commit 907bd8463bd62a3f6a445f52f9e27db3f1e63f50
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Oct 20 23:10:26 2021 -0300

    mwe por_causa_de

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 3bf9b49..b71b272 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -16599,10 +16599,10 @@
 25	asfalto	asfalto	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 26	recuperado	recuperar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	25	acl	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
-28	tendo	ter	AUX	_	VerbForm=Ger	30	aux	O	_
-29	em	em	ADP	_	_	30	mark	O	_
-30	vista	ver	VERB	_	VerbForm=Part	19	advcl	O	_
-31	variações	variação	NOUN	_	Gender=Fem|Number=Plur	30	obj	O	_
+28	tendo	ter	AUX	_	VerbForm=Ger	31	case	O	MWEPOS=ADP
+29	em	em	ADP	_	_	28	fixed	O	_
+30	vista	ver	VERB	_	VerbForm=Part	28	fixed	O	_
+31	variações	variação	NOUN	_	Gender=Fem|Number=Plur	19	obl	O	_
 32-33	na	_	_	_	_	_	_	_	_
 32	em	em	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
@@ -77999,12 +77999,12 @@
 
 # text = Por causa dessas propriedades e do seu baixo custo o aço passou a representar cerca de 90% de todos os metais consumidos pela civilização industrial.
 # sent_id = 161-20150810-MONOGRAFIA_0-244
-1	Por	por	ADP	_	_	2	case	O	_
-2	causa	causa	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
+1	Por	por	ADP	_	_	5	case	O	MWEPOS=ADP
+2	causa	causa	NOUN	_	Gender=Fem|Number=Sing	1	fixed	O	_
 3-4	dessas	_	_	_	_	_	_	_	_
-3	de	de	ADP	_	_	5	case	O	_
+3	de	de	ADP	_	_	1	fixed	O	_
 4	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	5	det	O	_
-5	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	2	nmod	O	_
+5	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	14	obl	O	_
 6	e	e	CCONJ	_	_	11	cc	O	_
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	11	case	O	_
@@ -83071,17 +83071,17 @@
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 8	estratégico	estratégico	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
-9	por	por	ADP	_	_	10	case	O	_
-10	causa	causa	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
+9	por	por	ADP	_	_	14	case	O	MWEPOS=ADP
+10	causa	causa	NOUN	_	Gender=Fem|Number=Sing	9	fixed	O	_
 11-12	da	_	_	_	_	_	_	_	_
-11	de	de	ADP	_	_	14	case	O	_
+11	de	de	ADP	_	_	9	fixed	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 13	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	14	det	O	_
-14	localização	localização	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
+14	localização	localização	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 15	intermediária	intermediário	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	entre	entre	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	terminal	terminal	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
+18	terminal	terminal	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 19-20	da	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
@@ -90119,12 +90119,12 @@
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
 30	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	26	obl	O	_
 31	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	30	acl	O	_
-32	por	por	ADP	_	_	33	case	O	_
-33	causa	causa	NOUN	_	Gender=Fem|Number=Sing	31	obl:agent	O	_
+32	por	por	ADP	_	_	36	case	O	MWEPOS=ADP
+33	causa	causa	NOUN	_	Gender=Fem|Number=Sing	32	fixed	O	_
 34-35	das	_	_	_	_	_	_	_	_
-34	de	de	ADP	_	_	36	case	O	_
+34	de	de	ADP	_	_	32	fixed	O	_
 35	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	36	det	O	_
-36	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	33	nmod	O	_
+36	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	26	obl	O	_
 37	de	de	ADP	_	_	38	case	O	_
 38	miscibilidade	miscibilidade	NOUN	_	Gender=Fem|Number=Sing	36	nmod	O	_
 39-40	do	_	_	_	_	_	_	_	_
@@ -91706,21 +91706,21 @@
 13	carboduto	carboduto	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	auxiliar	auxiliar	VERB	_	VerbForm=Inf	10	advcl	O	_
 15	,	,	PUNCT	_	_	14	punct	O	_
-16	por	por	ADP	_	_	17	case	O	_
-17	causa	causa	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
+16	por	por	ADP	_	_	20	case	O	MWEPOS=ADP
+17	causa	causa	NOUN	_	Gender=Fem|Number=Sing	16	fixed	O	_
 18-19	da	_	_	_	_	_	_	_	_
-18	de	de	ADP	_	_	20	case	O	_
+18	de	de	ADP	_	_	16	fixed	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	incerteza	incerteza	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+20	incerteza	incerteza	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 21-22	na	_	_	_	_	_	_	_	_
 21	em	em	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	quantificação	quantificação	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+23	quantificação	quantificação	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 26	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	28	nsubj:pass	O	_
 27	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	28	aux:pass	O	_
-28	reciclado	reciclar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	23	acl:relcl	O	_
+28	reciclado	reciclar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	25	acl:relcl	O	_
 29	em	em	ADP	_	_	31	case	O	_
 30	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	31	det	O	_
 31	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	28	obl	O	_
@@ -96004,16 +96004,16 @@
 13	carboduto	carboduto	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	auxiliar	auxiliar	VERB	_	VerbForm=Inf	10	advcl	O	_
 15	,	,	PUNCT	_	_	14	punct	O	_
-16	por	por	ADP	_	_	17	case	O	_
-17	causa	causa	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
+16	por	por	ADP	_	_	20	case	O	MWEPOS=ADP
+17	causa	causa	NOUN	_	Gender=Fem|Number=Sing	16	fixed	O	_
 18-19	da	_	_	_	_	_	_	_	_
-18	de	de	ADP	_	_	20	case	O	_
+18	de	de	ADP	_	_	16	fixed	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	incerteza	incerteza	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+20	incerteza	incerteza	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 21-22	na	_	_	_	_	_	_	_	_
 21	em	em	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	quantificação	quantificação	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+23	quantificação	quantificação	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 26	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	28	nsubj:pass	O	_
@@ -101607,11 +101607,11 @@
 
 # text = Tendo em vista os dados de Júpiter divulgados pela ANP a alternativa proposta neste trabalho visa o tratamento desse poço, produtor de gás, dividindo o processamento em três áreas: subsea, offshore (topside das FPSOs) e onshore.
 # sent_id = 334-20150521-MONOGRAFIA_0-111
-1	Tendo	ter	AUX	_	VerbForm=Ger	18	aux	O	_
-2	em	em	ADP	_	_	3	case	O	_
-3	vista	vista	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
+1	Tendo	ter	AUX	_	VerbForm=Ger	5	case	O	MWEPOS=ADP
+2	em	em	ADP	_	_	1	fixed	O	_
+3	vista	vista	NOUN	_	Gender=Fem|Number=Sing	1	fixed	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
-5	dados	dado	NOUN	_	Gender=Masc|Number=Plur	18	nsubj	O	_
+5	dados	dado	NOUN	_	Gender=Masc|Number=Plur	18	obl	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	Júpiter	Júpiter	PROPN	_	Gender=Masc|Number=Sing	5	nmod	O	_
 8	divulgados	divulgar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	5	acl	O	_
@@ -101645,7 +101645,7 @@
 33	três	três	NUM	_	NumType=Card	34	nummod	O	_
 34	áreas	área	NOUN	_	Gender=Fem|Number=Plur	29	obl	O	_
 35	:	:	PUNCT	_	_	36	punct	O	_
-36	subsea	subsea	ADJ	_	Gender=Fem|Number=Sing	34	amod	O	_
+36	subsea	subsea	NOUN	_	Gender=Fem|Number=Sing	34	appos	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
 38	offshore	offshore	NOUN	_	Gender=Fem|Number=Sing	36	conj	O	_
 39	(	(	PUNCT	_	_	40	punct	O	_
@@ -126532,11 +126532,11 @@
 19	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	cop	O	_
 20	altamente	altamente	ADV	_	_	21	advmod	O	_
 21	indesejável	indesejável	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
-22	tendo	ter	AUX	_	VerbForm=Ger	21	advcl	O	_
-23	em	em	ADP	_	_	24	case	O	_
-24	vista	vista	NOUN	_	Gender=Fem|Number=Sing	22	obl	O	_
+22	tendo	ter	AUX	_	VerbForm=Ger	26	case	O	MWEPOS=ADP
+23	em	em	ADP	_	_	22	fixed	O	_
+24	vista	vista	NOUN	_	Gender=Fem|Number=Sing	22	fixed	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
-26	incremento	incremento	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
+26	incremento	incremento	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 27-28	da	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
@@ -158751,15 +158751,15 @@
 18	teor	teor	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	metais	metal	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
-21	como	como	ADV	_	_	23	cc	O	_
+21	como	como	ADV	_	_	23	case	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
-23	Na	Na	PROPN	_	Gender=Fem|Number=Sing	20	conj	O	_
+23	Na	Na	PROPN	_	Gender=Fem|Number=Sing	20	nmod:appos	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
-25	K	K	PROPN	_	Gender=Masc|Number=Sing	20	conj	O	_
+25	K	K	PROPN	_	Gender=Masc|Number=Sing	23	conj	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
-27	Ca	Ca	PROPN	_	Gender=Masc|Number=Sing	20	conj	O	_
+27	Ca	Ca	PROPN	_	Gender=Masc|Number=Sing	23	conj	O	_
 28	e	e	CCONJ	_	_	29	cc	O	_
-29	Mg	Mg	PROPN	_	Gender=Masc|Number=Sing	20	conj	O	_
+29	Mg	Mg	PROPN	_	Gender=Masc|Number=Sing	23	conj	O	_
 30	,	,	PUNCT	_	_	36	punct	O	_
 31	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	32	det	O	_
 32	diferença	diferença	NOUN	_	Gender=Fem|Number=Sing	41	nsubj	O	_
@@ -158775,12 +158775,12 @@
 41	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	1	conj	O	_
 42	ser	ser	AUX	_	VerbForm=Inf	43	aux:pass	O	_
 43	explicados	explicar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	41	xcomp	O	_
-44	por	por	ADP	_	_	45	case	O	_
-45	causa	causa	NOUN	_	Gender=Fem|Number=Sing	43	obl:agent	O	_
+44	por	por	ADP	_	_	48	case	O	MWEPOS=ADP
+45	causa	causa	NOUN	_	Gender=Fem|Number=Sing	44	fixed	O	_
 46-47	dos	_	_	_	_	_	_	_	_
-46	de	de	ADP	_	_	48	case	O	_
+46	de	de	ADP	_	_	44	fixed	O	_
 47	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	48	det	O	_
-48	alimentos	alimento	NOUN	_	Gender=Masc|Number=Plur	45	nmod	O	_
+48	alimentos	alimento	NOUN	_	Gender=Masc|Number=Plur	43	obl	O	_
 49	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	51	nsubj:pass	O	_
 50	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	51	aux:pass	O	_
 51	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	48	acl:relcl	O	_
@@ -173693,20 +173693,20 @@
 9	uso	uso	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 10	como	como	ADP	_	_	11	case	O	_
 11	suportes	suporte	NOUN	_	Gender=Masc|Number=Plur	9	nmod	O	_
-12	,	,	PUNCT	_	_	14	punct	O	_
-13	por	por	ADP	_	_	14	case	O	_
-14	causa	causa	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
-15	de	de	ADP	_	_	16	case	O	_
-16	algumas	algum	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	14	nmod	O	_
+12	,	,	PUNCT	_	_	16	punct	O	_
+13	por	por	ADP	_	_	16	case	O	MWEPOS=ADP
+14	causa	causa	NOUN	_	Gender=Fem|Number=Sing	13	fixed	O	_
+15	de	de	ADP	_	_	13	fixed	O	_
+16	algumas	algum	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	5	obl	O	_
 17	de	de	ADP	_	_	19	case	O	_
 18	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	19	det	O	_
 19	características	característica	NOUN	_	Gender=Fem|Number=Plur	16	nmod	O	_
 20	,	,	PUNCT	_	_	25	punct	O	_
-21	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	25	cc	O	_
+21	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	25	case	O	MWEPOS=ADP
 22	como	como	ADV	_	_	21	fixed	O	_
 23	:	:	PUNCT	_	_	25	punct	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	obtenção	obtenção	NOUN	_	Gender=Fem|Number=Sing	3	appos	O	_
+25	obtenção	obtenção	NOUN	_	Gender=Fem|Number=Sing	19	nmod:appos	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	partículas	partícula	NOUN	_	Gender=Fem|Number=Plur	25	nmod	O	_
 28	de	de	ADP	_	_	29	case	O	_
@@ -181403,7 +181403,7 @@
 20-21	pelo	_	_	_	_	_	_	_	_
 20	por	por	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	pico	pico	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
+22	pico	pico	NOUN	_	Gender=Masc|Number=Sing	19	obl:agent	O	_
 23	com	com	ADP	_	_	24	case	O	_
 24	máximo	máximo	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
 25	em	em	ADP	_	_	27	case	O	_
@@ -181419,7 +181419,7 @@
 35	e	e	CCONJ	_	_	38	cc	O	_
 36	outros	outro	DET	_	Gender=Masc|Number=Plur|PronType=Ind	38	det	O	_
 37	dois	dois	NUM	_	NumType=Card	38	nummod	O	_
-38	picos	pico	NOUN	_	Gender=Masc|Number=Plur	19	conj	O	_
+38	picos	pico	NOUN	_	Gender=Masc|Number=Plur	22	conj	O	_
 39	largos	largo	ADJ	_	Gender=Masc|Number=Plur	38	amod	O	_
 40	com	com	ADP	_	_	41	case	O	_
 41	máximo	máximo	NOUN	_	Gender=Masc|Number=Sing	38	nmod	O	_
@@ -181450,12 +181450,12 @@
 66	,	,	PUNCT	_	_	68	punct	O	_
 67	com	com	ADP	_	_	68	case	O	_
 68	defeitos	defeito	NOUN	_	Gender=Masc|Number=Plur	61	obl	O	_
-69	por	por	ADP	_	_	70	case	O	_
-70	causa	causa	NOUN	_	Gender=Fem|Number=Sing	68	nmod	O	_
+69	por	por	ADP	_	_	73	case	O	MWEPOS=ADP
+70	causa	causa	NOUN	_	Gender=Fem|Number=Sing	69	fixed	O	_
 71-72	do	_	_	_	_	_	_	_	_
-71	de	de	ADP	_	_	73	case	O	_
+71	de	de	ADP	_	_	69	fixed	O	_
 72	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	73	det	O	_
-73	processo	processo	NOUN	_	Gender=Masc|Number=Sing	70	nmod	O	_
+73	processo	processo	NOUN	_	Gender=Masc|Number=Sing	68	nmod	O	_
 74	de	de	ADP	_	_	75	case	O	_
 75	tratamento	tratamento	NOUN	_	Gender=Masc|Number=Sing	73	nmod	O	_
 76	hidrotérmico	hidrotérmico	ADJ	_	Gender=Masc|Number=Sing	75	amod	O	_
@@ -249609,13 +249609,13 @@
 28	altamente	altamente	ADV	_	_	29	advmod	O	_
 29	sobrebalanceada	sobrebalancear	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	27	acl	O	_
 30	,	,	PUNCT	_	_	29	punct	O	_
-31	por	por	ADP	_	_	32	case	O	_
-32	causa	causa	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
+31	por	por	ADP	_	_	36	case	O	MWEPOS=ADP
+32	causa	causa	NOUN	_	Gender=Fem|Number=Sing	31	fixed	O	_
 33-34	das	_	_	_	_	_	_	_	_
-33	de	de	ADP	_	_	36	case	O	_
+33	de	de	ADP	_	_	31	fixed	O	_
 34	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	36	det	O	_
 35	grandes	grande	ADJ	_	Gender=Fem|Number=Plur	36	amod	O	_
-36	gargantas	garganta	NOUN	_	Gender=Fem|Number=Plur	32	nmod	O	_
+36	gargantas	garganta	NOUN	_	Gender=Fem|Number=Plur	6	obl	O	_
 37	de	de	ADP	_	_	38	case	O	_
 38	poro	poro	NOUN	_	Gender=Masc|Number=Sing	36	nmod	O	_
 39	e	e	CCONJ	_	_	42	cc	O	_
@@ -277212,7 +277212,7 @@
 8	mar	mar	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	diversas	diverso	DET	_	Gender=Fem|Number=Plur|PronType=Ind	11	det	O	_
-11	formas	forma	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
+11	formas	forma	NOUN	_	Gender=Fem|Number=Plur	5	obl	O	_
 12	,	,	PUNCT	_	_	14	punct	O	_
 13	como	como	ADP	_	_	14	case	O	_
 14	acidentes	acidente	NOUN	_	Gender=Masc|Number=Plur	11	nmod:appos	O	_
@@ -277227,7 +277227,7 @@
 22	,	,	PUNCT	_	_	14	punct	O	_
 23	durante	durante	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	lavagem	lavagem	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
+25	lavagem	lavagem	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
 26-27	dos	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	28	det	O	_
@@ -277253,15 +277253,15 @@
 44	tanques	tanque	NOUN	_	Gender=Masc|Number=Plur	41	obl:arg	O	_
 45	de	de	ADP	_	_	46	case	O	_
 46	refino	refino	NOUN	_	Gender=Masc|Number=Sing	44	nmod	O	_
-47	ou	ou	CCONJ	_	_	49	cc	O	_
-48	por	por	ADP	_	_	49	case	O	_
-49	causa	causa	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
-50	de	de	ADP	_	_	51	case	O	_
-51	vazamentos	vazamento	NOUN	_	Gender=Masc|Number=Plur	49	nmod	O	_
+47	ou	ou	CCONJ	_	_	51	cc	O	_
+48	por	por	ADP	_	_	51	case	O	MWEPOS=ADP
+49	causa	causa	NOUN	_	Gender=Fem|Number=Sing	48	fixed	O	_
+50	de	de	ADP	_	_	48	fixed	O	_
+51	vazamentos	vazamento	NOUN	_	Gender=Masc|Number=Plur	14	conj	O	_
 52-53	nas	_	_	_	_	_	_	_	_
 52	em	em	ADP	_	_	54	case	O	_
 53	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	54	det	O	_
-54	zonas	zona	NOUN	_	Gender=Fem|Number=Plur	49	nmod	O	_
+54	zonas	zona	NOUN	_	Gender=Fem|Number=Plur	51	nmod	O	_
 55	de	de	ADP	_	_	56	case	O	_
 56	extração	extração	NOUN	_	Gender=Fem|Number=Sing	54	nmod	O	_
 57	(	(	PUNCT	_	_	58	punct	O	_
