commit b4593ae3ca54af6ecaeb898481c8df332d902f84
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 8 01:29:09 2021 -0300

    alem_de

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b6ea396..763545f 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -139,7 +139,7 @@
 35	e	e	CCONJ	_	_	36	cc	O	_
 36	lístricas	lístrico	ADJ	_	Gender=Fem|Number=Plur	32	conj	O	_
 37	,	,	PUNCT	_	_	41	punct	O	_
-38	além	além	ADV	_	_	41	cc	O	_
+38	além	além	ADV	_	_	41	cc	O	MWEPOS=CCONJ
 39-40	da	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	38	fixed	O	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
@@ -2914,7 +2914,7 @@
 25	mesma	mesmo	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 26	idade	idade	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
-28	além	além	ADV	_	_	30	cc	O	_
+28	além	além	ADV	_	_	30	cc	O	MWEPOS=CCONJ
 29	de	de	ADP	_	_	28	fixed	O	_
 30	restos	resto	NOUN	_	Gender=Masc|Number=Plur	12	conj	O	_
 31-32	do	_	_	_	_	_	_	_	_
@@ -5983,7 +5983,7 @@
 24	Cassino	Cassino	PROPN	_	Gender=Masc|Number=Sing	22	conj	O	_
 25	respectivamente	respectivamente	ADV	_	_	29	advmod	O	_
 26	,	,	PUNCT	_	_	29	punct	O	_
-27	além	além	ADV	_	_	29	cc	O	_
+27	além	além	ADV	_	_	29	cc	O	MWEPOS=CCONJ
 28	de	de	ADP	_	_	27	fixed	O	_
 29	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	13	conj	B=LITOLOGIA|B=LOCAL:GEO	_
 30	antitéticas	antitético	ADJ	_	Gender=Fem|Number=Plur	29	amod	O	_
@@ -6782,7 +6782,7 @@
 31	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	29	appos	O	_
 32	14	14	PROPN	_	Number=Sing	31	flat:name	O	_
 33	)	)	PUNCT	_	_	31	punct	O	_
-34	além	além	ADV	_	_	36	cc	O	_
+34	além	além	ADV	_	_	36	cc	O	MWEPOS=CCONJ
 35	de	de	ADP	_	_	34	fixed	O	_
 36	direções	direção	NOUN	_	Gender=Fem|Number=Plur	11	conj	O	_
 37	ENE	ENE	PROPN	_	Gender=Fem|Number=Sing	36	nmod	O	_
@@ -7401,7 +7401,7 @@
 17	NW-SE	NW-SE	PROPN	_	Gender=Fem|Number=Sing	15	nmod	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
 19	NE-SW	NE-SW	PROPN	_	Gender=Fem|Number=Sing	17	conj	O	_
-20	além	além	ADV	_	_	22	cc	O	_
+20	além	além	ADV	_	_	22	cc	O	MWEPOS=CCONJ
 21	de	de	ADP	_	_	20	fixed	O	_
 22	direções	direção	NOUN	_	Gender=Fem|Number=Plur	15	conj	O	_
 23	WNW	WNW	PROPN	_	Gender=Fem|Number=Sing	22	nmod	O	_
@@ -7451,12 +7451,12 @@
 
 # text = Além destes dois feixes a direção NW-SE está correlacionada ao Lineamento Tibagi..
 # sent_id = 247-20140910-MONOGRAFIA_0-210
-1	Além	além	ADV	_	_	5	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 2-3	destes	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	5	det	O	_
 4	dois	dois	NUM	_	NumType=Card	5	nummod	O	_
-5	feixes	feixe	NOUN	_	Gender=Masc|Number=Plur	10	obl	O	_
+5	feixes	feixe	NOUN	_	Gender=Masc|Number=Plur	7	conj	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	direção	direção	NOUN	_	Gender=Fem|Number=Sing	10	nsubj	O	_
 8	NW-SE	NW-SE	PROPN	_	Gender=Fem|Number=Sing	7	nmod	O	_
@@ -8498,7 +8498,7 @@
 36	feições	feição	NOUN	_	Gender=Fem|Number=Plur	26	acl:relcl	O	_
 37	estratigráficas	estratigráfico	ADJ	_	Gender=Fem|Number=Plur	36	amod	O	_
 38	,	,	PUNCT	_	_	41	punct	O	_
-39	além	além	ADV	_	_	41	cc	O	_
+39	além	além	ADV	_	_	41	cc	O	MWEPOS=CCONJ
 40	de	de	ADP	_	_	39	fixed	O	_
 41	feições	feição	NOUN	_	Gender=Fem|Number=Plur	36	conj	O	_
 42	vulcânicas	vulcânico	ADJ	_	Gender=Fem|Number=Plur	41	amod	B=LITOLOGIA	_
@@ -10467,10 +10467,10 @@
 23	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	20	obl	B=INSTRUMENTO	_
 24	aeromagnetométrico	aeromagnetométrico	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 25	,	,	PUNCT	_	_	28	punct	O	_
-26	além	além	ADV	_	_	38	cc	O	_
+26	além	além	ADV	_	_	38	advmod	O	MWEPOS=ADV
 27-28	disso	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	26	fixed	O	_
-28	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	38	advcl	O	_
+28	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	26	fixed	O	_
 29	,	,	PUNCT	_	_	28	punct	O	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
 31	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	32	det	O	_
@@ -10718,11 +10718,11 @@
 
 # text = Além da projeção do Lineamento Tibagi, foi constatada a presença de feições vulcânicas próximo à Plataforma de Florianópolis, estas feições afetam os horizontes do Rifte, Albiano e com menor frequência os horizontes do Cretáceo Superior..
 # sent_id = 247-20140910-MONOGRAFIA_0-298
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	projeção	projeção	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
+4	projeção	projeção	NOUN	_	Gender=Fem|Number=Sing	13	conj	O	_
 5-6	do	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -10876,7 +10876,7 @@
 9	e	e	CCONJ	_	_	10	cc	O	_
 10	antitéticas	antitético	ADJ	_	Gender=Fem|Number=Plur	6	conj	O	_
 11	,	,	PUNCT	_	_	14	punct	O	_
-12	além	além	ADV	_	_	14	cc	O	_
+12	além	além	ADV	_	_	14	cc	O	MWEPOS=CCONJ
 13	de	de	ADP	_	_	12	fixed	O	_
 14	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	5	conj	B=LITOLOGIA|B=LOCAL:GEO	_
 15	normais	normal	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
@@ -12101,11 +12101,11 @@
 
 # text = Além dos componentes básicos que constituem a emulsão asfáltica (asfalto, água e agente emulsificante), em algumas situações a emulsão pode conter aditivos, tais como: estabilizantes, agentes melhoradores de adesão e controladores de quebra (AEMA, 2007)..
 # sent_id = 241-20140227-MONOGRAFIA_0-48
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	dos	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	25	obl	O	_
+4	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	27	conj	O	_
 5	básicos	básico	ADJ	_	Gender=Masc|Number=Plur	4	amod	O	_
 6	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	7	nsubj	O	_
 7	constituem	constituir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	acl:relcl	O	_
@@ -12542,10 +12542,10 @@
 
 # text = Além disso, o asfalto precisa ser disperso por ação mecânica que o transforme em pequenas partículas ou glóbulos (IBP, 1999; Hunter, 2000; ABEDA, 2001; Shell, 2003)..
 # sent_id = 241-20140227-MONOGRAFIA_0-60
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	asfalto	asfalto	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
@@ -21111,10 +21111,10 @@
 
 # text = Além disso, painéis fotográficos foram montados de modo a obter imagens mais abrangentes, ilustrando relações laterais e verticais..
 # sent_id = 284-20150917-MONOGRAFIA_0-62
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	8	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	8	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	painéis	painel	NOUN	_	Gender=Masc|Number=Plur	8	nsubj:pass	O	_
 6	fotográficos	fotográfico	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
@@ -24361,10 +24361,10 @@
 
 # text = Além disto, diversos autores afirmam que a ingressão marinha na Bacia do Araripe provém de nordeste pela Bacia do Parnaíba (Arai 2014) enquanto que outros contestam, afirmando ingressão por sudeste (Beurlen 1971)..
 # sent_id = 284-20150917-MONOGRAFIA_0-150
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disto	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	diversos	diverso	DET	_	Gender=Masc|Number=Plur|PronType=Ind	6	det	O	_
 6	autores	autor	NOUN	_	Gender=Masc|Number=Plur	7	nsubj	O	_
@@ -25625,7 +25625,7 @@
 56	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	54	obl	B=LITOLOGIA	_
 57	proximais	proximal	ADJ	_	Gender=Masc|Number=Plur	56	amod	O	_
 58	,	,	PUNCT	_	_	61	punct	O	_
-59	além	além	ADV	_	_	61	cc	O	_
+59	além	além	ADV	_	_	61	cc	O	MWEPOS=CCONJ
 60	de	de	ADP	_	_	59	fixed	O	_
 61	apresentar	apresentar	VERB	_	VerbForm=Inf	54	conj	O	_
 62	diminuição	diminuição	NOUN	_	Gender=Fem|Number=Sing	61	obj	O	_
@@ -25731,10 +25731,10 @@
 34	tornando	tornar	VERB	_	VerbForm=Ger	20	advcl	O	_
 35	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	34	expl	O	_
 36	,	,	PUNCT	_	_	34	punct	O	_
-37	além	além	ADV	_	_	34	cc	O	_
+37	além	além	ADV	_	_	34	advmod	O	MWEPOS=ADV
 38-39	disto	_	_	_	_	_	_	_	_
 38	de	de	ADP	_	_	37	fixed	O	_
-39	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	37	obl	O	_
+39	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	37	fixed	O	_
 40	,	,	PUNCT	_	_	34	punct	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
 42	mais	mais	ADV	_	_	43	advmod	O	_
@@ -29915,10 +29915,10 @@
 
 # text = Além disto, este autor afirma que, em direção ao topo, a partir da SIM, voltam a predominar condições continentais, dedutíveis pelo padrão de engrossamento textural ascendente e pela presença de fósseis não-marinhos..
 # sent_id = 284-20150917-MONOGRAFIA_0-330
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disto	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	6	det	O	_
 6	autor	autor	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
@@ -32417,11 +32417,11 @@
 
 # text = Além dos dinoflagelados, são encontrados em abundância restos de fósseis de peixes (Beurlen 1963), situados no interior de concreções calcárias, como Rhacolepis, Leptolepis, Aspidorhynchidae, Elopidae e Ichtyodectidae..
 # sent_id = 284-20150917-MONOGRAFIA_0-397
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	dos	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	dinoflagelados	dinoflagelado	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
+4	dinoflagelados	dinoflagelado	NOUN	_	Gender=Masc|Number=Plur	10	conj	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	O	_
 7	encontrados	encontrar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -34920,7 +34920,7 @@
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
 31	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	25	conj	O	_
 32	,	,	PUNCT	_	_	35	punct	O	_
-33	além	além	ADV	_	_	35	cc	O	_
+33	além	além	ADV	_	_	35	cc	O	MWEPOS=CCONJ
 34	de	de	ADP	_	_	33	fixed	O	_
 35	outras	outro	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	13	conj	O	_
 36	.	.	PUNCT	_	_	13	punct	O	_
@@ -59183,11 +59183,11 @@
 2	magnésio	magnésio	NOUN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 3	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
-5	além	além	ADV	_	_	8	cc	O	MWE=além_de|MWEPOS=CCONJ
+5	além	além	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6-7	da	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	5	fixed	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	substituição	substituição	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
+8	substituição	substituição	NOUN	_	Gender=Fem|Number=Sing	16	conj	O	_
 9	isomórfica	isomórfico	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 10	,	,	PUNCT	_	_	12	punct	O	_
 11	estar	estar	AUX	_	VerbForm=Inf	12	cop	O	_
@@ -61815,10 +61815,10 @@
 
 # text = Além disso, observando-se os resultados descritos na Tabela V.3, nota-se que a argila que teve a maior recuperação usando-se o poli (cloreto de dialildimetilamônio) como inibidor, foi a argila A4..
 # sent_id = 24-20150122-MONOGRAFIA_0-558
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	15	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	15	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
 5-6	observando-se	_	_	_	_	_	_	_	_
 5	observando	observar	VERB	_	VerbForm=Ger	39	advcl	O	_
@@ -69760,11 +69760,11 @@
 
 # text = Além da associação de fácies semelhante, a ausência de sedimentos pelíticos no registro sedimentar e a baixa dispersão das medidas de paleocorrentes também são indicativas de sedimentação fluvial do tipo entrelaçado para a porção inferior da Seção Córrego Tiririca..
 # sent_id = 283-20150917-MONOGRAFIA_0-186
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	associação	associação	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
+4	associação	associação	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	4	nmod	B=LITOLOGIA	_
 7	semelhante	semelhante	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
@@ -75446,11 +75446,11 @@
 
 # text = Além do quê, este sistema eólico ocorre interdigitado à fácies de sabkas, sugerindo maior umidade em todo o sistema..
 # sent_id = 283-20150917-MONOGRAFIA_0-403
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	9	advmod	O	MWEPOS=ADV
 2-3	do	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	quê	quê	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
+3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	1	fixed	O	_
+4	quê	quê	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	7	det	O	_
 7	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	9	nsubj	O	_
@@ -75489,7 +75489,7 @@
 10	de	de	ADP	_	_	11	case	O	_
 11	pelitos	pelito	NOUN	_	Gender=Masc|Number=Plur	9	nmod	B=LITOLOGIA	_
 12	,	,	PUNCT	_	_	7	punct	O	_
-13	além	além	ADV	_	_	23	cc	O	_
+13	além	além	ADV	_	_	23	cc	O	MWEPOS=CCONJ
 14-15	do	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	13	fixed	O	_
 15	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	13	fixed	O	_
@@ -77196,11 +77196,11 @@
 3	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	fornecer	fornecer	VERB	_	VerbForm=Inf	3	xcomp	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
-6	além	além	ADV	_	_	9	cc	O	MWE=além_de|MWEPOS=CCONJ
+6	além	além	ADV	_	_	9	cc	O	MWEPOS=CCONJ
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	6	fixed	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	rumo	rumo	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
+9	rumo	rumo	NOUN	_	Gender=Masc|Number=Sing	14	conj	O	_
 10-11	dos	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
@@ -78482,9 +78482,9 @@
 2	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	26	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	CCGS	CCGS	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-5	além	além	ADV	_	_	7	cc	O	_
+5	além	além	ADV	_	_	7	cc	O	MWEPOS=CCONJ
 6	de	de	ADP	_	_	5	fixed	O	_
-7	reduzir	reduzir	VERB	_	VerbForm=Inf	26	advcl	O	_
+7	reduzir	reduzir	VERB	_	VerbForm=Inf	27	conj	O	_
 8	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	9	det	O	_
 9	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
 10	de	de	ADP	_	_	11	case	O	_
@@ -78540,9 +78540,9 @@
 
 # text = Além de apresentar benefícios ambientais para as empresas, minimizando os impactos das emissões de GEE, e benefícios econômicos, através da EOR, o CCS também poderá gerar as Reduções Certificadas de Emissões (RCE) através do MDL, podendo essas reduções certificadas serem comercializadas em mercados de carbono ou mantidas pelas empresas participantes do projeto..
 # sent_id = 161-20150810-MONOGRAFIA_0-14
-1	Além	além	ADV	_	_	3	mark	O	_
+1	Além	além	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2	de	de	ADP	_	_	1	fixed	O	_
-3	apresentar	apresentar	VERB	_	VerbForm=Inf	31	advcl	O	_
+3	apresentar	apresentar	VERB	_	VerbForm=Inf	32	conj	O	_
 4	benefícios	benefício	NOUN	_	Gender=Masc|Number=Plur	3	obj	O	_
 5	ambientais	ambiental	ADJ	_	Gender=Masc|Number=Plur	4	amod	O	_
 6	para	para	ADP	_	_	8	case	O	_
@@ -83273,7 +83273,7 @@
 21	e	e	CCONJ	_	_	22	cc	O	_
 22	gás	gás	NOUN	_	Gender=Masc|Number=Sing	20	conj	B=FLUIDO	_
 23	,	,	PUNCT	_	_	26	punct	O	_
-24	além	além	ADV	_	_	26	cc	O	_
+24	além	além	ADV	_	_	26	cc	O	MWEPOS=CCONJ
 25	de	de	ADP	_	_	24	fixed	O	_
 26	utilizarem	utilizar	VERB	_	Number=Plur|Person=3|VerbForm=Inf	10	conj	O	_
 27	modelos	modelo	NOUN	_	Gender=Masc|Number=Plur	26	obj	O	_
@@ -84017,10 +84017,10 @@
 
 # text = Além disso, a rocha no ponto de injeção deve ser porosa e permeável para que o CO2 penetre com facilidade..
 # sent_id = 161-20150810-MONOGRAFIA_0-198
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	12	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	12	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
 6	rocha	rocha	NOUN	_	Gender=Fem|Number=Sing	12	nsubj	B=LITOLOGIA	_
@@ -86637,10 +86637,10 @@
 
 # text = Além disso, em função dos produtos que preponderam em suas linhas de produção, as usinas também podem ser assim classificadas:
 # sent_id = 161-20150810-MONOGRAFIA_0-288
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	21	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	21	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	em	em	ADP	_	_	6	case	O	_
 6	função	função	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
@@ -87685,11 +87685,11 @@
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
 12	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	9	nmod	B=QUÍMICA	_
 13	,	,	PUNCT	_	_	17	punct	O	_
-14	além	além	ADV	_	_	17	cc	O	_
+14	além	além	ADV	_	_	17	cc	O	MWEPOS=CCONJ
 15-16	da	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	14	fixed	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
-17	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	24	conj	O	_
+17	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	2	conj	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	solventes	solvente	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
 20	físicos	físico	ADJ	_	Gender=Masc|Number=Plur	19	amod	O	_
@@ -92175,7 +92175,7 @@
 30	baixas	baixo	ADJ	_	Gender=Fem|Number=Plur	31	amod	O	_
 31	temperaturas	temperatura	NOUN	_	Gender=Fem|Number=Plur	28	conj	O	_
 32	,	,	PUNCT	_	_	36	punct	O	_
-33	além	além	ADV	_	_	36	cc	O	_
+33	além	além	ADV	_	_	36	cc	O	MWEPOS=CCONJ
 34-35	da	_	_	_	_	_	_	_	_
 34	de	de	ADP	_	_	33	fixed	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
@@ -102297,10 +102297,10 @@
 
 # text = Além disso, o engajamento deste setor é crucial para a efetividade das ações de mitigação..
 # sent_id = 161-20150810-MONOGRAFIA_0-734
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	11	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	11	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	engajamento	engajamento	NOUN	_	Gender=Masc|Number=Sing	11	nsubj	O	_
@@ -108679,7 +108679,7 @@
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
 12	pré-sal	pré-sal	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 13	,	,	PUNCT	_	_	16	punct	O	_
-14	além	além	ADV	_	_	16	cc	O	_
+14	além	além	ADV	_	_	16	cc	O	MWEPOS=CCONJ
 15-16	daquelas	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	14	fixed	O	_
 16	aquelas	aquele	PRON	_	Gender=Fem|Number=Plur|PronType=Dem	1	conj	O	_
@@ -109620,11 +109620,11 @@
 
 # text = Além da Petrobras, outras empresas estão envolvidas na exploração da camada do Pré-sal, entre elas Royal Dutch Shell, ExxonMobil, BG Group, HessCorp, Galp, Repsol e Petrogal..
 # sent_id = 334-20150521-MONOGRAFIA_0-34
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	Petrobras	Petrobras	PROPN	_	Gender=Fem|Number=Sing	9	obl	O	_
+4	Petrobras	Petrobras	PROPN	_	Gender=Fem|Number=Sing	7	conj	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	outras	outro	DET	_	Gender=Fem|Number=Plur|PronType=Ind	7	det	O	_
 7	empresas	empresa	NOUN	_	Gender=Fem|Number=Plur	9	nsubj	O	_
@@ -110332,7 +110332,7 @@
 
 # text = Dentre os pontos citados no diagrama da Figura 5, podem-se destacar os seguintes desafios (FERRO e TEIXEIRA, 2009):
 # sent_id = 334-20150521-MONOGRAFIA_0-47
-1	Dentre	dentre	ADP	_	_	3	case	O	_
+1	Dentre	dentre	ADP	_	_	3	case	O	sentenciação
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	3	det	O	_
 3	pontos	ponto	NOUN	_	Gender=Masc|Number=Plur	13	obl	O	_
 4	citados	citar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	acl	O	_
@@ -113546,7 +113546,7 @@
 36	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	37	det	O	_
 37	FPSO	FPSO	PROPN	_	Gender=Fem|Number=Sing	34	obl	O	_
 38	)	)	PUNCT	_	_	29	punct	O	_
-39	além	além	ADV	_	_	42	cc	O	_
+39	além	além	ADV	_	_	42	cc	O	MWEPOS=CCONJ
 40	de	de	ADP	_	_	39	fixed	O	_
 41	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	42	nsubj	O	_
 42	substituir	substituir	VERB	_	VerbForm=Inf	18	conj	O	_
@@ -115311,11 +115311,11 @@
 # text = Pois além da complexidade trazida por tantas operações ao topside e ao elevado custo, uma das principais dificuldades é a manipulação dos componentes intermediários (C4-C5) que podem se enquadrar tanto na especificação de pressão de vapor do produto quanto na especificação do ponto de orvalho dos hidrocarbonetos (BOTHAMLEY et al, 2004)..
 # sent_id = 334-20150521-MONOGRAFIA_0-172
 1	Pois	pois	SCONJ	_	_	26	mark	O	_
-2	além	além	ADV	_	_	5	advmod	O	_
+2	além	além	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 3-4	da	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	2	fixed	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	complexidade	complexidade	NOUN	_	Gender=Fem|Number=Sing	26	obl	O	_
+5	complexidade	complexidade	NOUN	_	Gender=Fem|Number=Sing	26	conj	O	_
 6	trazida	trazer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	5	acl	O	_
 7	por	por	ADP	_	_	9	case	O	_
 8	tantas	tanto	DET	_	Gender=Fem|Number=Plur|PronType=Ind	9	det	O	_
@@ -119098,11 +119098,11 @@
 
 # text = Além do fato da importância da separação estar voltada para o interesse econômico nas frações constituídas por hidrocarbonetos (óleo e gás), a separação da mistura trifásica é muito importante sob o ponto de vista da fase gás, uma vez que a presença de água no duto poderá promover a formação de hidratos, produzindo depósitos que reduzem o diâmetro da tubulação..
 # sent_id = 334-20150521-MONOGRAFIA_0-271
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	do	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	fato	fato	NOUN	_	Gender=Masc|Number=Sing	37	obl	O	_
+4	fato	fato	NOUN	_	Gender=Masc|Number=Sing	30	conj	O	_
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	SCONJ	_	_	12	mark	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -119248,11 +119248,11 @@
 
 # text = Além da planta de processamento primário, as plataformas ainda possuem diversos sistemas (entre eles sistema de recuperação secundária e sistema de injeção de água do mar e/ou produzida) e redes de apoio, como sistemas de alívio, drenagem, ar comprimido, diesel, resfriamento e aquecimento (VAZ, 2009)..
 # sent_id = 334-20150521-MONOGRAFIA_0-273
-1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	planta	planta	NOUN	_	Gender=Fem|Number=Sing	12	obl	O	_
+4	planta	planta	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	processamento	processamento	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 7	primário	primário	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
@@ -124793,11 +124793,11 @@
 
 # text = Além dos inibidores, outros métodos podem ser citados, como controle de pressão, controle de temperatura e remoção de água/desidratação..
 # sent_id = 334-20150521-MONOGRAFIA_0-413
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	dos	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	10	obl	O	_
+4	inibidores	inibidor	NOUN	_	Gender=Masc|Number=Plur	7	conj	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	outros	outro	DET	_	Gender=Masc|Number=Plur|PronType=Ind	7	det	O	_
 7	métodos	método	NOUN	_	Gender=Masc|Number=Plur	8	nsubj	B=MÉTODO	_
@@ -124994,7 +124994,7 @@
 16	de	de	ADP	_	_	17	case	O	_
 17	forma	forma	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
 18	permanente	permanente	ADJ	_	Gender=Fem|Number=Sing	17	amod	O	_
-19	além	além	ADV	_	_	21	cc	O	_
+19	além	além	ADV	_	_	21	cc	O	MWEPOS=CCONJ
 20	de	de	ADP	_	_	19	fixed	O	_
 21	garantir	garantir	VERB	_	VerbForm=Inf	8	conj	O	_
 22	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
@@ -126674,11 +126674,11 @@
 
 # text = Além da função de anti-aglomerantes muitos desses aditivos exercem também outras funções na formulação dos fluidos, sendo alguns deles utilizados até como encapsuladores de argilas..
 # sent_id = 334-20150521-MONOGRAFIA_0-470
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	função	função	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
+4	função	função	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	anti-aglomerantes	anti-aglomerante	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 7	muitos	muito	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	6	nmod	O	_
@@ -128244,11 +128244,11 @@
 21	,	,	PUNCT	_	_	17	punct	O	_
 22	já	já	ADV	_	_	32	mark	O	MWEPOS=SCONJ
 23	que	que	SCONJ	_	_	22	fixed	O	_
-24	além	além	ADV	_	_	27	cc	O	_
+24	além	além	ADV	_	_	27	cc	O	MWEPOS=CCONJ
 25-26	do	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	24	fixed	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	custo	custo	NOUN	_	Gender=Masc|Number=Sing	32	conj	O	_
+27	custo	custo	NOUN	_	Gender=Masc|Number=Sing	33	conj	O	_
 28-29	das	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
@@ -128452,11 +128452,11 @@
 
 # text = Além do arranjo simples com apenas uma membrana mostrado na Figura 43, outros arranjos são possíveis dependendo da vazão de gás, concentração de CO2 e do grau de pureza requerido..
 # sent_id = 334-20150521-MONOGRAFIA_0-524
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	do	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	arranjo	arranjo	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
+4	arranjo	arranjo	NOUN	_	Gender=Masc|Number=Sing	17	conj	O	_
 5	simples	simples	ADJ	_	Gender=Masc|Number=Sing	4	amod	O	_
 6	com	com	ADP	_	_	9	case	O	_
 7	apenas	apenas	ADV	_	_	9	advmod	O	_
@@ -129144,7 +129144,7 @@
 
 # text = A recuperação secundária se refere à utilização de injeção de água ou gás imiscível para aumentar a energia do reservatório. • Recuperação Avançada: Faz uso de métodos especiais de recuperação que visam aumentar o fator de recuperação de óleo, além do que seria recuperado somente pelos métodos de recuperação primária e secundária..
 # sent_id = 334-20150521-MONOGRAFIA_0-541
-1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
+1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	sentenciação
 2	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
 3	secundária	secundário	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	5	expl	O	_
@@ -132967,7 +132967,7 @@
 21	estado	estado	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 22	líquido	líquido	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
 23	,	,	PUNCT	_	_	27	punct	O	_
-24	além	além	ADV	_	_	27	cc	O	_
+24	além	além	ADV	_	_	27	cc	O	MWEPOS=CCONJ
 25-26	da	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	24	fixed	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
@@ -133065,11 +133065,11 @@
 6	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	O	_
 7	geradas	gerar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 8	,	,	PUNCT	_	_	7	punct	O	_
-9	além	além	ADV	_	_	12	cc	O	_
+9	além	além	ADV	_	_	12	cc	O	MWEPOS=CCONJ
 10-11	da	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	9	fixed	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
+12	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	18	conj	O	_
 13	rica	rico	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 14	em	em	ADP	_	_	15	case	O	_
 15	MEG	MEG	PROPN	_	Gender=Fem|Number=Sing	12	nmod	O	_
@@ -133858,11 +133858,11 @@
 5	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	aux:pass	O	_
 6	mostradas	mostrar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 7	,	,	PUNCT	_	_	6	punct	O	_
-8	além	além	ADV	_	_	11	cc	O	_
+8	além	além	ADV	_	_	11	cc	O	MWEPOS=CCONJ
 9-10	das	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	8	fixed	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
-11	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	6	obl	O	_
+11	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	21	conj	O	_
 12-13	das	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	14	case	O	_
 13	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	14	det	O	_
@@ -136631,9 +136631,9 @@
 4	reinjeção	reinjeção	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	4	nmod	B=QUÍMICA	_
-7	além	além	ADV	_	_	9	cc	O	_
+7	além	além	ADV	_	_	9	cc	O	MWEPOS=CCONJ
 8	de	de	ADP	_	_	7	fixed	O	_
-9	aumentar	aumentar	VERB	_	VerbForm=Inf	2	conj	O	_
+9	aumentar	aumentar	VERB	_	VerbForm=Inf	23	conj	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -137067,7 +137067,7 @@
 30	além	além	ADV	_	_	33	cc	O	_
 31	de	de	ADP	_	_	30	fixed	O	_
 32	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	33	det	O	_
-33	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	21	conj	O	_
+33	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	11	conj	O	_
 34	inorgânicos	inorgânico	NOUN	_	Gender=Masc|Number=Plur	33	nmod	O	_
 35	sólidos	sólido	ADJ	_	Gender=Masc|Number=Plur	33	amod	O	_
 36	,	,	PUNCT	_	_	38	punct	O	_
@@ -137180,10 +137180,10 @@
 44	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	42	nmod	B=FLUIDO	_
 45	convencionais	convencional	ADJ	_	Gender=Masc|Number=Plur	44	amod	O	_
 46	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	52	nsubj	O	_
-47	além	além	ADV	_	_	50	cc	O	_
+47	além	além	ADV	_	_	50	cc	O	MWEPOS=CCONJ
 48	de	de	ADP	_	_	47	fixed	O	_
 49	menos	menos	ADV	_	_	50	advmod	O	_
-50	viscosos	viscoso	NOUN	_	Gender=Masc|Number=Plur	52	obl	O	_
+50	viscosos	viscoso	NOUN	_	Gender=Masc|Number=Plur	52	conj	O	_
 51	,	,	PUNCT	_	_	50	punct	O	_
 52	fornecem	fornecer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	42	acl:relcl	O	_
 53	,	,	PUNCT	_	_	52	punct	O	_
@@ -138509,10 +138509,10 @@
 29	estabilidade	estabilidade	NOUN	_	Gender=Fem|Number=Sing	28	obj	O	_
 30	termodinâmica	termodinâmica	ADJ	_	Gender=Fem|Number=Sing	29	amod	O	_
 31	,	,	PUNCT	_	_	35	punct	O	_
-32	além	além	ADV	_	_	35	cc	O	_
+32	além	além	ADV	_	_	35	cc	O	MWEPOS=CCONJ
 33	de	de	ADP	_	_	32	fixed	O	_
 34	serem	ser	AUX	_	Number=Plur|Person=3|VerbForm=Inf	35	aux:pass	O	_
-35	formadas	formar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	23	conj	O	_
+35	formadas	formar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	28	conj	O	_
 36	espontaneamente	espontaneamente	ADV	_	_	35	advmod	O	_
 37	,	,	PUNCT	_	_	35	punct	O	_
 38	porém	porém	CCONJ	_	_	35	cc	O	_
@@ -138984,10 +138984,10 @@
 
 # text = Além disso, é importante atuar na redução da tensão interfacial para que se possa diminuir o tamanho das gotículas sem que seja necessária a aplicação de quantidade muito grande de energia..
 # sent_id = 202-20140404-MONOGRAFIA_0-48
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	6	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	6	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	cop	O	_
 6	importante	importante	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
@@ -139718,7 +139718,7 @@
 31	e	e	CCONJ	_	_	32	cc	O	_
 32	não-iônicos	não-iônico	ADJ	_	Gender=Masc|Number=Plur	28	conj	O	_
 33	,	,	PUNCT	_	_	25	punct	O	_
-34	além	além	ADV	_	_	37	cc	O	_
+34	além	além	ADV	_	_	37	cc	O	MWEPOS=CCONJ
 35-36	dos	_	_	_	_	_	_	_	_
 35	de	de	ADP	_	_	34	fixed	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
@@ -140354,10 +140354,10 @@
 
 # text = Além disso, como se vê na Tabela 2, o conceito de HLB pode ser usado como uma primeira seleção de tensoativos para uma determinada aplicação..
 # sent_id = 202-20140404-MONOGRAFIA_0-92
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	17	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	17	obl	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	como	como	SCONJ	_	_	7	mark	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	7	expl	O	_
@@ -142119,11 +142119,11 @@
 # sent_id = 202-20140404-MONOGRAFIA_0-149
 1	Portanto	portanto	ADV	_	_	13	advmod	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
-3	além	além	ADV	_	_	6	cc	O	_
+3	além	além	ADV	_	_	6	cc	O	MWEPOS=CCONJ
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	3	fixed	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	fator	fator	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
+6	fator	fator	NOUN	_	Gender=Masc|Number=Sing	9	conj	O	_
 7	custo	custo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
 9	fator	fator	NOUN	_	Gender=Masc|Number=Sing	13	nsubj	O	_
@@ -142748,11 +142748,11 @@
 
 # text = Além da questão populacional, esses eventos representam um aumento do risco sísmico associado a áreas ricas em petróleo, que ocasionam tanto riscos de exploração como riscos ambientais..
 # sent_id = 258-20160602-TESEMSC_0-6
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	questão	questão	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
+4	questão	questão	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 5	populacional	populacional	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	,	,	PUNCT	_	_	4	punct	O	_
 7	esses	esse	DET	_	Gender=Masc|Number=Plur|PronType=Dem	8	det	O	_
@@ -143599,9 +143599,9 @@
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
 41	epicentro	epicentro	NOUN	_	Gender=Masc|Number=Sing	37	nmod	O	_
 42	,	,	PUNCT	_	_	45	punct	O	_
-43	além	além	ADV	_	_	45	cc	O	_
+43	além	além	ADV	_	_	45	cc	O	MWEPOS=CCONJ
 44	de	de	ADP	_	_	43	fixed	O	_
-45	identificar	identificar	VERB	_	VerbForm=Inf	34	conj	O	_
+45	identificar	identificar	VERB	_	VerbForm=Inf	23	conj	O	_
 46	intervalos	intervalo	NOUN	_	Gender=Masc|Number=Plur	45	obj	O	_
 47	estratigráficos	estratigráfico	ADJ	_	Gender=Masc|Number=Plur	46	amod	O	_
 48	e	e	CCONJ	_	_	50	cc	O	_
@@ -146880,7 +146880,7 @@
 
 # text = Além de discutir o termo neotectônica para o território brasileiro..
 # sent_id = 258-20160602-TESEMSC_0-115
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	advmod	O	MWEPOS=ADV
 2	de	de	ADP	_	_	1	fixed	O	_
 3	discutir	discutir	VERB	_	VerbForm=Inf	0	root	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
@@ -150803,7 +150803,7 @@
 9	e	e	CCONJ	_	_	10	cc	O	_
 10	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	8	conj	B=LITOLOGIA	_
 11	,	,	PUNCT	_	_	14	punct	O	_
-12	além	além	ADV	_	_	14	cc	O	_
+12	além	além	ADV	_	_	14	cc	O	MWEPOS=CCONJ
 13	de	de	ADP	_	_	12	fixed	O	_
 14	diamictitos	diamictito	NOUN	_	Gender=Masc|Number=Plur	8	conj	B=LITOLOGIA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
@@ -153356,7 +153356,7 @@
 63	marinho	marinho	NOUN	_	Gender=Masc|Number=Sing	58	nmod	O	_
 64	raso	raso	ADJ	_	Gender=Masc|Number=Sing	63	amod	O	_
 65	,	,	PUNCT	_	_	55	punct	O	_
-66	além	além	ADV	_	_	70	cc	O	_
+66	além	além	ADV	_	_	70	cc	O	MWEPOS=CCONJ
 67	de	de	ADP	_	_	66	fixed	O	_
 68	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	70	det	O	_
 69	espessa	espesso	ADJ	_	Gender=Fem|Number=Sing	70	amod	O	_
@@ -154404,7 +154404,7 @@
 38	1999	1999	NUM	_	NumType=Card	34	flat:name	O	_
 39	)	)	PUNCT	_	_	22	punct	O	_
 40	,	,	PUNCT	_	_	43	punct	O	_
-41	além	além	ADV	_	_	43	cc	O	_
+41	além	além	ADV	_	_	43	cc	O	MWEPOS=CCONJ
 42	de	de	ADP	_	_	41	fixed	O	_
 43	reativação	reativação	NOUN	_	Gender=Fem|Number=Sing	3	conj	O	_
 44	de	de	ADP	_	_	46	case	O	_
@@ -156616,7 +156616,7 @@
 
 # text = Além de conceitos metodológicos..
 # sent_id = 258-20160602-TESEMSC_0-339
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	advmod	O	MWEPOS=ADV
 2	de	de	ADP	_	_	1	fixed	O	_
 3	conceitos	conceito	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 4	metodológicos	metodológico	ADJ	_	Gender=Masc|Number=Plur	3	amod	O	_
@@ -157161,7 +157161,7 @@
 36	perfil	perfil	NOUN	_	Gender=Masc|Number=Sing	33	nmod	O	_
 37	sônico	sônico	ADJ	_	Gender=Masc|Number=Sing	36	amod	O	_
 38	,	,	PUNCT	_	_	41	punct	O	_
-39	além	além	ADV	_	_	41	cc	O	_
+39	além	além	ADV	_	_	41	cc	O	MWEPOS=CCONJ
 40	de	de	ADP	_	_	39	fixed	O	_
 41	informações	informação	NOUN	_	Gender=Fem|Number=Plur	33	conj	O	_
 42	disponíveis	disponível	ADJ	_	Gender=Fem|Number=Plur	41	amod	O	_
@@ -158589,9 +158589,9 @@
 17	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	15	obj	O	_
 18	,	,	PUNCT	_	_	15	punct	O	_
 19	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	34	nsubj	O	_
-20	além	além	ADV	_	_	22	cc	O	_
+20	além	além	ADV	_	_	22	cc	O	MWEPOS=CCONJ
 21	de	de	ADP	_	_	20	fixed	O	_
-22	representarem	representar	VERB	_	Number=Plur|Person=3|VerbForm=Inf	34	advcl	O	_
+22	representarem	representar	VERB	_	Number=Plur|Person=3|VerbForm=Inf	34	conj	O	_
 23	marcos	marco	NOUN	_	Gender=Masc|Number=Plur	22	obj	O	_
 24	estratigráficos	estratigráfico	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
 25	importantes	importante	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
@@ -158676,7 +158676,7 @@
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
 29	sinal	sinal	NOUN	_	Gender=Masc|Number=Sing	27	obj	O	_
 30	,	,	PUNCT	_	_	27	punct	O	_
-31	além	além	ADV	_	_	34	cc	O	_
+31	além	além	ADV	_	_	34	cc	O	MWEPOS=CCONJ
 32-33	do	_	_	_	_	_	_	_	_
 32	de	de	ADP	_	_	31	fixed	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
@@ -159705,7 +159705,7 @@
 50	,	,	PUNCT	_	_	51	punct	O	_
 51	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	49	appos	B=LITOLOGIA	_
 52	,	,	PUNCT	_	_	55	punct	O	_
-53	além	além	ADV	_	_	55	cc	O	_
+53	além	além	ADV	_	_	55	cc	O	MWEPOS=CCONJ
 54	de	de	ADP	_	_	53	fixed	O	_
 55	diamictitos	diamictito	NOUN	_	Gender=Masc|Number=Plur	49	conj	B=LITOLOGIA	_
 56	e	e	CCONJ	_	_	57	cc	O	_
@@ -160126,9 +160126,9 @@
 
 # text = Além de falhas planares, pode ser observada também a ocorrência de falhas lístricas ao longo das seções..
 # sent_id = 258-20160602-TESEMSC_0-423
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2	de	de	ADP	_	_	1	fixed	O	_
-3	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	8	obl	B=LITOLOGIA|B=LOCAL:GEO	_
+3	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	11	conj	B=LITOLOGIA|B=LOCAL:GEO	_
 4	planares	planar	ADJ	_	Gender=Fem|Number=Plur	3	amod	O	_
 5	,	,	PUNCT	_	_	3	punct	O	_
 6	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -161129,7 +161129,7 @@
 
 # text = Além de um trend NW-SE marcado por estruturas alongadas que ocorre em toda extensão da área..
 # sent_id = 258-20160602-TESEMSC_0-446
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	advmod	O	MWEPOS=ADV
 2	de	de	ADP	_	_	1	fixed	O	_
 3	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	trend	trend	X	_	Gender=Masc|Number=Sing	0	root	O	_
@@ -162234,7 +162234,7 @@
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
 24	sismo	sismo	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
 25	,	,	PUNCT	_	_	30	punct	O	_
-26	além	além	ADV	_	_	30	cc	O	_
+26	além	além	ADV	_	_	30	cc	O	MWEPOS=CCONJ
 27-28	das	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	26	fixed	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
@@ -167364,11 +167364,11 @@
 2	resinas	resina	NOUN	_	Gender=Fem|Number=Plur	16	nsubj	O	_
 3	puras	puro	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
 4	,	,	PUNCT	_	_	8	punct	O	_
-5	além	além	ADV	_	_	8	cc	O	_
+5	além	além	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6	de	de	ADP	_	_	5	fixed	O	_
 7	serem	ser	AUX	_	Number=Plur|Person=3|VerbForm=Inf	8	cop	O	_
-8	líquidos	líquido	ADJ	_	Gender=Masc|Number=Plur	2	amod	O	_
-9	pesados	pesar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	O	_
+8	líquidos	líquido	NOUN	_	Gender=Masc|Number=Plur	16	conj	O	_
+9	pesados	pesado	ADJ	_	Gender=Masc|Number=Plur	8	amod	O	_
 10	ou	ou	CCONJ	_	_	11	cc	O	_
 11	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
 12	pastosos	pastoso	ADJ	_	Gender=Masc|Number=Plur	11	amod	O	_
@@ -177446,9 +177446,9 @@
 # sent_id = 77-20141209-TESEMSC_0-361
 1	Primeiramente	primeiramente	ADV	_	_	9	advmod	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
-3	além	além	ADV	_	_	5	mark	O	_
+3	além	além	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 4	de	de	ADP	_	_	3	fixed	O	_
-5	turvar	turvar	VERB	_	VerbForm=Inf	9	advcl	O	_
+5	turvar	turvar	VERB	_	VerbForm=Inf	9	conj	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	água	água	NOUN	_	Gender=Fem|Number=Sing	5	obj	B=FLUIDO	_
 8	,	,	PUNCT	_	_	5	punct	O	_
@@ -182194,7 +182194,7 @@
 2	reação	reação	NOUN	_	Gender=Fem|Number=Sing	3	nsubj	O	_
 3	gera	gerar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	glicerina	glicerina	NOUN	_	Gender=Fem|Number=Sing	3	obj	O	_
-5	além	além	ADV	_	_	8	cc	O	_
+5	além	além	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6-7	do	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	5	fixed	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
@@ -182629,10 +182629,10 @@
 
 # text = Além disso, algumas características conferem ao biodiesel importância técnica, ambiental e econômica, são elas [1]:
 # sent_id = 279-20140530-MONOGRAFIA_0-23
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	algumas	algum	DET	_	Gender=Fem|Number=Plur|PronType=Ind	6	det	O	_
 6	características	característica	NOUN	_	Gender=Fem|Number=Plur	7	nsubj	O	_
@@ -184496,10 +184496,10 @@
 
 # text = Além disso, estas soluções devem ainda ser de fácil aplicabilidade para setor sucroalcooleiro..
 # sent_id = 279-20140530-MONOGRAFIA_0-72
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	6	det	O	_
 6	soluções	solução	NOUN	_	Gender=Fem|Number=Plur	7	nsubj	O	_
@@ -189912,7 +189912,7 @@
 42	2004	2004	PROPN	_	Number=Sing	41	flat:name	O	_
 43	)	)	PUNCT	_	_	39	punct	O	_
 44	,	,	PUNCT	_	_	48	punct	O	_
-45	além	além	ADV	_	_	48	cc	O	_
+45	além	além	ADV	_	_	48	cc	O	MWEPOS=CCONJ
 46-47	das	_	_	_	_	_	_	_	_
 46	de	de	ADP	_	_	45	fixed	O	_
 47	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	48	det	O	_
@@ -190524,7 +190524,7 @@
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 20	UFMG	UFMG	PROPN	_	Gender=Fem|Number=Sing	17	nmod	O	_
 21	,	,	PUNCT	_	_	24	punct	O	_
-22	além	além	ADV	_	_	24	cc	O	_
+22	além	além	ADV	_	_	24	cc	O	MWEPOS=CCONJ
 23	de	de	ADP	_	_	22	fixed	O	_
 24	aproveitar	aproveitar	VERB	_	VerbForm=Inf	7	conj	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
@@ -191315,7 +191315,7 @@
 23	e	e	CCONJ	_	_	24	cc	O	_
 24	cianobactérias	cianobactéria	NOUN	_	Gender=Fem|Number=Plur	16	conj	O	_
 25	,	,	PUNCT	_	_	28	punct	O	_
-26	além	além	ADV	_	_	28	cc	O	_
+26	além	além	ADV	_	_	28	cc	O	MWEPOS=CCONJ
 27	de	de	ADP	_	_	26	fixed	O	_
 28	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	16	conj	B=FLUIDO	_
 29	e	e	CCONJ	_	_	30	cc	O	_
@@ -196177,7 +196177,7 @@
 20			ADP	_	_	21	case	O	_
 21	L	L	PROPN	_	Gender=Masc|Number=Sing	19	nmod	O	_
 22	,	,	PUNCT	_	_	25	punct	O	_
-23	além	além	ADV	_	_	25	cc	O	_
+23	além	além	ADV	_	_	25	cc	O	MWEPOS=CCONJ
 24	de	de	ADP	_	_	23	fixed	O	_
 25	padrões	padrão	NOUN	_	Gender=Masc|Number=Plur	19	conj	O	_
 26	de	de	ADP	_	_	27	case	O	_
@@ -200959,7 +200959,7 @@
 14	de	de	ADP	_	_	15	case	O	_
 15	catalisador	catalisador	NOUN	_	Gender=Masc|Number=Sing	13	nmod	B=QUÍMICA	_
 16	,	,	PUNCT	_	_	20	punct	O	_
-17	além	além	ADV	_	_	20	cc	O	_
+17	além	além	ADV	_	_	20	cc	O	MWEPOS=CCONJ
 18-19	da	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	17	fixed	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
@@ -204772,11 +204772,11 @@
 4	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	2	nmod	B=PRODUTO	_
 5	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
-7	além	além	ADV	_	_	10	cc	O	_
+7	além	além	ADV	_	_	10	cc	O	MWEPOS=CCONJ
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	7	fixed	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	5	obl	B=PRODUTO	_
+10	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	13	conj	B=PRODUTO	_
 11	,	,	PUNCT	_	_	5	punct	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
 13	glicerina	glicerina	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
@@ -207739,7 +207739,7 @@
 45	a	a	ADP	_	_	46	case	O	_
 46	diesel	diesel	NOUN	_	Gender=Masc|Number=Sing	44	nmod	B=PRODUTO	_
 47	,	,	PUNCT	_	_	39	punct	O	_
-48	além	além	ADV	_	_	51	cc	O	_
+48	além	além	ADV	_	_	51	cc	O	MWEPOS=CCONJ
 49-50	do	_	_	_	_	_	_	_	_
 49	de	de	ADP	_	_	48	fixed	O	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	51	det	O	_
@@ -207785,7 +207785,7 @@
 26	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	27	det	O	_
 27	cenários	cenário	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 28	,	,	PUNCT	_	_	31	punct	O	_
-29	além	além	ADV	_	_	31	cc	O	_
+29	além	além	ADV	_	_	31	cc	O	MWEPOS=CCONJ
 30	de	de	ADP	_	_	29	fixed	O	_
 31	atuar	atuar	VERB	_	VerbForm=Inf	22	conj	O	_
 32	como	como	ADP	_	_	33	case	O	_
@@ -210561,11 +210561,11 @@
 
 # text = Além da matéria orgânica, as rochas sedimentares também têm suma importância na geração do petróleo, principalmente pelo acúmulo de fragmentos de outros minerais e detritos orgânicos, e que, quando se encontra num ambiente de pouca permeabilidade – o que inibe a ação de água circulante e diminui a quantidade de oxigênio existente – criam as condições necessárias para a formação do petróleo..
 # sent_id = 391-20160914-MONOGRAFIA_0-86
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	matéria	matéria	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
+4	matéria	matéria	NOUN	_	Gender=Fem|Number=Sing	8	conj	O	_
 5	orgânica	orgânico	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	,	,	PUNCT	_	_	4	punct	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
@@ -218070,7 +218070,7 @@
 
 # text = Além destes, também foram testados catalisadores de Ru/γAl2O3, Pt/γAl2O3 e Mo/γAl2O3, como referência..
 # sent_id = 25-20150123-MONOGRAFIA_0-83
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2-3	destes	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	7	conj	O	_
@@ -219019,7 +219019,7 @@
 31	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	27	conj	O	_
 32	;	;	PUNCT	_	_	33	punct	O	_
 33	●	●	NOUN	_	Gender=Masc|Number=Sing	17	conj	O	_
-34	Além	além	ADV	_	_	37	cc	O	_
+34	Além	além	ADV	_	_	37	cc	O	MWEPOS=CCONJ
 35-36	do	_	_	_	_	_	_	_	_
 35	de	de	ADP	_	_	34	fixed	O	_
 36	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	37	det	O	_
@@ -221140,7 +221140,7 @@
 10	ou	ou	CCONJ	_	_	11	cc	O	_
 11	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	9	conj	O	_
 12	,	,	PUNCT	_	_	15	punct	O	_
-13	além	além	ADV	_	_	15	cc	O	_
+13	além	além	ADV	_	_	15	cc	O	MWEPOS=CCONJ
 14	de	de	ADP	_	_	13	fixed	O	_
 15	adsorção	adsorção	NOUN	_	Gender=Fem|Number=Sing	9	conj	O	_
 16	com	com	ADP	_	_	17	case	O	_
@@ -231218,9 +231218,9 @@
 16	dados	dado	NOUN	_	Gender=Masc|Number=Plur	31	nsubj	O	_
 17	disponíveis	disponível	ADJ	_	Gender=Masc|Number=Plur	16	amod	O	_
 18	,	,	PUNCT	_	_	21	punct	O	_
-19	além	além	ADV	_	_	21	cc	O	_
+19	além	além	ADV	_	_	21	cc	O	MWEPOS=CCONJ
 20	de	de	ADP	_	_	19	fixed	O	_
-21	possuírem	possuir	VERB	_	Number=Plur|Person=3|VerbForm=Inf	31	advcl	O	_
+21	possuírem	possuir	VERB	_	Number=Plur|Person=3|VerbForm=Inf	31	conj	O	_
 22	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
 23	alto	alto	ADJ	_	Gender=Masc|Number=Sing	24	amod	O	_
 24	grau	grau	NOUN	_	Gender=Masc|Number=Sing	21	obj	O	_
@@ -236080,7 +236080,7 @@
 18	resposta	resposta	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 19	geofísica	geofísico	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 20	,	,	PUNCT	_	_	24	punct	O	_
-21	além	além	ADV	_	_	24	cc	O	_
+21	além	além	ADV	_	_	24	cc	O	MWEPOS=CCONJ
 22-23	do	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	21	fixed	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
@@ -238155,7 +238155,7 @@
 69	entre	entre	ADP	_	_	70	case	O	_
 70	outros	outro	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	66	nmod	O	_
 71	,	,	PUNCT	_	_	75	punct	O	_
-72	além	além	ADV	_	_	75	cc	O	_
+72	além	além	ADV	_	_	75	cc	O	MWEPOS=CCONJ
 73	de	de	ADP	_	_	72	fixed	O	_
 74	inúmeros	inúmero	ADJ	_	Gender=Masc|Number=Plur	75	amod	O	_
 75	diques	dique	NOUN	_	Gender=Masc|Number=Plur	50	conj	O	_
@@ -241671,7 +241671,7 @@
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	32	obl	O	_
 40	,	,	PUNCT	_	_	32	punct	O	_
-41	além	além	ADV	_	_	44	cc	O	_
+41	além	além	ADV	_	_	44	cc	O	MWEPOS=CCONJ
 42	de	de	ADP	_	_	41	fixed	O	_
 43	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	44	det	O	_
 44	intrusão	intrusão	NOUN	_	Gender=Fem|Number=Sing	23	conj	O	_
@@ -242598,7 +242598,7 @@
 
 # text = Além da aeromagnetometria, a única fonte de informação adicional sobre profundidades do embasamento raso se resume a algumas linhas sísmicas distribuídas esparsamente e gravimetria de fundo restrita à região do gráben da Barra do São João (Mohriak e Barros 1990)..
 # sent_id = 20-20140904-TESEDSC_0-288
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
@@ -243800,11 +243800,11 @@
 
 # text = Além da densidade variável pode-se perceber também as marcações radiais das linhas de navegação produzidas por radio-posicionamento..
 # sent_id = 20-20140904-TESEDSC_0-324
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
+4	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 5	variável	variável	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6-7	pode-se	_	_	_	_	_	_	_	_
 6	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -245915,7 +245915,7 @@
 30	mesma	mesmo	ADJ	_	Gender=Fem|Number=Sing	31	amod	O	_
 31	direção	direção	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 32	,	,	PUNCT	_	_	35	punct	O	_
-33	além	além	ADV	_	_	35	cc	O	_
+33	além	além	ADV	_	_	35	cc	O	MWEPOS=CCONJ
 34	de	de	ADP	_	_	33	fixed	O	_
 35	distorcer	distorcer	VERB	_	VerbForm=Inf	21	conj	O	_
 36	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	35	obj	O	_
@@ -246419,9 +246419,9 @@
 17	sinal	sinal	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 18	analítico	analítico	ADJ	_	Gender=Masc|Number=Sing	17	amod	O	_
 19	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	24	nsubj	O	_
-20	além	além	ADV	_	_	22	cc	O	_
+20	além	além	ADV	_	_	22	cc	O	MWEPOS=CCONJ
 21	de	de	ADP	_	_	20	fixed	O	_
-22	discriminar	discriminar	VERB	_	VerbForm=Inf	24	advcl	O	_
+22	discriminar	discriminar	VERB	_	VerbForm=Inf	24	conj	O	_
 23	,	,	PUNCT	_	_	22	punct	O	_
 24	enfatiza	enfatizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
@@ -246609,12 +246609,12 @@
 16	costa	costa	NOUN	_	Gender=Fem|Number=Sing	14	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17	,	,	PUNCT	_	_	11	punct	O	_
 18	aparecem	aparecer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-19	além	além	ADV	_	_	23	advmod	O	_
+19	além	além	ADV	_	_	23	cc	O	MWEPOS=CCONJ
 20-21	dos	_	_	_	_	_	_	_	_
 20	de	de	ADP	_	_	19	fixed	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
 22	muitos	muito	DET	_	Gender=Masc|Number=Plur|PronType=Ind	23	det	O	_
-23	diques	dique	NOUN	_	Gender=Masc|Number=Plur	18	obj	O	_
+23	diques	dique	NOUN	_	Gender=Masc|Number=Plur	27	conj	O	_
 24	alcalinos	alcalino	ADJ	_	Gender=Masc|Number=Plur	23	amod	B=QUÍMICA	_
 25	,	,	PUNCT	_	_	27	punct	O	_
 26	duas	dois	NUM	_	NumType=Card	27	nummod	O	_
@@ -249043,11 +249043,11 @@
 
 # text = Além do mais, pode-se mostrar que a estimativa das coordenadas horizontais da fonte magnética através da Deconvolução de Euler, é estável se a janela de dados for centralizada nas coordenadas do ponto de máxima amplitude do sinal analítico..
 # sent_id = 20-20140904-TESEDSC_0-442
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	6	advmod	O	MWEPOS=ADV
 2-3	do	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	mais	mais	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	6	obl	O	_
+3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	1	fixed	O	_
+4	mais	mais	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	1	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6-7	pode-se	_	_	_	_	_	_	_	_
 6	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -254165,11 +254165,11 @@
 
 # text = Além desta informação escassa, se for disponível informação extra sobre a geometria da fonte, pode-se verificar que o método de inversão compacta tende a recuperar cada vez melhor sua geometria à medida que esta informação aumenta e novos elementos geométricos são acrescentados modelando o corpo..
 # sent_id = 20-20140904-TESEDSC_0-569
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	desta	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	4	det	O	_
-4	informação	informação	NOUN	_	Gender=Fem|Number=Sing	19	obl	O	_
+4	informação	informação	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
 5	escassa	escasso	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	,	,	PUNCT	_	_	4	punct	O	_
 7	se	se	SCONJ	_	_	10	mark	O	_
@@ -254923,7 +254923,7 @@
 
 # text = Além disso, o corpo mais ao sul apresenta uma “envoltória” de susceptibilidade negativa totalmente artificial e o ajuste entre a anomalia observada (azul) e a teórica (vermelho) é caótico..
 # sent_id = 20-20140904-TESEDSC_0-589
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	11	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
@@ -255215,11 +255215,11 @@
 
 # text = Além das áreas recuperadas com a susceptibilidade limite atingida, também se observa um ajuste perfeito entre as duas anomalias..
 # sent_id = 20-20140904-TESEDSC_0-597
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	das	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
-4	áreas	área	NOUN	_	Gender=Fem|Number=Plur	14	obl	O	_
+4	áreas	área	NOUN	_	Gender=Fem|Number=Plur	16	conj	O	_
 5	recuperadas	recuperar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	4	acl	O	_
 6	com	com	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -256659,10 +256659,10 @@
 
 # text = Além destes, permite também a estimativa do índice estrutural como parte do vetor de parâmetros inserido no problema inverso..
 # sent_id = 20-20140904-TESEDSC_0-631
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2-3	destes	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	5	conj	O	_
+3	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	8	conj	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	permite	permitir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	também	também	ADV	_	_	5	advmod	O	_
@@ -260104,11 +260104,11 @@
 17	,	,	PUNCT	_	_	16	punct	O	_
 18	pois	pois	SCONJ	_	_	39	mark	O	_
 19	,	,	PUNCT	_	_	24	punct	O	_
-20	além	além	ADV	_	_	24	mark	O	_
+20	além	além	ADV	_	_	24	cc	O	MWEPOS=CCONJ
 21	de	de	ADP	_	_	20	fixed	O	_
 22	estar	estar	AUX	_	VerbForm=Inf	24	cop	O	_
 23	em	em	ADP	_	_	24	case	O	_
-24	conformidade	conformidade	NOUN	_	Gender=Fem|Number=Sing	39	advcl	O	_
+24	conformidade	conformidade	NOUN	_	Gender=Fem|Number=Sing	39	conj	O	_
 25	com	com	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	sísmica	sísmica	NOUN	_	Gender=Fem|Number=Sing	24	nmod	B=MÉTODO	_
@@ -260531,11 +260531,11 @@
 
 # text = Além da presença das falhas, que geram as maiores anomalias magnéticas verificadas no perfil, o modelo de inversão também sugere maior espessamento do pacote de sedimentos em relação à sua porção intermediaria..
 # sent_id = 20-20140904-TESEDSC_0-738
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	presença	presença	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
+4	presença	presença	NOUN	_	Gender=Fem|Number=Sing	27	conj	O	_
 5-6	das	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
@@ -262158,7 +262158,7 @@
 12	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	13	det	O	_
 13	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	6	nmod	O	_
 14	,	,	PUNCT	_	_	3	punct	O	_
-15	além	além	ADV	_	_	19	cc	O	_
+15	além	além	ADV	_	_	19	cc	O	MWEPOS=CCONJ
 16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	15	fixed	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -262800,11 +262800,11 @@
 
 # text = Além do mais, o ajuste obtido para esta hipótese entre as observações e a anomalia teórica é sofrível..
 # sent_id = 20-20140904-TESEDSC_0-794
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	20	advmod	O	MWEPOS=ADV
 2-3	do	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	mais	mais	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	20	obl	O	_
+3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	1	fixed	O	_
+4	mais	mais	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	1	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	ajuste	ajuste	NOUN	_	Gender=Masc|Number=Sing	20	nsubj	O	_
@@ -262966,7 +262966,7 @@
 5	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	6	det	O	_
 6	hipótese	hipótese	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7	,	,	PUNCT	_	_	3	punct	O	_
-8	além	além	ADV	_	_	11	cc	O	_
+8	além	além	ADV	_	_	11	cc	O	MWEPOS=CCONJ
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	8	fixed	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -264001,11 +264001,11 @@
 2	também	também	ADV	_	_	3	advmod	O	_
 3	observada	observar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
-5	além	além	ADV	_	_	8	cc	O	_
+5	além	além	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6-7	da	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	5	fixed	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	similaridade	similaridade	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
+8	similaridade	similaridade	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
 9	geométrica	geométrico	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 10	,	,	PUNCT	_	_	13	punct	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
@@ -264969,7 +264969,7 @@
 41	altitude	altitude	NOUN	_	Gender=Fem|Number=Sing	39	nmod	O	_
 42	)	)	PUNCT	_	_	39	punct	O	_
 43	,	,	PUNCT	_	_	46	punct	O	_
-44	além	além	ADV	_	_	46	cc	O	_
+44	além	além	ADV	_	_	46	cc	O	MWEPOS=CCONJ
 45	de	de	ADP	_	_	44	fixed	O	_
 46	apresentar	apresentar	VERB	_	VerbForm=Inf	24	conj	O	_
 47	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
@@ -269418,7 +269418,7 @@
 22	de	de	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
 24	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	20	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-25	além	além	ADV	_	_	28	cc	O	_
+25	além	além	ADV	_	_	28	cc	O	MWEPOS=CCONJ
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	25	fixed	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
@@ -272427,9 +272427,9 @@
 7	,	,	PUNCT	_	_	4	punct	O	_
 8	pois	pois	SCONJ	_	_	29	mark	O	_
 9	,	,	PUNCT	_	_	12	punct	O	_
-10	além	além	ADV	_	_	12	mark	O	_
+10	além	além	ADV	_	_	12	cc	O	MWEPOS=CCONJ
 11	de	de	ADP	_	_	10	fixed	O	_
-12	produzir	produzir	VERB	_	VerbForm=Inf	29	advcl	O	_
+12	produzir	produzir	VERB	_	VerbForm=Inf	29	conj	O	_
 13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 14	nítido	nítido	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
 15	deslocamento	deslocamento	NOUN	_	Gender=Masc|Number=Sing	12	obj	O	_
@@ -275902,7 +275902,7 @@
 17	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	19	nsubj	O	_
 18	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	19	expl	O	_
 19	estende	estender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	O	_
-20	além	além	ADV	_	_	23	cc	O	_
+20	além	além	ADV	_	_	23	case	O	MWEPOS=ADP
 21-22	dos	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	20	fixed	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
@@ -276440,7 +276440,7 @@
 35	Cabo	Cabo	PROPN	_	Number=Sing	33	flat:name	O	_
 36	Frio	Frio	PROPN	_	Number=Sing	33	flat:name	O	_
 37	,	,	PUNCT	_	_	40	punct	O	_
-38	além	além	ADV	_	_	40	cc	O	_
+38	além	além	ADV	_	_	40	cc	O	MWEPOS=CCONJ
 39	de	de	ADP	_	_	38	fixed	O	_
 40	ocupar	ocupar	VERB	_	VerbForm=Inf	25	conj	O	_
 41	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
@@ -278716,7 +278716,7 @@
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
 31	análise	análise	NOUN	_	Gender=Fem|Number=Sing	29	obj	O	_
 32	para	para	ADP	_	_	36	case	O	_
-33	além	além	ADV	_	_	36	advmod	O	_
+33	além	além	ADV	_	_	36	case	O	MWEPOS=ADP
 34-35	da	_	_	_	_	_	_	_	_
 34	de	de	ADP	_	_	33	fixed	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
@@ -278895,12 +278895,12 @@
 18	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	19	det	O	_
 19	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	10	nmod	B=BACIA:TIPO	_
 20	,	,	PUNCT	_	_	25	punct	O	_
-21	além	além	ADV	_	_	25	cc	O	_
+21	além	além	ADV	_	_	25	cc	O	MWEPOS=CCONJ
 22-23	dos	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	21	fixed	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	25	det	O	_
 24	proeminentes	proeminente	ADJ	_	Gender=Masc|Number=Plur	25	amod	O	_
-25	recursos	recurso	NOUN	_	Gender=Masc|Number=Plur	34	obl	O	_
+25	recursos	recurso	NOUN	_	Gender=Masc|Number=Plur	10	conj	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	processamento	processamento	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
 28	e	e	CCONJ	_	_	29	cc	O	_
@@ -283943,7 +283943,7 @@
 14	hipóteses	hipótese	NOUN	_	Gender=Fem|Number=Plur	12	nmod	O	_
 15	interpretativas	interpretativo	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
 16	,	,	PUNCT	_	_	19	punct	O	_
-17	além	além	ADV	_	_	19	cc	O	_
+17	além	além	ADV	_	_	19	cc	O	MWEPOS=CCONJ
 18	de	de	ADP	_	_	17	fixed	O	_
 19	viabilizar	viabilizar	VERB	_	VerbForm=Inf	4	conj	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
@@ -286986,10 +286986,10 @@
 
 # text = Além disso, devido ao maior tempo necessário para sua perfuração, a invasão de fluido em um poço horizontal tende a ser mais acentuada, provocando neste caso um dano mais severo do que ocorreria em um poço vertical (Rosa et al., 2006)..
 # sent_id = 119-20141209-TESEMSC_0-19
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	23	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	23	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	devido	devido	ADV	_	_	23	advmod	O	_
 6-7	ao	_	_	_	_	_	_	_	_
@@ -287183,10 +287183,10 @@
 11	,	,	PUNCT	_	_	5	punct	O	_
 12	pois	pois	SCONJ	_	_	22	mark	O	_
 13	,	,	PUNCT	_	_	17	punct	O	_
-14	além	além	ADV	_	_	17	mark	O	_
+14	além	além	ADV	_	_	17	cc	O	MWEPOS=CCONJ
 15	de	de	ADP	_	_	14	fixed	O	_
 16	não	não	ADV	_	Polarity=Neg	17	advmod	O	_
-17	viscosificar	viscosificar	VERB	_	VerbForm=Inf	22	advcl	O	_
+17	viscosificar	viscosificar	VERB	_	VerbForm=Inf	22	conj	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
 19	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	17	obj	O	_
 20	menos	menos	ADV	_	_	21	advmod	O	_
@@ -287399,10 +287399,10 @@
 
 # text = Além disso, há o consenso na indústria petrolífera que o “óleo fácil” já foi encontrado e que poderemos esperar um decréscimo de produção nas próximas décadas (Machado, 2009)..
 # sent_id = 119-20141209-TESEMSC_0-28
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	5	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	5	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	há	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -288153,10 +288153,10 @@
 
 # text = Além disso, são apresentados, também, o aparato experimental e o protocolo de testes utilizado..
 # sent_id = 119-20141209-TESEMSC_0-57
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	6	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	6	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	aux:pass	O	_
 6	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -288241,10 +288241,10 @@
 10	,	,	PUNCT	_	_	24	punct	O	_
 11	e	e	CCONJ	_	_	24	cc	O	_
 12	,	,	PUNCT	_	_	13	punct	O	_
-13	além	além	ADV	_	_	15	cc	O	_
+13	além	além	ADV	_	_	24	advmod	O	MWEPOS=ADV
 14-15	disso	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	13	fixed	O	_
-15	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	24	obl	O	_
+15	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	13	fixed	O	_
 16	,	,	PUNCT	_	_	15	punct	O	_
 17	sugestões	sugestão	NOUN	_	Gender=Fem|Number=Plur	24	nsubj:pass	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
@@ -290653,7 +290653,7 @@
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
 23	,	,	PUNCT	_	_	19	punct	O	_
-24	além	além	ADV	_	_	27	cc	O	_
+24	além	além	ADV	_	_	27	cc	O	MWEPOS=CCONJ
 25-26	dos	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	24	fixed	O	_
 26	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	27	det	O	_
@@ -295464,7 +295464,7 @@
 41	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	42	det	O	_
 42	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	39	nmod	O	_
 43	,	,	PUNCT	_	_	46	punct	O	_
-44	além	além	ADV	_	_	46	cc	O	_
+44	além	além	ADV	_	_	46	cc	O	MWEPOS=CCONJ
 45	de	de	ADP	_	_	44	fixed	O	_
 46	especificar	especificar	VERB	_	VerbForm=Inf	35	conj	O	_
 47	e	e	CCONJ	_	_	48	cc	O	_
@@ -304732,11 +304732,11 @@
 1	No	no	ADP	_	_	2	case	O	_
 2	ensaio	ensaio	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
-4	além	além	ADV	_	_	7	cc	O	_
+4	além	além	ADV	_	_	7	cc	O	MWEPOS=CCONJ
 5-6	dos	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	4	fixed	O	_
 6	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	7	det	O	_
-7	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	2	conj	B=FLUIDO	_
+7	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	13	conj	B=FLUIDO	_
 8	de	de	ADP	_	_	9	case	I=FLUIDO	_
 9	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	7	nmod	B=LITOLOGIA|I=FLUIDO	_
 10	,	,	PUNCT	_	_	7	punct	O	_
@@ -309899,9 +309899,9 @@
 
 # text = Além de remover o dano causado pela invasão do fluido de perfuração, espera-se, também, que este dano seja removido em um menor tempo..
 # sent_id = 119-20141209-TESEMSC_0-682
-1	Além	além	ADV	_	_	3	mark	O	_
+1	Além	além	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2	de	de	ADP	_	_	1	fixed	O	_
-3	remover	remover	VERB	_	VerbForm=Inf	16	advcl	O	_
+3	remover	remover	VERB	_	VerbForm=Inf	16	conj	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	dano	dano	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
 6	causado	causar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	5	acl	O	_
@@ -311615,10 +311615,10 @@
 
 # text = Além disso, uma vez que os poros foram bloqueados totalmente e o escoamento cessou, uma menor fração de argila e também de polímero foi transportada para dentro da amostra..
 # sent_id = 119-20141209-TESEMSC_0-722
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	28	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	11	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	11	mark	O	_
 6	vez	vez	NOUN	_	Gender=Fem|Number=Sing	5	fixed	O	_
@@ -318588,10 +318588,10 @@
 47	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	48	det	O	_
 48	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	42	nmod	B=LITOLOGIA	_
 49	,	,	PUNCT	_	_	52	punct	O	_
-50	além	além	ADV	_	_	52	cc	O	_
+50	além	além	ADV	_	_	54	advmod	O	MWEPOS=ADV
 51-52	disso	_	_	_	_	_	_	_	_
 51	de	de	ADP	_	_	50	fixed	O	_
-52	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	54	obl	O	_
+52	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	50	fixed	O	_
 53	,	,	PUNCT	_	_	52	punct	O	_
 54	deve	dever	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	conj	O	_
 55	levar	levar	VERB	_	VerbForm=Inf	54	xcomp	O	_
@@ -320857,10 +320857,10 @@
 # text = E além disso, um fluido de Bingham não flui até uma tensão de cisalhamento, 𝜏, exercer um valor mínimo, 𝜏𝑙..
 # sent_id = 240-20140220-MONOGRAFIA_0-173
 1	E	e	CCONJ	_	_	11	cc	O	_
-2	além	além	ADV	_	_	4	cc	O	_
+2	além	além	ADV	_	_	11	advmod	O	MWEPOS=ADV
 3-4	disso	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	2	fixed	O	_
-4	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	11	conj	O	_
+4	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	2	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	11	nsubj	O	_
@@ -322132,10 +322132,10 @@
 
 # text = Além disso, filtrados elevados contribuem fortemente para o processo de prisão diferencial (FARIAS, 2005)..
 # sent_id = 240-20140220-MONOGRAFIA_0-209
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	7	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	7	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	filtrados	filtrar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	acl	O	_
 6	elevados	elevar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	5	xcomp	O	_
@@ -322228,7 +322228,7 @@
 37	forças	força	NOUN	_	Gender=Fem|Number=Plur	33	conj	O	_
 38	géis	gel	ADJ	_	Gender=Masc|Number=Plur	37	amod	O	_
 39	,	,	PUNCT	_	_	42	punct	O	_
-40	além	além	ADV	_	_	42	cc	O	_
+40	além	além	ADV	_	_	42	cc	O	MWEPOS=CCONJ
 41	de	de	ADP	_	_	40	fixed	O	_
 42	aumentar	aumentar	VERB	_	VerbForm=Inf	24	conj	O	_
 43	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	44	det	O	_
@@ -327514,7 +327514,7 @@
 
 # text = Além de descobrir qual dos modelos melhor se adapta a estes fluidos, a partir do coeficiente de correlação (R²)..
 # sent_id = 240-20140220-MONOGRAFIA_0-374
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	advmod	O	MWEPOS=ADV
 2	de	de	ADP	_	_	1	fixed	O	_
 3	descobrir	descobrir	VERB	_	VerbForm=Inf	10	advcl	O	_
 4	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	10	nsubj	O	_
@@ -328322,7 +328322,7 @@
 
 # text = Além de descobrir qual dos modelos melhor se adapta a estes fluidos, a partir do coeficiente de correlação (R²)..
 # sent_id = 240-20140220-MONOGRAFIA_0-398
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	3	advmod	O	MWEPOS=ADV
 2	de	de	ADP	_	_	1	fixed	O	_
 3	descobrir	descobrir	VERB	_	VerbForm=Inf	10	advcl	O	_
 4	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	10	nsubj	O	_
@@ -328447,10 +328447,10 @@
 44	)	)	PUNCT	_	_	40	punct	O	_
 45	e	e	CCONJ	_	_	60	cc	O	_
 46	,	,	PUNCT	_	_	47	punct	O	_
-47	além	além	ADV	_	_	49	cc	O	_
+47	além	além	ADV	_	_	60	advmod	O	MWEPOS=ADV
 48-49	disso	_	_	_	_	_	_	_	_
 48	de	de	ADP	_	_	47	fixed	O	_
-49	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	52	det	O	_
+49	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	47	fixed	O	_
 50	,	,	PUNCT	_	_	49	punct	O	_
 51	para	para	ADP	_	_	52	case	O	_
 52	Atapulgita	Atapulgita	PROPN	_	Gender=Fem|Number=Sing	35	obl	B=LITOLOGIA	_
@@ -328799,10 +328799,10 @@
 
 # text = Além disso, comparando a magnitude tixotrópica, para as mesmas concentrações, dos fluidos com Goma Xantana e Atapulgita, nas Tabelas 7 e 9, respectivamente, fica evidente, que, com a utilização da Atapulgita, o fluido apresenta maior grau tixotrópico..
 # sent_id = 240-20140220-MONOGRAFIA_0-411
-1	Além	além	ADV	_	_	3	cc	O	_
+1	Além	além	ADV	_	_	33	advmod	O	MWEPOS=ADV
 2-3	disso	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
-3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	33	conj	O	_
+3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
 5	comparando	comparar	VERB	_	VerbForm=Ger	33	advcl	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -329022,7 +329022,7 @@
 20	a	a	ADP	_	_	21	case	O	_
 21	Atapulgita	Atapulgita	PROPN	_	Gender=Fem|Number=Sing	19	obl	B=LITOLOGIA	_
 22	,	,	PUNCT	_	_	25	punct	O	_
-23	além	além	ADV	_	_	25	cc	O	_
+23	além	além	ADV	_	_	25	cc	O	MWEPOS=CCONJ
 24	de	de	ADP	_	_	23	fixed	O	_
 25	demonstrar	demonstrar	VERB	_	VerbForm=Inf	13	conj	O	_
 26	que	que	SCONJ	_	_	28	mark	O	_
@@ -332187,11 +332187,11 @@
 6	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	7	det	O	_
 7	estrutura	estrutura	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 8	,	,	PUNCT	_	_	3	punct	O	_
-9	além	além	ADV	_	_	12	cc	O	_
+9	além	além	ADV	_	_	12	cc	O	MWEPOS=CCONJ
 10-11	da	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	9	fixed	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	introdução	introdução	NOUN	_	Gender=Fem|Number=Sing	3	conj	O	_
+12	introdução	introdução	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 13	,	,	PUNCT	_	_	3	punct	O	_
 14-15	no	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
@@ -338500,11 +338500,11 @@
 30	equipe	equipe	NOUN	_	Gender=Fem|Number=Sing	27	nmod	O	_
 31	,	,	PUNCT	_	_	22	punct	O	_
 32	pois	pois	SCONJ	_	_	47	mark	O	_
-33	além	além	ADV	_	_	36	cc	O	_
+33	além	além	ADV	_	_	36	cc	O	MWEPOS=CCONJ
 34-35	dos	_	_	_	_	_	_	_	_
 34	de	de	ADP	_	_	33	fixed	O	_
 35	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	36	det	O	_
-36	perigos	perigo	NOUN	_	Gender=Masc|Number=Plur	47	obl	O	_
+36	perigos	perigo	NOUN	_	Gender=Masc|Number=Plur	44	conj	O	_
 37	acima	acima	ADV	_	_	38	advmod	O	_
 38	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	36	acl	O	_
 39-40	na	_	_	_	_	_	_	_	_
@@ -346374,11 +346374,11 @@
 7	explicam	explicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	que	que	SCONJ	_	_	18	mark	O	_
 9	,	,	PUNCT	_	_	10	punct	O	_
-10	além	além	ADV	_	_	13	cc	O	_
+10	além	além	ADV	_	_	13	cc	O	MWEPOS=CCONJ
 11-12	destes	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	10	fixed	O	_
 12	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	13	det	O	_
-13	íons	íom	NOUN	_	Gender=Masc|Number=Plur	18	obl	O	_
+13	íons	íom	NOUN	_	Gender=Masc|Number=Plur	19	conj	O	_
 14	,	,	PUNCT	_	_	13	punct	O	_
 15	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	16	det	O	_
 16	águas	água	NOUN	_	Gender=Fem|Number=Plur	18	nsubj	B=FLUIDO	_
@@ -346771,11 +346771,11 @@
 7	explicam	explicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	que	que	SCONJ	_	_	28	mark	O	_
 9	,	,	PUNCT	_	_	10	punct	O	_
-10	além	além	ADV	_	_	13	cc	O	_
+10	além	além	ADV	_	_	13	cc	O	MWEPOS=CCONJ
 11-12	dos	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	10	fixed	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
-13	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	28	obl	O	_
+13	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	23	conj	O	_
 14	naturais	natural	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 15	presentes	presente	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 16-17	nas	_	_	_	_	_	_	_	_
