commit 977fa059d86f22188023c1a360045b2380c54cd0
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 8 04:23:29 2021 -0300

    correções de "referente"

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 4795448..fb504c7 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1172,7 +1172,7 @@
 8	banco	banco	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	dados	dado	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
-11	referente	referente	ADV	_	_	4	advmod	O	_
+11	referente	referente	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 12-13	ao	_	_	_	_	_	_	_	_
 12	a	a	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
@@ -1181,7 +1181,7 @@
 16	(	(	PUNCT	_	_	17	punct	O	_
 17	2011	2011	NUM	_	NumType=Card	14	nmod:appos	O	_
 18	)	)	PUNCT	_	_	17	punct	O	_
-19	gerado	gerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	14	acl	O	_
+19	gerado	gerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	acl	O	_
 20	através	através	ADV	_	_	19	advmod	O	_
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
@@ -8649,7 +8649,7 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-246
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
-3	referente	referente	ADV	_	_	7	advmod	O	_
+3	referente	referente	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
@@ -8694,7 +8694,7 @@
 11	de	de	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
 13	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
-14	referente	referente	ADV	_	_	4	advmod	O	_
+14	referente	referente	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 15-16	ao	_	_	_	_	_	_	_	_
 15	a	a	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
@@ -8703,13 +8703,13 @@
 19	toplap	toplap	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 20	com	com	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	topo	topo	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
+22	topo	topo	NOUN	_	Gender=Masc|Number=Sing	19	nmod	O	_
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
 25	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
-27	downlap	downlap	NOUN	_	Gender=Fem|Number=Sing	25	conj	O	_
+27	downlap	downlap	NOUN	_	Gender=Fem|Number=Sing	19	conj	O	_
 28-29	das	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
@@ -8723,7 +8723,7 @@
 36	Superior	Superior	PROPN	_	Number=Sing	35	flat:name	I=GEOCRONOLOGIA	_
 37	com	com	ADP	_	_	39	case	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	det	O	_
-39	topo	topo	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
+39	topo	topo	NOUN	_	Gender=Masc|Number=Sing	27	nmod	O	_
 40-41	do	_	_	_	_	_	_	_	_
 40	de	de	ADP	_	_	42	case	O	_
 41	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	42	det	O	_
@@ -8735,7 +8735,7 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-248
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	8	nsubj:pass	O	_
-3	referente	referente	ADV	_	_	2	advmod	O	_
+3	referente	referente	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
@@ -8758,7 +8758,7 @@
 20	Belo	Belo	PROPN	_	Number=Sing	19	flat:name	O	_
 21	(	(	PUNCT	_	_	23	punct	O	_
 22	apenas	apenas	ADV	_	_	23	advmod	O	_
-23	porção	porção	NOUN	_	Gender=Fem|Number=Sing	14	appos	O	_
+23	porção	porção	NOUN	_	Gender=Fem|Number=Sing	19	appos	O	_
 24	norte	norte	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	)	)	PUNCT	_	_	23	punct	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
@@ -8803,7 +8803,7 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-250
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	8	nsubj:pass	O	_
-3	referente	referente	ADV	_	_	2	advmod	O	_
+3	referente	referente	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
@@ -8828,20 +8828,20 @@
 22	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	21	obj	B=LITOLOGIA|B=LOCAL:GEO	_
 23	lístricas	lístrico	ADJ	_	Gender=Fem|Number=Plur	22	amod	O	_
 24	(	(	PUNCT	_	_	25	punct	O	_
-25	Anexo	Anexo	PROPN	_	Gender=Masc|Number=Sing	22	appos	O	_
+25	Anexo	Anexo	PROPN	_	Gender=Masc|Number=Sing	22	parataxis	O	_
 26	02	02	PROPN	_	Number=Sing	25	flat:name	O	_
 27	e	e	CCONJ	_	_	28	cc	O	_
 28	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	25	conj	O	_
 29	26	26	PROPN	_	Number=Sing	28	flat:name	O	_
 30	)	)	PUNCT	_	_	25	punct	O	_
 31	,	,	PUNCT	_	_	32	punct	O	_
-32	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	22	conj	B=LITOLOGIA|B=LOCAL:GEO	_
+32	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	38	nsubj:pass	B=LITOLOGIA|B=LOCAL:GEO	_
 33	em	em	ADP	_	_	34	case	O	_
 34	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 35	dominó	dominó	ADJ	_	Gender=Masc|Number=Sing	34	amod	O	_
 36	também	também	ADV	_	_	38	advmod	O	_
 37	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	38	aux:pass	O	_
-38	identificadas	identificar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	21	conj	O	_
+38	identificadas	identificar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	8	conj	O	_
 39-40	no	_	_	_	_	_	_	_	_
 39	em	em	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
@@ -8855,13 +8855,13 @@
 47	para	para	ADP	_	_	49	case	O	_
 48	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	49	det	O	_
 49	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	46	obl	O	_
-50	referente	referente	ADV	_	_	46	advmod	O	_
+50	referente	referente	ADJ	_	Gender=Masc|Number=Sing	49	amod	O	_
 51-52	ao	_	_	_	_	_	_	_	_
 51	a	a	ADP	_	_	53	case	O	_
 52	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	53	det	O	_
 53	Mioceno	Mioceno	PROPN	_	Gender=Masc|Number=Sing	50	obl	B=GEOCRONOLOGIA	_
 54	(	(	PUNCT	_	_	55	punct	O	_
-55	Anexo	Anexo	PROPN	_	Gender=Masc|Number=Sing	53	appos	O	_
+55	Anexo	Anexo	PROPN	_	Gender=Masc|Number=Sing	49	parataxis	O	_
 56	05	05	PROPN	_	Number=Sing	55	flat:name	O	_
 57	e	e	CCONJ	_	_	58	cc	O	_
 58	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	55	conj	O	_
@@ -8893,7 +8893,7 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-252
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	8	nsubj:pass	O	_
-3	referente	referente	ADV	_	_	2	advmod	O	_
+3	referente	referente	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
@@ -8937,7 +8937,7 @@
 40	e	e	CCONJ	_	_	41	cc	O	_
 41	antitéticas	antitético	ADJ	_	Gender=Fem|Number=Plur	37	conj	O	_
 42	provavelmente	provavelmente	ADV	_	_	43	advmod	O	_
-43	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	36	acl	O	_
+43	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	33	acl	O	_
 44-45	à	_	_	_	_	_	_	_	_
 44	a	a	ADP	_	_	46	case	O	_
 45	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	46	det	O	_
@@ -9095,7 +9095,7 @@
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
-12	referente	referente	ADV	_	_	6	advmod	O	_
+12	referente	referente	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 13-14	ao	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -9818,7 +9818,7 @@
 4	de	de	ADP	_	_	5	case	O	_
 5	contorno	contorno	NOUN	_	Gender=Masc|Number=Sing	3	nmod	O	_
 6	estrutural	estrutural	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
-7	referente	referente	ADV	_	_	3	advmod	O	_
+7	referente	referente	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 8-9	ao	_	_	_	_	_	_	_	_
 8	a	a	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
@@ -9954,7 +9954,7 @@
 1	Para	para	ADP	_	_	3	case	O	_
 2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
 3	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
-4	referente	referente	ADV	_	_	3	advmod	O	_
+4	referente	referente	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 5-6	ao	_	_	_	_	_	_	_	_
 5	a	a	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -19423,7 +19423,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-7
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
-3	referente	referente	ADV	_	_	2	advmod	O	_
+3	referente	referente	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
@@ -28762,7 +28762,7 @@
 10	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	16	obl	O	_
 11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	16	nsubj	B=LITOLOGIA	_
 12	de	de	ADP	_	_	13	case	O	_
-13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	obl	O	_
+13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 14	fluviais	fluvial	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 15	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	16	aux:pass	O	_
 16	sucedidas	sucedido	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	5	acl:relcl	O	_
@@ -51430,7 +51430,7 @@
 17	clivagem	clivagem	NOUN	_	Gender=Fem|Number=Sing	14	obj	B=LITOLOGIA	_
 18	,	,	PUNCT	_	_	20	punct	O	_
 19	em	em	ADP	_	_	20	case	O	_
-20	meio	meio	NOUN	_	Gender=Masc|Number=Sing	14	obl	O	_
+20	meio	meio	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
 21	líquido	líquido	ADJ	_	Gender=Masc|Number=Sing	20	amod	O	_
 22	,	,	PUNCT	_	_	25	punct	O	_
 23-24	das	_	_	_	_	_	_	_	_
@@ -51448,12 +51448,12 @@
 34	grande	grande	ADJ	_	Gender=Fem|Number=Sing	32	amod	O	_
 35	(	(	PUNCT	_	_	42	punct	O	_
 36	especialmente	especialmente	ADV	_	_	42	advmod	O	_
-37	quando	quando	ADV	_	_	42	advmod	O	_
+37	quando	quando	SCONJ	_	_	42	mark	O	_
 38	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	39	det	O	_
 39	cátions	cátion	NOUN	_	Gender=Masc|Number=Plur	42	nsubj	B=QUÍMICA	_
 40	saturantes	saturante	ADJ	_	Gender=Masc|Number=Plur	39	amod	O	_
 41	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	42	cop	O	_
-42	Na+	Na+	PROPN	_	Gender=Masc|Number=Sing	30	advcl	O	_
+42	Na+	Na+	PROPN	_	Gender=Masc|Number=Sing	32	advcl	O	_
 43	,	,	PUNCT	_	_	44	punct	O	_
 44	K+	K+	PROPN	_	Gender=Masc|Number=Sing	42	conj	O	_
 45	,	,	PUNCT	_	_	46	punct	O	_
@@ -51465,7 +51465,7 @@
 51-52	à	_	_	_	_	_	_	_	_
 51	a	a	ADP	_	_	53	case	O	_
 52	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	53	det	O	_
-53	separação	separação	NOUN	_	Gender=Fem|Number=Sing	42	obl	O	_
+53	separação	separação	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
 54-55	das	_	_	_	_	_	_	_	_
 54	de	de	ADP	_	_	56	case	O	_
 55	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	56	det	O	_
@@ -71070,7 +71070,7 @@
 1	Quiricó	quiricó	INTJ	_	_	0	root	O	_
 2	com	com	ADP	_	_	4	case	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	formação	formação	NOUN	_	Gender=Fem|Number=Sing	1	obl	O	_
+4	formação	formação	NOUN	_	Gender=Fem|Number=Sing	1	nmod	O	_
 5	sotoposta	sotoposta	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	.	.	PUNCT	_	_	4	punct	O	_
 7	.	.	PUNCT	_	_	1	punct	O	_
@@ -91181,7 +91181,7 @@
 9	percurso	percurso	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
 10	entre	entre	ADP	_	_	12	case	O	_
 11	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
+12	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	captura	captura	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
 15	e	e	CCONJ	_	_	17	cc	O	_
@@ -91189,14 +91189,14 @@
 17	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	injeção	injeção	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
-20	,	,	PUNCT	_	_	1	punct	O	_
+20	,	,	PUNCT	_	_	25	punct	O	_
 21	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	25	nsubj	O	_
 22	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	25	cop	O	_
 23	de	de	ADP	_	_	25	case	O	_
 24	474,5	474,5	NUM	_	NumType=Card	25	nummod	O	_
-25	km	km	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
+25	km	km	NOUN	_	Gender=Masc|Number=Plur	9	acl:relcl	O	_
 26	,	,	PUNCT	_	_	25	punct	O	_
-27	referente	referente	ADV	_	_	25	advmod	O	_
+27	referente	referente	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
 28-29	ao	_	_	_	_	_	_	_	_
 28	a	a	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
@@ -91208,7 +91208,7 @@
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
 36	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	33	conj	O	_
 37	P-37	P-37	PROPN	_	Gender=Fem|Number=Sing	36	nmod	O	_
-38	,	,	PUNCT	_	_	25	punct	O	_
+38	,	,	PUNCT	_	_	27	punct	O	_
 39	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	40	cop	O	_
 40	necessárias	necessário	ADJ	_	Gender=Fem|Number=Plur	0	root	O	_
 41	duas	dois	NUM	_	NumType=Card	42	nummod	O	_
@@ -93659,7 +93659,7 @@
 21	459	459	NUM	_	NumType=Card	22	nummod	O	_
 22	quilômetros	quilômetro	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
-24	referente	referente	ADV	_	_	22	advmod	O	_
+24	referente	referente	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 25-26	à	_	_	_	_	_	_	_	_
 25	a	a	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
@@ -98913,7 +98913,7 @@
 
 # text = A %EXTRA estimadas para os campos de Marlim e Barracuda foram:
 # sent_id = 161-20150810-MONOGRAFIA_0-641
-1	A	a	ADP	_	_	2	case	O	_
+1	A	a	ADP	_	_	2	case	O	sentenciação
 2	%	%	SYM	_	_	12	obl	O	_
 3	EXTRA	EXTRA	PROPN	_	Number=Sing	2	nmod	O	_
 4	estimadas	estimar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	2	acl	O	_
@@ -116748,7 +116748,7 @@
 17-18	no	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
-19	mar	mar	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
+19	mar	mar	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 20	ou	ou	CCONJ	_	_	22	cc	O	_
 21	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	22	aux:pass	O	_
 22	encaminhada	encaminhar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	3	conj	O	_
@@ -135248,13 +135248,13 @@
 
 # text = Porém somando-se o percentual referente à água das três correntes ricas
 # sent_id = 334-20150521-MONOGRAFIA_0-720
-1	Porém	porém	ADV	_	_	2	advmod	O	_
+1	Porém	porém	ADV	_	_	2	advmod	O	sentenciação
 2-3	somando-se	_	_	_	_	_	_	_	_
 2	somando	somar	VERB	_	VerbForm=Ger	0	root	O	_
 3	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	2	expl	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	percentual	percentual	NOUN	_	Gender=Masc|Number=Sing	2	obj	O	_
-6	referente	referente	ADV	_	_	2	advmod	O	_
+6	referente	referente	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
 7-8	à	_	_	_	_	_	_	_	_
 7	a	a	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -135613,7 +135613,7 @@
 2	conhecido	conhecer	VERB	_	VerbForm=Part	22	advcl	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	valor	valor	NOUN	_	Gender=Masc|Number=Sing	2	obj	O	_
-5	referente	referente	ADV	_	_	2	advmod	O	_
+5	referente	referente	ADJ	_	Gender=Masc|Number=Sing	4	amod	O	_
 6-7	ao	_	_	_	_	_	_	_	_
 6	a	a	ADP	_	_	8	case	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
@@ -135654,7 +135654,7 @@
 40	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	39	acl	O	_
 41	)	)	PUNCT	_	_	36	punct	O	_
 42	(	(	PUNCT	_	_	43	punct	O	_
-43	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	36	appos	O	_
+43	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	22	parataxis	O	_
 44	32	32	PROPN	_	Number=Sing	43	flat:name	O	_
 45	)	)	PUNCT	_	_	43	punct	O	_
 46	.	.	PUNCT	_	_	22	punct	O	_
@@ -146695,7 +146695,7 @@
 
 # text = Berrocal et al. (1984) expõem uma compilação de diversos dados sísmicos obtidos para diferentes regiões do Brasil, grande parte referente ao estudo C a p í t u l o 2 - S i s m i c i d a d e | 28 desenvolvido pelo IAG em 1981, e de acordo com a quantidade de eventos sísmicos e sua provável correlação com províncias tectônicas conhecidas, estabelecem uma regionalização sísmica para o país..
 # sent_id = 258-20160602-TESEMSC_0-112
-1	Berrocal	Berrocal	PROPN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
+1	Berrocal	Berrocal	PROPN	_	Gender=Masc|Number=Sing	7	nsubj	O	sentenciação
 2	et	et	PROPN	_	Number=Sing	1	flat:name	O	_
 3	al.	al.	PROPN	_	Number=Sing	1	flat:name	O	_
 4	(	(	PUNCT	_	_	1	flat:name	O	_
@@ -146719,12 +146719,12 @@
 21	,	,	PUNCT	_	_	23	punct	O	_
 22	grande	grande	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 23	parte	parte	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
-24	referente	referente	ADV	_	_	23	advmod	O	_
+24	referente	referente	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25-26	ao	_	_	_	_	_	_	_	_
 25	a	a	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
 27	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	24	obl	O	_
-28	C	c	ADJ	_	Gender=Masc|Number=Sing	27	amod	O	_
+28	C	c	ADJ	_	Gender=Masc|Number=Sing	27	amod	O	eliminar
 29	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
 30	p	p	PROPN	_	Gender=Masc|Number=Sing	27	nmod	O	_
 31	í	í	PROPN	_	Number=Sing	30	flat:name	O	_
@@ -165697,13 +165697,13 @@
 22	bilhões	bilhão	NUM	_	NumType=Card	21	flat	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	barris	barri	NOUN	_	Gender=Masc|Number=Plur	21	nmod	O	_
-25	,	,	PUNCT	_	_	27	punct	O	_
+25	,	,	PUNCT	_	_	30	punct	O	_
 26	sendo	ser	AUX	_	VerbForm=Ger	30	cop	O	_
 27	2,9	2,9	NUM	_	NumType=Card	30	nummod	O	_
 28	bilhões	bilhão	NUM	_	NumType=Card	27	flat	O	_
 29	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
 30	parcela	parcela	NOUN	_	Gender=Fem|Number=Sing	12	advcl	O	_
-31	referente	referente	ADV	_	_	30	advmod	O	_
+31	referente	referente	ADJ	_	Gender=Fem|Number=Sing	30	amod	O	_
 32	a	a	ADP	_	_	33	case	O	_
 33	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	31	obl	B=FLUIDO	_
 34	pesados	pesar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	33	acl	I=FLUIDO	_
@@ -165711,8 +165711,8 @@
 36	em	em	ADP	_	_	39	case	O	_
 37	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	39	det	O	_
 38	maior	grande	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
-39	parte	parte	NOUN	_	Gender=Fem|Number=Sing	30	nmod	O	_
-40	situada	situar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	39	acl	O	_
+39	parte	parte	NOUN	_	Gender=Fem|Number=Sing	40	obl	O	_
+40	situada	situar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	30	acl	O	_
 41	em	em	ADP	_	_	42	case	O	_
 42	campos	campo	NOUN	_	Gender=Masc|Number=Plur	40	obl	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
 43	marítimos	marítimo	ADJ	_	Gender=Masc|Number=Plur	42	amod	O	_
@@ -177176,7 +177176,7 @@
 5	de	de	ADP	_	_	8	case	O	_
 6	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	5	fixed	O	_
 7	com	com	ADP	_	_	5	fixed	O	_
-8	Silva	Silva	PROPN	_	Gender=Masc|Number=Sing	1	obl	O	_
+8	Silva	Silva	PROPN	_	Gender=Masc|Number=Sing	29	obl	O	_
 9	35	35	PROPN	_	Number=Sing	8	flat:name	O	_
 10	(	(	PUNCT	_	_	11	punct	O	_
 11	2003	2003	NUM	_	NumType=Card	8	flat:name	O	_
@@ -196756,11 +196756,11 @@
 5	separação	separação	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	fases	fase	NOUN	_	Gender=Fem|Number=Plur	5	nmod	O	_
-8	,	,	PUNCT	_	_	4	punct	O	_
+8	,	,	PUNCT	_	_	17	punct	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	camada	camada	NOUN	_	Gender=Fem|Number=Sing	17	nsubj:pass	O	_
 11	superior	alto	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
-12	referente	referente	ADV	_	_	17	advmod	O	_
+12	referente	referente	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 13-14	ao	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -196778,7 +196778,7 @@
 25	C	c	NOUN	_	Gender=Masc|Number=Sing	24	compound	O	_
 26	sob	sob	ADP	_	_	27	case	O	_
 27	vácuo	vácuo	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
-28	,	,	PUNCT	_	_	30	punct	O	_
+28	,	,	PUNCT	_	_	40	punct	O	_
 29	para	para	SCONJ	_	_	30	mark	O	_
 30	recuperar	recuperar	VERB	_	VerbForm=Inf	40	advcl	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
@@ -196790,7 +196790,7 @@
 37	produto	produto	NOUN	_	Gender=Masc|Number=Sing	32	conj	O	_
 38	obtido	obter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	37	acl	O	_
 39	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	40	aux:pass	O	_
-40	purificado	purificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	17	conj	O	_
+40	purificado	purificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	4	conj	O	_
 41	com	com	ADP	_	_	42	case	O	_
 42	resina	resina	NOUN	_	Gender=Fem|Number=Sing	40	obl	O	_
 43	de	de	ADP	_	_	44	case	O	_
@@ -197030,7 +197030,7 @@
 17	de	de	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
 19	pico	pico	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
-20	referente	referente	ADV	_	_	16	advmod	O	_
+20	referente	referente	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 21-22	aos	_	_	_	_	_	_	_	_
 21	a	a	ADP	_	_	23	case	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
@@ -197049,7 +197049,7 @@
 33	de	de	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 35	pico	pico	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
-36	referente	referente	ADV	_	_	32	advmod	O	_
+36	referente	referente	ADJ	_	Gender=Masc|Number=Sing	32	amod	O	_
 37-38	aos	_	_	_	_	_	_	_	_
 37	a	a	ADP	_	_	39	case	O	_
 38	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	39	det	O	_
@@ -215150,13 +215150,13 @@
 4	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
 6	Figuras	Figuras	PROPN	_	Gender=Fem|Number=Plur	4	nsubj:pass	O	_
-7	7	7	PROPN	_	Number=Sing	6	flat:name	O	_
+7	7	7	PROPN	_	Number=Sing	6	nummod	O	_
 8	,	,	PUNCT	_	_	9	punct	O	_
-9	8	8	NUM	_	NumType=Card	6	nmod:appos	O	_
+9	8	8	NUM	_	NumType=Card	6	conj	O	_
 10	e	e	CCONJ	_	_	11	cc	O	_
-11	9	9	NUM	_	NumType=Card	9	conj	O	_
+11	9	9	NUM	_	NumType=Card	6	conj	O	_
 12	,	,	PUNCT	_	_	13	punct	O	_
-13	referentes	referente	ADJ	_	Gender=Masc|Number=Plur	6	amod	O	_
+13	referentes	referente	ADJ	_	Gender=Fem|Number=Plur	6	amod	O	_
 14-15	aos	_	_	_	_	_	_	_	_
 14	a	a	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -215164,7 +215164,7 @@
 17	ternários	ternário	ADJ	_	Gender=Masc|Number=Plur	16	amod	O	_
 18	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	16	acl	O	_
 19-20	variando-se	_	_	_	_	_	_	_	_
-19	variando	variar	VERB	_	VerbForm=Ger	4	advcl	O	_
+19	variando	variar	VERB	_	VerbForm=Ger	18	advcl	O	_
 20	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	19	expl	O	_
 21	apenas	apenas	ADV	_	_	23	advmod	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
@@ -225684,7 +225684,7 @@
 48	número	número	NOUN	_	Gender=Masc|Number=Sing	32	acl:relcl	O	_
 49	de	de	ADP	_	_	50	case	O	_
 50	onda	onda	NOUN	_	Gender=Fem|Number=Sing	48	nmod	O	_
-51	referente	referente	ADV	_	_	48	advmod	O	_
+51	referente	referente	ADJ	_	Gender=Masc|Number=Sing	48	amod	O	_
 52-53	ao	_	_	_	_	_	_	_	_
 52	a	a	ADP	_	_	54	case	O	_
 53	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	54	det	O	_
@@ -247086,7 +247086,7 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
 3	capitulo	capitulo	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
-4	referente	referente	ADV	_	_	3	advmod	O	_
+4	referente	referente	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 5-6	à	_	_	_	_	_	_	_	_
 5	a	a	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -255891,7 +255891,7 @@
 12	contraste	contraste	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 13	negativo	negativo	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
-15	figuras	figura	NOUN	_	Gender=Fem|Number=Plur	20	parataxis	O	_
+15	figuras	figura	NOUN	_	Gender=Fem|Number=Plur	2	parataxis	O	_
 16	4.11	4.11	NUM	_	NumType=Card	15	nummod	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
 18	4.12	4.12	NUM	_	NumType=Card	16	conj	O	_
@@ -255926,10 +255926,10 @@
 42	São	São	PROPN	_	Number=Sing	38	flat:name	O	_
 43	João	João	PROPN	_	Number=Sing	38	flat:name	O	_
 44	,	,	PUNCT	_	_	45	punct	O	_
-45	associado	associar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	38	acl	O	_
+45	associado	associar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	27	acl	O	_
 46	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	47	det	O	_
 47	falha	falha	NOUN	_	Gender=Fem|Number=Sing	45	obj	B=LITOLOGIA|B=LOCAL:GEO	_
-48	,	,	PUNCT	_	_	45	punct	O	_
+48	,	,	PUNCT	_	_	51	punct	O	_
 49	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	51	nsubj:pass	O	_
 50	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	51	aux:pass	O	_
 51	abordado	abordar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	47	acl:relcl	O	_
@@ -255938,7 +255938,7 @@
 53	em	em	ADP	_	_	55	case	O	_
 54	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	55	det	O	_
 55	item	item	NOUN	_	Gender=Masc|Number=Sing	51	obl	O	_
-56	referente	referente	ADV	_	_	51	advmod	O	_
+56	referente	referente	ADJ	_	Gender=Masc|Number=Sing	55	amod	O	_
 57-58	à	_	_	_	_	_	_	_	_
 57	a	a	ADP	_	_	59	case	O	_
 58	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	59	det	O	_
@@ -256549,7 +256549,7 @@
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
 9	capitulo	capitulo	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 10	6	6	NUM	_	NumType=Card	9	nummod	O	_
-11	referente	referente	ADV	_	_	6	advmod	O	_
+11	referente	referente	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
 12-13	à	_	_	_	_	_	_	_	_
 12	a	a	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
@@ -258710,7 +258710,7 @@
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	capitulo	capitulo	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 11	7	7	NUM	_	NumType=Card	10	nummod	O	_
-12	referente	referente	ADV	_	_	7	advmod	O	_
+12	referente	referente	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 13-14	aos	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
@@ -269422,7 +269422,7 @@
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	25	fixed	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	demarcação	demarcação	NOUN	_	Gender=Fem|Number=Sing	6	conj	O	_
+28	demarcação	demarcação	NOUN	_	Gender=Fem|Number=Sing	5	conj	O	_
 29-30	das	_	_	_	_	_	_	_	_
 29	de	de	ADP	_	_	32	case	O	_
 30	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	32	det	O	_
@@ -269430,7 +269430,7 @@
 32	áreas	área	NOUN	_	Gender=Fem|Number=Plur	28	nmod	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	detalhamento	detalhamento	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
-35	referentes	referente	ADJ	_	Gender=Masc|Number=Plur	32	amod	O	_
+35	referentes	referente	ADJ	_	Gender=Fem|Number=Plur	32	amod	O	_
 36-37	às	_	_	_	_	_	_	_	_
 36	a	a	ADP	_	_	38	case	O	_
 37	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	38	det	O	_
@@ -312344,14 +312344,14 @@
 4-5	percebe-se	_	_	_	_	_	_	_	_
 4	percebe	perceber	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 5	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	4	expl	O	_
-6	,	,	PUNCT	_	_	4	punct	O	_
+6	,	,	PUNCT	_	_	7	punct	O	_
 7	novamente	novamente	ADV	_	_	4	advmod	O	_
-8	,	,	PUNCT	_	_	4	punct	O	_
+8	,	,	PUNCT	_	_	7	punct	O	_
 9	que	que	SCONJ	_	_	25	mark	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	filtrado	filtrado	NOUN	_	Gender=Masc|Number=Sing	25	nsubj	O	_
 12	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	11	acl	O	_
-13	referente	referente	ADV	_	_	12	advmod	O	_
+13	referente	referente	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 14-15	ao	_	_	_	_	_	_	_	_
 14	a	a	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -326903,7 +326903,7 @@
 5	,	,	PUNCT	_	_	8	punct	O	_
 6	após	após	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	escolha	escolha	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
+8	escolha	escolha	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -326914,7 +326914,7 @@
 15	ensaios	ensaio	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 16	reológicos	reológico	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	realizados	realizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	15	acl	O	_
-18	referentes	referente	ADJ	_	Gender=Masc|Number=Plur	17	xcomp	O	_
+18	referentes	referente	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 19-20	aos	_	_	_	_	_	_	_	_
 19	a	a	ADP	_	_	21	case	O	_
 20	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	21	det	O	_
