commit f62987545b7ae4380f3e527a3c3c26f3a303f2e5
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Dec 7 11:21:27 2021 -0300

    N com word terminando em a e lema terminando em o

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index ddf3610..0a45450 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -3198,7 +3198,7 @@
 # text = A primeira a partir do Terraço de Rio Grande até o Alto de Florianópolis e a segunda do Terraço em direção ao Alto de Polônio.
 # sent_id = 247-20140910-MONOGRAFIA_0-103
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
-2	primeira	primeiro	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
+2	primeira	primeiro	ADJ	_	Gender=Fem|Number=Sing|NumType=Ord	0	root	O	_
 3	a	a	ADP	_	_	7	case	O	MWEPOS=ADP
 4	partir	partir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5-6	do	_	_	_	_	_	_	_	_
@@ -3215,7 +3215,7 @@
 15	Florianópolis	Florianópolis	PROPN	_	Number=Sing	13	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 16	e	e	CCONJ	_	_	18	cc	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
-18	segunda	segundo	NOUN	_	Gender=Fem|Number=Sing	2	conj	O	_
+18	segunda	segundo	ADJ	_	Gender=Fem|Number=Sing|NumType=Ord	2	conj	O	_
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
@@ -4749,15 +4749,15 @@
 22	parte	parte	NOUN	_	Gender=Fem|Number=Sing	15	nmod:appos	O	_
 23	emersa	emerso	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
 24	)	)	PUNCT	_	_	22	punct	O	_
-25	,	,	PUNCT	_	_	28	punct	O	_
+25	,	,	PUNCT	_	_	27	punct	O	_
 26-27	da	_	_	_	_	_	_	_	_
-26	de	de	ADP	_	_	28	case	O	_
-27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	profunda	profundo	NOUN	_	Gender=Fem|Number=Sing	15	conj	O	_
-29	,	,	PUNCT	_	_	13	punct	O	_
+26	de	de	ADP	_	_	27	case	O	_
+27	a	o	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	13	obl:arg	O	_
+28	profunda	profundo	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
+29	,	,	PUNCT	_	_	27	punct	O	_
 30	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	32	nsubj	O	_
 31	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	32	expl	O	_
-32	inicia	iniciar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	acl:relcl	O	_
+32	inicia	iniciar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	27	acl:relcl	O	_
 33-34	no	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
@@ -6376,7 +6376,7 @@
 9	rosetas	roseta	NOUN	_	Gender=Fem|Number=Plur	7	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	frequentes	frequente	NOUN	_	Gender=Fem|Number=Plur	9	nmod	O	_
-12	absoluta	absoluto	NOUN	_	Gender=Fem|Number=Sing	11	compound	O	_
+12	absoluta	absoluto	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
 13	para	para	ADP	_	_	16	case	O	_
 14	todos	todo	DET	_	Gender=Masc|Number=Plur|PronType=Tot	16	det	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -25530,7 +25530,7 @@
 2	e	e	CCONJ	_	_	3	cc	O	_
 3	caracterizam	caracterizar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	1	conj	O	_
 4	paleoictiofauna	paleoictiofauna	NOUN	_	Gender=Fem|Number=Sing	3	obj	O	_
-5	marinha	marinho	NOUN	_	Gender=Fem|Number=Sing	4	nmod	B=SISTEMA_DEPOSICIONAL	_
+5	marinha	marinho	ADJ	_	Gender=Fem|Number=Sing	4	amod	B=SISTEMA_DEPOSICIONAL	_
 6	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = As concreções são mais abundantes na porção inferior do intervalo de folhelho, que pode atingir 5 metros de espessura.
@@ -25599,9 +25599,9 @@
 7	cores	cor	NOUN	_	Gender=Fem|Number=Plur	3	nmod	O	_
 8	cinza	cinza	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	,	,	PUNCT	_	_	10	punct	O	_
-10	creme	creme	NOUN	_	Gender=Masc|Number=Sing	7	conj	O	_
+10	creme	creme	ADJ	_	Gender=Masc|Number=Sing	8	conj	O	_
 11	e	e	CCONJ	_	_	12	cc	O	_
-12	branca	branco	NOUN	_	Gender=Fem|Number=Sing	7	conj	O	_
+12	branca	branco	ADJ	_	Gender=Fem|Number=Sing	8	conj	O	_
 13	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = A forma mais comum é o calcário laminado (Cal) e o calcário com topo e base ondulados.
@@ -31580,7 +31580,7 @@
 11	energia	energia	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	elétrica	elétrico	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	em	em	ADP	_	_	14	case	O	_
-14	térmica	térmico	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
+14	térmica	térmica	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 15	através	através	ADV	_	_	18	case	O	MWEPOS=ADP
 16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	15	fixed	O	_
@@ -33122,7 +33122,7 @@
 11	energia	energia	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 12	eletromagnética	eletromagnético	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	em	em	ADP	_	_	14	case	O	_
-14	térmica	térmico	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
+14	térmica	térmica	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 15	e	e	CCONJ	_	_	23	cc	O	_
 16	segundo	segundo	ADP	_	_	17	case	O	_
 17	Manichand	Manichand	PROPN	_	Gender=Masc|Number=Sing	23	obl	O	_
@@ -38799,7 +38799,7 @@
 19	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	20	nsubj	O	_
 20	transformada	transformar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	30	advcl	O	_
 21	em	em	ADP	_	_	22	case	O	_
-22	térmica	térmico	NOUN	_	Gender=Fem|Number=Sing	20	obl:arg	O	_
+22	térmica	térmica	NOUN	_	Gender=Fem|Number=Sing	20	obl:arg	O	_
 23-24	no	_	_	_	_	_	_	_	_
 23	em	em	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
@@ -41922,7 +41922,7 @@
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
 23	poço	poço	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
-25	mecânica	mecânico	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
+25	mecânica	mecânica	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
 27	quimicamente	quimicamente	ADV	_	_	25	conj	O	_
 28	;	;	PUNCT	_	_	29	punct	O	_
@@ -44246,7 +44246,7 @@
 27	por	por	ADP	_	_	29	case	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	22	conj	O	_
-30	ilita-montmorilonita	ilita-montmortmonito	NOUN	_	Gender=Fem|Number=Sing	29	nmod	O	_
+30	ilita-montmorilonita	ilita-montmorilonita	NOUN	_	Gender=Fem|Number=Sing	29	nmod	O	_
 31	(	(	PUNCT	_	_	32	punct	O	_
 32	WEAVER	WEAVER	PROPN	_	Gender=Fem|Number=Sing	14	parataxis	O	_
 33	,	,	PUNCT	_	_	32	flat:name	O	_
@@ -45659,7 +45659,7 @@
 24-25	chama-se	_	_	_	_	_	_	_	_
 24	chama	chamar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	conj	O	_
 25	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	24	expl	O	_
-26	brucita	brucito	NOUN	_	Gender=Masc|Number=Sing	24	obj	O	_
+26	brucita	brucita	NOUN	_	Gender=Masc|Number=Sing	24	obj	O	_
 27	,	,	PUNCT	_	_	24	punct	O	_
 28	cuja	cujo	DET	_	Gender=Fem|Number=Sing|PronType=Rel	29	det	O	_
 29	estrutura	estrutura	NOUN	_	Gender=Fem|Number=Sing	31	nsubj	O	_
@@ -57492,7 +57492,7 @@
 39	de	de	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
 41	argilomineral	argilomineral	NOUN	_	Gender=Masc|Number=Sing	38	nmod	O	_
-42	esmectita	esmectito	NOUN	_	Gender=Masc|Number=Sing	41	nmod	O	_
+42	esmectita	esmectita	NOUN	_	Gender=Masc|Number=Sing	41	nmod	O	_
 43	presente	presente	ADJ	_	Gender=Masc|Number=Sing	41	amod	O	_
 44-45	na	_	_	_	_	_	_	_	_
 44	em	em	ADP	_	_	46	case	O	_
@@ -70261,7 +70261,7 @@
 12	de	de	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
 14	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
-15	estufa	estufo	NOUN	_	Gender=Masc|Number=Sing	14	compound	O	_
+15	estufa	estufa	NOUN	_	Gender=Masc|Number=Sing	14	compound	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	10	acl	O	_
 18-19	pela	_	_	_	_	_	_	_	_
@@ -92715,7 +92715,7 @@
 17	gases	gase	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
-20	estufa	estufo	NOUN	_	Gender=Masc|Number=Sing	19	compound	O	_
+20	estufa	estufa	NOUN	_	Gender=Masc|Number=Sing	19	compound	O	_
 21	caso	caso	SCONJ	_	_	23	mark	O	_
 22	não	não	ADV	_	Polarity=Neg	23	advmod	O	_
 23	houvesse	haver	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	11	advcl	O	_
@@ -106843,7 +106843,7 @@
 44	separação	separação	NOUN	_	Gender=Fem|Number=Sing	42	obl	O	_
 45	compacta	compacto	ADJ	_	Gender=Fem|Number=Sing	44	amod	O	_
 46	,	,	PUNCT	_	_	47	punct	O	_
-47	outra	outro	DET	_	Gender=Fem|Number=Sing|PronType=Ind	44	appos	O	_
+47	outra	outro	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	44	appos	O	_
 48-49	na	_	_	_	_	_	_	_	_
 48	em	em	ADP	_	_	50	case	O	_
 49	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	50	det	O	_
@@ -106860,7 +106860,7 @@
 60	,	,	PUNCT	_	_	63	punct	O	_
 61	e	e	CCONJ	_	_	63	cc	O	_
 62	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	63	det	O	_
-63	terceira	terceiro	NOUN	_	Gender=Fem|Number=Sing	57	conj	O	_
+63	terceira	terceiro	ADJ	_	Gender=Fem|Number=Sing	57	conj	O	_
 64	,	,	PUNCT	_	_	66	punct	O	_
 65	e	e	CCONJ	_	_	66	cc	O	_
 66	principal	principal	ADJ	_	Gender=Fem|Number=Sing	63	conj	O	_
@@ -120032,7 +120032,7 @@
 7	energia	energia	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 8	elétrica	elétrico	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	e	e	CCONJ	_	_	10	cc	O	_
-10	térmica	térmico	NOUN	_	Gender=Fem|Number=Sing	4	conj	O	_
+10	térmica	térmica	NOUN	_	Gender=Fem|Number=Sing	4	conj	O	_
 11	por	por	ADP	_	_	12	case	O	_
 12	BTU/h	BTU/h	PROPN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 13	de	de	ADP	_	_	14	case	O	_
@@ -125034,7 +125034,7 @@
 53	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	56	nsubj	O	_
 54	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	56	det	O	_
 55	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	56	det	O	_
-56	inversa	inverso	NOUN	_	Gender=Fem|Number=Sing	52	obl	O	_
+56	inversa	inversa	NOUN	_	Gender=Fem|Number=Sing	52	obl	O	_
 57	.	.	PUNCT	_	_	12	punct	O	_
 
 # text = 3 MATERIAL E MÉTODOS
@@ -127487,8 +127487,8 @@
 23	cromatografia	cromatografia	NOUN	_	Gender=Fem|Number=Sing	17	nmod:appos	O	_
 24	gasosa	gasoso	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	e	e	CCONJ	_	_	27	cc	O	_
-26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	líquida	líquido	NOUN	_	Gender=Fem|Number=Sing	23	conj	O	_
+26	a	o	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	24	conj	O	_
+27	líquida	líquido	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 28	.	.	PUNCT	_	_	10	punct	O	_
 
 # text = A fração pesada residual que não é identificada inicialmente, passa um teste de comportamento de solubilidade em que estes componentes entram em contato com solventes orgânicos parafínicos e aromáticos, o que classifica as frações em saturados, aromáticos, resinas e asfaltenos (SARA), ilustrado na Figura 2.1. (Tissot e Welte, 1984).
@@ -131049,7 +131049,7 @@
 11	expostas	expor	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	9	acl	O	_
 12	a	a	ADP	_	_	13	case	O	_
 13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	obl:arg	O	_
-14	óleo/água	óleo/águo	NOUN	_	Gender=Masc|Number=Plur	13	nmod	O	_
+14	óleo/água	óleo/água	NOUN	_	Gender=Masc|Number=Plur	13	nmod	O	_
 15	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	16	aux:pass	O	_
 16	baseada	basear	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 17	em	em	ADP	_	_	18	case	O	_
@@ -139923,7 +139923,7 @@
 13	de	de	ADP	_	_	11	fixed	O	_
 14	equações	equação	NOUN	_	Gender=Fem|Number=Plur	10	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	mecânica	mecânico	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
+16	mecânica	mecânica	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 17	de	de	ADP	_	_	18	case	O	_
 18	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
 19	para	para	ADP	_	_	20	case	O	_
@@ -159391,7 +159391,7 @@
 40	calcinado	calcinar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	39	acl	O	_
 41	em	em	ADP	_	_	42	case	O	_
 42	forno	forno	NOUN	_	Gender=Masc|Number=Sing	40	obl	O	_
-43	mufla	muflo	NOUN	_	Gender=Fem|Number=Sing	42	compound	O	_
+43	mufla	mufla	NOUN	_	Gender=Fem|Number=Sing	42	compound	O	_
 44	a	a	ADP	_	_	46	case	O	_
 45	500	500	NUM	_	NumType=Card	46	nummod	O	_
 46	°	°	NOUN	_	Gender=Masc|Number=Plur	40	obl	O	_
@@ -181094,7 +181094,7 @@
 41-42	da	_	_	_	_	_	_	_	_
 41	de	de	ADP	_	_	44	case	O	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	44	det	O	_
-43	zeólita	zeólito	NOUN	_	Gender=Fem|Number=Sing	40	nmod	O	_
+43	zeólita	zeólita	NOUN	_	Gender=Fem|Number=Sing	40	nmod	O	_
 44	suporte	suporte	NOUN	_	Gender=Fem|Number=Sing	43	nmod	O	_
 45	CBV-720	CBV-720	PROPN	_	Gender=Fem|Number=Sing	43	nmod	O	_
 46	.	.	PUNCT	_	_	38	punct	O	_
@@ -182762,7 +182762,7 @@
 37	]	]	PUNCT	_	_	36	punct	O	_
 38	para	para	ADP	_	_	40	case	O	_
 39	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
-40	zeólita	zeólito	NOUN	_	Gender=Fem|Number=Sing	32	obl	O	_
+40	zeólita	zeólita	NOUN	_	Gender=Fem|Number=Sing	32	obl	O	_
 41	suporte	suporte	NOUN	_	Gender=Fem|Number=Sing	40	nmod	O	_
 42	com	com	ADP	_	_	44	case	O	_
 43	menor	pequeno	ADJ	_	Gender=Masc|Number=Sing	44	amod	O	_
@@ -198978,7 +198978,7 @@
 5	problema	problema	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 6	inverso	inverso	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
 7	em	em	ADP	_	_	8	case	O	_
-8	geofísica	geofísico	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
+8	geofísica	geofísica	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
 9	,	,	PUNCT	_	_	5	punct	O	_
 10	também	também	ADV	_	_	15	advmod	O	_
 11	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	13	det	O	_
@@ -205463,9 +205463,9 @@
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	anomalia	anomalia	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 12	observada	observar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	11	acl	O	_
-13	e	e	CCONJ	_	_	15	cc	O	_
-14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
-15	teórica	teórico	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
+13	e	e	CCONJ	_	_	14	cc	O	_
+14	a	o	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	12	conj	O	_
+15	teórica	teórico	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	cop	O	_
 17	adequado	adequado	ADJ	_	Gender=Masc|Number=Sing	3	ccomp	O	_
 18	para	para	ADP	_	_	21	case	O	_
@@ -205474,7 +205474,7 @@
 21	ensaios	ensaio	NOUN	_	Gender=Masc|Number=Plur	17	obl	O	_
 22	acima	acima	ADV	_	_	21	advmod	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
-24	ou	ou	CCONJ	_	_	31	cc	O	_
+24	ou	ou	CCONJ	_	_	31	cc	O	MWEPOS=CCONJ
 25	seja	ser	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	fixed	O	_
 26	,	,	PUNCT	_	_	24	punct	O	_
 27	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -205959,7 +205959,7 @@
 11	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 13	“	“	PUNCT	_	_	14	punct	O	_
-14	envoltória	envoltório	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	envoltória	envoltória	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
 15	”	”	PUNCT	_	_	14	punct	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	susceptibilidade	susceptibilidade	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
@@ -205976,11 +205976,11 @@
 28	(	(	PUNCT	_	_	29	punct	O	_
 29	azul	azul	NOUN	_	Gender=Masc|Number=Sing	26	parataxis	O	_
 30	)	)	PUNCT	_	_	29	punct	O	_
-31	e	e	CCONJ	_	_	33	cc	O	_
-32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
-33	teórica	teórico	NOUN	_	Gender=Fem|Number=Sing	27	conj	O	_
+31	e	e	CCONJ	_	_	32	cc	O	_
+32	a	o	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	27	conj	O	_
+33	teórica	teórico	ADJ	_	Gender=Fem|Number=Sing	32	amod	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
-35	vermelho	vermelho	NOUN	_	Gender=Masc|Number=Sing	33	parataxis	B=CAMPO	_
+35	vermelho	vermelho	NOUN	_	Gender=Masc|Number=Sing	32	parataxis	B=CAMPO	_
 36	)	)	PUNCT	_	_	35	punct	O	_
 37	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	38	cop	O	_
 38	caótico	caótico	ADJ	_	Gender=Masc|Number=Sing	11	conj	O	_
@@ -217679,7 +217679,7 @@
 13	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	aux:pass	O	_
 14	levantadas	levantar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 15	a	a	ADP	_	_	19	case	O	MWEPOS=ADP
-16	partir	partir	NOUN	_	_	15	fixed	O	_
+16	partir	partir	VERB	_	VerbForm=Inf	15	fixed	O	_
 17-18	dos	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	15	fixed	O	_
 18	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
@@ -217713,11 +217713,11 @@
 44	(	(	PUNCT	_	_	45	punct	O	_
 45	emersa	emerso	ADJ	_	Gender=Fem|Number=Sing	40	amod	O	_
 46	a	a	ADP	_	_	47	case	O	_
-47	W	w	NOUN	_	Gender=Fem|Number=Sing	45	obl	O	_
+47	W	W	PROPN	_	Gender=Fem|Number=Sing	45	obl	O	_
 48	e	e	CCONJ	_	_	49	cc	O	_
-49	submersa	submerso	NOUN	_	Gender=Fem|Number=Sing	45	conj	O	_
-50	a	a	ADP	_	_	49	case	O	_
-51	E	e	CCONJ	_	_	49	cc	O	_
+49	submersa	submerso	ADJ	_	Gender=Fem|Number=Sing	45	conj	O	_
+50	a	a	ADP	_	_	51	case	O	_
+51	E	E	PROPN	_	Gender=Fem|Number=Sing	49	obl	O	_
 52	)	)	PUNCT	_	_	45	punct	O	_
 53	e	e	CCONJ	_	_	55	cc	O	_
 54	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	55	det	O	_
@@ -224403,7 +224403,7 @@
 12	área	área	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 13	continental	continental	ADJ	_	Gender=Fem|Number=Sing	12	amod	B=SISTEMA_DEPOSICIONAL	_
 14	como	como	CCONJ	_	_	15	cc	O	_
-15	marinha	marinho	NOUN	_	Gender=Fem|Number=Sing	12	conj	B=SISTEMA_DEPOSICIONAL	_
+15	marinha	marinho	ADJ	_	Gender=Fem|Number=Sing	13	conj	B=SISTEMA_DEPOSICIONAL	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	demonstrando	demonstrar	VERB	_	VerbForm=Ger	4	advcl	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
@@ -248067,7 +248067,7 @@
 # text = A mecânica dos fluidos em poços horizontais difere daquela para poços verticais, assim como as permeabilidades horizontal e vertical na maioria das formações.
 # sent_id = 119-20141209-TESEMSC_0-464
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
-2	mecânica	mecânico	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
+2	mecânica	mecânica	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
 3-4	dos	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -269024,7 +269024,7 @@
 33	de	de	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 35	tipo	tipo	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
-36	óleo/água	óleo/águo	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
+36	óleo/água	óleo/água	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
 37	(	(	PUNCT	_	_	38	punct	O	_
 38	SILVA	Silva	PROPN	_	Gender=Masc|Number=Sing	3	parataxis	O	_
 39	,	,	PUNCT	_	_	38	flat:name	O	_
@@ -283250,17 +283250,17 @@
 56	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	57	det	O	_
 57	cartas	carta	NOUN	_	Gender=Fem|Number=Plur	53	nmod	O	_
 58	SAO	SAO	PROPN	_	Gender=Fem|Number=Sing	57	nmod	O	_
-59	,	,	PUNCT	_	_	61	punct	O	_
-60	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	61	det	O	_
-61	estratégica	estratégico	NOUN	_	Gender=Fem|Number=Sing	53	conj	O	_
+59	,	,	PUNCT	_	_	60	punct	O	_
+60	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	50	appos	O	_
+61	estratégica	estratégico	ADJ	_	Gender=Fem|Number=Sing	60	amod	O	_
 62	,	,	PUNCT	_	_	64	punct	O	_
 63	oito	oito	NUM	_	NumType=Card	64	nummod	O	_
-64	cartas	carta	NOUN	_	Gender=Fem|Number=Plur	61	conj	O	_
+64	cartas	carta	NOUN	_	Gender=Fem|Number=Plur	60	conj	O	_
 65	SAO	SAO	PROPN	_	Gender=Fem|Number=Sing	64	nmod	O	_
 66	táticas	tática	ADJ	_	Gender=Fem|Number=Plur	64	amod	O	_
 67	e	e	CCONJ	_	_	69	cc	O	_
 68	dez	dez	NUM	_	NumType=Card	69	nummod	O	_
-69	Cartas	Cartas	PROPN	_	Gender=Fem|Number=Plur	61	conj	O	_
+69	Cartas	Cartas	PROPN	_	Gender=Fem|Number=Plur	60	conj	O	_
 70	SAO	SAO	PROPN	_	Number=Sing	69	flat:name	O	_
 71	operacionais	operacional	ADJ	_	Gender=Fem|Number=Plur	69	amod	O	_
 72	.	.	PUNCT	_	_	32	punct	O	_
