commit 9420f0c51db5838645d749aba5ab5a35853e8c99
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Dec 7 21:55:55 2021 -0300

    correções de lema

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 0a45450..a9838cc 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -13280,7 +13280,7 @@
 # sent_id = 241-20140227-MONOGRAFIA_0-113
 1	Cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Tot	2	det	O	_
 2	floco	floco	NOUN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
-3	expandido	expander	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
+3	expandido	expandir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
 4	aprisiona	aprisionar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 5-6	consigo	_	_	_	_	_	_	_	_
 5	com	com	ADP	_	_	6	case	O	_
@@ -28504,7 +28504,7 @@
 15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nsubj	O	_
 16	heterolíticas	heterolítica	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
 17	intensamente	intensamente	ADV	_	_	18	advmod	O	_
-18	bioturbadas	bioturbar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	acl	O	_
+18	bioturbadas	bioturbado	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
 19	.	.	PUNCT	_	_	14	punct	O	_
 
 # text = Arenitos finos a médios com estratificação cruzada acanalada (fácies St) encerram a sequência.
@@ -37823,7 +37823,7 @@
 4	que	que	SCONJ	_	_	7	mark	O	_
 5	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	6	det	O	_
 6	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
-7	possua	possuar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
+7	possua	possuir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 8	horários	horário	NOUN	_	Gender=Masc|Number=Plur	7	obj	O	_
 9	predefinidos	predefinido	ADJ	_	Gender=Masc|Number=Plur	8	amod	O	_
 10	de	de	ADP	_	_	11	case	O	_
@@ -46376,7 +46376,7 @@
 27	chegando	chegar	VERB	_	VerbForm=Ger	23	advcl	O	_
 28	a	a	ADP	_	_	29	mark	O	_
 29-30	separá-las	_	_	_	_	_	_	_	_
-29	separá	separ	VERB	_	VerbForm=Inf	27	xcomp	O	_
+29	separá	separar	VERB	_	VerbForm=Inf	27	xcomp	O	_
 30	las	elas	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	29	obj	O	_
 31	totalmente	totalmente	ADV	_	_	29	advmod	O	_
 32	,	,	PUNCT	_	_	33	punct	O	_
@@ -74084,7 +74084,7 @@
 16	transportado	transportar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	13	xcomp	O	_
 17	em	em	ADP	_	_	18	case	O	_
 18	navios	navio	NOUN	_	Gender=Masc|Number=Plur	16	obl	O	_
-19	semi-refigerados	semi-refigerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	18	acl	O	_
+19	semi-refigerados	semi-refrigerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	18	acl	O	_
 20	(	(	PUNCT	_	_	23	punct	O	_
 21	-	-	SYM	_	_	23	punct	O	_
 22	50	50	NUM	_	NumType=Card	23	nummod	O	_
@@ -117939,7 +117939,7 @@
 50	de	de	ADP	_	_	51	case	O	_
 51	descida	descida	NOUN	_	Gender=Fem|Number=Sing	49	nmod	O	_
 52	)	)	PUNCT	_	_	44	punct	O	_
-53	atinja	atinjir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
+53	atinja	atingir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
 54	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	55	det	O	_
 55	especificação	especificação	NOUN	_	Gender=Fem|Number=Sing	53	obj	O	_
 56	(	(	PUNCT	_	_	58	punct	O	_
@@ -117957,7 +117957,7 @@
 66	em	em	ADP	_	_	68	case	O	_
 67	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	68	det	O	_
 68	duto	duto	NOUN	_	Gender=Masc|Number=Sing	65	nmod	O	_
-69	subsea	subsea	ADJ	_	_	68	amod	O	_
+69	subsea	subsea	ADJ	_	Gender=Masc|Number=Sing	68	amod	O	_
 70	)	)	PUNCT	_	_	58	punct	O	_
 71	.	.	PUNCT	_	_	9	punct	O	_
 
@@ -126845,7 +126845,7 @@
 17	a	a	ADP	_	_	19	case	O	_
 18	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
 19	poucos	pouco	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	20	nsubj	O	_
-20	remova	removar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
+20	remova	remover	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	20	obj	O	_
 23	acumulado	acumular	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	22	acl	O	_
@@ -134102,7 +134102,7 @@
 23	em	em	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
 25	manômetro	manômetro	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
-26	atinja	atinjir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	advcl	O	_
+26	atinja	atingir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	advcl	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	valor	valor	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
 29	de	de	ADP	_	_	31	case	O	_
@@ -134674,7 +134674,7 @@
 13	e	e	CCONJ	_	_	15	cc	O	_
 14	então	então	ADV	_	_	15	advmod	O	_
 15-16	ajustou-se	_	_	_	_	_	_	_	_
-15	abustou	abustar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	6	conj	O	_
+15	ajustou	ajustar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	6	conj	O	_
 16	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	15	expl	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	freqüência	freqüência	NOUN	_	Gender=Fem|Number=Sing	15	obj	O	_
@@ -134835,7 +134835,7 @@
 23	em	em	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
 25	manômetro	manômetro	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
-26	atinja	atinjir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	advcl	O	_
+26	atinja	atingir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	advcl	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	valor	valor	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
 29	de	de	ADP	_	_	31	case	O	_
@@ -137592,7 +137592,7 @@
 # sent_id = 77-20141209-TESEMSC_0-369
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
 2	desemulsificantes	desemulsificante	NOUN	_	Gender=Masc|Number=Plur	3	nsubj	O	_
-3	invertem	invertar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+3	invertem	inverter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	5	det	O	_
 5	gradiente	gradiente	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
 6	devido	devido	ADP	_	_	10	case	O	MWEPOS=ADP
@@ -159808,21 +159808,21 @@
 12	usina	usina	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 13	piloto	piloto	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 14	para	para	ADP	_	_	15	case	O	_
-15	produção	produção	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
+15	produção	produção	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
 18	a	a	ADP	_	_	21	case	O	MWEPOS=ADP
 19	partir	partir	VERB	_	VerbForm=Inf	18	fixed	O	_
 20	de	de	ADP	_	_	18	fixed	O	_
-21	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
+21	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	15	obl	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	origem	origem	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
 24	vegetal	vegetal	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
-25	,	,	PUNCT	_	_	5	punct	O	_
+25	,	,	PUNCT	_	_	27	punct	O	_
 26	seja	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	27	cop	O	_
-27	bruto	bruto	ADJ	_	Gender=Masc|Number=Sing	5	conj	O	_
+27	bruto	bruto	ADJ	_	Gender=Masc|Number=Sing	21	acl	O	_
 28	(	(	PUNCT	_	_	29	punct	O	_
-29	degomado	degomar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	27	parataxis	O	_
+29	degomado	degomado	ADJ	_	Gender=Masc|Number=Sing	27	nmod:appos	O	_
 30	)	)	PUNCT	_	_	29	punct	O	_
 31	,	,	PUNCT	_	_	34	punct	O	_
 32	ou	ou	CCONJ	_	_	34	cc	O	_
@@ -162166,7 +162166,7 @@
 6	de	de	ADP	_	_	7	case	O	_
 7	soja	soja	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
 8	bruto	bruto	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
-9	degomado	degomar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	5	acl	O	_
+9	degomado	degomado	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
 10	,	,	PUNCT	_	_	3	punct	O	_
 11	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	12	det	O	_
 12	contribuição	contribuição	NOUN	_	Gender=Fem|Number=Sing	13	nsubj	O	_
@@ -162830,7 +162830,7 @@
 14	este	este	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	12	nmod	O	_
 15	para	para	ADP	_	_	17	mark	O	_
 16	que	que	SCONJ	_	_	17	mark	O	_
-17	atinja	atinjir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	acl	O	_
+17	atinja	atingir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	acl	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 19	região	região	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
 20-21	do	_	_	_	_	_	_	_	_
@@ -170507,7 +170507,7 @@
 24	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	26	nsubj	O	_
 25	não	não	ADV	_	Polarity=Neg	26	advmod	O	_
 26-27	dissolveu-se	_	_	_	_	_	_	_	_
-26	disolveu	disolver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	1	conj	O	_
+26	dissolveu	dissolver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	1	conj	O	_
 27	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	26	expl:impers	O	_
 28	impedindo	impedir	VERB	_	VerbForm=Ger	26	advcl	O	_
 29	realização	realização	NOUN	_	Gender=Fem|Number=Sing	28	obj	O	_
@@ -187050,7 +187050,7 @@
 11	,	,	PUNCT	_	_	7	punct	O	_
 12	quando	quando	SCONJ	_	_	13	mark	O	_
 13-14	depositaram-se	_	_	_	_	_	_	_	_
-13	depostaram	depostar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	7	advcl	O	_
+13	depositaram	depositar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	7	advcl	O	_
 14	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	13	expl:impers	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
 16	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	13	obj	B=LITOLOGIA	_
@@ -192036,7 +192036,7 @@
 3	volume	volume	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
 4	de	de	ADP	_	_	5	case	O	_
 5	rocha	rocha	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
-6	soerguida	soerguir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	5	acl	O	_
+6	soerguida	soerguer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	5	acl	O	_
 7	possuía	possuir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	0	root	O	_
 8	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	9	det	O	_
 9	borda	borda	NOUN	_	Gender=Fem|Number=Sing	7	obj	O	_
@@ -201201,7 +201201,7 @@
 37	encontrar	encontrar	VERB	_	VerbForm=Inf	28	advcl	O	_
 38	aqueles	aquele	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	37	obj	O	_
 39	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	40	nsubj	O	_
-40	satisfaçam	satisfaçor	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	38	acl:relcl	O	_
+40	satisfaçam	satisfazer	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	38	acl:relcl	O	_
 41	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	42	det	O	_
 42	imposições	imposição	NOUN	_	Gender=Fem|Number=Plur	40	obj	O	_
 43-44	das	_	_	_	_	_	_	_	_
@@ -235707,7 +235707,7 @@
 13	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
 14	que	que	SCONJ	_	_	16	mark	O	_
 15	este	este	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	16	nsubj	O	_
-16	possua	possuar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	csubj	O	_
+16	possua	possuir	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	csubj	O	_
 17	certas	certo	DET	_	Gender=Fem|Number=Plur|PronType=Ind	18	det	O	_
 18	características	característica	NOUN	_	Gender=Fem|Number=Plur	16	obj	O	_
 19	,	,	PUNCT	_	_	21	punct	O	_
@@ -279553,7 +279553,7 @@
 64	todos	todo	DET	_	Gender=Masc|Number=Plur|PronType=Tot	66	det	O	_
 65	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	66	det	O	_
 66	fluxos	fluxo	NOUN	_	Gender=Masc|Number=Plur	67	nsubj	B=ESTRUTURA_DEPOSICIONAL	_
-67	interajam	interar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	61	ccomp	O	_
+67	interajam	interagir	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	61	ccomp	O	_
 68	entre	entre	ADP	_	_	69	case	O	_
 69	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	67	obl	O	_
 70	,	,	PUNCT	_	_	71	punct	O	_
@@ -279760,7 +279760,7 @@
 14	clima	clima	NOUN	_	Gender=Masc|Number=Sing	11	obl:agent	O	_
 15	equatorial	equatorial	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
-17	semiárido	semiárir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	14	acl	O	_
+17	semiárido	semiárido	ADJ	_	Gender=Masc|Number=Sing	15	conj	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
 19	recebe	receber	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	conj	O	_
 20	influência	influência	NOUN	_	Gender=Fem|Number=Sing	19	obj	O	_
@@ -291313,27 +291313,27 @@
 20	partes	parte	NOUN	_	Gender=Fem|Number=Plur	15	conj	O	_
 21	cíclicas	cíclico	ADJ	_	Gender=Fem|Number=Plur	20	amod	O	_
 22	)	)	PUNCT	_	_	15	punct	O	_
-23	,	,	PUNCT	_	_	8	punct	O	_
-24	pois	pois	SCONJ	_	_	30	mark	O	_
+23	,	,	PUNCT	_	_	34	punct	O	_
+24	pois	pois	CCONJ	_	_	42	cc	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
-26	carbonos	carbono	NOUN	_	Gender=Masc|Number=Plur	30	nsubj	O	_
+26	carbonos	carbono	NOUN	_	Gender=Masc|Number=Plur	42	nsubj	O	_
 27-28	dessa	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	29	det	O	_
 29	cadeia	cadeia	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
-30	apesarem	apesar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	8	advcl	O	_
+30	apesarem	apesar	ADP	_	_	34	case	O	MWEPOS=ADP
 31	de	de	SCONJ	_	_	34	mark	O	_
 32	serem	ser	AUX	_	Number=Plur|Person=3|VerbForm=Inf	34	cop	O	_
 33	mais	mais	ADV	_	_	34	advmod	O	_
-34	eletronegativos	eletronegativo	ADJ	_	Gender=Masc|Number=Plur	30	ccomp	O	_
+34	eletronegativos	eletronegativo	ADJ	_	Gender=Masc|Number=Plur	42	advcl	O	_
 35	que	que	SCONJ	_	_	42	mark	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
 37	átomos	átomo	NOUN	_	Gender=Masc|Number=Plur	42	nsubj	O	_
 38	de	de	ADP	_	_	39	case	O	_
 39	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	37	nmod	O	_
-40	,	,	PUNCT	_	_	42	punct	O	_
+40	,	,	PUNCT	_	_	34	punct	O	_
 41	não	não	ADV	_	Polarity=Neg	42	advmod	O	_
-42	formam	formar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	34	csubj	O	_
+42	formam	formar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	conj	O	_
 43	pólos	pólo	NOUN	_	Gender=Masc|Number=Plur	42	obj	O	_
 44	de	de	ADP	_	_	45	case	O	_
 45	concentração	concentração	NOUN	_	Gender=Fem|Number=Sing	43	nmod	O	_
@@ -293470,7 +293470,7 @@
 22	de	de	ADP	_	_	21	fixed	O	_
 23	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	25	nsubj:pass	O	_
 24	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	25	aux:pass	O	_
-25	homogenizado	homogenizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	1	advcl	O	_
+25	homogenizado	homogeneizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	1	advcl	O	_
 26	e	e	CCONJ	_	_	29	cc	O	_
 27	em	em	ADP	_	_	28	case	O	_
 28	seguida	seguida	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
