commit 3e3e48ea493b7aeea4db59da4abf09577e11859d
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Fri Oct 8 22:13:49 2021 -0300

    Referências como nmod:appos

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b44a284..aa3672a 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1062,9 +1062,9 @@
 4	Luiz	Luiz	PROPN	_	Gender=Masc|Number=Sing	20	obl	O	_
 5	&	&	PROPN	_	Number=Sing	4	flat:name	O	_
 6	Silva	Silva	PROPN	_	Number=Sing	4	flat:name	O	_
-7	(	(	PUNCT	_	_	4	flat:name	O	_
-8	1995	1995	NUM	_	NumType=Card	4	flat:name	O	_
-9	)	)	PUNCT	_	_	4	flat:name	O	_
+7	(	(	PUNCT	_	_	8	punct	O	_
+8	1995	1995	NUM	_	NumType=Card	4	nmod:appos	O	_
+9	)	)	PUNCT	_	_	8	punct	O	_
 10	,	,	PUNCT	_	_	4	punct	O	_
 11	frequentemente	frequentemente	ADV	_	_	20	advmod	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -14938,9 +14938,9 @@
 # sent_id = 241-20140227-MONOGRAFIA_0-164
 1	Segundo	segundo	ADP	_	_	2	case	O	_
 2	James	James	PROPN	_	Gender=Masc|Number=Sing	11	obl	O	_
-3	(	(	PUNCT	_	_	2	flat:name	O	_
-4	2006	2006	NUM	_	NumType=Card	2	flat:name	O	_
-5	)	)	PUNCT	_	_	2	flat:name	O	_
+3	(	(	PUNCT	_	_	4	punct	O	_
+4	2006	2006	NUM	_	NumType=Card	2	nmod:appos	O	_
+5	)	)	PUNCT	_	_	4	punct	O	_
 6	,	,	PUNCT	_	_	2	punct	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
 8	análises	análise	NOUN	_	Gender=Fem|Number=Plur	11	nsubj	O	_
@@ -20652,9 +20652,9 @@
 26	)	)	PUNCT	_	_	25	punct	O	_
 27	,	,	PUNCT	_	_	28	punct	O	_
 28	Assine	Assine	PROPN	_	Gender=Masc|Number=Sing	21	conj	O	_
-29	(	(	PUNCT	_	_	28	flat:name	O	_
-30	1990	1990	NUM	_	NumType=Card	28	flat:name	O	_
-31	)	)	PUNCT	_	_	28	punct	O	_
+29	(	(	PUNCT	_	_	30	punct	O	_
+30	1990	1990	NUM	_	NumType=Card	28	nmod:appos	O	_
+31	)	)	PUNCT	_	_	30	punct	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
 33	Arai	Arai	PROPN	_	Gender=Masc|Number=Sing	21	conj	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
@@ -22525,9 +22525,9 @@
 3	subsequente	subsequente	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	,	,	PUNCT	_	_	2	punct	O	_
 5	Beurlen	Beurlen	PROPN	_	Gender=Masc|Number=Sing	9	nsubj	O	_
-6	(	(	PUNCT	_	_	5	flat:name	O	_
-7	1971	1971	NUM	_	NumType=Card	5	flat:name	O	_
-8	)	)	PUNCT	_	_	5	flat:name	O	_
+6	(	(	PUNCT	_	_	7	punct	O	_
+7	1971	1971	NUM	_	NumType=Card	5	nmod:appos	O	_
+8	)	)	PUNCT	_	_	7	punct	O	_
 9-10	refere-se	_	_	_	_	_	_	_	_
 9	refere	referir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	9	expl	O	_
@@ -22993,9 +22993,9 @@
 33	,	,	PUNCT	_	_	26	punct	O	_
 34	segundo	segundo	ADP	_	_	35	case	O	_
 35	Beurlen	Beurlen	PROPN	_	Gender=Masc|Number=Sing	26	obl	O	_
-36	(	(	PUNCT	_	_	35	flat:name	O	_
-37	1966	1966	NUM	_	NumType=Card	35	flat:name	O	_
-38	)	)	PUNCT	_	_	35	punct	O	_
+36	(	(	PUNCT	_	_	37	punct	O	_
+37	1966	1966	NUM	_	NumType=Card	35	nmod:appos	O	_
+38	)	)	PUNCT	_	_	37	punct	O	_
 39	e	e	CCONJ	_	_	40	cc	O	_
 40	Coimbra	Coimbra	PROPN	_	Gender=Masc|Number=Sing	35	conj	O	_
 41	et	et	PROPN	_	Number=Sing	40	flat:name	O	_
@@ -27355,9 +27355,9 @@
 1	Bruno	Bruno	PROPN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
 2	et	et	PROPN	_	Number=Sing	1	flat:name	O	_
 3	al.	al.	PROPN	_	Number=Sing	1	flat:name	O	_
-4	(	(	PUNCT	_	_	1	flat:name	O	_
-5	2006	2006	NUM	_	NumType=Card	1	flat:name	O	_
-6	)	)	PUNCT	_	_	1	flat:name	O	_
+4	(	(	PUNCT	_	_	5	punct	O	_
+5	2006	2006	NUM	_	NumType=Card	1	nmod:appos	O	_
+6	)	)	PUNCT	_	_	5	punct	O	_
 7	reconheceram	reconhecer	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
 8	fortes	forte	ADJ	_	Gender=Fem|Number=Plur	9	amod	O	_
 9	influências	influência	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
@@ -29449,9 +29449,9 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-385
 1	Segundo	segundo	ADP	_	_	2	case	O	_
 2	Silva	Silva	PROPN	_	Gender=Masc|Number=Sing	9	obl	O	_
-3	(	(	PUNCT	_	_	2	flat:name	O	_
-4	1986	1986	NUM	_	NumType=Card	2	flat:name	O	_
-5	)	)	PUNCT	_	_	2	flat:name	O	_
+3	(	(	PUNCT	_	_	4	punct	O	_
+4	1986	1986	NUM	_	NumType=Card	2	nmod:appos	O	_
+5	)	)	PUNCT	_	_	4	punct	O	_
 6	,	,	PUNCT	_	_	2	punct	O	_
 7	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	8	det	O	_
 8	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
@@ -60111,24 +60111,24 @@
 13	formação	formação	NOUN	_	Gender=Fem|Number=Sing	7	conj	O	_
 14	por	por	ADP	_	_	15	case	O	_
 15	Lima	Lima	PROPN	_	Gender=Masc|Number=Sing	13	nmod	O	_
-16	(	(	PUNCT	_	_	15	flat:name	O	_
-17	1979	1979	NUM	_	NumType=Card	15	flat:name	O	_
+16	(	(	PUNCT	_	_	17	punct	O	_
+17	1979	1979	NUM	_	NumType=Card	15	nmod:appos	O	_
 18	)	)	PUNCT	_	_	17	punct	O	_
 19	,	,	PUNCT	_	_	20	punct	O	_
 20	Seer	Seer	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
-21	(	(	PUNCT	_	_	20	flat:name	O	_
-22	1989	1989	NUM	_	NumType=Card	20	flat:name	O	_
-23	)	)	PUNCT	_	_	20	punct	O	_
+21	(	(	PUNCT	_	_	22	punct	O	_
+22	1989	1989	NUM	_	NumType=Card	20	nmod:appos	O	_
+23	)	)	PUNCT	_	_	22	punct	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
 25	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
-26	(	(	PUNCT	_	_	25	flat:name	O	_
-27	1989	1989	NUM	_	NumType=Card	25	flat:name	O	_
-28	)	)	PUNCT	_	_	25	punct	O	_
+26	(	(	PUNCT	_	_	27	punct	O	_
+27	1989	1989	NUM	_	NumType=Card	25	nmod:appos	O	_
+28	)	)	PUNCT	_	_	27	punct	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
 30	Castro	Castro	PROPN	_	Gender=Masc|Number=Sing	7	conj	O	_
-31	(	(	PUNCT	_	_	30	flat:name	O	_
-32	1996	1996	NUM	_	NumType=Card	30	flat:name	O	_
-33	)	)	PUNCT	_	_	30	punct	O	_
+31	(	(	PUNCT	_	_	32	punct	O	_
+32	1996	1996	NUM	_	NumType=Card	30	nmod:appos	O	_
+33	)	)	PUNCT	_	_	32	punct	O	_
 34	.	.	PUNCT	_	_	3	punct	O	_
 
 # text = Contudo, desde os trabalhos de Campos e Dardenne (1997) há praticamente consenso em considerar como Grupo Areado.
@@ -60378,9 +60378,9 @@
 4	,	,	PUNCT	_	_	6	punct	O	_
 5	segundo	segundo	ADP	_	_	6	case	O	_
 6	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	11	obl	O	_
-7	(	(	PUNCT	_	_	6	flat:name	O	_
-8	2011	2011	NUM	_	NumType=Card	6	flat:name	O	_
-9	)	)	PUNCT	_	_	6	flat:name	O	_
+7	(	(	PUNCT	_	_	8	punct	O	_
+8	2011	2011	NUM	_	NumType=Card	6	nmod:appos	O	_
+9	)	)	PUNCT	_	_	8	punct	O	_
 10	,	,	PUNCT	_	_	6	punct	O	_
 11	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12-13	na	_	_	_	_	_	_	_	_
@@ -62297,9 +62297,9 @@
 25	modificado	modificar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	17	conj	O	_
 26	por	por	ADP	_	_	27	case	O	_
 27	Miall	Miall	PROPN	_	Gender=Masc|Number=Sing	25	obl:agent	O	_
-28	(	(	PUNCT	_	_	27	flat:name	O	_
-29	1996	1996	NUM	_	NumType=Card	27	flat:name	O	_
-30	)	)	PUNCT	_	_	27	punct	O	_
+28	(	(	PUNCT	_	_	29	punct	O	_
+29	1996	1996	NUM	_	NumType=Card	27	nmod:appos	O	_
+30	)	)	PUNCT	_	_	29	punct	O	_
 31	,	,	PUNCT	_	_	34	punct	O	_
 32	mas	mas	CCONJ	_	_	34	cc	O	_
 33	com	com	ADP	_	_	34	case	O	_
@@ -103605,9 +103605,9 @@
 1	Vichitrananda	Vichitrananda	PROPN	_	Gender=Fem|Number=Sing	7	nsubj	O	_
 2	et	et	PROPN	_	Number=Sing	1	flat:name	O	_
 3	al.	al.	PROPN	_	Number=Sing	1	flat:name	O	_
-4	(	(	PUNCT	_	_	1	flat:name	O	_
-5	2012	2012	NUM	_	NumType=Card	1	flat:name	O	_
-6	)	)	PUNCT	_	_	1	flat:name	O	_
+4	(	(	PUNCT	_	_	5	punct	O	_
+5	2012	2012	NUM	_	NumType=Card	1	nmod:appos	O	_
+6	)	)	PUNCT	_	_	5	punct	O	_
 7	destacam	destacar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	que	que	SCONJ	_	_	27	mark	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
@@ -134498,8 +134498,8 @@
 # text = Vanegas (1999) projetou o vaso separador gravitacional que foi utilizado na realização do presente trabalho e que também foi utilizado por Vara (2001), Rodrigues (2002), Trevisan (2003) e Barbosa (2004) na realização de seus trabalhos.
 # sent_id = 77-20141209-TESEMSC_0-205
 1	Vanegas	Vanegas	PROPN	_	Gender=Masc|Number=Sing	5	nsubj	O	_
-2	(	(	PUNCT	_	_	1	flat:name	O	_
-3	1999	1999	NUM	_	NumType=Card	1	flat:name	O	_
+2	(	(	PUNCT	_	_	3	punct	O	_
+3	1999	1999	NUM	_	NumType=Card	1	nmod:appos	O	_
 4	)	)	PUNCT	_	_	3	punct	O	_
 5	projetou	projetar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -143345,9 +143345,9 @@
 2	Grando	Grando	PROPN	_	Gender=Masc|Number=Sing	45	obl	O	_
 3	et	et	PROPN	_	Number=Sing	2	flat:name	O	_
 4	al.	al.	PROPN	_	Number=Sing	2	flat:name	O	_
-5	[	[	PUNCT	_	_	2	flat:name	O	_
-6	2	2	NUM	_	NumType=Card	2	flat:name	O	_
-7	]	]	PUNCT	_	_	2	flat:name	O	_
+5	[	[	PUNCT	_	_	6	punct	O	_
+6	2	2	NUM	_	NumType=Card	2	nmod:appos	O	_
+7	]	]	PUNCT	_	_	6	punct	O	_
 8	,	,	PUNCT	_	_	2	punct	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 10	crescente	crescente	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
@@ -186202,8 +186202,8 @@
 22	Silva	Silva	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
 23	e	e	CCONJ	_	_	24	cc	O	_
 24	Medeiros	Medeiros	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
-25	(	(	PUNCT	_	_	24	flat:name	O	_
-26	1999	1999	NUM	_	NumType=Card	24	flat:name	O	_
+25	(	(	PUNCT	_	_	26	punct	O	_
+26	1999	1999	NUM	_	NumType=Card	24	nmod:appos	O	_
 27	)	)	PUNCT	_	_	26	punct	O	_
 28	e	e	CCONJ	_	_	29	cc	O	_
 29	Silva	Silva	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
@@ -191186,9 +191186,9 @@
 30	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	22	acl	O	_
 31	por	por	ADP	_	_	32	case	O	_
 32	Riccomini	Riccomini	PROPN	_	Gender=Masc|Number=Sing	30	obl:agent	O	_
-33	(	(	PUNCT	_	_	32	flat:name	O	_
-34	1989	1989	NUM	_	NumType=Card	32	flat:name	O	_
-35	)	)	PUNCT	_	_	32	punct	O	_
+33	(	(	PUNCT	_	_	34	punct	O	_
+34	1989	1989	NUM	_	NumType=Card	32	nmod:appos	O	_
+35	)	)	PUNCT	_	_	34	punct	O	_
 36	.	.	PUNCT	_	_	13	punct	O	_
 
 # text = Sua área de ocorrência coincide com toda extensão da Serra do Mar e, em parte, com a da serra da Mantiqueira.
@@ -193622,9 +193622,9 @@
 37	)	)	PUNCT	_	_	36	punct	O	_
 38	e	e	CCONJ	_	_	39	cc	O	_
 39	Araujo	Araujo	PROPN	_	Gender=Masc|Number=Sing	25	conj	O	_
-40	(	(	PUNCT	_	_	39	flat:name	O	_
-41	1995	1995	NUM	_	NumType=Card	39	flat:name	O	_
-42	)	)	PUNCT	_	_	39	punct	O	_
+40	(	(	PUNCT	_	_	41	punct	O	_
+41	1995	1995	NUM	_	NumType=Card	39	nmod:appos	O	_
+42	)	)	PUNCT	_	_	41	punct	O	_
 43	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = Figura extraída de Ferrari (2001)
@@ -205481,9 +205481,9 @@
 1	Barbosa	Barbosa	PROPN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
 2	e	e	CCONJ	_	_	1	flat:name	O	_
 3	Silva	Silva	PROPN	_	Number=Sing	1	flat:name	O	_
-4	(	(	PUNCT	_	_	1	flat:name	O	_
-5	1994	1994	NUM	_	NumType=Card	1	flat:name	O	_
-6	)	)	PUNCT	_	_	1	flat:name	O	_
+4	(	(	PUNCT	_	_	5	punct	O	_
+5	1994	1994	NUM	_	NumType=Card	1	nmod:appos	O	_
+6	)	)	PUNCT	_	_	5	punct	O	_
 7	generalizaram	generalizar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
 9	método	método	NOUN	_	Gender=Masc|Number=Sing	7	obj	O	_
@@ -231715,9 +231715,9 @@
 48	trabalhos	trabalho	NOUN	_	Gender=Masc|Number=Plur	45	obl	O	_
 49	como	como	ADP	_	_	50	case	O	_
 50	Fetter	Fetter	PROPN	_	Gender=Masc|Number=Sing	45	xcomp	O	_
-51	(	(	PUNCT	_	_	50	flat:name	O	_
-52	2008	2008	NUM	_	NumType=Card	50	flat:name	O	_
-53	)	)	PUNCT	_	_	50	flat:name	O	_
+51	(	(	PUNCT	_	_	52	punct	O	_
+52	2008	2008	NUM	_	NumType=Card	50	nmod:appos	O	_
+53	)	)	PUNCT	_	_	52	punct	O	_
 54	e	e	CCONJ	_	_	55	cc	O	_
 55	Oreiro	Oreiro	PROPN	_	Gender=Masc|Number=Sing	50	conj	O	_
 56	et	et	PROPN	_	Number=Sing	55	flat:name	O	_
