commit 8b98a96c5d4e804102e3353e3ce92f0332d5fe20
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Sep 23 15:06:45 2021 -0300

    'ocorrer' não deve ter objeto

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 8b7ac26..c97d2f0 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -2518,7 +2518,7 @@
 7	,	,	PUNCT	_	_	3	punct	O	_
 8	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 9	marcante	marcante	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
-10	aloctonia	aloctonia	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
+10	aloctonia	aloctonia	NOUN	_	Gender=Fem|Number=Sing	8	nsubj	O	_
 11	e	e	CCONJ	_	_	13	cc	O	_
 12	elevado	elevado	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 13	grau	grau	NOUN	_	Gender=Masc|Number=Sing	10	conj	O	_
@@ -3186,7 +3186,7 @@
 19	,	,	PUNCT	_	_	20	punct	O	_
 20	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	2	advcl	O	_
 21	também	também	ADV	_	_	20	advmod	O	_
-22	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	20	obj	O	_
+22	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	20	nsubj	O	_
 23	síltico-argilosos	síltico-argiloso	ADJ	_	Gender=Masc|Number=Plur	22	amod	O	_
 24	e	e	CCONJ	_	_	25	cc	O	_
 25	carbonatos	carbonato	NOUN	_	Gender=Masc|Number=Plur	22	conj	O	_
@@ -5852,7 +5852,7 @@
 1	Ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	também	também	ADV	_	_	1	advmod	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
-4	deslocamento	deslocamento	NOUN	_	Gender=Masc|Number=Sing	1	obj	O	_
+4	deslocamento	deslocamento	NOUN	_	Gender=Masc|Number=Sing	1	nsubj	O	_
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -7033,7 +7033,7 @@
 26	também	também	ADV	_	_	27	advmod	O	_
 27	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	advcl	O	_
 28	elevadas	elevar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	29	acl	O	_
-29	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	27	obj	O	_
+29	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	27	nsubj	O	_
 30	gravimétricas	gravimétrico	ADJ	_	Gender=Fem|Number=Plur	29	amod	O	_
 31	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	35	nsubj	O	_
 32	também	também	ADV	_	_	35	advmod	O	_
@@ -12117,7 +12117,7 @@
 50	que	que	SCONJ	_	_	51	mark	O	_
 51	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	48	ccomp	O	_
 52	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	53	det	O	_
-53	coalescência	coalescência	NOUN	_	Gender=Fem|Number=Sing	51	obj	O	_
+53	coalescência	coalescência	NOUN	_	Gender=Fem|Number=Sing	51	nsubj	O	_
 54-55	das	_	_	_	_	_	_	_	_
 54	de	de	ADP	_	_	56	case	O	_
 55	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	56	det	O	_
@@ -14595,7 +14595,7 @@
 24	que	que	SCONJ	_	_	25	mark	O	_
 25	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	22	advcl	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	homogeneização	homogeneização	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
+27	homogeneização	homogeneização	NOUN	_	Gender=Fem|Number=Sing	25	nsubj	O	_
 28	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = 4-3-2-Obtenção da fase aquosa
@@ -17098,7 +17098,7 @@
 3	ter	ter	AUX	_	VerbForm=Inf	4	aux	O	_
 4	ocorrido	ocorrer	VERB	_	VerbForm=Part	9	advcl	O	_
 5	tal	tal	DET	_	Gender=Masc|Number=Sing|PronType=Ind	6	det	O	_
-6	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+6	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
 7	,	,	PUNCT	_	_	4	punct	O	_
 8	não	não	ADV	_	Polarity=Neg	9	advmod	O	_
 9	houve	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
@@ -20112,7 +20112,7 @@
 4	de	de	ADP	_	_	5	case	O	_
 5	ravinamento	ravinamento	NOUN	_	Gender=Masc|Number=Sing	3	nmod	O	_
 6	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-7	joeiramento	joeiramento	NOUN	_	Gender=Masc|Number=Sing	6	obj	O	_
+7	joeiramento	joeiramento	NOUN	_	Gender=Masc|Number=Sing	6	nsubj	O	_
 8-9	dos	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
@@ -27393,7 +27393,7 @@
 35	Romualdo	Romualdo	PROPN	_	Number=Sing	34	flat:name	O	_
 36	,	,	PUNCT	_	_	30	punct	O	_
 37	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	26	ccomp	O	_
-38	peixes	peixe	NOUN	_	Gender=Masc|Number=Plur	37	obj	O	_
+38	peixes	peixe	NOUN	_	Gender=Masc|Number=Plur	37	nsubj	O	_
 39	osteíctes	osteícte	ADJ	_	Gender=Masc|Number=Plur	38	amod	O	_
 40	marinhos	marinho	ADJ	_	Gender=Masc|Number=Plur	38	amod	O	_
 41	como	como	ADP	_	_	42	case	O	_
@@ -27566,7 +27566,7 @@
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	SIM	SIM	NOUN	_	Gender=Fem|Number=Sing	1	obl	O	_
 5	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-6	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
+6	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	5	nsubj	O	_
 7	regressivo	regressivo	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	10	nsubj	O	_
 9	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	10	expl	O	_
@@ -28678,7 +28678,7 @@
 12	carbonizados	carbonizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	10	acl	O	_
 13	;	;	PUNCT	_	_	14	punct	O	_
 14	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	obj	O	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nsubj	O	_
 16	heterolíticas	heterolítica	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
 17	intensamente	intensamente	ADV	_	_	18	advmod	O	_
 18	bioturbadas	bioturbar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	acl	O	_
@@ -29798,7 +29798,7 @@
 26	,	,	PUNCT	_	_	28	punct	O	_
 27	tendo	ter	AUX	_	VerbForm=Ger	28	aux	O	_
 28	ocorrido	ocorrer	VERB	_	VerbForm=Part	14	advcl	O	_
-29	evento	evento	NOUN	_	Gender=Masc|Number=Sing	28	obj	O	_
+29	evento	evento	NOUN	_	Gender=Masc|Number=Sing	28	nsubj	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	mortandade	mortandade	NOUN	_	Gender=Fem|Number=Sing	29	nmod	O	_
 32	quando	quando	ADV	_	_	28	advmod	O	_
@@ -30211,7 +30211,7 @@
 55	de	de	ADP	_	_	56	case	O	_
 56	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	57	nsubj	O	_
 57	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	54	acl:relcl	O	_
-58	raseamento	raseamento	NOUN	_	Gender=Masc|Number=Sing	57	obj	O	_
+58	raseamento	raseamento	NOUN	_	Gender=Masc|Number=Sing	57	nsubj	O	_
 59	para	para	ADP	_	_	61	case	O	_
 60	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	61	det	O	_
 61	topo	topo	NOUN	_	Gender=Masc|Number=Sing	57	obl	O	_
@@ -42898,7 +42898,7 @@
 24	quando	quando	SCONJ	_	_	26	mark	O	_
 25	tiver	ter	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	26	aux	O	_
 26	ocorrido	ocorrer	VERB	_	VerbForm=Part	16	advcl	O	_
-27	perda	perda	NOUN	_	Gender=Fem|Number=Sing	26	obj	O	_
+27	perda	perda	NOUN	_	Gender=Fem|Number=Sing	26	nsubj	O	_
 28	de	de	ADP	_	_	29	case	O	_
 29	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	27	nmod	O	_
 30	para	para	ADP	_	_	32	case	O	_
@@ -42988,7 +42988,7 @@
 # sent_id = 24-20150122-MONOGRAFIA_0-56
 1	Embora	embora	SCONJ	_	_	2	mark	O	_
 2	ocorram	ocorrer	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
-3	divergências	divergência	NOUN	_	Gender=Fem|Number=Plur	2	obj	O	_
+3	divergências	divergência	NOUN	_	Gender=Fem|Number=Plur	2	nsubj	O	_
 4	,	,	PUNCT	_	_	2	punct	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 6	principal	principal	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
@@ -43799,7 +43799,7 @@
 18	evitar	evitar	VERB	_	VerbForm=Inf	11	advcl	O	_
 19	que	que	SCONJ	_	_	20	mark	O	_
 20	ocorram	ocorrer	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	18	ccomp	O	_
-21	perdas	perda	NOUN	_	Gender=Fem|Number=Plur	20	obj	O	_
+21	perdas	perda	NOUN	_	Gender=Fem|Number=Plur	20	nsubj	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	circulação	circulação	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
 24	em	em	ADP	_	_	25	case	O	_
@@ -46505,7 +46505,7 @@
 12	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	csubj	O	_
 13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 14	pequeno	pequeno	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
-15	rearranjo	rearranjo	NOUN	_	Gender=Masc|Number=Sing	12	obj	O	_
+15	rearranjo	rearranjo	NOUN	_	Gender=Masc|Number=Sing	12	nsubj	O	_
 16-17	das	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	19	case	O	_
 17	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	19	det	O	_
@@ -47383,7 +47383,7 @@
 1	Praticamente	praticamente	ADV	_	_	3	advmod	O	_
 2	não	não	ADV	_	Polarity=Neg	3	advmod	O	_
 3	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-4	substituições	substituição	NOUN	_	Gender=Fem|Number=Plur	3	obj	O	_
+4	substituições	substituição	NOUN	_	Gender=Fem|Number=Plur	3	nsubj	O	_
 5	por	por	ADP	_	_	6	case	O	_
 6	cátions	cátion	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 7	dentro	dentro	ADV	_	_	3	advmod	O	_
@@ -55244,7 +55244,7 @@
 18	tenha	ter	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	19	aux	O	_
 19	ocorrido	ocorrer	VERB	_	VerbForm=Part	16	ccomp	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
-21	substituição	substituição	NOUN	_	Gender=Fem|Number=Sing	19	obj	O	_
+21	substituição	substituição	NOUN	_	Gender=Fem|Number=Sing	19	nsubj	O	_
 22	isomórfica	isomórfico	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
@@ -60568,7 +60568,7 @@
 # sent_id = 283-20150917-MONOGRAFIA_0-51
 1	Localmente	localmente	ADV	_	_	2	advmod	O	_
 2	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-3	calcários	calcário	NOUN	_	Gender=Masc|Number=Plur	2	obj	O	_
+3	calcários	calcário	NOUN	_	Gender=Masc|Number=Plur	2	nsubj	O	_
 4	micríticos	micrítico	ADJ	_	Gender=Masc|Number=Plur	3	amod	O	_
 5	cinza	cinza	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 6	esverdeados	esverdeado	ADJ	_	Gender=Masc|Number=Plur	5	compound	O	_
@@ -63171,7 +63171,7 @@
 4	Chapadão	Chapadão	PROPN	_	Gender=Fem|Number=Sing	3	nmod	O	_
 5	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	maiores	grande	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
-7	espessuras	espessura	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
+7	espessuras	espessura	NOUN	_	Gender=Fem|Number=Plur	5	nsubj	O	_
 8	para	para	ADP	_	_	10	case	O	_
 9	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	10	det	O	_
 10	associação	associação	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
@@ -63461,7 +63461,7 @@
 10	mas	mas	CCONJ	_	_	12	cc	O	_
 11	também	também	ADV	_	_	12	advmod	O	_
 12	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	conj	O	_
-13	sets	set	NOUN	_	Gender=Masc|Number=Plur	12	obj	O	_
+13	sets	set	NOUN	_	Gender=Masc|Number=Plur	12	nsubj	O	_
 14	cuneiformes	cuneiforme	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 15	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -64028,7 +64028,7 @@
 29	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	18	advcl	O	_
 30	apenas	apenas	ADV	_	_	29	advmod	O	_
 31	a	a	ADP	_	_	32	case	O	_
-32	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	29	obl:arg	O	_
+32	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	29	nsubj	O	_
 33	G	g	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 34	,	,	PUNCT	_	_	36	punct	O	_
 35	bem	bem	ADV	_	_	36	advmod	O	_
@@ -65128,7 +65128,7 @@
 6	,	,	PUNCT	_	_	1	punct	O	_
 7	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	aproximadamente	aproximadamente	ADV	_	_	7	advmod	O	_
-9	20m	20m	NOUN	_	Gender=Masc|Number=Plur	7	obj	O	_
+9	20m	20m	NOUN	_	Gender=Masc|Number=Plur	7	nsubj	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	seção	seção	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	psamítica	psamítico	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
@@ -65366,7 +65366,7 @@
 16	a	a	SCONJ	_	_	17	mark	O	_
 17	ocorrer	ocorrer	VERB	_	VerbForm=Inf	15	xcomp	O	_
 18	apenas	apenas	ADV	_	_	19	advmod	O	_
-19	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	17	obj	O	_
+19	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	17	nsubj	O	_
 20	arenosos	arenoso	ADJ	_	Gender=Masc|Number=Plur	19	amod	O	_
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
@@ -66622,7 +66622,7 @@
 # sent_id = 283-20150917-MONOGRAFIA_0-295
 1	Também	também	ADV	_	_	2	advmod	O	_
 2	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-3	interação	interação	NOUN	_	Gender=Fem|Number=Sing	2	obj	O	_
+3	interação	interação	NOUN	_	Gender=Fem|Number=Sing	2	nsubj	O	_
 4	flúvio-deltaica	flúvio-deltaico	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 5	,	,	PUNCT	_	_	6	punct	O	_
 6	representada	representar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	3	acl	O	_
@@ -67336,7 +67336,7 @@
 12	a	a	SCONJ	_	_	13	mark	O	_
 13	ocorrerem	ocorrer	VERB	_	Number=Plur|Person=3|VerbForm=Inf	11	xcomp	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
-15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	13	obj	O	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	13	nsubj	O	_
 16	Sm	Sm	PROPN	_	Gender=Fem|Number=Sing	13	nmod	O	_
 17	,	,	PUNCT	_	_	18	punct	O	_
 18	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	16	conj	O	_
@@ -67541,7 +67541,7 @@
 18	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	20	obl	O	_
 19	,	,	PUNCT	_	_	1	punct	O	_
 20	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-21	laminações	laminação	NOUN	_	Gender=Fem|Number=Plur	20	obj	O	_
+21	laminações	laminação	NOUN	_	Gender=Fem|Number=Plur	20	nsubj	O	_
 22	cruzadas	cruzar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	21	acl	O	_
 23	cavalgantes	cavalgante	ADJ	_	Gender=Fem|Number=Plur	21	amod	O	_
 24	(	(	PUNCT	_	_	25	punct	O	_
@@ -67734,7 +67734,7 @@
 11	a	a	SCONJ	_	_	12	mark	O	_
 12	ocorrer	ocorrer	VERB	_	VerbForm=Inf	10	xcomp	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	intercalação	intercalação	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
+14	intercalação	intercalação	NOUN	_	Gender=Fem|Number=Sing	12	nsubj	O	_
 15-16	das	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	17	det	O	_
@@ -67808,7 +67808,7 @@
 4	descontinuidade	descontinuidade	NOUN	_	Gender=Fem|Number=Sing	1	obl	O	_
 5	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	prioritariamente	prioritariamente	ADV	_	_	5	advmod	O	_
-7	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
+7	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	5	nsubj	O	_
 8-9	da	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
@@ -67928,7 +67928,7 @@
 1	St	St	PROPN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
 3	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-4	sets	set	NOUN	_	Gender=Masc|Number=Plur	3	obj	O	_
+4	sets	set	NOUN	_	Gender=Masc|Number=Plur	3	nsubj	O	_
 5	decimétricos	decimétrico	ADJ	_	Gender=Masc|Number=Plur	4	amod	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
@@ -68482,7 +68482,7 @@
 16	,	,	PUNCT	_	_	15	punct	O	_
 17	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	associação	associação	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
+19	associação	associação	NOUN	_	Gender=Fem|Number=Sing	17	nsubj	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	19	nmod	O	_
 22	St	St	PROPN	_	Gender=Masc|Number=Sing	21	nmod	O	_
@@ -68707,7 +68707,7 @@
 20	poderem	poder	VERB	_	Number=Plur|Person=3|VerbForm=Inf	30	conj	O	_
 21	ocorrer	ocorrer	VERB	_	VerbForm=Inf	20	xcomp	O	_
 22	grandes	grande	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
-23	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	21	obj	O	_
+23	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	21	nsubj	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	interdunas	interduna	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
 26	secas	seco	ADJ	_	Gender=Fem|Number=Plur	25	amod	O	_
@@ -69916,7 +69916,7 @@
 22	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
 23	ocorria	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	0	root	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	sedimentação	sedimentação	NOUN	_	Gender=Fem|Number=Sing	23	obj	O	_
+25	sedimentação	sedimentação	NOUN	_	Gender=Fem|Number=Sing	23	nsubj	O	_
 26	em	em	ADP	_	_	27	case	O	_
 27	lagos	lago	NOUN	_	Gender=Masc|Number=Plur	23	obl	O	_
 28	rasos	raso	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
@@ -70143,7 +70143,7 @@
 4	eólico	eólico	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 5	frequentemente	frequentemente	ADV	_	_	6	advmod	O	_
 6	ocorriam	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	0	root	O	_
-7	dunas	duna	NOUN	_	Gender=Fem|Number=Plur	6	obj	O	_
+7	dunas	duna	NOUN	_	Gender=Fem|Number=Plur	6	nsubj	O	_
 8	métricas	métrico	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 9	a	a	ADP	_	_	10	case	O	_
 10	decamétricas	decamétrica	NOUN	_	Gender=Fem|Number=Plur	7	nmod	O	_
@@ -73281,7 +73281,7 @@
 29	onde	onde	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	30	obl	O	_
 30	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	acl:relcl	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
-32	captura	captura	NOUN	_	Gender=Fem|Number=Sing	30	obj	O	_
+32	captura	captura	NOUN	_	Gender=Fem|Number=Sing	30	nsubj	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 35	.	.	PUNCT	_	_	28	punct	O	_
@@ -74839,7 +74839,7 @@
 17	mas	mas	CCONJ	_	_	21	cc	O	_
 18	se	se	SCONJ	_	_	19	mark	O	_
 19	ocorrer	ocorrer	VERB	_	Number=Sing|Person=3|VerbForm=Inf	21	advcl	O	_
-20	vazamentos	vazamento	NOUN	_	Gender=Masc|Number=Plur	19	obj	O	_
+20	vazamentos	vazamento	NOUN	_	Gender=Masc|Number=Plur	19	nsubj	O	_
 21	haverá	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	10	conj	O	_
 22	riscos	risco	NOUN	_	Gender=Masc|Number=Plur	21	obj	O	_
 23	potenciais	potencial	ADJ	_	Gender=Masc|Number=Plur	22	amod	O	_
@@ -75588,7 +75588,7 @@
 8	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	9	obl	O	_
 9	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	acl:relcl	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
-11	reação	reação	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
+11	reação	reação	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	15	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -80441,7 +80441,7 @@
 22	onde	onde	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	23	nsubj	O	_
 23	ocorrerá	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	21	acl:relcl	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	23	obj	O	_
+25	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	23	nsubj	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	captura	captura	NOUN	_	Gender=Fem|Number=Sing	25	nmod	O	_
 28-29	do	_	_	_	_	_	_	_	_
@@ -99224,7 +99224,7 @@
 21	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 22	ocorrer	ocorrer	VERB	_	VerbForm=Inf	21	xcomp	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	aprisionamento	aprisionamento	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
+24	aprisionamento	aprisionamento	NOUN	_	Gender=Masc|Number=Sing	22	nsubj	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	tubos	tubo	NOUN	_	Gender=Masc|Number=Plur	24	nmod	O	_
 27	durante	durante	ADP	_	_	29	case	O	_
@@ -108457,7 +108457,7 @@
 28	em	em	ADP	_	_	29	case	O	_
 29	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	30	obl	O	_
 30	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	25	acl:relcl	O	_
-31	aumento	aumento	NOUN	_	Gender=Masc|Number=Sing	30	obj	O	_
+31	aumento	aumento	NOUN	_	Gender=Masc|Number=Sing	30	nsubj	O	_
 32-33	da	_	_	_	_	_	_	_	_
 32	de	de	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
@@ -112220,7 +112220,7 @@
 9	e	e	CCONJ	_	_	10	cc	O	_
 10	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	conj	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	bloqueio	bloqueio	NOUN	_	Gender=Masc|Number=Sing	10	obj	O	_
+12	bloqueio	bloqueio	NOUN	_	Gender=Masc|Number=Sing	10	nsubj	O	_
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -112268,7 +112268,7 @@
 23	também	também	ADV	_	_	24	advmod	O	_
 24	ocorrer	ocorrer	VERB	_	VerbForm=Inf	22	xcomp	O	_
 25	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	26	det	O	_
-26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
+26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	24	nsubj	O	_
 27	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = Porém o uso de inibidores de hidratos é o método mais utilizado.
@@ -115078,7 +115078,7 @@
 60	topo	topo	NOUN	_	Gender=Masc|Number=Sing	56	conj	O	_
 61	sai	sair	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	44	conj	O	_
 62	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	63	det	O	_
-63	gás	gás	NOUN	_	Gender=Masc|Number=Sing	61	obj	O	_
+63	gás	gás	NOUN	_	Gender=Masc|Number=Sing	61	nsubj	O	_
 64	já	já	ADV	_	_	65	advmod	O	_
 65	tratado	tratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	63	acl	O	_
 66	.	.	PUNCT	_	_	4	punct	O	_
@@ -115137,7 +115137,7 @@
 23	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 24	sai	sair	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
+26	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	24	nsubj	O	_
 27	rica	rico	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 28	em	em	ADP	_	_	29	case	O	_
 29	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	27	obl	O	_
@@ -118987,7 +118987,7 @@
 20	que	que	SCONJ	_	_	21	mark	O	_
 21	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	advcl	O	_
 22	melhor	bom	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
-23	separação	separação	NOUN	_	Gender=Fem|Number=Sing	21	obj	O	_
+23	separação	separação	NOUN	_	Gender=Fem|Number=Sing	21	nsubj	O	_
 24	entre	entre	ADP	_	_	25	case	O	_
 25	água	água	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
@@ -126301,7 +126301,7 @@
 55	até	até	SCONJ	_	_	56	mark	O	_
 56	ocorrer	ocorrer	VERB	_	VerbForm=Inf	54	advcl	O	_
 57	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	58	det	O	_
-58	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	56	obj	O	_
+58	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	56	nsubj	O	_
 59	abrupta	abrupto	ADJ	_	Gender=Fem|Number=Sing	58	amod	O	_
 60-61	na	_	_	_	_	_	_	_	_
 60	em	em	ADP	_	_	62	case	O	_
@@ -127621,7 +127621,7 @@
 13	)	)	PUNCT	_	_	12	punct	O	_
 14	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 15	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	redução	redução	NOUN	_	Gender=Fem|Number=Sing	14	obj	O	_
+16	redução	redução	NOUN	_	Gender=Fem|Number=Sing	14	nsubj	O	_
 17-18	na	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -127977,7 +127977,7 @@
 7	que	que	SCONJ	_	_	8	mark	O	_
 8	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	ccomp	O	_
 9	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	intersecção	intersecção	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
+10	intersecção	intersecção	NOUN	_	Gender=Fem|Number=Sing	8	nsubj	O	_
 11	entre	entre	ADP	_	_	13	case	O	_
 12	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	13	det	O	_
 13	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	10	nmod	O	_
@@ -128429,7 +128429,7 @@
 12	coreflow	coreflow	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 13	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 14	ocorrer	ocorrer	VERB	_	VerbForm=Inf	13	xcomp	O	_
-15	imprevistos	imprevisto	NOUN	_	Gender=Masc|Number=Plur	14	obj	O	_
+15	imprevistos	imprevisto	NOUN	_	Gender=Masc|Number=Plur	14	nsubj	O	_
 16	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	17	nsubj	O	_
 17	ocasionem	ocasionar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	O	_
 18	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -130456,7 +130456,7 @@
 5	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	ocorrer	ocorrer	VERB	_	VerbForm=Inf	5	xcomp	O	_
 7	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	deficiência	deficiência	NOUN	_	Gender=Fem|Number=Sing	6	obj	O	_
+8	deficiência	deficiência	NOUN	_	Gender=Fem|Number=Sing	6	nsubj	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	resinas	resina	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
 11	e	e	CCONJ	_	_	12	cc	O	_
@@ -131423,7 +131423,7 @@
 11	flow	flow	X	_	_	9	nmod	O	_
 12	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 13	ocorrer	ocorrer	VERB	_	VerbForm=Inf	12	xcomp	O	_
-14	imprevistos	imprevisto	NOUN	_	Gender=Masc|Number=Plur	13	obj	O	_
+14	imprevistos	imprevisto	NOUN	_	Gender=Masc|Number=Plur	13	nsubj	O	_
 15	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	16	nsubj	O	_
 16	ocasionem	ocasionar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	acl:relcl	O	_
 17	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
@@ -140495,7 +140495,7 @@
 30	estabelece	estabelecer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	acl:relcl	O	_
 31	e	e	CCONJ	_	_	32	cc	O	_
 32	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	30	conj	O	_
-33	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	32	obj	O	_
+33	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	32	nsubj	O	_
 34-35	das	_	_	_	_	_	_	_	_
 34	de	de	ADP	_	_	36	case	O	_
 35	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	36	det	O	_
@@ -140960,7 +140960,7 @@
 27	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	19	acl:relcl	O	_
 28	certa	certo	DET	_	Gender=Fem|Number=Sing|PronType=Ind	30	det	O	_
 29	55	55	NUM	_	NumType=Card	30	nummod	O	_
-30	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	27	obj	O	_
+30	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	27	nsubj	O	_
 31	entre	entre	ADP	_	_	33	case	O	_
 32	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	33	det	O	_
 33	fases	fase	NOUN	_	Gender=Fem|Number=Plur	30	nmod	O	_
@@ -141014,7 +141014,7 @@
 12	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	8	advcl	O	_
 13	apenas	apenas	ADV	_	_	12	advmod	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	12	obj	O	_
+15	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	12	nsubj	O	_
 16-17	da	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
@@ -148142,7 +148142,7 @@
 # sent_id = 279-20140530-MONOGRAFIA_0-217
 1	Ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	também	também	ADV	_	_	1	advmod	O	_
-3	redução	redução	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
+3	redução	redução	NOUN	_	Gender=Fem|Number=Sing	1	nsubj	O	_
 4	para	para	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	3	nmod	O	_
@@ -157618,7 +157618,7 @@
 16	,	,	PUNCT	_	_	11	punct	O	_
 17	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
+19	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	17	nsubj	O	_
 20-21	dos	_	_	_	_	_	_	_	_
 20	de	de	ADP	_	_	22	case	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	22	det	O	_
@@ -157643,7 +157643,7 @@
 1	Ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	também	também	ADV	_	_	1	advmod	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
+4	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	1	nsubj	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	7	det	O	_
 7	silicatos	silicato	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
@@ -167025,7 +167025,7 @@
 4	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	5	obl	O	_
 5	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	acl:relcl	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	micelização	micelização	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+7	micelização	micelização	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
 8	,	,	PUNCT	_	_	12	punct	O	_
 9-10	numa	_	_	_	_	_	_	_	_
 9	em	em	ADP	_	_	12	case	O	_
@@ -169057,7 +169057,7 @@
 18	que	que	SCONJ	_	_	20	mark	O	_
 19	não	não	ADV	_	Polarity=Neg	20	advmod	O	_
 20	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	ccomp	O	_
-21	entupimento	entupimento	NOUN	_	Gender=Masc|Number=Sing	20	obj	O	_
+21	entupimento	entupimento	NOUN	_	Gender=Masc|Number=Sing	20	nsubj	O	_
 22-23	dos	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	24	det	O	_
@@ -172831,7 +172831,7 @@
 4	ocorrido	ocorrer	VERB	_	VerbForm=Part	2	xcomp	O	_
 5	devido	devido	ADV	_	_	4	advmod	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-7	clorofórmio	clorofórmio	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+7	clorofórmio	clorofórmio	NOUN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
 8	extrair	extrair	VERB	_	VerbForm=Inf	4	xcomp	O	_
 9	também	também	ADV	_	_	8	advmod	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
@@ -175444,7 +175444,7 @@
 11	que	que	SCONJ	_	_	12	mark	O	_
 12	ocorresse	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	3	advcl	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	difusão	difusão	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
+14	difusão	difusão	NOUN	_	Gender=Fem|Number=Sing	12	nsubj	O	_
 15-16	do	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
@@ -182469,9 +182469,9 @@
 13	OH	OH	PROPN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 14	por	por	SCONJ	_	_	15	mark	O	_
 15	ocorrer	ocorrer	VERB	_	VerbForm=Inf	8	advcl	O	_
-16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
+16	a	a	ADP	_	_	18	case	O	_
 17	baixa	baixo	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
-18	freqüência	freqüência	NOUN	_	Gender=Fem|Number=Sing	15	obj	O	_
+18	freqüência	freqüência	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 19	(	(	PUNCT	_	_	20	punct	O	_
 20	Low	Low	PROPN	_	Gender=Fem|Number=Sing	18	appos	O	_
 21	Frequency	Frequency	PROPN	_	Number=Sing	20	flat:name	O	_
@@ -184740,7 +184740,7 @@
 4	que	que	SCONJ	_	_	6	mark	O	_
 5	não	não	ADV	_	Polarity=Neg	6	advmod	O	_
 6	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	ccomp	O	_
-7	adsorção	adsorção	NOUN	_	Gender=Fem|Number=Sing	6	obj	O	_
+7	adsorção	adsorção	NOUN	_	Gender=Fem|Number=Sing	6	nsubj	O	_
 8	irreversível	irreversível	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	monóxido	monóxido	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
@@ -184775,7 +184775,7 @@
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
 4	catalisadores	catalisador	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
 5	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-6	quimissorção	quimissorção	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+6	quimissorção	quimissorção	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
 7	,	,	PUNCT	_	_	8	punct	O	_
 8	indicando	indicar	VERB	_	VerbForm=Ger	5	advcl	O	_
 9	que	que	SCONJ	_	_	13	mark	O	_
@@ -186456,7 +186456,7 @@
 27	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	28	obl	O	_
 28	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	acl:relcl	O	_
 29	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
-30	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	28	obj	O	_
+30	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	28	nsubj	O	_
 31-32	no	_	_	_	_	_	_	_	_
 31	em	em	ADP	_	_	33	case	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
@@ -189733,7 +189733,7 @@
 # sent_id = 20-20140904-TESEDSC_0-111
 1	Também	também	ADV	_	_	2	advmod	O	_
 2	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-3	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	2	obj	O	_
+3	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	2	nsubj	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
 5	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	3	conj	O	_
 6	ricos	rico	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
@@ -190880,7 +190880,7 @@
 4	unidades	unidade	NOUN	_	Gender=Fem|Number=Plur	1	obl:arg	O	_
 5	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
-7	manifestações	manifestação	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
+7	manifestações	manifestação	NOUN	_	Gender=Fem|Number=Plur	5	nsubj	O	_
 8	magmáticas	magmático	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 9	toleíticas	toleítico	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 10-11	do	_	_	_	_	_	_	_	_
@@ -193273,7 +193273,7 @@
 36	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	37	obl	O	_
 37	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	31	acl:relcl	O	_
 38	apenas	apenas	ADV	_	_	39	advmod	O	_
-39	blocos	bloco	NOUN	_	Gender=Masc|Number=Plur	37	obj	O	_
+39	blocos	bloco	NOUN	_	Gender=Masc|Number=Plur	37	nsubj	O	_
 40	de	de	ADP	_	_	41	case	O	_
 41	nefelina	nefelina	NOUN	_	Gender=Fem|Number=Sing	39	nmod	O	_
 42	sienito	sienito	NOUN	_	Gender=Masc|Number=Sing	41	compound	O	_
@@ -193438,7 +193438,7 @@
 5	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
 7	único	único	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
-8	dique	dique	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
+8	dique	dique	NOUN	_	Gender=Masc|Number=Sing	5	nsubj	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	brecha	brecha	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	com	com	ADP	_	_	16	case	O	_
@@ -193458,7 +193458,7 @@
 24	nordeste	nordeste	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 25	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	advcl	O	_
 26	dois	dois	NUM	_	NumType=Card	27	nummod	O	_
-27	diques	dique	NOUN	_	Gender=Masc|Number=Plur	25	obj	O	_
+27	diques	dique	NOUN	_	Gender=Masc|Number=Plur	25	nsubj	O	_
 28	,	,	PUNCT	_	_	29	punct	O	_
 29	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	27	appos	O	_
 30-31	deles	_	_	_	_	_	_	_	_
@@ -195067,7 +195067,7 @@
 # sent_id = 20-20140904-TESEDSC_0-241
 1	Ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	somente	somente	ADV	_	_	1	advmod	O	_
-3	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	1	obj	O	_
+3	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	1	nsubj	O	_
 4	interpretados	interpretar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	acl	O	_
 5	como	como	ADP	_	_	6	case	O	_
 6	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	4	xcomp	B=UNIDADE_LITOESTRATIGRÁFICA	_
@@ -196577,7 +196577,7 @@
 4	alinhamento	alinhamento	NOUN	_	Gender=Masc|Number=Sing	1	obl:arg	O	_
 5	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
-7	manifestações	manifestação	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
+7	manifestações	manifestação	NOUN	_	Gender=Fem|Number=Plur	5	nsubj	O	_
 8	vulcânicas	vulcânico	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 9	observadas	observar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	7	acl	O	_
 10-11	no	_	_	_	_	_	_	_	_
@@ -198045,7 +198045,7 @@
 17	,	,	PUNCT	_	_	5	punct	O	_
 18	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 19	diversos	diverso	DET	_	Gender=Masc|Number=Plur|PronType=Ind	20	det	O	_
-20	intervalos	intervalo	NOUN	_	Gender=Masc|Number=Plur	18	obj	O	_
+20	intervalos	intervalo	NOUN	_	Gender=Masc|Number=Plur	18	nsubj	O	_
 21	com	com	ADP	_	_	22	case	O	_
 22	falta	falta	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 23	de	de	ADP	_	_	24	case	O	_
@@ -200116,7 +200116,7 @@
 5	casos	caso	NOUN	_	Gender=Masc|Number=Plur	3	obj	O	_
 6	onde	onde	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	7	nsubj	O	_
 7	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	acl:relcl	O	_
-8	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	7	obj	O	_
+8	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	7	nsubj	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	remanescente	remanescente	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
@@ -200982,7 +200982,7 @@
 17-18	à	_	_	_	_	_	_	_	_
 17	a	a	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	charneira	charneira	NOUN	_	Gender=Fem|Number=Sing	15	obl:arg	O	_
+19	charneira	charneira	NOUN	_	Gender=Fem|Number=Sing	16	obl	O	_
 20	cretácea	cretáceo	ADJ	_	Gender=Fem|Number=Sing	19	amod	B=GEOCRONOLOGIA	_
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
@@ -209022,7 +209022,7 @@
 15	,	,	PUNCT	_	_	4	punct	O	_
 16	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 17	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
-18	zona	zona	NOUN	_	Gender=Fem|Number=Sing	16	obj	O	_
+18	zona	zona	NOUN	_	Gender=Fem|Number=Sing	16	nsubj	O	_
 19	de	de	ADP	_	_	21	case	O	_
 20	baixa	baixo	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
 21	susceptibilidade	susceptibilidade	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
@@ -212739,7 +212739,7 @@
 7	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 9	maior	grande	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
-10	espessamento	espessamento	NOUN	_	Gender=Masc|Number=Sing	7	obj	O	_
+10	espessamento	espessamento	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
 11	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	depocentro	depocentro	NOUN	_	Gender=Masc|Number=Sing	10	appos	O	_
@@ -215046,7 +215046,7 @@
 29	onde	onde	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	30	obl	O	_
 30	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	27	acl:relcl	O	_
 31	delgado	delgado	ADJ	_	Gender=Masc|Number=Sing	32	amod	O	_
-32	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	30	obj	O	_
+32	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	30	nsubj	O	_
 33	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	32	amod	O	_
 34	sobre	sobre	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
@@ -219671,7 +219671,7 @@
 8	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	advcl	O	_
 9	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 10	proeminente	proeminente	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
-11	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
+11	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	8	nsubj	O	_
 12	NE-SW	NE-SW	PROPN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	altos	alto	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
@@ -221073,7 +221073,7 @@
 10	,	,	PUNCT	_	_	4	punct	O	_
 11	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	conj	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	desaparecimento	desaparecimento	NOUN	_	Gender=Masc|Number=Sing	11	obj	O	_
+13	desaparecimento	desaparecimento	NOUN	_	Gender=Masc|Number=Sing	11	nsubj	O	_
 14-15	dos	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -222228,7 +222228,7 @@
 # sent_id = 20-20140904-TESEDSC_0-971
 1	Ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
-3	feixe	feixe	NOUN	_	Gender=Masc|Number=Sing	1	obj	O	_
+3	feixe	feixe	NOUN	_	Gender=Masc|Number=Sing	1	nsubj	O	_
 4	de	de	ADP	_	_	5	case	O	_
 5	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	3	nmod	O	_
 6	de	de	ADP	_	_	8	case	O	_
@@ -223581,7 +223581,7 @@
 4	porção	porção	NOUN	_	Gender=Fem|Number=Sing	1	obl	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 6	proeminente	proeminente	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
-7	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
+7	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	1	nsubj	O	_
 8	magmática	magmático	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	,	,	PUNCT	_	_	11	punct	O	_
 10	possivelmente	possivelmente	ADV	_	_	11	advmod	O	_
@@ -224770,7 +224770,7 @@
 23	também	também	ADV	_	_	22	advmod	O	_
 24	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
 25	sugestivo	sugestivo	ADJ	_	Gender=Masc|Number=Sing	26	amod	O	_
-26	deslocamento	deslocamento	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
+26	deslocamento	deslocamento	NOUN	_	Gender=Masc|Number=Sing	22	nsubj	O	_
 27	dextrógiro	dextrógiro	ADJ	_	Gender=Masc|Number=Sing	26	amod	O	_
 28-29	ao	_	_	_	_	_	_	_	_
 28	a	a	ADP	_	_	30	case	O	_
@@ -225496,7 +225496,7 @@
 10	Grande	Grande	PROPN	_	Number=Sing	9	flat:name	O	_
 11	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	proeminentes	proeminente	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
-13	alinhamentos	alinhamento	NOUN	_	Gender=Masc|Number=Plur	11	obj	O	_
+13	alinhamentos	alinhamento	NOUN	_	Gender=Masc|Number=Plur	11	nsubj	O	_
 14-15	nesta	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	17	case	O	_
 15	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	17	det	O	_
@@ -225556,7 +225556,7 @@
 43	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	conj	O	_
 44	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
 45	novo	novo	ADJ	_	Gender=Masc|Number=Sing	46	amod	O	_
-46	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	43	obj	O	_
+46	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	43	nsubj	O	_
 47-48	destes	_	_	_	_	_	_	_	_
 47	de	de	ADP	_	_	49	case	O	_
 48	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	49	det	O	_
@@ -227562,7 +227562,7 @@
 5	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 6	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 7	significativa	significativo	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
-8	concentração	concentração	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	concentração	concentração	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
 9-10	destes	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	11	det	O	_
@@ -227958,7 +227958,7 @@
 7	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	mais	mais	ADV	_	_	9	advmod	O	_
 9	duas	dois	NUM	_	NumType=Card	10	nummod	O	_
-10	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
+10	anomalias	anomalia	NOUN	_	Gender=Fem|Number=Plur	7	nsubj	O	_
 11	expressivas	expressivo	ADJ	_	Gender=Fem|Number=Plur	10	amod	O	_
 12	.	.	PUNCT	_	_	7	punct	O	_
 
@@ -241413,7 +241413,7 @@
 # sent_id = 119-20141209-TESEMSC_0-135
 1	Embora	embora	SCONJ	_	_	2	mark	O	_
 2	ocorram	ocorrer	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	advcl	O	_
-3	divergências	divergência	NOUN	_	Gender=Fem|Number=Plur	2	obj	O	_
+3	divergências	divergência	NOUN	_	Gender=Fem|Number=Plur	2	nsubj	O	_
 4	,	,	PUNCT	_	_	2	punct	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 6	principal	principal	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
@@ -243560,7 +243560,7 @@
 11	aumenta	aumentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	advcl	O	_
 12	,	,	PUNCT	_	_	11	punct	O	_
 13	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-14	redução	redução	NOUN	_	Gender=Fem|Number=Sing	13	obj	O	_
+14	redução	redução	NOUN	_	Gender=Fem|Number=Sing	13	nsubj	O	_
 15-16	na	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
@@ -254480,7 +254480,7 @@
 38	,	,	PUNCT	_	_	35	punct	O	_
 39	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	5	conj	O	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
-41	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	39	obj	O	_
+41	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	39	nsubj	O	_
 42-43	do	_	_	_	_	_	_	_	_
 42	de	de	ADP	_	_	44	case	O	_
 43	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	44	det	O	_
@@ -257392,7 +257392,7 @@
 3	que	que	SCONJ	_	_	4	mark	O	_
 4	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	ccomp	O	_
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	processo	processo	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+6	processo	processo	NOUN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	filtração	filtração	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 9-10	do	_	_	_	_	_	_	_	_
@@ -258875,7 +258875,7 @@
 22	(	(	PUNCT	_	_	24	punct	O	_
 23	não	não	ADV	_	Polarity=Neg	24	advmod	O	_
 24	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	18	parataxis	O	_
-25	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
+25	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	24	nsubj	O	_
 26-27	do	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
@@ -261058,7 +261058,7 @@
 25	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	26	obl	O	_
 26	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	23	acl:relcl	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
-28	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
+28	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	26	nsubj	O	_
 29	(	(	PUNCT	_	_	31	punct	O	_
 30	1,5	1,5	NUM	_	NumType=Card	31	nummod	O	_
 31	%	%	SYM	_	_	28	appos	O	_
@@ -261716,7 +261716,7 @@
 15	não	não	ADV	_	Polarity=Neg	16	advmod	O	_
 16	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	12	acl:relcl	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	16	obj	O	_
+18	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	16	nsubj	O	_
 19	(	(	PUNCT	_	_	20	punct	O	_
 20	D1	D1	PROPN	_	Gender=Masc|Number=Sing	18	appos	O	_
 21	e	e	CCONJ	_	_	22	cc	O	_
@@ -261758,7 +261758,7 @@
 11	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	12	obl	O	_
 12	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	9	acl:relcl	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	12	obj	O	_
+14	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	12	nsubj	O	_
 15	(	(	PUNCT	_	_	16	punct	O	_
 16	D3	D3	PROPN	_	Gender=Masc|Number=Sing	14	appos	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
@@ -262032,7 +262032,7 @@
 18	não	não	ADV	_	Polarity=Neg	19	advmod	O	_
 19	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	15	acl:relcl	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	19	obj	O	_
+21	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	19	nsubj	O	_
 22	,	,	PUNCT	_	_	19	punct	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
 24	maior	grande	ADJ	_	Gender=Masc|Number=Sing	25	amod	O	_
@@ -262552,7 +262552,7 @@
 24	continha	conter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	19	acl:relcl	O	_
 25	GX	GX	PROPN	_	Gender=Masc|Number=Sing	24	obj	O	_
 26	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	advcl	O	_
-27	produção	produção	NOUN	_	Gender=Fem|Number=Sing	26	obj	O	_
+27	produção	produção	NOUN	_	Gender=Fem|Number=Sing	26	nsubj	O	_
 28	de	de	ADP	_	_	29	case	O	_
 29	filtrado	filtrado	NOUN	_	Gender=Masc|Number=Sing	27	nmod	O	_
 30	em	em	ADP	_	_	32	case	O	_
@@ -262575,7 +262575,7 @@
 6	onde	onde	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	7	obl	O	_
 7	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	4	acl:relcl	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-9	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	7	obj	O	_
+9	irrupção	irrupção	NOUN	_	Gender=Fem|Number=Sing	7	nsubj	O	_
 10	(	(	PUNCT	_	_	11	punct	O	_
 11	breakthrough	breakthrough	NOUN	_	Gender=Masc|Number=Sing	9	appos	O	_
 12	)	)	PUNCT	_	_	11	punct	O	_
@@ -263484,7 +263484,7 @@
 1	Enquanto	enquanto	SCONJ	_	_	2	mark	O	_
 2	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	advcl	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	2	obj	O	_
+4	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	2	nsubj	O	_
 5	,	,	PUNCT	_	_	2	punct	O	_
 6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
 7	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
@@ -271034,7 +271034,7 @@
 20	,	,	PUNCT	_	_	12	punct	O	_
 21	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 22	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	21	obj	O	_
+23	mudança	mudança	NOUN	_	Gender=Fem|Number=Sing	21	nsubj	O	_
 24	brusca	brusco	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25-26	nestas	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
@@ -278498,7 +278498,7 @@
 93	caso	caso	SCONJ	_	_	94	mark	O	_
 94	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	83	advcl	O	_
 95	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	96	det	O	_
-96	vazamento	vazamento	NOUN	_	Gender=Masc|Number=Sing	94	obj	O	_
+96	vazamento	vazamento	NOUN	_	Gender=Masc|Number=Sing	94	nsubj	O	_
 97	de	de	ADP	_	_	98	case	O	_
 98	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	96	nmod	O	_
 99-100	na	_	_	_	_	_	_	_	_
@@ -282616,7 +282616,7 @@
 36	,	,	PUNCT	_	_	38	punct	O	_
 37	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	38	nsubj	O	_
 38	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	33	acl:relcl	O	_
-39	acúmulo	acúmulo	NOUN	_	Gender=Masc|Number=Sing	38	obj	O	_
+39	acúmulo	acúmulo	NOUN	_	Gender=Masc|Number=Sing	38	nsubj	O	_
 40	diferenciado	diferenciar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	39	acl	O	_
 41	de	de	ADP	_	_	42	case	O	_
 42	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	39	nmod	O	_
@@ -283489,7 +283489,7 @@
 13	quando	quando	SCONJ	_	_	14	mark	O	_
 14	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	advcl	O	_
 15	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	emergência	emergência	NOUN	_	Gender=Fem|Number=Sing	14	obj	O	_
+16	emergência	emergência	NOUN	_	Gender=Fem|Number=Sing	14	nsubj	O	_
 17-18	nesses	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	esses	esse	DET	_	Gender=Masc|Number=Plur|PronType=Dem	19	det	O	_
@@ -283673,7 +283673,7 @@
 44	não	não	ADV	_	Polarity=Neg	45	advmod	O	_
 45	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	41	advcl	O	_
 46	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	47	det	O	_
-47	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	45	obj	O	_
+47	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	45	nsubj	O	_
 48	imediata	imediato	ADJ	_	Gender=Fem|Number=Sing	47	amod	O	_
 49	,	,	PUNCT	_	_	51	punct	O	_
 50	logo	logo	ADV	_	_	51	advmod	O	_
@@ -284800,7 +284800,7 @@
 21	caso	caso	SCONJ	_	_	22	mark	O	_
 22	ocorra	ocorrer	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	20	advcl	O	_
 23	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	incidente	incidente	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
+24	incidente	incidente	NOUN	_	Gender=Masc|Number=Sing	22	nsubj	O	_
 25	com	com	ADP	_	_	27	case	O	_
 26	algum	algum	DET	_	Gender=Masc|Number=Sing|PronType=Ind	27	det	O	_
 27	membro	membro	NOUN	_	Gender=Masc|Number=Sing	22	obl	O	_
@@ -285818,7 +285818,7 @@
 27	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	advcl	O	_
 28	ocorrer	ocorrer	VERB	_	VerbForm=Inf	27	xcomp	O	_
 29	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
-30	aumento	aumento	NOUN	_	Gender=Masc|Number=Sing	28	obj	O	_
+30	aumento	aumento	NOUN	_	Gender=Masc|Number=Sing	28	nsubj	O	_
 31	de	de	ADP	_	_	34	case	O	_
 32	até	até	ADP	_	_	34	case	O	_
 33	cinco	cinco	NUM	_	NumType=Card	34	nummod	O	_
@@ -287660,7 +287660,7 @@
 13	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 14	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 15	maior	grande	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
-16	impacto	impacto	NOUN	_	Gender=Masc|Number=Sing	13	obj	O	_
+16	impacto	impacto	NOUN	_	Gender=Masc|Number=Sing	13	nsubj	O	_
 17	ambiental	ambiental	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
 19	econômico	econômico	ADJ	_	Gender=Masc|Number=Sing	17	conj	O	_
@@ -287703,7 +287703,7 @@
 24	,	,	PUNCT	_	_	25	punct	O	_
 25	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	33	advcl	O	_
 26	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	27	det	O	_
-27	processo	processo	NOUN	_	Gender=Masc|Number=Sing	25	obj	O	_
+27	processo	processo	NOUN	_	Gender=Masc|Number=Sing	25	nsubj	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
 29	polotas	polota	NOUN	_	Gender=Fem|Number=Plur	33	nsubj	O	_
 30-31	do	_	_	_	_	_	_	_	_
@@ -292315,7 +292315,7 @@
 8	)	)	PUNCT	_	_	6	punct	O	_
 9	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
-11	separação	separação	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
+11	separação	separação	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
 12	gás/líquido	gás/líquido	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 13	,	,	PUNCT	_	_	9	punct	O	_
 14	enquanto	enquanto	SCONJ	_	_	24	mark	O	MWEPOS=SCONJ
@@ -292334,7 +292334,7 @@
 26	também	também	ADV	_	_	24	advmod	O	_
 27	,	,	PUNCT	_	_	24	punct	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
-29	separação	separação	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
+29	separação	separação	NOUN	_	Gender=Fem|Number=Sing	24	nsubj	O	_
 30	óleo/água	óleo/águo	ADJ	_	Gender=Fem|Number=Sing	29	amod	O	_
 31	.	.	PUNCT	_	_	9	punct	O	_
 
