commit 7ff2ee15b2a0ed18c7cce19f026fb5099b75cc90
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Aug 5 13:05:26 2021 -0300

    Correções de 'tem' como AUX

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 2999093..a113325 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -18584,14 +18584,14 @@
 8	não	não	ADV	_	Polarity=Neg	11	advmod	O	_
 9	tenha	ter	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	aux	O	_
 10	sido	ser	AUX	_	VerbForm=Part	11	aux	O	_
-11	o	o	PRON	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Dem	1	ccomp	O	_
-12	bastante	bastante	ADJ	_	_	11	amod	O	_
+11	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	ccomp	O	_
+12	bastante	bastante	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 13	para	para	SCONJ	_	_	14	mark	O	_
 14	homogeneizar	homogeneizar	VERB	_	VerbForm=Inf	12	advcl	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
 16	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	14	obj	O	_
 17	100	100	NUM	_	NumType=Card	18	nummod	O	_
-18	%	%	SYM	_	_	16	nmod	O	_
+18	%	%	SYM	_	_	14	obl	O	_
 19	,	,	PUNCT	_	_	20	punct	O	_
 20	fazendo	fazer	VERB	_	VerbForm=Ger	11	advcl	O	_
 21	com	com	SCONJ	_	_	27	mark	O	_
@@ -48305,10 +48305,10 @@
 13	de	de	ADP	_	_	14	case	O	_
 14	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=FLUIDO	_
 15	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	aux	O	_
-16	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+16	resultado	resultar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	0	root	O	_
 17	em	em	ADP	_	_	19	case	O	_
 18	severas	severo	ADJ	_	Gender=Fem|Number=Plur	19	amod	O	_
-19	restrições	restrição	NOUN	_	Gender=Fem|Number=Plur	15	obl	O	_
+19	restrições	restrição	NOUN	_	Gender=Fem|Number=Plur	16	obl:arg	O	_
 20-21	à	_	_	_	_	_	_	_	_
 20	a	a	ADP	_	_	23	case	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
@@ -48316,7 +48316,7 @@
 23	utilização	utilização	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
 24	em	em	ADP	_	_	26	case	O	_
 25	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	26	det	O	_
-26	partes	parte	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
+26	partes	parte	NOUN	_	Gender=Fem|Number=Plur	19	nmod	O	_
 27-28	do	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -48342,7 +48342,7 @@
 47	meio	meio	NOUN	_	Gender=Masc|Number=Sing	44	obl	O	_
 48	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	47	compound	O	_
 49	,	,	PUNCT	_	_	44	punct	O	_
-50	os	o	DET	_	Gender=Masc|Number=Plur|PronType=Art	51	det	O	_
+50	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	51	det	O	_
 51	quais	qual	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	54	nsubj	O	_
 52	não	não	ADV	_	Polarity=Neg	54	advmod	O	_
 53	somente	somente	ADV	_	_	54	advmod	O	_
@@ -51522,9 +51522,9 @@
 23	por	por	ADP	_	_	24	case	O	_
 24	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	27	obl	O	_
 25	não	não	ADV	_	Polarity=Neg	27	advmod	O	_
-26	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	27	aux	O	_
-27	sentido	sentido	NOUN	_	Gender=Masc|Number=Sing	18	conj	O	_
-28	medir	medir	VERB	_	VerbForm=Inf	27	csubj	O	_
+26	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	conj	O	_
+27	sentido	sentido	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
+28	medir	medir	VERB	_	VerbForm=Inf	26	csubj	O	_
 29	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
 30	distribuição	distribuição	NOUN	_	Gender=Fem|Number=Sing	28	obj	O	_
 31	granulométrica	granulométrico	ADJ	_	Gender=Fem|Number=Sing	30	amod	O	_
@@ -51550,7 +51550,7 @@
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
 50	eletrólito	eletrólito	NOUN	_	Gender=Masc|Number=Sing	45	conj	B=QUÍMICA	_
 51	dispersante	dispersante	ADJ	_	Gender=Masc|Number=Sing	50	amod	O	_
-52	provocam	provocar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	28	advcl	O	_
+52	provocam	provocar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	26	advcl	O	_
 53	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	54	det	O	_
 54	separação	separação	NOUN	_	Gender=Fem|Number=Sing	52	obj	O	_
 55	ou	ou	CCONJ	_	_	56	cc	O	_
@@ -64014,15 +64014,15 @@
 5	fato	fato	NOUN	_	Gender=Masc|Number=Sing	10	nsubj	O	_
 6	de	de	SCONJ	_	_	8	mark	O	_
 7	serem	ser	AUX	_	Number=Plur|Person=3|VerbForm=Inf	8	cop	O	_
-8	reativas	reativo	ADJ	_	Gender=Fem|Number=Plur	5	amod	O	_
+8	reativas	reativo	ADJ	_	Gender=Fem|Number=Plur	5	acl	O	_
 9	não	não	ADV	_	Polarity=Neg	10	advmod	O	_
 10	significa	significar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-11	que	que	SCONJ	_	_	16	mark	O	_
-12	terão	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	16	aux	O	_
+11	que	que	SCONJ	_	_	12	mark	O	_
+12	terão	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	10	ccomp	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
-14	necessariamente	necessariamente	ADV	_	_	16	advmod	O	_
+14	necessariamente	necessariamente	ADV	_	_	12	advmod	O	_
 15	,	,	PUNCT	_	_	14	punct	O	_
-16	afinidade	afinidade	NOUN	_	Gender=Fem|Number=Sing	10	ccomp	O	_
+16	afinidade	afinidade	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
 17-18	pelo	_	_	_	_	_	_	_	_
 17	por	por	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
@@ -117172,11 +117172,11 @@
 15	de	de	ADP	_	_	16	case	O	_
 16	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	11	nmod	B=FLUIDO	_
 17	,	,	PUNCT	_	_	20	punct	O	_
-18	tendo	ter	AUX	_	VerbForm=Ger	20	cop	O	_
+18	tendo	ter	VERB	_	VerbForm=Ger	5	advcl	O	_
 19	como	como	ADP	_	_	20	case	O	_
-20	destino	destino	NOUN	_	Gender=Masc|Number=Sing	7	xcomp	O	_
+20	destino	destino	NOUN	_	Gender=Masc|Number=Sing	18	xcomp	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
-22	reinjeção	reinjeção	NOUN	_	Gender=Fem|Number=Sing	20	nsubj	O	_
+22	reinjeção	reinjeção	NOUN	_	Gender=Fem|Number=Sing	18	obj	O	_
 23-24	no	_	_	_	_	_	_	_	_
 23	em	em	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
@@ -124663,11 +124663,11 @@
 15	la	ela	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	14	obj	O	_
 16	,	,	PUNCT	_	_	14	punct	O	_
 17-18	tem-se	_	_	_	_	_	_	_	_
-17	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	aux	O	_
+17	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	parataxis	O	_
 18	se	se	PRON	_	_	17	expl	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-20	então	então	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
-21	tampão	tampão	NOUN	_	Gender=Masc|Number=Sing	12	obj	O	_
+20	então	então	ADV	_	_	21	advmod	O	_
+21	tampão	tampão	NOUN	_	Gender=Masc|Number=Sing	17	obj	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	hidrato	hidrato	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 24	(	(	PUNCT	_	_	25	punct	O	_
@@ -196842,7 +196842,7 @@
 # text = A Ressonância Magnética Nuclear de Hidrogênio tem sido muito usada para quantificar o teor de biodiesel, ésteres alquílicos, nas reações de transesterificação..
 # sent_id = 271-20150810-MONOGRAFIA_0-221
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
-2	Ressonância	Ressonância	PROPN	_	Gender=Fem|Number=Sing	10	nsubj	O	_
+2	Ressonância	Ressonância	PROPN	_	Gender=Fem|Number=Sing	10	nsubj:pass	O	_
 3	Magnética	Magnética	PROPN	_	Number=Sing	2	flat:name	O	_
 4	Nuclear	Nuclear	PROPN	_	Number=Sing	2	flat:name	O	_
 5	de	de	ADP	_	_	2	flat:name	O	_
@@ -196850,7 +196850,7 @@
 7	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	aux	O	_
 8	sido	ser	AUX	_	VerbForm=Part	10	cop	O	_
 9	muito	muito	ADV	_	_	10	advmod	O	_
-10	usada	usar	ADJ	_	Gender=Fem|Number=Sing	0	root	O	_
+10	usada	usar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 11	para	para	SCONJ	_	_	12	mark	O	_
 12	quantificar	quantificar	VERB	_	VerbForm=Inf	10	advcl	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
@@ -277008,10 +277008,10 @@
 8	Cabo	Cabo	PROPN	_	Number=Sing	6	flat:name	O	_
 9	Frio	Frio	PROPN	_	Number=Sing	6	flat:name	O	_
 10	teriam	ter	AUX	_	Mood=Cnd|Number=Plur|Person=3|VerbForm=Fin	11	aux	O	_
-11	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+11	resultado	resultar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	0	root	O	_
 12	de	de	ADP	_	_	14	case	O	_
 13	diferentes	diferente	DET	_	Gender=Masc|Number=Plur|PronType=Ind	14	det	O	_
-14	pulsos	pulso	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
+14	pulsos	pulso	NOUN	_	Gender=Masc|Number=Plur	11	obl	O	_
 15	magmáticos	magmático	ADJ	_	Gender=Masc|Number=Plur	14	amod	O	_
 16	como	como	SCONJ	_	_	17	mark	O	_
 17	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	advcl	O	_
@@ -297715,11 +297715,11 @@
 
 # text = Pela Figura 2.10 tem-se que, com o aumento da temperatura, a viscosidade do fluido tende a diminuir..
 # sent_id = 119-20141209-TESEMSC_0-357
-1	Pela	pela	PROPN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
-2	Figura	Figura	PROPN	_	Number=Sing	1	flat:name	O	_
-3	2.10	2.10	NUM	_	NumType=Card	1	nummod	O	_
+1	Pela	pela	ADP	_	_	2	case	O	tokenização
+2	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	4	obl	O	_
+3	2.10	2.10	NUM	_	NumType=Card	2	nummod	O	_
 4-5	tem-se	_	_	_	_	_	_	_	_
-4	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+4	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 5	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	4	expl	O	_
 6	que	que	SCONJ	_	_	20	mark	O	_
 7	,	,	PUNCT	_	_	10	punct	O	_
@@ -297737,10 +297737,10 @@
 17	de	de	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
 19	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
-20	tende	tender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	ccomp	O	_
+20	tende	tender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	ccomp	O	_
 21	a	a	SCONJ	_	_	22	mark	O	_
 22	diminuir	diminuir	VERB	_	VerbForm=Inf	20	xcomp	O	_
-23	..	..	PUNCT	_	_	1	punct	O	_
+23	..	..	PUNCT	_	_	4	punct	O	_
 
 # text = Observa-se também que as curvas de fluxo se convergem sendo a distância entre as curvas maior em A do que em B..
 # sent_id = 119-20141209-TESEMSC_0-358
@@ -297843,10 +297843,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	2.12	2.12	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	_	5	expl	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	gráfico	gráfico	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+8	gráfico	gráfico	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -297857,7 +297857,7 @@
 15	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 16	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	17	obl	O	_
 17-18	observa-se	_	_	_	_	_	_	_	_
-17	observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	acl:relcl	O	_
+17	observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	acl:relcl	O	_
 18	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	17	expl	O	_
 19	,	,	PUNCT	_	_	17	punct	O	_
 20	também	também	ADV	_	_	17	advmod	O	_
@@ -297879,13 +297879,13 @@
 34-35	na	_	_	_	_	_	_	_	_
 34	em	em	ADP	_	_	36	case	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
-36	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
+36	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 37-38	do	_	_	_	_	_	_	_	_
 37	de	de	ADP	_	_	39	case	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	det	O	_
 39	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	36	nmod	O	_
-40	.	.	PUNCT	_	_	8	punct	O	_
-41	.	.	PUNCT	_	_	8	punct	O	_
+40	.	.	PUNCT	_	_	5	punct	O	_
+41	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Nesta figura é possível identificar o comportamento típico da curva de fluxo de um fluido polimérico, como apresentado na Figura 2.6..
 # sent_id = 119-20141209-TESEMSC_0-363
@@ -297973,10 +297973,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	2.13	2.13	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	5	expl	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+8	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
 9	reológico	reológico	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 10-11	da	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
@@ -297984,7 +297984,7 @@
 12	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 13	versus	versus	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
-15	taxa	taxa	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
+15	taxa	taxa	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
 18	para	para	ADP	_	_	20	case	O	_
@@ -297994,8 +297994,8 @@
 22	(	(	PUNCT	_	_	23	punct	O	_
 23	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	20	nmod:appos	B=QUÍMICA	_
 24	)	)	PUNCT	_	_	23	punct	O	_
-25	.	.	PUNCT	_	_	8	punct	O	_
-26	.	.	PUNCT	_	_	8	punct	O	_
+25	.	.	PUNCT	_	_	5	punct	O	_
+26	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Pelo gráfico é possível observar, os efeitos da salinidade e do peso molecular à temperatura ambiente no comportamento da solução de PHPA..
 # sent_id = 119-20141209-TESEMSC_0-367
@@ -298902,12 +298902,12 @@
 40	e	e	CCONJ	_	_	43	cc	O	_
 41	com	com	ADP	_	_	43	case	O	_
 42	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	43	det	O	_
-43	reinício	reinício	NOUN	_	Gender=Masc|Number=Sing	39	conj	O	_
+43	reinício	reinício	NOUN	_	Gender=Masc|Number=Sing	44	obl	O	_
 44-45	tem-se	_	_	_	_	_	_	_	_
-44	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	47	aux	O	_
+44	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	32	conj	O	_
 45	se	se	PRON	_	_	44	expl	O	_
 46	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	47	det	O	_
-47	pico	pico	NOUN	_	Gender=Masc|Number=Sing	32	conj	O	_
+47	pico	pico	NOUN	_	Gender=Masc|Number=Sing	44	obj	O	_
 48	(	(	PUNCT	_	_	49	punct	O	_
 49	Sorbie	Sorbie	PROPN	_	Gender=Masc|Number=Sing	11	parataxis	O	_
 50	,	,	PUNCT	_	_	49	flat:name	O	_
@@ -299566,10 +299566,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	2.16	2.16	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	_	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	mecanismos	mecanismo	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
+8	mecanismos	mecanismo	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	bloqueio	bloqueio	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 11-12	da	_	_	_	_	_	_	_	_
@@ -299590,8 +299590,8 @@
 24	migração	migração	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	finos	fino	NOUN	_	Gender=Masc|Number=Plur	24	nmod	O	_
-27	.	.	PUNCT	_	_	8	punct	O	_
-28	.	.	PUNCT	_	_	8	punct	O	_
+27	.	.	PUNCT	_	_	5	punct	O	_
+28	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Figura 2.16–
 # sent_id = 119-20141209-TESEMSC_0-411
@@ -302123,7 +302123,7 @@
 # text = De acordo com Porter, 1989, e Mungan, 1989, apud Civan, 2007, para uma boa caracterização do dano de formação, testes laboratoriais de fluxo em testemunho devem ser conduzidos seguindo algumas condições:
 # sent_id = 119-20141209-TESEMSC_0-476
 1	De	de	ADP	_	_	4	case	O	MWEPOS=ADP
-2	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	_
+2	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	sentenciação
 3	com	com	ADP	_	_	1	fixed	O	_
 4	Porter	Porter	PROPN	_	Gender=Masc|Number=Sing	34	obl	O	_
 5	,	,	PUNCT	_	_	4	flat:name	O	_
@@ -302142,7 +302142,7 @@
 18	para	para	ADP	_	_	21	case	O	_
 19	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 20	boa	bom	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
-21	caracterização	caracterização	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
+21	caracterização	caracterização	NOUN	_	Gender=Fem|Number=Sing	34	obl	O	_
 22-23	do	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
@@ -302150,7 +302150,7 @@
 25	de	de	ADP	_	_	26	case	O	_
 26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 27	,	,	PUNCT	_	_	28	punct	O	_
-28	testes	teste	NOUN	_	Gender=Masc|Number=Plur	34	nsubj	B=MÉTODO	_
+28	testes	teste	NOUN	_	Gender=Masc|Number=Plur	34	nsubj:pass	B=MÉTODO	_
 29	laboratoriais	laboratorial	ADJ	_	Gender=Masc|Number=Plur	28	amod	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	28	nmod	O	_
@@ -302195,11 +302195,11 @@
 68	devem	dever	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	63	conj	O	_
 69	ser	ser	AUX	_	VerbForm=Inf	70	aux:pass	O	_
 70	projetados	projetar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	68	xcomp	O	_
-71	tendo	ter	AUX	_	VerbForm=Ger	73	cop	O	_
+71	tendo	ter	VERB	_	VerbForm=Ger	70	advcl	O	_
 72	em	em	ADP	_	_	73	case	O	_
-73	vista	vista	NOUN	_	Gender=Fem|Number=Sing	70	obl	O	_
+73	vista	vista	NOUN	_	Gender=Fem|Number=Sing	71	obl	O	_
 74	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	75	det	O	_
-75	condições	condição	NOUN	_	Gender=Fem|Number=Plur	70	nsubj:pass	O	_
+75	condições	condição	NOUN	_	Gender=Fem|Number=Plur	71	obj	O	_
 76	de	de	ADP	_	_	79	case	O	_
 77	todas	todo	DET	_	Gender=Fem|Number=Plur|PronType=Tot	79	det	O	_
 78	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	79	det	O	_
@@ -302207,7 +302207,7 @@
 80	de	de	ADP	_	_	81	case	O	_
 81	campo	campo	NOUN	_	Gender=Masc|Number=Sing	79	nmod	B=CAMPO:TIPO	_
 82	,	,	PUNCT	_	_	83	punct	O	_
-83	incluindo	incluir	VERB	_	VerbForm=Ger	70	advcl	O	_
+83	incluindo	incluir	VERB	_	VerbForm=Ger	71	advcl	O	_
 84	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	83	obj	B=LITOLOGIA	_
 85	,	,	PUNCT	_	_	86	punct	O	_
 86	completação	completação	NOUN	_	Gender=Fem|Number=Sing	84	conj	O	_
@@ -303456,10 +303456,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	3.1	3.1	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	amostras	amostra	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
+8	amostras	amostra	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	arenito	arenito	NOUN	_	Gender=Masc|Number=Sing	8	nmod	B=LITOLOGIA	_
 11	utilizadas	utilizar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	8	acl	O	_
@@ -303471,8 +303471,8 @@
 16	com	com	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	15	obl	B=QUÍMICA	_
-19	.	.	PUNCT	_	_	8	punct	O	_
-20	.	.	PUNCT	_	_	8	punct	O	_
+19	.	.	PUNCT	_	_	5	punct	O	_
+20	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Os valores das propriedades geométricas e permo-porosas obtidos durante a caracterização das amostras estão detalhados na Tabela 3.3..
 # sent_id = 119-20141209-TESEMSC_0-511
@@ -303903,16 +303903,16 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	3.3	3.3	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
+8	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
 9	reológicas	reológico	ADJ	_	Gender=Fem|Number=Plur	8	amod	O	_
 10-11	dos	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
 12	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
-13	contendo	conter	VERB	_	VerbForm=Ger	8	acl	O	_
+13	contendo	conter	VERB	_	VerbForm=Ger	12	acl	O	_
 14	4,5	4,5	NUM	_	NumType=Card	15	nummod	O	_
 15	lb/bbl	lb/bbl	NOUN	_	Gender=Masc|Number=Plur	13	obj	O	_
 16	de	de	ADP	_	_	17	case	O	_
@@ -303924,7 +303924,7 @@
 22	massa	massa	NOUN	_	Gender=Fem|Number=Sing	13	obl	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	bentonita	bentonita	NOUN	_	Gender=Fem|Number=Sing	22	nmod	B=LITOLOGIA	_
-25	variando	variar	VERB	_	VerbForm=Ger	13	advcl	O	_
+25	variando	variar	VERB	_	VerbForm=Ger	5	advcl	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	ordem	ordem	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
 28	de	de	ADP	_	_	29	case	O	_
@@ -303933,7 +303933,7 @@
 30	de	de	ADP	_	_	32	case	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
 32	sal.	sal.	NOUN	_	Gender=Masc|Number=Sing	29	nmod	O	_
-33	.	.	PUNCT	_	_	8	punct	O	_
+33	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = A partir delas é possível comparar o efeito da adição do sal como elemento inicial ou final na solução resultante..
 # sent_id = 119-20141209-TESEMSC_0-528
@@ -306895,13 +306895,13 @@
 
 # text = À esquerda tem-se o resultado obtido para o fluido sem argila e à direita têm-se os resultados obtidos para os fluidos com argila..
 # sent_id = 119-20141209-TESEMSC_0-616
-1	À	à	NOUN	_	Gender=Masc|Number=Sing	6	nsubj	O	_
-2	esquerda	esquerdo	ADJ	_	Gender=Fem|Number=Sing	1	amod	O	_
+1	À	à	ADP	_	_	2	case	O	_
+2	esquerda	esquerda	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 3-4	tem-se	_	_	_	_	_	_	_	_
-3	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	aux	O	_
+3	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	se	se	PRON	_	_	3	expl	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+6	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
 7	obtido	obter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	acl	O	_
 8	para	para	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
@@ -306912,20 +306912,20 @@
 14-15	à	_	_	_	_	_	_	_	_
 14	a	a	ADP	_	_	16	case	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	direita	direita	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
+16	direita	direita	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
 17-18	têm-se	_	_	_	_	_	_	_	_
-17	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	20	aux	O	_
+17	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	3	conj	O	_
 18	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	17	expl	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
-20	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	6	conj	O	_
+20	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	17	obj	O	_
 21	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	20	acl	O	_
 22	para	para	ADP	_	_	24	case	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	24	det	O	_
 24	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	21	obl	O	_
 25	com	com	ADP	_	_	26	case	O	_
 26	argila	argila	NOUN	_	Gender=Fem|Number=Sing	24	nmod	B=LITOLOGIA	_
-27	.	.	PUNCT	_	_	6	punct	O	_
-28	.	.	PUNCT	_	_	6	punct	O	_
+27	.	.	PUNCT	_	_	3	punct	O	_
+28	.	.	PUNCT	_	_	3	punct	O	_
 
 # text = Pelas figuras percebe-se que, para os fluidos que continham argila, esta tem papel fundamental na redução da invasão de fluido, pois as curvas para estes fluidos, decorrido certo tempo, mostraram uma queda na inclinação tendendo a se horizontalizar..
 # sent_id = 119-20141209-TESEMSC_0-617
@@ -307529,10 +307529,10 @@
 6	b	b	PROPN	_	Gender=Masc|Number=Sing	3	appos	O	_
 7	)	)	PUNCT	_	_	6	punct	O	_
 8-9	têm-se	_	_	_	_	_	_	_	_
-8	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	aux	O	_
+8	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 9	se	se	PRON	_	_	8	expl	O	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	detalhamento	detalhamento	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+11	detalhamento	detalhamento	NOUN	_	Gender=Masc|Number=Sing	8	obj	O	_
 12-13	destas	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	14	case	O	_
 13	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	14	det	O	_
@@ -307540,7 +307540,7 @@
 15	para	para	SCONJ	_	_	18	mark	O	_
 16	que	que	SCONJ	_	_	18	mark	O	_
 17	se	se	PRON	_	_	18	expl	O	_
-18	possa	poder	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	advcl	O	_
+18	possa	poder	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	advcl	O	_
 19	observar	observar	VERB	_	VerbForm=Inf	18	xcomp	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
 21	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	19	obj	O	_
@@ -307560,8 +307560,8 @@
 33	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 34	sem	sem	ADP	_	_	35	case	O	_
 35	argila	argila	NOUN	_	Gender=Fem|Number=Sing	33	nmod	B=LITOLOGIA	_
-36	.	.	PUNCT	_	_	11	punct	O	_
-37	.	.	PUNCT	_	_	11	punct	O	_
+36	.	.	PUNCT	_	_	8	punct	O	_
+37	.	.	PUNCT	_	_	8	punct	O	_
 
 # text = Por esta figura, observa-se que a vazão total para o fluido que não continha argila apresentou uma redução em relação à vazão inicial, porém, a curva se estabilizou, mostrando que esta vazão não chegou a ser interrompida, ou seja, o fluido que continha apenas PHPA não tendeu a bloquear as gargantas de poro da amostra por completo..
 # sent_id = 119-20141209-TESEMSC_0-630
@@ -308460,17 +308460,17 @@
 18	sem	sem	ADP	_	_	19	case	O	_
 19	argila	argila	NOUN	_	Gender=Fem|Number=Sing	17	nmod	B=LITOLOGIA	_
 20-21	têm-se	_	_	_	_	_	_	_	_
-20	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	25	aux	O	_
+20	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 21	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	20	expl	O	_
 22	,	,	PUNCT	_	_	25	punct	O	_
 23-24	na	_	_	_	_	_	_	_	_
 23	em	em	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	0	root	O	_
+25	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	20	obl	O	_
 26	4.9	4.9	NUM	_	NumType=Card	25	nummod	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
 28	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	29	det	O	_
-29	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	25	obl	O	_
+29	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	20	obj	O	_
 30	plotados	plotar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	29	acl	O	_
 31	em	em	ADP	_	_	32	case	O	_
 32	escala	escala	NOUN	_	Gender=Fem|Number=Sing	30	obl	O	_
@@ -309056,10 +309056,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.11	4.11	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
+8	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
 9	reológicas	reológico	ADJ	_	Gender=Fem|Number=Plur	8	amod	O	_
 10-11	dos	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
@@ -309080,8 +309080,8 @@
 24	preparados	preparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	23	acl	O	_
 25	com	com	ADP	_	_	26	case	O	_
 26	PHPA	PHPA	PROPN	_	Gender=Masc|Number=Sing	24	obl	B=QUÍMICA	_
-27	.	.	PUNCT	_	_	8	punct	O	_
-28	.	.	PUNCT	_	_	8	punct	O	_
+27	.	.	PUNCT	_	_	5	punct	O	_
+28	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Observa-se que os filtrados não apresentam as mesmas características dos fluidos injetados, principalmente para baixas taxas de cisalhamento, pois observa-se que a viscosidade, antes elevada, reduziu consideravelmente..
 # sent_id = 119-20141209-TESEMSC_0-665
@@ -309390,14 +309390,14 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.12	4.12	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
+8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
 9	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	O	_
 10	para	para	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	redução	redução	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
+12	redução	redução	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 13	e	e	CCONJ	_	_	16	cc	O	_
 14	para	para	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -309414,14 +309414,14 @@
 25-26	na	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	23	conj	O	_
+27	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	29	obl	O	_
 28	4.13	4.13	NUM	_	NumType=Card	27	nummod	O	_
 29-30	têm-se	_	_	_	_	_	_	_	_
-29	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	33	aux	O	_
+29	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 30	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	29	expl	O	_
 31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	33	det	O	_
 32	mesmos	mesmo	ADJ	_	Gender=Masc|Number=Plur	33	amod	O	_
-33	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
+33	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	29	obj	O	_
 34	,	,	PUNCT	_	_	36	punct	O	_
 35	porém	porém	CCONJ	_	_	36	cc	O	_
 36	plotados	plotar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	33	acl	O	_
@@ -309792,14 +309792,14 @@
 3	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.3	4.3	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	5	expl	O	_
 7	,	,	PUNCT	_	_	9	punct	O	_
 8	em	em	ADP	_	_	9	case	O	_
 9	números	número	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
 10	,	,	PUNCT	_	_	9	punct	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+12	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
 13-14	da	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
@@ -309816,7 +309816,7 @@
 24	preparados	preparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	23	acl	O	_
 25	com	com	ADP	_	_	26	case	O	_
 26	PHPA	PHPA	PROPN	_	Gender=Masc|Number=Sing	24	obl	B=QUÍMICA	_
-27	..	..	PUNCT	_	_	12	punct	O	_
+27	..	..	PUNCT	_	_	5	punct	O	_
 
 # text = Para podermos comparar os resultados obtidos, os valores foram interpolados para o máximo valor comum de VPI dentre os experimentos realizados com PHPA, no caso, 24,2 VPI..
 # sent_id = 119-20141209-TESEMSC_0-679
@@ -309978,15 +309978,15 @@
 2	se	se	PRON	_	_	1	expl	O	_
 3	que	que	SCONJ	_	_	23	mark	O	_
 4	,	,	PUNCT	_	_	8	punct	O	_
-5	apesar	apesar	ADV	_	_	8	case	O	_
+5	apesar	apesar	ADV	_	_	13	mark	O	_
 6	de	de	ADP	_	_	5	fixed	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	23	obl	O	_
+8	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	13	nsubj	O	_
 9	sem	sem	ADP	_	_	10	case	O	_
 10	argila	argila	NOUN	_	Gender=Fem|Number=Sing	8	nmod	B=LITOLOGIA	_
 11	ter	ter	AUX	_	VerbForm=Inf	13	aux	O	_
 12	sido	ser	AUX	_	VerbForm=Part	13	cop	O	_
-13	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	advcl	O	_
+13	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	23	advcl	O	_
 14	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	15	nsubj	O	_
 15	apresentou	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	acl:relcl	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
@@ -311441,10 +311441,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.21	4.21	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	_	5	expl	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+8	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -312304,10 +312304,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.25	4.25	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
+8	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
 9	com	com	ADP	_	_	11	case	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
 11	curvas	curva	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
@@ -312321,7 +312321,7 @@
 18-19	dos	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
-20	filtrados	filtrado	NOUN	_	Gender=Masc|Number=Plur	11	conj	O	_
+20	filtrados	filtrado	NOUN	_	Gender=Masc|Number=Plur	15	conj	O	_
 21	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	20	acl	O	_
 22-23	no	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
@@ -312588,14 +312588,14 @@
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.26	4.26	NUM	_	NumType=Card	3	nummod	O	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
+8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
 9	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	O	_
 10	para	para	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	razão	razão	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
+12	razão	razão	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	produtividade	produtividade	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
 15	versus	versus	ADP	_	_	16	case	O	_
@@ -312605,17 +312605,17 @@
 19-20	na	_	_	_	_	_	_	_	_
 19	em	em	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
-21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
+21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	23	obl	O	_
 22	4.27	4.27	NUM	_	NumType=Card	21	nummod	O	_
 23-24	têm-se	_	_	_	_	_	_	_	_
-23	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	27	aux	O	_
+23	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 24	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	23	expl	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	27	det	O	_
 26	mesmos	mesmo	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
-27	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
-28	,	,	PUNCT	_	_	29	punct	O	_
+27	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	23	obj	O	_
+28	,	,	PUNCT	_	_	31	punct	O	_
 29	porém	porém	CCONJ	_	_	31	cc	O	_
-30	,	,	PUNCT	_	_	29	punct	O	_
+30	,	,	PUNCT	_	_	31	punct	O	_
 31	plotados	plotar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	27	acl	O	_
 32	em	em	ADP	_	_	36	case	O	_
 33	relação	relação	NOUN	_	Gender=Fem|Number=Sing	32	fixed	O	_
@@ -312634,7 +312634,7 @@
 44	em	em	ADP	_	_	45	case	O	_
 45	escala	escala	NOUN	_	Gender=Fem|Number=Sing	42	conj	O	_
 46	log-log.	log-log.	NOUN	_	Gender=Fem|Number=Sing	45	nmod	O	_
-47	.	.	PUNCT	_	_	8	punct	O	_
+47	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Por estas figuras percebe-se que o retorno da razão de produtividade em amostras danificadas por fluidos preparados com GX é muito pequeno..
 # sent_id = 119-20141209-TESEMSC_0-748
@@ -312730,10 +312730,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.28	4.28	NUM	_	NumType=Card	3	nummod	O	_
 5-6	têm-se	_	_	_	_	_	_	_	_
-5	têm	ter	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	detalhes	detalhe	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
+8	detalhes	detalhe	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
 9-10	das	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
@@ -312743,7 +312743,7 @@
 14	b	b	PROPN	_	Gender=Masc|Number=Sing	11	appos	O	_
 15	)	)	PUNCT	_	_	14	punct	O	_
 16	e	e	CCONJ	_	_	17	cc	O	_
-17	4.27	4.27	NUM	_	NumType=Card	11	conj	O	_
+17	4.27	4.27	NUM	_	NumType=Card	12	conj	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
 19	b	b	NOUN	_	Gender=Masc|Number=Sing	17	appos	O	_
 20	)	)	PUNCT	_	_	19	punct	O	_
@@ -312784,7 +312784,7 @@
 52	a	a	ADP	_	_	50	fixed	O	_
 53	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	54	det	O	_
 54	VPI	VPI	PROPN	_	Gender=Fem|Number=Sing	51	nmod	O	_
-55	..	..	PUNCT	_	_	8	punct	O	_
+55	..	..	PUNCT	_	_	5	punct	O	_
 
 # text = Figura 4.28–
 # sent_id = 119-20141209-TESEMSC_0-751
@@ -312914,10 +312914,10 @@
 3	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.29	4.29	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	5	expl	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	comparação	comparação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
+8	comparação	comparação	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
 9-10	dos	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
@@ -312938,8 +312938,8 @@
 24	para	para	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	22	conj	B=QUÍMICA	_
-27	.	.	PUNCT	_	_	8	punct	O	_
-28	.	.	PUNCT	_	_	8	punct	O	_
+27	.	.	PUNCT	_	_	5	punct	O	_
+28	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Figura 4.29–
 # sent_id = 119-20141209-TESEMSC_0-756
@@ -313068,14 +313068,14 @@
 3	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	4.5	4.5	NUM	_	NumType=Card	3	nummod	O	_
 5-6	tem-se	_	_	_	_	_	_	_	_
-5	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	aux	O	_
+5	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	5	expl	O	_
 7	,	,	PUNCT	_	_	9	punct	O	_
 8	em	em	ADP	_	_	9	case	O	_
 9	números	número	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
 10	,	,	PUNCT	_	_	9	punct	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+12	retorno	retorno	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
 13-14	da	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
@@ -313167,10 +313167,10 @@
 4	4.30(a	4.30(a	NUM	_	NumType=Card	3	nummod	O	_
 5	)	)	PUNCT	_	_	3	punct	O	_
 6-7	tem-se	_	_	_	_	_	_	_	_
-6	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	aux	O	_
+6	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 7	se	se	PRON	_	_	6	expl	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+9	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	6	obj	O	_
 10	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
 11	para	para	SCONJ	_	_	12	mark	O	_
 12	atingir	atingir	VERB	_	VerbForm=Inf	10	advcl	O	_
@@ -313185,7 +313185,7 @@
 20	de	de	ADP	_	_	21	case	O	_
 21	produtividade	produtividade	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
 22	acima	acima	ADV	_	_	12	advmod	O	_
-23	..	..	PUNCT	_	_	9	punct	O	_
+23	..	..	PUNCT	_	_	6	punct	O	_
 
 # text = Percebe-se que, nos experimentos para os quais os fluidos não atravessaram a amostra por completo, foram obtidos os maiores retornos em um menor tempo..
 # sent_id = 119-20141209-TESEMSC_0-763
@@ -333502,9 +333502,9 @@
 3	leve	leve	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	considerado	considerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-6	tendo	ter	AUX	_	VerbForm=Ger	8	cop	O	_
+6	tendo	ter	VERB	_	VerbForm=Ger	5	advcl	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	grau	grau	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
+8	grau	grau	NOUN	_	Gender=Masc|Number=Sing	6	obj	O	_
 9	API	API	PROPN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 10	maior	grande	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 11	..	..	PUNCT	_	_	5	punct	O	_
@@ -350638,25 +350638,25 @@
 2	baixas	baixo	ADJ	_	Gender=Fem|Number=Plur	3	amod	O	_
 3	concentrações	concentração	NOUN	_	Gender=Fem|Number=Plur	14	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
-5	entretanto	entretanto	CCONJ	_	_	39	cc	O	_
+5	entretanto	entretanto	CCONJ	_	_	14	cc	O	_
 6	,	,	PUNCT	_	_	7	punct	O	_
-7	abaixo	abaixo	ADV	_	_	5	conj	O	_
+7	abaixo	abaixo	ADV	_	_	10	case	O	MWEPOS=ADP
 8-9	da	_	_	_	_	_	_	_	_
-8	de	de	ADP	_	_	10	case	O	_
+8	de	de	ADP	_	_	7	fixed	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
+10	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	Krafft	Krafft	PROPN	_	Gender=Unsp|Number=Sing	10	nmod	O	_
 13	,	,	PUNCT	_	_	7	punct	O	_
 14-15	tem-se	_	_	_	_	_	_	_	_
-14	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	aux	O	_
+14	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	se	se	PRON	_	_	14	expl	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
-17	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+17	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	14	obj	O	_
 18	hidratado	hidratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	acl	O	_
 19	sob	sob	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
-21	forma	forma	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
+21	forma	forma	NOUN	_	Gender=Fem|Number=Sing	17	nmod	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	precipitado	precipitar	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 24	(	(	PUNCT	_	_	25	punct	O	_
@@ -350668,15 +350668,15 @@
 30	concentrações	concentração	NOUN	_	Gender=Fem|Number=Plur	39	obl	O	_
 31	elevadas	elevar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	30	acl	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
-33	acima	acima	ADV	_	_	31	conj	O	_
+33	acima	acima	ADV	_	_	36	case	O	MWEPOS=ADP
 34-35	da	_	_	_	_	_	_	_	_
-34	de	de	ADP	_	_	36	case	O	_
+34	de	de	ADP	_	_	33	fixed	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
-36	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	33	obl	O	_
+36	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	30	conj	O	_
 37	de	de	ADP	_	_	38	case	O	_
 38	Krafft	Krafft	PROPN	_	Gender=Unsp|Number=Sing	36	nmod	O	_
 39-40	verifica-se	_	_	_	_	_	_	_	_
-39	verifica	verificar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	conj	O	_
+39	verifica	verificar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	conj	O	_
 40	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	39	expl	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
 42	formação	formação	NOUN	_	Gender=Fem|Number=Sing	39	obj	O	_
@@ -350693,8 +350693,8 @@
 53	,	,	PUNCT	_	_	52	flat:name	O	_
 54	2008	2008	NUM	_	NumType=Card	52	flat:name	O	_
 55	)	)	PUNCT	_	_	52	punct	O	_
-56	.	.	PUNCT	_	_	39	punct	O	_
-57	.	.	PUNCT	_	_	39	punct	O	_
+56	.	.	PUNCT	_	_	14	punct	O	_
+57	.	.	PUNCT	_	_	14	punct	O	_
 
 # text = Barros Neto (1999) explica que, a temperatura de grafft pode ser medida através de métodos alternativos, sendo eles: método espectrofotométrico ou turbidimétrico..
 # sent_id = 243-20141015-MONOGRAFIA_1-195
