commit 9118124f74957948ee3dec47459cae2b374b3902
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Dec 21 19:34:46 2021 -0300

    correções de verbos intransitivos

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index bf7159a..f072407 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -36318,7 +36318,7 @@
 30	condutividade	condutividade	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	,	,	PUNCT	_	_	28	punct	O	_
 32	flui	fluir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	acl:relcl	O	_
-33	calor	calor	NOUN	_	Gender=Masc|Number=Sing	32	obj	O	_
+33	calor	calor	NOUN	_	Gender=Masc|Number=Sing	32	nsubj	O	_
 34	até	até	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
 36	equilíbrio	equilíbrio	NOUN	_	Gender=Masc|Number=Sing	32	obl	O	_
@@ -39006,13 +39006,13 @@
 17	a	a	ADP	_	_	18	mark	O	_
 18	crescer	crescer	VERB	_	VerbForm=Inf	16	acl	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	18	obj	O	_
+20	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	18	nsubj	O	_
 21	a	a	ADP	_	_	24	mark	O	MWEPOS=SCONJ
 22	medida	medida	ADP	_	_	21	fixed	O	_
 23	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	21	fixed	O	_
 24	cresce	crescer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	advcl	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	frequência	frequência	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
+26	frequência	frequência	NOUN	_	Gender=Fem|Number=Sing	24	nsubj	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
 28	assim	assim	ADV	_	_	30	mark	O	MWEPOS=SCONJ
 29	como	como	ADP	_	_	28	fixed	O	_
@@ -96352,7 +96352,7 @@
 7	em	em	ADP	_	_	8	case	O	_
 8	operação	operação	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	FPSO	FPSO	PROPN	_	Gender=Fem|Number=Sing	6	obj	O	_
+10	FPSO	FPSO	PROPN	_	Gender=Fem|Number=Sing	6	nsubj	O	_
 11	Cidade	Cidade	PROPN	_	Number=Sing	10	flat:name	O	_
 12	de	de	ADP	_	_	10	flat:name	O	_
 13	Paraty	Paraty	PROPN	_	Number=Sing	10	flat:name	O	_
@@ -119196,9 +119196,9 @@
 20	de	de	ADP	_	_	21	case	O	_
 21	MEG	MEG	PROPN	_	Gender=Fem|Number=Sing	19	nmod	O	_
 22	pobre	pobre	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
-23	antes	antes	ADV	_	_	26	mark	O	_
+23	antes	antes	ADV	_	_	26	mark	O	MWEPOS=ADV
 24	de	de	ADP	_	_	23	fixed	O	_
-25	esta	este	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	26	obj	O	_
+25	esta	este	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	26	nsubj	O	_
 26	entrar	entrar	VERB	_	VerbForm=Inf	17	advcl	O	_
 27-28	no	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
@@ -249772,7 +249772,7 @@
 20	não	não	ADV	_	Polarity=Neg	21	advmod	O	_
 21	deixar	deixar	VERB	_	VerbForm=Inf	18	acl	O	_
 22	entrar	entrar	VERB	_	VerbForm=Inf	21	xcomp	O	_
-23	ar	ar	NOUN	_	Gender=Masc|Number=Sing	22	obj	O	_
+23	ar	ar	NOUN	_	Gender=Masc|Number=Sing	22	nsubj	O	_
 24-25	no	_	_	_	_	_	_	_	_
 24	em	em	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
