commit cada8310640e99453c71506dac6c337a584fd1f5
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu May 13 02:06:45 2021 -0300

    det filho de nummod -- erro?

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 9255a7e..e501143 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -501,7 +501,7 @@
 47	,	,	PUNCT	_	_	40	punct	O	_
 48-49	destes	_	_	_	_	_	_	O	_
 48	de	de	ADP	_	_	50	case	O	_
-49	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	50	det	O	_
+49	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	52	det	O	_
 50	278	278	NUM	_	NumType=Card	52	nummod	O	_
 51	048	048	NUM	_	NumType=Card	52	nummod	O	_
 52	Km²	Km²	PROPN	_	Gender=Masc|Number=Plur	54	nsubj	O	_
@@ -22759,7 +22759,7 @@
 4	original	original	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	7	det	O	_
-7	uma	um	NUM	_	NumType=Card	2	nummod	O	_
+7	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	2	nmod	O	_
 8-9	das	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	11	case	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
@@ -34782,7 +34782,7 @@
 17	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	15	conj	O	_
 18	em	em	ADP	_	_	20	case	O	_
 19	cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Tot	20	det	O	_
-20	um	um	NUM	_	NumType=Card	17	nummod	O	_
+20	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	13	obl	O	_
 21-22	dos	_	_	_	_	_	_	O	_
 21	de	de	ADP	_	_	23	case	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
@@ -54985,10 +54985,10 @@
 14-15	à	_	_	_	_	_	_	O	_
 14	a	a	ADP	_	_	16	case	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
-16	1000ºC	1000ºC	ADJ	_	Gender=Fem|NumType=Ord|Number=Sing	13	nummod	O	_
+16	1000ºC	1000ºC	SYM	_	Gender=Masc|Number=Plur	7	obl	O	_
 17	por	por	ADP	_	_	19	case	O	_
 18	30	30	NUM	_	NumType=Card	19	nummod	O	_
-19	minutos	minuto	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
+19	minutos	minuto	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	formando	formar	VERB	_	VerbForm=Ger	7	advcl	O	_
 22	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
@@ -134516,7 +134516,7 @@
 # text = Já o segundo riser, de descida, leva o gás contido nesta corrente e separado no topside, para ser injetado no duto de gás subsea e transportado para o continente. • DUTOS: A simulação de cada um dos três dutos propostos foi divida em três segmentos..
 # sent_id = 334-20150521-MONOGRAFIA_0-707
 1	Já	já	ADV	_	_	9	advmod	O	_
-2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
+2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	sentenciação
 3	segundo	segundo	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	4	amod	O	_
 4	riser	riser	NOUN	_	Gender=Masc|Number=Sing	9	nsubj	B=INSTRUMENTO	_
 5	,	,	PUNCT	_	_	7	punct	O	_
@@ -139372,7 +139372,7 @@
 30	natureza	natureza	NOUN	_	Gender=Fem|Number=Sing	27	obj	O	_
 31	de	de	ADP	_	_	33	case	O	_
 32	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	33	det	O	_
-33	uma	um	NUM	_	NumType=Card	30	nummod	O	_
+33	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	30	nmod	O	_
 34-35	das	_	_	_	_	_	_	O	_
 34	de	de	ADP	_	_	36	case	O	_
 35	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	36	det	O	_
@@ -158316,7 +158316,7 @@
 
 # text = Após a filtragem foram gerados os mapas finais de anomalia Bouguer e ar livre e realizada a interpretação dos mesmos, que consistiu na identificação e C a p í t u l o 4 - M e t o d o l o g i a | 63 interpretação de altos e baixos gravimétricos e feições estruturais ao longo dos dados..
 # sent_id = 258-20160602-TESEMSC_0-382
-1	Após	após	ADP	_	_	3	case	O	_
+1	Após	após	ADP	_	_	3	case	O	sentenciação
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	filtragem	filtragem	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 4	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	5	aux:pass	O	_
@@ -179928,8 +179928,8 @@
 17	escala	escala	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	512	512	NUM	_	NumType=Card	17	nummod	O	_
-20	mmH2	mmH2	ADP	_	_	19	compound	O	_
-21	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
+20	mmH2	mmH2	SYM	_	_	19	compound	O	_
+21	O	o	SYM	_	_	19	compound	O	_
 22	,	,	PUNCT	_	_	12	punct	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	modo	modo	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
@@ -179944,7 +179944,7 @@
 33	(	(	PUNCT	_	_	34	punct	O	_
 34	A	A	PROPN	_	Gender=Fem|Number=Sing	32	appos	O	_
 35	,	,	PUNCT	_	_	36	punct	O	_
-36	B,	B,	PROPN	_	Gender=Masc|Number=Sing	34	conj	O	_
+36	B,	B,	PROPN	_	Gender=Masc|Number=Sing	34	conj	O	tokenização
 37	C	C	PROPN	_	Number=Sing	36	flat:name	O	_
 38	)	)	PUNCT	_	_	34	punct	O	_
 39	,	,	PUNCT	_	_	28	punct	O	_
@@ -186295,7 +186295,7 @@
 12	ensaios	ensaio	NOUN	_	Gender=Masc|Number=Plur	10	nsubj:pass	O	_
 13	para	para	ADP	_	_	15	case	O	_
 14	cada	cada	DET	_	Gender=Masc|Number=Sing	15	det	O	_
-15	um	um	NUM	_	NumType=Card	12	nummod	O	_
+15	um	um	PRON	_	Gender=Masc|Number=Sing	10	obl	O	_
 16-17	dos	_	_	_	_	_	_	O	_
 16	de	de	ADP	_	_	18	case	O	_
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
@@ -193922,7 +193922,7 @@
 23	benefícios	benefício	NOUN	_	Gender=Masc|Number=Plur	21	conj	O	_
 24	de	de	ADP	_	_	26	case	O	_
 25	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	26	det	O	_
-26	uma	um	NUM	_	NumType=Card	21	nummod	O	_
+26	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	21	nmod	O	_
 27-28	das	_	_	_	_	_	_	O	_
 27	de	de	ADP	_	_	29	case	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -204647,7 +204647,7 @@
 5	preço	preço	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	de	de	ADP	_	_	8	case	O	_
 7	qualquer	qualquer	DET	_	Gender=Masc|Number=Sing|PronType=Ind	8	det	O	_
-8	um	um	NUM	_	NumType=Card	5	nummod	O	_
+8	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	nmod	O	_
 9-10	dos	_	_	_	_	_	_	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
@@ -262878,7 +262878,7 @@
 14	diferentes	diferente	ADJ	_	Gender=Fem|Number=Plur	13	amod	O	_
 15	para	para	ADP	_	_	17	case	O	_
 16	cada	cada	DET	_	Gender=Masc|Number=Sing	17	det	O	_
-17	um	um	NUM	_	NumType=Card	13	nummod	O	_
+17	um	um	PRON	_	Gender=Masc|Number=Sing	12	obl	O	_
 18-19	dos	_	_	_	_	_	_	O	_
 18	de	de	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
@@ -327144,7 +327144,7 @@
 22	Waale	Waale	PROPN	_	Number=Sing	20	flat:name	O	_
 23	para	para	ADP	_	_	25	case	O	_
 24	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	25	det	O	_
-25	uma	um	NUM	_	NumType=Card	14	nummod	O	_
+25	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	3	obl	O	_
 26-27	destas	_	_	_	_	_	_	O	_
 26	de	de	ADP	_	_	28	case	O	_
 27	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	28	det	O	_
@@ -334452,7 +334452,7 @@
 27	eficiência	eficiência	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
 28	de	de	ADP	_	_	30	case	O	_
 29	cada	cada	DET	_	Gender=Masc|Number=Sing	30	det	O	_
-30	um	um	NUM	_	NumType=Card	27	nummod	O	_
+30	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	27	nmod	O	_
 31-32	desses	_	_	_	_	_	_	O	_
 31	de	de	ADP	_	_	33	case	O	_
 32	esses	esse	DET	_	Gender=Masc|Number=Plur|PronType=Dem	33	det	O	_
@@ -343483,12 +343483,12 @@
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	resina	resina	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 28	entre	entre	ADP	_	_	30	case	O	_
-29	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
-30	2	2	NUM	_	NumType=Card	27	nummod	O	_
+29	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	34	det	O	_
+30	2	2	NUM	_	NumType=Card	34	nummod	O	_
 31	(	(	PUNCT	_	_	32	punct	O	_
-32	duas	dois	NUM	_	NumType=Card	30	conj	O	_
+32	duas	dois	NUM	_	NumType=Card	30	appos	O	_
 33	)	)	PUNCT	_	_	32	punct	O	_
-34	pets..	pets..	PUNCT	_	_	14	punct	O	_
+34	pets..	pets..	PUNCT	_	_	23	obl	O	_
 
 # text = Esse teste feito em uma piscina, teve um tempo de 1 (uma) hora..
 # sent_id = 411-20150702-MONOGRAFIA_0-318
@@ -344956,7 +344956,7 @@
 11	,	,	PUNCT	_	_	12	punct	O	_
 12	CONAMA	CONAMA	PROPN	_	Gender=Fem|Number=Sing	5	appos	B=ORGANIZAÇÃO	_
 13	,	,	PUNCT	_	_	12	punct	O	_
-14-15	n°	_	_	_	_	_	_	O	_
+14-15	n°	_	_	_	_	_	_	O	tokenização
 14	em	em	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	357/2005	357/2005	NUM	_	NumType=Card	2	nummod	O	_
