commit 51713966b5603e3e4f2c04fc63f436788212b63d
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Jun 9 13:20:48 2021 -0300

    fácies sedimentar

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index dbf4c42..1e9d768 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -20327,7 +20327,7 @@
 1	Análise	Análise	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 2	de	de	ADP	_	_	3	case	O	_
 3	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	1	nmod	B=LITOLOGIA	_
-4	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	3	compound	I=LITOLOGIA	_
+4	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	3	amod	I=LITOLOGIA	_
 
 # text = A análise de fácies é um método fundamental de caracterização de rocha com atributos litológicos, físicos e biológicos singulares e distintos das demais..
 # sent_id = 284-20150917-MONOGRAFIA_0-43
@@ -20472,8 +20472,8 @@
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	17	nsubj	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	11	nmod	B=LITOLOGIA	_
-14	sedimentares	sedimentar	NOUN	_	Gender=Masc|Number=Plur	13	nmod	I=LITOLOGIA	_
+13	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	11	nmod	B=LITOLOGIA	_
+14	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	13	amod	I=LITOLOGIA	_
 15	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	cop	O	_
 16	particularmente	particularmente	ADV	_	_	17	advmod	O	_
 17	relevante	relevante	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
@@ -28424,7 +28424,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-291
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nsubj	B=LITOLOGIA	_
-3	sedimentares	sedimentar	NOUN	_	Gender=Fem|Number=Plur	2	nmod	I=LITOLOGIA	_
+3	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	2	amod	I=LITOLOGIA	_
 4	acima	acima	ADV	_	_	5	advmod	O	_
 5	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	2	acl	O	_
 6-7	encontram-se	_	_	_	_	_	_	_	_
@@ -32060,7 +32060,7 @@
 11	presente	presente	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 12	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 13	,	,	PUNCT	_	_	7	punct	O	_
-14	fácies	fácie	NOUN	_	Gender=Fem|Number=Plur	7	nsubj:pass	B=LITOLOGIA	_
+14	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	7	nsubj:pass	B=LITOLOGIA	_
 15	conglomeráticas	conglomerático	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
 16	(	(	PUNCT	_	_	17	punct	O	_
 17	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	amod	B=LITOLOGIA	_
@@ -64328,7 +64328,7 @@
 13	análise	análise	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	13	nmod	B=LITOLOGIA	_
-16	sedimentares	sedimentar	NOUN	_	Gender=Masc|Number=Plur	15	nmod	I=LITOLOGIA	_
+16	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	15	amod	I=LITOLOGIA	_
 17	,	,	PUNCT	_	_	18	punct	O	_
 18	contribuindo	contribuir	VERB	_	VerbForm=Ger	3	advcl	O	_
 19	para	para	ADP	_	_	21	case	O	_
@@ -64800,8 +64800,8 @@
 8	de	de	ADP	_	_	9	case	O	_
 9	análise	análise	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	9	nmod	B=LITOLOGIA	_
-12	sedimentares	sedimentar	NOUN	_	Gender=Masc|Number=Plur	11	nmod	I=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	9	nmod	B=LITOLOGIA	_
+12	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	11	amod	I=LITOLOGIA	_
 13	e	e	CCONJ	_	_	15	cc	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	paleocorrentes	paleocorrente	NOUN	_	Gender=Masc|Number=Plur	11	conj	O	_
@@ -64849,8 +64849,8 @@
 12	e	e	CCONJ	_	_	13	cc	O	_
 13	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	13	nmod	B=LITOLOGIA	_
-16	sedimentares	sedimentar	NOUN	_	Gender=Masc|Number=Plur	15	nmod	I=LITOLOGIA	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	13	nmod	B=LITOLOGIA	_
+16	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	15	amod	I=LITOLOGIA	_
 17-18	do	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
@@ -73409,7 +73409,7 @@
 16	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	15	acl	I=LITOLOGIA	_
 17	cavalgante	cavalgante	ADJ	_	Gender=Fem|Number=Sing	15	amod	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
-19	fácies	fácie	ADJ	_	Gender=Fem|Number=Plur	15	nmod:appos	B=LITOLOGIA	_
+19	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	15	nmod:appos	B=LITOLOGIA	_
 20	Sr	Sr	PROPN	_	Gender=Masc|Number=Sing	19	nmod	O	_
 21	)	)	PUNCT	_	_	19	punct	O	_
 22	(	(	PUNCT	_	_	23	punct	O	_
@@ -74683,7 +74683,7 @@
 2	associação	associação	NOUN	_	Gender=Fem|Number=Sing	16	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
-5	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	4	acl	O	_
+5	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	2	acl	O	_
 6-7	na	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -74718,7 +74718,7 @@
 35	Sm	Sm	PROPN	_	Gender=Masc|Number=Sing	31	appos	O	_
 36	)	)	PUNCT	_	_	35	punct	O	_
 37	e	e	CCONJ	_	_	38	cc	O	_
-38	fácies	fácie	ADJ	_	Gender=Fem|Number=Plur	28	conj	B=LITOLOGIA	_
+38	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	28	conj	B=LITOLOGIA	_
 39	provenientes	proveniente	ADJ	_	Gender=Fem|Number=Plur	38	amod	O	_
 40	de	de	ADP	_	_	41	case	O	_
 41	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	39	obl	O	_
@@ -77482,9 +77482,9 @@
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
 12	associações	associação	NOUN	_	Gender=Fem|Number=Plur	7	conj	O	_
 13	de	de	ADP	_	_	14	case	O	_
-14	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	12	nmod	B=LITOLOGIA	_
-15	sedimentares	sedimentar	NOUN	_	Gender=Masc|Number=Plur	14	nmod	I=LITOLOGIA	_
-16	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	17	nsubj	O	_
+14	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	12	nmod	B=LITOLOGIA	_
+15	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	14	amod	I=LITOLOGIA	_
+16	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	17	nsubj	O	_
 17	permitiram	permitir	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	14	acl:relcl	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 19	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
