commit b7f3d42913ca7db33f3c2927609ba56ecc3efa5c
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu May 13 06:35:52 2021 -0300

    "um de" indicates PRON PronType=Ind

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index e501143..ab5cabc 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -2522,7 +2522,7 @@
 27	Província	Província	PROPN	_	Gender=Fem|Number=Sing	22	nmod	O	_
 28	Mantiqueira	Mantiqueira	PROPN	_	Number=Sing	27	flat:name	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
-30	um	um	NUM	_	NumType=Card	27	conj	O	_
+30	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	22	conj	O	_
 31-32	dos	_	_	_	_	_	_	O	_
 31	de	de	ADP	_	_	34	case	O	_
 32	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	34	det	O	_
@@ -5583,7 +5583,7 @@
 17	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	18	nsubj	O	_
 18	corresponde	corresponder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	O	_
 19	a	a	ADP	_	_	20	case	O	_
-20	um	um	NUM	_	NumType=Card	18	obj	O	_
+20	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	18	obj	O	_
 21-22	dos	_	_	_	_	_	_	O	_
 21	de	de	ADP	_	_	23	case	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
@@ -11108,7 +11108,7 @@
 9	físicas	físico	ADJ	_	Gender=Fem|Number=Plur	7	conj	O	_
 10	,	,	PUNCT	_	_	3	punct	O	_
 11	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	12	det	O	_
-12	uma	um	NUM	_	NumType=Card	3	nsubj:pass	O	_
+12	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	15	nsubj:pass	O	_
 13-14	delas	_	_	_	_	_	_	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	12	nmod	O	_
@@ -14270,13 +14270,13 @@
 11	nm	nm	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
 12	em	em	ADP	_	_	15	case	O	_
 13-14	pelo	_	_	_	_	_	_	O	_
-13	por	por	ADP	_	_	15	case	O	_
-14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	menos	menos	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
-16	uma	um	NUM	_	NumType=Card	15	nummod	O	_
+13	por	por	ADP	_	_	16	case	O	_
+14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	fixed	O	_
+15	menos	menos	NOUN	_	Gender=Masc|Number=Plur	13	fixed	O	_
+16	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	8	obl	O	_
 17	de	de	ADP	_	_	19	case	O	_
 18	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	19	det	O	_
-19	dimensões	dimensão	NOUN	_	Gender=Fem|Number=Plur	15	nmod	O	_
+19	dimensões	dimensão	NOUN	_	Gender=Fem|Number=Plur	16	nmod	O	_
 20	,	,	PUNCT	_	_	5	punct	O	_
 21	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	22	nsubj	O	_
 22	conferem	conferir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	O	_
@@ -22189,12 +22189,12 @@
 36	)	)	PUNCT	_	_	34	punct	O	_
 37	,	,	PUNCT	_	_	26	punct	O	_
 38	sendo	ser	AUX	_	VerbForm=Ger	43	cop	O	_
-39	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
+39	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	26	advcl	O	_
 40-41	das	_	_	_	_	_	_	O	_
 40	de	de	ADP	_	_	43	case	O	_
 41	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	43	det	O	_
 42	maiores	grande	ADJ	_	Gender=Fem|Number=Plur	43	amod	O	_
-43	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	26	obl	B=BACIA:TIPO	_
+43	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	39	nmod	B=BACIA:TIPO	_
 44-45	do	_	_	_	_	_	_	O	_
 44	de	de	ADP	_	_	46	case	O	_
 45	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
@@ -34127,7 +34127,7 @@
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
 19	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	15	nmod	B=FLUIDO	_
 20	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	cop	O	_
-21	uma	um	NUM	_	NumType=Card	3	conj	O	_
+21	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	3	conj	O	_
 22-23	das	_	_	_	_	_	_	O	_
 22	de	de	ADP	_	_	24	case	O	_
 23	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	24	det	O	_
@@ -39137,7 +39137,7 @@
 8	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	7	obj	O	_
 9	eletromagnético	eletromagnético	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 10	como	como	ADP	_	_	11	case	O	_
-11	uma	um	NUM	_	NumType=Card	7	xcomp	O	_
+11	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	7	xcomp	O	_
 12-13	das	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	14	case	O	_
 13	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	14	det	O	_
@@ -51854,7 +51854,7 @@
 5	íons	íom	NOUN	_	Gender=Masc|Number=Plur	3	nmod	O	_
 6-7	numa	_	_	_	_	_	_	O	_
 6	em	em	ADP	_	_	7	case	O	_
-7	uma	um	NUM	_	NumType=Card	3	nummod	O	_
+7	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	3	nmod	O	_
 8-9	das	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	10	case	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
@@ -54571,7 +54571,7 @@
 6	realizado	realizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 7	para	para	ADP	_	_	9	case	O	_
 8	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	9	det	O	_
-9	uma	um	NUM	_	NumType=Card	6	obl	O	_
+9	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	6	obl	O	_
 10-11	das	_	_	_	_	_	_	O	_
 10	de	de	ADP	_	_	12	case	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
@@ -56125,7 +56125,7 @@
 22	III.6	III.6	PROPN	_	Number=Sing	21	flat:name	O	_
 23	para	para	ADP	_	_	25	case	O	_
 24	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	25	det	O	_
-25	uma	um	NUM	_	NumType=Card	18	obl	O	_
+25	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	18	obl	O	_
 26-27	das	_	_	_	_	_	_	O	_
 26	de	de	ADP	_	_	29	case	O	_
 27	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -65466,7 +65466,7 @@
 10	desértico	desértico	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
 11	,	,	PUNCT	_	_	13	punct	O	_
 12	sendo	ser	AUX	_	VerbForm=Ger	13	cop	O	_
-13	uma	um	NUM	_	NumType=Card	6	advcl	O	_
+13	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	6	advcl	O	_
 14	de	de	ADP	_	_	16	case	O	_
 15	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	16	det	O	_
 16	características	característica	NOUN	_	Gender=Fem|Number=Plur	13	nmod	O	_
@@ -68479,7 +68479,7 @@
 8	de	de	ADP	_	_	9	case	O	_
 9	espessura	espessura	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 10	,	,	PUNCT	_	_	11	punct	O	_
-11	uma	um	NUM	_	NumType=Card	7	appos	O	_
+11	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	7	appos	O	_
 12-13	das	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	14	case	O	_
 13	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	14	det	O	_
@@ -73095,7 +73095,7 @@
 11	de	de	ADP	_	_	12	case	O	_
 12	silexitos	silexito	NOUN	_	Gender=Masc|Number=Plur	10	nmod	B=LITOLOGIA	_
 13	compreendem	compreender	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-14	um	um	NUM	_	NumType=Card	13	obj	O	_
+14	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	13	obj	O	_
 15-16	dos	_	_	_	_	_	_	O	_
 15	de	de	ADP	_	_	18	case	O	_
 16	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
@@ -74146,7 +74146,7 @@
 
 # text = Um dos principais exemplos destas fácies pelíticas é o afloramento no Ribeirão Três Barras (Ponto 64 – Tabela 2)..
 # sent_id = 283-20150917-MONOGRAFIA_0-352
-1	Um	um	NUM	_	NumType=Card	12	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	12	nsubj	O	_
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	5	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -78467,7 +78467,7 @@
 35	de	de	ADP	_	_	34	flat:name	O	_
 36	Quioto	Quioto	PROPN	_	Number=Sing	34	flat:name	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
-38	um	um	NUM	_	NumType=Card	31	appos	O	_
+38	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	31	appos	O	_
 39-40	dos	_	_	_	_	_	_	O	_
 39	de	de	ADP	_	_	42	case	O	_
 40	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	42	det	O	_
@@ -79836,7 +79836,7 @@
 21	,	,	PUNCT	_	_	16	punct	O	_
 22	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	23	aux:pass	O	_
 23	adotado	adotar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-24	uma	um	NUM	_	NumType=Card	23	nsubj:pass	O	_
+24	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	23	nsubj:pass	O	_
 25-26	das	_	_	_	_	_	_	O	_
 25	de	de	ADP	_	_	28	case	O	_
 26	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	28	det	O	_
@@ -80672,7 +80672,7 @@
 3	de	de	ADP	_	_	4	case	O	_
 4	absorção	absorção	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	cop	O	_
-6	uma	um	NUM	_	NumType=Card	0	root	O	_
+6	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	0	root	O	_
 7-8	das	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	10	case	O	_
 8	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
@@ -83210,7 +83210,7 @@
 6	gás	gás	NOUN	_	Gender=Masc|Number=Sing	4	conj	B=FLUIDO	_
 7	depletados	depletar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	2	acl	O	_
 8	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
-9	uma	um	NUM	_	NumType=Card	0	root	O	_
+9	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	0	root	O	_
 10-11	das	_	_	_	_	_	_	O	_
 10	de	de	ADP	_	_	12	case	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
@@ -84872,7 +84872,7 @@
 15	ArcGIS	ArcGIS	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	considerado	considerar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	14	acl	O	_
-18	um	um	NUM	_	NumType=Card	17	obj	O	_
+18	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	17	obj	O	_
 19-20	dos	_	_	_	_	_	_	O	_
 19	de	de	ADP	_	_	21	case	O	_
 20	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	21	det	O	_
@@ -86265,7 +86265,7 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	escória	escória	NOUN	_	Gender=Fem|Number=Sing	4	nsubj	O	_
 3	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
-4	uma	um	NUM	_	NumType=Card	0	root	O	_
+4	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	0	root	O	_
 5-6	das	_	_	_	_	_	_	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
@@ -90504,11 +90504,11 @@
 42	de	de	ADP	_	_	43	case	O	_
 43	carboduto	carboduto	NOUN	_	Gender=Masc|Number=Sing	40	nmod	O	_
 44	distinto	distinto	ADJ	_	Gender=Masc|Number=Sing	43	amod	O	_
-45	um	um	NUM	_	NumType=Card	43	nummod	O	_
+45	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	44	obl	O	_
 46-47	do	_	_	_	_	_	_	O	_
 46	de	de	ADP	_	_	48	case	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
-48	outro	outro	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	43	nmod	O	_
+48	outro	outro	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	45	nmod	O	_
 49	,	,	PUNCT	_	_	52	punct	O	_
 50	para	para	ADP	_	_	52	case	O	_
 51	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	52	det	O	_
@@ -90630,7 +90630,7 @@
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	PNA-1	PNA-1	PROPN	_	Gender=Fem|Number=Sing	11	obj	O	_
 15	,	,	PUNCT	_	_	20	punct	O	_
-16	um	um	NUM	_	NumType=Card	20	nsubj	O	_
+16	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	20	nsubj	O	_
 17-18	dos	_	_	_	_	_	_	O	_
 17	de	de	ADP	_	_	19	case	O	_
 18	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
@@ -93418,7 +93418,7 @@
 46	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	47	det	O	_
 47	Petrobras	Petrobras	PROPN	_	Gender=Fem|Number=Sing	48	nsubj	O	_
 48	montou	montar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	42	acl:relcl	O	_
-49	um	um	NUM	_	NumType=Card	48	obj	O	_
+49	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	48	obj	O	_
 50-51	dos	_	_	_	_	_	_	O	_
 50	de	de	ADP	_	_	52	case	O	_
 51	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	52	det	O	_
@@ -98310,7 +98310,7 @@
 3	de	de	ADP	_	_	4	case	O	_
 4	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=QUÍMICA	_
 5	,	,	PUNCT	_	_	6	punct	O	_
-6	um	um	NUM	_	NumType=Card	2	appos	O	_
+6	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	2	appos	O	_
 7-8	dos	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	10	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
@@ -113369,7 +113369,7 @@
 10	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	proposta	proposta	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
 12	para	para	ADP	_	_	13	case	O	_
-13	um	um	NUM	_	NumType=Card	11	nummod	O	_
+13	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	nmod	O	_
 14-15	dos	_	_	_	_	_	_	O	_
 14	de	de	ADP	_	_	17	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	17	det	O	_
@@ -114407,7 +114407,7 @@
 
 # text = Um dos desafios encontrados pela Petrobras na exploração do petróleo da área do pré-sal reside na grande quantidade de gás natural associado..
 # sent_id = 334-20150521-MONOGRAFIA_0-150
-1	Um	um	NUM	_	NumType=Card	21	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	21	nsubj	O	_
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
@@ -114547,7 +114547,7 @@
 5	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	6	det	O	_
 6	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	3	nmod	B=QUÍMICA	_
 7	,	,	PUNCT	_	_	3	punct	O	_
-8	uma	um	NUM	_	NumType=Card	18	nsubj	O	_
+8	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	18	nsubj	O	_
 9-10	das	_	_	_	_	_	_	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
@@ -115331,7 +115331,7 @@
 16	elevado	elevado	ADJ	_	Gender=Masc|Number=Sing	17	amod	O	_
 17	custo	custo	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
 18	,	,	PUNCT	_	_	6	punct	O	_
-19	uma	um	NUM	_	NumType=Card	26	nsubj	O	_
+19	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	26	nsubj	O	_
 20-21	das	_	_	_	_	_	_	O	_
 20	de	de	ADP	_	_	23	case	O	_
 21	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	23	det	O	_
@@ -115495,7 +115495,7 @@
 
 # text = Uma das áreas mais desenvolvidas na exploração de petróleo em águas ultraprofundas é o Golfo do México..
 # sent_id = 334-20150521-MONOGRAFIA_0-176
-1	Uma	um	NUM	_	NumType=Card	17	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	17	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -115947,7 +115947,7 @@
 # sent_id = 334-20150521-MONOGRAFIA_0-185
 1	Este	este	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	3	nsubj	O	_
 2	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	cop	O	_
-3	um	um	NUM	_	NumType=Card	0	root	O	_
+3	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	0	root	O	_
 4-5	dos	_	_	_	_	_	_	O	_
 4	de	de	ADP	_	_	6	case	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
@@ -116078,7 +116078,7 @@
 
 # text = Uma das principais características a ser observada durante a escolha do tipo de processamento a ser adotado é a razão entre os volumes produzidos de gás associado e óleo (RGO)..
 # sent_id = 334-20150521-MONOGRAFIA_0-187
-1	Uma	um	NUM	_	NumType=Card	22	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	22	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	5	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	5	det	O	_
@@ -119393,7 +119393,7 @@
 
 # text = Uma das grandes empresas do setor de petróleo, a norueguesa Statoil já trabalha com tecnologias subsea há décadas, porém só nos últimos anos o setor petrolífero passou a reconhecer e priorizar as tecnologias de processamento submarino..
 # sent_id = 334-20150521-MONOGRAFIA_0-277
-1	Uma	um	NUM	_	NumType=Card	16	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	16	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	5	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	5	det	O	_
@@ -120415,7 +120415,7 @@
 24	três	três	NUM	_	NumType=Card	25	nummod	O	_
 25	frentes	frente	NOUN	_	Gender=Fem|Number=Plur	22	obl:agent	O	_
 26	sendo	ser	AUX	_	VerbForm=Ger	27	cop	O	_
-27	uma	um	NUM	_	NumType=Card	22	nsubj:pass	O	_
+27	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	22	nsubj:pass	O	_
 28-29	delas	_	_	_	_	_	_	O	_
 28	de	de	ADP	_	_	29	case	O	_
 29	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	27	nmod	O	_
@@ -121347,7 +121347,7 @@
 
 # text = Uma delas é que o separador submarino submerso utilizado em Pazflor é de duas fases, enquanto que o utilizado em Tordis é trifásico..
 # sent_id = 334-20150521-MONOGRAFIA_0-323
-1	Uma	um	NUM	_	NumType=Card	4	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	4	nsubj	O	_
 2-3	delas	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	3	case	O	_
 3	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	1	nmod	O	_
@@ -123365,7 +123365,7 @@
 7	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	mark	O	_
 8	vez	vez	NOUN	_	Gender=Fem|Number=Sing	7	fixed	O	_
 9	que	que	SCONJ	_	_	7	fixed	O	_
-10	uma	um	NUM	_	NumType=Card	23	nsubj	O	_
+10	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	23	nsubj	O	_
 11-12	das	_	_	_	_	_	_	O	_
 11	de	de	ADP	_	_	14	case	O	_
 12	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	14	det	O	_
@@ -123490,7 +123490,7 @@
 
 # text = Um dos principais exemplos de desenvolvimento e aplicação dos dutos submarinos é o campo de Snøhvit localizado no Mar de Barents, a 140 quilômetros da costa da Noruega..
 # sent_id = 334-20150521-MONOGRAFIA_0-378
-1	Um	um	NUM	_	NumType=Card	16	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	16	nsubj	O	_
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	5	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -126725,7 +126725,7 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	monoetilenoglicol	monoetilenoglicol	NOUN	_	Gender=Masc|Number=Sing	4	nsubj	O	_
 3	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
-4	um	um	NUM	_	NumType=Card	0	root	O	_
+4	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	0	root	O	_
 5-6	dos	_	_	_	_	_	_	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	7	det	O	_
@@ -127195,7 +127195,7 @@
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
 13	pré-sal	pré-sal	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 14	representam	representar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-15	um	um	NUM	_	NumType=Card	14	obj	O	_
+15	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	14	obj	O	_
 16-17	dos	_	_	_	_	_	_	O	_
 16	de	de	ADP	_	_	19	case	O	_
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	19	det	O	_
@@ -129913,7 +129913,7 @@
 # sent_id = 334-20150521-MONOGRAFIA_0-565
 1	Isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	2	nsubj	O	_
 2	resolve	resolver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-3	uma	um	NUM	_	NumType=Card	2	obj	O	_
+3	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	2	obj	O	_
 4-5	das	_	_	_	_	_	_	O	_
 4	de	de	ADP	_	_	6	case	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
@@ -134021,7 +134021,7 @@
 # text = Apenas um deles conta com fluidos de processos no lado quente e no lado frio, os outros dois são trocadores de calor que funcionam com água de resfriamento. • AQUECEDORES: Este tipo de equipamento é requerido nas três etapas do processo, subsea, topside e onshore..
 # sent_id = 334-20150521-MONOGRAFIA_0-698
 1	Apenas	apenas	ADV	_	_	2	advmod	O	_
-2	um	um	NUM	_	NumType=Card	5	nsubj	O	_
+2	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	nsubj	O	_
 3-4	deles	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	eles	eles	PRON	_	Gender=Masc|Number=Plur|Person=3|PronType=Prs	2	nmod	O	_
@@ -134159,7 +134159,7 @@
 # text = Apenas uma das cinco bombas não é subsea e está localizada no continente para pressurizar a corrente de MEG pobre antes de esta entrar no duto de transporte..
 # sent_id = 334-20150521-MONOGRAFIA_0-700
 1	Apenas	apenas	ADV	_	_	9	advmod	O	_
-2	uma	um	NUM	_	NumType=Card	9	nsubj	O	_
+2	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	9	nsubj	O	_
 3-4	das	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	6	case	O	_
 4	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
@@ -134561,7 +134561,7 @@
 41	simulação	simulação	NOUN	_	Gender=Fem|Number=Sing	51	nsubj:pass	O	_
 42	de	de	ADP	_	_	44	case	O	_
 43	cada	cada	DET	_	Gender=Masc|Number=Sing	44	det	O	_
-44	um	um	NUM	_	NumType=Card	41	nummod	O	_
+44	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	41	nmod	O	_
 45-46	dos	_	_	_	_	_	_	O	_
 45	de	de	ADP	_	_	48	case	O	_
 46	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	48	det	O	_
@@ -137288,7 +137288,7 @@
 4	sido	ser	AUX	_	VerbForm=Part	5	aux:pass	O	_
 5	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 6	e	e	CCONJ	_	_	7	cc	O	_
-7	uma	um	NUM	_	NumType=Card	5	conj	O	_
+7	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	5	conj	O	_
 8-9	delas	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	7	nmod	O	_
@@ -139048,7 +139048,7 @@
 12	aplicações	aplicação	NOUN	_	Gender=Fem|Number=Plur	10	conj	O	_
 13	,	,	PUNCT	_	_	15	punct	O	_
 14	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	15	det	O	_
-15	uma	um	NUM	_	NumType=Card	8	conj	O	_
+15	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	8	conj	O	_
 16-17	delas	_	_	_	_	_	_	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	15	nmod	O	_
@@ -140181,7 +140181,7 @@
 
 # text = Uma das ideias básica que norteiam a seleção de tensoativos é que os mais solúveis em água tendem a formar emulsões
 # sent_id = 202-20140404-MONOGRAFIA_0-87
-1	Uma	um	NUM	_	NumType=Card	0	root	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	0	root	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -140398,7 +140398,7 @@
 6	em	em	ADP	_	_	7	case	O	_
 7	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	4	conj	B=FLUIDO	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
-9	um	um	NUM	_	NumType=Card	0	root	O	_
+9	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	0	root	O	_
 10-11	dos	_	_	_	_	_	_	O	_
 10	de	de	ADP	_	_	13	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -141621,7 +141621,7 @@
 9	agentes	agente	NOUN	_	Gender=Masc|Number=Plur	7	nsubj:pass	O	_
 10	tensoativos	tensoativo	ADJ	_	Gender=Masc|Number=Plur	9	amod	O	_
 11	,	,	PUNCT	_	_	12	punct	O	_
-12	um	um	NUM	_	NumType=Card	9	appos	O	_
+12	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	9	appos	O	_
 13-14	deles	_	_	_	_	_	_	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	eles	eles	PRON	_	Gender=Masc|Number=Plur|Person=3|PronType=Prs	12	nmod	O	_
@@ -148511,7 +148511,7 @@
 15	côncava	côncavo	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	compreende	compreender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	acl:relcl	O	_
-18	uma	um	NUM	_	NumType=Card	17	obj	O	_
+18	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	17	obj	O	_
 19-20	das	_	_	_	_	_	_	O	_
 19	de	de	ADP	_	_	22	case	O	_
 20	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	22	det	O	_
@@ -149064,7 +149064,7 @@
 1	Essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	2	det	O	_
 2	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	3	nsubj	O	_
 3	representa	representar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-4	um	um	NUM	_	NumType=Card	3	obj	O	_
+4	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	3	obj	O	_
 5-6	dos	_	_	_	_	_	_	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	7	det	O	_
@@ -157482,7 +157482,7 @@
 17	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 18	para	para	ADP	_	_	20	case	O	_
 19	cada	cada	DET	_	Gender=Masc|Number=Sing	20	det	O	_
-20	um	um	NUM	_	NumType=Card	11	obl	O	_
+20	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	obl	O	_
 21-22	dos	_	_	_	_	_	_	O	_
 21	de	de	ADP	_	_	23	case	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
@@ -160511,7 +160511,7 @@
 
 # text = Uma das explicações para a formação dessas estruturas está relacionada a atenuação de uma força principal na direção vertical (força de gravidade) resultante do acréscimo de forças extensionais em margens divergentes, ou seja são formadas de forma reativa aos processos extensionais (COBBOLD & VENDEVILLE, 1987; SZATMARI & AIRES, 1987; DUVAL ET AL., 1992; VENDEVILLE & JACKSON, 1992 apud MOHRIAK, 2008)..
 # sent_id = 258-20160602-TESEMSC_0-432
-1	Uma	um	NUM	_	NumType=Card	12	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	12	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -160995,12 +160995,12 @@
 6	em	em	ADP	_	_	7	case	O	_
 7	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 8	para	para	ADP	_	_	9	case	O	_
-9	cada	cada	DET	_	Gender=Masc|Number=Sing	2	nmod	O	_
-10	um	um	NUM	_	NumType=Card	9	nummod	O	_
+9	cada	cada	DET	_	Gender=Masc|Number=Sing	10	det	O	_
+10	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	2	nmod	O	_
 11-12	dos	_	_	_	_	_	_	O	_
 11	de	de	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
-13	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	9	nmod	O	_
+13	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
 14	apresentam	apresentar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
 16	visualização	visualização	NOUN	_	Gender=Fem|Number=Sing	14	obj	O	_
@@ -167818,7 +167818,7 @@
 56	se	se	PRON	_	_	55	expl	O	_
 57	degradado	degradar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	55	xcomp	O	_
 58	por	por	ADP	_	_	59	case	O	_
-59	um	um	PRON	_	PronType=Ind	57	obl	O	_
+59	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	57	obl	O	_
 60-61	dos	_	_	_	_	_	_	O	_
 60	de	de	ADP	_	_	63	case	O	_
 61	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	63	det	O	_
@@ -171295,7 +171295,7 @@
 5	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	3	nmod	B=QUÍMICA	_
 6	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	7	expl	O	_
 7	aproximam	aproximar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	15	advcl	O	_
-8	uma	um	NUM	_	NumType=Card	7	obj	O	_
+8	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	7	obj	O	_
 9-10	da	_	_	_	_	_	_	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -172020,7 +172020,7 @@
 49	alto	alto	ADJ	_	Gender=Masc|Number=Sing	47	amod	O	_
 50	)	)	PUNCT	_	_	47	punct	O	_
 51	e	e	CCONJ	_	_	52	cc	O	_
-52	uma	um	NUM	_	NumType=Card	41	conj	O	_
+52	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	41	conj	O	_
 53	de	de	ADP	_	_	55	case	O	_
 54	baixo	baixo	ADJ	_	Gender=Masc|Number=Sing	55	amod	O	_
 55	peso	peso	NOUN	_	Gender=Masc|Number=Sing	52	nmod	O	_
@@ -172653,11 +172653,11 @@
 5	distam	dizer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	1	1	NUM	_	NumType=Card	7	nummod	O	_
 7	metro	metro	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
-8	um	um	NUM	_	NumType=Card	7	nummod	O	_
+8	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	obl	O	_
 9-10	do	_	_	_	_	_	_	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	outro	outro	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	7	nmod	O	_
+11	outro	outro	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	8	nmod	O	_
 12	.	.	PUNCT	_	_	5	punct	O	_
 13	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -173341,7 +173341,7 @@
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	aux:pass	O	_
 6	medida	medir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 7	por	por	ADP	_	_	8	case	O	_
-8	um	um	NUM	_	NumType=Card	6	obl	O	_
+8	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	6	obl	O	_
 9-10	dos	_	_	_	_	_	_	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
@@ -173362,7 +173362,7 @@
 23	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	25	expl	O	_
 24	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	25	aux	O	_
 25	utilizando	utilizar	VERB	_	VerbForm=Ger	19	acl:relcl	O	_
-26	um	um	NUM	_	NumType=Card	25	obj	O	_
+26	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	25	obj	O	_
 27-28	dos	_	_	_	_	_	_	O	_
 27	de	de	ADP	_	_	29	case	O	_
 28	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	29	det	O	_
@@ -173691,11 +173691,11 @@
 4	pressão	pressão	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 5	distam	dizer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	1m	1m	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
-7	uma	um	NUM	_	NumType=Card	6	nummod	O	_
+7	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	5	obl	O	_
 8-9	da	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	outra	outro	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	6	nmod	O	_
+10	outra	outro	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	7	nmod	O	_
 11	e	e	CCONJ	_	_	13	cc	O	_
 12	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	cop	O	_
 13	localizadas	localizar	ADJ	_	Gender=Fem|Number=Plur	5	conj	O	_
@@ -176253,7 +176253,7 @@
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	abertura	abertura	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	uma	um	NUM	_	NumType=Card	11	nummod	O	_
+13	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	11	nmod	O	_
 14-15	das	_	_	_	_	_	_	O	_
 14	de	de	ADP	_	_	16	case	O	_
 15	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	16	det	O	_
@@ -176490,7 +176490,7 @@
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	abertura	abertura	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	uma	um	NUM	_	NumType=Card	14	nummod	O	_
+16	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	14	nmod	O	_
 17-18	das	_	_	_	_	_	_	O	_
 17	de	de	ADP	_	_	19	case	O	_
 18	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	19	det	O	_
@@ -182093,7 +182093,7 @@
 # sent_id = 279-20140530-MONOGRAFIA_0-6
 1	Atualmente	atualmente	ADV	_	_	11	advmod	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
-3	um	um	NUM	_	NumType=Card	11	nsubj	O	_
+3	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	nsubj	O	_
 4-5	dos	_	_	_	_	_	_	O	_
 4	de	de	ADP	_	_	6	case	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
@@ -182688,12 +182688,12 @@
 # text = Segundo ANP um dos princípios do marco regulatório para produção de biodiesel é o aproveitamento das oleaginosas de acordo com as diversidades regionais, fortalecendo as potencialidades de cada região para produção da matéria-prima..
 # sent_id = 279-20140530-MONOGRAFIA_0-25
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	ANP	ANP	PROPN	_	Gender=Masc|Number=Sing	17	nmod	O	_
-3	um	um	NUM	_	NumType=Card	2	nummod	O	_
+2	ANP	ANP	PROPN	_	Gender=Masc|Number=Sing	17	obl	O	_
+3	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	17	nsubj	O	_
 4-5	dos	_	_	_	_	_	_	O	_
 4	de	de	ADP	_	_	6	case	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
-6	princípios	princípio	NOUN	_	Gender=Masc|Number=Plur	2	nmod	O	_
+6	princípios	princípio	NOUN	_	Gender=Masc|Number=Plur	3	nmod	O	_
 7-8	do	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
@@ -186035,7 +186035,7 @@
 31	funcionando	funcionar	VERB	_	VerbForm=Ger	23	acl	O	_
 32	com	com	ADP	_	_	34	case	O	_
 33	cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Tot	34	det	O	_
-34	um	um	NUM	_	NumType=Card	31	obj	O	_
+34	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	31	obj	O	_
 35-36	dos	_	_	_	_	_	_	O	_
 35	de	de	ADP	_	_	37	case	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
@@ -186295,7 +186295,7 @@
 12	ensaios	ensaio	NOUN	_	Gender=Masc|Number=Plur	10	nsubj:pass	O	_
 13	para	para	ADP	_	_	15	case	O	_
 14	cada	cada	DET	_	Gender=Masc|Number=Sing	15	det	O	_
-15	um	um	PRON	_	Gender=Masc|Number=Sing	10	obl	O	_
+15	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	10	obl	O	_
 16-17	dos	_	_	_	_	_	_	O	_
 16	de	de	ADP	_	_	18	case	O	_
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
@@ -192712,7 +192712,7 @@
 4	álcool/óleo	álcool/óleo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 5	vegetal	vegetal	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	constitui	constituir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-7	um	um	NUM	_	NumType=Card	6	obj	O	_
+7	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	6	obj	O	_
 8-9	dos	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	11	case	O	_
 9	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
@@ -194027,7 +194027,7 @@
 9	ser	ser	AUX	_	VerbForm=Inf	10	aux:pass	O	_
 10	analisadas	analisar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	8	xcomp	O	_
 11	,	,	PUNCT	_	_	10	punct	O	_
-12	uma	um	NUM	_	NumType=Card	31	nsubj	O	_
+12	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	31	nsubj	O	_
 13-14	das	_	_	_	_	_	_	O	_
 13	de	de	ADP	_	_	15	case	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -195480,7 +195480,7 @@
 1	X	X	NOUN	_	Gender=Masc|Number=Sing	2	nsubj	O	_
 2	consiste	consistir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 3	em	em	ADP	_	_	4	case	O	_
-4	uma	um	NUM	_	NumType=Card	2	obj	O	_
+4	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	2	obj	O	_
 5-6	das	_	_	_	_	_	_	O	_
 5	de	de	ADP	_	_	8	case	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
@@ -198171,7 +198171,7 @@
 # text = Como um dos objetivos deste trabalho é apresentar uma análise de sensibilidade de investimento de uma usina piloto para produção de biodiesel localizada na FUMG e consumo próprio, os cálculos como os custos de logística de produção (fretes de matérias primas, até a usina piloto, e fretes de produtos finais até os clientes não serão necessários) pois a matéria prima será obtida dos restaurantes da universidade e de postos de coleta seletiva, disponiveis no campus para que os alunos possam descartar corretamente o óleo utilizado em suas casas..
 # sent_id = 271-20150810-MONOGRAFIA_0-262
 1	Como	como	ADP	_	_	2	case	O	_
-2	um	um	NUM	_	NumType=Card	10	obl	O	_
+2	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	10	obl	O	_
 3-4	dos	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	5	case	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -208415,7 +208415,7 @@
 26	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	27	nsubj	O	_
 27	consiste	consistir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	acl:relcl	O	_
 28	em	em	ADP	_	_	29	case	O	_
-29	um	um	NUM	_	NumType=Card	27	obj	O	_
+29	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	27	obj	O	_
 30-31	dos	_	_	_	_	_	_	O	_
 30	de	de	ADP	_	_	32	case	O	_
 31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	32	det	O	_
@@ -208465,12 +208465,12 @@
 26	espectrofotometria	espectrofotometria	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 27	,	,	PUNCT	_	_	22	punct	O	_
 28	sendo	ser	AUX	_	VerbForm=Ger	29	cop	O	_
-29	um	um	NUM	_	NumType=Card	22	nsubj	O	_
+29	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	22	advcl	O	_
 30-31	dos	_	_	_	_	_	_	O	_
 30	de	de	ADP	_	_	33	case	O	_
-31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	33	det	O	_
+31	os	o	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	29	nmod	O	_
 32	mais	mais	ADV	_	_	33	advmod	O	_
-33	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	29	advcl	O	_
+33	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	31	acl	O	_
 34	dentre	dentre	ADP	_	_	36	case	O	_
 35	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	36	det	O	_
 36	análises	anális	NOUN	_	Gender=Fem|Number=Plur	33	obl	O	_
@@ -208844,7 +208844,7 @@
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	interface	interface	NOUN	_	Gender=Fem|Number=Sing	16	obj	O	_
 19	onde	onde	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	26	obl	O	_
-20	uma	um	NUM	_	NumType=Card	26	nsubj	O	_
+20	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	26	nsubj	O	_
 21-22	das	_	_	_	_	_	_	O	_
 21	de	de	ADP	_	_	23	case	O	_
 22	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	23	det	O	_
@@ -210271,7 +210271,7 @@
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
 8	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	10	nsubj	B=FLUIDO	_
 9	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	cop	O	_
-10	uma	um	NUM	_	NumType=Card	0	root	O	_
+10	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	0	root	O	_
 11-12	das	_	_	_	_	_	_	O	_
 11	de	de	ADP	_	_	13	case	O	_
 12	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	13	det	O	_
@@ -215393,7 +215393,7 @@
 15	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 16	em	em	ADP	_	_	18	case	O	_
 17	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	18	det	O	_
-18	uma	um	NUM	_	NumType=Card	11	obl	O	_
+18	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	11	obl	O	_
 19-20	delas	_	_	_	_	_	_	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	elas	ele	PRON	_	Gender=Fem|Number=Plur|Person=3|PronType=Prs	18	nmod	O	_
@@ -217055,7 +217055,7 @@
 
 # text = Uma das aplicações dos carbetos está na catálise heterogênea..
 # sent_id = 25-20150123-MONOGRAFIA_0-55
-1	Uma	um	NUM	_	NumType=Card	11	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	11	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -217624,7 +217624,7 @@
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	5	aux:pass	O	_
 5	publicado	publicar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-6	um	um	NUM	_	NumType=Card	5	nsubj:pass	O	_
+6	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	nsubj:pass	O	_
 7-8	dos	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	10	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
@@ -219888,7 +219888,7 @@
 
 # text = Uma das estruturas formadas mais conhecida é o octaedro truncado, que é conhecido como cavidade beta ou cavidade sodalíta (β-cage), gerado a partir de anéis de quatro e seis membros [9]..
 # sent_id = 25-20150123-MONOGRAFIA_0-129
-1	Uma	um	NUM	_	NumType=Card	10	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	10	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -229096,7 +229096,7 @@
 6	com	com	ADP	_	_	9	case	O	_
 7	mais	mais	ADV	_	_	9	advmod	O	_
 8	de	de	ADP	_	_	7	fixed	O	_
-9	um	um	NUM	_	NumType=Card	5	obl	O	_
+9	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	obl	O	_
 10-11	destes	_	_	_	_	_	_	O	_
 10	de	de	ADP	_	_	12	case	O	_
 11	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	12	det	O	_
@@ -236224,7 +236224,7 @@
 2	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	3	aux:pass	O	_
 3	discutido	discutir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	13	advcl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
-5	um	um	NUM	_	NumType=Card	13	nsubj	O	_
+5	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	13	nsubj	O	_
 6-7	dos	_	_	_	_	_	_	O	_
 6	de	de	ADP	_	_	9	case	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
@@ -236485,7 +236485,7 @@
 
 # text = Uma das características que chama a atenção nas serras é o fato de apresentarem seus cimos aplainados por superfícies de erosão ,as quais, se corretamente correlacionadas, servem de excelentes marcos para rastrear a movimentação tectônica diferencial sofrida pelos seus vários segmentos (Zalán e Oliveira, 2005)..
 # sent_id = 20-20140904-TESEDSC_0-150
-1	Uma	um	NUM	_	NumType=Card	14	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	14	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -238295,7 +238295,7 @@
 10	portanto	portanto	CCONJ	_	_	8	cc	O	_
 11	,	,	PUNCT	_	_	8	punct	O	_
 12	em	em	ADP	_	_	13	case	O	_
-13	um	um	NUM	_	NumType=Card	8	obj	O	_
+13	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	8	obj	O	_
 14-15	dos	_	_	_	_	_	_	O	_
 14	de	de	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -238805,7 +238805,7 @@
 26	dois	dois	NUM	_	NumType=Card	27	nummod	O	_
 27	diques	dique	NOUN	_	Gender=Masc|Number=Plur	25	obj	O	_
 28	,	,	PUNCT	_	_	29	punct	O	_
-29	um	um	NUM	_	NumType=Card	27	appos	O	_
+29	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	27	appos	O	_
 30-31	deles	_	_	_	_	_	_	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	eles	eles	PRON	_	Gender=Masc|Number=Plur|Person=3|PronType=Prs	29	nmod	O	_
@@ -241654,7 +241654,7 @@
 23	extremidade	extremidade	NOUN	_	Gender=Fem|Number=Sing	21	obj	O	_
 24	nordeste	nordeste	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	de	de	ADP	_	_	26	case	O	_
-26	uma	um	NUM	_	NumType=Card	23	nummod	O	_
+26	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	23	nmod	O	_
 27-28	das	_	_	_	_	_	_	O	_
 27	de	de	ADP	_	_	29	case	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -248530,7 +248530,7 @@
 33	opera	operar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	30	acl:relcl	O	_
 34	em	em	ADP	_	_	36	case	O	_
 35	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	36	det	O	_
-36	uma	um	NUM	_	NumType=Card	33	obj	O	_
+36	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	33	obj	O	_
 37-38	das	_	_	_	_	_	_	O	_
 37	de	de	ADP	_	_	40	case	O	_
 38	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	40	det	O	_
@@ -251436,7 +251436,7 @@
 41	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	40	expl	O	_
 42	a	a	ADP	_	_	44	case	O	_
 43	cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Tot	44	det	O	_
-44	um	um	NUM	_	NumType=Card	40	obj	O	_
+44	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	40	obj	O	_
 45-46	desses	_	_	_	_	_	_	O	_
 45	de	de	ADP	_	_	47	case	O	_
 46	esses	esse	DET	_	Gender=Masc|Number=Plur|PronType=Dem	47	det	O	_
@@ -252640,7 +252640,7 @@
 7	correspondentes	correspondente	ADJ	_	Gender=Masc|Number=Plur	4	amod	O	_
 8	a	a	ADP	_	_	10	case	O	_
 9	cada	cada	DET	_	Gender=Masc|Number=Sing	10	det	O	_
-10	um	um	NUM	_	NumType=Card	7	obl	O	_
+10	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	7	obl	O	_
 11-12	dos	_	_	_	_	_	_	O	_
 11	de	de	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -258178,7 +258178,7 @@
 18	,	,	PUNCT	_	_	11	punct	O	_
 19	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	20	expl	O	_
 20	observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-21	uma	um	NUM	_	NumType=Card	20	nsubj	O	_
+21	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	20	nsubj	O	_
 22-23	das	_	_	_	_	_	_	O	_
 22	de	de	ADP	_	_	25	case	O	_
 23	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	25	det	O	_
@@ -259284,7 +259284,7 @@
 
 # text = Uma das linhas sísmicas descritas por estes autores (a 28-RL-64) se encontra plotada no mapa de interpretação do anexo 1 e está representada na figura 6.1..
 # sent_id = 20-20140904-TESEDSC_0-711
-1	Uma	um	NUM	_	NumType=Card	15	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	15	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -261063,7 +261063,7 @@
 14	extremidade	extremidade	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
 15	NE	NE	PROPN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	um	um	NUM	_	NumType=Card	14	nummod	O	_
+17	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	14	nmod	O	_
 18-19	dos	_	_	_	_	_	_	O	_
 18	de	de	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
@@ -262878,7 +262878,7 @@
 14	diferentes	diferente	ADJ	_	Gender=Fem|Number=Plur	13	amod	O	_
 15	para	para	ADP	_	_	17	case	O	_
 16	cada	cada	DET	_	Gender=Masc|Number=Sing	17	det	O	_
-17	um	um	PRON	_	Gender=Masc|Number=Sing	12	obl	O	_
+17	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	12	obl	O	_
 18-19	dos	_	_	_	_	_	_	O	_
 18	de	de	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
@@ -266625,7 +266625,7 @@
 22	item	item	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
 23	anterior	anterior	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 24	,	,	PUNCT	_	_	19	punct	O	_
-25	um	um	NUM	_	NumType=Card	29	nsubj	O	_
+25	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	29	nsubj	O	_
 26-27	dos	_	_	_	_	_	_	O	_
 26	de	de	ADP	_	_	28	case	O	_
 27	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	28	det	O	_
@@ -273026,7 +273026,7 @@
 
 # text = Um destes alinhamentos, observado tanto no mapa de sinal analítico como no campo total, coincide com o alinhamento topográfico do Pontal do Atalaia (Arraial do Cabo) e o SW da Ilha de Cabo Frio (figura 2.5)..
 # sent_id = 20-20140904-TESEDSC_0-1049
-1	Um	um	NUM	_	NumType=Card	20	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	20	nsubj	O	_
 2-3	destes	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	4	det	O	_
@@ -276257,7 +276257,7 @@
 33	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	34	expl	O	_
 34	associa	associar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	ccomp	O	_
 35	a	a	ADP	_	_	36	case	O	_
-36	uma	um	NUM	_	NumType=Card	34	obj	O	_
+36	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	34	obj	O	_
 37-38	das	_	_	_	_	_	_	O	_
 37	de	de	ADP	_	_	39	case	O	_
 38	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	39	det	O	_
@@ -286545,7 +286545,7 @@
 19	,	,	PUNCT	_	_	22	punct	O	_
 20	sendo	ser	AUX	_	VerbForm=Ger	22	cop	O	_
 21	cada	cada	DET	_	Gender=Masc|Number=Sing	22	det	O	_
-22	um	um	NUM	_	NumType=Card	11	conj	O	_
+22	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	advcl	O	_
 23-24	deles	_	_	_	_	_	_	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	eles	eles	PRON	_	Gender=Masc|Number=Plur|Person=3|PronType=Prs	22	nmod	O	_
@@ -286887,7 +286887,7 @@
 
 # text = Um dos principais objetivos quando se utiliza um poço horizontal é aumentar o índice de produtividade do mesmo, quando comparado ao de um poço vertical (Rosa et al., 2006)..
 # sent_id = 119-20141209-TESEMSC_0-16
-1	Um	um	NUM	_	NumType=Card	13	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	13	nsubj	O	_
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	5	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -288922,9 +288922,9 @@
 17	,	,	PUNCT	_	_	18	punct	O	_
 18	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	8	appos	O	_
 19	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	21	nsubj	O	_
-20	o	o	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	21	nsubj	O	_
+20	o	o	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	21	obj	O	_
 21	torna	tornar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	acl:relcl	O	_
-22	um	um	NUM	_	NumType=Card	21	obj	O	_
+22	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	21	xcomp	O	_
 23-24	dos	_	_	_	_	_	_	O	_
 23	de	de	ADP	_	_	25	case	O	_
 24	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	25	det	O	_
@@ -294539,7 +294539,7 @@
 
 # text = Um dos fatos mais marcantes na reologia é que, sob cisalhamento, a grande maioria das soluções poliméricas exibe toda ou parte da curva de fluxo conforme a Figura 2.6..
 # sent_id = 119-20141209-TESEMSC_0-242
-1	Um	um	NUM	_	NumType=Card	10	nsubj	O	_
+1	Um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	10	nsubj	O	_
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
@@ -294976,11 +294976,11 @@
 6	de	de	ADP	_	_	8	case	O	_
 7	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	8	det	O	_
 8	polímero	polímero	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
-9	um	um	NUM	_	NumType=Card	8	nummod	O	_
+9	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	xcomp	O	_
 10-11	dos	_	_	_	_	_	_	O	_
 10	de	de	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
-12	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
+12	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	9	nmod	O	_
 13	de	de	ADP	_	_	15	case	O	_
 14	grande	grande	ADJ	_	Gender=Fem|Number=Sing	15	amod	O	_
 15	aplicação	aplicação	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
@@ -302907,7 +302907,7 @@
 19	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	20	aux:pass	O	_
 20	injetado	injetar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 21	sob	sob	ADP	_	_	22	case	O	_
-22	um	um	NUM	_	NumType=Card	20	obl	O	_
+22	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	20	obl	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	diferencial	diferencial	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
 25	de	de	ADP	_	_	26	case	O	_
@@ -308835,7 +308835,7 @@
 12	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
 13	béquer	béquer	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 14	com	com	ADP	_	_	15	case	O	_
-15	um	um	NUM	_	NumType=Card	13	nummod	O	_
+15	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	13	nmod	O	_
 16-17	dos	_	_	_	_	_	_	O	_
 16	de	de	ADP	_	_	18	case	O	_
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
@@ -311762,8 +311762,8 @@
 23	de	de	ADP	_	_	24	case	O	_
 24	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	21	nmod	B=FLUIDO	_
 25	para	para	ADP	_	_	26	case	O	_
-26	cada	cada	DET	_	Gender=Masc|Number=Sing	21	nmod	O	_
-27	um	um	NUM	_	NumType=Card	26	nummod	O	_
+26	cada	cada	DET	_	Gender=Masc|Number=Sing	27	det	O	_
+27	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	obl	O	_
 28-29	dos	_	_	_	_	_	_	O	_
 28	de	de	ADP	_	_	30	case	O	_
 29	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	30	det	O	_
@@ -315497,14 +315497,14 @@
 # sent_id = 240-20140220-MONOGRAFIA_0-4
 1	Sendo	ser	AUX	_	VerbForm=Ger	3	cop	O	_
 2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
-3	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	25	nsubj	B=FLUIDO	_
+3	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	6	nsubj	B=FLUIDO	_
 4	de	de	ADP	_	_	5	case	I=FLUIDO	_
 5	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	3	nmod	B=LITOLOGIA|I=FLUIDO	_
-6	um	um	NUM	_	NumType=Card	5	nummod	O	_
+6	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	25	advcl	O	_
 7-8	dos	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	9	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
-9	fatores	fator	NOUN	_	Gender=Masc|Number=Plur	3	nmod	O	_
+9	fatores	fator	NOUN	_	Gender=Masc|Number=Plur	6	nmod	O	_
 10	de	de	ADP	_	_	12	case	O	_
 11	grande	grande	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 12	importância	importância	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
@@ -315947,7 +315947,7 @@
 4	de	de	ADP	_	_	5	case	O	_
 5	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	3	nmod	B=LITOLOGIA	_
 6	,	,	PUNCT	_	_	3	punct	O	_
-7	uma	um	NUM	_	NumType=Card	20	nsubj	O	_
+7	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	3	appos	O	_
 8-9	das	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	10	case	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
@@ -323350,7 +323350,7 @@
 
 # text = Uma das propriedades fundamentais dos tensoativos é a propriedade de adsorção nas interfaces (limite entre duas fases imiscíveis) ou superfícies de um dado sistema..
 # sent_id = 240-20140220-MONOGRAFIA_0-252
-1	Uma	um	NUM	_	NumType=Card	11	nsubj	O	_
+1	Uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	11	nsubj	O	_
 2-3	das	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	4	case	O	_
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
@@ -323815,13 +323815,13 @@
 18	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	14	conj	O	_
 19	,	,	PUNCT	_	_	25	punct	O	_
 20	cada	cada	DET	_	Gender=Masc|Number=Sing	25	nsubj	O	_
-21	um	um	NUM	_	NumType=Card	25	nsubj	O	_
+21	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	25	nsubj	O	_
 22-23	destes	_	_	_	_	_	_	O	_
 22	de	de	ADP	_	_	24	case	O	_
 23	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	24	det	O	_
 24	componentes	componente	NOUN	_	Gender=Masc|Number=Plur	21	nmod	O	_
 25	assumem	assumir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
-26	um	um	NUM	_	NumType=Card	25	obj	O	_
+26	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	25	obj	O	_
 27-28	dos	_	_	_	_	_	_	O	_
 27	de	de	ADP	_	_	29	case	O	_
 28	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	29	det	O	_
@@ -324030,7 +324030,7 @@
 16	triangulares	triangular	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	em	em	ADP	_	_	18	case	O	_
 18	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	23	obl	O	_
-19	um	um	NUM	_	NumType=Card	23	nsubj	O	_
+19	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	23	nsubj	O	_
 20-21	dos	_	_	_	_	_	_	O	_
 20	de	de	ADP	_	_	22	case	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	22	det	O	_
@@ -328378,7 +328378,7 @@
 22	Waale	Waale	PROPN	_	Number=Sing	20	flat:name	O	_
 23	para	para	ADP	_	_	25	case	O	_
 24	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	25	det	O	_
-25	uma	um	NUM	_	NumType=Card	3	obl	O	_
+25	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	3	obl	O	_
 26-27	destas	_	_	_	_	_	_	O	_
 26	de	de	ADP	_	_	28	case	O	_
 27	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	28	det	O	_
@@ -329878,7 +329878,7 @@
 80	)	)	PUNCT	_	_	71	punct	O	_
 81	,	,	PUNCT	_	_	82	punct	O	_
 82	possuindo	possuir	VERB	_	VerbForm=Ger	52	advcl	O	_
-83	uma	um	NUM	_	NumType=Card	82	obj	O	_
+83	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	82	obj	O	_
 84-85	das	_	_	_	_	_	_	O	_
 84	de	de	ADP	_	_	87	case	O	_
 85	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	87	det	O	_
@@ -336726,7 +336726,7 @@
 11-12	encontra-se	_	_	_	_	_	_	O	_
 11	encontra	encontrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	11	expl	O	_
-13	um	um	NUM	_	NumType=Card	11	nsubj	O	_
+13	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	11	nsubj	O	_
 14-15	dos	_	_	_	_	_	_	O	_
 14	de	de	ADP	_	_	17	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	17	det	O	_
@@ -342557,7 +342557,7 @@
 10	apoio	apoio	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 11	,	,	PUNCT	_	_	13	punct	O	_
 12	sendo	ser	AUX	_	VerbForm=Ger	13	cop	O	_
-13	uma	um	NUM	_	NumType=Card	7	nsubj	O	_
+13	uma	um	PRON	_	Gender=Fem|Number=Sing|PronType=Ind	7	advcl	O	_
 14-15	da	_	_	_	_	_	_	O	_
 14	de	de	ADP	_	_	16	case	O	_
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
@@ -342915,7 +342915,7 @@
 66	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	67	det	O	_
 67	garrafas	garrafa	NOUN	_	Gender=Fem|Number=Plur	68	nsubj	O	_
 68	tornaram	tornar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	55	conj	O	_
-69	um	um	NUM	_	NumType=Card	68	obj	O	_
+69	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	68	obj	O	_
 70-71	dos	_	_	_	_	_	_	O	_
 70	de	de	ADP	_	_	73	case	O	_
 71	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	73	det	O	_
@@ -347395,7 +347395,7 @@
 # sent_id = 243-20141015-MONOGRAFIA_1-82
 1	Flotação	flotação	NOUN	_	Gender=Fem|Number=Sing	3	nsubj	O	_
 2	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	cop	O	_
-3	um	um	NUM	_	NumType=Card	0	root	O	_
+3	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	0	root	O	_
 4-5	dos	_	_	_	_	_	_	O	_
 4	de	de	ADP	_	_	6	case	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
