commit 2d35600b53011449d91bafca81793aa513f607f7
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Nov 23 12:50:19 2021 -0300

    lemma de caracteriza-se

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 2995b56..3cbe0c6 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -5614,7 +5614,7 @@
 13	de	de	ADP	_	_	12	flat:name	O	_
 14	Florianópolis	Florianópolis	PROPN	_	Number=Sing	12	flat:name	O	_
 15-16	caracteriza-se	_	_	_	_	_	_	_	_
-15	caracteziza	caractezizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+15	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	15	expl	O	_
 17	por	por	ADP	_	_	18	mark	O	_
 18	apresentar	apresentar	VERB	_	VerbForm=Inf	15	advcl	O	_
@@ -20413,7 +20413,7 @@
 28	pré-rifte	pré-rifte	ADJ	_	Gender=Masc|Number=Sing	27	amod	O	_
 29	,	,	PUNCT	_	_	30	punct	O	_
 30-31	caracteriza-se	_	_	_	_	_	_	_	_
-30	caracteziza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	parataxis	O	_
+30	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	parataxis	O	_
 31	se	se	PRON	_	_	30	expl	O	_
 32	por	por	ADP	_	_	33	case	O	_
 33	subsidência	subsidência	NOUN	_	Gender=Fem|Number=Sing	30	obl	O	_
@@ -20934,7 +20934,7 @@
 12	Santo	Santo	PROPN	_	Number=Sing	10	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 13	,	,	PUNCT	_	_	7	punct	O	_
 14-15	caracteriza-se	_	_	_	_	_	_	_	_
-14	caracteziza	caractezizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+14	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	14	expl	O	_
 16	por	por	ADP	_	_	17	case	O	_
 17	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	14	obl	B=LITOLOGIA	_
@@ -76278,7 +76278,7 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	SIG	SIG	PROPN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 3-4	caracteriza-se	_	_	_	_	_	_	_	_
-3	caracteziza	caractezizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+3	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	3	expl	O	_
 5	por	por	ADP	_	_	6	mark	O	_
 6	permitir	permitir	VERB	_	VerbForm=Inf	3	xcomp	O	_
@@ -127033,7 +127033,7 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 3-4	caracteriza-se	_	_	_	_	_	_	_	_
-3	caracteziza	caractezizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+3	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	3	expl	O	_
 5	por	por	SCONJ	_	_	8	mark	O	_
 6	ser	ser	AUX	_	VerbForm=Inf	8	cop	O	_
@@ -256348,7 +256348,7 @@
 9	meio	meio	NOUN	_	Gender=Masc|Number=Sing	5	nmod	O	_
 10	poroso	poroso	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
 11-12	caracteriza-se	_	_	_	_	_	_	_	_
-11	caracteziza	caractezizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	advcl	O	_
+11	caracteriza	caracterizar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	advcl	O	_
 12	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	11	expl	O	_
 13	por	por	ADP	_	_	14	case	O	_
 14	velocidades	velocidade	NOUN	_	Gender=Fem|Number=Plur	11	obl	O	_
