commit 14edfcfbf766a231a0dc31f58454bbff6069484c
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Sat Oct 23 03:02:49 2021 -0300

    Lema de superior

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 977239b..ee3a550 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -80,7 +80,7 @@
 27	de	de	ADP	_	_	26	flat:name	I=BACIA	_
 28	Pelotas	Pelotas	PROPN	_	Number=Sing	26	flat:name	I=BACIA	_
 29	verificando	verificar	VERB	_	VerbForm=Ger	5	advcl	O	_
-30	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	31	expl	O	_
+30	se	se	SCONJ	_	_	31	mark	O	_
 31	há	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	ccomp	O	_
 32	correspondência	correspondência	NOUN	_	Gender=Fem|Number=Sing	31	obj	O	_
 33	entre	entre	ADP	_	_	35	case	O	_
@@ -94,7 +94,7 @@
 40	porção	porção	NOUN	_	Gender=Fem|Number=Sing	37	obl	O	_
 41	continental	continental	ADJ	_	Gender=Fem|Number=Sing	40	amod	O	_
 42	e	e	CCONJ	_	_	43	cc	O	_
-43	offshore	offshore	NOUN	_	Gender=Fem|Number=Sing	41	conj	O	_
+43	offshore	offshore	ADJ	_	Gender=Fem|Number=Sing	41	conj	O	_
 44	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = A caracterização estrutural para a porção norte da Bacia de Pelotas foi realizada através de 17 seções sísmicas, onde foram caracterizadas falhas normais, em padrão dominó, falhas sintéticas e antitéticas e lístricas, além da identificação de estruturas como o Arco de Torres (Fonseca 2006), (Alves 1981) a Plataforma de Florianópolis individualizada por Gonçalves et al. (1979), Zona de Falha do Rio Grande (Miranda 1970), (Kowsmann 1974).
@@ -8190,7 +8190,7 @@
 7	de	de	ADP	_	_	8	case	O	_
 8	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
 9	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
-10	superior	alto	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
+10	superior	superior	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
 11	a	a	ADP	_	_	13	case	O	_
 12	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	13	det	O	_
 13	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	10	obl	O	_
@@ -17634,7 +17634,7 @@
 35	em	em	ADP	_	_	37	case	O	_
 36	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	37	det	O	_
 37	Jurássico	Jurássico	PROPN	_	Gender=Masc|Number=Sing	14	obl	B=GEOCRONOLOGIA	_
-38	superior	alto	ADJ	_	Gender=Masc|Number=Sing	37	amod	I=GEOCRONOLOGIA	_
+38	superior	superior	ADJ	_	Gender=Masc|Number=Sing	37	amod	I=GEOCRONOLOGIA	_
 39	/	/	PUNCT	_	_	40	punct	O	_
 40	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	37	nmod	B=GEOCRONOLOGIA	_
 41	inferior	inferior	ADJ	_	Gender=Masc|Number=Sing	37	amod	I=GEOCRONOLOGIA	_
@@ -20191,7 +20191,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-91
 1	Seu	seu	DET	_	Gender=Masc|Number=Sing|PronType=Prs	2	det	O	_
 2	limite	limite	NOUN	_	Gender=Masc|Number=Sing	5	nsubj:pass	O	_
-3	superior	alto	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	definido	definir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 6-7	pela	_	_	_	_	_	_	_	_
@@ -21735,7 +21735,7 @@
 20	Romualdo	Romualdo	PROPN	_	Number=Sing	19	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 21	(	(	PUNCT	_	_	22	punct	O	_
 22	porção	porção	NOUN	_	Gender=Fem|Number=Sing	19	appos	O	_
-23	superior	alto	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
+23	superior	superior	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
 24	)	)	PUNCT	_	_	22	punct	O	_
 25	(	(	PUNCT	_	_	26	punct	O	_
 26	Assine	Assine	PROPN	_	Gender=Masc|Number=Sing	5	parataxis	O	_
@@ -22016,7 +22016,7 @@
 12	como	como	ADP	_	_	13	case	O	_
 13	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	5	xcomp	B=UNIDADE_LITOESTRATIGRÁFICA	_
 14	Romualdo	Romualdo	PROPN	_	Number=Sing	13	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
-15	superior	alto	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
+15	superior	superior	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	pertencente	pertencente	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 18-19	à	_	_	_	_	_	_	_	_
@@ -24838,7 +24838,7 @@
 11	regime	regime	NOUN	_	Gender=Masc|Number=Sing	16	nsubj:pass	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
-14	superior	alto	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
+14	superior	superior	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 15	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	aux:pass	O	_
 16	alcançado	alcançar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	4	ccomp	O	_
 17	,	,	PUNCT	_	_	16	punct	O	_
@@ -24901,11 +24901,11 @@
 19	regime	regime	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	19	nmod	O	_
-22	superior	alto	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
+22	superior	superior	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
 23	,	,	PUNCT	_	_	2	punct	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
 25	porção	porção	NOUN	_	Gender=Fem|Number=Sing	32	nsubj:pass	O	_
-26	superior	alto	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
+26	superior	superior	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
 27-28	das	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -26199,7 +26199,7 @@
 35	topo	topo	NOUN	_	Gender=Masc|Number=Sing	23	conj	O	_
 36	(	(	PUNCT	_	_	37	punct	O	_
 37	limite	limite	NOUN	_	Gender=Masc|Number=Sing	35	appos	O	_
-38	superior	alto	ADJ	_	Gender=Masc|Number=Sing	37	amod	O	_
+38	superior	superior	ADJ	_	Gender=Masc|Number=Sing	37	amod	O	_
 39-40	da	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
@@ -26302,7 +26302,7 @@
 84	de	de	ADP	_	_	85	case	O	_
 85	sequência	sequência	NOUN	_	Gender=Fem|Number=Sing	83	nmod	O	_
 86	deposicional	deposicional	ADJ	_	Gender=Fem|Number=Sing	85	amod	O	_
-87	superior	alto	ADJ	_	Gender=Fem|Number=Sing	85	amod	O	_
+87	superior	superior	ADJ	_	Gender=Fem|Number=Sing	85	amod	O	_
 88	(	(	PUNCT	_	_	89	punct	O	_
 89	LSDs	LSDs	PROPN	_	Gender=Fem|Number=Sing	85	appos	O	_
 90	)	)	PUNCT	_	_	89	punct	O	_
@@ -26405,7 +26405,7 @@
 61	em	em	ADP	_	_	63	case	O	_
 62	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	63	det	O	_
 63	seção	seção	NOUN	_	Gender=Fem|Number=Sing	58	nmod	O	_
-64	superior	alto	ADJ	_	Gender=Fem|Number=Sing	63	amod	O	_
+64	superior	superior	ADJ	_	Gender=Fem|Number=Sing	63	amod	O	_
 65	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = O TST é de caráter transgressivo, no qual fácies de sistemas fluviais são sucedidas por fácies costeiras que culminam em depósitos marinhos rasos.
@@ -26864,7 +26864,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-312
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	limite	limite	NOUN	_	Gender=Masc|Number=Sing	11	nsubj	O	_
-3	superior	alto	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4-5	da	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -28279,7 +28279,7 @@
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
 6	parte	parte	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 7	mais	mais	ADV	_	_	8	advmod	O	_
-8	superior	alto	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
+8	superior	superior	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
 9	/	/	PUNCT	_	_	10	punct	O	_
 10	final	final	ADJ	_	Gender=Fem|Number=Sing	8	conj	O	_
 11-12	do	_	_	_	_	_	_	_	_
@@ -28441,7 +28441,7 @@
 5	em	em	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	porção	porção	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
-8	superior	alto	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
+8	superior	superior	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -28521,7 +28521,7 @@
 37	em	em	ADP	_	_	39	case	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	parte	parte	NOUN	_	Gender=Fem|Number=Sing	34	obl	O	_
-40	superior	alto	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
+40	superior	superior	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
 41-42	da	_	_	_	_	_	_	_	_
 41	de	de	ADP	_	_	43	case	O	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
@@ -29473,7 +29473,7 @@
 21	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	9	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
 22	Romualdo	Romualdo	PROPN	_	Number=Sing	21	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 23	(	(	PUNCT	_	_	24	punct	O	_
-24	superior	alto	ADJ	_	Gender=Masc|Number=Sing	21	nmod:appos	O	_
+24	superior	superior	ADJ	_	Gender=Masc|Number=Sing	21	nmod:appos	O	_
 25	)	)	PUNCT	_	_	24	punct	O	_
 26	sendo	ser	AUX	_	VerbForm=Ger	27	aux:pass	O	_
 27	marcada	marcar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	9	advcl	O	_
@@ -30107,7 +30107,7 @@
 20	em	em	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	intervalo	intervalo	NOUN	_	Gender=Masc|Number=Sing	19	obl	O	_
-23	superior	alto	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
+23	superior	superior	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 24-25	do	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
@@ -43299,7 +43299,7 @@
 5	aquela	aquele	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	0	root	O	_
 6	com	com	ADP	_	_	7	case	O	_
 7	salinidade	salinidade	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
-8	superior	alto	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
+8	superior	superior	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	a	a	ADP	_	_	11	case	O	_
 10	1.000	1.000	NUM	_	NumType=Card	11	nummod	O	_
 11	ppm	ppm	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
@@ -60989,11 +60989,11 @@
 
 # text = Como por exemplo, se houve uma possível ingressão marinha, atestada por radiolaritos descritos por Kattah (1991).
 # sent_id = 283-20150917-MONOGRAFIA_0-63
-1	Como	como	ADP	_	_	2	case	O	_
-2	por	por	ADP	_	_	6	obl	O	_
+1	Como	como	ADV	_	_	6	advmod	O	_
+2	por	por	ADP	_	_	6	obl	O	MWEPOS=ADV
 3	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	2	fixed	O	_
 4	,	,	PUNCT	_	_	2	punct	O	_
-5	se	se	PRON	_	Case=Acc|Gender=Unsp|Person=3|PronType=Prs	6	expl	O	_
+5	se	se	SCONJ	_	_	6	mark	O	_
 6	houve	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 7	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
 8	possível	possível	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
@@ -63061,7 +63061,7 @@
 # sent_id = 283-20150917-MONOGRAFIA_0-146
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	porção	porção	NOUN	_	Gender=Fem|Number=Sing	5	nsubj:pass	O	_
-3	superior	alto	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	composta	compor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 6	por	por	ADP	_	_	7	case	O	_
@@ -63108,7 +63108,7 @@
 20	observar	observar	VERB	_	VerbForm=Inf	7	advcl	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	contato	contato	NOUN	_	Gender=Masc|Number=Sing	20	obj	O	_
-23	superior	alto	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
+23	superior	superior	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 24	entre	entre	ADP	_	_	26	case	O	_
 25	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	26	det	O	_
 26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
@@ -63139,7 +63139,7 @@
 14	associação	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	O	_
-17	superior	alto	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
+17	superior	superior	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 18-19	desta	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	20	det	O	_
@@ -63177,7 +63177,7 @@
 8	para	para	ADP	_	_	10	case	O	_
 9	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	10	det	O	_
 10	associação	associação	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
-11	superior	alto	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
+11	superior	superior	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	17	case	O	_
 13	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	12	fixed	O	_
@@ -63728,7 +63728,7 @@
 # sent_id = 283-20150917-MONOGRAFIA_0-177
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	porção	porção	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
-3	superior	alto	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4-5	da	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -64185,7 +64185,7 @@
 3	para	para	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
 5	porção	porção	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
-6	superior	alto	ADJ	_	Gender=Fem|Number=Sing	5	amod	O	_
+6	superior	superior	ADJ	_	Gender=Fem|Number=Sing	5	amod	O	_
 7-8	da	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -65221,7 +65221,7 @@
 9	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	contato	contato	NOUN	_	Gender=Masc|Number=Sing	9	nsubj:pass	O	_
-12	superior	alto	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
+12	superior	superior	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 13-14	da	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
@@ -66726,7 +66726,7 @@
 4	Barras	Barras	PROPN	_	Number=Sing	2	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 5	,	,	PUNCT	_	_	6	punct	O	_
 6	porção	porção	NOUN	_	Gender=Fem|Number=Sing	2	appos	O	_
-7	superior	alto	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
+7	superior	superior	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
@@ -67056,7 +67056,7 @@
 14	inferior	inferior	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
 15	e	e	CCONJ	_	_	17	cc	O	_
 16	outra	outro	DET	_	Gender=Fem|Number=Sing|PronType=Ind	17	det	O	_
-17	superior	alto	ADJ	_	Gender=Fem|Number=Sing	14	conj	O	_
+17	superior	superior	ADJ	_	Gender=Fem|Number=Sing	14	conj	O	_
 18	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = A inferior compreende arenitos de granulometria fina a grossa, seleção moderada, com cimento carbonático e com estratificações plano-paralela, cruzada acanalada ou sem estruturas aparentes.
@@ -67096,7 +67096,7 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	associação	associação	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
-4	superior	alto	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
+4	superior	superior	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
 6	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	9	nsubj:pass	O	_
 7	não	não	ADV	_	Polarity=Neg	9	advmod	O	_
@@ -67129,7 +67129,7 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	3	det	O	_
 3	porção	porção	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
-4	superior	alto	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
+4	superior	superior	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 5	também	também	ADV	_	_	7	advmod	O	_
 6	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	O	_
 7	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -68117,7 +68117,7 @@
 45	regime	regime	NOUN	_	Gender=Masc|Number=Sing	39	obl	O	_
 46	de	de	ADP	_	_	47	case	O	_
 47	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	45	nmod	O	_
-48	superior	alto	ADJ	_	Gender=Masc|Number=Sing	47	amod	O	_
+48	superior	superior	ADJ	_	Gender=Masc|Number=Sing	47	amod	O	_
 49	(	(	PUNCT	_	_	50	punct	O	_
 50	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	47	appos	O	_
 51	)	)	PUNCT	_	_	50	punct	O	_
@@ -91606,7 +91606,7 @@
 30	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
 31	para	para	SCONJ	_	_	32	mark	O	_
 32	verificar	verificar	VERB	_	VerbForm=Inf	10	advcl	O	_
-33	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	34	expl	O	_
+33	se	se	SCONJ	_	_	34	mark	O	_
 34	há	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	32	ccomp	O	_
 35	vazamentos	vazamento	NOUN	_	Gender=Masc|Number=Plur	34	obj	O	_
 36	de	de	ADP	_	_	37	case	O	_
@@ -103130,7 +103130,7 @@
 1	Tal	tal	DET	_	Gender=Fem|Number=Sing|PronType=Ind	2	det	O	_
 2	concentração	concentração	NOUN	_	Gender=Fem|Number=Sing	4	nsubj	O	_
 3	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
-4	superior	alto	ADJ	_	Gender=Fem|Number=Sing	0	root	O	_
+4	superior	superior	ADJ	_	Gender=Fem|Number=Sing	0	root	O	_
 5-6	à	_	_	_	_	_	_	_	_
 5	a	a	ADP	_	_	6	case	O	_
 6	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	4	obl	O	_
@@ -113669,7 +113669,7 @@
 2	de	de	ADP	_	_	1	fixed	O	_
 3	ter	ter	VERB	_	VerbForm=Inf	19	advcl	O	_
 4	custo	custo	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
-5	superior	alto	ADJ	_	Gender=Masc|Number=Sing	4	amod	O	_
+5	superior	superior	ADJ	_	Gender=Masc|Number=Sing	4	amod	O	_
 6	em	em	ADP	_	_	8	case	O	_
 7	50	50	NUM	_	NumType=Card	8	nummod	O	_
 8	%	%	SYM	_	_	3	obl	O	_
@@ -114754,7 +114754,7 @@
 41	e	e	CCONJ	_	_	42	cc	O	_
 42	igual	igual	ADJ	_	Gender=Masc|Number=Sing	40	conj	O	_
 43	ou	ou	CCONJ	_	_	44	cc	O	_
-44	superior	alto	ADJ	_	Gender=Masc|Number=Sing	40	conj	O	_
+44	superior	superior	ADJ	_	Gender=Masc|Number=Sing	40	conj	O	_
 45	a	a	ADP	_	_	47	case	O	_
 46	75	75	NUM	_	NumType=Card	47	nummod	O	_
 47	%	%	SYM	_	_	44	obl	O	_
@@ -117690,7 +117690,7 @@
 22	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	cop	O	_
 23	igual	igual	ADJ	_	Gender=Masc|Number=Sing	15	acl:relcl	O	_
 24	ou	ou	CCONJ	_	_	25	cc	O	_
-25	superior	alto	ADJ	_	Gender=Masc|Number=Sing	23	conj	O	_
+25	superior	superior	ADJ	_	Gender=Masc|Number=Sing	23	conj	O	_
 26	a	a	ADP	_	_	27	case	O	_
 27	20	20	NUM	_	NumType=Card	25	obl	O	_
 28	.	.	PUNCT	_	_	5	punct	O	_
@@ -122711,7 +122711,7 @@
 10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	forma	forma	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
-13	superior	alto	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
+13	superior	superior	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 14-15	aos	_	_	_	_	_	_	_	_
 14	a	a	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -124563,7 +124563,7 @@
 25	para	para	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	fase	fase	NOUN	_	Gender=Fem|Number=Sing	24	obl	O	_
-28	superior	alto	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
+28	superior	superior	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
 29	,	,	PUNCT	_	_	30	punct	O	_
 30	impulsionando	impulsionar	VERB	_	VerbForm=Ger	24	advcl	O	_
 31	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	32	det	O	_
@@ -128471,7 +128471,7 @@
 18	em	em	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 20	parte	parte	NOUN	_	Gender=Fem|Number=Sing	13	obl	O	_
-21	superior	alto	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
+21	superior	superior	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22-23	da	_	_	_	_	_	_	_	_
 22	de	de	ADP	_	_	24	case	O	_
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
@@ -128524,7 +128524,7 @@
 24	em	em	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	parte	parte	NOUN	_	Gender=Fem|Number=Sing	23	obl	O	_
-27	superior	alto	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
+27	superior	superior	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 28	.	.	PUNCT	_	_	8	punct	O	_
 
 # text = A repartida de linha precisa ser realizada sem a ocorrência de picos de pressão e de forma a remover o plugue de óleo acumulado durante o período em que o sistema ficou em repouso.
@@ -133841,7 +133841,7 @@
 8	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	9	det	O	_
 9	atividade	atividade	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 10	superficial	superficial	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
-11	superior	alto	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
+11	superior	superior	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
 12	,	,	PUNCT	_	_	13	punct	O	_
 13	criando	criar	VERB	_	VerbForm=Ger	3	advcl	O	_
 14	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -135697,7 +135697,7 @@
 37	em	em	ADP	_	_	39	case	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	parte	parte	NOUN	_	Gender=Fem|Number=Sing	36	obl	O	_
-40	superior	alto	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
+40	superior	superior	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
 41-42	da	_	_	_	_	_	_	_	_
 41	de	de	ADP	_	_	43	case	O	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
@@ -136045,7 +136045,7 @@
 24	em	em	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	parte	parte	NOUN	_	Gender=Fem|Number=Sing	23	obl	O	_
-27	superior	alto	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
+27	superior	superior	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
@@ -136430,7 +136430,7 @@
 37	em	em	ADP	_	_	39	case	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	parte	parte	NOUN	_	Gender=Fem|Number=Sing	36	obl	O	_
-40	superior	alto	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
+40	superior	superior	ADJ	_	Gender=Fem|Number=Sing	39	amod	O	_
 41-42	da	_	_	_	_	_	_	_	_
 41	de	de	ADP	_	_	43	case	O	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
@@ -136942,7 +136942,7 @@
 24	em	em	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	parte	parte	NOUN	_	Gender=Fem|Number=Sing	23	obl	O	_
-27	superior	alto	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
+27	superior	superior	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
@@ -137665,7 +137665,7 @@
 9	de	de	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	parte	parte	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
-12	superior	alto	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
+12	superior	superior	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -139273,7 +139273,7 @@
 9	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	10	det	O	_
 10	atividade	atividade	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 11	superficial	superficial	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
-12	superior	alto	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
+12	superior	superior	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
 14	criando	criar	VERB	_	VerbForm=Ger	3	advcl	O	_
 15	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -150298,7 +150298,7 @@
 # sent_id = 271-20150810-MONOGRAFIA_0-47
 1	Uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	fase	fase	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
-3	superior	alto	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	composta	compor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	2	acl	O	_
 5-6	pelos	_	_	_	_	_	_	_	_
 5	por	por	ADP	_	_	7	case	O	_
@@ -153497,7 +153497,7 @@
 9	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 11	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	12	cop	O	_
-12	superior	alto	ADJ	_	Gender=Masc|Number=Sing	29	advcl	O	_
+12	superior	superior	ADJ	_	Gender=Masc|Number=Sing	29	advcl	O	_
 13-14	ao	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -153524,7 +153524,7 @@
 34	que	que	SCONJ	_	_	35	mark	O	_
 35	apresente	apresentar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	31	advcl	O	_
 36	TIR	TIR	PROPN	_	Gender=Fem|Number=Sing	35	obj	O	_
-37	superior	alto	ADJ	_	Gender=Fem|Number=Sing	36	amod	O	_
+37	superior	superior	ADJ	_	Gender=Fem|Number=Sing	36	amod	O	_
 38-39	ao	_	_	_	_	_	_	_	_
 38	a	a	ADP	_	_	40	case	O	_
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
@@ -155299,7 +155299,7 @@
 8	,	,	PUNCT	_	_	17	punct	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	camada	camada	NOUN	_	Gender=Fem|Number=Sing	17	nsubj:pass	O	_
-11	superior	alto	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
+11	superior	superior	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 12	referente	referente	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 13-14	ao	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
@@ -158410,7 +158410,7 @@
 9	sem	sem	SCONJ	_	_	10	mark	O	_
 10	calcinar	calcinar	VERB	_	VerbForm=Inf	4	acl	O	_
 11	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	cop	O	_
-12	superior	alto	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
+12	superior	superior	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
 13	que	que	ADP	_	_	16	case	O	_
 14-15	da	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
@@ -159077,7 +159077,7 @@
 5	de	de	ADP	_	_	7	case	O	_
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
-8	superior	alto	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
+8	superior	superior	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
 9	a	a	ADP	_	_	11	case	O	_
 10	30	30	NUM	_	NumType=Card	11	nummod	O	_
 11	minutos	minuto	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
@@ -162977,7 +162977,7 @@
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	período	período	NOUN	_	Gender=Masc|Number=Sing	24	nsubj	O	_
 23	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	24	cop	O	_
-24	superior	alto	ADJ	_	Gender=Masc|Number=Sing	15	advcl	O	_
+24	superior	superior	ADJ	_	Gender=Masc|Number=Sing	15	advcl	O	_
 25	a	a	ADP	_	_	27	case	O	_
 26	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	27	det	O	_
 27	estabelecido	estabelecer	NOUN	_	Gender=Masc|Number=Sing	24	obl	O	_
@@ -164488,7 +164488,7 @@
 42	soja	soja	NOUN	_	Gender=Fem|Number=Sing	40	nmod	O	_
 43	comercial	comercial	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 44	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	cop	O	_
-45	superior	alto	ADJ	_	Gender=Masc|Number=Sing	21	conj	O	_
+45	superior	superior	ADJ	_	Gender=Masc|Number=Sing	21	conj	O	_
 46-47	ao	_	_	_	_	_	_	_	_
 46	a	a	ADP	_	_	48	case	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
@@ -175516,7 +175516,7 @@
 17	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	8	conj	O	_
 19	pouco	pouco	ADV	_	_	20	advmod	O	_
-20	superior	alto	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
+20	superior	superior	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 21-22	à	_	_	_	_	_	_	_	_
 21	a	a	ADP	_	_	22	case	O	_
 22	a	o	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	20	obl	O	_
@@ -179266,7 +179266,7 @@
 20	unindo	unir	VERB	_	VerbForm=Ger	4	advcl	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
 22	parte	parte	NOUN	_	Gender=Fem|Number=Sing	20	obj	O	_
-23	superior	alto	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
+23	superior	superior	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
 24-25	à	_	_	_	_	_	_	_	_
 24	a	a	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
@@ -179355,7 +179355,7 @@
 # sent_id = 25-20150123-MONOGRAFIA_0-237
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	parte	parte	NOUN	_	Gender=Fem|Number=Sing	5	nsubj:pass	O	_
-3	superior	alto	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	formada	formar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 6	por	por	ADP	_	_	8	case	O	_
@@ -180393,7 +180393,7 @@
 15	sinais	sinal	NOUN	_	Gender=Masc|Number=Plur	14	obj	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	m/e	m/e	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
-18	superior	alto	ADJ	_	Gender=Masc|Number=Sing	17	amod	O	_
+18	superior	superior	ADJ	_	Gender=Masc|Number=Sing	17	amod	O	_
 19	a	a	ADP	_	_	20	case	O	_
 20	cinco	cinco	NUM	_	NumType=Card	18	obl	O	_
 21	(	(	PUNCT	_	_	22	punct	O	_
@@ -181168,7 +181168,7 @@
 25	de	de	ADP	_	_	26	case	O	_
 26	dessorção	dessorção	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 27	(	(	PUNCT	_	_	28	punct	O	_
-28	superior	alto	ADJ	_	Gender=Fem|Number=Sing	24	amod	O	_
+28	superior	superior	ADJ	_	Gender=Fem|Number=Sing	24	amod	O	_
 29	)	)	PUNCT	_	_	28	punct	O	_
 30	,	,	PUNCT	_	_	35	punct	O	_
 31	isto	isto	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	35	cc	O	_
@@ -185447,7 +185447,7 @@
 22	apresentou	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 23	atividade	atividade	NOUN	_	Gender=Fem|Number=Sing	22	obj	O	_
 24	bem	bem	ADV	_	_	25	advmod	O	_
-25	superior	alto	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
+25	superior	superior	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 26	(	(	PUNCT	_	_	28	punct	O	_
 27	12	12	NUM	_	NumType=Card	28	nummod	O	_
 28	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	23	appos	O	_
@@ -189801,7 +189801,7 @@
 # sent_id = 20-20140904-TESEDSC_0-113
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	parte	parte	NOUN	_	Gender=Fem|Number=Sing	16	nsubj:pass	O	_
-3	superior	alto	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
+3	superior	superior	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 4-5	da	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -204357,7 +204357,7 @@
 26	de	de	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	corpo	corpo	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
-29	superior	alto	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
+29	superior	superior	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
 30	através	através	ADV	_	_	33	case	O	MWEPOS=ADP
 31-32	do	_	_	_	_	_	_	_	_
 31	de	de	ADP	_	_	30	fixed	O	_
@@ -207515,7 +207515,7 @@
 19	bem	bem	ADV	_	_	17	advmod	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	parte	parte	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
-22	superior	alto	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
+22	superior	superior	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
 23-24	da	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
@@ -211779,7 +211779,7 @@
 27	seja	ser	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	26	fixed	O	_
 28	,	,	PUNCT	_	_	26	punct	O	_
 29	bem	bem	ADV	_	_	30	advmod	O	_
-30	superior	alto	ADJ	_	Gender=Masc|Number=Sing	24	amod	O	_
+30	superior	superior	ADJ	_	Gender=Masc|Number=Sing	24	amod	O	_
 31-32	à	_	_	_	_	_	_	_	_
 31	a	a	ADP	_	_	33	case	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
@@ -224897,7 +224897,7 @@
 46	atingido	atingir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	42	conj	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
 48	manto	manto	NOUN	_	Gender=Masc|Number=Sing	46	obj	O	_
-49	superior	alto	ADJ	_	Gender=Masc|Number=Sing	48	amod	O	_
+49	superior	superior	ADJ	_	Gender=Masc|Number=Sing	48	amod	O	_
 50	.	.	PUNCT	_	_	13	punct	O	_
 
 # text = Chamam a atenção também para uma importante estrutura regional que atravessa o Alto de Cabo Frio e se une à zona de deformação oceânica Cruzeiro do Sul (figura 2.1).
@@ -227924,7 +227924,7 @@
 42	amplitude	amplitude	NOUN	_	Gender=Fem|Number=Sing	34	conj	O	_
 43	absoluta	absoluto	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 44	máxima	grande	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
-45	superior	alto	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
+45	superior	superior	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 46	a	a	ADP	_	_	48	case	O	_
 47	800	800	NUM	_	NumType=Card	48	nummod	O	_
 48	nT	nt	NOUN	_	Gender=Masc|Number=Plur	45	obl	O	_
@@ -230661,7 +230661,7 @@
 56	em	em	ADP	_	_	58	case	O	_
 57	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	58	det	O	_
 58	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	55	obl	B=GEOCRONOLOGIA	_
-59	superior	alto	ADJ	_	Gender=Masc|Number=Sing	58	amod	I=GEOCRONOLOGIA	_
+59	superior	superior	ADJ	_	Gender=Masc|Number=Sing	58	amod	I=GEOCRONOLOGIA	_
 60-61	ao	_	_	_	_	_	_	_	_
 60	a	a	ADP	_	_	62	case	O	_
 61	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	62	det	O	_
@@ -231688,7 +231688,7 @@
 20	em	em	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	18	obl	B=GEOCRONOLOGIA	_
-23	superior	alto	ADJ	_	Gender=Masc|Number=Sing	22	amod	I=GEOCRONOLOGIA	_
+23	superior	superior	ADJ	_	Gender=Masc|Number=Sing	22	amod	I=GEOCRONOLOGIA	_
 24	e	e	CCONJ	_	_	25	cc	O	_
 25	Terciário	Terciário	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=GEOCRONOLOGIA	_
 26	)	)	PUNCT	_	_	12	punct	O	_
@@ -245870,7 +245870,7 @@
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	enquanto	enquanto	SCONJ	_	_	11	mark	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-9	superior	alto	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
+9	superior	superior	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
 10	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	11	expl	O	_
 11	desloca	deslocar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	advcl	O	_
 12	a	a	ADP	_	_	14	case	O	_
@@ -245923,7 +245923,7 @@
 18	a	a	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 20	placa	placa	NOUN	_	Gender=Fem|Number=Sing	16	nmod	O	_
-21	superior	alto	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
+21	superior	superior	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22-23	move-se	_	_	_	_	_	_	_	_
 22	move	mover	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	advcl	O	_
 23	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	22	expl	O	_
@@ -272017,7 +272017,7 @@
 5	em	em	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	parte	parte	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
-8	superior	alto	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
+8	superior	superior	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9-10	à	_	_	_	_	_	_	_	_
 9	a	a	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
@@ -272071,7 +272071,7 @@
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	fase	fase	NOUN	_	Gender=Fem|Number=Sing	14	advcl	O	_
-8	superior	alto	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
+8	superior	superior	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
 9	,	,	PUNCT	_	_	7	punct	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	microemulsão	microemulsão	NOUN	_	Gender=Fem|Number=Sing	14	nsubj	O	_
@@ -281692,7 +281692,7 @@
 19	em	em	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	parte	parte	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
-22	superior	alto	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
+22	superior	superior	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
@@ -291310,7 +291310,7 @@
 10	gotas	gota	NOUN	_	Gender=Fem|Number=Plur	8	obl:agent	O	_
 11	com	com	ADP	_	_	12	case	O	_
 12	diâmetro	diâmetro	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
-13	superior	alto	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
+13	superior	superior	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	a	a	ADP	_	_	15	case	O	_
 15	150	150	NUM	_	NumType=Card	13	obl	O	_
 16	µ	µ	ADP	_	_	17	case	O	_
@@ -295289,7 +295289,7 @@
 12	por	por	ADP	_	_	13	case	O	_
 13	volume	volume	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 14	seja	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	cop	O	_
-15	superior	alto	ADJ	_	Gender=Masc|Number=Sing	3	acl:relcl	O	_
+15	superior	superior	ADJ	_	Gender=Masc|Number=Sing	3	acl:relcl	O	_
 16	a	a	ADP	_	_	19	case	O	_
 17	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
 18	determinado	determinado	DET	_	Gender=Masc|Number=Sing|PronType=Ind	19	det	O	_
