commit ceb12f5b35f8582d9149f4ca7a1b677ebb9b9290
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Jan 4 17:58:50 2022 -0300

    os verbos viver|ocorrer|crescer|entrar não devem ter argumento

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 21dcc59..3e053c7 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1769,7 +1769,7 @@
 4	BPS	BPS	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 5	0007	0007	NUM	_	NumType=Card	4	flat:name	O	_
 6	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-7	localizado	localizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	xcomp	O	_
+7	localizado	localizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	advcl	O	_
 8	próximo	próximo	ADV	_	_	7	advmod	O	_
 9-10	à	_	_	_	_	_	_	_	_
 9	a	a	ADP	_	_	11	case	O	_
@@ -24974,7 +24974,7 @@
 8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod	O	_
 9	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	10	nsubj	O	_
 10	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	acl:relcl	O	_
-11	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	10	xcomp	O	_
+11	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	10	advcl	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	forma	forma	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 14	heterolítica	heterolítico	ADJ	_	Gender=Fem|Number=Sing	13	amod	O	_
@@ -29847,7 +29847,7 @@
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
 2	equinóides	equinóide	NOUN	_	Gender=Masc|Number=Plur	3	nsubj	O	_
 3	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-4	associados	associar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	xcomp	O	_
+4	associados	associar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	advcl	O	_
 5-6	aos	_	_	_	_	_	_	_	_
 5	a	a	ADP	_	_	7	case	O	_
 6	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	7	det	O	_
@@ -43927,7 +43927,7 @@
 8	animais	animal	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 9	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	10	nsubj	O	_
 10	vivem	viver	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	acl:relcl	O	_
-11	associados	associado	ADJ	_	Gender=Masc|Number=Plur	10	xcomp	O	_
+11	associados	associado	ADJ	_	Gender=Masc|Number=Plur	10	advcl	O	_
 12-13	ao	_	_	_	_	_	_	_	_
 12	a	a	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
@@ -65256,7 +65256,7 @@
 2	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	3	nsubj	B=LITOLOGIA	_
 3	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	como	como	ADP	_	_	5	case	O	_
-5	lentes	lente	NOUN	_	Gender=Fem|Number=Plur	3	xcomp	O	_
+5	lentes	lente	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 6	e	e	CCONJ	_	_	7	cc	O	_
 7	sets	set	NOUN	_	Gender=Masc|Number=Plur	5	conj	O	_
 8	tabulares	tabular	ADJ	_	Gender=Masc|Number=Plur	7	amod	O	_
@@ -67980,7 +67980,7 @@
 7	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	9	nsubj	O	_
 8	eólico	eólico	ADJ	_	Gender=Masc|Number=Sing	7	amod	B=SISTEMA_DEPOSICIONAL	_
 9	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-10	interdigitado	interdigitar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	9	xcomp	O	_
+10	interdigitado	interdigitar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	9	advcl	O	_
 11-12	à	_	_	_	_	_	_	_	_
 11	a	a	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
@@ -80924,7 +80924,7 @@
 50	residenciais	residencial	ADJ	_	Gender=Fem|Number=Plur	49	amod	O	_
 51	)	)	PUNCT	_	_	39	punct	O	_
 52	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	53	aux:pass	O	_
-53	reduzido	reduzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	2	ccomp	O	_
+53	reduzido	reduzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	2	conj	O	_
 54	,	,	PUNCT	_	_	53	punct	O	_
 55	já	já	ADV	_	_	67	mark	O	MWEPOS=SCONJ
 56	que	que	SCONJ	_	_	55	fixed	O	_
@@ -92025,9 +92025,9 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	EOD	EOD	PROPN	_	Gender=Fem|Number=Sing	3	nsubj	O	_
 3	verificará	verificar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	0	root	O	_
-4	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	3	expl	O	_
+4	se	se	SCONJ	_	_	15	mark	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
-6	reduções	redução	NOUN	_	Gender=Fem|Number=Plur	3	obj	O	_
+6	reduções	redução	NOUN	_	Gender=Fem|Number=Plur	15	nsubj	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	6	nmod	O	_
 9	de	de	ADP	_	_	10	case	O	_
@@ -92036,9 +92036,9 @@
 12	efeito	efeito	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 13	estufa	estufa	NOUN	_	Gender=Fem|Number=Sing	12	compound	O	_
 14	monitoradas	monitorar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	8	acl	O	_
-15	ocorreram	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	3	conj	O	_
+15	ocorreram	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	3	ccomp	O	_
 16	como	como	ADP	_	_	17	case	O	_
-17	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	15	xcomp	O	_
+17	resultado	resultado	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 18-19	da	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
@@ -112833,7 +112833,7 @@
 6	impurezas	impureza	NOUN	_	Gender=Fem|Number=Plur	9	nsubj:pass	O	_
 7	não	não	ADV	_	Polarity=Neg	9	advmod	O	_
 8	forem	ser	AUX	_	Mood=Sub|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	9	aux:pass	O	_
-9	tratadas	tratar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	3	ccomp	O	_
+9	tratadas	tratar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	3	advcl	O	_
 10	e	e	CCONJ	_	_	11	cc	O	_
 11	removidas	remover	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	9	conj	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -147300,7 +147300,7 @@
 33	processo	processo	NOUN	_	Gender=Masc|Number=Sing	30	obl	O	_
 34	como	como	ADP	_	_	36	case	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
-36	separação	separação	NOUN	_	Gender=Fem|Number=Sing	30	xcomp	O	_
+36	separação	separação	NOUN	_	Gender=Fem|Number=Sing	29	nmod:appos	O	_
 37	entre	entre	ADP	_	_	39	case	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	det	O	_
 39	biodiesel	biodiesel	NOUN	_	Gender=Masc|Number=Sing	36	nmod	O	_
@@ -160636,7 +160636,7 @@
 3	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	como	como	ADP	_	_	6	case	O	_
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	subproduto	subproduto	NOUN	_	Gender=Masc|Number=Sing	3	xcomp	O	_
+6	subproduto	subproduto	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	em	em	ADP	_	_	9	case	O	_
 8	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
 9	planta	planta	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
@@ -187387,7 +187387,7 @@
 7	a	a	ADP	_	_	8	case	O	_
 8	campanianos	campaniano	ADJ	_	Gender=Masc|Number=Plur	6	obl	B=UNIDADE_CRONOESTRATIGRÁFICA	_
 9	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-10	intercalados	intercalar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	xcomp	O	_
+10	intercalados	intercalar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	advcl	O	_
 11	a	a	ADP	_	_	12	case	O	_
 12	rochas	rocha	NOUN	_	Gender=Fem|Number=Plur	10	obl:arg	O	_
 13	vulcanoclásticas	vulcanoclástico	ADJ	_	Gender=Fem|Number=Plur	12	amod	O	_
@@ -191074,7 +191074,7 @@
 15	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	como	como	ADP	_	_	18	case	O	_
 17	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
-18	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	15	xcomp	O	_
+18	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 19	espessa	espesso	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	brecha	brecha	NOUN	_	Gender=Fem|Number=Sing	18	nmod	B=LITOLOGIA	_
@@ -198714,8 +198714,8 @@
 12	em	em	ADP	_	_	14	case	O	_
 13	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	14	det	O	_
 14	caso	caso	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
-15	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	5	advcl	O	_
-16	próxima	próximo	ADJ	_	Gender=Fem|Number=Sing	15	xcomp	O	_
+15	ocorrendo	ocorrer	VERB	_	VerbForm=Ger	9	acl	O	_
+16	próxima	próximo	ADV	_	_	15	advmod	O	_
 17-18	à	_	_	_	_	_	_	_	_
 17	a	a	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -281493,7 +281493,7 @@
 6	animais	animal	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 7	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	8	nsubj	O	_
 8	vivem	viver	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	acl:relcl	O	_
-9	associados	associar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	xcomp	O	_
+9	associados	associar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	advcl	O	_
 10-11	aos	_	_	_	_	_	_	_	_
 10	a	a	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
