commit a5334dd7e0cacedc64cf32ef16fc19ce4f0807d8
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Jun 23 20:06:54 2021 -0300

    MWE a_seguir

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b6ea396..890c2be 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -619,8 +619,8 @@
 
 # text = A seguir, são apresentadas as etapas e a metodologia que foi adotada no trabalho..
 # sent_id = 247-20140910-MONOGRAFIA_0-16
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	5	advcl	O	_
+1	A	a	ADP	_	_	5	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -3904,8 +3904,8 @@
 13	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	14	det	O	_
 14	supersequências	supersequência	NOUN	_	Gender=Fem|Number=Plur	11	obl:agent	O	_
 15	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	14	acl	O	_
-16	a	a	SCONJ	_	_	17	mark	O	_
-17	seguir	seguir	VERB	_	VerbForm=Inf	15	advcl	O	_
+16	a	a	ADP	_	_	15	obl	O	MWEPOS=ADV
+17	seguir	seguir	VERB	_	VerbForm=Inf	16	fixed	O	_
 18	,	,	PUNCT	_	_	17	punct	O	_
 19	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	20	nsubj	O	_
 20	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	acl:relcl	O	_
@@ -8285,8 +8285,8 @@
 37	,	,	PUNCT	_	_	39	punct	O	_
 38	conforme	conforme	SCONJ	_	_	39	mark	O	_
 39	apresentado	apresentar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	32	advcl	O	_
-40	a	a	SCONJ	_	_	41	mark	O	_
-41	seguir	seguir	VERB	_	VerbForm=Inf	39	xcomp	O	_
+40	a	a	ADP	_	_	39	obl	O	MWEPOS=ADV
+41	seguir	seguir	VERB	_	VerbForm=Inf	40	fixed	O	_
 42	e	e	CCONJ	_	_	45	cc	O	_
 43-44	nos	_	_	_	_	_	_	_	_
 43	em	em	ADP	_	_	45	case	O	_
@@ -14864,8 +14864,8 @@
 
 # text = A seguir será feita uma breve discussão sobre cada constituinte presente na emulsão asfáltica. • Asfalto: o asfalto é considerado o principal constituinte da emulsão asfáltica, podendo variar na proporção de 40 até 70% em volume..
 # sent_id = 241-20140227-MONOGRAFIA_0-123
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	feita	fazer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 5	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -15837,8 +15837,8 @@
 16	asfáltica	asfáltico	ADJ	_	Gender=Fem|Number=Sing	15	amod	B=LITOLOGIA	_
 17	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	18	aux:pass	O	_
 18	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-19	a	a	SCONJ	_	_	20	mark	O	_
-20	seguir	seguir	VERB	_	VerbForm=Inf	18	advcl	O	_
+19	a	a	ADP	_	_	18	obl	O	MWEPOS=ADV
+20	seguir	seguir	VERB	_	VerbForm=Inf	19	fixed	O	_
 21	.	.	PUNCT	_	_	18	punct	O	_
 22	.	.	PUNCT	_	_	18	punct	O	_
 
@@ -17615,8 +17615,8 @@
 17	,	,	PUNCT	_	_	15	punct	O	_
 18	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	19	aux:pass	O	_
 19	analisadas	analisar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-20	a	a	SCONJ	_	_	21	mark	O	_
-21	seguir	seguir	VERB	_	VerbForm=Inf	19	advcl	O	_
+20	a	a	ADP	_	_	19	obl	O	MWEPOS=ADV
+21	seguir	seguir	VERB	_	VerbForm=Inf	20	fixed	O	_
 22	.	.	PUNCT	_	_	19	punct	O	_
 23	.	.	PUNCT	_	_	19	punct	O	_
 
@@ -28446,10 +28446,10 @@
 
 # text = A seguir, tabela 3, síntese da associação de fácies e interpretação do sistema paleodeposicional: Tabela 3..
 # sent_id = 284-20150917-MONOGRAFIA_0-292
-1	A	a	SCONJ	_	_	2	mark	O	_
+1	A	a	ADP	_	_	4	amod	O	MWEPOS=ADJ
 2	seguir	seguir	VERB	_	VerbForm=Inf	0	root	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
-4	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	2	obj	O	_
+4	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 5	3	3	NUM	_	NumType=Card	4	nummod	O	_
 6	,	,	PUNCT	_	_	7	punct	O	_
 7	síntese	síntese	NOUN	_	Gender=Fem|Number=Sing	4	conj	O	_
@@ -28811,8 +28811,8 @@
 
 # text = A seguir, são apresentadas características dos limites da sequência deposicional, a superfície de inundação máxima, os tratos de sistemas e uma discussão dos resultados obtidos em relação aos trabalhos disponíveis na literatura..
 # sent_id = 284-20150917-MONOGRAFIA_0-300
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	5	advcl	O	_
+1	A	a	ADP	_	_	5	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -46567,8 +46567,8 @@
 
 # text = A seguir, é novamente bombeado para o poço e o ciclo recomeça (JUNIOR, 2005)..
 # sent_id = 24-20150122-MONOGRAFIA_0-49
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	6	advcl	O	_
+1	A	a	ADP	_	_	6	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	aux:pass	O	_
 5	novamente	novamente	ADV	_	_	6	advmod	O	_
@@ -81344,8 +81344,8 @@
 21	tecnologias	tecnologia	NOUN	_	Gender=Fem|Number=Plur	23	nsubj	O	_
 22	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	23	cop	O	_
 23	descritas	descrito	ADJ	_	Gender=Fem|Number=Plur	7	conj	O	_
-24	a	a	SCONJ	_	_	25	mark	O	_
-25	seguir	seguir	VERB	_	VerbForm=Inf	23	advcl	O	_
+24	a	a	ADP	_	_	23	obl	O	MWEPOS=ADV
+25	seguir	seguir	VERB	_	VerbForm=Inf	24	fixed	O	_
 26	(	(	PUNCT	_	_	27	punct	O	_
 27	Gupta	Gupta	PROPN	_	Gender=Masc|Number=Sing	23	parataxis	O	_
 28	,	,	PUNCT	_	_	27	flat:name	O	_
@@ -82797,8 +82797,8 @@
 
 # text = A seguir serão descritos os mecanismos..
 # sent_id = 161-20150810-MONOGRAFIA_0-160
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	descritos	descrever	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
@@ -85430,8 +85430,8 @@
 1	Serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	2	aux:pass	O	_
 2	descritos	descrever	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	2	advcl	O	_
+4	a	a	ADP	_	_	2	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	,	,	PUNCT	_	_	2	punct	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
 8	formas	forma	NOUN	_	Gender=Fem|Number=Plur	2	nsubj:pass	O	_
@@ -89177,8 +89177,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-365
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	5	nsubj	B=INSTRUMENTO	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	evidencia	evidenciar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	localização	localização	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
@@ -89452,8 +89452,8 @@
 17	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 18	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	19	cop	O	_
 19	descritos	descrito	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-20	a	a	SCONJ	_	_	21	mark	O	_
-21	seguir	seguir	VERB	_	VerbForm=Inf	19	advcl	O	_
+20	a	a	ADP	_	_	19	obl	O	MWEPOS=ADV
+21	seguir	seguir	VERB	_	VerbForm=Inf	20	fixed	O	_
 22	..	..	PUNCT	_	_	19	punct	O	_
 
 # text = Os resultados dos cálculos serão apresentados posteriormente no capítulo 8..
@@ -89706,8 +89706,8 @@
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	4	nsubj:pass	O	_
 3	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-5	a	a	SCONJ	_	_	6	mark	O	_
-6	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+5	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+6	seguir	seguir	VERB	_	VerbForm=Inf	5	fixed	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
 8	adaptado	adaptar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	advcl	O	_
 9	de	de	ADP	_	_	10	case	O	_
@@ -90728,8 +90728,8 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	lista	lista	NOUN	_	Gender=Fem|Number=Sing	7	nsubj	O	_
 3	,	,	PUNCT	_	_	5	punct	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	7	advcl	O	_
+4	a	a	ADP	_	_	7	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	resume	resumir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
@@ -91459,8 +91459,8 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	quadro	quadro	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
 3	,	,	PUNCT	_	_	5	punct	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	7	advcl	O	_
+4	a	a	ADP	_	_	7	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	destaca	destacar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
@@ -91956,8 +91956,8 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	lista	lista	NOUN	_	Gender=Fem|Number=Sing	7	nsubj	O	_
 3	,	,	PUNCT	_	_	5	punct	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	7	advcl	O	_
+4	a	a	ADP	_	_	7	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	contém	conter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
@@ -92213,8 +92213,8 @@
 17	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 18	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	19	cop	O	_
 19	descritos	descrito	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-20	a	a	SCONJ	_	_	21	mark	O	_
-21	seguir	seguir	VERB	_	VerbForm=Inf	19	advcl	O	_
+20	a	a	ADP	_	_	19	obl	O	MWEPOS=ADV
+21	seguir	seguir	VERB	_	VerbForm=Inf	20	fixed	O	_
 22	.	.	PUNCT	_	_	19	punct	O	_
 23	.	.	PUNCT	_	_	19	punct	O	_
 
@@ -92630,8 +92630,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-467
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	resume	resumir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	cálculo	cálculo	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
@@ -93094,8 +93094,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-482
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	5	nsubj	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	resume	resumir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	7	det	O	_
 7	processo	processo	NOUN	_	Gender=Masc|Number=Sing	5	obj	O	_
@@ -93944,8 +93944,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-504
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	12	nsubj	B=INSTRUMENTO	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	,	,	PUNCT	_	_	6	punct	O	_
 6	elaborado	elaborar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
 7	através	através	ADV	_	_	6	advmod	O	_
@@ -96059,8 +96059,8 @@
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
 32	função	função	NOUN	_	Gender=Fem|Number=Sing	29	obl:agent	O	_
 33	gráfica	gráfico	ADJ	_	Gender=Fem|Number=Sing	32	amod	O	_
-34	a	a	SCONJ	_	_	35	mark	O	_
-35	seguir	seguir	VERB	_	VerbForm=Inf	29	advcl	O	_
+34	a	a	ADP	_	_	29	obl	O	MWEPOS=ADV
+35	seguir	seguir	VERB	_	VerbForm=Inf	34	fixed	O	_
 36	:	:	PUNCT	_	_	14	punct	O	_
 37	Tanto	tanto	ADV	_	_	40	advmod	O	_
 38	para	para	ADP	_	_	40	case	O	_
@@ -97810,8 +97810,8 @@
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	4	nsubj	O	_
 3	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	4	expl	O	_
 4	encontra	encontrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-5	a	a	SCONJ	_	_	6	mark	O	_
-6	seguir	seguir	VERB	_	VerbForm=Inf	4	xcomp	O	_
+5	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+6	seguir	seguir	VERB	_	VerbForm=Inf	5	fixed	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
 8	Stevens	Stevens	PROPN	_	Gender=Masc|Number=Sing	4	parataxis	O	_
 9	,	,	PUNCT	_	_	8	flat:name	O	_
@@ -98626,8 +98626,8 @@
 5	a	a	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	equação	equação	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
-8	a	a	SCONJ	_	_	9	mark	O	_
-9	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+8	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+9	seguir	seguir	VERB	_	VerbForm=Inf	8	fixed	O	_
 10	(	(	PUNCT	_	_	11	punct	O	_
 11	adaptado	adaptar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	7	acl	O	_
 12	de	de	ADP	_	_	13	case	O	_
@@ -100051,8 +100051,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-671
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	5	nsubj	B=INSTRUMENTO	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	toda	todo	DET	_	Gender=Fem|Number=Sing|PronType=Tot	8	det	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -100550,8 +100550,8 @@
 19	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 20	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	cop	O	_
 21	descritos	descrito	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-22	a	a	SCONJ	_	_	23	mark	O	_
-23	seguir	seguir	VERB	_	VerbForm=Inf	21	advcl	O	_
+22	a	a	ADP	_	_	21	obl	O	MWEPOS=ADV
+23	seguir	seguir	VERB	_	VerbForm=Inf	22	fixed	O	_
 24	..	..	PUNCT	_	_	21	punct	O	_
 
 # text = Os resultados dos cálculos serão apresentados posteriormente no capítulo 8..
@@ -101036,8 +101036,8 @@
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	4	nsubj:pass	O	_
 3	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-5	a	a	SCONJ	_	_	6	mark	O	_
-6	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+5	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+6	seguir	seguir	VERB	_	VerbForm=Inf	5	fixed	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
 8	adaptado	adaptar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	advcl	O	_
 9	de	de	ADP	_	_	10	case	O	_
@@ -101191,8 +101191,8 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-705
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	8	nsubj	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	já	já	ADV	_	_	8	advmod	O	_
 6	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	cop	O	_
 7	em	em	ADP	_	_	8	case	O	_
@@ -104731,8 +104731,8 @@
 2	equação	equação	NOUN	_	Gender=Fem|Number=Sing	4	nsubj	O	_
 3	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	4	expl	O	_
 4	encontra	encontrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-5	a	a	SCONJ	_	_	6	mark	O	_
-6	seguir	seguir	VERB	_	VerbForm=Inf	4	xcomp	O	_
+5	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+6	seguir	seguir	VERB	_	VerbForm=Inf	5	fixed	O	_
 7	:	:	PUNCT	_	_	8	punct	O	_
 8	Emissões	Emissões	PROPN	_	Gender=Fem|Number=Plur	4	parataxis	O	_
 9			PROPN	_	Number=Sing	8	flat:name	O	_
@@ -104971,8 +104971,8 @@
 8	siderúrgicas	siderúrgica	NOUN	_	Gender=Fem|Number=Plur	5	obl	O	_
 9	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	10	cop	O	_
 10	descritos	descrito	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-11	a	a	SCONJ	_	_	12	mark	O	_
-12	seguir	seguir	VERB	_	VerbForm=Inf	10	advcl	O	_
+11	a	a	ADP	_	_	10	obl	O	MWEPOS=ADV
+12	seguir	seguir	VERB	_	VerbForm=Inf	11	fixed	O	_
 13	:	:	PUNCT	_	_	10	punct	O	_
 14	32	32	NUM	_	NumType=Card	10	appos	O	_
 
@@ -105343,8 +105343,8 @@
 70	de	de	ADP	_	_	72	case	O	_
 71	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	72	det	O	_
 72	equação	equação	NOUN	_	Gender=Fem|Number=Sing	69	obl	O	_
-73	a	a	SCONJ	_	_	74	mark	O	_
-74	seguir	seguir	VERB	_	VerbForm=Inf	72	acl	O	_
+73	a	a	ADP	_	_	72	amod	O	MWEPOS=ADJ
+74	seguir	seguir	VERB	_	VerbForm=Inf	73	fixed	O	_
 75	.	.	PUNCT	_	_	61	punct	O	_
 76	.	.	PUNCT	_	_	61	punct	O	_
 
@@ -106819,8 +106819,8 @@
 32	em	em	ADP	_	_	34	case	O	_
 33	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	34	det	O	_
 34	quadros	quadro	NOUN	_	Gender=Masc|Number=Plur	31	obl	O	_
-35	a	a	SCONJ	_	_	36	mark	O	_
-36	seguir	seguir	VERB	_	VerbForm=Inf	31	advcl	O	_
+35	a	a	ADP	_	_	31	amod	O	MWEPOS=ADJ
+36	seguir	seguir	VERB	_	VerbForm=Inf	35	fixed	O	_
 37	,	,	PUNCT	_	_	47	punct	O	_
 38	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	39	det	O	_
 39	valores	valor	NOUN	_	Gender=Masc|Number=Plur	47	nsubj	O	_
@@ -107316,8 +107316,8 @@
 24	quadros	quadro	NOUN	_	Gender=Masc|Number=Plur	21	obl	O	_
 25	e	e	CCONJ	_	_	26	cc	O	_
 26	gráficos	gráfico	ADJ	_	Gender=Masc|Number=Plur	24	conj	O	_
-27	a	a	SCONJ	_	_	28	mark	O	_
-28	seguir	seguir	VERB	_	VerbForm=Inf	21	advcl	O	_
+27	a	a	ADP	_	_	21	obl	O	MWEPOS=ADV
+28	seguir	seguir	VERB	_	VerbForm=Inf	27	fixed	O	_
 29	:	:	PUNCT	_	_	30	punct	O	_
 30	Quadro	Quadro	NOUN	_	Gender=Masc|Number=Sing	24	appos	O	_
 31	14	14	PROPN	_	Number=Sing	30	nmod	O	_
@@ -109721,8 +109721,8 @@
 2	blocos	bloco	NOUN	_	Gender=Masc|Number=Plur	4	nsubj:pass	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	citados	citar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-5	a	a	SCONJ	_	_	6	mark	O	_
-6	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+5	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+6	seguir	seguir	VERB	_	VerbForm=Inf	5	fixed	O	_
 7	:	:	PUNCT	_	_	4	punct	O	_
 8	-	-	PUNCT	_	_	9	punct	O	_
 9	BM-S-8	BM-S-8	PROPN	_	Gender=Masc|Number=Sing	4	appos	O	_
@@ -112496,8 +112496,8 @@
 13	área	área	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 14	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	15	aux:pass	O	_
 15	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-16	a	a	SCONJ	_	_	17	mark	O	_
-17	seguir	seguir	VERB	_	VerbForm=Inf	15	advcl	O	_
+16	a	a	ADP	_	_	15	obl	O	MWEPOS=ADV
+17	seguir	seguir	VERB	_	VerbForm=Inf	16	fixed	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
 19	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	15	parataxis	O	_
 20	5	5	PROPN	_	Number=Sing	19	flat:name	O	_
@@ -121685,8 +121685,8 @@
 8	submarino	submarino	ADJ	_	Gender=Masc|Number=Sing	7	amod	B=TRANSPORTE	_
 9	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	10	aux:pass	O	_
 10	citadas	citar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-11	a	a	SCONJ	_	_	12	mark	O	_
-12	seguir	seguir	VERB	_	VerbForm=Inf	10	xcomp	O	_
+11	a	a	ADP	_	_	10	obl	O	MWEPOS=ADV
+12	seguir	seguir	VERB	_	VerbForm=Inf	11	fixed	O	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	MORAIS	MORAIS	PROPN	_	Gender=Masc|Number=Sing	10	parataxis	O	_
 15	,	,	PUNCT	_	_	16	punct	O	_
@@ -123915,8 +123915,8 @@
 
 # text = A seguir (Tabela 7) são apresentados os dados dos dutos envolvidos no projeto..
 # sent_id = 334-20150521-MONOGRAFIA_0-389
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	8	advcl	O	_
+1	A	a	ADP	_	_	8	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	(	(	PUNCT	_	_	4	punct	O	_
 4	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	2	vocative	O	_
 5	7	7	PROPN	_	Number=Sing	4	flat:name	O	_
@@ -125487,8 +125487,8 @@
 # text = Temperatura a seguir (Figura 38)..
 # sent_id = 334-20150521-MONOGRAFIA_0-432
 1	Temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
-2	a	a	SCONJ	_	_	3	mark	O	_
-3	seguir	seguir	VERB	_	VerbForm=Inf	1	acl	O	_
+2	a	a	ADP	_	_	1	amod	O	MWEPOS=ADJ
+3	seguir	seguir	VERB	_	VerbForm=Inf	2	fixed	O	_
 4	(	(	PUNCT	_	_	5	punct	O	_
 5	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	1	appos	O	_
 6	38	38	PROPN	_	Number=Sing	5	flat:name	O	_
@@ -127792,8 +127792,8 @@
 21	,	,	PUNCT	_	_	23	punct	O	_
 22	conforme	conforme	SCONJ	_	_	23	mark	O	_
 23	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	7	advcl	O	_
-24	a	a	SCONJ	_	_	25	mark	O	_
-25	seguir	seguir	VERB	_	VerbForm=Inf	23	xcomp	O	_
+24	a	a	ADP	_	_	23	obl	O	MWEPOS=ADV
+25	seguir	seguir	VERB	_	VerbForm=Inf	24	fixed	O	_
 26	(	(	PUNCT	_	_	29	punct	O	_
 27	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	29	obl	O	_
 28	B	B	PROPN	_	Gender=Masc|Number=Sing	29	nsubj	O	_
@@ -128497,8 +128497,8 @@
 
 # text = A seguir são mostradas outras possibilidades de arranjos de membrana (Figura 45). (a): três estágios em paralelo; (b) dois estágios em série; (c) um estágio com reciclo parcial do permeado; (d)dois estágios com reciclo total do retentado do segundo estágio; (e)dois estágios com reciclo total do permeado do segundo estágio..
 # sent_id = 334-20150521-MONOGRAFIA_0-525
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	mostradas	mostrar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 5	outras	outro	DET	_	Gender=Fem|Number=Plur|PronType=Ind	6	det	O	_
@@ -132342,8 +132342,8 @@
 
 # text = A seguir, esta corrente é então encaminhada a um separador bifásico..
 # sent_id = 334-20150521-MONOGRAFIA_0-647
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	8	advcl	O	_
+1	A	a	ADP	_	_	8	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	5	det	O	_
 5	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	8	nsubj:pass	O	_
@@ -138622,8 +138622,8 @@
 10	emulsões	emulsão	NOUN	_	Gender=Fem|Number=Plur	8	nmod	B=QUÍMICA	_
 11	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	aux:pass	O	_
 12	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-13	a	a	SCONJ	_	_	14	mark	O	_
-14	seguir	seguir	VERB	_	VerbForm=Inf	12	advcl	O	_
+13	a	a	ADP	_	_	12	obl	O	MWEPOS=ADV
+14	seguir	seguir	VERB	_	VerbForm=Inf	13	fixed	O	_
 15	:	:	PUNCT	_	_	16	punct	O	_
 16			NOUN	_	Gender=Masc|Number=Sing	14	appos	O	_
 
@@ -157034,8 +157034,8 @@
 63	(	(	PUNCT	_	_	64	punct	O	_
 64	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	61	appos	O	_
 65	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	64	acl	O	_
-66	a	a	SCONJ	_	_	67	mark	O	_
-67	seguir	seguir	VERB	_	VerbForm=Inf	65	advcl	O	_
+66	a	a	ADP	_	_	65	obl	O	MWEPOS=ADV
+67	seguir	seguir	VERB	_	VerbForm=Inf	66	fixed	O	_
 68	)	)	PUNCT	_	_	64	punct	O	_
 69	,	,	PUNCT	_	_	71	punct	O	_
 70	melhor	melhor	ADV	_	_	71	advmod	O	_
@@ -158556,8 +158556,8 @@
 12	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 13	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	14	aux:pass	O	_
 14	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-15	a	a	SCONJ	_	_	16	mark	O	_
-16	seguir	seguir	VERB	_	VerbForm=Inf	14	advcl	O	_
+15	a	a	ADP	_	_	14	obl	O	MWEPOS=ADV
+16	seguir	seguir	VERB	_	VerbForm=Inf	15	fixed	O	_
 17	..	..	PUNCT	_	_	14	punct	O	_
 
 # text = 5.1 Sísmica de reflexão
@@ -185913,8 +185913,8 @@
 7	B100	B100	PROPN	_	Gender=Masc|Number=Sing	2	flat:name	O	_
 8	,	,	PUNCT	_	_	9	punct	O	_
 9	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
-10	a	a	SCONJ	_	_	11	mark	O	_
-11	seguir	seguir	VERB	_	VerbForm=Inf	9	advcl	O	_
+10	a	a	ADP	_	_	9	obl	O	MWEPOS=ADV
+11	seguir	seguir	VERB	_	VerbForm=Inf	10	fixed	O	_
 12	,	,	PUNCT	_	_	13	punct	O	_
 13	atestando	atestar	VERB	_	VerbForm=Ger	9	advcl	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -186261,8 +186261,8 @@
 3	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	5	nsubj:pass	O	_
 4	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	5	aux:pass	O	_
 5	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	2	acl:relcl	O	_
-6	a	a	SCONJ	_	_	7	mark	O	_
-7	seguir	seguir	VERB	_	VerbForm=Inf	5	advcl	O	_
+6	a	a	ADP	_	_	5	obl	O	MWEPOS=ADV
+7	seguir	seguir	VERB	_	VerbForm=Inf	6	fixed	O	_
 8	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	9	aux:pass	O	_
 9	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 10-11	dos	_	_	_	_	_	_	_	_
@@ -186701,8 +186701,8 @@
 32	em	em	ADP	_	_	34	case	O	_
 33	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	34	det	O	_
 34	tabelas	tabela	NOUN	_	Gender=Fem|Number=Plur	31	obl	O	_
-35	a	a	SCONJ	_	_	36	mark	O	_
-36	seguir	seguir	VERB	_	VerbForm=Inf	31	advcl	O	_
+35	a	a	ADP	_	_	31	obl	O	MWEPOS=ADV
+36	seguir	seguir	VERB	_	VerbForm=Inf	35	fixed	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
 38	utilizando	utilizar	VERB	_	VerbForm=Ger	31	advcl	O	_
 39	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	40	det	O	_
@@ -186758,8 +186758,8 @@
 32	em	em	ADP	_	_	34	case	O	_
 33	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	34	det	O	_
 34	tabelas	tabela	NOUN	_	Gender=Fem|Number=Plur	31	obl	O	_
-35	a	a	SCONJ	_	_	36	mark	O	_
-36	seguir	seguir	VERB	_	VerbForm=Inf	31	advcl	O	_
+35	a	a	ADP	_	_	31	obl	O	MWEPOS=ADV
+36	seguir	seguir	VERB	_	VerbForm=Inf	35	fixed	O	_
 37	,	,	PUNCT	_	_	38	punct	O	_
 38	utilizando	utilizar	VERB	_	VerbForm=Ger	31	advcl	O	_
 39	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	40	det	O	_
@@ -186796,8 +186796,8 @@
 # sent_id = 279-20140530-MONOGRAFIA_0-151
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
 2	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	5	nsubj	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	tiveram	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
 6	como	como	ADP	_	_	7	case	O	_
 7	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	5	xcomp	O	_
@@ -186885,8 +186885,8 @@
 4	e	e	CCONJ	_	_	5	cc	O	_
 5	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	2	conj	O	_
 6	5-7	5-7	PROPN	_	Number=Sing	5	flat:name	O	_
-7	a	a	SCONJ	_	_	8	mark	O	_
-8	seguir	seguir	VERB	_	VerbForm=Inf	9	advcl	O	_
+7	a	a	ADP	_	_	9	obl	O	MWEPOS=ADV
+8	seguir	seguir	VERB	_	VerbForm=Inf	7	fixed	O	_
 9	indicam	indicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
 11	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	9	obj	O	_
@@ -187029,8 +187029,8 @@
 4	e	e	CCONJ	_	_	5	cc	O	_
 5	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	2	conj	O	_
 6	5-7	5-7	PROPN	_	Number=Sing	5	flat:name	O	_
-7	a	a	SCONJ	_	_	8	mark	O	_
-8	seguir	seguir	VERB	_	VerbForm=Inf	9	advcl	O	_
+7	a	a	ADP	_	_	9	obl	O	MWEPOS=ADV
+8	seguir	seguir	VERB	_	VerbForm=Inf	7	fixed	O	_
 9	indicam	indicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	11	det	O	_
 11	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	9	obj	O	_
@@ -187237,8 +187237,8 @@
 # sent_id = 279-20140530-MONOGRAFIA_0-169
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
 2	gráficos	gráfico	NOUN	_	Gender=Masc|Number=Plur	5	nsubj	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	2	acl	O	_
+3	a	a	ADP	_	_	2	amod	O	MWEPOS=ADJ
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	mostram	mostrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	variação	variação	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
@@ -187285,8 +187285,8 @@
 
 # text = A seguir mostra-se o gráfico de consumo específico..
 # sent_id = 279-20140530-MONOGRAFIA_0-171
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	3	advcl	O	_
+1	A	a	ADP	_	_	3	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3-4	mostra-se	_	_	_	_	_	_	_	_
 3	mostra	mostrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Sing|Person=3|PronType=Prs	3	expl	O	_
@@ -188323,8 +188323,8 @@
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	18	obj	O	_
 22	5-22	5-22	PROPN	_	Number=Sing	21	flat:name	O	_
-23	a	a	SCONJ	_	_	24	mark	O	_
-24	seguir	seguir	VERB	_	VerbForm=Inf	18	advcl	O	_
+23	a	a	ADP	_	_	18	obl	O	MWEPOS=ADV
+24	seguir	seguir	VERB	_	VerbForm=Inf	23	fixed	O	_
 25	[	[	PUNCT	_	_	26	punct	O	_
 26	15	15	NUM	_	NumType=Card	24	parataxis	O	_
 27	]	]	PUNCT	_	_	26	punct	O	_
@@ -209377,8 +209377,8 @@
 
 # text = A seguir será feita uma breve descrição de algumas destas propriedades..
 # sent_id = 391-20160914-MONOGRAFIA_0-52
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	feita	fazer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 5	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
@@ -215144,8 +215144,8 @@
 
 # text = A seguir serão apresentados as Figuras 7, 8 e 9, referentes aos diagramas ternários obtidos variando-se apenas o tensoativo..
 # sent_id = 391-20160914-MONOGRAFIA_0-231
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
@@ -215690,7 +215690,7 @@
 14	quais	qual	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	16	nsubj	O	_
 15	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	16	cop	O	_
 16	descritas	descrito	ADJ	_	Gender=Fem|Number=Plur	12	acl:relcl	O	_
-17	a	a	SCONJ	_	_	18	mark	O	_
+17	a	a	ADP	_	_	16	obl	O	MWEPOS=ADV
 18	seguir	seguir	VERB	_	VerbForm=Inf	16	advcl	O	_
 19	:	:	PUNCT	_	_	20	punct	O	_
 20	1.	1.	NUM	_	NumType=Card	23	appos	O	_
@@ -232698,8 +232698,8 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	3	det	O	_
 3	itens	item	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	3	acl	O	_
+4	a	a	ADP	_	_	3	amod	O	MWEPOS=ADJ
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	7	aux:pass	O	_
 7	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 8	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -232834,8 +232834,8 @@
 
 # text = A seguir estão relacionadas algumas destas estruturas, conforme descrição do referido autor, que podem apresentar eventual efeito magnético observado na área desta pesquisa:
 # sent_id = 20-20140904-TESEDSC_0-56
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
 4	relacionadas	relacionar	ADJ	_	Gender=Fem|Number=Plur	0	root	O	_
 5	algumas	algum	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	4	nsubj	O	_
@@ -233850,8 +233850,8 @@
 11	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	12	aux:pass	O	_
 12	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 13	sucintamente	sucintamente	ADV	_	_	12	advmod	O	_
-14	a	a	SCONJ	_	_	15	mark	O	_
-15	seguir	seguir	VERB	_	VerbForm=Inf	12	advcl	O	_
+14	a	a	ADP	_	_	12	obl	O	MWEPOS=ADV
+15	seguir	seguir	VERB	_	VerbForm=Inf	14	fixed	O	_
 16	com	com	ADP	_	_	17	case	O	_
 17	base	base	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 18	em	em	ADP	_	_	19	case	O	_
@@ -240341,8 +240341,8 @@
 16	,	,	PUNCT	_	_	12	punct	O	_
 17	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	18	aux:pass	O	_
 18	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-19	a	a	SCONJ	_	_	20	mark	O	_
-20	seguir	seguir	VERB	_	VerbForm=Inf	18	advcl	O	_
+19	a	a	ADP	_	_	18	obl	O	MWEPOS=ADV
+20	seguir	seguir	VERB	_	VerbForm=Inf	19	fixed	O	_
 21	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
 22	sucinta	sucinto	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 23	descrição	descrição	NOUN	_	Gender=Fem|Number=Sing	20	obj	O	_
@@ -242688,8 +242688,8 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	3	det	O	_
 3	itens	item	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	7	advcl	O	_
+4	a	a	ADP	_	_	7	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	7	aux:pass	O	_
 7	dada	dar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 8	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
@@ -243370,8 +243370,8 @@
 5	levantamento	levantamento	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
 7	relacionados	relacionar	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-8	a	a	SCONJ	_	_	9	mark	O	_
-9	seguir	seguir	VERB	_	VerbForm=Inf	7	advcl	O	_
+8	a	a	ADP	_	_	7	obl	O	MWEPOS=ADV
+9	seguir	seguir	VERB	_	VerbForm=Inf	8	fixed	O	_
 10	:	:	PUNCT	_	_	12	punct	O	_
 11	-	-	PUNCT	_	_	12	punct	O	_
 12	Total	Total	PROPN	_	Gender=Masc|Number=Sing	7	appos	O	_
@@ -244954,8 +244954,8 @@
 10	em	em	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
 12	itens	item	NOUN	_	Gender=Masc|Number=Plur	9	obl	O	_
-13	a	a	SCONJ	_	_	14	mark	O	_
-14	seguir	seguir	VERB	_	VerbForm=Inf	9	advcl	O	_
+13	a	a	ADP	_	_	9	obl	O	MWEPOS=ADV
+14	seguir	seguir	VERB	_	VerbForm=Inf	13	fixed	O	_
 15	.	.	PUNCT	_	_	9	punct	O	_
 16	.	.	PUNCT	_	_	9	punct	O	_
 
@@ -245524,8 +245524,8 @@
 47	com	com	ADP	_	_	49	case	O	_
 48	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	49	det	O	_
 49	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	46	nmod	O	_
-50	a	a	SCONJ	_	_	51	mark	O	_
-51	seguir	seguir	VERB	_	VerbForm=Inf	49	acl	O	_
+50	a	a	ADP	_	_	49	amod	O	MWEPOS=ADJ
+51	seguir	seguir	VERB	_	VerbForm=Inf	50	fixed	O	_
 52	.	.	PUNCT	_	_	6	punct	O	_
 53	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -246498,8 +246498,8 @@
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
 15	dois	dois	NUM	_	NumType=Card	16	nummod	O	_
 16	casos	caso	NOUN	_	Gender=Masc|Number=Plur	13	nsubj:pass	O	_
-17	a	a	SCONJ	_	_	18	mark	O	_
-18	seguir	seguir	VERB	_	VerbForm=Inf	13	advcl	O	_
+17	a	a	ADP	_	_	13	obl	O	MWEPOS=ADV
+18	seguir	seguir	VERB	_	VerbForm=Inf	17	fixed	O	_
 19	:	:	PUNCT	_	_	13	punct	O	_
 20	-	-	PUNCT	_	_	23	punct	O	_
 21-22	No	_	_	_	_	_	_	_	_
@@ -247987,8 +247987,8 @@
 7	em	em	ADP	_	_	9	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
 9	itens	item	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
-10	a	a	SCONJ	_	_	11	mark	O	_
-11	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+10	a	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+11	seguir	seguir	VERB	_	VerbForm=Inf	10	fixed	O	_
 12	.	.	PUNCT	_	_	4	punct	O	_
 13	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -248446,8 +248446,8 @@
 15	com	com	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
-18	a	a	SCONJ	_	_	19	mark	O	_
-19	seguir	seguir	VERB	_	VerbForm=Inf	6	advcl	O	_
+18	a	a	ADP	_	_	17	amod	O	MWEPOS=ADJ
+19	seguir	seguir	VERB	_	VerbForm=Inf	18	fixed	O	_
 20	:	:	PUNCT	_	_	6	punct	O	_
 21	Considerando	considerar	VERB	_	VerbForm=Ger	19	advcl	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
@@ -248568,8 +248568,8 @@
 13	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	14	det	O	_
 14	critérios	critério	NOUN	_	Gender=Masc|Number=Plur	6	obl	O	_
 15	descritos	descrever	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	14	acl	O	_
-16	a	a	SCONJ	_	_	17	mark	O	_
-17	seguir	seguir	VERB	_	VerbForm=Inf	15	xcomp	O	_
+16	a	a	ADP	_	_	15	obl	O	MWEPOS=ADV
+17	seguir	seguir	VERB	_	VerbForm=Inf	16	fixed	O	_
 18	.	.	PUNCT	_	_	6	punct	O	_
 19	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -249396,8 +249396,8 @@
 29	mostram	mostrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	10	advcl	O	_
 30	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	31	det	O	_
 31	expressões	expressão	NOUN	_	Gender=Fem|Number=Plur	29	obj	O	_
-32	a	a	SCONJ	_	_	33	mark	O	_
-33	seguir	seguir	VERB	_	VerbForm=Inf	29	advcl	O	_
+32	a	a	ADP	_	_	29	obl	O	MWEPOS=ADV
+33	seguir	seguir	VERB	_	VerbForm=Inf	32	fixed	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	9	9	NUM	_	NumType=Card	33	obl	O	_
 36	)	)	PUNCT	_	_	35	punct	O	_
@@ -251027,8 +251027,8 @@
 23	como	como	SCONJ	_	_	25	mark	O	_
 24	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	25	aux:pass	O	_
 25	discutido	discutir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	10	advcl	O	_
-26	a	a	SCONJ	_	_	27	mark	O	_
-27	seguir	seguir	VERB	_	VerbForm=Inf	25	xcomp	O	_
+26	a	a	ADP	_	_	25	obl	O	MWEPOS=ADV
+27	seguir	seguir	VERB	_	VerbForm=Inf	26	fixed	O	_
 28	.	.	PUNCT	_	_	10	punct	O	_
 29	.	.	PUNCT	_	_	10	punct	O	_
 
@@ -251967,8 +251967,8 @@
 9	generalizada	generalizado	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	6	acl	O	_
 10	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	aux:pass	O	_
 11	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-12	a	a	SCONJ	_	_	13	mark	O	_
-13	seguir	seguir	VERB	_	VerbForm=Inf	11	advcl	O	_
+12	a	a	ADP	_	_	11	obl	O	MWEPOS=ADV
+13	seguir	seguir	VERB	_	VerbForm=Inf	12	fixed	O	_
 14	.	.	PUNCT	_	_	11	punct	O	_
 15	.	.	PUNCT	_	_	11	punct	O	_
 
@@ -256105,8 +256105,8 @@
 7	4.14	4.14	NUM	_	NumType=Card	5	conj	O	_
 8	e	e	CCONJ	_	_	9	cc	O	_
 9	4.15	4.15	NUM	_	NumType=Card	5	conj	O	_
-10	a	a	SCONJ	_	_	11	mark	O	_
-11	seguir	seguir	VERB	_	VerbForm=Inf	1	advcl	O	_
+10	a	a	ADP	_	_	1	obl	O	MWEPOS=ADV
+11	seguir	seguir	VERB	_	VerbForm=Inf	10	fixed	O	_
 12	que	que	SCONJ	_	_	13	mark	O	_
 13	correspondem	corresponder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	ccomp	O	_
 14	,	,	PUNCT	_	_	13	punct	O	_
@@ -256718,8 +256718,8 @@
 22	sintético	sintético	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
 23	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	24	aux:pass	O	_
 24	mostrado	mostrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	8	conj	O	_
-25	a	a	SCONJ	_	_	26	mark	O	_
-26	seguir	seguir	VERB	_	VerbForm=Inf	24	advcl	O	_
+25	a	a	ADP	_	_	24	obl	O	MWEPOS=ADV
+26	seguir	seguir	VERB	_	VerbForm=Inf	25	fixed	O	_
 27	:	:	PUNCT	_	_	28	punct	O	_
 28	Dados	dado	NOUN	_	Gender=Masc|Number=Plur	26	obj	O	_
 29	de	de	ADP	_	_	30	case	O	_
@@ -256945,8 +256945,8 @@
 27	sintético	sintético	ADJ	_	Gender=Masc|Number=Sing	26	amod	O	_
 28	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	29	aux:pass	O	_
 29	apresentado	apresentar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	5	conj	O	_
-30	a	a	SCONJ	_	_	31	mark	O	_
-31	seguir	seguir	VERB	_	VerbForm=Inf	29	advcl	O	_
+30	a	a	ADP	_	_	29	obl	O	MWEPOS=ADV
+31	seguir	seguir	VERB	_	VerbForm=Inf	30	fixed	O	_
 32	:	:	PUNCT	_	_	33	punct	O	_
 33	Dados	dado	NOUN	_	Gender=Masc|Number=Plur	31	obj	O	_
 34	de	de	ADP	_	_	35	case	O	_
@@ -257448,8 +257448,8 @@
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	capítulo	capítulo	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 12	5	5	NUM	_	NumType=Card	11	nummod	O	_
-13	a	a	SCONJ	_	_	14	mark	O	_
-14	seguir	seguir	VERB	_	VerbForm=Inf	8	advcl	O	_
+13	a	a	ADP	_	_	8	obl	O	MWEPOS=ADV
+14	seguir	seguir	VERB	_	VerbForm=Inf	13	fixed	O	_
 15	..	..	PUNCT	_	_	8	punct	O	_
 
 # text = 5 O EMBASAMENTO MAGNÉTICO
@@ -261920,8 +261920,8 @@
 31	em	em	ADP	_	_	33	case	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
 33	item	item	NOUN	_	Gender=Masc|Number=Sing	30	obl	O	_
-34	a	a	SCONJ	_	_	35	mark	O	_
-35	seguir	seguir	VERB	_	VerbForm=Inf	30	advcl	O	_
+34	a	a	ADP	_	_	30	obl	O	MWEPOS=ADV
+35	seguir	seguir	VERB	_	VerbForm=Inf	34	fixed	O	_
 36	.	.	PUNCT	_	_	5	punct	O	_
 37	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -264429,8 +264429,8 @@
 1	Em	em	ADP	_	_	3	case	O	_
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	3	det	O	_
 3	itens	item	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
-4	a	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	3	acl	O	_
+4	a	a	ADP	_	_	3	amod	O	MWEPOS=ADJ
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	7	aux:pass	O	_
 7	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 8	interpretações	interpretação	NOUN	_	Gender=Fem|Number=Plur	7	nsubj:pass	O	_
@@ -264499,8 +264499,8 @@
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	item	item	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-6	a	a	SCONJ	_	_	7	mark	O	_
-7	seguir	seguir	VERB	_	VerbForm=Inf	8	advcl	O	_
+6	a	a	ADP	_	_	8	obl	O	MWEPOS=ADV
+7	seguir	seguir	VERB	_	VerbForm=Inf	6	fixed	O	_
 8	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 9	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	magnetização	magnetização	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
@@ -267060,8 +267060,8 @@
 30	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	31	det	O	_
 31	capítulo	capítulo	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
 32	7	7	NUM	_	NumType=Card	31	nummod	O	_
-33	a	a	SCONJ	_	_	34	mark	O	_
-34	seguir	seguir	VERB	_	VerbForm=Inf	27	advcl	O	_
+33	a	a	ADP	_	_	27	obl	O	MWEPOS=ADV
+34	seguir	seguir	VERB	_	VerbForm=Inf	33	fixed	O	_
 35	.	.	PUNCT	_	_	16	punct	O	_
 36	.	.	PUNCT	_	_	16	punct	O	_
 
@@ -280921,8 +280921,8 @@
 11	metodologia	metodologia	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 12	permite	permitir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 13	listar	listar	VERB	_	VerbForm=Inf	12	xcomp	O	_
-14	a	a	SCONJ	_	_	15	mark	O	_
-15	seguir	seguir	VERB	_	VerbForm=Inf	13	xcomp	O	_
+14	a	a	ADP	_	_	13	obl	O	MWEPOS=ADV
+15	seguir	seguir	VERB	_	VerbForm=Inf	14	fixed	O	_
 16	algumas	algum	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	det	O	_
 17	conclusões	conclusão	NOUN	_	Gender=Fem|Number=Plur	15	obj	O	_
 18	:	:	PUNCT	_	_	12	punct	O	_
@@ -283357,8 +283357,8 @@
 9	conclusões	conclusão	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
 10	,	,	PUNCT	_	_	11	punct	O	_
 11	listadas	listar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	9	acl	O	_
-12	a	a	SCONJ	_	_	13	mark	O	_
-13	seguir	seguir	VERB	_	VerbForm=Inf	11	advcl	O	_
+12	a	a	ADP	_	_	11	obl	O	MWEPOS=ADV
+13	seguir	seguir	VERB	_	VerbForm=Inf	12	fixed	O	_
 14	,	,	PUNCT	_	_	13	punct	O	_
 15	sobre	sobre	ADP	_	_	16	case	O	_
 16	problemas	problema	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
@@ -294781,8 +294781,8 @@
 
 # text = A seguir será feita uma descrição dos polímeros atualmente em uso nos fluidos de perfuração. • Biopolímeros
 # sent_id = 119-20141209-TESEMSC_0-253
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	feita	fazer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 5	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -297030,7 +297030,7 @@
 # sent_id = 119-20141209-TESEMSC_0-333
 1	Se	se	SCONJ	_	_	6	mark	O	_
 2	,	,	PUNCT	_	_	4	punct	O	_
-3	a	a	SCONJ	_	_	6	mark	O	_
+3	a	a	ADP	_	_	6	obl	O	MWEPOS=ADV
 4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	submetermos	submeter	VERB	_	Number=Plur|Person=1|VerbForm=Inf	22	advcl	O	_
@@ -302956,8 +302956,8 @@
 10	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	acl	O	_
 11	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	aux:pass	O	_
 12	apresentados	apresentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-13	a	a	SCONJ	_	_	14	mark	O	_
-14	seguir	seguir	VERB	_	VerbForm=Inf	12	advcl	O	_
+13	a	a	ADP	_	_	12	obl	O	MWEPOS=ADV
+14	seguir	seguir	VERB	_	VerbForm=Inf	13	fixed	O	_
 15	..	..	PUNCT	_	_	12	punct	O	_
 
 # text = Os aditivos empregados nesta pesquisa para a formulação dos fluidos de perfuração, bem como o óleo utilizado estão citados na Tabela 3.1..
@@ -306490,8 +306490,8 @@
 3	usados	usar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	2	acl	O	_
 4	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	detalhados	detalhar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-6	a	a	SCONJ	_	_	7	mark	O	_
-7	seguir	seguir	VERB	_	VerbForm=Inf	5	advcl	O	_
+6	a	a	ADP	_	_	5	obl	O	MWEPOS=ADV
+7	seguir	seguir	VERB	_	VerbForm=Inf	6	fixed	O	_
 8	..	..	PUNCT	_	_	5	punct	O	_
 
 # text = Primeiramente mediu-se a massa das amostras de arenito saturadas com óleo mineral ( )..
@@ -315891,7 +315891,7 @@
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
 12	leitor	leitor	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 13	,	,	PUNCT	_	_	4	punct	O	_
-14	a	a	SCONJ	_	_	17	mark	O	_
+14	a	a	ADP	_	_	17	obl	O	MWEPOS=ADV
 15	seguir	seguir	VERB	_	VerbForm=Inf	14	fixed	O	_
 16	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	17	aux:pass	O	_
 17	explanado	explanar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -316749,8 +316749,8 @@
 1	4	4	NUM	_	NumType=Card	9	parataxis	O	_
 2	.	.	PUNCT	_	_	1	punct	O	_
 3	-	-	PUNCT	_	_	9	punct	O	_
-4	A	a	SCONJ	_	_	5	mark	O	_
-5	seguir	seguir	VERB	_	VerbForm=Inf	9	advcl	O	_
+4	A	a	ADP	_	_	9	obl	O	MWEPOS=ADV
+5	seguir	seguir	VERB	_	VerbForm=Inf	4	fixed	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	lama	lama	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
@@ -320565,8 +320565,8 @@
 
 # text = A seguir será explanado com mais detalhes o modelo Newtoniano, de Bingham e o de Ostwald Waale ou fluido de potência (pode ser usado para representar os fluidos pseudoplásticos), pois estes modelos serão utilizados para os resultados deste trabalho..
 # sent_id = 240-20140220-MONOGRAFIA_0-165
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	aux:pass	O	_
 4	explanado	explanar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 5	com	com	ADP	_	_	7	case	O	_
@@ -320809,8 +320809,8 @@
 16	por	por	ADP	_	_	18	case	O	_
 17	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	18	det	O	_
 18	equações	equação	NOUN	_	Gender=Fem|Number=Plur	15	obl:agent	O	_
-19	a	a	SCONJ	_	_	20	mark	O	_
-20	seguir	seguir	VERB	_	VerbForm=Inf	15	advcl	O	_
+19	a	a	ADP	_	_	15	obl	O	MWEPOS=ADV
+20	seguir	seguir	VERB	_	VerbForm=Inf	19	fixed	O	_
 21	:	:	PUNCT	_	_	15	punct	O	_
 22	Sendo	ser	AUX	_	VerbForm=Ger	24	cop	O	_
 23	:	:	PUNCT	_	_	24	punct	O	_
@@ -327097,8 +327097,8 @@
 
 # text = A seguir, seguem respectivamente as Figuras 20 e 21, comparativos entre as curva de fluxo e viscosidade
 # sent_id = 240-20140220-MONOGRAFIA_0-362
-1	A	a	SCONJ	_	_	2	mark	O	_
-2	seguir	seguir	VERB	_	VerbForm=Inf	4	advcl	O	_
+1	A	a	ADP	_	_	4	obl	O	MWEPOS=ADV
+2	seguir	seguir	VERB	_	VerbForm=Inf	1	fixed	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	seguem	seguir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 5	respectivamente	respectivamente	ADV	_	_	4	advmod	O	_
@@ -332871,8 +332871,8 @@
 32	,	,	PUNCT	_	_	37	punct	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
 34	seção	seção	NOUN	_	Gender=Fem|Number=Sing	37	nsubj	O	_
-35	a	a	SCONJ	_	_	36	mark	O	_
-36	seguir	seguir	VERB	_	VerbForm=Inf	37	advcl	O	_
+35	a	a	ADP	_	_	37	obl	O	MWEPOS=ADV
+36	seguir	seguir	VERB	_	VerbForm=Inf	35	fixed	O	_
 37	abordará	abordar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	21	advcl	O	_
 38	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	39	det	O	_
 39	características	característica	NOUN	_	Gender=Fem|Number=Plur	37	obj	O	_
@@ -334573,8 +334573,8 @@
 24	em	em	ADP	_	_	26	case	O	_
 25	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	26	det	O	_
 26	figuras	figura	NOUN	_	Gender=Fem|Number=Plur	23	obl	O	_
-27	a	a	SCONJ	_	_	28	mark	O	_
-28	seguir	seguir	VERB	_	VerbForm=Inf	23	advcl	O	_
+27	a	a	ADP	_	_	23	obl	O	MWEPOS=ADV
+28	seguir	seguir	VERB	_	VerbForm=Inf	27	fixed	O	_
 29	a	a	ADP	_	_	32	mark	O	_
 30	fim	fim	NOUN	_	Gender=Masc|Number=Sing	29	fixed	O	_
 31	de	de	ADP	_	_	29	fixed	O	_
@@ -335150,8 +335150,8 @@
 21	de	de	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
 23	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	20	nmod	B=INSTRUMENTO	_
-24	a	a	SCONJ	_	_	25	mark	O	_
-25	seguir	seguir	VERB	_	VerbForm=Inf	17	advcl	O	_
+24	a	a	ADP	_	_	17	obl	O	MWEPOS=ADV
+25	seguir	seguir	VERB	_	VerbForm=Inf	24	fixed	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
 27	município	município	NOUN	_	Gender=Masc|Number=Sing	25	obj	O	_
 28	de	de	ADP	_	_	29	case	O	_
@@ -337818,8 +337818,8 @@
 # sent_id = 411-20150702-MONOGRAFIA_0-176
 1	Trataremos	tratar	VERB	_	Mood=Ind|Number=Plur|Person=1|Tense=Fut|VerbForm=Fin	0	root	O	_
 2	,	,	PUNCT	_	_	1	punct	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	seguir	seguir	VERB	_	VerbForm=Inf	1	advcl	O	_
+3	a	a	ADP	_	_	1	obl	O	MWEPOS=ADV
+4	seguir	seguir	VERB	_	VerbForm=Inf	3	fixed	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6-7	dos	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	8	case	O	_
@@ -343107,8 +343107,8 @@
 20	,	,	PUNCT	_	_	5	punct	O	_
 21	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	22	cop	O	_
 22	descritos	descrito	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-23	a	a	SCONJ	_	_	24	mark	O	_
-24	seguir	seguir	VERB	_	VerbForm=Inf	22	advcl	O	_
+23	a	a	ADP	_	_	22	obl	O	MWEPOS=ADV
+24	seguir	seguir	VERB	_	VerbForm=Inf	23	fixed	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	24	obl	O	_
 27	com	com	ADP	_	_	29	case	O	_
