commit 226aa79550f0842f205240d0c43935cfd428d81f
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Aug 18 19:22:06 2021 -0300

    Regras para B=CAMPO

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 529be08..b211860 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -8,7 +8,7 @@
 # text = A Bacia de Pelotas é a mais meridional na costa brasileira (Figura 1) e apresenta poucos trabalhos publicados a seu respeito se comparada às outras bacias de margem continental brasileiras.
 # sent_id = 247-20140910-MONOGRAFIA_0-2
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
-2	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	8	nsubj	O	_
+2	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	8	nsubj	B=BACIA	_
 3	de	de	ADP	_	_	2	flat:name	O	_
 4	Pelotas	Pelotas	PROPN	_	Number=Sing	2	flat:name	B=BACIA	_
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	cop	O	_
@@ -76,7 +76,7 @@
 24-25	da	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	23	nmod	O	_
+26	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	23	nmod	B=BACIA	_
 27	de	de	ADP	_	_	26	flat:name	O	_
 28	Pelotas	Pelotas	PROPN	_	Number=Sing	26	flat:name	B=BACIA	_
 29	verificando	verificar	VERB	_	VerbForm=Ger	5	advcl	O	_
@@ -109,7 +109,7 @@
 8-9	da	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	6	nmod	O	_
+10	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	6	nmod	B=BACIA	_
 11	de	de	ADP	_	_	10	flat:name	O	_
 12	Pelotas	Pelotas	PROPN	_	Number=Sing	10	flat:name	B=BACIA	_
 13	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	14	aux:pass	O	_
@@ -212,7 +212,7 @@
 17-18	da	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
-19	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	15	nmod	O	_
+19	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	15	nmod	B=BACIA	_
 20	de	de	ADP	_	_	19	flat:name	O	_
 21	Pelotas	Pelotas	PROPN	_	Number=Sing	19	flat:name	B=BACIA	_
 22	.	.	PUNCT	_	_	5	punct	O	_
@@ -281,7 +281,7 @@
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	4	flat:name	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	flat:name	O	_
-7	Bacia	bacia	PROPN	_	Number=Sing	4	flat:name	O	_
+7	Bacia	Bacia	PROPN	_	Number=Sing	4	flat:name	B=BACIA	_
 8	de	de	ADP	_	_	4	flat:name	O	_
 9	Pelotas	Pelotas	PROPN	_	Number=Sing	4	flat:name	B=BACIA	_
 
@@ -318,7 +318,7 @@
 19-20	da	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
-21	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	17	nmod	O	_
+21	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	17	nmod	B=BACIA	_
 22	de	de	ADP	_	_	21	flat:name	O	_
 23	Pelotas	Pelotas	PROPN	_	Number=Sing	21	flat:name	B=BACIA	_
 24	e	e	CCONJ	_	_	26	cc	O	_
@@ -429,7 +429,7 @@
 29	emersa	emerso	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 30	e	e	CCONJ	_	_	32	cc	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
-32	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	28	conj	O	_
+32	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	28	conj	B=BACIA	_
 33	de	de	ADP	_	_	32	flat:name	O	_
 34	Pelotas	Pelotas	PROPN	_	Number=Sing	32	flat:name	B=BACIA	_
 35-36	na	_	_	_	_	_	_	_	_
@@ -517,7 +517,7 @@
 27	sísmicas	sísmico	ADJ	_	Gender=Fem|Number=Plur	26	amod	O	_
 28	referente	referente	ADV	_	_	30	advmod	O	_
 29	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
-30	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	26	nmod	O	_
+30	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	26	nmod	B=BACIA	_
 31	de	de	ADP	_	_	30	flat:name	O	_
 32	Pelotas	Pelotas	PROPN	_	Number=Sing	30	flat:name	B=BACIA	_
 33	,	,	PUNCT	_	_	34	punct	O	_
@@ -1432,7 +1432,7 @@
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	24	nmod	O	_
+28	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	24	nmod	B=BACIA	_
 29	de	de	ADP	_	_	28	flat:name	O	_
 30	Pelotas	Pelotas	PROPN	_	Number=Sing	28	flat:name	B=BACIA	_
 31	,	,	PUNCT	_	_	24	punct	O	_
@@ -2715,7 +2715,7 @@
 # text = A Bacia Sedimentar do Paraná (Figura 6) se localiza no Continente Sul-Americano nas porções territoriais do Brasil meridional, Paraguai oriental, nordeste da Argentina e norte do Uruguai, totalizando uma área próxima de 1,5 milhão de quilômetros quadrados.
 # sent_id = 247-20140910-MONOGRAFIA_0-82
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
-2	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	12	nsubj	O	_
+2	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	12	nsubj	B=BACIA	_
 3	Sedimentar	Sedimentar	PROPN	_	Number=Sing	2	flat:name	O	_
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	2	flat:name	O	_
@@ -2780,7 +2780,7 @@
 5	DE	de	ADP	_	_	7	case	O	_
 6	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	BACIA	bacia	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
-8	DO	DO	PROPN	_	Number=Sing	7	flat:name	O	_
+8	DO	DO	PROPN	_	Number=Sing	7	flat:name	O	sentenciação
 
 # text = PARANÁ
 # sent_id = 247-20140910-MONOGRAFIA_0-85
@@ -81903,7 +81903,7 @@
 27-28	no	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
-29	campo	campo	NOUN	_	Gender=Masc|Number=Sing	26	obl	B=BACIA	_
+29	campo	campo	NOUN	_	Gender=Masc|Number=Sing	26	obl	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	29	nmod	B=CAMPO	_
 32	e	e	CCONJ	_	_	33	cc	O	_
@@ -82538,7 +82538,7 @@
 18	localizada	localizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	14	acl	O	_
 19	sobre	sobre	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	18	obl	B=BACIA	_
+21	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	18	obl	B=CAMPO	_
 22	de	de	ADP	_	_	21	flat:name	O	_
 23	Namorado	Namorado	PROPN	_	Number=Sing	21	flat:name	B=CAMPO	_
 24	,	,	PUNCT	_	_	18	punct	O	_
@@ -82552,7 +82552,7 @@
 31-32	no	_	_	_	_	_	_	_	_
 31	em	em	ADP	_	_	33	case	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
-33	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	25	obl	B=BACIA	_
+33	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	25	obl	B=CAMPO	_
 34	de	de	ADP	_	_	33	flat:name	O	_
 35	Garoupa	Garoupa	PROPN	_	Number=Sing	33	flat:name	B=CAMPO	_
 36	.	.	PUNCT	_	_	11	punct	O	_
@@ -82589,7 +82589,7 @@
 26-27	no	_	_	_	_	_	_	_	_
 26	em	em	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
-28	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	20	obl	B=BACIA	_
+28	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	20	obl	B=CAMPO	_
 29	de	de	ADP	_	_	28	flat:name	O	_
 30	Barracuda	Barracuda	PROPN	_	Number=Sing	28	flat:name	B=CAMPO	_
 31	e	e	CCONJ	_	_	33	cc	O	_
@@ -82604,7 +82604,7 @@
 40-41	no	_	_	_	_	_	_	_	_
 40	em	em	ADP	_	_	42	case	O	_
 41	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	42	det	O	_
-42	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	34	obl	B=BACIA	_
+42	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	34	obl	B=CAMPO	_
 43	de	de	ADP	_	_	42	flat:name	O	_
 44	Marlim	Marlim	PROPN	_	Number=Sing	42	flat:name	B=CAMPO	_
 45	,	,	PUNCT	_	_	48	punct	O	_
@@ -84994,7 +84994,7 @@
 16-17	do	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	15	nmod	B=BACIA	_
+18	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	15	nmod	B=CAMPO	_
 19	de	de	ADP	_	_	18	flat:name	O	_
 20	Garoupa	Garoupa	PROPN	_	Number=Sing	18	flat:name	B=CAMPO	_
 21	.	.	PUNCT	_	_	8	punct	O	_
@@ -85020,7 +85020,7 @@
 17-18	no	_	_	_	_	_	_	_	_
 17	em	em	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
-19	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	11	obl	B=BACIA	_
+19	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	11	obl	B=CAMPO	_
 20	de	de	ADP	_	_	19	flat:name	O	_
 21	Enchova	Enchova	PROPN	_	Number=Sing	19	flat:name	B=CAMPO	_
 22	.	.	PUNCT	_	_	11	punct	O	_
@@ -85485,7 +85485,7 @@
 # text = Os campos de Marlim e Barracuda foram os escolhidos para armazenar todo o CO2 capturado.
 # sent_id = 161-20150810-MONOGRAFIA_0-501
 1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
-2	campos	campo	NOUN	_	Gender=Masc|Number=Plur	9	nsubj	B=BACIA	_
+2	campos	campo	NOUN	_	Gender=Masc|Number=Plur	9	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 5	e	e	CCONJ	_	_	6	cc	O	_
@@ -85599,11 +85599,11 @@
 22	a	a	ADP	_	_	20	fixed	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	25	det	O	_
 24	demais	demais	DET	_	Gender=Masc|Number=Plur|PronType=Ind	25	det	O	_
-25	campos	campo	NOUN	_	Gender=Masc|Number=Plur	12	obl	B=BACIA	_
+25	campos	campo	NOUN	_	Gender=Masc|Number=Plur	12	obl	O	_
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	25	nmod	O	_
+28	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	25	nmod	B=BACIA	_
 29	de	de	ADP	_	_	28	flat:name	O	_
 30	Campos	Campos	PROPN	_	Number=Sing	28	flat:name	O	_
 31	.	.	PUNCT	_	_	12	punct	O	_
@@ -85625,7 +85625,7 @@
 
 # text = Campo de Marlim
 # sent_id = 161-20150810-MONOGRAFIA_0-506
-1	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	0	root	B=BACIA	_
+1	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	0	root	B=CAMPO	_
 2	de	de	ADP	_	_	1	flat:name	O	_
 3	Marlim	Marlim	PROPN	_	Number=Sing	1	flat:name	B=CAMPO	_
 
@@ -85646,7 +85646,7 @@
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	campo	campo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+5	campo	campo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	5	nmod	B=CAMPO	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	aux:pass	O	_
@@ -85747,7 +85747,7 @@
 5-6	do	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-7	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	B=BACIA	_
+7	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	7	nmod	B=CAMPO	_
 10	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	11	nsubj	O	_
@@ -85766,7 +85766,7 @@
 22-23	no	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	campo	campo	NOUN	_	Gender=Masc|Number=Sing	21	obl	B=BACIA	_
+24	campo	campo	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	Voador	Voador	PROPN	_	Gender=Masc|Number=Sing	24	nmod	B=CAMPO	_
 27	.	.	PUNCT	_	_	2	punct	O	_
@@ -86123,7 +86123,7 @@
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 6	de	de	ADP	_	_	5	flat:name	O	_
 7	Marlim	Marlim	PROPN	_	Number=Sing	5	flat:name	B=CAMPO	_
 8	abriga	abrigar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -86227,7 +86227,7 @@
 11-12	do	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=BACIA	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	.	.	PUNCT	_	_	7	punct	O	_
@@ -86253,7 +86253,7 @@
 
 # text = Campo de Barracuda
 # sent_id = 161-20150810-MONOGRAFIA_0-528
-1	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	0	root	B=BACIA	_
+1	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	0	root	B=CAMPO	_
 2	de	de	ADP	_	_	1	flat:name	O	_
 3	Barracuda	Barracuda	PROPN	_	Number=Sing	1	flat:name	B=CAMPO	_
 
@@ -86274,7 +86274,7 @@
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 6	de	de	ADP	_	_	5	flat:name	O	_
 7	Barracuda	Barracuda	PROPN	_	Number=Sing	5	flat:name	B=CAMPO	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	aux:pass	O	_
@@ -86304,7 +86304,7 @@
 32	localizada	localizar	ADJ	_	Gender=Fem|Number=Sing	9	advcl	O	_
 33	sobre	sobre	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
-35	campo	campo	NOUN	_	Gender=Masc|Number=Sing	32	obl	B=BACIA	_
+35	campo	campo	NOUN	_	Gender=Masc|Number=Sing	32	obl	O	_
 36	de	de	ADP	_	_	37	case	O	_
 37	Caratinga	Caratinga	PROPN	_	Gender=Unsp|Number=Sing	35	nmod	B=CAMPO	_
 38	,	,	PUNCT	_	_	32	punct	O	_
@@ -86334,7 +86334,7 @@
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	B=BACIA	_
+10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	Caratinga	Caratinga	PROPN	_	Gender=Unsp|Number=Sing	10	nmod	B=CAMPO	_
 13	.	.	PUNCT	_	_	5	punct	O	_
@@ -86375,7 +86375,7 @@
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+6	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 7	de	de	ADP	_	_	6	flat:name	O	_
 8	Barracuda	Barracuda	PROPN	_	Number=Sing	6	flat:name	B=CAMPO	_
 9	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	10	cop	O	_
@@ -86417,7 +86417,7 @@
 42	compartilhado	compartilhar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	37	acl	O	_
 43	com	com	ADP	_	_	45	case	O	_
 44	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	45	det	O	_
-45	campo	campo	NOUN	_	Gender=Masc|Number=Sing	42	obl	B=BACIA	_
+45	campo	campo	NOUN	_	Gender=Masc|Number=Sing	42	obl	O	_
 46	de	de	ADP	_	_	47	case	O	_
 47	Caratinga	Caratinga	PROPN	_	Gender=Unsp|Number=Sing	45	nmod	B=CAMPO	_
 48	)	)	PUNCT	_	_	37	punct	O	_
@@ -86494,7 +86494,7 @@
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 6	de	de	ADP	_	_	5	flat:name	O	_
 7	Barracuda	Barracuda	PROPN	_	Number=Sing	5	flat:name	B=CAMPO	_
 8	abriga	abrigar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -86584,7 +86584,7 @@
 11-12	do	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=BACIA	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	.	.	PUNCT	_	_	7	punct	O	_
@@ -86632,7 +86632,7 @@
 9-10	do	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	B=BACIA	_
+11	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	11	nmod	B=CAMPO	_
 
@@ -86705,7 +86705,7 @@
 15-16	nos	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
 16	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	17	det	O	_
-17	campos	campo	NOUN	_	Gender=Masc|Number=Plur	8	obl	B=BACIA	_
+17	campos	campo	NOUN	_	Gender=Masc|Number=Plur	8	obl	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	17	nmod	B=CAMPO	_
 20	e	e	CCONJ	_	_	21	cc	O	_
@@ -86860,7 +86860,7 @@
 6-7	nos	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
-8	campos	campo	NOUN	_	Gender=Masc|Number=Plur	5	obl	B=BACIA	_
+8	campos	campo	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	8	nmod	B=CAMPO	_
 11	e	e	CCONJ	_	_	12	cc	O	_
@@ -87138,13 +87138,13 @@
 37	Tanto	tanto	ADV	_	_	40	advmod	O	_
 38	para	para	ADP	_	_	40	case	O	_
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
-40	campo	campo	NOUN	_	Gender=Masc|Number=Sing	62	obl	B=BACIA	_
+40	campo	campo	NOUN	_	Gender=Masc|Number=Sing	62	obl	O	_
 41	de	de	ADP	_	_	42	case	O	_
 42	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	40	nmod	B=CAMPO	_
 43	quanto	quanto	CCONJ	_	_	46	cc	O	_
 44	para	para	ADP	_	_	46	case	O	_
 45	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
-46	campo	campo	NOUN	_	Gender=Masc|Number=Sing	40	conj	B=BACIA	_
+46	campo	campo	NOUN	_	Gender=Masc|Number=Sing	40	conj	O	_
 47	de	de	ADP	_	_	48	case	O	_
 48	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	46	nmod	B=CAMPO	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
@@ -87201,7 +87201,7 @@
 3-4	dos	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
-5	campos	campo	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=BACIA	_
+5	campos	campo	NOUN	_	Gender=Masc|Number=Plur	2	nmod	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	5	nmod	B=CAMPO	_
 8	e	e	CCONJ	_	_	9	cc	O	_
@@ -87314,7 +87314,7 @@
 9	injetividade	injetividade	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 10	para	para	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
-12	campos	campo	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=BACIA	_
+12	campos	campo	NOUN	_	Gender=Masc|Number=Plur	6	nmod	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	12	nmod	B=CAMPO	_
 15	e	e	CCONJ	_	_	16	cc	O	_
@@ -87580,7 +87580,7 @@
 9-10	do	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	campo	campo	NOUN	_	Gender=Masc|Number=Sing	8	nmod	B=BACIA	_
+11	campo	campo	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	11	nmod	B=CAMPO	_
 14	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	cop	O	_
@@ -87599,7 +87599,7 @@
 27-28	do	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
-29	campo	campo	NOUN	_	Gender=Masc|Number=Sing	26	nmod	B=BACIA	_
+29	campo	campo	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
 30	de	de	ADP	_	_	31	case	O	_
 31	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	29	nmod	B=CAMPO	_
 32	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	33	aux:pass	O	_
@@ -89131,7 +89131,7 @@
 1	Tanto	tanto	ADV	_	_	19	advmod	O	_
 2	para	para	ADP	_	_	4	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	campos	campo	NOUN	_	Gender=Masc|Number=Plur	11	obl	B=BACIA	_
+4	campos	campo	NOUN	_	Gender=Masc|Number=Plur	11	obl	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	4	nmod	B=CAMPO	_
 7	e	e	CCONJ	_	_	9	cc	O	_
@@ -89835,7 +89835,7 @@
 17	extra	extra	ADJ	_	Gender=Fem|Number=Sing	16	amod	O	_
 18	para	para	ADP	_	_	20	case	O	_
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
-20	campos	campo	NOUN	_	Gender=Masc|Number=Plur	14	nmod	B=BACIA	_
+20	campos	campo	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 21	de	de	ADP	_	_	22	case	O	_
 22	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	20	nmod	B=CAMPO	_
 23	e	e	CCONJ	_	_	24	cc	O	_
@@ -90025,7 +90025,7 @@
 21-22	nos	_	_	_	_	_	_	_	_
 21	em	em	ADP	_	_	23	case	O	_
 22	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
-23	campos	campo	NOUN	_	Gender=Masc|Number=Plur	20	obl	B=BACIA	_
+23	campos	campo	NOUN	_	Gender=Masc|Number=Plur	20	obl	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	23	nmod	B=CAMPO	_
 26	e	e	CCONJ	_	_	27	cc	O	_
@@ -90365,7 +90365,7 @@
 12	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 13	sobre	sobre	ADP	_	_	15	case	O	_
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
-15	campos	campo	NOUN	_	Gender=Masc|Number=Plur	8	nmod	B=BACIA	_
+15	campos	campo	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	15	nmod	B=CAMPO	_
 18	e	e	CCONJ	_	_	19	cc	O	_
@@ -90971,7 +90971,7 @@
 12-13	nos	_	_	_	_	_	_	_	_
 12	em	em	ADP	_	_	14	case	O	_
 13	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	14	det	O	_
-14	campos	campo	NOUN	_	Gender=Masc|Number=Plur	10	obl	B=BACIA	_
+14	campos	campo	NOUN	_	Gender=Masc|Number=Plur	10	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 17-18	da	_	_	_	_	_	_	_	_
@@ -91122,7 +91122,7 @@
 33-34	nos	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	35	case	O	_
 34	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	35	det	O	_
-35	campos	campo	NOUN	_	Gender=Masc|Number=Plur	20	obl	B=BACIA	_
+35	campos	campo	NOUN	_	Gender=Masc|Number=Plur	20	obl	O	_
 36	de	de	ADP	_	_	37	case	O	_
 37	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
 38	em	em	ADP	_	_	39	case	O	_
@@ -95106,7 +95106,7 @@
 10-11	nos	_	_	_	_	_	_	_	_
 10	em	em	ADP	_	_	12	case	O	_
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
-12	campos	campo	NOUN	_	Gender=Masc|Number=Plur	6	obl	B=BACIA	_
+12	campos	campo	NOUN	_	Gender=Masc|Number=Plur	6	obl	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	12	nmod	B=CAMPO	_
 15	e	e	CCONJ	_	_	16	cc	O	_
@@ -96404,7 +96404,7 @@
 8-9	dos	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
-10	campos	campo	NOUN	_	Gender=Masc|Number=Plur	7	nmod	B=BACIA	_
+10	campos	campo	NOUN	_	Gender=Masc|Number=Plur	7	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	10	nmod	B=CAMPO	_
 13	e	e	CCONJ	_	_	14	cc	O	_
@@ -97326,7 +97326,7 @@
 11-12	dos	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
-13	campos	campo	NOUN	_	Gender=Masc|Number=Plur	10	nmod	B=BACIA	_
+13	campos	campo	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	e	e	CCONJ	_	_	17	cc	O	_
@@ -97725,7 +97725,7 @@
 21-22	no	_	_	_	_	_	_	_	_
 21	em	em	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
-23	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	20	obl	B=BACIA	_
+23	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	20	obl	B=CAMPO	_
 24	de	de	ADP	_	_	23	flat:name	O	_
 25	Jubarte	Jubarte	PROPN	_	Number=Sing	23	flat:name	B=CAMPO	_
 26	(	(	PUNCT	_	_	27	punct	O	_
@@ -97842,9 +97842,9 @@
 42-43	no	_	_	_	_	_	_	_	_
 42	em	em	ADP	_	_	44	case	O	_
 43	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	44	det	O	_
-44	campo	campo	NOUN	_	Gender=Masc|Number=Sing	37	obl	B=BACIA	_
+44	campo	campo	NOUN	_	Gender=Masc|Number=Sing	37	obl	O	_
 45	de	de	ADP	_	_	46	case	O	_
-46	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	44	nmod	O	_
+46	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	44	nmod	B=CAMPO	_
 47	,	,	PUNCT	_	_	37	punct	O	_
 48-49	na	_	_	_	_	_	_	_	_
 48	em	em	ADP	_	_	50	case	O	_
@@ -97867,7 +97867,7 @@
 65-66	no	_	_	_	_	_	_	_	_
 65	em	em	ADP	_	_	67	case	O	_
 66	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	67	det	O	_
-67	campo	campo	NOUN	_	Gender=Masc|Number=Sing	60	obl	B=BACIA	_
+67	campo	campo	NOUN	_	Gender=Masc|Number=Sing	60	obl	O	_
 68	de	de	ADP	_	_	69	case	O	_
 69	Baleia	Baleia	PROPN	_	Gender=Fem|Number=Sing	67	nmod	B=CAMPO	_
 70	Azul	Azul	PROPN	_	Number=Sing	69	flat:name	I=CAMPO	_
@@ -97894,7 +97894,7 @@
 90-91	no	_	_	_	_	_	_	_	_
 90	em	em	ADP	_	_	92	case	O	_
 91	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	92	det	O	_
-92	campo	campo	NOUN	_	Gender=Masc|Number=Sing	85	obl	B=BACIA	_
+92	campo	campo	NOUN	_	Gender=Masc|Number=Sing	85	obl	O	_
 93	de	de	ADP	_	_	94	case	O	_
 94	Sapinhoá	Sapinhoá	PROPN	_	Gender=Masc|Number=Sing	92	nmod	B=CAMPO	_
 95	,	,	PUNCT	_	_	85	punct	O	_
@@ -98643,7 +98643,7 @@
 19-20	no	_	_	_	_	_	_	_	_
 19	em	em	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	17	obl	B=BACIA	_
+21	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	17	obl	B=CAMPO	_
 22	de	de	ADP	_	_	21	flat:name	O	_
 23	Jubarte	Jubarte	PROPN	_	Number=Sing	21	flat:name	B=CAMPO	_
 24	,	,	PUNCT	_	_	17	punct	O	_
@@ -98723,7 +98723,7 @@
 23-24	do	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
-25	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	19	nmod	B=BACIA	_
+25	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	19	nmod	B=CAMPO	_
 26	de	de	ADP	_	_	25	flat:name	O	_
 27	Merluza	Merluza	PROPN	_	Number=Sing	25	flat:name	B=CAMPO	_
 28	.	.	PUNCT	_	_	6	punct	O	_
@@ -98782,7 +98782,7 @@
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	3	nmod	B=BACIA	_
+6	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	3	nmod	B=CAMPO	_
 7	de	de	ADP	_	_	6	flat:name	O	_
 8	Mexilhão	Mexilhão	PROPN	_	Number=Sing	6	flat:name	B=CAMPO	_
 9	,	,	PUNCT	_	_	3	punct	O	_
@@ -99818,7 +99818,7 @@
 # text = O campo de Marlim Sul, na Bacia de Campos, foi o de maior produção diária de petróleo.
 # sent_id = 334-20150521-MONOGRAFIA_0-57
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
-2	campo	campo	NOUN	_	Gender=Masc|Number=Sing	14	nsubj	B=BACIA	_
+2	campo	campo	NOUN	_	Gender=Masc|Number=Sing	14	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=CAMPO	_
 5	Sul	Sul	PROPN	_	Number=Sing	4	flat:name	I=CAMPO	_
@@ -99851,9 +99851,9 @@
 7	natural	natural	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
 8	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	10	cop	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	0	root	B=BACIA	_
+10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 11	de	de	ADP	_	_	12	case	O	_
-12	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	10	nmod	O	_
+12	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	10	nmod	B=CAMPO	_
 13	,	,	PUNCT	_	_	16	punct	O	_
 14-15	na	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
@@ -104310,9 +104310,9 @@
 8-9	no	_	_	_	_	_	_	_	_
 8	em	em	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	obl	B=BACIA	_
+10	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
 11	de	de	ADP	_	_	12	case	O	_
-12	Tahiti	Tahiti	PROPN	_	Gender=Masc|Number=Sing	10	nmod	O	_
+12	Tahiti	Tahiti	PROPN	_	Gender=Masc|Number=Sing	10	nmod	B=CAMPO	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	400	400	NUM	_	NumType=Card	12	nmod:appos	O	_
 15	a	a	ADP	_	_	16	case	O	_
@@ -108206,7 +108206,7 @@
 7-8	no	_	_	_	_	_	_	_	_
 7	em	em	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	B=BACIA	_
+9	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	9	nmod	B=CAMPO	_
 12	,	,	PUNCT	_	_	15	punct	O	_
@@ -108442,7 +108442,7 @@
 22-23	no	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
-24	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	obl	B=BACIA	_
+24	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	24	nmod	B=CAMPO	_
 27	(	(	PUNCT	_	_	28	punct	O	_
@@ -108866,9 +108866,9 @@
 11-12	no	_	_	_	_	_	_	_	_
 11	em	em	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	obl	B=BACIA	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	13	nmod	O	_
+15	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	(	(	PUNCT	_	_	17	punct	O	_
 17	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	15	appos	O	_
 18	25	25	NUM	_	NumType=Card	17	nummod	O	_
@@ -109019,9 +109019,9 @@
 25-26	no	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	20	obl	B=BACIA	_
+27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	20	obl	O	_
 28	de	de	ADP	_	_	29	case	O	_
-29	Tordis	Tordis	PROPN	_	Gender=Unsp|Number=Sing	27	nmod	O	_
+29	Tordis	Tordis	PROPN	_	Gender=Unsp|Number=Sing	27	nmod	B=CAMPO	_
 30	.	.	PUNCT	_	_	20	punct	O	_
 
 # text = Assim como o de Troll, este separador trabalha em lâminas de água mais rasas (200 metros) que as encontradas no Pré-sal brasileiro.
@@ -109258,9 +109258,9 @@
 13-14	no	_	_	_	_	_	_	_	_
 13	em	em	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
-15	campo	campo	NOUN	_	Gender=Masc|Number=Sing	12	obl	B=BACIA	_
+15	campo	campo	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	Pazflor	Pazflor	PROPN	_	Gender=Unsp|Number=Sing	15	nmod	O	_
+17	Pazflor	Pazflor	PROPN	_	Gender=Unsp|Number=Sing	15	nmod	B=CAMPO	_
 18	,	,	PUNCT	_	_	21	punct	O	_
 19	a	a	ADP	_	_	21	case	O	_
 20	150	150	NUM	_	NumType=Card	21	nummod	O	_
@@ -110660,9 +110660,9 @@
 19	opção	opção	NOUN	_	Gender=Fem|Number=Sing	15	conj	O	_
 20	para	para	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	campo	campo	NOUN	_	Gender=Masc|Number=Sing	19	nmod	B=BACIA	_
+22	campo	campo	NOUN	_	Gender=Masc|Number=Sing	19	nmod	O	_
 23	de	de	ADP	_	_	24	case	O	_
-24	Asgard	Asgard	PROPN	_	Gender=Masc|Number=Sing	22	nmod	O	_
+24	Asgard	Asgard	PROPN	_	Gender=Masc|Number=Sing	22	nmod	B=CAMPO	_
 25	.	.	PUNCT	_	_	15	punct	O	_
 
 # text = Compressores na faixa de 12-13
@@ -110841,9 +110841,9 @@
 25-26	no	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	21	nmod	B=BACIA	_
+27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 28	de	de	ADP	_	_	29	case	O	_
-29	Gullfaks	Gullfaks	PROPN	_	Gender=Unsp|Number=Sing	27	nmod	O	_
+29	Gullfaks	Gullfaks	PROPN	_	Gender=Unsp|Number=Sing	27	nmod	B=CAMPO	_
 30	(	(	PUNCT	_	_	31	punct	O	_
 31	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	10	parataxis	O	_
 32	31	31	NUM	_	NumType=Card	31	nummod	O	_
@@ -110970,9 +110970,9 @@
 11-12	no	_	_	_	_	_	_	_	_
 11	em	em	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	obl	B=BACIA	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	13	nmod	O	_
+15	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	,	,	PUNCT	_	_	18	punct	O	_
 17	sendo	ser	AUX	_	VerbForm=Ger	18	aux:pass	O	_
 18	usada	usar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	6	advcl	O	_
@@ -110980,9 +110980,9 @@
 20-21	no	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	campo	campo	NOUN	_	Gender=Masc|Number=Sing	18	obl	B=BACIA	_
+22	campo	campo	NOUN	_	Gender=Masc|Number=Sing	18	obl	O	_
 23	de	de	ADP	_	_	24	case	O	_
-24	Tordis	Tordis	PROPN	_	Gender=Unsp|Number=Sing	22	nmod	O	_
+24	Tordis	Tordis	PROPN	_	Gender=Unsp|Number=Sing	22	nmod	B=CAMPO	_
 25	em	em	ADP	_	_	26	case	O	_
 26	2007	2007	NUM	_	NumType=Card	22	nummod	O	_
 27	,	,	PUNCT	_	_	18	punct	O	_
@@ -111015,7 +111015,7 @@
 11-12	no	_	_	_	_	_	_	_	_
 11	em	em	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	obl	B=BACIA	_
+13	campo	campo	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=CAMPO	_
 16	,	,	PUNCT	_	_	17	punct	O	_
@@ -111080,7 +111080,7 @@
 37-38	no	_	_	_	_	_	_	_	_
 37	em	em	ADP	_	_	39	case	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	det	O	_
-39	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	32	obl	B=BACIA	_
+39	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	32	obl	B=CAMPO	_
 40	de	de	ADP	_	_	39	flat:name	O	_
 41	Roncador	Roncador	PROPN	_	Number=Sing	39	flat:name	B=CAMPO	_
 42-43	pela	_	_	_	_	_	_	_	_
@@ -111402,9 +111402,9 @@
 13	submarinos	submarino	ADJ	_	Gender=Masc|Number=Plur	12	amod	O	_
 14	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	cop	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	campo	campo	NOUN	_	Gender=Masc|Number=Sing	0	root	B=BACIA	_
+16	campo	campo	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 17	de	de	ADP	_	_	18	case	O	_
-18	Snøhvit	Snøhvit	PROPN	_	Gender=Unsp|Number=Sing	16	nmod	O	_
+18	Snøhvit	Snøhvit	PROPN	_	Gender=Unsp|Number=Sing	16	nmod	B=CAMPO	_
 19	localizado	localizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	16	acl	O	_
 20-21	no	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	22	case	O	_
@@ -113279,9 +113279,9 @@
 18	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	19	nsubj	O	_
 19	liga	ligar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	acl:relcl	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	campo	campo	NOUN	_	Gender=Masc|Number=Sing	19	obj	B=BACIA	_
+21	campo	campo	NOUN	_	Gender=Masc|Number=Sing	19	obj	O	_
 22	de	de	ADP	_	_	23	case	O	_
-23	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	21	nmod	O	_
+23	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	21	nmod	B=CAMPO	_
 24-25	à	_	_	_	_	_	_	_	_
 24	a	a	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
@@ -113310,14 +113310,14 @@
 48	)	)	PUNCT	_	_	45	punct	O	_
 49	e	e	CCONJ	_	_	51	cc	O	_
 50	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	51	det	O	_
-51	campos	campo	NOUN	_	Gender=Masc|Number=Plur	15	conj	B=BACIA	_
+51	campos	campo	NOUN	_	Gender=Masc|Number=Plur	15	conj	O	_
 52-53	da	_	_	_	_	_	_	_	_
 52	de	de	ADP	_	_	54	case	O	_
 53	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	54	det	O	_
 54	Noruega	Noruega	PROPN	_	Gender=Fem|Number=Sing	51	nmod	O	_
 55	,	,	PUNCT	_	_	57	punct	O	_
 56	como	como	ADP	_	_	57	case	O	_
-57	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	51	nmod:appos	O	_
+57	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	51	nmod:appos	B=CAMPO	_
 58	,	,	PUNCT	_	_	59	punct	O	_
 59	Snøhvit	Snøhvit	PROPN	_	Gender=Masc|Number=Sing	57	conj	O	_
 60	,	,	PUNCT	_	_	61	punct	O	_
@@ -114817,9 +114817,9 @@
 5	maior	grande	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
 6	para	para	ADP	_	_	8	case	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	campo	campo	NOUN	_	Gender=Masc|Number=Sing	5	obl	B=BACIA	_
+8	campo	campo	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
-10	Júpiter	Júpiter	PROPN	_	Gender=Masc|Number=Sing	8	nmod	O	_
+10	Júpiter	Júpiter	PROPN	_	Gender=Masc|Number=Sing	8	nmod	B=CAMPO	_
 11	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	13	obl	O	_
 12	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	aux:pass	O	_
 13	atingidas	atingir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	8	acl:relcl	O	_
@@ -117149,9 +117149,9 @@
 12-13	no	_	_	_	_	_	_	_	_
 12	em	em	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	B=BACIA	_
+14	campo	campo	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
+16	Lula	Lula	PROPN	_	Gender=Masc|Number=Sing	14	nmod	B=CAMPO	_
 17	.	.	PUNCT	_	_	19	punct	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
 19	Fonte:	Fonte:	PROPN	_	Gender=Masc|Number=Sing	1	appos	O	_
@@ -168169,12 +168169,12 @@
 5-6	no	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-7	campo	campo	NOUN	_	Gender=Masc|Number=Sing	16	obl	B=BACIA	_
+7	campo	campo	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
 8	de	de	ADP	_	_	9	case	O	_
-9	Carmopólis	Carmopólis	PROPN	_	Gender=Unsp|Number=Sing	7	nmod	O	_
+9	Carmopólis	Carmopólis	PROPN	_	Gender=Unsp|Number=Sing	7	nmod	B=CAMPO	_
 10	,	,	PUNCT	_	_	7	punct	O	_
 11	em	em	ADP	_	_	12	case	O	_
-12	Sergipe	Sergipe	PROPN	_	Gender=Masc|Number=Sing	7	nmod	B=BACIA	_
+12	Sergipe	Sergipe	PROPN	_	Gender=Masc|Number=Sing	7	nmod	O	_
 13	–	–	PUNCT	_	_	12	punct	O	_
 14	Brasil	Brasil	PROPN	_	Gender=Masc|Number=Sing	12	flat:name	O	_
 15	,	,	PUNCT	_	_	12	punct	O	_
@@ -210955,7 +210955,7 @@
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	B=BACIA	_
+5	Campo	Campo	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	Magnético	Magnético	PROPN	_	Number=Sing	5	flat:name	O	_
 7	Anômalo	Anômalo	PROPN	_	Number=Sing	5	flat:name	O	_
 
@@ -281655,7 +281655,7 @@
 25-26	no	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	obl	B=BACIA	_
+27	campo	campo	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 28	de	de	ADP	_	_	29	case	O	_
 29	Xaréu	Xaréu	PROPN	_	Gender=Masc|Number=Sing	27	nmod	B=CAMPO	_
 30	em	em	ADP	_	_	31	case	O	_
