commit 081a59fcd72436e3ee95e2689894990abd6398db
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Fri Sep 10 18:35:10 2021 +0000

    Novas entidades a partir de coordenações

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 0a8d6ae..0ccdb6a 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -2815,12 +2815,12 @@
 24	conglomeráticos	conglomerático	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
 25	)	)	PUNCT	_	_	21	punct	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
-27	Iapó	Iapó	PROPN	_	Gender=Masc|Number=Sing	18	conj	O	_
+27	Iapó	Iapó	PROPN	_	Gender=Masc|Number=Sing	18	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 28	(	(	PUNCT	_	_	29	punct	O	_
 29	diamictitos	diamictito	NOUN	_	Gender=Masc|Number=Plur	27	parataxis	O	_
 30	)	)	PUNCT	_	_	29	punct	O	_
 31	e	e	CCONJ	_	_	32	cc	O	_
-32	Vila	Vila	PROPN	_	Gender=Fem|Number=Sing	18	conj	O	_
+32	Vila	Vila	PROPN	_	Gender=Fem|Number=Sing	18	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 33	Maria	Maria	PROPN	_	Number=Sing	32	flat:name	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	32	parataxis	O	_
@@ -2920,9 +2920,9 @@
 11	grupos	grupo	NOUN	_	Gender=Masc|Number=Plur	8	obl:agent	O	_
 12	Itararé	Itararé	PROPN	_	Gender=Masc|Number=Sing	11	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	,	,	PUNCT	_	_	14	punct	O	_
-14	Guatá	Guatá	PROPN	_	Gender=Masc|Number=Sing	12	conj	O	_
+14	Guatá	Guatá	PROPN	_	Gender=Masc|Number=Sing	12	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Passa	Passa	PROPN	_	Gender=Masc|Number=Sing	12	conj	O	_
+16	Passa	Passa	PROPN	_	Gender=Masc|Number=Sing	12	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17	Dois	Dois	PROPN	_	Number=Sing	16	flat:name	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
 19	representando	representar	VERB	_	VerbForm=Ger	8	advcl	O	_
@@ -2992,7 +2992,7 @@
 9	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	8	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 10	Bonito	Bonito	PROPN	_	Number=Sing	9	flat:name	B=CAMPO	_
 11	e	e	CCONJ	_	_	12	cc	O	_
-12	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	9	conj	O	_
+12	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	(	(	PUNCT	_	_	16	punct	O	_
 14-15	no	_	_	_	_	_	_	_	_
 14	em	em	ADP	_	_	16	case	O	_
@@ -3027,12 +3027,12 @@
 9	formações	formação	NOUN	_	Gender=Fem|Number=Plur	6	obl:agent	O	_
 10	Irati	Irati	PROPN	_	Gender=Fem|Number=Sing	9	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 11	,	,	PUNCT	_	_	12	punct	O	_
-12	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	10	conj	B=CAMPO	_
+12	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	10	conj	B=CAMPO|B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	Alta	Alta	PROPN	_	Number=Sing	12	flat:name	O	_
 14	,	,	PUNCT	_	_	15	punct	O	_
-15	Teresina	Teresina	PROPN	_	Gender=Fem|Number=Sing	10	conj	O	_
+15	Teresina	Teresina	PROPN	_	Gender=Fem|Number=Sing	10	conj	B=CAMPO|B=UNIDADE_LITOESTRATIGRÁFICA	_
 16	e	e	CCONJ	_	_	17	cc	O	_
-17	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	10	conj	O	_
+17	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	10	conj	B=CAMPO|B=UNIDADE_LITOESTRATIGRÁFICA	_
 18-19	do	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	17	flat:name	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	flat:name	O	_
@@ -3058,7 +3058,7 @@
 37	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	36	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 38	Alta	Alta	PROPN	_	Number=Sing	37	flat:name	O	_
 39	,	,	PUNCT	_	_	40	punct	O	_
-40	Teresina	Teresina	PROPN	_	Gender=Fem|Number=Sing	37	conj	O	_
+40	Teresina	Teresina	PROPN	_	Gender=Fem|Number=Sing	37	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 41	e	e	CCONJ	_	_	42	cc	O	_
 42	parte	parte	NOUN	_	Gender=Fem|Number=Sing	37	conj	O	_
 43-44	da	_	_	_	_	_	_	_	_
@@ -3095,7 +3095,7 @@
 11	formações	formação	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
 12	Pirambóia	Pirambóia	PROPN	_	Gender=Fem|Number=Sing	11	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
-14	Rosário	Rosário	PROPN	_	Gender=Fem|Number=Sing	12	conj	O	_
+14	Rosário	Rosário	PROPN	_	Gender=Fem|Number=Sing	12	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15-16	do	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	14	flat:name	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	flat:name	O	_
@@ -3136,7 +3136,7 @@
 15	eólicos	eólico	ADJ	_	Gender=Masc|Number=Plur	14	amod	O	_
 16	)	)	PUNCT	_	_	15	punct	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
-18	Serra	Serra	PROPN	_	Gender=Masc|Number=Sing	12	conj	B=CAMPO	_
+18	Serra	Serra	PROPN	_	Gender=Masc|Number=Sing	12	conj	B=CAMPO|B=UNIDADE_LITOESTRATIGRÁFICA	_
 19	Geral	Geral	PROPN	_	Number=Sing	18	flat:name	O	_
 20	(	(	PUNCT	_	_	21	punct	O	_
 21	derrames	derrame	NOUN	_	Gender=Masc|Number=Plur	18	parataxis	O	_
@@ -3160,7 +3160,7 @@
 9	grupos	grupo	NOUN	_	Gender=Masc|Number=Plur	7	obj	O	_
 10	Caiuá	Caiuá	PROPN	_	Gender=Masc|Number=Sing	9	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 11	e	e	CCONJ	_	_	12	cc	O	_
-12	Bauru	Bauru	PROPN	_	Gender=Masc|Number=Sing	10	conj	O	_
+12	Bauru	Bauru	PROPN	_	Gender=Masc|Number=Sing	10	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = São representados por depósitos continentais areno-conglomeráticos, com seixos de diversas litologias, maciços ou com estratificação cruzada, ocorrendo também depósitos síltico-argilosos e carbonatos de caliche.
@@ -3464,7 +3464,7 @@
 13	grupos	grupo	NOUN	_	Gender=Masc|Number=Plur	9	parataxis	O	_
 14	Guatá	Guatá	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Passa	Passa	PROPN	_	Gender=Masc|Number=Sing	14	conj	O	_
+16	Passa	Passa	PROPN	_	Gender=Masc|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17	Dois	Dois	PROPN	_	Number=Sing	16	flat:name	O	_
 18	)	)	PUNCT	_	_	13	punct	O	_
 19	e	e	CCONJ	_	_	21	cc	O	_
@@ -3476,7 +3476,7 @@
 25	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	24	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	Geral	Geral	PROPN	_	Number=Sing	25	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 27	e	e	CCONJ	_	_	28	cc	O	_
-28	Botucatu	Botucatu	PROPN	_	Gender=Fem|Number=Sing	25	conj	O	_
+28	Botucatu	Botucatu	PROPN	_	Gender=Fem|Number=Sing	25	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 29	)	)	PUNCT	_	_	24	punct	O	_
 30	em	em	ADP	_	_	31	case	O	_
 31	dados	dado	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
@@ -3568,9 +3568,9 @@
 14	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	Bonito	Bonito	PROPN	_	Number=Sing	14	flat:name	B=CAMPO	_
 16	,	,	PUNCT	_	_	17	punct	O	_
-17	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	14	conj	O	_
+17	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	e	e	CCONJ	_	_	19	cc	O	_
-19	Irati	Irati	PROPN	_	Gender=Masc|Number=Sing	14	conj	O	_
+19	Irati	Irati	PROPN	_	Gender=Masc|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	desenvolvidos	desenvolver	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	13	acl	O	_
 22	em	em	ADP	_	_	23	case	O	_
@@ -4066,8 +4066,8 @@
 64	de	de	ADP	_	_	66	case	O	_
 65	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	66	det	O	_
 66	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	63	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-67	Porto	Porto	PROPN	_	Number=Sing	66	flat:name	O	_
-68	Belo	Belo	PROPN	_	Number=Sing	66	flat:name	O	_
+67	Porto	Porto	PROPN	_	Number=Sing	66	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+68	Belo	Belo	PROPN	_	Number=Sing	66	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 69	e	e	CCONJ	_	_	72	cc	O	_
 70	,	,	PUNCT	_	_	72	punct	O	_
 71	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	72	det	O	_
@@ -4800,7 +4800,7 @@
 5	Zona	zona	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	cop	O	_
 7	possivelmente	possivelmente	ADV	_	_	8	advmod	O	_
-8	pré-neógeno	pré-neógeno	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
+8	pré-neógeno	pré-neógeno	NOUN	_	Gender=Masc|Number=Sing	0	root	B=GEOCRONOLOGIA	_
 9	ou	ou	CCONJ	_	_	10	cc	O	_
 10	neógeno	neógeno	NOUN	_	Gender=Masc|Number=Sing	8	conj	B=GEOCRONOLOGIA	_
 11	inferior	inferior	ADJ	_	Gender=Masc|Number=Sing	8	amod	O	_
@@ -5399,9 +5399,9 @@
 39	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	40	nsubj	O	_
 40	fragmentou	fragmentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	37	acl:relcl	O	_
 41	rochas	rocha	NOUN	_	Gender=Fem|Number=Plur	40	obj	O	_
-42	crustais	crustal	ADJ	_	Gender=Fem|Number=Plur	41	amod	O	_
+42	crustais	crustal	ADJ	_	Gender=Fem|Number=Plur	41	amod	O	B=LITOLOGIA
 43	e	e	CCONJ	_	_	44	cc	O	_
-44	supracrustais	supracrustal	ADJ	_	Gender=Fem|Number=Plur	42	conj	O	_
+44	supracrustais	supracrustal	ADJ	_	Gender=Fem|Number=Plur	42	conj	B=LITOLOGIA	_
 45	pré-cambrianas	pré-cambriano	ADJ	_	Gender=Fem|Number=Plur	41	amod	B=GEOCRONOLOGIA	_
 46	e	e	CCONJ	_	_	47	cc	O	_
 47	paleozóicas	paleozóico	ADJ	_	Gender=Fem|Number=Plur	42	conj	B=GEOCRONOLOGIA	_
@@ -5472,7 +5472,7 @@
 21	formações	formação	NOUN	_	Gender=Fem|Number=Plur	13	conj	O	_
 22	Imbituba	Imbituba	PROPN	_	Gender=Fem|Number=Sing	21	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 23	e	e	CCONJ	_	_	24	cc	O	_
-24	Cassino	Cassino	PROPN	_	Gender=Masc|Number=Sing	22	conj	O	_
+24	Cassino	Cassino	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 25	respectivamente	respectivamente	ADV	_	_	29	advmod	O	_
 26	,	,	PUNCT	_	_	29	punct	O	_
 27	além	além	ADV	_	_	29	cc	O	MWEPOS=CCONJ
@@ -5822,7 +5822,7 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-175
 1	Entre	entre	ADP	_	_	3	case	O	_
 2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
-3	Albo-Aptiano	Albo-Aptiano	PROPN	_	Gender=Masc|Number=Sing	7	obl	O	_
+3	Albo-Aptiano	Albo-Aptiano	PROPN	_	Gender=Masc|Number=Sing	7	obl	B=GEOCRONOLOGIA	_
 4	e	e	CCONJ	_	_	6	cc	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	Mioceno	Mioceno	PROPN	_	Gender=Masc|Number=Sing	3	conj	B=GEOCRONOLOGIA	_
@@ -8063,19 +8063,19 @@
 14	formações	formação	NOUN	_	Gender=Fem|Number=Plur	11	nmod	O	_
 15	Curumin	Curumin	PROPN	_	Gender=Fem|Number=Sing	14	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 16	,	,	PUNCT	_	_	17	punct	O	_
-17	Ariri	Ariri	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
+17	Ariri	Ariri	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	e	e	CCONJ	_	_	19	cc	O	_
-19	Porto	Porto	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
-20	Belo	Belo	PROPN	_	Number=Sing	19	flat:name	O	_
+19	Porto	Porto	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+20	Belo	Belo	PROPN	_	Number=Sing	19	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 21	(	(	PUNCT	_	_	23	punct	O	_
 22	apenas	apenas	ADV	_	_	23	advmod	O	_
 23	porção	porção	NOUN	_	Gender=Fem|Number=Sing	19	appos	O	_
 24	norte	norte	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	)	)	PUNCT	_	_	23	punct	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
-27	Tramandaí	Tramandaí	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+27	Tramandaí	Tramandaí	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 28	e	e	CCONJ	_	_	29	cc	O	_
-29	Atlântida	Atlântida	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+29	Atlântida	Atlântida	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 30	.	.	PUNCT	_	_	8	punct	O	_
 
 # text = O topo do Cretáceo Superior é marcado por rochas das formações Atlântida, Imbé, Tramandaí e Cidreira, apresentando falhas lístricas.
@@ -8097,11 +8097,11 @@
 13	formações	formação	NOUN	_	Gender=Fem|Number=Plur	10	nmod	O	_
 14	Atlântida	Atlântida	PROPN	_	Gender=Fem|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	,	,	PUNCT	_	_	16	punct	O	_
-16	Imbé	Imbé	PROPN	_	Gender=Masc|Number=Sing	14	conj	O	_
+16	Imbé	Imbé	PROPN	_	Gender=Masc|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17	,	,	PUNCT	_	_	18	punct	O	_
-18	Tramandaí	Tramandaí	PROPN	_	Gender=Fem|Number=Sing	14	conj	O	_
+18	Tramandaí	Tramandaí	PROPN	_	Gender=Fem|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 19	e	e	CCONJ	_	_	20	cc	O	_
-20	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	14	conj	O	_
+20	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 21	,	,	PUNCT	_	_	22	punct	O	_
 22	apresentando	apresentar	VERB	_	VerbForm=Ger	8	advcl	O	_
 23	falhas	falha	NOUN	_	Gender=Fem|Number=Plur	22	obj	O	_
@@ -8125,7 +8125,7 @@
 11	formações	formação	NOUN	_	Gender=Fem|Number=Plur	8	obl:agent	O	_
 12	Imbé	Imbé	PROPN	_	Gender=Fem|Number=Sing	11	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
-14	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	12	conj	O	_
+14	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	12	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	,	,	PUNCT	_	_	21	punct	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
 17	topo	topo	NOUN	_	Gender=Masc|Number=Sing	21	nsubj	O	_
@@ -8215,7 +8215,7 @@
 13	formações	formação	NOUN	_	Gender=Fem|Number=Plur	10	nmod	O	_
 14	Imbé	Imbé	PROPN	_	Gender=Fem|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	14	conj	O	_
+16	Cidreira	Cidreira	PROPN	_	Gender=Fem|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17	,	,	PUNCT	_	_	18	punct	O	_
 18-19	encontra-se	_	_	_	_	_	_	_	_
 18	encontra	encontrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	conj	O	_
@@ -17737,11 +17737,11 @@
 20	membros	membro	NOUN	_	Gender=Masc|Number=Plur	18	obj	O	_
 21	Barbalha	Barbalha	PROPN	_	Gender=Masc|Number=Sing	20	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 22	,	,	PUNCT	_	_	23	punct	O	_
-23	Crato	Crato	PROPN	_	Gender=Masc|Number=Sing	21	conj	O	_
+23	Crato	Crato	PROPN	_	Gender=Masc|Number=Sing	21	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 24	,	,	PUNCT	_	_	25	punct	O	_
-25	Ipubi	Ipubi	PROPN	_	Gender=Masc|Number=Sing	21	conj	O	_
+25	Ipubi	Ipubi	PROPN	_	Gender=Masc|Number=Sing	21	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	e	e	CCONJ	_	_	27	cc	O	_
-27	Romualdo	Romualdo	PROPN	_	Gender=Masc|Number=Sing	21	conj	O	_
+27	Romualdo	Romualdo	PROPN	_	Gender=Masc|Number=Sing	21	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 28	.	.	PUNCT	_	_	11	punct	O	_
 
 # text = O Membro Romualdo consiste no último registro litoestratigráfico da sequência pós-rifte I e marca as incursões marinhas na Bacia do Araripe (Assine 2007).
@@ -18437,7 +18437,7 @@
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
 30	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	22	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-31	Romualdo	Romualdo	PROPN	_	Number=Sing	30	flat:name	O	_
+31	Romualdo	Romualdo	PROPN	_	Number=Sing	30	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 32	e	e	CCONJ	_	_	35	cc	O	_
 33-34	da	_	_	_	_	_	_	_	_
 33	de	de	ADP	_	_	35	case	O	_
@@ -21028,7 +21028,7 @@
 23	Brejo	Brejo	PROPN	_	Gender=Fem|Number=Sing	22	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 24	Santo	Santo	PROPN	_	Number=Sing	23	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 25	e	e	CCONJ	_	_	26	cc	O	_
-26	Missão	Missão	PROPN	_	Gender=Fem|Number=Sing	23	conj	O	_
+26	Missão	Missão	PROPN	_	Gender=Fem|Number=Sing	23	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 27	Velha	Velha	PROPN	_	Number=Sing	26	flat:name	O	_
 28	(	(	PUNCT	_	_	29	punct	O	_
 29	Assine	Assine	PROPN	_	Gender=Fem|Number=Sing	5	parataxis	O	_
@@ -21439,10 +21439,10 @@
 20-21	pelas	_	_	_	_	_	_	_	_
 20	por	por	ADP	_	_	22	case	O	_
 21	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	22	det	O	_
-22	Formações	Formações	PROPN	_	Gender=Fem|Number=Plur	19	obl:agent	O	_
-23	Barbalha	Barbalha	PROPN	_	Number=Sing	22	flat:name	O	_
+22	Formações	Formações	PROPN	_	Gender=Fem|Number=Plur	19	obl:agent	B=UNIDADE_LITOESTRATIGRÁFICA	_
+23	Barbalha	Barbalha	PROPN	_	Number=Sing	22	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 24	e	e	CCONJ	_	_	25	cc	O	_
-25	Santana	Santana	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=CAMPO	_
+25	Santana	Santana	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	(	(	PUNCT	_	_	27	punct	O	_
 27	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	22	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
 28	Araripe	Araripe	PROPN	_	Number=Sing	27	flat:name	B=BACIA	_
@@ -21730,7 +21730,7 @@
 17	)	)	PUNCT	_	_	15	punct	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
 19	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
-20	Romualdo	Romualdo	PROPN	_	Number=Sing	19	flat:name	O	_
+20	Romualdo	Romualdo	PROPN	_	Number=Sing	19	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
 21	(	(	PUNCT	_	_	22	punct	O	_
 22	porção	porção	NOUN	_	Gender=Fem|Number=Sing	19	appos	O	_
 23	superior	alto	ADJ	_	Gender=Fem|Number=Sing	22	amod	O	_
@@ -22033,7 +22033,7 @@
 30	intermediário	intermediário	NOUN	_	Gender=Masc|Number=Sing	28	parataxis	O	_
 31	)	)	PUNCT	_	_	30	punct	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
-33	Crato	Crato	PROPN	_	Gender=Masc|Number=Sing	28	conj	O	_
+33	Crato	Crato	PROPN	_	Gender=Masc|Number=Sing	28	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	inferior	inferior	ADJ	_	Gender=Masc|Number=Sing	33	parataxis	O	_
 36	)	)	PUNCT	_	_	35	punct	O	_
@@ -26588,7 +26588,7 @@
 16	membros	membro	NOUN	_	Gender=Masc|Number=Plur	13	nmod	O	_
 17	Crato	Crato	PROPN	_	Gender=Masc|Number=Sing	16	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	e	e	CCONJ	_	_	19	cc	O	_
-19	Romualdo	Romualdo	PROPN	_	Gender=Masc|Number=Sing	17	conj	O	_
+19	Romualdo	Romualdo	PROPN	_	Gender=Masc|Number=Sing	17	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 20	.	.	PUNCT	_	_	13	punct	O	_
 
 # text = Esta superfície se caracteriza por ser uma desconformidade na região proximal da borda da bacia, que passa gradualmente a uma conformidade correlata no sentido do centro da bacia (Figura 26).
@@ -60367,7 +60367,7 @@
 14	:	:	PUNCT	_	_	15	punct	O	_
 15	Carmo	Carmo	PROPN	_	Gender=Masc|Number=Sing	13	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
 16	e	e	CCONJ	_	_	17	cc	O	_
-17	Canabrava	Canabrava	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=CAMPO	_
+17	Canabrava	Canabrava	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	.	.	PUNCT	_	_	7	punct	O	_
 
 # text = O Membro Carmo, segundo Sgarbi (2011), ocorre na região de Carmo do Paranaíba (MG) e é constituído por conglomerados matriz-sustentado e arenitos líticos depositados por leques aluviais e fluxos aquosos que geraram depósitos do tipo wadi.
@@ -61410,7 +61410,7 @@
 38	autores	autor	NOUN	_	Gender=Masc|Number=Plur	34	obl:agent	O	_
 39	,	,	PUNCT	_	_	41	punct	O	_
 40	respectivamente	respectivamente	ADV	_	_	41	advmod	O	_
-41	neojurássica-eocretácea	neojurássica-eocretáceo	ADJ	_	Gender=Fem|Number=Sing	27	appos	O	_
+41	neojurássica-eocretácea	neojurássica-eocretáceo	ADJ	_	Gender=Fem|Number=Sing	27	appos	B=GEOCRONOLOGIA	_
 42	e	e	CCONJ	_	_	43	cc	O	_
 43	berriasiana	berriasiano	ADJ	_	Gender=Fem|Number=Plur	41	conj	B=GEOCRONOLOGIA	_
 44	.	.	PUNCT	_	_	26	punct	O	_
@@ -63109,7 +63109,7 @@
 23	superior	alto	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 24	entre	entre	ADP	_	_	26	case	O	_
 25	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	26	det	O	_
-26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
+26	formação	formação	NOUN	_	Gender=Fem|Number=Sing	22	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 27	e	e	CCONJ	_	_	29	cc	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	26	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
@@ -65146,7 +65146,7 @@
 2	contato	contato	NOUN	_	Gender=Masc|Number=Sing	13	nsubj	O	_
 3	entre	entre	ADP	_	_	5	case	O	_
 4	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	5	det	O	_
-5	formação	formação	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+5	formação	formação	NOUN	_	Gender=Fem|Number=Sing	2	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 6	e	e	CCONJ	_	_	8	cc	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	5	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
@@ -65385,7 +65385,7 @@
 33	transicional	transicional	ADJ	_	Gender=Masc|Number=Sing	32	amod	O	_
 34	entre	entre	ADP	_	_	36	case	O	_
 35	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	36	det	O	_
-36	formação	formação	NOUN	_	Gender=Fem|Number=Sing	29	nmod	O	_
+36	formação	formação	NOUN	_	Gender=Fem|Number=Sing	29	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 37	e	e	CCONJ	_	_	39	cc	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	36	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
@@ -68080,7 +68080,7 @@
 11	formações	formação	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
 12	Quiricó	Quiricó	PROPN	_	Gender=Fem|Number=Sing	11	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
-14	Três	Três	PROPN	_	Gender=Fem|Number=Sing	12	conj	O	_
+14	Três	Três	PROPN	_	Gender=Fem|Number=Sing	12	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15	Barras	Barras	PROPN	_	Number=Sing	14	flat:name	O	_
 16	provem	provar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 17	de	de	ADP	_	_	18	case	O	_
@@ -87915,7 +87915,7 @@
 11	4126,98	4126,98	NUM	_	NumType=Card	9	conj	O	_
 12	=	=	PUNCT	_	_	14	punct	O	_
 13	4,7	4,7	NUM	_	NumType=Card	14	nummod	O	_
-14	Poços	poço	NOUN	_	Gender=Masc|Number=Plur	4	conj	O	_
+14	Poços	poço	NOUN	_	Gender=Masc|Number=Plur	4	conj	O	sentenciação
 
 # text = nescessários em Barracuda
 # sent_id = 161-20150810-MONOGRAFIA_0-593
@@ -100825,10 +100825,10 @@
 10	de	de	ADP	_	_	11	case	O	_
 11	Tupi	Tupi	PROPN	_	Gender=Masc|Number=Sing	9	nmod	B=CAMPO	_
 12	,	,	PUNCT	_	_	13	punct	O	_
-13	Iara	Iara	PROPN	_	Gender=Fem|Number=Sing	11	conj	O	_
+13	Iara	Iara	PROPN	_	Gender=Fem|Number=Sing	11	conj	B=CAMPO	_
 14	Júpiter	Júpiter	PROPN	_	Number=Sing	13	flat:name	O	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Franco	Franco	PROPN	_	Gender=Masc|Number=Sing	11	conj	O	_
+16	Franco	Franco	PROPN	_	Gender=Masc|Number=Sing	11	conj	B=CAMPO	_
 17	.	.	PUNCT	_	_	3	punct	O	_
 
 # text = A GCA examinou e auditou dados técnicos para se estimar o volume de petróleo a ser explorado em cada descoberta.
@@ -113147,12 +113147,12 @@
 56	como	como	ADP	_	_	57	case	O	_
 57	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	51	nmod:appos	B=CAMPO	_
 58	,	,	PUNCT	_	_	59	punct	O	_
-59	Snøhvit	Snøhvit	PROPN	_	Gender=Masc|Number=Sing	57	conj	O	_
+59	Snøhvit	Snøhvit	PROPN	_	Gender=Masc|Number=Sing	57	conj	B=CAMPO	_
 60	,	,	PUNCT	_	_	61	punct	O	_
-61	Ormen	Ormen	PROPN	_	Gender=Masc|Number=Sing	57	conj	O	_
+61	Ormen	Ormen	PROPN	_	Gender=Masc|Number=Sing	57	conj	B=CAMPO	_
 62	Lange	Lange	PROPN	_	Number=Sing	61	flat:name	O	_
 63	e	e	CCONJ	_	_	64	cc	O	_
-64	Åsgard	Åsgard	PROPN	_	Gender=Masc|Number=Sing	57	conj	O	_
+64	Åsgard	Åsgard	PROPN	_	Gender=Masc|Number=Sing	57	conj	B=CAMPO	_
 65	(	(	PUNCT	_	_	66	punct	O	_
 66	BRUSTAD	BRUSTAD	PROPN	_	Gender=Masc|Number=Sing	6	parataxis	O	_
 67	et	et	PROPN	_	Number=Sing	66	flat:name	O	_
@@ -134005,7 +134005,7 @@
 30	de	de	ADP	_	_	29	flat:name	O	_
 31	Campos	Campos	PROPN	_	Number=Sing	29	flat:name	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
-33	pertencente	pertencente	ADJ	_	Gender=Masc|Number=Sing	29	conj	O	_
+33	pertencente	pertencente	ADJ	_	Gender=Masc|Number=Sing	26	conj	O	_
 34-35	à	_	_	_	_	_	_	_	_
 34	a	a	ADP	_	_	36	case	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
@@ -148626,7 +148626,7 @@
 3	SOUZA	SOUZA	PROPN	_	Gender=Masc|Number=Sing	1	appos	B=BACIA	_
 4	JUNIOR	JUNIOR	PROPN	_	Number=Sing	3	flat:name	O	_
 5	,	,	PUNCT	_	_	6	punct	O	_
-6	G.	G.	PROPN	_	Gender=Fem|Number=Sing	3	conj	O	_
+6	G.	G.	PROPN	_	Gender=Fem|Number=Sing	3	conj	O	sentenciação
 7	C.	C.	PROPN	_	Number=Sing	6	flat:name	O	_
 8	,	,	PUNCT	_	_	9	punct	O	_
 9	Simulação	Simulação	PROPN	_	Gender=Fem|Number=Sing	3	conj	O	_
@@ -186881,7 +186881,7 @@
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
 12	Bacias	Bacias	PROPN	_	Gender=Fem|Number=Plur	9	nmod	O	_
 13	de	de	ADP	_	_	12	flat:name	O	_
-14	Campos	Campos	PROPN	_	Number=Sing	12	flat:name	O	_
+14	Campos	Campos	PROPN	_	Number=Sing	12	flat:name	B=BACIA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
 16	Santos	Santos	PROPN	_	Number=Sing	12	conj	B=BACIA	_
 17	.	.	PUNCT	_	_	6	punct	O	_
@@ -188580,7 +188580,7 @@
 8	de	de	ADP	_	_	9	case	O	_
 9	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	7	nmod	B=BACIA	_
 10	e	e	CCONJ	_	_	11	cc	O	_
-11	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	9	conj	O	_
+11	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=BACIA	_
 12	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	13	nsubj	O	_
 13	evoluíram	evoluir	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	7	acl:relcl	O	_
 14	a	a	ADP	_	_	18	case	O	MWEPOS=ADP
@@ -189083,7 +189083,7 @@
 9	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 10	entre	entre	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	Neocomiano	Neocomiano	PROPN	_	Gender=Masc|Number=Sing	9	obl	O	_
+12	Neocomiano	Neocomiano	PROPN	_	Gender=Masc|Number=Sing	9	obl	B=GEOCRONOLOGIA	_
 13	e	e	CCONJ	_	_	15	cc	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 15	Eoaptiano	Eoaptiano	PROPN	_	Gender=Masc|Number=Sing	12	conj	B=GEOCRONOLOGIA	_
@@ -189262,9 +189262,9 @@
 7	formações	formação	NOUN	_	Gender=Fem|Number=Plur	5	xcomp	O	_
 8	Florianópolis	Florianópolis	PROPN	_	Gender=Fem|Number=Sing	7	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 9	,	,	PUNCT	_	_	10	punct	O	_
-10	Guarujá	Guarujá	PROPN	_	Gender=Masc|Number=Sing	8	conj	O	_
+10	Guarujá	Guarujá	PROPN	_	Gender=Masc|Number=Sing	8	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 11	e	e	CCONJ	_	_	12	cc	O	_
-12	Itanhaém	Itanhaém	PROPN	_	Gender=Unsp|Number=Sing	8	conj	O	_
+12	Itanhaém	Itanhaém	PROPN	_	Gender=Unsp|Number=Sing	8	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	,	,	PUNCT	_	_	5	punct	O	_
 14	consiste	consistir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	de	de	ADP	_	_	19	case	O	_
@@ -191553,7 +191553,7 @@
 13	,	,	PUNCT	_	_	14	punct	O	_
 14	Resende	Resende	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=BACIA	_
 15	,	,	PUNCT	_	_	16	punct	O	_
-16	Volta	Volta	PROPN	_	Gender=Fem|Number=Sing	9	conj	O	_
+16	Volta	Volta	PROPN	_	Gender=Fem|Number=Sing	9	conj	B=BACIA	_
 17	Redonda	Redonda	PROPN	_	Number=Sing	16	flat:name	O	_
 18	,	,	PUNCT	_	_	20	punct	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
@@ -192800,24 +192800,24 @@
 47	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	48	det	O	_
 48	plugs	plug	NOUN	_	Gender=Masc|Number=Plur	28	conj	O	_
 49	de	de	ADP	_	_	50	case	O	_
-50	Serra	Serra	PROPN	_	Gender=Masc|Number=Sing	48	nmod	B=CAMPO	_
+50	Serra	Serra	PROPN	_	Gender=Masc|Number=Sing	48	nmod	B=BACIA|B=CAMPO	_
 51-52	dos	_	_	_	_	_	_	_	_
 51	de	de	ADP	_	_	50	flat:name	O	_
 52	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	50	flat:name	O	_
 53	Tomazes	Tomazes	PROPN	_	Number=Sing	50	flat:name	O	_
 54	,	,	PUNCT	_	_	55	punct	O	_
-55	Marapicu	Marapicu	PROPN	_	Gender=Fem|Number=Sing	50	conj	O	_
+55	Marapicu	Marapicu	PROPN	_	Gender=Fem|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 56	,	,	PUNCT	_	_	57	punct	O	_
-57	Itaúna	Itaúna	PROPN	_	Gender=Masc|Number=Sing	50	conj	O	_
+57	Itaúna	Itaúna	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 58	,	,	PUNCT	_	_	59	punct	O	_
-59	Monjolos	Monjolos	PROPN	_	Gender=Masc|Number=Sing	50	conj	O	_
+59	Monjolos	Monjolos	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 60	,	,	PUNCT	_	_	61	punct	O	_
-61	Itaboraí	Itaboraí	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA	_
+61	Itaboraí	Itaboraí	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 62	,	,	PUNCT	_	_	63	punct	O	_
-63	Country	Country	PROPN	_	Gender=Masc|Number=Sing	50	conj	O	_
+63	Country	Country	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 64	Club	Club	PROPN	_	Number=Sing	63	flat:name	O	_
 65	e	e	CCONJ	_	_	66	cc	O	_
-66	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	50	conj	O	_
+66	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	50	conj	B=BACIA|B=CAMPO	_
 67	Frio	Frio	PROPN	_	Number=Sing	66	flat:name	O	_
 68	,	,	PUNCT	_	_	70	punct	O	_
 69	entre	entre	ADP	_	_	70	case	O	_
@@ -194241,7 +194241,7 @@
 34	de	de	ADP	_	_	35	case	O	_
 35	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	33	nmod	B=BACIA	_
 36	,	,	PUNCT	_	_	37	punct	O	_
-37	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	35	conj	O	_
+37	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	35	conj	B=BACIA	_
 38	e	e	CCONJ	_	_	39	cc	O	_
 39	Paraná	Paraná	PROPN	_	Gender=Masc|Number=Sing	35	conj	B=BACIA	_
 40	.	.	PUNCT	_	_	17	punct	O	_
@@ -194317,7 +194317,7 @@
 19	de	de	ADP	_	_	20	case	O	_
 20	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	18	nmod	B=BACIA	_
 21	e	e	CCONJ	_	_	22	cc	O	_
-22	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	20	conj	O	_
+22	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	20	conj	B=BACIA	_
 23	.	.	PUNCT	_	_	7	punct	O	_
 
 # text = Tal situação tornou-se isostaticamente instável, e o colapso gravitacional do megaplanalto em direção às áreas deprimidas iniciou-se 7
@@ -194430,7 +194430,7 @@
 22	de	de	ADP	_	_	23	case	O	_
 23	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	21	nmod	B=BACIA	_
 24	e	e	CCONJ	_	_	25	cc	O	_
-25	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	23	conj	O	_
+25	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	23	conj	B=BACIA	_
 26	.	.	PUNCT	_	_	12	punct	O	_
 
 # text = Os remanescentes topográficos deste megaplanalto constituem hoje as partes altas das serras do Mar e da Mantiqueira, modificadas por rebate elástico e basculamento dos blocos.
@@ -194638,7 +194638,7 @@
 56	de	de	ADP	_	_	57	case	O	_
 57	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	55	nmod	B=BACIA	_
 58	e	e	CCONJ	_	_	59	cc	O	_
-59	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	57	conj	O	_
+59	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	57	conj	B=BACIA	_
 60	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = No entender destes autores os aspectos que apontam para a natureza tectônica dos grábens do RCSB são: (1) planícies litorâneas ou vales fluviais limitados por desníveis topográficos abruptos, retilíneos e de grande expressão (centenas a milhares de metros); (2) perfis morfotectônicos que podem ser interpretados como resultantes de tectônica de blocos planares rotacionados (tectônica dominó), em se tomando como referência as superfícies de aplainamento (superfície Japi por exemplo); (3) cotas altimétricas coerentes com patamares tectônicos escalonados desde a Serra da Mantiqueira até a plataforma continental da Bacia de Santos; (4) inserção em um contexto regional de quebramento/basculamento/abatimento cenozóico, comprovado em algumas das bacias sedimentares estudadas em grande detalhe.
@@ -194988,28 +194988,28 @@
 13-14	da	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
-15	Barra	Barra	PROPN	_	Gender=Fem|Number=Sing	12	nmod	O	_
+15	Barra	Barra	PROPN	_	Gender=Fem|Number=Sing	12	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 16-17	do	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	15	flat:name	O	_
-17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	flat:name	O	_
-18	São	São	PROPN	_	Number=Sing	15	flat:name	O	_
-19	João	João	PROPN	_	Number=Sing	15	flat:name	O	_
+17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+18	São	São	PROPN	_	Number=Sing	15	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+19	João	João	PROPN	_	Number=Sing	15	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 20	,	,	PUNCT	_	_	21	punct	O	_
-21	Guanabara	Guanabara	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+21	Guanabara	Guanabara	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 22	,	,	PUNCT	_	_	23	punct	O	_
-23	Ubatuba	Ubatuba	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+23	Ubatuba	Ubatuba	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 24	,	,	PUNCT	_	_	25	punct	O	_
-25	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=BACIA	_
+25	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	,	,	PUNCT	_	_	27	punct	O	_
-27	Ribeira	Ribeira	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+27	Ribeira	Ribeira	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 28-29	do	_	_	_	_	_	_	_	_
-28	de	de	ADP	_	_	27	flat:name	O	_
-29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	flat:name	O	_
-30	Iguape	Iguape	PROPN	_	Number=Sing	27	flat:name	O	_
+28	de	de	ADP	_	_	27	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+30	Iguape	Iguape	PROPN	_	Number=Sing	27	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 31	,	,	PUNCT	_	_	32	punct	O	_
-32	Cananéia	Cananéia	PROPN	_	Gender=Fem|Number=Sing	15	conj	O	_
+32	Cananéia	Cananéia	PROPN	_	Gender=Fem|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 33	e	e	CCONJ	_	_	34	cc	O	_
-34	Paranaguá	Paranaguá	PROPN	_	Gender=Masc|Number=Sing	15	conj	O	_
+34	Paranaguá	Paranaguá	PROPN	_	Gender=Masc|Number=Sing	15	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 35	(	(	PUNCT	_	_	36	punct	O	_
 36	fig.2.6	fig.2.6	NOUN	_	Gender=Fem|Number=Sing	1	parataxis	O	_
 37	)	)	PUNCT	_	_	36	punct	O	_
@@ -195559,7 +195559,7 @@
 45	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	42	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 46	Brusque	Brusque	PROPN	_	Number=Sing	45	flat:name	O	_
 47	,	,	PUNCT	_	_	48	punct	O	_
-48	Heilbron	Heilbron	PROPN	_	Gender=Masc|Number=Sing	45	conj	O	_
+48	Heilbron	Heilbron	PROPN	_	Gender=Masc|Number=Sing	42	parataxis	O	_
 49	et	et	PROPN	_	Number=Sing	48	flat:name	O	_
 50	al.	al.	PROPN	_	Number=Sing	48	flat:name	O	_
 51	2004	2004	PROPN	_	Number=Sing	48	flat:name	O	_
@@ -230674,7 +230674,7 @@
 71-72	nas	_	_	_	_	_	_	_	_
 71	em	em	ADP	_	_	73	case	O	_
 72	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	73	det	O	_
-73	Bacias	Bacias	PROPN	_	Gender=Fem|Number=Plur	68	nmod	O	_
+73	Bacias	Bacias	PROPN	_	Gender=Fem|Number=Plur	68	nmod	B=BACIA	_
 74	de	de	ADP	_	_	73	flat:name	O	_
 75	Campos	Campos	PROPN	_	Number=Sing	73	flat:name	O	_
 76	e	e	CCONJ	_	_	77	cc	O	_
@@ -231686,7 +231686,7 @@
 22	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	18	obl	B=GEOCRONOLOGIA	_
 23	superior	alto	ADJ	_	Gender=Masc|Number=Sing	22	amod	I=GEOCRONOLOGIA	_
 24	e	e	CCONJ	_	_	25	cc	O	_
-25	Terciário	Terciário	PROPN	_	Gender=Masc|Number=Sing	22	conj	O	_
+25	Terciário	Terciário	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=GEOCRONOLOGIA	_
 26	)	)	PUNCT	_	_	12	punct	O	_
 27-28	na	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
@@ -231706,7 +231706,7 @@
 39	de	de	ADP	_	_	40	case	O	_
 40	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	38	nmod	B=BACIA	_
 41	e	e	CCONJ	_	_	42	cc	O	_
-42	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	40	conj	O	_
+42	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	40	conj	B=BACIA	_
 43	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	aux	O	_
 44	sido	ser	AUX	_	VerbForm=Part	45	aux:pass	O	_
 45	apontada	apontar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -231869,7 +231869,7 @@
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
 24	Cretáceo	Cretáceo	PROPN	_	Gender=Masc|Number=Sing	1	obl	B=GEOCRONOLOGIA	_
 25	e	e	CCONJ	_	_	26	cc	O	_
-26	Terciário	Terciário	PROPN	_	Gender=Masc|Number=Sing	24	conj	O	_
+26	Terciário	Terciário	PROPN	_	Gender=Masc|Number=Sing	24	conj	B=GEOCRONOLOGIA	_
 27	.	.	PUNCT	_	_	1	punct	O	_
 
 # text = Estas novas idéias, baseadas na análise de sísmica 3D e seus atributos, apontam, portanto, para o papel fundamental que desempenhou o embasamento no desenvolvimento do sistema petrolífero desta bacia.
@@ -237041,7 +237041,7 @@
 22	de	de	ADP	_	_	23	case	O	_
 23	Santos	Santos	PROPN	_	Gender=Masc|Number=Sing	21	nmod	B=BACIA	_
 24	e	e	CCONJ	_	_	25	cc	O	_
-25	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	23	conj	O	_
+25	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	23	conj	B=BACIA	_
 26	e	e	CCONJ	_	_	27	cc	O	_
 27	correspondem	corresponder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	conj	O	_
 28-29	ao	_	_	_	_	_	_	_	_
@@ -249141,7 +249141,7 @@
 38	sem	sem	ADP	_	_	39	case	O	_
 39	avaliação	avaliação	NOUN	_	Gender=Fem|Number=Sing	37	nmod	O	_
 40	de	de	ADP	_	_	41	case	O	_
-41	campo	campo	NOUN	_	Gender=Masc|Number=Sing	39	nmod	B=BACIA	_
+41	campo	campo	NOUN	_	Gender=Masc|Number=Sing	39	nmod	O	_
 42	e	e	CCONJ	_	_	43	cc	O	_
 43	laboratorial	laboratorial	ADJ	_	Gender=Masc|Number=Sing	41	conj	O	_
 44	.	.	PUNCT	_	_	7	punct	O	_
