commit eb928fd2e4709a6df8385ab0e0b6b60a3b223564
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed May 19 18:40:27 2021 -0300

    correções mclara petroles_3

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b4203c4..ac81bf2 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -268,13 +268,13 @@
 42	2000	2000	NUM	_	NumType=Card	38	nmod:appos	O	_
 43	)	)	PUNCT	_	_	42	punct	O	_
 44	,	,	PUNCT	_	_	47	punct	O	_
-45	além	além	ADV	_	_	47	cc	O	_
+45	além	além	ADV	_	_	47	cc	O	MWE=além_de|MWEPOS=CCONJ
 46	de	de	ADP	_	_	45	fixed	O	_
-47	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	24	conj	B=INSTRUMENTO	_
+47	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	11	conj	B=INSTRUMENTO	_
 48	gravimétrico	gravimétrico	ADJ	_	Gender=Masc|Number=Sing	47	amod	O	_
 49	e	e	CCONJ	_	_	50	cc	O	_
 50	aeromagnetométrico	aeromagnetométrico	ADJ	_	Gender=Masc|Number=Sing	48	conj	O	_
-51	.	.	PUNCT	_	_	47	punct	O	_
+51	.	.	PUNCT	_	_	10	punct	O	_
 52	.	.	PUNCT	_	_	10	punct	O	_
 
 # text = Figura 1: Localização da Bacia de Pelotas
@@ -4882,7 +4882,7 @@
 27	ou	ou	CCONJ	_	_	28	cc	O	_
 28	duplex	duplex	NOUN	_	Gender=Fem|Number=Sing	25	conj	O	_
 29	,	,	PUNCT	_	_	25	punct	O	_
-30	além	além	ADV	_	_	33	cc	O	_
+30	além	além	ADV	_	_	33	cc	O	MWE=além_de|MWEPOS=CCONJ
 31-32	da	_	_	_	_	_	_	O	_
 31	de	de	ADP	_	_	30	fixed	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
@@ -7451,7 +7451,7 @@
 
 # text = Além destes dois feixes a direção NW-SE está correlacionada ao Lineamento Tibagi..
 # sent_id = 247-20140910-MONOGRAFIA_0-210
-1	Além	além	ADV	_	_	5	cc	O	_
+1	Além	além	ADV	_	_	5	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	destes	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	5	det	O	_
@@ -10718,7 +10718,7 @@
 
 # text = Além da projeção do Lineamento Tibagi, foi constatada a presença de feições vulcânicas próximo à Plataforma de Florianópolis, estas feições afetam os horizontes do Rifte, Albiano e com menor frequência os horizontes do Cretáceo Superior..
 # sent_id = 247-20140910-MONOGRAFIA_0-298
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
@@ -13041,7 +13041,7 @@
 14	têm	ter	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	como	como	ADP	_	_	16	case	O	_
 16	emulsificante	emulsificante	NOUN	_	Gender=Masc|Number=Sing	14	xcomp	B=QUÍMICA	_
-17	sais	sai	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
+17	sais	sal	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	aminas	amina	NOUN	_	Gender=Fem|Number=Plur	17	nmod	B=QUÍMICA	_
 20	ou	ou	CCONJ	_	_	21	cc	O	_
@@ -16334,7 +16334,7 @@
 5	)	)	PUNCT	_	_	2	flat:name	O	_
 6	,	,	PUNCT	_	_	2	punct	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	análises	anális	NOUN	_	Gender=Fem|Number=Plur	11	nsubj	O	_
+8	análises	análise	NOUN	_	Gender=Fem|Number=Plur	11	nsubj	O	_
 9	com	com	ADP	_	_	10	case	O	_
 10	emulsões	emulsão	NOUN	_	Gender=Fem|Number=Plur	8	nmod	B=QUÍMICA	_
 11	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -16383,7 +16383,7 @@
 52	2	2	NUM	_	NumType=Card	55	parataxis	O	_
 53	)	)	PUNCT	_	_	52	punct	O	_
 54	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	55	det	O	_
-55	análises	anális	NOUN	_	Gender=Fem|Number=Plur	35	conj	O	_
+55	análises	análise	NOUN	_	Gender=Fem|Number=Plur	35	conj	O	_
 56	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	57	nsubj	O	_
 57	classificam	classificar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	55	acl:relcl	O	_
 58	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	59	det	O	_
@@ -16419,7 +16419,7 @@
 88	3	3	NUM	_	NumType=Card	91	parataxis	O	_
 89	)	)	PUNCT	_	_	88	punct	O	_
 90	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	91	det	O	_
-91	análises	anális	NOUN	_	Gender=Fem|Number=Plur	73	conj	O	_
+91	análises	análise	NOUN	_	Gender=Fem|Number=Plur	73	conj	O	_
 92	sobre	sobre	ADP	_	_	94	case	O	_
 93	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	94	det	O	_
 94	resíduo	resíduo	NOUN	_	Gender=Masc|Number=Sing	91	nmod	O	_
@@ -17080,8 +17080,8 @@
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	8	nmod	B=QUÍMICA	_
 12	asfáltica	asfáltico	ADJ	_	Gender=Fem|Number=Sing	11	amod	B=LITOLOGIA	_
-13	quanto	quanto	ADV	_	_	15	advmod	O	_
-14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
+13	quanto	quanto	ADP	_	_	15	case	O	MWE=quanto_a|MWEPOS=ADP
+14	a	a	ADP	_	_	13	fixed	O	_
 15	sedimentação	sedimentação	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	verificando	verificar	VERB	_	VerbForm=Ger	3	advcl	O	_
@@ -17090,7 +17090,7 @@
 20	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	21	nsubj	B=QUÍMICA	_
 21	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	ccomp	O	_
 22	estabilidade	estabilidade	NOUN	_	Gender=Fem|Number=Sing	21	obj	O	_
-23	quanto	quanto	ADV	_	_	25	case	O	_
+23	quanto	quanto	ADP	_	_	25	case	O	MWE=quanto_a|MWEPOS=ADP
 24	a	a	ADP	_	_	23	fixed	O	_
 25	estocagem	estocagem	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
 26	(	(	PUNCT	_	_	28	punct	O	_
@@ -32229,7 +32229,7 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	condições	condição	NOUN	_	Gender=Fem|Number=Plur	14	nsubj:pass	O	_
 3	ambientais	ambiental	ADJ	_	Gender=Fem|Number=Plur	2	amod	O	_
-4	,	,	PUNCT	_	_	2	punct	O	_
+4	,	,	PUNCT	_	_	11	punct	O	_
 5-6	às	_	_	_	_	_	_	O	_
 5	a	a	ADP	_	_	7	case	O	_
 6	as	o	DET	_	Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
@@ -32245,12 +32245,12 @@
 16	Arai	Arai	PROPN	_	Gender=Masc|Number=Sing	14	obl:agent	O	_
 17	&	&	PROPN	_	Number=Sing	16	flat:name	O	_
 18	Coimbra	Coimbra	PROPN	_	Number=Sing	16	flat:name	O	_
-19	(	(	PUNCT	_	_	20	punct	O	_
-20	1990	1990	NUM	_	NumType=Card	16	nmod:appos	O	_
-21	)	)	PUNCT	_	_	20	punct	O	_
-22	como	como	ADV	_	_	24	case	O	_
+19	(	(	PUNCT	_	_	16	flat:name	O	_
+20	1990	1990	NUM	_	NumType=Card	16	flat:name	O	_
+21	)	)	PUNCT	_	_	16	flat:name	O	_
+22	como	como	ADP	_	_	24	case	O	_
 23	de	de	ADP	_	_	24	case	O	_
-24	água	água	NOUN	_	Gender=Fem|Number=Sing	16	conj	B=FLUIDO	_
+24	água	água	NOUN	_	Gender=Fem|Number=Sing	14	xcomp	B=FLUIDO	_
 25	doce	doce	ADJ	_	Gender=Fem|Number=Sing	24	amod	I=FLUIDO	_
 26	,	,	PUNCT	_	_	28	punct	O	_
 27	tendo	ter	AUX	_	VerbForm=Ger	28	aux	O	_
@@ -32417,7 +32417,7 @@
 
 # text = Além dos dinoflagelados, são encontrados em abundância restos de fósseis de peixes (Beurlen 1963), situados no interior de concreções calcárias, como Rhacolepis, Leptolepis, Aspidorhynchidae, Elopidae e Ichtyodectidae..
 # sent_id = 284-20150917-MONOGRAFIA_0-397
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
@@ -34875,7 +34875,7 @@
 15	de	de	ADP	_	_	17	case	O	_
 16	alto	alto	ADJ	_	Gender=Masc|Number=Sing	17	amod	O	_
 17	valor	valor	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
-18	quando	quando	ADP	_	_	22	case	O	_
+18	quando	quando	ADV	_	_	19	advmod	O	_
 19	refinado	refinar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	22	acl	O	_
 20	e	e	CCONJ	_	_	21	cc	O	_
 21	elevado	elevar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	19	conj	O	_
@@ -41046,7 +41046,7 @@
 7	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	8	det	O	_
 8	características	característica	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 9	,	,	PUNCT	_	_	12	punct	O	_
-10	porem	porem	ADV	_	_	12	advmod	O	_
+10	porem	porém	CCONJ	_	_	12	cc	O	_
 11	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	aux:pass	O	_
 12	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	3	conj	O	_
 13	aqui	aqui	ADV	_	_	12	advmod	O	_
@@ -41153,7 +41153,7 @@
 8	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	9	det	O	_
 9	características	característica	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
 10	,	,	PUNCT	_	_	4	punct	O	_
-11	porem	porem	ADV	_	_	13	advmod	O	_
+11	porem	porém	CCONJ	_	_	13	cc	O	_
 12	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	13	aux:pass	O	_
 13	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	4	conj	O	_
 14	aqui	aqui	ADV	_	_	13	advmod	O	_
@@ -41247,7 +41247,7 @@
 7	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	8	det	O	_
 8	características	característica	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 9	,	,	PUNCT	_	_	12	punct	O	_
-10	porem	porem	ADV	_	_	12	advmod	O	_
+10	porem	porém	CCONJ	_	_	12	cc	O	_
 11	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	aux:pass	O	_
 12	apresentadas	apresentar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	3	conj	O	_
 13	aqui	aqui	ADV	_	_	12	advmod	O	_
@@ -46188,7 +46188,7 @@
 29	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	24	obl	B=LITOLOGIA	_
 30	e	e	CCONJ	_	_	37	cc	O	_
 31	,	,	PUNCT	_	_	33	punct	O	_
-32	quando	quando	ADP	_	_	33	case	O	_
+32	quando	quando	ADV	_	_	33	advmod	O	_
 33	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	37	amod	O	_
 34	,	,	PUNCT	_	_	33	punct	O	_
 35-36	na	_	_	_	_	_	_	O	_
@@ -48491,7 +48491,7 @@
 20	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	21	det	O	_
 21	corais	coral	NOUN	_	Gender=Masc|Number=Plur	14	nmod:appos	B=CAMPO	_
 22	(	(	PUNCT	_	_	23	punct	O	_
-23	Wikipédia	Wikipédia	PROPN	_	Gender=Fem|Number=Sing	21	appos	O	_
+23	Wikipédia	Wikipédia	PROPN	_	Gender=Fem|Number=Sing	8	appos	O	_
 24	)	)	PUNCT	_	_	23	punct	O	_
 25	.	.	PUNCT	_	_	8	punct	O	_
 26	.	.	PUNCT	_	_	8	punct	O	_
@@ -49388,7 +49388,7 @@
 
 # text = Além dos argilominerais, as argilas contêm geralmente, outros materiais e minerais, tais como matéria orgânica, sais solúveis e partículas de quartzo, pirita, mica, calcita, dolomita e outros minerais residuais, e podem conter também minerais não-cristalinos ou amorfos..
 # sent_id = 24-20150122-MONOGRAFIA_0-140
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	dos	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
@@ -59183,7 +59183,7 @@
 2	magnésio	magnésio	NOUN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 3	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
-5	além	além	ADV	_	_	8	cc	O	_
+5	além	além	ADV	_	_	8	cc	O	MWE=além_de|MWEPOS=CCONJ
 6-7	da	_	_	_	_	_	_	O	_
 6	de	de	ADP	_	_	5	fixed	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -66385,7 +66385,7 @@
 17	Kattah	Kattah	PROPN	_	Gender=Masc|Number=Sing	15	obl:agent	O	_
 18	(	(	PUNCT	_	_	17	flat:name	O	_
 19	1991	1991	NUM	_	NumType=Card	17	flat:name	O	_
-20	)	)	PUNCT	_	_	19	punct	O	_
+20	)	)	PUNCT	_	_	17	flat:name	O	_
 21	.	.	PUNCT	_	_	6	punct	O	_
 22	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -68019,7 +68019,7 @@
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	obter	obter	VERB	_	VerbForm=Inf	1	xcomp	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
-5	quando	quando	ADP	_	_	6	case	O	_
+5	quando	quando	ADV	_	_	6	advmod	O	_
 6	possível	possível	ADJ	_	Gender=Masc|Number=Sing	3	obl	O	_
 7	,	,	PUNCT	_	_	3	punct	O	_
 8	mínimo	mínimo	ADJ	_	Gender=Masc|Number=Sing	3	xcomp	O	_
@@ -69760,7 +69760,7 @@
 
 # text = Além da associação de fácies semelhante, a ausência de sedimentos pelíticos no registro sedimentar e a baixa dispersão das medidas de paleocorrentes também são indicativas de sedimentação fluvial do tipo entrelaçado para a porção inferior da Seção Córrego Tiririca..
 # sent_id = 283-20150917-MONOGRAFIA_0-186
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
@@ -70880,7 +70880,7 @@
 
 # text = Além do quê, esta seção compreende a maior espessura observada para a Fm..
 # sent_id = 283-20150917-MONOGRAFIA_0-228
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	do	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
@@ -72472,7 +72472,7 @@
 6-7	destas	_	_	_	_	_	_	O	_
 6	de	de	ADP	_	_	8	case	O	_
 7	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	8	det	O	_
-8	fácies	fácie	NOUN	_	Gender=Fem|Number=Plur	5	nmod	B=LITOLOGIA	_
+8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	5	nmod	B=LITOLOGIA	_
 9	como	como	ADP	_	_	10	case	O	_
 10	playa	playa	NOUN	_	Gender=Fem|Number=Sing	5	nmod	O	_
 11	lake	lake	X	_	_	10	amod	O	_
@@ -72482,11 +72482,11 @@
 15	pouco	pouco	ADV	_	_	16	advmod	O	_
 16	expressiva	expressivo	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 17	de	de	ADP	_	_	19	case	O	_
-18	fácies	fácie	ADJ	_	Gender=Fem|Number=Plur	19	amod	B=LITOLOGIA	_
-19	evaporíticas	evaporítica	NOUN	_	Gender=Fem|Number=Plur	14	nmod	O	_
+18	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=LITOLOGIA	_
+19	evaporíticas	evaporítica	ADJ	_	Gender=Fem|Number=Plur	18	amod	O	_
 20	,	,	PUNCT	_	_	22	punct	O	_
 21	normalmente	normalmente	ADV	_	_	22	advmod	O	_
-22	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	19	acl	O	_
+22	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	18	acl	O	_
 23	para	para	ADP	_	_	25	case	O	_
 24	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	25	det	O	_
 25	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	22	obl	O	_
@@ -75446,7 +75446,7 @@
 
 # text = Além do quê, este sistema eólico ocorre interdigitado à fácies de sabkas, sugerindo maior umidade em todo o sistema..
 # sent_id = 283-20150917-MONOGRAFIA_0-403
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	do	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
@@ -76516,8 +76516,8 @@
 18	de	de	ADP	_	_	17	flat:name	O	_
 19	Galena	Galena	PROPN	_	Number=Sing	17	flat:name	B=LITOLOGIA	_
 20	(	(	PUNCT	_	_	21	punct	O	_
-21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	17	appos	O	_
-22	27	27	PROPN	_	Number=Sing	21	flat:name	O	_
+21	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	53	parataxis	O	_
+22	27	27	PROPN	_	Number=Sing	21	nummod	O	_
 23	)	)	PUNCT	_	_	21	punct	O	_
 24	,	,	PUNCT	_	_	30	punct	O	_
 25	e	e	CCONJ	_	_	30	cc	O	_
@@ -76548,7 +76548,7 @@
 49	,	,	PUNCT	_	_	48	flat:name	O	_
 50	2011	2011	NUM	_	NumType=Card	48	flat:name	O	_
 51	)	)	PUNCT	_	_	42	punct	O	_
-52	,	,	PUNCT	_	_	42	punct	O	_
+52	,	,	PUNCT	_	_	30	punct	O	_
 53-54	interpretou-se	_	_	_	_	_	_	O	_
 53	interpretou	interpretar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 54	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	53	expl	O	_
@@ -77196,7 +77196,7 @@
 3	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	fornecer	fornecer	VERB	_	VerbForm=Inf	3	xcomp	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
-6	além	além	ADV	_	_	9	cc	O	_
+6	além	além	ADV	_	_	9	cc	O	MWE=além_de|MWEPOS=CCONJ
 7-8	do	_	_	_	_	_	_	O	_
 7	de	de	ADP	_	_	6	fixed	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
@@ -77207,9 +77207,9 @@
 12	paleoventos	paleovento	NOUN	_	Gender=Masc|Number=Plur	9	nmod	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
 14	informações	informação	NOUN	_	Gender=Fem|Number=Plur	4	obj	O	_
-15	quanto	quanto	ADV	_	_	18	case	O	_
+15	quanto	quanto	ADP	_	_	18	case	O	MWE=quanto_a|MWEPOS=ADP
 16-17	ao	_	_	_	_	_	_	O	_
-16	a	a	ADP	_	_	18	case	O	_
+16	a	a	ADP	_	_	15	fixed	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
 18	tipo	tipo	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
@@ -109620,7 +109620,7 @@
 
 # text = Além da Petrobras, outras empresas estão envolvidas na exploração da camada do Pré-sal, entre elas Royal Dutch Shell, ExxonMobil, BG Group, HessCorp, Galp, Repsol e Petrogal..
 # sent_id = 334-20150521-MONOGRAFIA_0-34
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
@@ -112575,9 +112575,9 @@
 # text = Já quanto ao óleo original in situ (OOIP) para o Entorno de Júpiter estima-se que varia de 1,21 Bbbl (baixa estimativa) a 2,57 Bbbl (alta estimativa), sendo a melhor estimativa de 1,83 bbl..
 # sent_id = 334-20150521-MONOGRAFIA_0-103
 1	Já	já	ADV	_	_	17	advmod	O	_
-2	quanto	quanto	ADV	_	_	5	case	O	_
+2	quanto	quanto	ADP	_	_	5	case	O	MWE=quanto_a|MWEPOS=ADP
 3-4	ao	_	_	_	_	_	_	O	_
-3	a	a	ADP	_	_	5	case	O	_
+3	a	a	ADP	_	_	2	fixed	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	17	obl	B=FLUIDO	_
 6	original	original	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
@@ -116365,7 +116365,7 @@
 17	e	e	CCONJ	_	_	18	cc	O	_
 18	água	água	NOUN	_	Gender=Fem|Number=Sing	14	conj	B=FLUIDO	_
 19	,	,	PUNCT	_	_	22	punct	O	_
-20	além	além	ADV	_	_	22	cc	O	_
+20	além	além	ADV	_	_	22	cc	O	MWE=além_de|MWEPOS=CCONJ
 21	de	de	ADP	_	_	20	fixed	O	_
 22	impurezas	impureza	NOUN	_	Gender=Fem|Number=Plur	14	conj	O	_
 23	como	como	ADP	_	_	24	case	O	_
@@ -117256,7 +117256,7 @@
 8	por	por	ADP	_	_	9	case	O	_
 9	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 10	,	,	PUNCT	_	_	7	punct	O	_
-11	além	além	ADV	_	_	14	cc	O	_
+11	além	além	ADV	_	_	14	cc	O	MWE=além_de|MWEPOS=CCONJ
 12-13	da	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	11	fixed	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
@@ -117709,9 +117709,9 @@
 # text = Já quanto à estrutura de suporte, as plataformas podem ser fixas (jacket), flutuantes (floating) ou autoelevatorias (jack-up)..
 # sent_id = 334-20150521-MONOGRAFIA_0-237
 1	Já	já	ADV	_	_	11	advmod	O	_
-2	quanto	quanto	ADV	_	_	5	case	O	_
+2	quanto	quanto	ADP	_	_	5	case	O	MWE=quanto_a|MWEPOS=ADP
 3-4	à	_	_	_	_	_	_	O	_
-3	a	a	ADP	_	_	5	case	O	_
+3	a	a	ADP	_	_	2	fixed	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
 5	estrutura	estrutura	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 6	de	de	ADP	_	_	7	case	O	_
@@ -119248,7 +119248,7 @@
 
 # text = Além da planta de processamento primário, as plataformas ainda possuem diversos sistemas (entre eles sistema de recuperação secundária e sistema de injeção de água do mar e/ou produzida) e redes de apoio, como sistemas de alívio, drenagem, ar comprimido, diesel, resfriamento e aquecimento (VAZ, 2009)..
 # sent_id = 334-20150521-MONOGRAFIA_0-273
-1	Além	além	ADV	_	_	4	cc	O	_
+1	Além	além	ADV	_	_	4	cc	O	MWE=além_de|MWEPOS=CCONJ
 2-3	da	_	_	_	_	_	_	O	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
@@ -127938,7 +127938,7 @@
 27	de	de	ADP	_	_	28	case	O	_
 28	regeneração	regeneração	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 29	,	,	PUNCT	_	_	33	punct	O	_
-30	além	além	ADV	_	_	33	cc	O	_
+30	além	além	ADV	_	_	33	cc	O	MWE=além_de|MWEPOS=CCONJ
 31-32	da	_	_	_	_	_	_	O	_
 31	de	de	ADP	_	_	30	fixed	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
@@ -129598,9 +129598,9 @@
 2	pré-sal	pré-sal	NOUN	_	Gender=Masc|Number=Sing	3	nsubj	O	_
 3	apresenta	apresentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 4	vantagens	vantagem	NOUN	_	Gender=Fem|Number=Plur	3	obj	O	_
-5	quanto	quanto	ADV	_	_	8	case	O	_
+5	quanto	quanto	ADP	_	_	8	case	O	MWE=quanto_a|MWEPOS=ADP
 6-7	à	_	_	_	_	_	_	O	_
-6	a	a	ADP	_	_	8	case	O	_
+6	a	a	ADP	_	_	5	fixed	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	reinjeção	reinjeção	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
@@ -152651,7 +152651,7 @@
 15	de	de	ADP	_	_	16	case	O	_
 16	charneira	charneira	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
-18	domos	domo	NOUN	_	Gender=Masc|Number=Plur	16	conj	O	_
+18	domos	domo	NOUN	_	Gender=Masc|Number=Plur	14	conj	O	_
 19	salinos	salino	ADJ	_	Gender=Masc|Number=Plur	18	amod	O	_
 20-21	na	_	_	_	_	_	_	O	_
 20	em	em	ADP	_	_	22	case	O	_
@@ -168798,7 +168798,7 @@
 20	cujos	cujo	DET	_	Gender=Masc|Number=Plur|PronType=Rel	22	det	O	_
 21	bons	bom	ADJ	_	Gender=Masc|Number=Plur	22	amod	O	_
 22	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	23	nsubj	O	_
-23	indicam	indicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	17	acl:relcl	O	_
+23	indicam	indicar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	3	acl:relcl	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
 25	viabilidade	viabilidade	NOUN	_	Gender=Fem|Number=Sing	23	obj	O	_
 26	prática	prático	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
@@ -198309,7 +198309,7 @@
 23	qual	qual	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	26	obl	O	_
 24	,	,	PUNCT	_	_	23	punct	O	_
 25	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	26	cop	O	_
-26	possível	possível	ADJ	_	Gender=Masc|Number=Sing	15	acl:relcl	O	_
+26	possível	possível	ADJ	_	Gender=Masc|Number=Sing	11	acl:relcl	O	_
 27	alterar	alterar	VERB	_	VerbForm=Inf	26	csubj	O	_
 28	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	29	det	O	_
 29	dados	dado	NOUN	_	Gender=Masc|Number=Plur	27	obj	O	_
@@ -209992,9 +209992,9 @@
 
 # text = Devido ao sistema desenvolvido por Griffin, a seleção de um tensoativo (ou mistura de tensoativos) mais adequado para cada tipo de emulsão, ou seja, água em óleo (A/O) ou óleo em água (O/A) tornou-se mais simples e racional..
 # sent_id = 391-20160914-MONOGRAFIA_0-72
-1	Devido	devido	ADV	_	_	4	case	O	_
+1	Devido	devido	ADV	_	_	4	case	O	MWE=devido_a|MWEPOS=ADP
 2-3	ao	_	_	_	_	_	_	O	_
-2	a	a	ADP	_	_	4	case	O	_
+2	a	a	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 5	desenvolvido	desenvolver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	4	acl	O	_
@@ -213501,7 +213501,7 @@
 26	xileno	xileno	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 27	como	como	ADP	_	_	29	case	O	_
 28	poluente	poluente	ADJ	_	Gender=Masc|Number=Sing	29	amod	O	_
-29	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	23	nmod:appos	O	_
+29	modelo	modelo	NOUN	_	Gender=Masc|Number=Sing	28	nmod	O	_
 30	demonstrou	demonstrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	1	ccomp	O	_
 31	ser	ser	AUX	_	VerbForm=Inf	32	aux:pass	O	_
 32	apropriada	apropriar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	30	xcomp	O	_
@@ -259001,8 +259001,8 @@
 8	inseridos	inserir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	6	acl	O	_
 9-10	nestes	_	_	_	_	_	_	O	_
 9	em	em	ADP	_	_	11	case	O	_
-10	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	11	det	O	_
-11	maiores	grande	ADJ	_	Gender=Masc|Number=Plur	8	obl	O	_
+10	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	8	obl	O	_
+11	maiores	grande	ADJ	_	Gender=Masc|Number=Plur	10	amod	O	_
 12	,	,	PUNCT	_	_	8	punct	O	_
 13	conforme	conforme	ADP	_	_	15	case	O	_
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
@@ -291093,7 +291093,7 @@
 3	fluidos	fluido	NOUN	_	Gender=Masc|Number=Plur	7	nsubj	O	_
 4	de	de	ADP	_	_	5	case	O	_
 5	base	base	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
-6	óleo	óleo	ADJ	_	Gender=Masc|Number=Sing	5	amod	B=FLUIDO	_
+6	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	5	nmod	B=FLUIDO	_
 7	tenham	ter	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	25	advcl	O	_
 8	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
 9	únicas	único	ADJ	_	Gender=Fem|Number=Plur	8	amod	O	_
@@ -291115,7 +291115,7 @@
 25	possui	possuir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 26	algumas	algum	DET	_	Gender=Fem|Number=Plur|PronType=Ind	27	det	O	_
 27	dificuldades	dificuldade	NOUN	_	Gender=Fem|Number=Plur	25	obj	O	_
-28	como	como	ADV	_	_	31	case	O	_
+28	como	como	ADP	_	_	31	case	O	_
 29	:	:	PUNCT	_	_	31	punct	O	_
 30	alto	alto	ADJ	_	Gender=Masc|Number=Sing	31	amod	O	_
 31	custo	custo	NOUN	_	Gender=Masc|Number=Sing	27	appos	O	_
@@ -295055,7 +295055,7 @@
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	Goma	Goma	PROPN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
 8	Guar	Guar	PROPN	_	Number=Sing	7	flat:name	O	_
-9	forma	forma	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+9	forma	formar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	gel	gel	NOUN	_	Gender=Masc|Number=Sing	9	obj	O	_
 12	altamente	altamente	ADV	_	_	13	advmod	O	_
@@ -295070,7 +295070,7 @@
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	indústria	indústria	NOUN	_	Gender=Fem|Number=Sing	18	obl:agent	O	_
 22	como	como	ADP	_	_	23	case	O	_
-23	espessante	espessante	NOUN	_	Gender=Masc|Number=Sing	21	nmod:appos	O	_
+23	espessante	espessante	NOUN	_	Gender=Masc|Number=Sing	18	xcomp	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
 25	gelificante	gelificante	NOUN	_	Gender=Masc|Number=Sing	23	conj	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
@@ -319861,7 +319861,7 @@
 6	de	de	ADP	_	_	7	case	O	_
 7	classificações	classificação	NOUN	_	Gender=Fem|Number=Plur	5	nmod	O	_
 8	reológicas	reológico	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
-9	quanto	quanto	ADV	_	_	12	case	O	_
+9	quanto	quanto	ADP	_	_	12	case	O	MWE=quanto_a|MWEPOS=ADP
 10-11	à	_	_	_	_	_	_	O	_
 10	a	a	ADP	_	_	9	fixed	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
@@ -319869,9 +319869,9 @@
 13	,	,	PUNCT	_	_	3	punct	O	_
 14	e	e	CCONJ	_	_	19	cc	O	_
 15	também	também	ADV	_	_	19	advmod	O	_
-16	quanto	quanto	ADV	_	_	19	case	O	_
+16	quanto	quanto	ADP	_	_	19	case	O	MWE=quanto_a|MWEPOS=ADP
 17-18	à	_	_	_	_	_	_	O	_
-17	a	a	ADP	_	_	19	case	O	_
+17	a	a	ADP	_	_	16	fixed	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 19	relação	relação	NOUN	_	Gender=Fem|Number=Sing	3	obl	O	_
 20	entre	entre	ADP	_	_	22	case	O	_
@@ -322942,7 +322942,7 @@
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	13	obl	O	_
 18	ativa	ativo	ADJ	_	Gender=Fem|Number=Sing	17	amod	O	_
-19	quando	quando	ADV	_	_	21	case	O	_
+19	quando	quando	ADV	_	_	21	advmod	O	_
 20	em	em	ADP	_	_	21	case	O	_
 21	solução	solução	NOUN	_	Gender=Fem|Number=Sing	13	obl	B=QUÍMICA	_
 22	aquosa	aquoso	ADJ	_	Gender=Fem|Number=Sing	21	amod	I=QUÍMICA	_
@@ -327007,9 +327007,9 @@
 56	tipos	tipo	NOUN	_	Gender=Masc|Number=Plur	52	nmod	O	_
 57	de	de	ADP	_	_	58	case	O	_
 58	viscosificante	viscosificante	NOUN	_	Gender=Masc|Number=Plur	56	nmod	B=QUÍMICA	_
-59	frente	frente	ADV	_	_	62	case	O	_
+59	frente	frente	ADV	_	_	62	case	O	MWE=frente_a|MWEPOS=ADP
 60-61	ao	_	_	_	_	_	_	O	_
-60	a	a	ADP	_	_	62	case	O	_
+60	a	a	ADP	_	_	59	fixed	O	_
 61	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	62	det	O	_
 62	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	56	nmod	O	_
 63	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	62	acl	O	_
@@ -333109,7 +333109,7 @@
 70	%	%	SYM	_	_	65	nmod:appos	O	_
 71	)	)	PUNCT	_	_	67	punct	O	_
 72	,	,	PUNCT	_	_	75	punct	O	_
-73	bem	bem	ADP	_	_	75	case	O	_
+73	bem	bem	ADP	_	_	75	cc	O	MWE=bem_como|MWEPOS=CCONJ
 74	como	como	ADP	_	_	73	fixed	O	_
 75	metais	metal	NOUN	_	Gender=Masc|Number=Plur	65	nmod:appos	O	_
 76	como	como	ADP	_	_	77	case	O	_
@@ -354768,4 +354768,3 @@
 54	.	.	PUNCT	_	_	2	punct	O	_
 55	.	.	PUNCT	_	_	2	punct	O	_
 
-
