commit 5eba1aa6265970f3e1275bcc5a10adc8b305aa39
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 22 18:54:41 2021 -0300

    Correção de preposições erradas

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index c6979b2..3325dae 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1021,7 +1021,7 @@
 15-16	nas	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
 16	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	17	det	O	_
-17	rochas	rocha	NOUN	_	Gender=Fem|Number=Plur	14	obj	B=LITOLOGIA	_
+17	rochas	rocha	NOUN	_	Gender=Fem|Number=Plur	14	obl	B=LITOLOGIA	_
 18-19	da	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
@@ -2806,7 +2806,7 @@
 6-7	na	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	porção	porção	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
+8	porção	porção	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 9	meridional	meridional	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 10-11	da	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	12	case	O	_
@@ -4282,7 +4282,7 @@
 57	recoberta	recobrir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	45	acl:relcl	O	_
 58	em	em	ADP	_	_	59	case	O	_
 59	discordância	discordância	NOUN	_	Gender=Fem|Number=Sing	57	obl	B=LITOLOGIA	_
-60	tanto	tanto	ADV	_	_	57	advmod	O	_
+60	tanto	tanto	ADV	_	_	63	advmod	O	_
 61-62	pelos	_	_	_	_	_	_	_	_
 61	por	por	ADP	_	_	63	case	O	_
 62	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	63	det	O	_
@@ -4292,11 +4292,11 @@
 65	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	66	det	O	_
 66	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	63	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 67	Ariri	Ariri	PROPN	_	Number=Sing	66	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
-68	quanto	quanto	ADV	_	_	71	case	O	_
+68	quanto	quanto	CCONJ	_	_	71	cc	O	_
 69-70	pela	_	_	_	_	_	_	_	_
 69	por	por	ADP	_	_	71	case	O	_
 70	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	71	det	O	_
-71	seção	seção	NOUN	_	Gender=Fem|Number=Sing	57	obl	O	_
+71	seção	seção	NOUN	_	Gender=Fem|Number=Sing	63	conj	O	_
 72	carbonática	carbonático	ADJ	_	Gender=Fem|Number=Sing	71	amod	O	_
 73-74	da	_	_	_	_	_	_	_	_
 73	de	de	ADP	_	_	75	case	O	_
@@ -5221,7 +5221,7 @@
 26-27	na	_	_	_	_	_	_	_	_
 26	em	em	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
+28	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
 29	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	22	acl	O	_
 30-31	à	_	_	_	_	_	_	_	_
 30	a	a	ADP	_	_	32	case	O	_
@@ -5276,7 +5276,7 @@
 33-34	no	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
-35	centro	centro	NOUN	_	Gender=Masc|Number=Sing	32	obj	O	_
+35	centro	centro	NOUN	_	Gender=Masc|Number=Sing	32	obl	O	_
 36-37	da	_	_	_	_	_	_	_	_
 36	de	de	ADP	_	_	38	case	O	_
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	O	_
@@ -5444,7 +5444,7 @@
 18-19	na	_	_	_	_	_	_	_	_
 18	em	em	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
+20	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
 21	transversalmente	transversalmente	ADV	_	_	17	advmod	O	_
 22-23	à	_	_	_	_	_	_	_	_
 22	a	a	ADP	_	_	24	case	O	_
@@ -5926,7 +5926,7 @@
 2	rifte	rifte	NOUN	_	Gender=Masc|Number=Sing	3	nsubj	B=LOCAL:GEO	_
 3	atuou	atuar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 4	em	em	ADP	_	_	5	case	O	_
-5	áreas	área	NOUN	_	Gender=Fem|Number=Plur	3	obj	O	_
+5	áreas	área	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 6	onde	onde	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	8	obl	O	_
 7	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	8	cop	O	_
 8	localizadas	localizar	ADJ	_	Gender=Fem|Number=Plur	5	acl:relcl	O	_
@@ -7024,7 +7024,7 @@
 5-6	no	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
-7	diagrama	diagrama	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
+7	diagrama	diagrama	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	rosetas	roseta	NOUN	_	Gender=Fem|Number=Plur	7	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
@@ -10257,7 +10257,7 @@
 25-26	no	_	_	_	_	_	_	_	_
 25	em	em	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
-27	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	24	obj	O	_
+27	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	24	obl	O	_
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
@@ -12560,7 +12560,7 @@
 15	transforme	transformar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	acl:relcl	O	_
 16	em	em	ADP	_	_	18	case	O	_
 17	pequenas	pequeno	ADJ	_	Gender=Fem|Number=Plur	18	amod	O	_
-18	partículas	partícula	NOUN	_	Gender=Fem|Number=Plur	15	obl	O	_
+18	partículas	partícula	NOUN	_	Gender=Fem|Number=Plur	15	obj	O	_
 19	ou	ou	CCONJ	_	_	20	cc	O	_
 20	glóbulos	glóbulo	NOUN	_	Gender=Masc|Number=Plur	18	conj	O	_
 21	(	(	PUNCT	_	_	22	punct	O	_
@@ -12915,7 +12915,7 @@
 18-19	na	_	_	_	_	_	_	_	_
 18	em	em	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	17	obj	O	_
+20	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	17	obl	O	_
 21-22	do	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
@@ -13543,7 +13543,7 @@
 12	,	,	PUNCT	_	_	13	punct	O	_
 13	atuando	atuar	VERB	_	VerbForm=Ger	8	advcl	O	_
 14	em	em	ADP	_	_	15	case	O	_
-15	condições	condição	NOUN	_	Gender=Fem|Number=Plur	13	obj	O	_
+15	condições	condição	NOUN	_	Gender=Fem|Number=Plur	13	obl	O	_
 16	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	18	nsubj	O	_
 17	eram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	18	cop	O	_
 18	restritas	restrito	ADJ	_	Gender=Fem|Number=Plur	15	acl:relcl	O	_
@@ -14402,7 +14402,7 @@
 16	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	17	expl	O	_
 17	transforma	transformar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 18	em	em	ADP	_	_	19	case	O	_
-19	vapor	vapor	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
+19	vapor	vapor	NOUN	_	Gender=Masc|Number=Sing	17	obj	O	_
 20	fazendo	fazer	VERB	_	VerbForm=Ger	17	advcl	O	_
 21	com	com	SCONJ	_	_	25	mark	O	_
 22	que	que	SCONJ	_	_	25	mark	O	_
@@ -14413,7 +14413,7 @@
 27	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	28	expl	O	_
 28	transformem	transformar	VERB	_	Mood=Sub|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	25	conj	O	_
 29	em	em	ADP	_	_	30	case	O	_
-30	flocos	floco	NOUN	_	Gender=Masc|Number=Plur	28	obl	O	_
+30	flocos	floco	NOUN	_	Gender=Masc|Number=Plur	28	obj	O	_
 31	sanfonados	sanfonar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	30	acl	O	_
 32	.	.	PUNCT	_	_	17	punct	O	_
 33	.	.	PUNCT	_	_	17	punct	O	_
@@ -15101,7 +15101,7 @@
 12-13	na	_	_	_	_	_	_	_	_
 12	em	em	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	proporção	proporção	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	proporção	proporção	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	30	30	NUM	_	NumType=Card	14	nummod	O	_
 17	até	até	ADP	_	_	19	case	O	_
@@ -16695,7 +16695,7 @@
 12	devendo	dever	VERB	_	VerbForm=Ger	5	advcl	O	_
 13	ficar	ficar	VERB	_	VerbForm=Inf	12	xcomp	O	_
 14	em	em	ADP	_	_	15	case	O	_
-15	banho-maria	banho-maria	NOUN	_	Gender=Masc|Number=Sing	13	obj	B=MÉTODO	_
+15	banho-maria	banho-maria	NOUN	_	Gender=Masc|Number=Sing	13	obl	B=MÉTODO	_
 16	até	até	SCONJ	_	_	17	mark	O	_
 17	atingir	atingir	VERB	_	VerbForm=Inf	13	advcl	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -19783,7 +19783,7 @@
 5-6	nas	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
-7	proximidades	proximidade	NOUN	_	Gender=Fem|Number=Plur	3	obj	O	_
+7	proximidades	proximidade	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 8-9	das	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
@@ -21732,7 +21732,7 @@
 57	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	58	nsubj	O	_
 58	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	55	acl:relcl	O	_
 59	ocorrer	ocorrer	VERB	_	VerbForm=Inf	58	xcomp	O	_
-60	tanto	tanto	ADV	_	_	59	advmod	O	_
+60	tanto	tanto	ADV	_	_	62	advmod	O	_
 61	entre	entre	ADP	_	_	62	case	O	_
 62	estratos	estrato	NOUN	_	Gender=Masc|Number=Plur	59	obl	B=LITOLOGIA	_
 63	levemente	levemente	ADV	_	_	64	advmod	O	_
@@ -21749,7 +21749,7 @@
 73	downlap	downlap	NOUN	_	Gender=Fem|Number=Sing	70	appos	O	_
 74	)	)	PUNCT	_	_	73	punct	O	_
 75	,	,	PUNCT	_	_	78	punct	O	_
-76	quanto	quanto	ADV	_	_	78	case	O	_
+76	quanto	quanto	CCONJ	_	_	78	cc	O	_
 77	entre	entre	ADP	_	_	78	case	O	_
 78	estratos	estrato	NOUN	_	Gender=Masc|Number=Plur	62	conj	B=LITOLOGIA	_
 79	concordantes	concordante	ADJ	_	Gender=Masc|Number=Plur	78	amod	O	_
@@ -22828,7 +22828,7 @@
 5-6	na	_	_	_	_	_	_	_	_
 5	em	em	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	porção	porção	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
+7	porção	porção	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8	leste	leste	ADJ	_	Gender=Masc|Number=Sing	7	amod	O	_
 9-10	da	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
@@ -23600,11 +23600,11 @@
 36	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	flat:name	O	_
 37	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	34	flat:name	B=BACIA|B=LOCAL:POL	_
 38	,	,	PUNCT	_	_	31	punct	O	_
-39	quanto	quanto	ADV	_	_	42	case	O	_
+39	quanto	quanto	CCONJ	_	_	42	cc	O	_
 40-41	na	_	_	_	_	_	_	_	_
 40	em	em	ADP	_	_	42	case	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
-42	porção	porção	NOUN	_	Gender=Fem|Number=Sing	31	obl	O	_
+42	porção	porção	NOUN	_	Gender=Fem|Number=Sing	26	conj	O	_
 43	sul	sul	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
 44	,	,	PUNCT	_	_	45	punct	O	_
 45	correspondente	correspondente	ADJ	_	Gender=Fem|Number=Sing	42	amod	O	_
@@ -24646,7 +24646,7 @@
 
 # text = Do mesmo modo, Santos (1982) apresenta uma composição faunística coerente para ambiente marinho raso..
 # sent_id = 284-20150917-MONOGRAFIA_0-154
-1	Do	Do	ADP	_	_	3	case	O	_
+1	Do	Do	ADP	_	_	3	case	O	tokenização
 2	mesmo	mesmo	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 3	modo	modo	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
@@ -25198,7 +25198,7 @@
 
 # text = Do mesmo modo, Chagas (2006) utilizando a categoria de membro, interpreta a sucessão de litofácies carbonato-pelíticas como ambiente lagunar transicionando para ambientes marinhos rasos de águas muito calmas, evidenciado pela preservação de peixes marinhos nas concreções carbonáticas..
 # sent_id = 284-20150917-MONOGRAFIA_0-167
-1	Do	Do	ADP	_	_	3	case	O	_
+1	Do	Do	ADP	_	_	3	case	O	tokenização
 2	mesmo	mesmo	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 3	modo	modo	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
@@ -25437,8 +25437,8 @@
 16	sedimentação	sedimentação	NOUN	_	Gender=Fem|Number=Sing	17	nsubj	O	_
 17	ocorreu	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	ccomp	O	_
 18	em	em	ADP	_	_	20	case	O	_
-19	paleoambiente	paleoambiente	ADV	_	_	20	advmod	O	_
-20	marinho	marinho	NOUN	_	Gender=Masc|Number=Sing	17	obj	O	_
+19	paleoambiente	paleoambiente	NOUN	_	_	17	obl	O	_
+20	marinho	marinho	ADJ	_	Gender=Masc|Number=Sing	19	amod	O	_
 21	restrito	restrito	ADJ	_	Gender=Masc|Number=Sing	20	amod	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
 23	baseado	basear	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	O	_
@@ -45838,7 +45838,7 @@
 
 # text = Do ponto de vista químico, eles podem assumir aspectos de suspensão, dispersão coloidal ou emulsão, dependendo do estado físico dos componentes (THOMAS, 2001)..
 # sent_id = 24-20150122-MONOGRAFIA_0-34
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -46939,7 +46939,7 @@
 
 # text = Do ponto de vista industrial para aplicação em fluidos de perfuração, a água doce não necessita de pré-tratamento químico porque praticamente não afeta o desempenho dos aditivos empregados no preparo do fluido..
 # sent_id = 24-20150122-MONOGRAFIA_0-66
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -68272,7 +68272,7 @@
 # text = Próximo a Presidente Olegário (MG) foram descritos neste estudo pacotes siliciclásticos relativamente espessos (até 60 m) pertencentes a esta formação..
 # sent_id = 283-20150917-MONOGRAFIA_0-127
 1	Próximo	próximo	ADV	_	_	9	advmod	O	_
-2	a	o	ADP	_	_	3	case	O	_
+2	a	a	ADP	_	_	3	case	O	_
 3	Presidente	Presidente	PROPN	_	Gender=Masc|Number=Sing	9	nsubj:pass	O	_
 4	Olegário	Olegário	PROPN	_	Number=Sing	3	flat:name	O	_
 5	(	(	PUNCT	_	_	6	punct	O	_
@@ -73190,7 +73190,7 @@
 5	de	de	ADP	_	_	6	case	O	_
 6	granulometria	granulometria	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 7	fina	fino	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
-8	a	o	ADP	_	_	9	case	O	_
+8	a	a	ADP	_	_	9	case	O	_
 9	grossa	grosso	NOUN	_	Gender=Fem|Number=Sing	6	amod	O	_
 10	,	,	PUNCT	_	_	11	punct	O	_
 11	seleção	seleção	NOUN	_	Gender=Fem|Number=Sing	6	conj	O	_
@@ -74637,7 +74637,7 @@
 25-26	encontra-se	_	_	_	_	_	_	_	_
 25	encontra	encontrar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 26	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	25	expl	O	_
-27	muitas	muitas	ADP	_	_	28	case	O	_
+27	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	28	det	O	_
 28	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	25	obl	O	_
 29	deformada	deformar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	25	xcomp	O	_
 30	plasticamente	plasticamente	ADV	_	_	29	advmod	O	_
@@ -75410,7 +75410,7 @@
 24	métricas	métrico	ADJ	_	Gender=Fem|Number=Plur	23	amod	O	_
 25	,	,	PUNCT	_	_	29	punct	O	_
 26	e	e	CCONJ	_	_	29	cc	O	_
-27	muitas	muitas	ADP	_	_	28	case	O	_
+27	muitas	muito	DET	Gender=Fem|Number=Plur|PronType=Ind	_	28	det	O	_
 28	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	29	obl	O	_
 29-30	encontravam-se	_	_	_	_	_	_	_	_
 29	encontravam	encontrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	18	conj	O	_
@@ -76877,7 +76877,7 @@
 1	Ambientes	ambiente	NOUN	_	Gender=Masc|Number=Plur	8	nsubj	O	_
 2	de	de	ADP	_	_	3	case	O	_
 3	interduna	interduna	NOUN	_	Gender=Fem|Number=Sing	1	nmod	O	_
-4	muitas	muitas	ADP	_	_	5	case	O	_
+4	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	5	det	O	_
 5	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	3	nmod	O	_
 6	eram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	8	cop	O	_
 7	bem	bem	ADV	_	_	8	advmod	O	_
@@ -89383,7 +89383,7 @@
 
 # text = No caso da tecnologia Top Gas Recycling Blast Furnace (TGRBF) para altos-fornos siderúrgicos, os custos estarão associados as etapas de geração de oxigênio puro, reciclagem do gás e a separação do CO2..
 # sent_id = 161-20150810-MONOGRAFIA_0-372
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 3-4	da	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -91405,7 +91405,7 @@
 2	photos	photo	NOUN	_	Gender=Masc|Number=Plur	1	compound	O	_
 3	/	/	PUNCT	_	_	1	punct	O	_
 4	romulocam	romulocar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-5	pos	pos	ADP	_	_	6	case	O	_
+5	pos	pos	ADP	_	_	6	case	O	sentenciação
 6	/5769663608	/5769663608	NUM	_	NumType=Card	4	obl	O	_
 7	/	/	PUNCT	_	_	4	punct	O	_
 
@@ -96068,10 +96068,10 @@
 40	campo	campo	NOUN	_	Gender=Masc|Number=Sing	35	obl	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
 41	de	de	ADP	_	_	42	case	O	_
 42	Marlim	Marlim	PROPN	_	Gender=Masc|Number=Sing	40	nmod	B=CAMPO	_
-43	quanto	quanto	ADV	_	_	46	case	O	_
+43	quanto	quanto	CCONJ	_	_	46	cc	O	_
 44	para	para	ADP	_	_	46	case	O	_
 45	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
-46	campo	campo	NOUN	_	Gender=Masc|Number=Sing	40	nmod	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
+46	campo	campo	NOUN	_	Gender=Masc|Number=Sing	40	conj	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
 47	de	de	ADP	_	_	48	case	O	_
 48	Barracuda	Barracuda	PROPN	_	Gender=Masc|Number=Sing	46	nmod	B=CAMPO	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
@@ -102503,7 +102503,7 @@
 
 # text = No caso do CCGS, a adicionalidade estará diretamente ligada à questão da quantificação do armazenamento do dióxido de carbono em comparação com as emissões das fontes onde houve a captura..
 # sent_id = 161-20150810-MONOGRAFIA_0-742
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -102713,7 +102713,7 @@
 
 # text = No caso do CCGS, as emissões de projeto são equivalentes às emissões que irão ocorrer em razão das atividades de captura, transporte e injeção de CO2 abatendo-se, desse valor, a quantidade de gás carbônico que foi capturado..
 # sent_id = 161-20150810-MONOGRAFIA_0-748
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -110605,7 +110605,7 @@
 
 # text = No caso da Petrobras, a maior parte do investimento já era direcionada para exploração e produção (E&P) antes da descoberta do pré-sal..
 # sent_id = 334-20150521-MONOGRAFIA_0-49
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 3-4	da	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -114024,7 +114024,7 @@
 
 # text = Muitas vezes a descoberta de jazidas de gás natural se dá em função da pesquisa exploratória em busca do petróleo..
 # sent_id = 334-20150521-MONOGRAFIA_0-138
-1	Muitas	muitas	ADP	_	_	2	case	O	_
+1	Muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	2	case	O	_
 2	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	11	obl	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	descoberta	descoberta	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
@@ -115352,7 +115352,7 @@
 35	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	29	acl:relcl	O	_
 36	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	37	expl	O	_
 37	enquadrar	enquadrar	VERB	_	VerbForm=Inf	35	xcomp	O	_
-38	tanto	tanto	ADV	_	_	37	advmod	O	_
+38	tanto	tanto	ADV	_	_	41	advmod	O	_
 39-40	na	_	_	_	_	_	_	_	_
 39	em	em	ADP	_	_	41	case	O	_
 40	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	41	det	O	_
@@ -115365,11 +115365,11 @@
 46	de	de	ADP	_	_	48	case	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	det	O	_
 48	produto	produto	NOUN	_	Gender=Masc|Number=Sing	45	nmod	O	_
-49	quanto	quanto	ADV	_	_	52	case	O	_
+49	quanto	quanto	CCONJ	_	_	52	cc	O	_
 50-51	na	_	_	_	_	_	_	_	_
 50	em	em	ADP	_	_	52	case	O	_
 51	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	52	det	O	_
-52	especificação	especificação	NOUN	_	Gender=Fem|Number=Sing	37	obl	O	_
+52	especificação	especificação	NOUN	_	Gender=Fem|Number=Sing	41	conj	O	_
 53-54	do	_	_	_	_	_	_	_	_
 53	de	de	ADP	_	_	55	case	O	_
 54	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	55	det	O	_
@@ -120691,7 +120691,7 @@
 31	produzidos	produzir	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	30	acl	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
 33	elevados	elevar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	31	conj	O	_
-34	ate	ate	ADP	_	_	36	case	O	_
+34	ate	até	ADP	_	_	36	case	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
 36	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	33	obl	O	_
 37	e	e	CCONJ	_	_	41	cc	O	_
@@ -122218,7 +122218,7 @@
 9	operar	operar	VERB	_	VerbForm=Inf	7	acl	O	_
 10	continuamente	continuamente	ADV	_	_	9	advmod	O	_
 11	com	com	ADP	_	_	14	case	O	_
-12	ate	ate	ADP	_	_	14	case	O	_
+12	ate	até	ADP	_	_	14	case	O	_
 13	95	95	NUM	_	NumType=Card	14	nummod	O	_
 14	%	%	SYM	_	_	9	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
@@ -126131,7 +126131,7 @@
 
 # text = Pra alguns desses produtos ainda não se tem o claro conhecimento de como estes afetam o crescimento ou a morfologia do hidrato..
 # sent_id = 334-20150521-MONOGRAFIA_0-450
-1	Pra	pra	ADP	_	_	2	case	O	_
+1	Pra	para	ADP	_	_	2	case	O	_
 2	alguns	algum	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	9	obl	O	_
 3-4	desses	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -141736,7 +141736,7 @@
 
 # text = Do ponto de vista econômico, seria mais rentável diminuir o tanto quanto possível a quantidade de tensoativo necessária para promover a estabilização completa da emulsão durante as 4 horas de observação..
 # sent_id = 202-20140404-MONOGRAFIA_0-137
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -148795,7 +148795,7 @@
 
 # text = A cada fase tectônica deste processo corresponde um pacote sedimentar cujas características são o reflexo do respectivo contexto geológico em que esteve inserida a bacia no momento da deposição..
 # sent_id = 258-20160602-TESEMSC_0-160
-1	A	o	ADP	_	_	3	case	O	_
+1	A	a	ADP	_	_	3	case	O	_
 2	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	3	det	O	_
 3	fase	fase	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
 4	tectônica	tectônico	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
@@ -155507,7 +155507,7 @@
 13	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 14	interpretados	interpretar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	13	acl	O	_
 15	,	,	PUNCT	_	_	18	punct	O	_
-16	muitas	muitas	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	case	O	_
+16	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	det	O	_
 17	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	18	obl	O	_
 18	realizada	realizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	acl	O	_
 19-20	pela	_	_	_	_	_	_	_	_
@@ -155591,7 +155591,7 @@
 1	Tais	tais	DET	_	Gender=Fem|Number=Plur|PronType=Dem	2	det	O	_
 2	pistas	pista	NOUN	_	Gender=Fem|Number=Plur	7	nsubj	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
-4	muitas	muitas	ADP	_	_	5	case	O	_
+4	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	5	det	O	_
 5	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	7	nmod	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	chave	chave	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
@@ -155978,7 +155978,7 @@
 
 # text = No caso da ocorrência de falhas verticais ocorre um aumento progressivo da anomalia gravimétrica ao longo do lado soerguido, interpretado como sendo devido ao deslocamento para cima de material mais denso, o que representa um contraste horizontal de densidade (LOWRIE, 2007)..
 # sent_id = 258-20160602-TESEMSC_0-320
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 3-4	da	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -164488,7 +164488,7 @@
 57	em	em	ADP	_	_	58	case	O	_
 58	média	média	NOUN	_	Gender=Fem|Number=Sing	56	obl	O	_
 59	40º	40º	NOUN	_	Gender=Masc|NumType=Ord|Number=Sing	58	nmod	O	_
-60	pra	pra	ADP	_	_	61	case	O	_
+60	pra	para	ADP	_	_	61	case	O	_
 61	oeste	oeste	NOUN	_	Gender=Masc|Number=Sing	59	obl	O	_
 62	.	.	PUNCT	_	_	42	punct	O	_
 63	.	.	PUNCT	_	_	42	punct	O	_
@@ -168174,7 +168174,7 @@
 
 # text = Do ponto de vista ambiental, faz sentido se os óleos pesados fossem usados como matéria-prima das refinarias ao invés de simplesmente combustíveis para queimar..
 # sent_id = 77-20141209-TESEMSC_0-79
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -169782,7 +169782,7 @@
 37	com	com	ADP	_	_	39	case	O	_
 38	apenas	apenas	ADV	_	_	39	advmod	O	_
 39	água	água	NOUN	_	Gender=Fem|Number=Sing	34	nmod	B=FLUIDO	_
-40	ate	ate	ADP	_	_	42	case	O	_
+40	ate	até	ADP	_	_	42	case	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
 42	queda	queda	NOUN	_	Gender=Fem|Number=Sing	39	nmod	O	_
 43	de	de	ADP	_	_	44	case	O	_
@@ -170479,7 +170479,7 @@
 
 # text = No caso do fluido 1 ser óleo e o fluido 2 ser água, para θ > 90° diz-se que o sólido é hidrofílico e para θ < 90° o sólido é oleofílico..
 # sent_id = 77-20141209-TESEMSC_0-140
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -193854,7 +193854,7 @@
 
 # text = Muitas vezes são usados catalisadores ditos heterogêneos, que na verdade lixiviam e atuam como homogêneos..
 # sent_id = 271-20150810-MONOGRAFIA_0-120
-1	Muitas	muitas	ADP	_	_	2	case	O	_
+1	Muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	2	det	O	_
 2	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	usados	usar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -198728,7 +198728,7 @@
 
 # text = No caso do catalisador dopado com Li, foi também considerado o preço do reagente utilizado para dopagem..
 # sent_id = 271-20150810-MONOGRAFIA_0-277
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3-4	do	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -201147,7 +201147,7 @@
 
 # text = No caso da catálise homogênea é comum usar razão molar álcool:óleo de 6:1, embora a razão estequiométrica seja de 3:1..
 # sent_id = 271-20150810-MONOGRAFIA_0-363
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	caso	caso	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 3-4	da	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -201600,10 +201600,10 @@
 16	para	para	ADP	_	_	17	case	O	_
 17	óleos	óleo	NOUN	_	Gender=Masc|Number=Plur	8	obl	B=FLUIDO	_
 18	residuais	residual	ADJ	_	Gender=Masc|Number=Plur	17	amod	I=FLUIDO	_
-19	quanto	quanto	ADV	_	_	22	case	O	_
+19	quanto	quanto	CCONJ	_	_	22	cc	O	_
 20	para	para	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
-22	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	17	nmod	B=PRODUTO|B=FLUIDO	_
+22	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	17	conj	B=PRODUTO|B=FLUIDO	_
 23	comercial	comercial	ADJ	_	Gender=Masc|Number=Sing	22	amod	I=PRODUTO	_
 24	,	,	PUNCT	_	_	25	punct	O	_
 25	indicando	indicar	VERB	_	VerbForm=Ger	8	advcl	O	_
@@ -206701,11 +206701,11 @@
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
 13	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	10	nmod	B=PRODUTO|B=FLUIDO	_
 14	comercial	comercial	ADJ	_	Gender=Masc|Number=Sing	13	amod	I=PRODUTO	_
-15	quanto	quanto	ADV	_	_	18	case	O	_
+15	quanto	quanto	CCONJ	_	_	18	cc	O	_
 16-17	do	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	13	nmod	B=FLUIDO	_
+18	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	13	conj	B=FLUIDO	_
 19	residual	residual	ADJ	_	Gender=Masc|Number=Sing	18	amod	I=FLUIDO	_
 20	,	,	PUNCT	_	_	10	punct	O	_
 21	houve	haver	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
@@ -212055,7 +212055,7 @@
 
 # text = Pra remoção dos compostos citados, são usados vários processos físicos, químicos e biológicos (FAKHRU’L-RAZI et al., 2009)..
 # sent_id = 391-20160914-MONOGRAFIA_0-126
-1	Pra	pra	ADP	_	_	2	case	O	_
+1	Pra	para	ADP	_	_	2	case	O	_
 2	remoção	remoção	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 3-4	dos	_	_	_	_	_	_	_	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -237544,7 +237544,7 @@
 38	tensões	tensão	NOUN	_	Gender=Fem|Number=Plur	36	nmod	O	_
 39	propícios	propício	ADJ	_	Gender=Fem|Number=Plur	38	amod	O	_
 40	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	45	nsubj	O	_
-41	muitas	muitas	ADP	_	_	42	case	O	_
+41	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	42	det	O	_
 42	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	45	obl	O	_
 43	não	não	ADV	_	Polarity=Neg	45	advmod	O	_
 44	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	45	expl	O	_
@@ -239195,7 +239195,7 @@
 20	domínios	domínio	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
 21	escarpados	escarpar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	20	acl	O	_
 22	,	,	PUNCT	_	_	24	punct	O	_
-23	muitas	muitas	ADP	_	_	24	case	O	_
+23	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	24	det	O	_
 24	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
 25	de	de	ADP	_	_	27	case	O	_
 26	difícil	difícil	ADJ	_	Gender=Masc|Number=Sing	27	amod	O	_
@@ -247019,7 +247019,7 @@
 
 # text = Do ponto de vista geológico, pode se associar, comparativamente à ocorrência da Ilha de Cabo Frio, a um plug principal formado pelo antigo duto magmático que deu origem à suíte alcalina, acompanhado a leste por um possante dique anelar oriundo do mesmo processo magmático..
 # sent_id = 20-20140904-TESEDSC_0-391
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -256804,7 +256804,7 @@
 3	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	A)	A)	PROPN	_	Gender=Fem|Number=Sing	2	obj	O	_
 5			PROPN	_	Number=Sing	4	flat:name	O	_
-6	¹	¹	ADP	_	_	8	case	O	_
+6	¹	¹	ADP	_	_	8	case	O	sentenciação
 7	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	h0	h0	PROPN	_	Gender=Fem|Number=Sing	2	obl	O	_
 
@@ -277729,7 +277729,7 @@
 
 # text = Do ponto de vista magnético, e também gravimétrico (Mohriak e Barros 1990), esta falha é menos expressiva que a anterior, que condiciona o flanco sul e apresenta um maior rejeito..
 # sent_id = 20-20140904-TESEDSC_0-1154
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -280761,7 +280761,7 @@
 
 # text = Do ponto de vista quantitativo, a metodologia desenvolvida se baseia na conjugação de três métodos, isto é, amplitude do sinal analítico, deconvolução de Euler e inversão compacta generalizada..
 # sent_id = 20-20140904-TESEDSC_0-1222
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -288187,14 +288187,14 @@
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
 8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	6	nsubj:pass	O	_
 9	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	O	_
-10	tanto	tanto	ADV	_	_	9	advmod	O	_
+10	tanto	tanto	ADV	_	_	13	advmod	O	_
 11	para	para	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
 13	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	9	obl	B=QUÍMICA	_
-14	quanto	quanto	ADV	_	_	17	case	O	_
+14	quanto	quanto	CCONJ	_	_	17	cc	O	_
 15	para	para	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
-17	GX	GX	PROPN	_	Gender=Fem|Number=Sing	9	obl	O	_
+17	GX	GX	PROPN	_	Gender=Fem|Number=Sing	13	conj	O	_
 18	,	,	PUNCT	_	_	21	punct	O	_
 19	e	e	CCONJ	_	_	21	cc	O	_
 20	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	aux:pass	O	_
@@ -288830,7 +288830,7 @@
 
 # text = Do ponto de vista químico, eles podem assumir aspectos de suspensão, dispersão coloidal ou emulsão, dependendo do estado físico dos componentes (Thomas, 2001)..
 # sent_id = 119-20141209-TESEMSC_0-82
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -291048,7 +291048,7 @@
 
 # text = Do ponto de vista operacional, particularmente em áreas remotas de operações offshore, o custo do transporte de água doce para certas áreas pode ser proibitivo (Chilingarian & Vorabutr 1983)..
 # sent_id = 119-20141209-TESEMSC_0-143
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	26	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -291431,7 +291431,7 @@
 23	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 24	ser	ser	AUX	_	VerbForm=Inf	25	aux:pass	O	_
 25	melhorada	melhorar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	23	xcomp	O	_
-26	muitas	muitas	ADP	_	_	27	case	O	_
+26	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	27	cc	O	_
 27	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	25	obl	O	_
 28	,	,	PUNCT	_	_	29	punct	O	_
 29	tornando	tornar	VERB	_	VerbForm=Ger	25	advcl	O	_
@@ -292985,7 +292985,7 @@
 11	de	de	ADP	_	_	12	case	O	_
 12	água	água	NOUN	_	Gender=Fem|Number=Sing	10	nmod	B=FLUIDO	_
 13	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	16	cop	O	_
-14	muitas	muitas	ADP	_	_	15	case	O	_
+14	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	15	cc	O	_
 15	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	16	obl	O	_
 16	adicionadas	adicionar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	0	root	O	_
 17-18	na	_	_	_	_	_	_	_	_
@@ -300827,7 +300827,7 @@
 8	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	B=FLUIDO	_
 9	e	e	CCONJ	_	_	10	cc	O	_
 10	gás	gás	NOUN	_	Gender=Masc|Number=Sing	8	conj	B=FLUIDO	_
-11	muitas	muitas	ADP	_	_	12	case	O	_
+11	muitas	muito	DET	_	Gender=Fem|Number=Plur|PronType=Ind	12	cc	O	_
 12	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	10	nmod	O	_
 13	requer	requerer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 14	altas	alto	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
@@ -301287,9 +301287,9 @@
 25	tanto	tanto	ADV	_	_	27	advmod	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	24	nmod	B=LITOLOGIA	_
-28	quanto	quanto	ADV	_	_	30	case	O	_
+28	quanto	quanto	CCONJ	_	_	30	cc	O	_
 29	de	de	ADP	_	_	30	case	O	_
-30	completação	completação	NOUN	_	Gender=Fem|Number=Sing	27	nmod	O	_
+30	completação	completação	NOUN	_	Gender=Fem|Number=Sing	27	conj	O	_
 31	.	.	PUNCT	_	_	17	punct	O	_
 32	.	.	PUNCT	_	_	17	punct	O	_
 
@@ -304729,7 +304729,7 @@
 
 # text = No ensaio, além dos fluidos de perfuração, foi utilizado óleo mineral (~ 24 cp @23º C)..
 # sent_id = 119-20141209-TESEMSC_0-547
-1	No	no	ADP	_	_	2	case	O	_
+1	No	no	ADP	_	_	2	case	O	tokenização
 2	ensaio	ensaio	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	além	além	ADV	_	_	7	cc	O	MWEPOS=CCONJ
@@ -306307,14 +306307,14 @@
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
 5	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	3	nsubj:pass	O	_
 6	obtidos	obter	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	5	acl	O	_
-7	tanto	tanto	ADV	_	_	6	advmod	O	_
+7	tanto	tanto	ADV	_	_	10	advmod	O	_
 8	para	para	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	6	obl	B=QUÍMICA	_
-11	quanto	quanto	ADV	_	_	14	case	O	_
+11	quanto	quanto	CCONJ	_	_	14	cc	O	_
 12	para	para	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	GX..	GX..	PROPN	_	Gender=Fem|Number=Sing	6	obl	O	_
+14	GX..	GX..	PROPN	_	Gender=Fem|Number=Sing	10	conj	O	_
 
 # text = Os resultados são comparados com vistas à influência tanto da concentração de argila como do tipo de polímero..
 # sent_id = 119-20141209-TESEMSC_0-597
@@ -310926,10 +310926,10 @@
 16	para	para	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	GX	GX	PROPN	_	Gender=Fem|Number=Sing	6	obl	O	_
-19	quanto	quanto	ADV	_	_	22	case	O	_
+19	quanto	quanto	CCONJ	_	_	22	cc	O	_
 20	para	para	ADP	_	_	22	case	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
-22	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	18	nmod	B=QUÍMICA	_
+22	PHPA	PHPA	PROPN	_	Gender=Fem|Number=Sing	18	conj	B=QUÍMICA	_
 23	,	,	PUNCT	_	_	18	punct	O	_
 24	para	para	SCONJ	_	_	27	mark	O	_
 25	que	que	SCONJ	_	_	27	mark	O	_
@@ -312234,7 +312234,7 @@
 
 # text = Do mesmo modo que os experimentos realizados com a PHPA, foi realizado também o estudo reológico dos filtrados produzidos nos experimentos com a GX..
 # sent_id = 119-20141209-TESEMSC_0-738
-1	Do	Do	ADP	_	_	3	case	O	_
+1	Do	Do	ADP	_	_	3	case	O	tokenização
 2	mesmo	mesmo	ADJ	_	Gender=Masc|Number=Sing	3	amod	O	_
 3	modo	modo	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
 4	que	que	SCONJ	_	_	13	mark	O	_
@@ -316091,7 +316091,7 @@
 
 # text = Do ponto de vista químico, o fluido assume aspectos de dispersão coloidal ou emulsão, dependendo da temperatura..
 # sent_id = 240-20140220-MONOGRAFIA_0-20
-1	Do	Do	ADP	_	_	2	case	O	_
+1	Do	Do	ADP	_	_	2	case	O	tokenização
 2	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	vista	vista	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -333387,8 +333387,8 @@
 27	não	não	ADV	_	Polarity=Neg	29	advmod	O	_
 28	quando	quando	SCONJ	_	_	29	mark	O	_
 29	lançados	lançar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	25	advcl	O	_
-30	n	n	ADP	_	_	32	case	O	_
-31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
+30	n	n	ADP	_	_	32	case	O	tokenização
+31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	tokenização
 32	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
 33	d'água	d'água	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
 34	,	,	PUNCT	_	_	29	punct	O	_
@@ -334053,7 +334053,7 @@
 12	em	em	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
-15	d’água	d’água	ADP	_	_	19	case	O	_
+15	d’água	d’água	NOUN	_	_	14	nmod	O	_
 16	logo	logo	ADV	_	_	11	advmod	O	_
 17	após	após	ADP	_	_	19	case	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
@@ -337517,7 +337517,7 @@
 38	local	local	NOUN	_	Gender=Masc|Number=Sing	35	conj	O	_
 39	para	para	ADP	_	_	40	case	O	_
 40	reprodução	reprodução	NOUN	_	Gender=Fem|Number=Sing	35	nmod	O	_
-41	pra	pra	ADP	_	_	42	case	O	_
+41	pra	para	ADP	_	_	42	case	O	_
 42	espécies	espécie	NOUN	_	Gender=Fem|Number=Plur	40	nmod	O	_
 43	de	de	ADP	_	_	44	case	O	_
 44	moluscos	molusco	NOUN	_	Gender=Masc|Number=Plur	42	nmod	O	_
