commit 36c40d555e50791ebc581305f64bd676347bb7ef
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Jul 14 09:28:27 2021 -0300

    Segundo_ADP

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 50324d0..537a91d 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -12696,7 +12696,7 @@
 # text = Segundo Bauer (1999), a emulsão asfáltica é uma dispersão composta de 50 a 70% de cimento asfáltico, pequenas frações de emulsificante (1 a 2%) e o restante de água..
 # sent_id = 241-20140227-MONOGRAFIA_0-65
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Bauer	Bauer	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
+2	Bauer	Bauer	PROPN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	1999	1999	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -36001,14 +36001,14 @@
 12	eletromagnética	eletromagnético	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	em	em	ADP	_	_	14	case	O	_
 14	térmica	térmico	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
-15	e	e	CCONJ	_	_	17	cc	O	_
+15	e	e	CCONJ	_	_	23	cc	O	_
 16	segundo	segundo	ADP	_	_	17	case	O	_
-17	Manichand	Manichand	PROPN	_	Gender=Masc|Number=Sing	14	conj	O	_
+17	Manichand	Manichand	PROPN	_	Gender=Masc|Number=Sing	23	obl	O	_
 18	(	(	PUNCT	_	_	17	punct	O	_
 19	2002	2002	NUM	_	NumType=Card	17	flat:name	O	_
 20	)	)	PUNCT	_	_	17	punct	O	_
-21	,	,	PUNCT	_	_	5	punct	O	_
-22	se	se	PRON	_	Case=Acc|Gender=Unsp|Person=3|PronType=Prs	23	expl	O	_
+21	,	,	PUNCT	_	_	17	punct	O	_
+22	se	se	PRON	_	_	23	expl	O	_
 23	deve	dever	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	conj	O	_
 24	através	através	ADV	_	_	27	case	O	MWEPOS=ADP
 25-26	da	_	_	_	_	_	_	_	_
@@ -36030,7 +36030,7 @@
 39	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	40	det	O	_
 40	meio	meio	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
 41	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	42	nsubj	O	_
-42	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	40	acl:relcl	O	_
+42	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	35	acl:relcl	O	_
 43	ser	ser	AUX	_	VerbForm=Inf	44	cop	O	_
 44	íons	íom	NOUN	_	Gender=Masc|Number=Plur	42	xcomp	O	_
 45	ou	ou	CCONJ	_	_	46	cc	O	_
@@ -37362,9 +37362,9 @@
 
 # text = A medida que o petróleo é extraído e transportado desde o poço à superfície, vai perdendo calor para o ambiente, com esta diminuição da temperatura, segundo Morán (2007), mais precisamente no ponto de névoa, cada componente parafínico torna-se menos solúvel até que a cristalização da parafina se inicie..
 # sent_id = 398-20160721-MONOGRAFIA_0-135
-1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-2	medida	medida	ADP	_	_	3	case	O	_
-3	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	7	obl	O	_
+1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	mark	O	MWEPOS=SCONJ
+2	medida	medida	ADP	_	_	1	fixed	O	_
+3	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	1	fixed	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	7	nsubj:pass	B=FLUIDO	_
 6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	O	_
@@ -37377,38 +37377,38 @@
 13-14	à	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
-15	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
-16	,	,	PUNCT	_	_	18	punct	O	_
+15	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
+16	,	,	PUNCT	_	_	7	punct	O	_
 17	vai	ir	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	aux	O	_
 18	perdendo	perder	VERB	_	VerbForm=Ger	0	root	O	_
 19	calor	calor	NOUN	_	Gender=Masc|Number=Sing	18	obj	O	_
 20	para	para	ADP	_	_	22	case	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	ambiente	ambiente	NOUN	_	Gender=Masc|Number=Sing	18	obl	O	_
-23	,	,	PUNCT	_	_	18	punct	O	_
+23	,	,	PUNCT	_	_	48	punct	O	_
 24	com	com	ADP	_	_	26	case	O	_
 25	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	26	det	O	_
-26	diminuição	diminuição	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
+26	diminuição	diminuição	NOUN	_	Gender=Fem|Number=Sing	48	obl	O	_
 27-28	da	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	29	case	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 30	,	,	PUNCT	_	_	32	punct	O	_
 31	segundo	segundo	ADP	_	_	32	case	O	_
-32	Morán	Morán	PROPN	_	Gender=Fem|Number=Sing	26	nmod	O	_
+32	Morán	Morán	PROPN	_	Gender=Fem|Number=Sing	48	obl	O	_
 33	(	(	PUNCT	_	_	34	punct	O	_
 34	2007	2007	NUM	_	NumType=Card	32	flat:name	O	_
 35	)	)	PUNCT	_	_	34	punct	O	_
-36	,	,	PUNCT	_	_	38	punct	O	_
+36	,	,	PUNCT	_	_	41	punct	O	_
 37	mais	mais	ADV	_	_	38	advmod	O	_
-38	precisamente	precisamente	ADV	_	_	48	advmod	O	_
+38	precisamente	precisamente	ADV	_	_	41	advmod	O	_
 39-40	no	_	_	_	_	_	_	_	_
 39	em	em	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
-41	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	38	obl	O	_
+41	ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	48	obl	O	_
 42	de	de	ADP	_	_	43	case	O	_
 43	névoa	névoa	NOUN	_	Gender=Fem|Number=Sing	41	nmod	O	_
-44	,	,	PUNCT	_	_	38	punct	O	_
+44	,	,	PUNCT	_	_	41	punct	O	_
 45	cada	cada	DET	_	Gender=Masc|Number=Sing|PronType=Tot	46	det	O	_
 46	componente	componente	NOUN	_	Gender=Masc|Number=Sing	48	nsubj	O	_
 47	parafínico	parafínico	ADJ	_	Gender=Masc|Number=Sing	46	amod	O	_
@@ -37480,10 +37480,10 @@
 4	deposição	deposição	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 5	,	,	PUNCT	_	_	7	punct	O	_
 6	segundo	segundo	ADP	_	_	7	case	O	_
-7	Morán	Morán	PROPN	_	Gender=Fem|Number=Sing	2	nmod	O	_
+7	Morán	Morán	PROPN	_	Gender=Fem|Number=Sing	13	obl	O	_
 8	(	(	PUNCT	_	_	7	flat:name	O	_
 9	2007	2007	NUM	_	NumType=Card	7	flat:name	O	_
-10	)	)	PUNCT	_	_	7	punct	O	_
+10	)	)	PUNCT	_	_	7	flat:name	O	_
 11	,	,	PUNCT	_	_	7	punct	O	_
 12	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	13	expl	O	_
 13	deve	dever	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -37496,11 +37496,11 @@
 20	difusão	difusão	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
 21	browniana	browniano	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 22	,	,	PUNCT	_	_	23	punct	O	_
-23	dispersão	dispersão	NOUN	_	Gender=Fem|Number=Sing	15	conj	O	_
+23	dispersão	dispersão	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
 24	por	por	ADP	_	_	25	case	O	_
 25	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
-27	deposição	deposição	NOUN	_	Gender=Fem|Number=Sing	15	conj	O	_
+27	deposição	deposição	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
 28	gravitacional	gravitacional	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
 29	.	.	PUNCT	_	_	13	punct	O	_
 30	.	.	PUNCT	_	_	13	punct	O	_
@@ -37709,10 +37709,10 @@
 # text = Segundo COMSOL (2013) o problema da análise eletromagnética num nível macroscópico é a de resolver as equações de Maxwell sujeitas a certas condições de contorno..
 # sent_id = 398-20160721-MONOGRAFIA_0-145
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	COMSOL	COMSOL	PROPN	_	Gender=Masc|Number=Sing	17	nmod	O	_
-3	(	(	PUNCT	_	_	4	punct	O	_
+2	COMSOL	COMSOL	PROPN	_	Gender=Masc|Number=Sing	17	obl	O	_
+3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2013	2013	NOUN	_	Gender=Masc|Number=Plur	2	flat:name	O	_
-5	)	)	PUNCT	_	_	4	punct	O	_
+5	)	)	PUNCT	_	_	2	flat:name	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	problema	problema	NOUN	_	Gender=Masc|Number=Sing	17	nsubj	O	_
 8-9	da	_	_	_	_	_	_	_	_
@@ -37723,7 +37723,7 @@
 12-13	num	_	_	_	_	_	_	_	_
 12	em	em	ADP	_	_	14	case	O	_
 13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	nível	nível	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
+14	nível	nível	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 15	macroscópico	macroscópico	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	cop	O	_
 17	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	0	root	O	_
@@ -38681,7 +38681,7 @@
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	2013	2013	NOUN	_	Gender=Masc|Number=Plur	12	nmod:appos	O	_
 15	)	)	PUNCT	_	_	14	punct	O	_
-16	,	,	PUNCT	_	_	18	punct	O	_
+16	,	,	PUNCT	_	_	12	punct	O	_
 17	nenhum	nenhum	DET	_	Gender=Masc|Number=Sing|PronType=Neg	18	det	O	_
 18	campo	campo	NOUN	_	Gender=Masc|Number=Sing	6	conj	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
 19	elétrico	elétrico	ADJ	_	Gender=Masc|Number=Sing	18	amod	I=CAMPO:TIPO	_
@@ -40190,20 +40190,20 @@
 1	Ela	ela	PRON	_	Case=Nom|Gender=Fem|Number=Sing|Person=3|PronType=Prs	2	nsubj	O	_
 2	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 3	como	como	ADP	_	_	4	case	O	_
-4	consequência	consequência	NOUN	_	Gender=Fem|Number=Sing	2	xcomp	O	_
+4	consequência	consequência	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	diferentes	diferente	DET	_	Gender=Masc|Number=Plur|PronType=Ind	7	det	O	_
 7	mecanismos	mecanismo	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 8	em	em	ADP	_	_	10	case	O	_
 9	diferentes	diferente	DET	_	Gender=Masc|Number=Plur|PronType=Ind	10	det	O	_
 10	meios	meio	NOUN	_	Gender=Masc|Number=Plur	7	nmod	O	_
-11	e	e	CCONJ	_	_	13	cc	O	_
+11	e	e	CCONJ	_	_	20	cc	O	_
 12	segundo	segundo	ADP	_	_	13	case	O	_
-13	COMSOL	COMSOL	PROPN	_	Gender=Masc|Number=Sing	10	conj	O	_
-14	(	(	PUNCT	_	_	15	punct	O	_
+13	COMSOL	COMSOL	PROPN	_	Gender=Masc|Number=Sing	20	nmod	O	_
+14	(	(	PUNCT	_	_	13	flat:name	O	_
 15	2006	2006	NUM	_	NumType=Card	13	flat:name	O	_
-16	)	)	PUNCT	_	_	15	punct	O	_
-17	,	,	PUNCT	_	_	2	punct	O	_
+16	)	)	PUNCT	_	_	13	flat:name	O	_
+17	,	,	PUNCT	_	_	13	punct	O	_
 18	através	através	ADV	_	_	20	case	O	MWEPOS=ADP
 19	de	de	ADP	_	_	18	fixed	O	_
 20	colisões	colisão	NOUN	_	Gender=Fem|Number=Plur	2	obl	O	_
@@ -64723,7 +64723,7 @@
 # text = Segundo Sgarbi (2011), as principais feições geomorfológicas da área são morrotes de morfologia plana, colinas baixas, cristas elevadas e vales encaixados..
 # sent_id = 283-20150917-MONOGRAFIA_0-19
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	15	nmod	O	_
+2	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	15	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2011	2011	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -72154,7 +72154,7 @@
 # text = Segundo Briere (2000), sistema lacustre é àquele que passa ao menos 75% do tempo com água..
 # sent_id = 283-20150917-MONOGRAFIA_0-282
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Briere	Briere	PROPN	_	Gender=Masc|Number=Sing	11	nmod	O	_
+2	Briere	Briere	PROPN	_	Gender=Masc|Number=Sing	11	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2000	2000	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -72280,7 +72280,7 @@
 # text = Segundo Nichols (2009), lagos efêmeros são massas de água que secam periodicamente, típicos de ambientes áridos e semi-áridos, com longos períodos secos interrompidos por chuvas torrenciais..
 # sent_id = 283-20150917-MONOGRAFIA_0-286
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Nichols	Nichols	PROPN	_	Gender=Masc|Number=Sing	10	nmod	O	_
+2	Nichols	Nichols	PROPN	_	Gender=Masc|Number=Sing	10	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2009	2009	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -99260,7 +99260,7 @@
 # sent_id = 161-20150810-MONOGRAFIA_0-649
 1	Segundo	segundo	ADP	_	_	3	case	O	_
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
-3	IEA	IEA	PROPN	_	Gender=Fem|Number=Sing	16	nmod	O	_
+3	IEA	IEA	PROPN	_	Gender=Fem|Number=Sing	16	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
 6	taxas	taxa	NOUN	_	Gender=Fem|Number=Plur	16	nsubj	O	_
@@ -117906,7 +117906,7 @@
 
 # text = Segundo Morais (2013), o FPSO
 # sent_id = 334-20150521-MONOGRAFIA_0-243
-1	Segundo	segundo	ADP	_	_	2	case	O	_
+1	Segundo	segundo	ADP	_	_	2	case	O	sentenciação
 2	Morais	Morais	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2013	2013	NOUN	_	Gender=Masc|Number=Plur	2	flat:name	O	_
@@ -121588,7 +121588,7 @@
 17	2.000	2.000	PROPN	_	Number=Sing	16	flat:name	O	_
 18	(	(	PUNCT	_	_	20	punct	O	_
 19	segundo	segundo	ADP	_	_	20	case	O	_
-20	Programa	Programa	PROPN	_	Gender=Masc|Number=Sing	16	nmod	O	_
+20	Programa	Programa	PROPN	_	Gender=Masc|Number=Sing	16	obl	O	_
 21	de	de	ADP	_	_	20	flat:name	O	_
 22	Capacitação	Capacitação	PROPN	_	Number=Sing	20	flat:name	O	_
 23	em	em	ADP	_	_	20	flat:name	O	_
@@ -148993,7 +148993,7 @@
 
 # text = 3.2.1 Embasamento Segundo Moreira et al. (2007) o embasamento cristalino da Bacia de Santos é caracterizado por granitos e gnaisses de idade pré-cambriana pertencentes ao Complexo Costeiro e metassedimentos da Faixa Ribeira, sobrepostos pelo embasamento econômico da Bacia de Santos definido pelos basaltos da Formação Camboriú..
 # sent_id = 258-20160602-TESEMSC_0-166
-1	3.2.1	3.2.1	NUM	_	NumType=Card	2	nummod	O	_
+1	3.2.1	3.2.1	NUM	_	NumType=Card	2	nummod	O	sentenciação
 2	Embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	19	nsubj:pass	B=UNIDADE_LITOESTRATIGRÁFICA	_
 3	Segundo	segundo	ADP	_	_	4	case	O	_
 4	Moreira	Moreira	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
@@ -160230,7 +160230,7 @@
 # text = Segundo Zalán (1986), nas bacias de Santos e Campos, essas estruturas em flor são indicadores de transcorrência..
 # sent_id = 258-20160602-TESEMSC_0-426
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Zalán	Zalán	PROPN	_	Gender=Masc|Number=Sing	20	nmod	O	_
+2	Zalán	Zalán	PROPN	_	Gender=Masc|Number=Sing	20	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	1986	1986	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -164434,7 +164434,7 @@
 9	de	de	ADP	_	_	10	case	O	_
 10	falha	falha	NOUN	_	Gender=Fem|Number=Sing	8	nmod	B=LITOLOGIA|B=LOCAL:GEO	_
 11	(	(	PUNCT	_	_	12	punct	O	_
-12	um	um	NUM	_	NumType=Card	8	appos	O	_
+12	um	um	PRON	_	Gender=Masc|Number=Sing|PronType=Ind	5	appos	O	_
 13-14	no	_	_	_	_	_	_	_	_
 13	em	em	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
@@ -164463,10 +164463,10 @@
 33	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	30	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 34	,	,	PUNCT	_	_	36	punct	O	_
 35	segundo	segundo	ADP	_	_	36	case	O	_
-36	Constantino	Constantino	PROPN	_	Gender=Masc|Number=Sing	30	nmod	O	_
+36	Constantino	Constantino	PROPN	_	Gender=Masc|Number=Sing	42	obl	O	_
 37	(	(	PUNCT	_	_	38	punct	O	_
-38	2012	2012	NUM	_	NumType=Card	36	flat:name	O	_
-39	)	)	PUNCT	_	_	36	punct	O	_
+38	2012	2012	NUM	_	NumType=Card	36	nmod:appos	O	_
+39	)	)	PUNCT	_	_	38	punct	O	_
 40	,	,	PUNCT	_	_	36	punct	O	_
 41	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	42	cop	O	_
 42	possível	possível	ADJ	_	Gender=Masc|Number=Sing	0	root	O	_
@@ -164481,15 +164481,15 @@
 50	de	de	ADP	_	_	51	case	O	_
 51	falha	falha	NOUN	_	Gender=Fem|Number=Sing	49	nmod	B=LITOLOGIA|B=LOCAL:GEO	_
 52	mapeado	mapear	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	49	acl	O	_ ou mergulho (45)
-53	,	,	PUNCT	_	_	52	punct	O	_
+53	,	,	PUNCT	_	_	56	punct	O	_
 54	o	o	DET	_	Gender=Masc|Number=Sing|PronType=Art	55	det	O	_
 55	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	56	nsubj	O	_
-56	mergulha	mergulhar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	acl:relcl	O	_
+56	mergulha	mergulhar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	49	acl:relcl	O	_
 57	em	em	ADP	_	_	58	case	O	_
 58	média	média	NOUN	_	Gender=Fem|Number=Sing	56	obl	O	_
 59	40º	40º	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	58	amod	O	_
 60	pra	pra	ADP	_	_	61	case	O	_
-61	oeste	oeste	NOUN	_	Gender=Masc|Number=Sing	58	nmod	O	_
+61	oeste	oeste	NOUN	_	Gender=Masc|Number=Sing	59	obl	O	_
 62	.	.	PUNCT	_	_	42	punct	O	_
 63	.	.	PUNCT	_	_	42	punct	O	_
 
@@ -173249,10 +173249,10 @@
 19	1,253x10-6	1,253x10-6	NUM	_	NumType=Card	21	nummod	O	_
 20	m3	m3	NUM	_	NumType=Card	21	nummod	O	_
 21	/s	/s	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
-22	,	,	PUNCT	_	_	21	punct	O	_
+22	,	,	PUNCT	_	_	25	punct	O	_
 23	segundo	segundo	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
-25	carta	carta	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
+25	carta	carta	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	calibração	calibração	NOUN	_	Gender=Fem|Number=Sing	25	nmod	O	_
 28	e	e	CCONJ	_	_	29	cc	O	_
@@ -182110,9 +182110,9 @@
 # sent_id = 279-20140530-MONOGRAFIA_0-7
 1	Segundo	segundo	ADP	_	_	3	case	O	_
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
-3	ANP	ANP	PROPN	_	Gender=Fem|Number=Sing	24	nmod	O	_
+3	ANP	ANP	PROPN	_	Gender=Fem|Number=Sing	24	obl	O	_
 4	[	[	PUNCT	_	_	5	punct	O	_
-5	1	1	PROPN	_	Gender=Fem|Number=Sing	3	appos	O	_
+5	1	1	PROPN	_	Gender=Fem|Number=Sing	3	nmod:appos	O	_
 6	]	]	PUNCT	_	_	5	punct	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
 8	Agência	Agência	PROPN	_	Gender=Fem|Number=Sing	3	appos	O	_
@@ -182135,7 +182135,7 @@
 24	combustível	combustível	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 25	composto	compor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	24	acl	O	_
 26	de	de	ADP	_	_	27	case	O	_
-27	mono-ésteres	mono-éster	NOUN	_	Gender=Masc|Number=Plur	24	nmod	O	_
+27	mono-ésteres	mono-éster	NOUN	_	Gender=Masc|Number=Plur	25	obl	O	_
 28	alquílicos	alquílico	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
 29	de	de	ADP	_	_	30	case	O	_
 30	ácidos	ácido	NOUN	_	Gender=Masc|Number=Plur	27	nmod	O	_
@@ -212170,7 +212170,7 @@
 # text = Segundo Samuel Feitoza (2012), a espectrofotometria é um método de análises bastante usado em investigações biológicas e físico-químicas..
 # sent_id = 391-20160914-MONOGRAFIA_0-131
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Samuel	Samuel	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
+2	Samuel	Samuel	PROPN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3	Feitoza	Feitoza	PROPN	_	Number=Sing	2	flat:name	O	_
 4	(	(	PUNCT	_	_	2	flat:name	O	_
 5	2012	2012	NUM	_	NumType=Card	2	flat:name	O	_
@@ -212474,11 +212474,11 @@
 
 # text = Absorção de Luz Segundo ELEUTÉRIO (2014), Einstein propôs, em 1905, que a luz é quantizada em “pacotes”, aos quais deu o nome de quantum..
 # sent_id = 391-20160914-MONOGRAFIA_0-140
-1	Absorção	absorção	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
+1	Absorção	absorção	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	sentenciação
 2	de	de	ADP	_	_	3	case	O	_
 3	Luz	Luz	PROPN	_	Gender=Fem|Number=Sing	1	nmod	O	_
 4	Segundo	segundo	ADP	_	_	5	case	O	_
-5	ELEUTÉRIO	ELEUTÉRIO	PROPN	_	Number=Sing	1	nmod	O	_
+5	ELEUTÉRIO	ELEUTÉRIO	PROPN	_	Number=Sing	11	obl	O	_
 6	(	(	PUNCT	_	_	7	punct	O	_
 7	2014	2014	NUM	_	NumType=Card	5	flat:name	O	_
 8	)	)	PUNCT	_	_	7	punct	O	_
@@ -226287,7 +226287,7 @@
 14	,	,	PUNCT	_	_	17	punct	O	_
 15	e	e	CCONJ	_	_	17	cc	O	_
 16	segundo	segundo	ADP	_	_	17	case	O	_
-17	Kawai	Kawai	PROPN	_	Gender=Masc|Number=Sing	11	conj	O	_
+17	Kawai	Kawai	PROPN	_	Gender=Masc|Number=Sing	29	conj	O	_
 18	e	e	CCONJ	_	_	17	flat:name	O	_
 19	Tsutsumi	Tsutsumi	PROPN	_	Gender=Masc|Number=Sing	17	flat:name	O	_
 20	[	[	PUNCT	_	_	21	punct	O	_
@@ -228207,27 +228207,27 @@
 10	de	de	ADP	_	_	11	case	O	_
 11	silício	silício	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 12	com	com	ADP	_	_	16	case	O	_
-13	um	um	NUM	_	NumType=Card	16	nummod	O	_
-14	ou	ou	CCONJ	_	_	16	cc	O	_
-15	nenhum	nenhum	DET	_	Gender=Masc|Number=Sing|PronType=Neg	16	det	O	_
+13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
+14	ou	ou	CCONJ	_	_	15	cc	O	_
+15	nenhum	nenhum	DET	_	Gender=Masc|Number=Sing|PronType=Neg	13	det	O	_
 16	átomo	átomo	NOUN	_	Gender=Masc|Number=Sing	9	conj	O	_
 17	de	de	ADP	_	_	18	case	O	_
 18	alumínio	alumínio	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 19	como	como	ADP	_	_	21	case	O	_
 20	segundo	segundo	ADP	_	_	21	case	O	_
-21	vizinho	vizinho	NOUN	_	Gender=Masc|Number=Sing	16	nmod:appos	O	_
+21	vizinho	vizinho	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 22	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	25	cop	O	_
 23	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	25	det	O	_
 24	mais	mais	ADV	_	_	25	advmod	O	_
 25	abundantes	abundante	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
 26	,	,	PUNCT	_	_	27	punct	O	_
-27	Si(1Al	Si(1Al	PROPN	_	Gender=Masc|Number=Sing	25	appos	O	_
+27	Si(1Al	Si(1Al	PROPN	_	Gender=Masc|Number=Sing	25	appos	O	tokenização
 28	)	)	PUNCT	_	_	27	punct	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
-30	Si(0Al	Si(0Al	PROPN	_	Gender=Masc|Number=Sing	25	conj	O	_
-31	)	)	PUNCT	_	_	25	punct	O	_
-32	,	,	PUNCT	_	_	25	punct	O	_
-33	respectivamente	respectivamente	ADV	_	_	25	advmod	O	_
+30	Si(0Al	Si(0Al	PROPN	_	Gender=Masc|Number=Sing	27	conj	O	_
+31	)	)	PUNCT	_	_	30	punct	O	_
+32	,	,	PUNCT	_	_	33	punct	O	_
+33	respectivamente	respectivamente	ADV	_	_	27	advmod	O	_
 34	.	.	PUNCT	_	_	25	punct	O	_
 35	.	.	PUNCT	_	_	25	punct	O	_
 
@@ -229238,7 +229238,7 @@
 24	segundo	segundo	ADP	_	_	27	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 26	seguinte	seguinte	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
-27	equação	equação	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
+27	equação	equação	NOUN	_	Gender=Fem|Number=Sing	22	obl	O	_
 28	:	:	PUNCT	_	_	22	punct	O	_
 29	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	30	det	O	_
 30	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	32	nsubj	O	_
@@ -237782,7 +237782,7 @@
 3	segundo	segundo	ADP	_	_	6	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 5	mesmo	mesmo	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
-6	autor	autor	NOUN	_	Gender=Masc|Number=Sing	9	nmod	O	_
+6	autor	autor	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 7	,	,	PUNCT	_	_	6	punct	O	_
 8	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
 9	diques	dique	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
@@ -241186,7 +241186,7 @@
 2	segundo	segundo	ADP	_	_	5	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
 4	mesmos	mesmo	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
-5	autores	autor	NOUN	_	Gender=Masc|Number=Plur	21	nmod	O	_
+5	autores	autor	NOUN	_	Gender=Masc|Number=Plur	21	obl	O	_
 6	,	,	PUNCT	_	_	5	punct	O	_
 7	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	situação	situação	NOUN	_	Gender=Fem|Number=Sing	21	nsubj	O	_
@@ -284906,7 +284906,7 @@
 17	,	,	PUNCT	_	_	20	punct	O	_
 18	segundo	segundo	ADP	_	_	20	case	O	_
 19	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	20	det	O	_
-20	autores	autor	NOUN	_	Gender=Masc|Number=Plur	22	nmod	O	_
+20	autores	autor	NOUN	_	Gender=Masc|Number=Plur	22	obl	O	_
 21	,	,	PUNCT	_	_	20	punct	O	_
 22	locais	local	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 23	de	de	ADP	_	_	25	case	O	_
@@ -291704,7 +291704,7 @@
 # text = Segundo Gallino et al., 2001, um fluido drill-in é um fluido com menor poder de dano, que pode ser utilizado na perfuração através de formações produtoras, especialmente quando longas seções horizontais devem ser perfuradas..
 # sent_id = 119-20141209-TESEMSC_0-159
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Gallino	Gallino	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
+2	Gallino	Gallino	PROPN	_	Gender=Masc|Number=Sing	14	obl	O	_
 3	et	et	PROPN	_	Number=Sing	2	flat:name	O	_
 4	al.	al.	PROPN	_	Number=Sing	2	flat:name	O	_
 5	,	,	PUNCT	_	_	2	flat:name	O	_
@@ -293475,7 +293475,7 @@
 # text = Segundo Darley & Gray, 1988, a bentonita é a argila comercial mais utilizada em fluidos à base de água doce..
 # sent_id = 119-20141209-TESEMSC_0-209
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Darley	Darley	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
+2	Darley	Darley	PROPN	_	Gender=Masc|Number=Sing	12	obl	O	_
 3	&	&	PROPN	_	Number=Sing	2	flat:name	O	_
 4	Gray	Gray	PROPN	_	Number=Sing	2	flat:name	O	_
 5	,	,	PUNCT	_	_	2	flat:name	O	_
@@ -294264,7 +294264,7 @@
 # text = Segundo Rosa et al., 2006, um polímero é uma molécula muito grande formada por milhares de blocos que se repetem, chamados monômeros..
 # sent_id = 119-20141209-TESEMSC_0-233
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Rosa	Rosa	PROPN	_	Gender=Fem|Number=Sing	12	nmod	O	_
+2	Rosa	Rosa	PROPN	_	Gender=Fem|Number=Sing	12	obl	O	_
 3	et	et	PROPN	_	Number=Sing	2	flat:name	O	_
 4	al.	al.	PROPN	_	Number=Sing	2	flat:name	O	_
 5	,	,	PUNCT	_	_	2	flat:name	O	_
@@ -300889,7 +300889,7 @@
 # sent_id = 119-20141209-TESEMSC_0-443
 1	Ainda	ainda	ADV	_	_	3	advmod	O	_
 2	segundo	segundo	ADP	_	_	3	case	O	_
-3	Iscan	Iscan	PROPN	_	Gender=Masc|Number=Sing	24	nmod	O	_
+3	Iscan	Iscan	PROPN	_	Gender=Masc|Number=Sing	24	obl	O	_
 4	et	et	PROPN	_	Number=Sing	3	flat:name	O	_
 5	al.	al.	PROPN	_	Number=Sing	3	flat:name	O	_
 6	,	,	PUNCT	_	_	3	flat:name	O	_
@@ -301215,7 +301215,7 @@
 # text = Segundo Gallino et al., 2001, um fluido drill-in é um fluido que minimiza o dano e que pode ser usado em perfurações através de formações produtoras, especialmente quando longas seções horizontais precisam ser perfuradas..
 # sent_id = 119-20141209-TESEMSC_0-453
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Gallino	Gallino	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
+2	Gallino	Gallino	PROPN	_	Gender=Masc|Number=Sing	14	obl	O	_
 3	et	et	PROPN	_	Number=Sing	2	flat:name	O	_
 4	al.	al.	PROPN	_	Number=Sing	2	flat:name	O	_
 5	,	,	PUNCT	_	_	2	flat:name	O	_
@@ -321358,7 +321358,7 @@
 # text = Segundo Machado (2002), fluidos de perfuração são exemplos de fluidos tixotrópicos..
 # sent_id = 240-20140220-MONOGRAFIA_0-188
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Machado	Machado	PROPN	_	Gender=Masc|Number=Sing	11	nmod	O	_
+2	Machado	Machado	PROPN	_	Gender=Masc|Number=Sing	11	obl	O	_
 3	(	(	PUNCT	_	_	2	flat:name	O	_
 4	2002	2002	NUM	_	NumType=Card	2	flat:name	O	_
 5	)	)	PUNCT	_	_	2	flat:name	O	_
@@ -321462,7 +321462,7 @@
 1	Densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	21	parataxis	O	_
 2	:	:	PUNCT	_	_	4	punct	O	_
 3	Segundo	segundo	ADP	_	_	4	case	O	_
-4	Annis	Annis	PROPN	_	Gender=Masc|Number=Sing	1	nmod	O	_
+4	Annis	Annis	PROPN	_	Gender=Masc|Number=Sing	21	obl	O	_
 5	e	e	CCONJ	_	_	4	flat:name	O	_
 6	Smith	Smith	PROPN	_	Gender=Masc|Number=Sing	4	flat:name	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
@@ -337281,7 +337281,7 @@
 # text = Segundo Duarte e Vieira (1997), o estuário é como a entrada de um braço de mar em uma bacia hidrográfica..
 # sent_id = 411-20150702-MONOGRAFIA_0-165
 1	Segundo	segundo	ADP	_	_	2	case	O	_
-2	Duarte	Duarte	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
+2	Duarte	Duarte	PROPN	_	Gender=Masc|Number=Sing	14	obl	O	_
 3	e	e	CCONJ	_	_	2	flat:name	O	_
 4	Vieira	Vieira	PROPN	_	Gender=Fem|Number=Sing	2	flat:name	O	_
 5	(	(	PUNCT	_	_	2	flat:name	O	_
