commit 44088d4fc25dc0c6252bbd65408d38d38795a22f
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Oct 19 14:27:39 2021 -0300

    correções de frases com 'colocar'

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 0ccdb6a..c1441e7 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -152768,18 +152768,18 @@
 18	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	21	nsubj	O	_
 19	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	cop	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
-21	sólidos	sólido	ADJ	_	Gender=Masc|Number=Plur	17	amod	O	_
+21	sólidos	sólido	ADJ	_	Gender=Masc|Number=Plur	13	acl:relcl	O	_
 22-23	na	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
 24	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 25-26	dos	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
-26	os	o	DET	_	Gender=Masc|Number=Plur|PronType=Art	27	det	O	_
-27	quais	qual	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	24	nmod	O	_
-28	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	29	expl	O	_
+26	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	27	det	O	_
+27	quais	qual	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	29	obl	O	_
+28	se	se	PRON	_	_	29	expl	O	_
 29	colocam	colocar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	24	acl:relcl	O	_
-30	complexos	complexo	ADJ	_	Gender=Masc|Number=Plur	29	xcomp	O	_
+30	complexos	complexo	NOUN	_	Gender=Masc|Number=Plur	29	obj	O	_
 31	de	de	ADP	_	_	32	case	O	_
 32	coordenação	coordenação	NOUN	_	Gender=Fem|Number=Sing	30	obl	O	_
 33	,	,	PUNCT	_	_	34	punct	O	_
