commit 2beb4fece79cf40fa68889ea0f297be7b284f67c
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Jul 27 11:31:01 2021 -0300

    Obl ou obj: lista de verbos seguidos de "por"

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b1ba9f5..b664fbf 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -40817,7 +40817,7 @@
 4-5	pelo	_	_	_	_	_	_	_	_
 4	por	por	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	fato	fato	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
+6	fato	fato	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7-8	da	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -43181,7 +43181,7 @@
 4-5	pelo	_	_	_	_	_	_	_	_
 4	por	por	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	fato	fato	NOUN	_	Gender=Masc|Number=Sing	3	obj	O	_
+6	fato	fato	NOUN	_	Gender=Masc|Number=Sing	3	obl	O	_
 7-8	da	_	_	_	_	_	_	_	_
 7	de	de	SCONJ	_	_	15	mark	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -46135,7 +46135,7 @@
 25-26	pela	_	_	_	_	_	_	_	_
 25	por	por	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
-27	broca	broca	NOUN	_	Gender=Fem|Number=Sing	24	obj	B=INSTRUMENTO	_
+27	broca	broca	NOUN	_	Gender=Fem|Number=Sing	24	obl	B=INSTRUMENTO	_
 28	e	e	CCONJ	_	_	29	cc	O	_
 29	retornando	retornar	VERB	_	VerbForm=Ger	24	conj	O	_
 30-31	à	_	_	_	_	_	_	_	_
@@ -80883,7 +80883,7 @@
 9	passa	passar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 10	por	por	ADP	_	_	12	case	O	_
 11	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
+12	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	absorção	absorção	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
 15	(	(	PUNCT	_	_	16	punct	O	_
@@ -84477,7 +84477,7 @@
 8-9	pela	_	_	_	_	_	_	_	_
 8	por	por	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
-10	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	7	obj	O	_
+10	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 11-12	do	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
@@ -114464,7 +114464,7 @@
 12-13	pela	_	_	_	_	_	_	_	_
 12	por	por	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	destinação	destinação	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	destinação	destinação	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15-16	dessa	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	18	case	O	_
 16	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	18	det	O	_
@@ -126201,7 +126201,7 @@
 14	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	15	nsubj	O	_
 15	interagem	interager	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	acl:relcl	O	_
 16	por	por	ADP	_	_	17	case	O	_
-17	ligação	ligação	NOUN	_	Gender=Fem|Number=Sing	15	obj	O	_
+17	ligação	ligação	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
 20	com	com	ADP	_	_	22	case	O	_
@@ -131462,7 +131462,7 @@
 8	passa	passar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 9	por	por	ADP	_	_	11	case	O	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	processo	processo	NOUN	_	Gender=Masc|Number=Sing	8	obj	O	_
+11	processo	processo	NOUN	_	Gender=Masc|Number=Sing	8	obl	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	separação	separação	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 14	.	.	PUNCT	_	_	8	punct	O	_
@@ -133023,7 +133023,7 @@
 15	passa	passar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	por	por	ADP	_	_	18	case	O	_
 17	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	aquecedor	aquecedor	NOUN	_	Gender=Masc|Number=Sing	15	obj	O	_
+18	aquecedor	aquecedor	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 19	para	para	SCONJ	_	_	20	mark	O	_
 20	ter	ter	VERB	_	VerbForm=Inf	15	advcl	O	_
 21	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	22	det	O	_
@@ -133304,7 +133304,7 @@
 13	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	14	nsubj	O	_
 14	passa	passar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 15	por	por	ADP	_	_	16	case	O	_
-16	despressurização	despressurização	NOUN	_	Gender=Fem|Number=Sing	14	obj	O	_
+16	despressurização	despressurização	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
 17	seguida	seguer	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	16	acl	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
@@ -133366,7 +133366,7 @@
 20	passa	passar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	12	conj	O	_
 21	por	por	ADP	_	_	23	case	O	_
 22	mais	mais	DET	_	Gender=Fem|Number=Sing|PronType=Ind	23	det	O	_
-23	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	20	obj	O	_
+23	etapa	etapa	NOUN	_	Gender=Fem|Number=Sing	20	obl	O	_
 24	de	de	ADP	_	_	25	case	O	_
 25	despressurização	despressurização	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
@@ -139486,7 +139486,7 @@
 6	ocorrer	ocorrer	VERB	_	VerbForm=Inf	5	xcomp	O	_
 7	por	por	ADP	_	_	9	case	O	_
 8	vários	vário	DET	_	Gender=Masc|Number=Plur|PronType=Ind	9	det	O	_
-9	mecanismos	mecanismo	NOUN	_	Gender=Masc|Number=Plur	6	obj	O	_
+9	mecanismos	mecanismo	NOUN	_	Gender=Masc|Number=Plur	6	obl	O	_
 10	,	,	PUNCT	_	_	13	punct	O	_
 11	como	como	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -149668,7 +149668,7 @@
 6-7	pela	_	_	_	_	_	_	_	_
 6	por	por	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	deposição	deposição	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	deposição	deposição	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	calcários	calcário	NOUN	_	Gender=Masc|Number=Plur	8	nmod	B=LITOLOGIA	_
 
@@ -151546,7 +151546,7 @@
 5	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	4	expl	O	_
 6	por	por	ADP	_	_	8	case	O	_
 7	dois	dois	NUM	_	NumType=Card	8	nummod	O	_
-8	altos	alto	NOUN	_	Gender=Masc|Number=Plur	4	obj	O	_
+8	altos	alto	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
 9-10	do	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
@@ -174076,7 +174076,7 @@
 6-7	pela	_	_	_	_	_	_	_	_
 6	por	por	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 9	por	por	ADP	_	_	11	case	O	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	período	período	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
@@ -174825,7 +174825,7 @@
 6-7	pela	_	_	_	_	_	_	_	_
 6	por	por	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
-8	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 9	por	por	ADP	_	_	11	case	O	_
 10	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
 11	período	período	NOUN	_	Gender=Masc|Number=Sing	5	obl	O	_
@@ -179772,7 +179772,7 @@
 9	por	por	ADP	_	_	12	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
 11	maior	grande	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
-12	velocidade	velocidade	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
+12	velocidade	velocidade	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 13	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	17	obj	O	_
 14	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
 15	água	água	NOUN	_	Gender=Fem|Number=Sing	17	nsubj	B=FLUIDO	_
@@ -202915,7 +202915,7 @@
 10-11	pela	_	_	_	_	_	_	_	_
 10	por	por	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	extração	extração	NOUN	_	Gender=Fem|Number=Sing	9	obj	O	_
+12	extração	extração	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 15	sulfurados	sulfurar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	14	acl	O	_
@@ -203464,7 +203464,7 @@
 7	não	não	ADV	_	Polarity=Neg	8	advmod	O	_
 8	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	3	ccomp	O	_
 9	por	por	ADP	_	_	10	case	O	_
-10	rota	rota	NOUN	_	Gender=Fem|Number=Sing	8	obj	O	_
+10	rota	rota	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	catálise	catálise	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
 13	homogênea	homogêneo	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
@@ -204459,7 +204459,7 @@
 16-17	pelo	_	_	_	_	_	_	_	_
 16	por	por	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
-18	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	15	obj	O	_
+18	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	15	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	vida	vida	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 21-22	dos	_	_	_	_	_	_	_	_
@@ -231091,12 +231091,12 @@
 
 # text = justifica primeiramente pela disponibilidade de dados
 # sent_id = 20-20140904-TESEDSC_0-20
-1	justifica	justificar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	justifica	justificar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	tokenização
 2	primeiramente	primeiramente	ADV	_	_	1	advmod	O	_
 3-4	pela	_	_	_	_	_	_	_	_
 3	por	por	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	disponibilidade	disponibilidade	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
+5	disponibilidade	disponibilidade	NOUN	_	Gender=Fem|Number=Sing	1	obl	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	dados	dado	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 
@@ -274349,7 +274349,7 @@
 11	por	por	ADP	_	_	14	case	O	_
 12	toda	todo	DET	_	Gender=Fem|Number=Sing|PronType=Tot	14	det	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	área	área	NOUN	_	Gender=Fem|Number=Sing	10	obj	O	_
+14	área	área	NOUN	_	Gender=Fem|Number=Sing	10	obl	O	_
 15	investigada	investigar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	14	acl	O	_
 16	.	.	PUNCT	_	_	10	punct	O	_
 17	.	.	PUNCT	_	_	10	punct	O	_
@@ -286238,7 +286238,7 @@
 9	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	8	expl	O	_
 10	estendendo	estender	VERB	_	VerbForm=Ger	8	advcl	O	_
 11	por	por	ADP	_	_	12	case	O	_
-12	centenas	centena	NOUN	_	Gender=Fem|Number=Plur	10	obj	O	_
+12	centenas	centena	NOUN	_	Gender=Fem|Number=Plur	10	obl	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	quilômetros	quilômetro	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 15-16	na	_	_	_	_	_	_	_	_
@@ -288725,7 +288725,7 @@
 7	deslocam	deslocar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	7	expl	O	_
 9	por	por	ADP	_	_	10	case	O	_
-10	tubulações	tubulação	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
+10	tubulações	tubulação	NOUN	_	Gender=Fem|Number=Plur	7	obl	O	_
 11	até	até	SCONJ	_	_	12	mark	O	_
 12	entrar	entrar	VERB	_	VerbForm=Inf	7	advcl	O	_
 13-14	na	_	_	_	_	_	_	_	_
@@ -316888,7 +316888,7 @@
 10-11	pelo	_	_	_	_	_	_	_	_
 10	por	por	ADP	_	_	12	case	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	deslizador	deslizador	NOUN	_	Gender=Masc|Number=Sing	9	obj	O	_
+12	deslizador	deslizador	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	xisto	xisto	NOUN	_	Gender=Masc|Number=Sing	12	nmod	B=LITOLOGIA	_
 15	para	para	SCONJ	_	_	17	mark	O	_
@@ -328425,7 +328425,7 @@
 23-24	pelo	_	_	_	_	_	_	_	_
 23	por	por	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
-25	índice	índice	NOUN	_	Gender=Masc|Number=Sing	21	obj	O	_
+25	índice	índice	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	comportamento	comportamento	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
 28	(	(	PUNCT	_	_	29	punct	O	_
@@ -329782,7 +329782,7 @@
 7	por	por	ADP	_	_	10	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 9	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	10	det	O	_
-10	importância	importância	NOUN	_	Gender=Fem|Number=Sing	6	obj	O	_
+10	importância	importância	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 11	.	.	PUNCT	_	_	6	punct	O	_
 12	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -332818,7 +332818,7 @@
 11-12	pela	_	_	_	_	_	_	_	_
 11	por	por	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
-13	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	10	obj	O	_
+13	decomposição	decomposição	NOUN	_	Gender=Fem|Number=Sing	10	obl	O	_
 14-15	dos	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
@@ -333419,7 +333419,7 @@
 57	,	,	PUNCT	_	_	58	punct	O	_
 58	passando	passar	VERB	_	VerbForm=Ger	48	advcl	O	_
 59	por	por	ADP	_	_	60	case	O	_
-60	imterperização	intemperização	NOUN	_	Gender=Fem|Number=Sing	58	obj	O	_
+60	imterperização	intemperização	NOUN	_	Gender=Fem|Number=Sing	58	obl	O	_
 61	adicional	adicional	ADJ	_	Gender=Fem|Number=Sing	60	amod	O	_
 62	,	,	PUNCT	_	_	66	punct	O	_
 63	ou	ou	CCONJ	_	_	66	cc	O	_
@@ -339104,7 +339104,7 @@
 21-22	pela	_	_	_	_	_	_	_	_
 21	por	por	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	combinação	combinação	NOUN	_	Gender=Fem|Number=Sing	20	obj	O	_
+23	combinação	combinação	NOUN	_	Gender=Fem|Number=Sing	20	obl	O	_
 24-25	dos	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
@@ -341410,7 +341410,7 @@
 5-6	pela	_	_	_	_	_	_	_	_
 5	por	por	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	função	função	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
+7	função	função	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8-9	dos	_	_	_	_	_	_	_	_
 8	de	de	SCONJ	_	_	11	mark	O	_
 9	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	10	det	O	_
@@ -354120,7 +354120,7 @@
 2-3	pelos	_	_	_	_	_	_	_	_
 2	por	por	ADP	_	_	4	case	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	valores	valor	NOUN	_	Gender=Masc|Number=Plur	1	obj	O	_
+4	valores	valor	NOUN	_	Gender=Masc|Number=Plur	1	obl	O	_
 5-6	da	_	_	_	_	_	_	_	_
 5	de	de	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
