commit 2d66ae1805ed6b38581c60e3b0e37fbc0a9d2cdc
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Sun Jul 11 02:32:30 2021 -0300

    cunha(s) NOUN

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index c614236..f54e0d9 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -8984,12 +8984,12 @@
 11	identificar	identificar	VERB	_	VerbForm=Inf	10	csubj	O	_
 12	com	com	ADP	_	_	14	case	O	_
 13	maior	grande	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
-14	facilidade	facilidade	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
-15	cunhas	cunho	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
-16	clásticas	clástico	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
-17	progradantes	progradante	ADJ	_	Gender=Fem|Number=Plur	14	amod	O	_
+14	facilidade	facilidade	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
+15	cunhas	cunha	NOUN	_	Gender=Fem|Number=Plur	11	obj	O	_
+16	clásticas	clástico	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
+17	progradantes	progradante	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
 18	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	19	nsubj	O	_
-19	passaram	passar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	14	acl:relcl	O	_
+19	passaram	passar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	15	acl:relcl	O	_
 20	por	por	ADP	_	_	21	case	O	_
 21	períodos	período	NOUN	_	Gender=Masc|Number=Plur	19	obl	O	_
 22	retrogradacionais	retrogradacional	ADJ	_	Gender=Masc|Number=Plur	21	amod	O	_
@@ -73644,14 +73644,14 @@
 # text = Notam-se, em alguns casos, cunhas com gradação inversa no foreset (produto de grain flow), como observado na figura 19, e com estrutura pin stripe (Figura 19)..
 # sent_id = 283-20150917-MONOGRAFIA_0-339
 1-2	Notam-se	_	_	_	_	_	_	_	_
-1	Eotam	eutar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
-2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
+1	Notam	notar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin|Voice=Pass	0	root	O	_
+2	se	se	PRON	_	_	1	expl	O	_
 3	,	,	PUNCT	_	_	1	punct	O	_
 4	em	em	ADP	_	_	6	case	O	_
 5	alguns	algum	DET	_	Gender=Masc|Number=Plur|PronType=Ind	6	det	O	_
 6	casos	caso	NOUN	_	Gender=Masc|Number=Plur	1	obl	O	_
 7	,	,	PUNCT	_	_	1	punct	O	_
-8	cunhas	cunha	NOUN	_	Gender=Fem|Number=Plur	1	obj	O	_
+8	cunhas	cunha	NOUN	_	Gender=Fem|Number=Plur	1	nsubj:pass	O	_
 9	com	com	ADP	_	_	10	case	O	_
 10	gradação	gradação	NOUN	_	Gender=Fem|Number=Sing	8	nmod	B=LITOLOGIA	_
 11	inversa	inverso	ADJ	_	Gender=Fem|Number=Sing	10	amod	I=LITOLOGIA	_
@@ -73676,7 +73676,7 @@
 28	,	,	PUNCT	_	_	23	punct	O	_
 29	e	e	CCONJ	_	_	31	cc	O	_
 30	com	com	ADP	_	_	31	case	O	_
-31	estrutura	estrutura	NOUN	_	Gender=Fem|Number=Sing	23	conj	O	_
+31	estrutura	estrutura	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
 32	pin	pin	X	_	_	31	nmod	O	_
 33	stripe	stripe	X	_	_	31	nmod	O	_
 34	(	(	PUNCT	_	_	35	punct	O	_
