commit 3dc61b031f1841b76092d1464d836ebc81c1c60d
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Aug 5 13:17:15 2021 -0300

    Correções de root e ciclo

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index a113325..f362ff8 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -4522,7 +4522,7 @@
 4	embasamento	embasamento	NOUN	_	Gender=Masc|Number=Sing	2	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 5	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	aux	O	_
 6-7	subparalelas	_	_	_	_	_	_	_	_
-6	subparale	subparalelo	ADJ	_	Gender=Fem|Number=Plur	0	amod	O	tokenização
+6	subparale	subparalelo	ADJ	_	Gender=Fem|Number=Plur	0	root	O	tokenização
 7	las	elas	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	6	obj	O	_
 8-9	à	_	_	_	_	_	_	_	_
 8	a	a	ADP	_	_	10	case	O	_
@@ -4541,9 +4541,9 @@
 21	mais	mais	ADV	_	_	22	advmod	O	_
 22	predominantes	predominante	ADJ	_	Gender=Fem|Number=Plur	20	amod	O	_
 23	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	24	cop	O	_
-24	grabens	graben	NOUN	_	Gender=Masc|Number=Plur	6	conj	O	_
+24	grabens	graben	NOUN	_	Gender=Masc|Number=Plur	6	parataxis	O	_
 25	assimétricos	assimétrico	ADJ	_	Gender=Masc|Number=Plur	24	amod	O	_
-26	,	,	PUNCT	_	_	24	punct	O	_
+26	,	,	PUNCT	_	_	29	punct	O	_
 27	com	com	ADP	_	_	29	case	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	parte	parte	NOUN	_	Gender=Fem|Number=Sing	24	obl	O	_
@@ -73953,7 +73953,7 @@
 27	de	de	ADP	_	_	29	case	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	seção	seção	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
-30	(	(	PUNCT	_	_	33	punct	O	_
+30	(	(	PUNCT	_	_	31	punct	O	_
 31	E	E	PROPN	_	_	19	parataxis	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
 33	F)	F)	PROPN	_	Gender=Masc|Number=Sing	31	conj	O	tokenização
@@ -74009,7 +74009,7 @@
 81	de	de	ADP	_	_	84	case	O	_
 82	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	84	det	O	_
 83	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	80	nmod	B=LITOLOGIA	_
-84	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	84	nmod	O	_
+84	Sh	Sh	PROPN	_	Gender=Masc|Number=Sing	83	nmod	O	_
 85-86	do	_	_	_	_	_	_	_	_
 85	de	de	ADP	_	_	87	case	O	_
 86	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	87	det	O	_
@@ -125417,30 +125417,30 @@
 # text = Como pode ser observado pela tabela acima, quanto menor o peso molécula do inibidor, melhor sua atuação..
 # sent_id = 334-20150521-MONOGRAFIA_0-429
 1	Como	como	SCONJ	_	_	2	mark	O	_
-2	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	21	root	O	_
-3	ser	ser	AUX	_	VerbForm=Inf	4	aux:pass	O	_
-4	observado	observar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	2	advcl	O	_
+2	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	19	advcl	O	_
+3	ser	ser	AUX	_	VerbForm=Inf	4	aux	O	_
+4	observado	observar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	xcomp	O	_
 5-6	pela	_	_	_	_	_	_	_	_
 5	por	por	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
-7	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	4	obl:agent	O	_
+7	tabela	tabela	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 8	acima	acima	ADV	_	_	4	advmod	O	_
 9	,	,	PUNCT	_	_	2	punct	O	_
 10	quanto	quanto	SCONJ	_	_	11	mark	O	_
 11	menor	pequeno	ADJ	_	Gender=Masc|Number=Sing	19	advcl	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	13	det	O	_
-13	peso	peso	NOUN	_	Gender=Masc|Number=Sing	21	advcl	O	_
+13	peso	peso	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 14	molécula	molecular	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
 15-16	do	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
 17	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
-18	,	,	PUNCT	_	_	13	punct	O	_
-19	melhor	bom	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
+18	,	,	PUNCT	_	_	11	punct	O	_
+19	melhor	bom	ADJ	_	Gender=Fem|Number=Sing	0	root	O	_
 20	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	21	det	O	_
-21	atuação	atuação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
-22	.	.	PUNCT	_	_	21	punct	O	_
-23	.	.	PUNCT	_	_	21	punct	O	_
+21	atuação	atuação	NOUN	_	Gender=Fem|Number=Sing	19	nsubj	O	_
+22	.	.	PUNCT	_	_	19	punct	O	_
+23	.	.	PUNCT	_	_	19	punct	O	_
 
 # text = Por isso o MEG possui melhor desempenho entre os glicóis e o metanol entre os alcoóis..
 # sent_id = 334-20150521-MONOGRAFIA_0-430
@@ -130281,9 +130281,9 @@
 58	(	(	PUNCT	_	_	59	punct	O	_
 59	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	54	appos	O	_
 60	“	"	PUNCT	_	NumType=Card	61	punct	O	_
-61	%	%	SYM	_	_	61	nummod	O	_
+61	%	%	SYM	_	_	59	nmod	O	_
 62	sem	sem	ADP	_	_	63	case	O	_
-63	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	59	nmod	B=QUÍMICA	_
+63	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	61	nmod	B=QUÍMICA	_
 64	”	"	PUNCT	_	Number=Sing	61	punct	O	_
 65	)	)	PUNCT	_	_	59	punct	O	_
 66	.	.	PUNCT	_	_	16	punct	O	_
@@ -251323,14 +251323,14 @@
 34	M	m	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 35	x	x	ADP	_	_	36	case	O	_
 36	M	m	NOUN	_	Gender=Masc|Number=Sing	32	nmod	O	_
-37	(	(	PUNCT	_	_	41	punct	O	_
-38	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	38	obl	O	_
+37	(	(	PUNCT	_	_	39	punct	O	_
+38	onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	39	obl	O	_
 39	M	m	NOUN	_	Gender=Masc|Number=Sing	32	parataxis	O	_
 40	=	=	PUNCT	_	_	41	punct	O	_
 41	número	número	NOUN	_	Gender=Masc|Number=Sing	39	appos	O	_
 42	de	de	ADP	_	_	43	case	O	_
 43	parâmetros	parâmetro	NOUN	_	Gender=Masc|Number=Plur	41	nmod	O	_
-44	)	)	PUNCT	_	_	41	punct	O	_
+44	)	)	PUNCT	_	_	39	punct	O	_
 45	cujo	cujo	DET	_	Gender=Masc|Number=Sing|PronType=Rel	47	det	O	_
 46	j-ésimo	j-ésimo	ADJ	_	Gender=Masc|Number=Sing	47	amod	O	_
 47	elemento	elemento	NOUN	_	Gender=Masc|Number=Sing	49	nsubj:pass	O	_
@@ -251493,7 +251493,7 @@
 # sent_id = 20-20140904-TESEDSC_0-504
 1	Em	em	ADP	_	_	3	case	O	_
 2	outras	outro	DET	_	Gender=Fem|Number=Plur|PronType=Ind	3	det	O	_
-3	palavras	palavra	NOUN	_	Gender=Fem|Number=Plur	8	obl	O	_
+3	palavras	palavra	NOUN	_	Gender=Fem|Number=Plur	10	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
 5	H	h	NOUN	_	Gender=Masc|Number=Sing	10	nsubj	O	tokenização
 6	(	(	PUNCT	_	_	7	punct	O	_
@@ -251511,7 +251511,7 @@
 17	susceptibilidades	susceptibilidade	NOUN	_	Gender=Fem|Number=Plur	15	nmod	O	_
 18	não	não	ADV	_	Polarity=Neg	19	advmod	O	_
 19-20	nulas	_	_	_	_	_	_	_	tokenização
-19	em	em	ADP	_	_	19	case	O	_
+19	em	em	ADP	_	_	17	nmod	O	_
 20	ulas	ula	NOUN	_	Gender=Fem|Number=Plur	17	nmod	O	_
 21	sujeitas	sujeitar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	17	acl	O	_
 22	a	a	SCONJ	_	_	23	mark	O	_
@@ -251539,7 +251539,7 @@
 44	de	de	ADP	_	_	45	case	O	_
 45	prismas	prisma	NOUN	_	Gender=Masc|Number=Plur	43	nmod	O	_
 46	(	(	PUNCT	_	_	47	punct	O	_
-47	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	43	parataxis	O	_
+47	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	45	parataxis	O	_
 48	de	de	ADP	_	_	50	case	O	_
 49	mínimo	mínimo	ADJ	_	Gender=Masc|Number=Sing	50	amod	O	_
 50	volume	volume	NOUN	_	Gender=Masc|Number=Sing	47	nmod	O	_
@@ -277529,11 +277529,11 @@
 34	revela	revelar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 35	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
 36	depocentro	depocentro	NOUN	_	Gender=Masc|Number=Sing	34	obj	O	_
-37	local	local	ADJ	_	Gender=Masc|Number=Sing	37	amod	O	_
+37	local	local	ADJ	_	Gender=Masc|Number=Sing	36	amod	O	_
 38-39	da	_	_	_	_	_	_	_	_
 38	de	de	ADP	_	_	40	case	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
-40	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	37	nmod	B=BACIA:TIPO	_
+40	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	36	nmod	B=BACIA:TIPO	_
 41	condicionado	condicionar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	36	acl	O	_
 42	por	por	ADP	_	_	44	case	O	_
 43	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	44	det	O	_
@@ -316936,15 +316936,15 @@
 8	por	por	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	espaço	espaço	NOUN	_	Gender=Masc|Number=Sing	7	nmod	O	_
-11	anular	anular	ADJ	_	Number=Sing|VerbForm=Inf	11	amod	O	_
-12	,	,	PUNCT	_	_	11	punct	O	_
+11	anular	anular	ADJ	_	Number=Sing|VerbForm=Inf	10	amod	O	_
+12	,	,	PUNCT	_	_	15	punct	O	_
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	15	case	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 15	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	7	nmod	B=FLUIDO	_
 16	de	de	ADP	_	_	17	case	I=FLUIDO	_
 17	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	15	nmod	B=LITOLOGIA|I=FLUIDO	_
-18	,	,	PUNCT	_	_	19	punct	O	_
+18	,	,	PUNCT	_	_	15	punct	O	_
 19	transportando	transportar	VERB	_	VerbForm=Ger	5	advcl	O	_
 20	consigo	consigo	PRON	_	_	19	obl	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	22	det	O	_
@@ -333370,10 +333370,10 @@
 10	derivados	derivado	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
 11	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	cop	O	_
 12	entre	entre	ADP	_	_	13	case	O	_
-13	0,78	0,78	NUM	_	NumType=Card	19	root	O	_
+13	0,78	0,78	NUM	_	NumType=Card	0	root	O	_
 14	e	e	CCONJ	_	_	15	cc	O	_
 15	1,00	1,00	NUM	_	NumType=Card	13	conj	O	_
-16	,	,	PUNCT	_	_	13	punct	O	_
+16	,	,	PUNCT	_	_	19	punct	O	_
 17	esses	esse	DET	_	Gender=Masc|Number=Plur|PronType=Dem	18	det	O	_
 18	valores	valor	NOUN	_	Gender=Masc|Number=Plur	19	nsubj	O	_
 19	faz	fazer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	13	parataxis	O	_
