commit 50824b0432986749b03a99255f32e71da23d6d79
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed May 19 19:08:30 2021 -0300

    correções aline

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b4203c4..11a420d 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -361,7 +361,7 @@
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	continente	continente	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 17	com	com	ADP	_	_	18	case	O	_
-18	dados	dado	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
+18	dados	dado	NOUN	_	Gender=Masc|Number=Plur	9	nmod	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	subsuperfície	subsuperfície	NOUN	_	Gender=Fem|Number=Sing	18	nmod	B=LOCAL:GEO	_
 21	(	(	PUNCT	_	_	22	punct	O	_
@@ -1327,7 +1327,7 @@
 6	sobre	sobre	ADP	_	_	9	case	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	9	det	O	_
 8	diferentes	diferentes	DET	_	Gender=Fem|Number=Plur|PronType=Ind	9	det	O	_
-9	distribuições	distribuição	NOUN	_	Gender=Fem|Number=Plur	4	obl	O	_
+9	distribuições	distribuição	NOUN	_	Gender=Fem|Number=Plur	5	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	em	em	ADP	_	_	13	case	O	_
@@ -22759,7 +22759,7 @@
 4	original	original	ADJ	_	Gender=Fem|Number=Sing	2	amod	O	_
 5	de	de	ADP	_	_	7	case	O	_
 6	cada	cada	DET	_	Gender=Fem|Number=Sing|PronType=Tot	7	det	O	_
-7	uma	um	NUM	_	NumType=Card	2	nummod	O	_
+7	uma	um	PRON	_	PronType=Ind	2	nmod	O	_
 8-9	das	_	_	_	_	_	_	O	_
 8	de	de	ADP	_	_	11	case	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
@@ -22813,7 +22813,7 @@
 53	de	de	ADP	_	_	52	flat:name	O	_
 54	Pernambuco	Pernambuco	PROPN	_	Number=Sing	52	flat:name	O	_
 55	(	(	PUNCT	_	_	56	punct	O	_
-56	Assine	Assine	PROPN	_	Gender=Masc|Number=Sing	52	appos	O	_
+56	Assine	Assine	PROPN	_	Gender=Masc|Number=Sing	14	parataxis	O	_
 57	2007	2007	PROPN	_	Number=Sing	56	flat:name	O	_
 58	)	)	PUNCT	_	_	56	punct	O	_
 59	.	.	PUNCT	_	_	14	punct	O	_
@@ -25782,8 +25782,8 @@
 23	empilhamento	empilhamento	NOUN	_	Gender=Masc|Number=Sing	14	conj	O	_
 24	progradacional	progradacional	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 25	,	,	PUNCT	_	_	26	punct	O	_
-26	granodecrescente	granodecrescente	NOUN	_	Gender=Masc|Number=Sing	23	nmod	O	_
-27	ascendente	ascendente	ADJ	_	Gender=Masc|Number=Sing	26	amod	O	_
+26	granodecrescente	granodecrescente	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
+27	ascendente	ascendente	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
 28	.	.	PUNCT	_	_	4	punct	O	_
 29	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -56998,7 +56998,7 @@
 
 # text = B = Absorbância C = Concentração (mol/100mL) da solução não diluída = [(A+0,0061)/70076]xB D = nº de mol trocado = 1,31.10-3 mol/L (branco) – C.
 # sent_id = 24-20150122-MONOGRAFIA_0-376
-1	B	B	PROPN	_	Gender=Masc|Number=Sing	0	root	O	ELIMINAR
+1	B	B	PROPN	_	Gender=Masc|Number=Sing	0	root	O	eliminar
 2	=	=	PUNCT	_	_	3	punct	O	_
 3	Absorbância	Absorbância	PROPN	_	Gender=Fem|Number=Sing	1	appos	O	_
 4	C	C	PROPN	_	Number=Sing	3	flat:name	O	_
@@ -57115,7 +57115,7 @@
 
 # text = do percentual de massa recuperada no ensaio de rolamento.
 # sent_id = 24-20150122-MONOGRAFIA_0-382
-1	do	do	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
+1	do	do	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	sentenciação
 2	percentual	percentual	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	massa	massa	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -88644,7 +88644,7 @@
 41	line	line	X	_	_	39	flat:foreign	O	_
 42	,	,	PUNCT	_	_	44	punct	O	_
 43	como	como	ADP	_	_	44	case	O	_
-44	tratores	trator	NOUN	_	Gender=Masc|Number=Plur	39	nmod:appos	B=TRANSPORTE	_
+44	tratores	trator	NOUN	_	Gender=Masc|Number=Plur	36	nmod:appos	B=TRANSPORTE	_
 45	,	,	PUNCT	_	_	46	punct	O	_
 46	guindastes	guindaste	NOUN	_	Gender=Masc|Number=Plur	44	conj	O	_
 47	e	e	CCONJ	_	_	48	cc	O	_
@@ -110706,13 +110706,13 @@
 # text = Do montante total em E&P, 73% é destinado ao desenvolvimento da produção, o que corresponde a 135,9 bilhões de dólares..
 # sent_id = 334-20150521-MONOGRAFIA_0-52
 1	Do	Do	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
-2	montante	montante	NOUN	_	Gender=Masc|Number=Sing	10	nsubj:pass	O	_
+2	montante	montante	NOUN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 3	total	total	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	em	em	ADP	_	_	5	case	O	_
 5	E&P	E&P	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 6	,	,	PUNCT	_	_	8	punct	O	_
 7	73	73	NUM	_	NumType=Card	8	nummod	O	_
-8	%	%	SYM	_	_	2	appos	O	_
+8	%	%	SYM	_	_	10	nsubj:pass	O	_
 9	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	aux:pass	O	_
 10	destinado	destinar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 11-12	ao	_	_	_	_	_	_	O	_
@@ -112600,8 +112600,8 @@
 22	1,21	1,21	NUM	_	NumType=Card	23	nummod	O	_
 23	Bbbl	bbbl	NOUN	_	Gender=Masc|Number=Plur	20	obj	O	_
 24	(	(	PUNCT	_	_	25	punct	O	_
-25	baixa	baixa	NOUN	_	Gender=Fem|Number=Sing	23	appos	O	_
-26	estimativa	estimativa	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
+25	baixa	baixa	ADJ	_	Gender=Fem|Number=Sing	26	amod	O	_
+26	estimativa	estimativa	NOUN	_	Gender=Fem|Number=Sing	23	appos	O	_
 27	)	)	PUNCT	_	_	25	punct	O	_
 28	a	a	ADP	_	_	30	case	O	_
 29	2,57	2,57	NUM	_	NumType=Card	30	nummod	O	_
@@ -114945,7 +114945,7 @@
 
 # text = De acordo com Bothamley et al.(2004), a decisão por qual direção um projeto deve seguir depende de diversos fatores, dentre os quais se pode citar:
 # sent_id = 334-20150521-MONOGRAFIA_0-165
-1	De	de	ADP	_	_	4	case	O	_
+1	De	de	ADP	_	_	4	case	O	sentenciação
 2	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	_
 3	com	com	ADP	_	_	1	fixed	O	_
 4	Bothamley	Bothamley	PROPN	_	Gender=Masc|Number=Sing	18	obl	O	_
@@ -115974,7 +115974,7 @@
 25	Chevron	Chevron	PROPN	_	Gender=Fem|Number=Sing	22	nmod	B=LITOLOGIA	_
 26	(	(	PUNCT	_	_	28	punct	O	_
 27	37,5	37,5	NUM	_	NumType=Card	28	nummod	O	_
-28	%	%	SYM	_	_	25	appos	O	_
+28	%	%	SYM	_	_	25	nmod:appos	O	_
 29	)	)	PUNCT	_	_	28	punct	O	_
 30	e	e	CCONJ	_	_	31	cc	O	_
 31	BP	BP	PROPN	_	Gender=Masc|Number=Sing	25	conj	O	_
@@ -115982,7 +115982,7 @@
 33	27,55	27,55	NUM	_	NumType=Card	31	nmod:appos	O	_
 34	)	)	PUNCT	_	_	36	punct	O	_
 35	(	(	PUNCT	_	_	36	punct	O	_
-36	www.	www.	NOUN	_	Gender=Masc|Number=Sing	31	appos	O	_
+36	www.	www.	NOUN	_	Gender=Masc|Number=Sing	3	parataxis	O	tokenização
 37	shell.	shell.	X	_	_	36	flat:foreign	O	_
 38	com	com	ADP	_	_	36	flat:name	O	_
 39	)	)	PUNCT	_	_	36	punct	O	_
@@ -134616,7 +134616,7 @@
 
 # text = Na Tabela 26, são apresentadas apenas as temperaturas e pressões de entrada e de saída dos dutos, não sendo mostrados os dados intermediários entre os segmentos. • MEMBRANAS: O método de remoção de CO2 que se mostrou mais eficiente durante a simulação foi através de duas membranas, em série, de 1,6E+5 m2de área cada e cujas pressões de permeado e retentado são apresentadas na Tabela 27, assim como as respectivas vazões e temperaturas..
 # sent_id = 334-20150521-MONOGRAFIA_0-709
-1-2	Na	_	_	_	_	_	_	O	_
+1-2	Na	_	_	_	_	_	_	O	sentenciação
 1	Em	em	ADP	_	_	3	case	O	_
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	7	obl	O	_
@@ -135268,7 +135268,7 @@
 
 # text = neste componente (ÁGUA TOPSIDE, ÁGUA+CO2 to EOR e ÁGUA+CO2) é obtido
 # sent_id = 334-20150521-MONOGRAFIA_0-721
-1	neste	neste	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
+1	neste	neste	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	sentenciação
 2	componente	componente	NOUN	_	Gender=Masc|Number=Sing	14	nsubj:pass	O	_
 3	(	(	PUNCT	_	_	4	punct	O	_
 4	ÁGUA	ÁGUA	PROPN	_	Gender=Fem|Number=Sing	2	appos	B=FLUIDO	_
@@ -137903,7 +137903,7 @@
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
 28	molécula	molécula	NOUN	_	Gender=Fem|Number=Sing	25	nmod	O	_
 29	(	(	PUNCT	_	_	30	punct	O	_
-30	apolar	apolar	ADJ	_	Gender=Fem|Number=Sing	25	appos	O	_
+30	apolar	apolar	ADJ	_	Gender=Fem|Number=Sing	25	amod	O	_
 31	ou	ou	CCONJ	_	_	32	cc	O	_
 32	lipofílica	lipofílico	ADJ	_	Gender=Fem|Number=Sing	30	conj	O	_
 33	)	)	PUNCT	_	_	30	punct	O	_
@@ -143002,8 +143002,8 @@
 22	hipocentro	hipocentro	ADJ	_	Gender=Masc|Number=Sing	20	conj	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
 24	afirmação	afirmação	NOUN	_	Gender=Fem|Number=Sing	17	conj	O	_
-25	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	26	det	O	_
-26	aceita	aceitar	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
+25	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	24	det	O	_
+26	aceita	aceitar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	24	nmod	O	_
 27-28	nesse	_	_	_	_	_	_	O	_
 27	em	em	ADP	_	_	29	case	O	_
 28	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	29	det	O	_
@@ -147999,8 +147999,8 @@
 
 # text = Os registros das polaridades da onda
 # sent_id = 258-20160602-TESEMSC_0-139
-1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	Sentenciação
-2	registros	registro	NOUN	_	Gender=Masc|Number=Plur	0	root	O	sentenciação
+1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	sentenciação
+2	registros	registro	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
 3-4	das	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	5	case	O	_
 4	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	5	det	O	_
@@ -148012,7 +148012,7 @@
 
 # text = P tanto das estações regionais como das telesismicas localizadas nos diferentes quadrantes, mostram um plano nodal aproximadamente N-S, que permitiu definir a orientação da falha (strike) de 341º e um mergulho (dip) de 89º..
 # sent_id = 258-20160602-TESEMSC_0-140
-1	P	P	PROPN	_	Gender=Masc|Number=Sing	17	nsubj	O	Sentenciação
+1	P	P	PROPN	_	Gender=Masc|Number=Sing	17	nsubj	O	sentenciação
 2	tanto	tanto	ADV	_	_	5	advmod	O	_
 3-4	das	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	5	case	O	_
@@ -148081,7 +148081,7 @@
 
 # text = C a p í t u l o 2 - S i s m i c i d a d e | 34 Com a análise da razão das amplitudes das fases pP/P outro plano nodal foi estabelecido, aproximadamente E-W mergulhando para sul, o qual apresenta orientação da falha (strike) de 82º e mergulho (dip) de 3º..
 # sent_id = 258-20160602-TESEMSC_0-142
-1	C	C	PROPN	_	Gender=Masc|Number=Sing	42	nsubj:pass	O	_
+1	C	C	PROPN	_	Gender=Masc|Number=Sing	42	nsubj:pass	sentenciação	_
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	p	p	PROPN	_	Gender=Masc|Number=Sing	1	flat:name	O	_
 4	í	í	PROPN	_	Number=Sing	3	flat:name	O	_
@@ -148172,7 +148172,7 @@
 12	de	de	ADP	_	_	13	case	O	_
 13	11º	11º	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	0	root	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
-15	ASSUMPÇÃO	ASSUMPç-O	PROPN	_	Gender=Masc|Number=Sing	13	parataxis	O	_
+15	ASSUMPÇÃO	ASSUMPÇÃO	PROPN	_	Gender=Masc|Number=Sing	13	parataxis	O	_
 16	et	et	PROPN	_	Number=Sing	15	flat:name	O	_
 17	al.	al.	PROPN	_	Number=Sing	15	flat:name	O	_
 18	,	,	PUNCT	_	_	15	flat:name	O	_
@@ -155507,9 +155507,9 @@
 13	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 14	interpretados	interpretar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	13	acl	O	_
 15	,	,	PUNCT	_	_	18	punct	O	_
-16	muitas	muitas	ADP	_	_	17	case	O	_
-17	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	6	obl	O	_
-18	realizada	realizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	17	acl	O	_
+16	muitas	muitas	DET	_	Gender=Fem|Number=Plur|PronType=Ind	17	case	O	_
+17	vezes	vez	NOUN	_	Gender=Fem|Number=Plur	18	obl	O	_
+18	realizada	realizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	8	acl	O	_
 19-20	pela	_	_	_	_	_	_	O	_
 19	por	por	ADP	_	_	21	case	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
@@ -166952,24 +166952,24 @@
 # text = Como conseqüência, segundo estas quantidades, diferentes serão as características dos tipos de petróleo..
 # sent_id = 77-20141209-TESEMSC_0-39
 1	Como	como	ADP	_	_	2	case	O	_
-2	conseqüência	conseqüência	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
+2	conseqüência	conseqüência	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	segundo	segundo	ADP	_	_	6	case	O	_
 5	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	6	det	O	_
-6	quantidades	quantidade	NOUN	_	Gender=Fem|Number=Plur	11	nmod	O	_
+6	quantidades	quantidade	NOUN	_	Gender=Fem|Number=Plur	8	obl	O	_
 7	,	,	PUNCT	_	_	6	punct	O	_
-8	diferentes	diferentes	PRON	_	Gender=Masc|Number=Plur|PronType=Ind	11	nsubj	O	_
+8	diferentes	diferentes	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
 9	serão	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Fut|VerbForm=Fin	11	cop	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
-11	características	característica	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
+11	características	característica	NOUN	_	Gender=Fem|Number=Plur	8	nsubj	O	_
 12-13	dos	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	14	case	O	_
 13	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	14	det	O	_
 14	tipos	tipo	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	14	nmod	B=FLUIDO	_
-17	.	.	PUNCT	_	_	11	punct	O	_
-18	.	.	PUNCT	_	_	11	punct	O	_
+17	.	.	PUNCT	_	_	8	punct	O	_
+18	.	.	PUNCT	_	_	8	punct	O	_
 
 # text = Para a caracterização da composição do petróleo utilizam-se métodos de fracionamento que utilizam técnicas de destilação, como a cromatografia gasosa e a líquida..
 # sent_id = 77-20141209-TESEMSC_0-40
@@ -189283,7 +189283,7 @@
 
 # text = [8] http://www.tnsustentavel..
 # sent_id = 279-20140530-MONOGRAFIA_0-239
-1	[8	[8	NUM	_	NumType=Card	3	parataxis	O	Sentenciação
+1	[8	[8	NUM	_	NumType=Card	3	parataxis	O	sentenciação
 2	]	]	PUNCT	_	_	3	punct	O	_
 3	http:/	http:/	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 4	/	/	PUNCT	_	_	3	punct	O	_
@@ -215120,7 +215120,7 @@
 24	elaboração	elaboração	NOUN	_	Gender=Fem|Number=Sing	22	obj	O	_
 25	de	de	ADP	_	_	28	case	O	_
 26	quatro	quatro	NUM	_	NumType=Card	28	nummod	O	_
-27	diferentes	diferentes	DET	_	Gender=Masc|Number=Plur|PronType=Ind	28	det	O	_
+27	diferentes	diferente	ADJ	_	Gender=Masc|Number=Plur	28	amod	O	_
 28	diagramas	diagrama	NOUN	_	Gender=Masc|Number=Plur	24	nmod	O	_
 29	ternários	ternário	ADJ	_	Gender=Masc|Number=Plur	28	amod	O	_
 30	permanecendo	permanecer	VERB	_	VerbForm=Ger	22	advcl	O	_
@@ -216908,7 +216908,7 @@
 13	características	característica	NOUN	_	Gender=Fem|Number=Plur	11	obj	O	_
 14	de	de	ADP	_	_	17	case	O	_
 15	três	três	NUM	_	NumType=Card	17	nummod	O	_
-16	diferentes	diferentes	DET	_	Gender=Masc|Number=Plur|PronType=Ind	17	det	O	_
+16	diferentes	diferente	ADJ	_	Gender=Fem|Number=Plur	17	amod	O	_
 17	classes	classe	NOUN	_	Gender=Fem|Number=Plur	13	nmod	O	_
 18	de	de	ADP	_	_	19	case	O	_
 19	materiais	material	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
@@ -219225,7 +219225,7 @@
 
 # text = O2 que são responsáveis pela atividade estável do catalisador; ●O estado de oxidação médio dos átomos de molibdênio mostrou-se bastante sensível à acidez protônica da zeólita: quanto menor a acidez protônica, mais baixo o estado de oxidação do molibdênio suportado..
 # sent_id = 25-20150123-MONOGRAFIA_0-112
-1	O2	O2	PROPN	_	Gender=Masc|Number=Sing	24	nsubj	O	_
+1	O2	O2	PROPN	_	Gender=Masc|Number=Sing	24	nsubj	O	sentenciação
 2	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	4	nsubj	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	cop	O	_
 4	responsáveis	responsável	ADJ	_	Gender=Masc|Number=Plur	1	acl:relcl	O	_
@@ -221565,7 +221565,7 @@
 
 # text = Os sais formados foram dissolvidos em 10 mL de HCl 1
 # sent_id = 25-20150123-MONOGRAFIA_0-190
-1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	_
+1	Os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	2	det	O	sentenciação
 2	sais	sai	NOUN	_	Gender=Masc|Number=Plur	5	nsubj:pass	O	_
 3	formados	formar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	2	acl	O	_
 4	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	5	aux:pass	O	_
@@ -221600,7 +221600,7 @@
 
 # text = Foram adicionados 5
 # sent_id = 25-20150123-MONOGRAFIA_0-192
-1	Foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	2	aux:pass	O	_
+1	Foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	2	aux:pass	O	sentenciação
 2	adicionados	adicionar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 3	5	5	NUM	_	NumType=Card	2	nsubj:pass	O	_
 
@@ -221623,7 +221623,7 @@
 
 # text = Em seguida foram adicionados 10
 # sent_id = 25-20150123-MONOGRAFIA_0-194
-1	Em	em	ADP	_	_	2	case	O	_
+1	Em	em	ADP	_	_	2	case	O	sentenciação
 2	seguida	seguida	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 3	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	4	aux:pass	O	_
 4	adicionados	adicionar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -224347,7 +224347,7 @@
 
 # text = As condições de análise cromatográfica com MSD se encontram abaixo: ► Temperatura da coluna: 90 °C.
 # sent_id = 25-20150123-MONOGRAFIA_0-278
-1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
+1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	sentenciação
 2	condições	condição	NOUN	_	Gender=Fem|Number=Plur	9	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	análise	análise	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -225246,7 +225246,7 @@
 
 # text = do catalisador por massa de zeólita..
 # sent_id = 25-20150123-MONOGRAFIA_0-308
-1	do	do	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
+1	do	do	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	eliminar
 2	catalisador	catalisador	NOUN	_	Gender=Masc|Number=Sing	0	root	B=QUÍMICA	_
 3	por	por	ADP	_	_	4	case	O	_
 4	massa	massa	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
@@ -324432,7 +324432,7 @@
 
 # text = Com isso, os diferentes tipos de equilíbrio existentes entre as microemulsões, fase aquosa e oleosa, foram classificados por Winsor em 1938, como:
 # sent_id = 240-20140220-MONOGRAFIA_0-289
-1	Com	com	ADP	_	_	2	case	O	_
+1	Com	com	ADP	_	_	2	case	O	sentenciação
 2	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	20	obl	O	_
 3	,	,	PUNCT	_	_	2	punct	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
@@ -324842,7 +324842,7 @@
 12	adicionar	adicionar	VERB	_	VerbForm=Inf	1	advcl	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	quantidade	quantidade	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
-15	certa	certo	DET	_	Gender=Fem|Number=Sing|PronType=Ind	14	det	O	_
+15	certa	certo	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	adensante	adensante	NOUN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 18	para	para	SCONJ	_	_	19	mark	O	_
@@ -345962,7 +345962,7 @@
 
 # text = Fernandes Jr. (2002), destaca que, a água produzida é salina (salmora) e contém sólidos dispersos (areia, argila, lodo, outros silicatos, gipsita) e teores de sais variando de 15.000mg/L a 300.000mg/L.
 # sent_id = 243-20141015-MONOGRAFIA_1-35
-1	Fernandes	Fernandes	PROPN	_	Gender=Masc|Number=Sing	8	nsubj	O	_
+1	Fernandes	Fernandes	PROPN	_	Gender=Masc|Number=Sing	8	nsubj	O	sentenciação
 2	Jr	Jr	PROPN	_	Number=Sing	1	flat:name	O	_
 3	.	.	PUNCT	_	_	1	flat:name	O	_
 4	(	(	PUNCT	_	_	1	flat:name	O	_
@@ -354768,4 +354768,3 @@
 54	.	.	PUNCT	_	_	2	punct	O	_
 55	.	.	PUNCT	_	_	2	punct	O	_
 
-
