commit 4ee4a880b18bf53a004c02240c668bfee158d425
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Fri Jul 30 12:46:12 2021 -0300

    Confusões obl e nmod

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index e888c8a..36e34c7 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -478,7 +478,7 @@
 26-27	na	_	_	_	_	_	_	_	_
 26	em	em	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	porção	porção	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
+28	porção	porção	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 29	emersa	emerso	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 30	e	e	CCONJ	_	_	32	cc	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
@@ -500,7 +500,7 @@
 46	Km²	Km²	PROPN	_	Gender=Masc|Number=Sing	42	nmod	O	_
 47	,	,	PUNCT	_	_	40	punct	O	_
 48-49	destes	_	_	_	_	_	_	_	_
-48	de	de	ADP	_	_	50	case	O	_
+48	de	de	ADP	_	_	52	case	O	_
 49	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	52	det	O	_
 50	278	278	NUM	_	NumType=Card	52	nummod	O	_
 51	048	048	NUM	_	NumType=Card	52	nummod	O	_
@@ -854,7 +854,7 @@
 2	de	de	ADP	_	_	1	fixed	O	_
 3	estas	este	DET	_	Gender=Fem|Number=Plur|PronType=Dem	4	det	O	_
 4	imagens	imagem	NOUN	_	Gender=Fem|Number=Plur	7	obl	O	_
-5	,	,	PUNCT	_	_	1	punct	O	_
+5	,	,	PUNCT	_	_	4	punct	O	_
 6	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	7	aux:pass	O	_
 7	localizadas	localizar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 8	feições	feição	NOUN	_	Gender=Fem|Number=Plur	7	nsubj:pass	O	_
@@ -890,7 +890,7 @@
 36-37	ao	_	_	_	_	_	_	_	_
 36	a	a	ADP	_	_	38	case	O	_
 37	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	38	det	O	_
-38	longo	longo	NOUN	_	Gender=Masc|Number=Sing	35	obl	O	_
+38	longo	longo	NOUN	_	Gender=Masc|Number=Sing	34	nmod	O	_
 39-40	do	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
@@ -1302,7 +1302,7 @@
 14	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	15	det	O	_
 15	lineamentos	lineamento	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 16	em	em	ADP	_	_	17	case	O	_
-17	mapas	mapa	NOUN	_	Gender=Masc|Number=Plur	10	obl	B=INSTRUMENTO	_
+17	mapas	mapa	NOUN	_	Gender=Masc|Number=Plur	10	xcomp	B=INSTRUMENTO	_
 18	magnéticos	magnético	ADJ	_	Gender=Masc|Number=Plur	17	amod	O	_
 19	.	.	PUNCT	_	_	10	punct	O	_
 20	.	.	PUNCT	_	_	10	punct	O	_
@@ -2457,7 +2457,7 @@
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	7	obl	O	_
 11	4	4	NUM	_	Number=Sing	10	nummod	O	_
-12	,	,	PUNCT	_	_	7	punct	O	_
+12	,	,	PUNCT	_	_	14	punct	O	_
 13	com	com	ADP	_	_	14	case	O	_
 14	integração	integração	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
@@ -2468,8 +2468,8 @@
 20	em	em	ADP	_	_	23	case	O	_
 21	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
 22	única	único	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
-23	base	base	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
-24	georreferenciada	georreferenciar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	23	acl	O	_
+23	base	base	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
+24	georreferenciada	georreferenciado	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	.	.	PUNCT	_	_	7	punct	O	_
 26	.	.	PUNCT	_	_	7	punct	O	_
 
@@ -3244,7 +3244,7 @@
 18	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	16	nmod:appos	B=LITOLOGIA	_
 19	branco-amarelados	branco-amarelado	ADJ	_	Gender=Masc|Number=Plur	18	amod	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
-21	cauliníticos	caulinítico	ADJ	_	Gender=Masc|Number=Plur	18	conj	O	_
+21	cauliníticos	caulinítico	ADJ	_	Gender=Masc|Number=Plur	19	conj	O	_
 22	com	com	ADP	_	_	23	case	O	_
 23	estratificações	estratificação	NOUN	_	Gender=Fem|Number=Plur	21	obl	B=LITOLOGIA	_
 24	cruzadas	cruzado	ADJ	_	Gender=Fem|Number=Plur	23	amod	O	_
@@ -3485,7 +3485,7 @@
 26	finos	fino	ADJ	_	Gender=Masc|Number=Plur	24	obl	O	_
 27	,	,	PUNCT	_	_	29	punct	O	_
 28	com	com	ADP	_	_	29	case	O	_
-29	estratificação	estratificação	NOUN	_	Gender=Fem|Number=Sing	24	obl	B=LITOLOGIA	_
+29	estratificação	estratificação	NOUN	_	Gender=Fem|Number=Sing	19	obl	B=LITOLOGIA	_
 30	cruzada	cruzado	ADJ	_	Gender=Fem|Number=Sing	29	amod	O	_
 31	)	)	PUNCT	_	_	19	punct	O	_
 32	..	..	PUNCT	_	_	1	punct	O	_
@@ -3665,7 +3665,7 @@
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
 21	Terraço	Terraço	PROPN	_	Gender=Masc|Number=Sing	18	obl	O	_
 22	em	em	ADP	_	_	23	case	O	_
-23	direção	direção	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
+23	direção	direção	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 24-25	ao	_	_	_	_	_	_	_	_
 24	a	a	ADP	_	_	26	case	O	_
 25	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	det	O	_
@@ -4195,17 +4195,17 @@
 31	borda	borda	NOUN	_	Gender=Fem|Number=Sing	28	obj	O	_
 32	falhada	falhar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	31	acl	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
-34	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	32	obj	B=BACIA:TIPO	_
+34	bacia	bacia	NOUN	_	Gender=Fem|Number=Sing	28	obj	B=BACIA:TIPO	_
 35	conjugada	conjugar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	34	acl	O	_
 36-37	da	_	_	_	_	_	_	_	_
 36	de	de	ADP	_	_	38	case	O	_
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	O	_
 38	Namíbia	Namíbia	PROPN	_	Gender=Fem|Number=Sing	34	nmod	O	_
-39	,	,	PUNCT	_	_	35	punct	O	_
+39	,	,	PUNCT	_	_	42	punct	O	_
 40-41	na	_	_	_	_	_	_	_	_
 40	em	em	ADP	_	_	42	case	O	_
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
-42	África	África	PROPN	_	Gender=Fem|Number=Sing	35	obl	B=LOCAL:POL	_
+42	África	África	PROPN	_	Gender=Fem|Number=Sing	38	nmod	B=LOCAL:POL	_
 43	.	.	PUNCT	_	_	3	punct	O	_
 44	.	.	PUNCT	_	_	3	punct	O	_
 
@@ -7667,10 +7667,10 @@
 12	lineamentos	lineamento	NOUN	_	Gender=Masc|Number=Plur	11	obj	O	_
 13	com	com	ADP	_	_	15	case	O	_
 14	variadas	variar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	acl	O	_
-15	direções	direção	NOUN	_	Gender=Fem|Number=Plur	11	obl	O	_
+15	direções	direção	NOUN	_	Gender=Fem|Number=Plur	12	nmod	O	_
 16	,	,	PUNCT	_	_	18	punct	O	_
 17	sendo	ser	AUX	_	VerbForm=Ger	18	cop	O	_
-18	elas	ele	PRON	_	Case=Nom|Gender=Fem|Number=Plur|Person=3|PronType=Prs	11	advcl	O	_
+18	elas	ele	PRON	_	Case=Nom|Gender=Fem|Number=Plur|Person=3|PronType=Prs	15	acl	O	_
 19	NNW-SSE	NNW-SSE	PROPN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	WNW-WSE	WNW-WSE	PROPN	_	Gender=Fem|Number=Sing	19	conj	O	_
@@ -7775,12 +7775,12 @@
 35	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	32	nmod	O	_
 36	,	,	PUNCT	_	_	37	punct	O	_
 37	anastomosados	anastomosar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	32	acl	O	_
-38	,	,	PUNCT	_	_	41	punct	O	_
+38	,	,	PUNCT	_	_	40	punct	O	_
 39	com	com	ADP	_	_	40	case	O	_
 40	planos	plano	NOUN	_	Gender=Masc|Number=Plur	28	obl	O	_
 41	estriados	estriar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	40	acl	O	_
 42	bem	bem	ADV	_	_	43	advmod	O	_
-43	caracterizados	caracterizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	41	conj	O	_
+43	caracterizados	caracterizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	40	acl	O	_
 44	.	.	PUNCT	_	_	4	punct	O	_
 45	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -7985,15 +7985,15 @@
 14	e	e	CCONJ	_	_	15	cc	O	_
 15	NNE-SSW	NNE-SSW	PROPN	_	Gender=Fem|Number=Sing	13	conj	O	_
 16	(	(	PUNCT	_	_	17	punct	O	_
-17	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	15	appos	O	_
+17	Figura	Figura	PROPN	_	Gender=Fem|Number=Sing	9	appos	O	_
 18	24	24	PROPN	_	Number=Sing	17	flat:name	O	_
 19	)	)	PUNCT	_	_	17	punct	O	_
 20	,	,	PUNCT	_	_	9	punct	O	_
 21	com	com	ADP	_	_	22	case	O	_
-22	médio	médio	ADJ	_	Gender=Masc|Number=Sing	6	nmod	O	_
-23	a	a	ADP	_	_	25	case	O	_
-24	alto	alto	ADJ	_	Gender=Masc|Number=Sing	25	amod	O	_
-25	ângulo	ângulo	NOUN	_	Gender=Masc|Number=Sing	22	obl	O	_
+22	médio	médio	ADJ	_	Gender=Masc|Number=Sing	25	amod	O	_
+23	a	a	ADP	_	_	24	case	O	_
+24	alto	alto	ADJ	_	Gender=Masc|Number=Sing	22	obl	O	_
+25	ângulo	ângulo	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
 26	de	de	ADP	_	_	27	case	O	_
 27	mergulho	mergulho	NOUN	_	Gender=Masc|Number=Sing	25	nmod	O	_
 28	.	.	PUNCT	_	_	6	punct	O	_
@@ -11657,7 +11657,7 @@
 4	em	em	ADP	_	_	7	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 6	mesma	mesmo	ADJ	_	Gender=Fem|Number=Sing	7	amod	O	_
-7	molécula	molécula	NOUN	_	Gender=Fem|Number=Sing	15	obl	O	_
+7	molécula	molécula	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 8	,	,	PUNCT	_	_	7	punct	O	_
 9	de	de	ADP	_	_	11	case	O	_
 10	duas	dois	NUM	_	NumType=Card	11	nummod	O	_
@@ -12708,10 +12708,10 @@
 11	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
 12	dispersão	dispersão	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 13	composta	compor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	12	acl	O	_
-14	de	de	ADP	_	_	15	case	O	_
-15	50	50	NUM	_	NumType=Card	12	nummod	O	_
-16	a	a	ADP	_	_	18	case	O	_
-17	70	70	NUM	_	NumType=Card	18	nummod	O	_
+14	de	de	ADP	_	_	18	case	O	_
+15	50	50	NUM	_	NumType=Card	18	nummod	O	_
+16	a	a	ADP	_	_	17	case	O	_
+17	70	70	NUM	_	NumType=Card	15	nummod	O	_
 18	%	%	SYM	_	_	13	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	cimento	cimento	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
@@ -12721,15 +12721,15 @@
 24	frações	fração	NOUN	_	Gender=Fem|Number=Plur	18	conj	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	emulsificante	emulsificante	NOUN	_	Gender=Masc|Number=Sing	24	nmod	B=QUÍMICA	_
-27	(	(	PUNCT	_	_	28	punct	O	_
-28	1	1	NUM	_	NumType=Card	24	appos	O	_
-29	a	a	ADP	_	_	31	case	O	_
-30	2	2	NUM	_	NumType=Card	31	nummod	O	_
-31	%	%	SYM	_	_	28	nmod	O	_
-32	)	)	PUNCT	_	_	28	punct	O	_
+27	(	(	PUNCT	_	_	31	punct	O	_
+28	1	1	NUM	_	NumType=Card	31	nummod	O	_
+29	a	a	ADP	_	_	30	case	O	_
+30	2	2	NUM	_	NumType=Card	28	nummod	O	_
+31	%	%	SYM	_	_	24	appos	O	_
+32	)	)	PUNCT	_	_	31	punct	O	_
 33	e	e	CCONJ	_	_	35	cc	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
-35	restante	restante	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
+35	restante	restante	NOUN	_	Gender=Masc|Number=Sing	18	conj	O	_
 36	de	de	ADP	_	_	37	case	O	_
 37	água	água	NOUN	_	Gender=Fem|Number=Sing	35	nmod	B=FLUIDO	_
 38	.	.	PUNCT	_	_	12	punct	O	_
