commit d05c5a8ed8190af9b716fc202aa29970e5fb40c7
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 8 01:34:29 2021 -0300

    com_vistas_a

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b6ea396..cfde9d0 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -112114,7 +112114,7 @@
 32-33	à	_	_	_	_	_	_	_	_
 32	a	a	ADP	_	_	30	fixed	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
-34	valoração	valoração	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
+34	valoração	valoração	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 35-36	dos	_	_	_	_	_	_	_	_
 35	de	de	ADP	_	_	37	case	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
@@ -231413,12 +231413,12 @@
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 26	dados	dado	NOUN	_	Gender=Masc|Number=Plur	23	obl:agent	O	_
 27	)	)	PUNCT	_	_	23	punct	O	_
-28	com	com	ADP	_	_	29	case	O	_
-29	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	7	advcl	O	_
+28	com	com	ADP	_	_	32	case	O	MWEPOS=ADP
+29	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	28	fixed	O	_
 30-31	à	_	_	_	_	_	_	_	_
-30	a	a	ADP	_	_	32	case	O	_
+30	a	a	ADP	_	_	28	fixed	O	_
 31	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	32	det	O	_
-32	construção	construção	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
+32	construção	construção	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
 33	de	de	ADP	_	_	35	case	O	_
 34	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 35	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	32	nmod	B=INSTRUMENTO	_
@@ -242531,7 +242531,7 @@
 7	sísmica	sísmica	NOUN	_	Gender=Fem|Number=Sing	5	nmod	B=MÉTODO	_
 8	e	e	CCONJ	_	_	9	cc	O	_
 9	batimetria	batimetria	NOUN	_	Gender=Fem|Number=Sing	7	conj	O	_
-10	com	com	ADP	_	_	11	mark	O	MWEPOS=SCONJ
+10	com	com	ADP	_	_	13	mark	O	MWEPOS=SCONJ
 11	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	10	fixed	O	_
 12	a	a	ADP	_	_	10	fixed	O	_
 13	enriquecer	enriquecer	VERB	_	VerbForm=Inf	4	advcl	O	_
@@ -243432,10 +243432,10 @@
 14	tecnicamente	tecnicamente	ADV	_	_	13	advmod	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
 16	grids	grid	NOUN	_	Gender=Masc|Number=Plur	13	obj	O	_
-17	com	com	ADP	_	_	20	mark	O	_
-18	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	16	acl	O	_
-19	a	a	SCONJ	_	_	20	mark	O	_
-20	obter	obter	VERB	_	VerbForm=Inf	18	advcl	O	_
+17	com	com	ADP	_	_	20	mark	O	MWEPOS=SCONJ
+18	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	17	fixed	O	_
+19	a	a	ADP	_	_	17	fixed	O	_
+20	obter	obter	VERB	_	VerbForm=Inf	13	advcl	O	_
 21	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	22	det	O	_
 22	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	20	obj	B=INSTRUMENTO	_
 23	integrado	integrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	22	acl	O	_
@@ -243665,12 +243665,12 @@
 
 # text = Com vistas à obtenção de um mapa batimétrico integrado de cunho regional, os dados foram organizados em um grid regular de valores, obtidos por interpolações lineares utilizando o método conhecido como Krigagem (disponível no Geosoft)..
 # sent_id = 20-20140904-TESEDSC_0-320
-1	Com	com	ADP	_	_	2	mark	O	_
-2	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	18	advcl	O	_
+1	Com	com	ADP	_	_	5	case	O	MWEPOS=ADP
+2	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	1	fixed	O	_
 3-4	à	_	_	_	_	_	_	_	_
-3	a	a	ADP	_	_	5	case	O	_
+3	a	a	ADP	_	_	1	fixed	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	obtenção	obtenção	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
+5	obtenção	obtenção	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 6	de	de	ADP	_	_	8	case	O	_
 7	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
 8	mapa	mapa	NOUN	_	Gender=Masc|Number=Sing	5	nmod	B=INSTRUMENTO	_
@@ -243924,10 +243924,10 @@
 
 # text = Com vistas a enfatizar as feições sutis do relevo topográfico, encontrado numa área tipicamente de relevo inexpressivo como é a plataforma continental, foi aplicado um sombreamento digital no mapa com o posicionamento da fonte de luz a baixa inclinação e perpendicular ao trend magnético observado..
 # sent_id = 20-20140904-TESEDSC_0-327
-1	Com	com	ADP	_	_	2	mark	O	_
-2	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	28	advcl	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	enfatizar	enfatizar	VERB	_	VerbForm=Inf	2	xcomp	O	_
+1	Com	com	ADP	_	_	4	mark	O	MWEPOS=SCONJ
+2	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	1	fixed	O	_
+3	a	a	ADP	_	_	1	fixed	O	_
+4	enfatizar	enfatizar	VERB	_	VerbForm=Inf	28	advcl	O	_
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
 6	feições	feição	NOUN	_	Gender=Fem|Number=Plur	4	obj	O	_
 7	sutis	sutil	ADJ	_	Gender=Fem|Number=Plur	6	amod	O	_
@@ -244241,10 +244241,10 @@
 4	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	6	nsubj:pass	O	_
 5	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	6	aux:pass	O	_
 6	elaborado	elaborar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
-7	com	com	ADP	_	_	8	case	O	_
-8	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	6	advcl	O	_
-9	a	a	SCONJ	_	_	10	mark	O	_
-10	avaliar	avaliar	VERB	_	VerbForm=Inf	8	xcomp	O	_
+7	com	com	ADP	_	_	10	mark	O	MWEPOS=SCONJ
+8	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	7	fixed	O	_
+9	a	a	ADP	_	_	7	fixed	O	_
+10	avaliar	avaliar	VERB	_	VerbForm=Inf	6	advcl	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
 12	propriedades	propriedade	NOUN	_	Gender=Fem|Number=Plur	10	obj	O	_
 13-14	do	_	_	_	_	_	_	_	_
@@ -249180,10 +249180,10 @@
 
 # text = Com vistas a obter um spray de soluções, a janela é deslocada, a intervalos regulares, em ambos os sentidos a partir da posição onde está centralizada..
 # sent_id = 20-20140904-TESEDSC_0-445
-1	Com	com	ADP	_	_	2	mark	O	_
-2	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	13	advcl	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	obter	obter	VERB	_	VerbForm=Inf	2	xcomp	O	_
+1	Com	com	ADP	_	_	4	mark	O	MWEPOS=SCONJ
+2	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	1	fixed	O	_
+3	a	a	ADP	_	_	1	fixed	O	_
+4	obter	obter	VERB	_	VerbForm=Inf	13	advcl	O	_
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	spray	spray	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
 7	de	de	ADP	_	_	8	case	O	_
@@ -252354,7 +252354,7 @@
 
 # text = Com vistas a produzir o efeito de congelamento das estimativas que ultrapassaram este valor limite, é realizado o seguinte procedimento..
 # sent_id = 20-20140904-TESEDSC_0-525
-1	Com	com	ADP	_	_	2	mark	O	MWEPOS=SCONJ
+1	Com	com	ADP	_	_	4	mark	O	MWEPOS=SCONJ
 2	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	1	fixed	O	_
 3	a	a	ADP	_	_	1	fixed	O	_
 4	produzir	produzir	VERB	_	VerbForm=Inf	19	advcl	O	_
@@ -252599,10 +252599,10 @@
 
 # text = Com vistas a introduzir a informação a priori sobre os valores limites (toleráveis) das susceptibilidades das fontes, criou-se um vetor
 # sent_id = 20-20140904-TESEDSC_0-530
-1	Com	com	ADP	_	_	2	mark	O	_
-2	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	23	advcl	O	_
-3	a	a	SCONJ	_	_	4	mark	O	_
-4	introduzir	introduzir	VERB	_	VerbForm=Inf	2	xcomp	O	_
+1	Com	com	ADP	_	_	4	mark	O	MWEPOS=SCONJ
+2	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	1	fixed	O	_
+3	a	a	ADP	_	_	1	fixed	O	_
+4	introduzir	introduzir	VERB	_	VerbForm=Inf	23	advcl	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
 6	informação	informação	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
 7	a	a	ADP	_	_	8	case	O	_
@@ -253451,10 +253451,10 @@
 14	nenhum	nenhum	DET	_	Gender=Masc|Number=Sing|PronType=Neg	15	det	O	_
 15	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	12	nsubj:pass	O	_
 16	teórico	teórico	ADJ	_	Gender=Masc|Number=Sing	15	amod	O	_
-17	com	com	ADP	_	_	18	case	O	_
-18	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	acl	O	_
-19	a	a	SCONJ	_	_	20	mark	O	_
-20	determinar	determinar	VERB	_	VerbForm=Inf	18	advcl	O	_
+17	com	com	ADP	_	_	20	mark	O	MWEPOS=SCONJ
+18	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	17	fixed	O	_
+19	a	a	ADP	_	_	17	fixed	O	_
+20	determinar	determinar	VERB	_	VerbForm=Inf	12	advcl	O	_
 21	relações	relação	NOUN	_	Gender=Fem|Number=Plur	20	obj	O	_
 22	matemáticas	matemático	ADJ	_	Gender=Fem|Number=Plur	21	amod	O	_
 23	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	24	nsubj	O	_
@@ -253939,11 +253939,11 @@
 20	Deconvolução	Deconvolução	PROPN	_	Gender=Fem|Number=Sing	18	nsubj:pass	O	_
 21	de	de	ADP	_	_	20	flat:name	O	_
 22	Euler	Euler	PROPN	_	Number=Sing	20	flat:name	O	_
-23	com	com	ADP	_	_	24	case	O	_
-24	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	18	advcl	O	_
-25	a	a	SCONJ	_	_	27	mark	O	_
+23	com	com	ADP	_	_	27	mark	O	MWEPOS=SCONJ
+24	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	23	fixed	O	_
+25	a	a	ADP	_	_	23	fixed	O	_
 26	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Sing|PronType=Prs	27	expl	O	_
-27	obter	obter	VERB	_	VerbForm=Inf	24	advcl	O	_
+27	obter	obter	VERB	_	VerbForm=Inf	18	advcl	O	_
 28	alguma	algum	DET	_	Gender=Fem|Number=Sing|PronType=Ind	29	det	O	_
 29	pista	pista	NOUN	_	Gender=Fem|Number=Sing	27	obj	O	_
 30	.	.	PUNCT	_	_	18	punct	O	_
@@ -257212,10 +257212,10 @@
 10	figura	figura	NOUN	_	Gender=Fem|Number=Sing	3	parataxis	O	_
 11	3.5	3.5	NUM	_	NumType=Card	10	nummod	O	_
 12	)	)	PUNCT	_	_	10	punct	O	_
-13	com	com	ADP	_	_	14	mark	O	_
-14	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	7	acl	O	_
-15	a	a	SCONJ	_	_	16	mark	O	_
-16	acrescentar	acrescentar	VERB	_	VerbForm=Inf	14	advcl	O	_
+13	com	com	ADP	_	_	16	mark	O	MWEPOS=SCONJ
+14	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	13	fixed	O	_
+15	a	a	ADP	_	_	13	fixed	O	_
+16	acrescentar	acrescentar	VERB	_	VerbForm=Inf	3	advcl	O	_
 17	informação	informação	NOUN	_	Gender=Fem|Number=Sing	16	obj	O	_
 18	e	e	CCONJ	_	_	19	cc	O	_
 19	enriquecer	enriquecer	VERB	_	VerbForm=Inf	16	conj	O	_
@@ -259196,7 +259196,7 @@
 7	entanto	entanto	NOUN	_	Gender=Masc|Number=Sing	5	fixed	O	_
 8	,	,	PUNCT	_	_	5	punct	O	_
 9	novos	novo	ADJ	_	Gender=Masc|Number=Plur	10	amod	O	_
-10	perfis	perfi	NOUN	_	Gender=Masc|Number=Plur	11	nsubj	O	_
+10	perfis	perfil	NOUN	_	Gender=Masc|Number=Plur	11	nsubj	O	_
 11	podem	poder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 12	ser	ser	AUX	_	VerbForm=Inf	13	aux:pass	O	_
 13	traçados	traçar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	11	xcomp	O	_
@@ -259209,10 +259209,10 @@
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
 20	anteriores	anterior	NOUN	_	Gender=Masc|Number=Plur	17	obl	O	_
 21	,	,	PUNCT	_	_	23	punct	O	_
-22	com	com	ADP	_	_	23	case	O	_
-23	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	13	advcl	O	_
-24	a	a	ADP	_	_	25	case	O	_
-25	complementar	complementar	NOUN	_	Gender=Fem|Number=Sing	23	obl	O	_
+22	com	com	ADP	_	_	25	mark	O	MWEPOS=SCONJ
+23	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	22	fixed	O	_
+24	a	a	ADP	_	_	22	fixed	O	_
+25	complementar	complementar	NOUN	_	Gender=Fem|Number=Sing	13	advcl	O	_
 26	e	e	CCONJ	_	_	27	cc	O	_
 27	detalhar	detalhar	VERB	_	VerbForm=Inf	23	conj	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
@@ -269517,10 +269517,10 @@
 11	figura	figura	NOUN	_	Gender=Fem|Number=Sing	3	parataxis	O	_
 12	3.5	3.5	NUM	_	NumType=Card	11	nummod	O	_
 13	)	)	PUNCT	_	_	11	punct	O	_
-14	com	com	ADP	_	_	15	mark	O	_
-15	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	8	acl	O	_
-16	a	a	SCONJ	_	_	17	mark	O	_
-17	mapear	mapear	VERB	_	VerbForm=Inf	15	xcomp	O	_
+14	com	com	ADP	_	_	17	mark	O	MWEPOS=SCONJ
+15	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	14	fixed	O	_
+16	a	a	ADP	_	_	14	fixed	O	_
+17	mapear	mapear	VERB	_	VerbForm=Inf	3	advcl	O	_
 18	possíveis	possível	ADJ	_	Gender=Masc|Number=Plur	19	amod	O	_
 19	afloramentos	afloramento	NOUN	_	Gender=Masc|Number=Plur	17	obj	B=LITOLOGIA	_
 20-21	do	_	_	_	_	_	_	_	_
@@ -278661,12 +278661,12 @@
 32	de	de	ADP	_	_	34	case	O	_
 33	novos	novo	ADJ	_	Gender=Masc|Number=Plur	34	amod	O	_
 34	projetos	projeto	NOUN	_	Gender=Masc|Number=Plur	31	nmod	O	_
-35	com	com	ADP	_	_	36	case	O	_
-36	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	29	advcl	O	_
+35	com	com	ADP	_	_	36	case	O	MWEPOS=ADP
+36	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	35	fixed	O	_
 37-38	à	_	_	_	_	_	_	_	_
-37	a	a	ADP	_	_	39	case	O	_
+37	a	a	ADP	_	_	35	fixed	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
-39	delineação	delineação	NOUN	_	Gender=Fem|Number=Sing	36	obl	O	_
+39	delineação	delineação	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
 40	de	de	ADP	_	_	42	case	O	_
 41	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	42	det	O	_
 42	geometria	geometria	NOUN	_	Gender=Fem|Number=Sing	39	nmod	O	_
@@ -280631,10 +280631,10 @@
 15	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	ser	ser	AUX	_	VerbForm=Inf	17	aux:pass	O	_
 17	realizada	realizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	15	xcomp	O	_
-18	com	com	ADP	_	_	19	case	O	_
-19	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	17	xcomp	O	_
-20	a	a	SCONJ	_	_	21	mark	O	_
-21	acrescentar	acrescentar	VERB	_	VerbForm=Inf	19	advcl	O	_
+18	com	com	ADP	_	_	21	mark	O	MWEPOS=SCONJ
+19	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	18	fixed	O	_
+20	a	a	ADP	_	_	18	fixed	O	_
+21	acrescentar	acrescentar	VERB	_	VerbForm=Inf	15	advcl	O	_
 22	alguma	algum	DET	_	Gender=Fem|Number=Sing|PronType=Ind	23	det	O	_
 23	contribuição	contribuição	NOUN	_	Gender=Fem|Number=Sing	21	obj	O	_
 24-25	aos	_	_	_	_	_	_	_	_
@@ -287701,12 +287701,12 @@
 15	reverso	reverso	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	14	nmod	B=FLUIDO	_
-18	com	com	ADP	_	_	19	case	O	_
-19	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	14	acl	O	_
+18	com	com	ADP	_	_	22	case	O	MWEPOS=ADP
+19	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	18	fixed	O	_
 20-21	à	_	_	_	_	_	_	_	_
-20	a	a	ADP	_	_	22	case	O	_
+20	a	a	ADP	_	_	18	fixed	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
-22	análise	análise	NOUN	_	Gender=Fem|Number=Sing	19	obl	O	_
+22	análise	análise	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 23-24	da	_	_	_	_	_	_	_	_
 23	de	de	ADP	_	_	25	case	O	_
 24	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	det	O	_
@@ -301595,12 +301595,12 @@
 73	de	de	ADP	_	_	74	case	O	_
 74	levantamentos	levantamento	NOUN	_	Gender=Masc|Number=Plur	72	nmod	O	_
 75	precisos	preciso	ADJ	_	Gender=Masc|Number=Plur	74	amod	O	_
-76	com	com	ADP	_	_	77	case	O	_
-77	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	74	acl	O	_
+76	com	com	ADP	_	_	80	case	O	MWEPOS=ADP
+77	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	76	fixed	O	_
 78-79	à	_	_	_	_	_	_	_	_
-78	a	a	ADP	_	_	80	case	O	_
+78	a	a	ADP	_	_	76	fixed	O	_
 79	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	80	det	O	_
-80	remoção	remoção	NOUN	_	Gender=Fem|Number=Sing	77	obl	O	_
+80	remoção	remoção	NOUN	_	Gender=Fem|Number=Sing	70	obl	O	_
 81-82	do	_	_	_	_	_	_	_	_
 81	de	de	ADP	_	_	83	case	O	_
 82	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	83	det	O	_
@@ -306322,12 +306322,12 @@
 2	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	4	nsubj:pass	O	_
 3	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux:pass	O	_
 4	comparados	comparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-5	com	com	ADP	_	_	9	case	O	_
-6	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	4	xcomp	O	_
+5	com	com	ADP	_	_	9	case	O	MWEPOS=ADP
+6	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	5	fixed	O	_
 7-8	à	_	_	_	_	_	_	_	_
-7	a	a	ADP	_	_	9	case	O	_
+7	a	a	ADP	_	_	5	fixed	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
-9	influência	influência	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
+9	influência	influência	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
 10	tanto	tanto	ADV	_	_	13	advmod	O	_
 11-12	da	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
