commit 7fc9bff36097f635effbfd32b37b8f46d6c5a792
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Jun 9 14:02:51 2021 -0300

    feats de fácies

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 1e9d768..556a671 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -20092,15 +20092,15 @@
 3	em	em	ADP	_	_	5	case	O	_
 4	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	5	det	O	_
 5	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	2	obl	O	_
-6	técnicas	técnico	ADJ	_	Gender=Fem|Number=Plur	5	amod	O	_
+6	técnicas	técnica	NOUN	_	Gender=Fem|Number=Plur	2	nsubj:pass	O	_
 7	e	e	CCONJ	_	_	8	cc	O	_
-8	métodos	método	NOUN	_	Gender=Masc|Number=Plur	5	conj	B=MÉTODO	_
+8	métodos	método	NOUN	_	Gender=Masc|Number=Plur	6	conj	B=MÉTODO	_
 9	de	de	ADP	_	_	10	case	O	_
 10	descrição	descrição	NOUN	_	Gender=Fem|Number=Sing	8	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
-12	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	10	nmod	B=LITOLOGIA	_
+12	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	10	nmod	B=LITOLOGIA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
-14	análise	análise	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
+14	análise	análise	NOUN	_	Gender=Fem|Number=Sing	6	conj	O	_
 15-16	da	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
@@ -20122,8 +20122,8 @@
 8	,	,	PUNCT	_	_	9	punct	O	_
 9	descrições	descrição	NOUN	_	Gender=Fem|Number=Plur	6	conj	O	_
 10	de	de	ADP	_	_	11	case	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	9	nmod	B=LITOLOGIA	_
-12	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	11	amod	I=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	9	nmod	B=LITOLOGIA	_
+12	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	11	amod	I=LITOLOGIA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
 14	levantamento	levantamento	NOUN	_	Gender=Masc|Number=Sing	6	conj	O	_
 15	de	de	ADP	_	_	16	case	O	_
@@ -20306,8 +20306,8 @@
 11	:	:	PUNCT	_	_	12	punct	O	_
 12	análise	análise	NOUN	_	Gender=Fem|Number=Sing	8	appos	O	_
 13	de	de	ADP	_	_	14	case	O	_
-14	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	12	nmod	B=LITOLOGIA	_
-15	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	14	amod	I=LITOLOGIA	_
+14	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	12	nmod	B=LITOLOGIA	_
+15	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	14	amod	I=LITOLOGIA	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	levantamento	levantamento	NOUN	_	Gender=Masc|Number=Sing	12	conj	O	_
 18	de	de	ADP	_	_	19	case	O	_
@@ -20326,15 +20326,15 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-42
 1	Análise	Análise	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 2	de	de	ADP	_	_	3	case	O	_
-3	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	1	nmod	B=LITOLOGIA	_
-4	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	3	amod	I=LITOLOGIA	_
+3	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	1	nmod	B=LITOLOGIA	_
+4	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	3	amod	I=LITOLOGIA	_
 
 # text = A análise de fácies é um método fundamental de caracterização de rocha com atributos litológicos, físicos e biológicos singulares e distintos das demais..
 # sent_id = 284-20150917-MONOGRAFIA_0-43
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	análise	análise	NOUN	_	Gender=Fem|Number=Sing	7	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	cop	O	_
 6	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
 7	método	método	NOUN	_	Gender=Masc|Number=Sing	0	root	B=MÉTODO	_
@@ -20597,7 +20597,7 @@
 25	de	de	ADP	_	_	24	flat:name	O	_
 26	Identificação	Identificação	PROPN	_	Number=Sing	24	flat:name	O	_
 27	de	de	ADP	_	_	24	flat:name	O	_
-28	Fácies	fácies	PROPN	_	Number=Sing	24	flat:name	B=LITOLOGIA	_
+28	Fácies	fácies	PROPN	_	Gender=Fem|Number=Plur	24	flat:name	B=LITOLOGIA	_
 29	.	.	PUNCT	_	_	10	punct	O	_
 30	.	.	PUNCT	_	_	10	punct	O	_
 
@@ -20828,7 +20828,7 @@
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	análise	análise	NOUN	_	Gender=Fem|Number=Sing	13	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
-20	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	18	nmod	B=LITOLOGIA	_
+20	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	18	nmod	B=LITOLOGIA	_
 21	e	e	CCONJ	_	_	24	cc	O	_
 22-23	na	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
@@ -21366,7 +21366,7 @@
 14	sucessão	sucessão	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 15	vertical	vertical	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	14	nmod	B=LITOLOGIA	_
+17	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=LITOLOGIA	_
 18	.	.	PUNCT	_	_	6	punct	O	_
 19	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -21625,7 +21625,7 @@
 14	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	15	det	O	_
 15	cunha	cunha	NOUN	_	Gender=Fem|Number=Sing	12	obl:agent	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	15	nmod	B=LITOLOGIA	_
+17	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	15	nmod	B=LITOLOGIA	_
 18	em	em	ADP	_	_	20	case	O	_
 19	cujo	cujo	DET	_	Gender=Masc|Number=Sing|PronType=Rel	20	det	O	_
 20	ápice	ápice	NOUN	_	Gender=Masc|Number=Sing	21	nsubj	O	_
@@ -23661,8 +23661,8 @@
 1	Esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	2	det	O	_
 2	complexidade	complexidade	NOUN	_	Gender=Fem|Number=Sing	6	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
-5	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	4	amod	I=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
+5	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	4	amod	I=LITOLOGIA	_
 6	levou	levar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 7	a	a	ADP	_	_	9	case	O	_
 8	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -25484,7 +25484,7 @@
 5	ESTRATIGRÁFICA	estratigráfico	ADJ	_	Gender=Fem|Number=Sing	4	flat:foreign	O	_
 6	DE	de	ADP	_	_	7	case	O	_
 7	FÁCIES	fácies	NOUN	_	Gender=Fem|Number=Plur	4	nmod	B=LITOLOGIA	_
-8	SEDIMENTARES	sedimentar	ADJ	_	Gender=Masc|Number=Plur	7	amod	I=LITOLOGIA	_
+8	SEDIMENTARES	sedimentar	ADJ	_	Gender=Fem|Number=Plur	7	amod	I=LITOLOGIA	_
 
 # text = Três seções colunares na escala 1:100 (localização na Tabela 1) foram levantadas na porção leste da Bacia do Araripe..
 # sent_id = 284-20150917-MONOGRAFIA_0-173
@@ -25531,7 +25531,7 @@
 5	sucessão	sucessão	NOUN	_	Gender=Fem|Number=Sing	3	obj	O	_
 6	vertical	vertical	ADJ	_	Gender=Fem|Number=Sing	5	amod	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	5	nmod	B=LITOLOGIA	_
+8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	5	nmod	B=LITOLOGIA	_
 9	(	(	PUNCT	_	_	10	punct	O	_
 10	granodecrescência	granodecrescência	NOUN	_	Gender=Fem|Number=Sing	5	appos	O	_
 11	ascendente	ascendente	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
@@ -25546,7 +25546,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-175
 1	Foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	2	aux:pass	O	_
 2	descritas	descrever	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
-3	nove	nove	NUM	_	NumType=Card	4	nummod	O	_
+3	nove	nove	NUM	_	Gender=Fem|NumType=Card	4	nummod	O	_
 4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nsubj:pass	B=LITOLOGIA	_
 5	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	4	amod	I=LITOLOGIA	_
 6	,	,	PUNCT	_	_	9	punct	O	_
@@ -25622,8 +25622,8 @@
 53	estarem	estar	AUX	_	Number=Plur|Person=3|VerbForm=Inf	54	cop	O	_
 54	dispostas	disposto	ADJ	_	Gender=Fem|Number=Plur	42	advcl	O	_
 55	sobre	sobre	ADP	_	_	56	case	O	_
-56	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	54	obl	B=LITOLOGIA	_
-57	proximais	proximal	ADJ	_	Gender=Masc|Number=Plur	56	amod	O	_
+56	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	54	obl	B=LITOLOGIA	_
+57	proximais	proximal	ADJ	_	Gender=Fem|Number=Plur	56	amod	O	_
 58	,	,	PUNCT	_	_	61	punct	O	_
 59	além	além	ADV	_	_	61	cc	O	_
 60	de	de	ADP	_	_	59	fixed	O	_
@@ -25675,8 +25675,8 @@
 28	seção	seção	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 29	aflorante	aflorante	ADJ	_	Gender=Fem|Number=Sing	28	amod	O	_
 30	com	com	ADP	_	_	31	case	O	_
-31	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	26	nmod	B=LITOLOGIA	_
-32	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	33	nsubj	O	_
+31	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	26	nmod	B=LITOLOGIA	_
+32	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	33	nsubj	O	_
 33	apresentam	apresentar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	31	acl:relcl	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
 35	mesmo	mesmo	ADJ	_	Gender=Masc|Number=Sing	36	amod	O	_
@@ -25909,8 +25909,8 @@
 4	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	I=LITOLOGIA	_
 5	acanalada	acanalar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	I=LITOLOGIA	_
 6	(	(	PUNCT	_	_	7	punct	O	_
-7	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	5	appos	B=LITOLOGIA	_
-8	Gt	Gt	PROPN	_	Number=Sing	7	flat:name	O	_
+7	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	5	appos	B=LITOLOGIA	_
+8	Gt	Gt	PROPN	_	Gender=Fem|Number=Sing	7	flat:name	O	_
 9	)	)	PUNCT	_	_	7	punct	O	_
 
 # text = Descrição:
@@ -26187,8 +26187,8 @@
 4	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	I=LITOLOGIA	_
 5	acanalada	acanalar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	I=LITOLOGIA	_
 6	(	(	PUNCT	_	_	7	punct	O	_
-7	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	1	appos	B=LITOLOGIA	_
-8	St	St	PROPN	_	Number=Sing	7	flat:name	I=LITOLOGIA	_
+7	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	1	appos	B=LITOLOGIA	_
+8	St	St	PROPN	_	Gender=Fem|Number=Sing	7	flat:name	I=LITOLOGIA	_
 9	)	)	PUNCT	_	_	7	punct	O	_
 
 # text = Descrição:
@@ -26262,7 +26262,7 @@
 # text = A fácies
 # sent_id = 284-20150917-MONOGRAFIA_0-204
 1	A	a	ADP	_	_	2	case	O	_
-2	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	0	root	B=LITOLOGIA	_
+2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	0	root	B=LITOLOGIA	_
 
 # text = St apresenta estratificação cruzada acanalada, com tamanhos médios de 30 cm..
 # sent_id = 284-20150917-MONOGRAFIA_0-205
@@ -26535,8 +26535,8 @@
 7	climbing	climbing	X	_	_	5	conj	O	_
 8	ripples	ripples	X	_	_	7	flat:foreign	O	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	7	appos	B=LITOLOGIA	_
-11	Sr	Sr	PROPN	_	Number=Sing	10	flat:name	O	_
+10	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	7	appos	B=LITOLOGIA	_
+11	Sr	Sr	PROPN	_	Gender=Fem|Number=Sing	10	flat:name	O	_
 12	)	)	PUNCT	_	_	10	punct	O	_
 13	Descrição	descrição	NOUN	_	Gender=Fem|Number=Sing	1	conj	O	_
 14	:	:	PUNCT	_	_	15	punct	O	_
@@ -26701,8 +26701,8 @@
 4	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	O	_
 5	sigmoide	sigmoide	ADJ	_	Gender=Masc|Number=Sing	4	xcomp	B=LITOLOGIA	_
 6	(	(	PUNCT	_	_	7	punct	O	_
-7	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	1	appos	B=LITOLOGIA	_
-8	Ssg	Ssg	PROPN	_	Number=Sing	7	flat:name	O	_
+7	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	1	appos	B=LITOLOGIA	_
+8	Ssg	Ssg	PROPN	_	Gender=Fem|Number=Sing	7	flat:name	O	_
 9	)	)	PUNCT	_	_	7	punct	O	_
 
 # text = Descrição:
@@ -27155,8 +27155,8 @@
 3	Arenito	Arenito	PROPN	_	Gender=Masc|Number=Sing	0	root	B=LITOLOGIA	_
 4	Maciço	Maciço	PROPN	_	Number=Sing	3	flat:name	B=LITOLOGIA	_
 5	(	(	PUNCT	_	_	6	punct	O	_
-6	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	3	appos	B=LITOLOGIA	_
-7	Sm	Sm	PROPN	_	Number=Sing	6	flat:name	O	_
+6	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	3	appos	B=LITOLOGIA	_
+7	Sm	Sm	PROPN	_	Gender=Fem|Number=Sing	6	flat:name	O	_
 8	)	)	PUNCT	_	_	6	punct	O	_
 
 # text = Descrição:
@@ -27364,8 +27364,8 @@
 7	e	e	CCONJ	_	_	8	cc	O	_
 8	linsen	linsen	NOUN	_	Gender=Masc|Number=Sing	3	conj	O	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	1	appos	B=LITOLOGIA	_
-11	H	h	PROPN	_	Number=Sing	10	flat:name	I=LITOLOGIA	_
+10	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	1	appos	B=LITOLOGIA	_
+11	H	h	PROPN	_	Gender=Fem|Number=Sing	10	flat:name	I=LITOLOGIA	_
 12	)	)	PUNCT	_	_	10	punct	O	_
 
 # text = Descrição:
@@ -27376,7 +27376,7 @@
 # text = A fácies
 # sent_id = 284-20150917-MONOGRAFIA_0-249
 1	A	a	ADP	_	_	2	case	O	_
-2	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	0	root	B=LITOLOGIA	_
+2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	0	root	B=LITOLOGIA	_
 
 # text = H é aqui considerada um conjunto de fácies que ocorrem associadas de forma heterolítica complexa, caracterizando uma associação genética..
 # sent_id = 284-20150917-MONOGRAFIA_0-250
@@ -27387,7 +27387,7 @@
 5	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
 6	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	4	nsubj:pass	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=LITOLOGIA	_
+8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod	B=LITOLOGIA	_
 9	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	10	nsubj	O	_
 10	ocorrem	ocorrer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	6	acl:relcl	O	_
 11	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	10	xcomp	O	_
@@ -28458,7 +28458,7 @@
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	associação	associação	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
-12	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	10	nmod	B=LITOLOGIA	_
+12	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	10	nmod	B=LITOLOGIA	_
 13	e	e	CCONJ	_	_	14	cc	O	_
 14	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
 15-16	do	_	_	_	_	_	_	_	_
@@ -28474,7 +28474,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-293
 1	Associação	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 2	de	de	ADP	_	_	3	case	O	_
-3	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	1	nmod	B=LITOLOGIA	_
+3	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	1	nmod	B=LITOLOGIA	_
 4	e	e	CCONJ	_	_	5	cc	O	_
 5	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	3	conj	O	_
 6-7	do	_	_	_	_	_	_	_	_
@@ -28760,9 +28760,9 @@
 8	em	em	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	qual	qual	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	16	obl	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	16	nsubj	B=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	16	nsubj	B=LITOLOGIA	_
 12	de	de	ADP	_	_	13	case	O	_
-13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	obl	O	_
+13	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 14	fluviais	fluvial	ADJ	_	Gender=Masc|Number=Plur	13	amod	O	_
 15	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	16	aux:pass	O	_
 16	sucedidas	sucedido	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	5	acl:relcl	O	_
@@ -29004,13 +29004,13 @@
 11	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	12	det	O	_
 12	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	7	nmod	B=LITOLOGIA	_
 13	(	(	PUNCT	_	_	14	punct	O	_
-14	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	12	appos	B=LITOLOGIA	_
+14	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	12	appos	B=LITOLOGIA	_
 15	St	St	PROPN	_	Gender=Masc|Number=Sing	14	nmod	I=LITOLOGIA	_
 16	)	)	PUNCT	_	_	14	punct	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
 18	conglomerados	conglomerar	NOUN	_	Gender=Masc|Number=Plur	14	conj	O	_
 19	(	(	PUNCT	_	_	20	punct	O	_
-20	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	18	nmod:appos	B=LITOLOGIA	_
+20	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	18	nmod:appos	B=LITOLOGIA	_
 21	Gt	Gt	PROPN	_	Gender=Masc|Number=Sing	20	nmod	O	_
 22	)	)	PUNCT	_	_	20	punct	O	_
 23-24	da	_	_	_	_	_	_	_	_
@@ -29026,13 +29026,13 @@
 31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	32	det	O	_
 32	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	18	nmod	B=LITOLOGIA	_
 33	(	(	PUNCT	_	_	34	punct	O	_
-34	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	32	appos	B=LITOLOGIA	_
-35	Fl	Fl	PROPN	_	Gender=Masc|Number=Sing	34	parataxis	O	_
+34	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	32	appos	B=LITOLOGIA	_
+35	Fl	Fl	PROPN	_	Gender=Fem|Number=Sing	34	parataxis	O	_
 36	)	)	PUNCT	_	_	34	punct	O	_
 37	e	e	CCONJ	_	_	38	cc	O	_
 38	gipsita	gipsita	NOUN	_	Gender=Fem|Number=Sing	12	conj	B=LITOLOGIA	_
 39	(	(	PUNCT	_	_	40	punct	O	_
-40	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	38	nmod:appos	B=LITOLOGIA	_
+40	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	38	nmod:appos	B=LITOLOGIA	_
 41	GIP	GIP	PROPN	_	Gender=Masc|Number=Sing	40	nmod	O	_
 42	)	)	PUNCT	_	_	40	punct	O	_
 43-44	das	_	_	_	_	_	_	_	_
@@ -29362,8 +29362,8 @@
 20	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	19	acl	I=LITOLOGIA	_
 21	acanalada	acanalar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	19	acl	I=LITOLOGIA	_
 22	(	(	PUNCT	_	_	23	punct	O	_
-23	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	19	appos	B=LITOLOGIA	_
-24	St	St	PROPN	_	Number=Sing	23	flat:name	I=LITOLOGIA	_
+23	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	19	appos	B=LITOLOGIA	_
+24	St	St	PROPN	_	Gender=Fem|Number=Sing	23	flat:name	I=LITOLOGIA	_
 25	)	)	PUNCT	_	_	23	punct	O	_
 26	em	em	ADP	_	_	27	case	O	_
 27	contato	contato	NOUN	_	Gender=Masc|Number=Sing	9	obl	O	_
@@ -29379,8 +29379,8 @@
 37	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	36	acl	I=LITOLOGIA	_
 38	acanalada	acanalar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	36	acl	I=LITOLOGIA	_
 39	(	(	PUNCT	_	_	40	punct	O	_
-40	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	36	appos	B=LITOLOGIA	_
-41	St	St	PROPN	_	Number=Sing	40	flat:name	I=LITOLOGIA	_
+40	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	36	appos	B=LITOLOGIA	_
+41	St	St	PROPN	_	Gender=Fem|Number=Sing	40	flat:name	I=LITOLOGIA	_
 42	)	)	PUNCT	_	_	40	punct	O	_
 43	,	,	PUNCT	_	_	45	punct	O	_
 44	como	como	SCONJ	_	_	45	mark	O	_
@@ -29576,7 +29576,7 @@
 24	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	21	nmod	B=LITOLOGIA	_
 25	pretos	preto	ADJ	_	Gender=Masc|Number=Plur	24	amod	I=LITOLOGIA	_
 26	(	(	PUNCT	_	_	27	punct	O	_
-27	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	24	nmod:appos	B=LITOLOGIA	_
+27	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	24	nmod:appos	B=LITOLOGIA	_
 28	Fl	Fl	PROPN	_	Gender=Masc|Number=Plur	27	nmod	O	_
 29	)	)	PUNCT	_	_	27	punct	O	_
 30	.	.	PUNCT	_	_	4	punct	O	_
@@ -29841,10 +29841,10 @@
 25	de	de	ADP	_	_	26	case	O	_
 26	camadas	camada	NOUN	_	Gender=Fem|Number=Plur	24	nmod	O	_
 27	de	de	ADP	_	_	28	case	O	_
-28	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	26	nmod	B=LITOLOGIA	_
+28	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	26	nmod	B=LITOLOGIA	_
 29	heterolítica	heterolítica	NOUN	_	Gender=Fem|Number=Sing	28	compound	O	_
 30	(	(	PUNCT	_	_	31	punct	O	_
-31	H	h	NOUN	_	Gender=Masc|Number=Sing	28	appos	O	_
+31	H	h	NOUN	_	Gender=Fem|Number=Sing	28	appos	O	_
 32	)	)	PUNCT	_	_	31	punct	O	_
 33	,	,	PUNCT	_	_	34	punct	O	_
 34	abaixo	abaixo	ADV	_	_	19	advmod	O	_
@@ -29852,8 +29852,8 @@
 36	com	com	ADP	_	_	37	case	O	_
 37	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	45	obl	B=LITOLOGIA	_
 38	(	(	PUNCT	_	_	39	punct	O	_
-39	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	37	nmod:appos	B=LITOLOGIA	_
-40	Fl	Fl	PROPN	_	Gender=Masc|Number=Sing	39	parataxis	O	_
+39	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	37	nmod:appos	B=LITOLOGIA	_
+40	Fl	Fl	PROPN	_	Gender=Fem|Number=Sing	39	parataxis	O	_
 41	)	)	PUNCT	_	_	39	punct	O	_
 42	,	,	PUNCT	_	_	43	punct	O	_
 43	acima	acima	ADV	_	_	45	advmod	O	_
@@ -30102,16 +30102,16 @@
 21	calcíferos	calcífero	ADJ	_	Gender=Masc|Number=Plur	20	amod	O	_
 22	esverdeados	esverdear	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	20	acl	O	_
 23	(	(	PUNCT	_	_	24	punct	O	_
-24	Fácies	fácies	PROPN	_	Gender=Masc|Number=Sing	20	appos	B=LITOLOGIA	_
-25	Fl	Fl	PROPN	_	Number=Sing	24	flat:name	O	_
+24	Fácies	fácies	PROPN	_	Gender=Fem|Number=Sing	20	appos	B=LITOLOGIA	_
+25	Fl	Fl	PROPN	_	Gender=Fem|Number=Sing	24	flat:name	O	_
 26	)	)	PUNCT	_	_	24	punct	O	_
 27	em	em	ADP	_	_	28	case	O	_
 28	contato	contato	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
 29	com	com	ADP	_	_	30	case	O	_
 30	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	28	nmod	B=LITOLOGIA	_
 31	(	(	PUNCT	_	_	32	punct	O	_
-32	Fácies	Fácies	PROPN	_	Gender=Masc|Number=Sing	30	appos	B=LITOLOGIA	_
-33	Gt	Gt	PROPN	_	Number=Sing	32	flat:name	O	_
+32	Fácies	Fácies	PROPN	_	Gender=Fem|Number=Sing	30	appos	B=LITOLOGIA	_
+33	Gt	Gt	PROPN	_	Gender=Fem|Number=Sing	32	flat:name	O	_
 34	)	)	PUNCT	_	_	32	punct	O	_
 35	,	,	PUNCT	_	_	36	punct	O	_
 36	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	30	conj	O	_
@@ -30166,7 +30166,7 @@
 1	Estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	2	det	O	_
 2	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	7	nsubj	B=LITOLOGIA	_
 3	(	(	PUNCT	_	_	4	punct	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	appos	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	appos	B=LITOLOGIA	_
 5	Gt	Gt	PROPN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 6	)	)	PUNCT	_	_	4	punct	O	_
 7	aparecem	aparecer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -30195,7 +30195,7 @@
 30	a	a	ADP	_	_	31	case	O	_
 31	grossos	grosso	ADJ	_	Gender=Masc|Number=Plur	29	obl	O	_
 32	(	(	PUNCT	_	_	33	punct	O	_
-33	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	26	nmod:appos	B=LITOLOGIA	_
+33	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	26	nmod:appos	B=LITOLOGIA	_
 34	St	St	PROPN	_	Gender=Masc|Number=Sing	33	nmod	I=LITOLOGIA	_
 35	)	)	PUNCT	_	_	33	punct	O	_
 36	.	.	PUNCT	_	_	7	punct	O	_
@@ -30435,7 +30435,7 @@
 22	cavalgante	cavalgante	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
 23	(	(	PUNCT	_	_	24	punct	O	_
 24	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	18	nmod:appos	B=LITOLOGIA	_
-25	Sr	Sr	PROPN	_	Gender=Masc|Number=Sing	24	appos	O	_
+25	Sr	Sr	PROPN	_	Gender=Fem|Number=Sing	24	appos	O	_
 26	)	)	PUNCT	_	_	24	punct	O	_
 27	e	e	CCONJ	_	_	28	cc	O	_
 28	heterolíticas	heterolítico	ADJ	_	Gender=Fem|Number=Plur	22	conj	O	_
@@ -30506,7 +30506,7 @@
 7	de	de	ADP	_	_	8	case	O	_
 8	folhelhos	folhelho	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=LITOLOGIA	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=LITOLOGIA	_
+10	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod	B=LITOLOGIA	_
 11	Fl	Fl	PROPN	_	Gender=Masc|Number=Plur	10	nmod	O	_
 12	)	)	PUNCT	_	_	10	punct	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
@@ -30519,7 +30519,7 @@
 20	com	com	ADP	_	_	21	case	O	_
 21	calcários	calcário	NOUN	_	Gender=Masc|Number=Plur	19	obl	B=LITOLOGIA	_
 22	(	(	PUNCT	_	_	23	punct	O	_
-23	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	21	nmod:appos	B=LITOLOGIA	_
+23	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	21	nmod:appos	B=LITOLOGIA	_
 24	Ca	Ca	PROPN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 25	)	)	PUNCT	_	_	23	punct	O	_
 26	com	com	ADP	_	_	29	case	O	_
@@ -31024,7 +31024,7 @@
 16	gradam	grar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	acl:relcl	O	_
 17	a	a	ADP	_	_	18	case	O	_
 18	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	16	obj	B=LITOLOGIA	_
-19	sigmoides	sigmoide	ADJ	_	Gender=Masc|Number=Plur	18	amod	B=LITOLOGIA	_
+19	sigmoides	sigmoide	ADJ	_	Gender=Fem|Number=Plur	18	amod	B=LITOLOGIA	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	heterolíticas	heterolítico	ADJ	_	Gender=Fem|Number=Plur	19	conj	O	_
 22	e	e	CCONJ	_	_	23	cc	O	_
@@ -31081,7 +31081,7 @@
 30	barras	barra	NOUN	_	Gender=Fem|Number=Plur	28	nmod	O	_
 31	sigmoides	sigmoide	ADJ	_	Gender=Fem|Number=Plur	30	amod	B=LITOLOGIA	_
 32	(	(	PUNCT	_	_	33	punct	O	_
-33	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	30	nmod:appos	B=LITOLOGIA	_
+33	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	30	nmod:appos	B=LITOLOGIA	_
 34	Ssg	Ssg	PROPN	_	Gender=Masc|Number=Plur	33	nmod	O	_
 35	)	)	PUNCT	_	_	33	punct	O	_
 36	e	e	CCONJ	_	_	38	cc	O	_
@@ -31090,8 +31090,8 @@
 39	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	38	acl	O	_
 40	cavalgante	cavalgante	ADJ	_	Gender=Fem|Number=Sing	38	amod	O	_
 41	(	(	PUNCT	_	_	42	punct	O	_
-42	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	38	nmod:appos	B=LITOLOGIA	_
-43	Sr	Sr	PROPN	_	Gender=Masc|Number=Sing	42	parataxis	O	_
+42	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	38	nmod:appos	B=LITOLOGIA	_
+43	Sr	Sr	PROPN	_	Gender=Fem|Number=Sing	42	parataxis	O	_
 44	)	)	PUNCT	_	_	42	punct	O	_
 45	.	.	PUNCT	_	_	15	punct	O	_
 46	.	.	PUNCT	_	_	15	punct	O	_
@@ -31130,7 +31130,7 @@
 7	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	6	acl	I=LITOLOGIA	_
 8	acanalada	acanalar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	6	acl	I=LITOLOGIA	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	B=LITOLOGIA	_
+10	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	8	nmod:appos	B=LITOLOGIA	_
 11	St	St	PROPN	_	Gender=Masc|Number=Sing	10	nmod	I=LITOLOGIA	_
 12	)	)	PUNCT	_	_	10	punct	O	_
 13	encerram	encerrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -31447,8 +31447,8 @@
 13	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
 14	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	7	conj	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	14	nmod	B=LITOLOGIA	_
-17	composto	compor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	16	acl	O	_
+16	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=LITOLOGIA	_
+17	composto	compor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	14	acl	O	_
 18	por	por	ADP	_	_	19	case	O	_
 19	seção	seção	NOUN	_	Gender=Fem|Number=Sing	17	obl:agent	O	_
 20	arenosa	arenoso	ADJ	_	Gender=Fem|Number=Sing	19	amod	B=LITOLOGIA	_
@@ -33159,7 +33159,7 @@
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	sucessão	sucessão	NOUN	_	Gender=Fem|Number=Sing	4	conj	O	_
 28	de	de	ADP	_	_	29	case	O	_
-29	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	27	nmod	B=LITOLOGIA	_
+29	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	27	nmod	B=LITOLOGIA	_
 30	típica	típico	NOUN	_	Gender=Fem|Number=Sing	29	nmod	O	_
 31	de	de	ADP	_	_	32	case	O	_
 32	influências	influência	NOUN	_	Gender=Fem|Number=Plur	30	nmod	O	_
@@ -64327,8 +64327,8 @@
 12	e	e	CCONJ	_	_	13	cc	O	_
 13	análise	análise	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	13	nmod	B=LITOLOGIA	_
-16	sedimentares	sedimentar	ADJ	_	Gender=Masc|Number=Plur	15	amod	I=LITOLOGIA	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	13	nmod	B=LITOLOGIA	_
+16	sedimentares	sedimentar	ADJ	_	Gender=Fem|Number=Plur	15	amod	I=LITOLOGIA	_
 17	,	,	PUNCT	_	_	18	punct	O	_
 18	contribuindo	contribuir	VERB	_	VerbForm=Ger	3	advcl	O	_
 19	para	para	ADP	_	_	21	case	O	_
@@ -65475,7 +65475,7 @@
 19	variação	variação	NOUN	_	Gender=Fem|Number=Sing	13	appos	O	_
 20	lateral	lateral	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	de	de	ADP	_	_	22	case	O	_
-22	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	19	nmod	B=LITOLOGIA	_
+22	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	19	nmod	B=LITOLOGIA	_
 23	decorrentes	decorrente	NOUN	_	Gender=Masc|Number=Plur	22	nmod	O	_
 24-25	dos	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	27	case	O	_
@@ -66024,7 +66024,7 @@
 10	variação	variação	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 11	lateral	lateral	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	10	nmod	B=LITOLOGIA	_
+13	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	10	nmod	B=LITOLOGIA	_
 14	,	,	PUNCT	_	_	16	punct	O	_
 15	com	com	ADP	_	_	16	case	O	_
 16	atuação	atuação	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
@@ -67811,8 +67811,8 @@
 13	e	e	CCONJ	_	_	14	cc	O	_
 14	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	12	conj	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	14	nmod	B=LITOLOGIA	_
-17	usuais	usual	ADJ	_	Gender=Masc|Number=Plur	16	amod	O	_
+16	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=LITOLOGIA	_
+17	usuais	usual	ADJ	_	Gender=Fem|Number=Plur	16	amod	O	_
 18	em	em	ADP	_	_	19	case	O	_
 19	análise	análise	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 20	sedimentar	sedimentar	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
@@ -67824,7 +67824,7 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	código	código	NOUN	_	Gender=Masc|Number=Sing	9	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
 5	utilizado	utilizar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
 6-7	neste	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
@@ -67838,7 +67838,7 @@
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
 14	código	código	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	14	nmod	B=LITOLOGIA	_
+16	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=LITOLOGIA	_
 17	proposto	propor	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	14	acl	O	_
 18	inicialmente	inicialmente	ADV	_	_	17	advmod	O	_
 19	por	por	ADP	_	_	20	case	O	_
@@ -67909,7 +67909,7 @@
 6	sucessão	sucessão	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
 7	vertical	vertical	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
 8	de	de	ADP	_	_	9	case	O	_
-9	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=LITOLOGIA	_
+9	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod	B=LITOLOGIA	_
 10	e	e	CCONJ	_	_	11	cc	O	_
 11	entender	entender	VERB	_	VerbForm=Inf	4	conj	O	_
 12	melhor	melhor	ADV	_	_	11	advmod	O	_
@@ -68582,7 +68582,7 @@
 10	a	a	ADP	_	_	11	case	O	_
 11	associações	associação	NOUN	_	Gender=Fem|Number=Plur	9	obl	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	11	nmod	B=LITOLOGIA	_
+13	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	11	nmod	B=LITOLOGIA	_
 14	.	.	PUNCT	_	_	4	punct	O	_
 15	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -68754,8 +68754,8 @@
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	associação	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 8	de	de	ADP	_	_	9	case	O	_
-9	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	7	nmod	B=LITOLOGIA	_
-10	superior	alto	ADJ	_	Gender=Masc|Number=Sing	9	amod	O	_
+9	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	7	nmod	B=LITOLOGIA	_
+10	superior	alto	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
 11-12	desta	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	13	det	O	_
@@ -68880,8 +68880,8 @@
 1	4.1	4.1	NUM	_	NumType=Card	2	nummod	O	_
 2	ASSOCIAÇÃO	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 3	DE	de	ADP	_	_	4	case	O	_
-4	FÁCIES	fácies	PROPN	_	Number=Sing	2	nmod	B=LITOLOGIA	_
-5	SEDIMENTARES	sedimentar	ADJ	_	Gender=Masc|Number=Plur	2	amod	I=LITOLOGIA	_
+4	FÁCIES	fácies	PROPN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
+5	SEDIMENTARES	sedimentar	ADJ	_	Gender=Fem|Number=Plur	4	amod	I=LITOLOGIA	_
 
 # text = As seções-colunares levantadas para a Fm..
 # sent_id = 283-20150917-MONOGRAFIA_0-156
@@ -69049,7 +69049,7 @@
 5	conglomerados	conglomerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	3	xcomp	O	_
 6	clasto-sustentados	clastobsusustentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	5	xcomp	O	_
 7	(	(	PUNCT	_	_	8	punct	O	_
-8	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	6	nmod:appos	B=LITOLOGIA	_
+8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod:appos	B=LITOLOGIA	_
 9	G	g	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 10	)	)	PUNCT	_	_	8	punct	O	_
 11	,	,	PUNCT	_	_	12	punct	O	_
@@ -69268,7 +69268,7 @@
 12	conglomerados	conglomerar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	10	xcomp	O	_
 13	matriz-sustentado	matriz-sustentado	ADJ	_	Gender=Masc|Number=Sing	12	amod	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
-15	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	12	nmod:appos	B=LITOLOGIA	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	12	nmod:appos	B=LITOLOGIA	_
 16	Gmm	Gmm	PROPN	_	Gender=Masc|Number=Plur	15	nmod	O	_
 17	)	)	PUNCT	_	_	15	punct	O	_
 18	.	.	PUNCT	_	_	10	punct	O	_
@@ -69766,8 +69766,8 @@
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	associação	associação	NOUN	_	Gender=Fem|Number=Sing	29	obl	O	_
 5	de	de	ADP	_	_	6	case	O	_
-6	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	4	nmod	B=LITOLOGIA	_
-7	semelhante	semelhante	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
+6	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	4	nmod	B=LITOLOGIA	_
+7	semelhante	semelhante	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 8	,	,	PUNCT	_	_	4	punct	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	ausência	ausência	NOUN	_	Gender=Fem|Number=Sing	29	nsubj	O	_
@@ -69849,7 +69849,7 @@
 2	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	29	nsubj:pass	B=LITOLOGIA	_
 3	matriz-sustentados	matriz-sustentado	ADJ	_	Gender=Masc|Number=Plur	2	amod	O	_
 4	(	(	PUNCT	_	_	5	punct	O	_
-5	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	appos	B=LITOLOGIA	_
+5	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	appos	B=LITOLOGIA	_
 6	Gmm	Gmm	PROPN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 7	)	)	PUNCT	_	_	5	punct	O	_
 8	pertencentes	pertencente	ADJ	_	Gender=Masc|Number=Plur	2	amod	O	_
@@ -69894,7 +69894,7 @@
 # text = A fácies
 # sent_id = 283-20150917-MONOGRAFIA_0-189
 1	A	a	ADP	_	_	2	case	O	_
-2	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	0	root	B=LITOLOGIA	_
+2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	0	root	B=LITOLOGIA	_
 
 # text = St descrita para a porção superior da Formação Abaeté resulta da migração de dunas de cristas sinuosas, em fluxo unidirecional e sob regime de fluxo inferior (Miall, 2006)..
 # sent_id = 283-20150917-MONOGRAFIA_0-190
@@ -70008,7 +70008,7 @@
 8	eólicas	eólica	ADJ	_	Gender=Fem|Number=Plur	7	amod	O	_
 9	interdigitadas	interdigitar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	7	acl	O	_
 10	com	com	ADP	_	_	11	case	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	9	obl	B=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	9	obl	B=LITOLOGIA	_
 12	de	de	ADP	_	_	13	case	O	_
 13	leque	leque	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
 14	aluvial	aluvial	ADJ	_	Gender=Masc|Number=Sing	13	amod	O	_
@@ -70103,7 +70103,7 @@
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
 8	clastos	clasto	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	8	appos	B=LITOLOGIA	_
+10	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	8	appos	B=LITOLOGIA	_
 11	G	g	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
 12	)	)	PUNCT	_	_	10	punct	O	_
 13	..	..	PUNCT	_	_	1	punct	O	_
@@ -71383,7 +71383,7 @@
 3	ASSOCIAÇÃO	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 4	DE	de	ADP	_	_	5	case	O	_
 5	FÁCIES	fácies	NOUN	_	Gender=Fem|Number=Plur	3	nmod	B=LITOLOGIA	_
-6	SEDIMENTARES	sedimentar	ADJ	_	Gender=Masc|Number=Plur	5	amod	I=LITOLOGIA	_
+6	SEDIMENTARES	sedimentar	ADJ	_	Gender=Fem|Number=Plur	5	amod	I=LITOLOGIA	_
 
 # text = A Seção-colunar São José Romão encerra fácies únicas para a Fm. Quiricó..
 # sent_id = 283-20150917-MONOGRAFIA_0-253
@@ -71628,7 +71628,7 @@
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	associação	associação	NOUN	_	Gender=Fem|Number=Sing	18	obj	O	_
 22	de	de	ADP	_	_	23	case	O	_
-23	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	21	nmod	B=LITOLOGIA	_
+23	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	21	nmod	B=LITOLOGIA	_
 24	típica	típico	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	23	acl	O	_
 26	para	para	ADP	_	_	28	case	O	_
@@ -71661,7 +71661,7 @@
 7	fácies	fácies	NOUN	_	Gender=Fem|Number=Sing	13	nsubj	B=LITOLOGIA	_
 8	heterolítica	heterolítica	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 9	(	(	PUNCT	_	_	10	punct	O	_
-10	H	h	NOUN	_	Gender=Masc|Number=Sing	7	appos	O	_
+10	H	h	NOUN	_	Gender=Fem|Number=Sing	7	appos	O	_
 11	)	)	PUNCT	_	_	10	punct	O	_
 12	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	13	cop	O	_
 13	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	0	root	O	_
@@ -71873,7 +71873,7 @@
 8	dobras	dobra	NOUN	_	Gender=Fem|Number=Plur	4	conj	O	_
 9	convolutas	convoluto	ADJ	_	Gender=Fem|Number=Plur	8	amod	O	_
 10	(	(	PUNCT	_	_	11	punct	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	8	nmod:appos	B=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	8	nmod:appos	B=LITOLOGIA	_
 12	Sd	Sd	PROPN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 13	)	)	PUNCT	_	_	11	punct	O	_
 14-15	nestes	_	_	_	_	_	_	_	_
@@ -72105,7 +72105,7 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	associação	associação	NOUN	_	Gender=Fem|Number=Sing	36	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
 5	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	4	acl	O	_
 6	para	para	ADP	_	_	8	case	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -72119,8 +72119,8 @@
 14	Romão	Romão	PROPN	_	Number=Sing	11	flat:name	O	_
 15	,	,	PUNCT	_	_	18	punct	O	_
 16	com	com	ADP	_	_	17	case	O	_
-17	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	5	obl	B=LITOLOGIA	_
-18	provenientes	proveniente	ADJ	_	Gender=Masc|Number=Plur	17	amod	O	_
+17	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	5	obl	B=LITOLOGIA	_
+18	provenientes	proveniente	ADJ	_	Gender=Fem|Number=Plur	17	amod	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	decantação	decantação	NOUN	_	Gender=Fem|Number=Sing	18	obl	O	_
 21	,	,	PUNCT	_	_	18	punct	O	_
@@ -72536,7 +72536,7 @@
 16	haviam	haver	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	17	aux	O	_
 17	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	0	root	O	_
 18	anteriormente	anteriormente	ADV	_	_	19	advmod	O	_
-19	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	17	obj	B=LITOLOGIA	_
+19	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	17	obj	B=LITOLOGIA	_
 20	evaporíticas	evaporítica	ADJ	_	Gender=Fem|Number=Plur	19	amod	O	_
 21	associadas	associar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	19	acl	O	_
 22	a	a	ADP	_	_	24	case	O	_
@@ -72618,7 +72618,7 @@
 29	pelitos	pelito	NOUN	_	Gender=Masc|Number=Plur	23	nmod	B=LITOLOGIA	_
 30	expostos	expor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	29	acl	O	_
 31	(	(	PUNCT	_	_	32	punct	O	_
-32	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	29	nmod:appos	B=LITOLOGIA	_
+32	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	29	nmod:appos	B=LITOLOGIA	_
 33	Sr	Sr	PROPN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 34	e	e	CCONJ	_	_	35	cc	O	_
 35	St	St	PROPN	_	Gender=Masc|Number=Sing	33	conj	O	_
@@ -73171,7 +73171,7 @@
 8	distintas	distinto	ADJ	_	Gender=Fem|Number=Plur	9	amod	O	_
 9	associações	associação	NOUN	_	Gender=Fem|Number=Plur	5	obj	O	_
 10	de	de	ADP	_	_	11	case	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	9	nmod	B=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	9	nmod	B=LITOLOGIA	_
 12	,	,	PUNCT	_	_	14	punct	O	_
 13	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	inferior	inferior	ADJ	_	Gender=Fem|Number=Sing	9	amod	O	_
@@ -73279,7 +73279,7 @@
 3	ASSOCIAÇÃO	associação	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 4	DE	de	ADP	_	_	5	case	O	_
 5	FÁCIES	fácies	NOUN	_	Gender=Fem|Number=Plur	3	nmod	B=LITOLOGIA	_
-6	SEDIMENTARES	sedimentar	ADJ	_	Gender=Masc|Number=Plur	5	amod	I=LITOLOGIA	_
+6	SEDIMENTARES	sedimentar	ADJ	_	Gender=Fem|Number=Plur	5	amod	I=LITOLOGIA	_
 
 # text = O contato entre os sedimentos da Fm..
 # sent_id = 283-20150917-MONOGRAFIA_0-322
@@ -73367,7 +73367,7 @@
 6	de	de	ADP	_	_	7	case	O	_
 7	finos	fino	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 8	(	(	PUNCT	_	_	10	punct	O	_
-9	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	7	nmod:appos	B=LITOLOGIA	_
+9	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	7	nmod:appos	B=LITOLOGIA	_
 10	H	h	NOUN	_	Gender=Masc|Number=Plur	7	nmod	I=LITOLOGIA	_
 11	e	e	CCONJ	_	_	12	cc	O	_
 12	M	m	NOUN	_	Gender=Masc|Number=Plur	10	conj	O	_
@@ -73437,7 +73437,7 @@
 12	convolutas	convoluto	ADJ	_	Gender=Fem|Number=Plur	11	amod	O	_
 13	sinsedimentares	sinsedimentar	ADJ	_	Gender=Fem|Number=Plur	11	amod	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
-15	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	7	nmod:appos	B=LITOLOGIA	_
+15	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	7	nmod:appos	B=LITOLOGIA	_
 16	Sd	Sd	PROPN	_	Gender=Masc|Number=Sing	15	nmod	O	_
 17	)	)	PUNCT	_	_	15	punct	O	_
 18	,	,	PUNCT	_	_	5	punct	O	_
@@ -73699,7 +73699,7 @@
 9	grande	grande	ADJ	_	Gender=Masc|Number=Sing	10	amod	O	_
 10	porte	porte	NOUN	_	Gender=Masc|Number=Sing	4	nmod	O	_
 11	(	(	PUNCT	_	_	12	punct	O	_
-12	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	10	nmod:appos	B=LITOLOGIA	_
+12	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	10	nmod:appos	B=LITOLOGIA	_
 13	St	St	PROPN	_	Gender=Masc|Number=Sing	12	nmod	I=LITOLOGIA	_
 14	)	)	PUNCT	_	_	12	punct	O	_
 15	,	,	PUNCT	_	_	1	punct	O	_
@@ -73887,7 +73887,7 @@
 14	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	15	det	O	_
 15	associação	associação	NOUN	_	Gender=Fem|Number=Sing	12	obj	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	15	nmod	B=LITOLOGIA	_
+17	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	15	nmod	B=LITOLOGIA	_
 18	,	,	PUNCT	_	_	21	punct	O	_
 19	sendo	ser	AUX	_	VerbForm=Ger	21	cop	O	_
 20	recorrente	recorrente	ADJ	_	Gender=Fem|Number=Sing	21	amod	O	_
@@ -74107,8 +74107,8 @@
 172	porte	porte	NOUN	_	Gender=Masc|Number=Sing	168	nmod	O	_
 173	decimétrico	decimétrico	ADJ	_	Gender=Masc|Number=Sing	172	amod	O	_
 174	(	(	PUNCT	_	_	175	punct	O	_
-175	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	172	nmod:appos	B=LITOLOGIA	_
-176	St	St	PROPN	_	Gender=Masc|Number=Sing	175	parataxis	I=LITOLOGIA	_
+175	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	172	nmod:appos	B=LITOLOGIA	_
+176	St	St	PROPN	_	Gender=Fem|Number=Sing	175	parataxis	I=LITOLOGIA	_
 177	)	)	PUNCT	_	_	175	punct	O	_
 178	(	(	PUNCT	_	_	179	punct	O	_
 179	Ponto	ponto	NOUN	_	Gender=Masc|Number=Sing	175	parataxis	O	_
@@ -74130,7 +74130,7 @@
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
 9	associação	associação	NOUN	_	Gender=Fem|Number=Sing	6	obl	O	_
 10	de	de	ADP	_	_	11	case	O	_
-11	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	9	nmod	B=LITOLOGIA	_
+11	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	9	nmod	B=LITOLOGIA	_
 12	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	11	acl	O	_
 13	acima	acima	ADV	_	_	12	advmod	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
@@ -74216,7 +74216,7 @@
 18	descritos	descrever	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 19	argilitos	argilito	NOUN	_	Gender=Masc|Number=Plur	18	nsubj:pass	B=LITOLOGIA	_
 20	(	(	PUNCT	_	_	21	punct	O	_
-21	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	19	nmod:appos	B=LITOLOGIA	_
+21	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	19	nmod:appos	B=LITOLOGIA	_
 22	Fm	Fm	PROPN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 23	)	)	PUNCT	_	_	21	punct	O	_
 24	,	,	PUNCT	_	_	28	punct	O	_
@@ -74303,7 +74303,7 @@
 25	cruzada	cruzar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	24	acl	I=LITOLOGIA	_
 26	cavalgante	cavalgante	ADJ	_	Gender=Fem|Number=Sing	24	amod	O	_
 27	(	(	PUNCT	_	_	28	punct	O	_
-28	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	24	nmod:appos	B=LITOLOGIA	_
+28	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	24	nmod:appos	B=LITOLOGIA	_
 29	Sr	Sr	PROPN	_	Gender=Masc|Number=Sing	28	nmod	O	_
 30	)	)	PUNCT	_	_	28	punct	O	_
 31	,	,	PUNCT	_	_	25	punct	O	_
@@ -74587,7 +74587,7 @@
 1	Arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	9	nsubj:pass	B=LITOLOGIA	_
 2	deformados	deformar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	1	acl	O	_
 3	(	(	PUNCT	_	_	4	punct	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	1	nmod:appos	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	1	nmod:appos	B=LITOLOGIA	_
 5	Sd	Sd	PROPN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 6	)	)	PUNCT	_	_	4	punct	O	_
 7	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	9	aux:pass	O	_
@@ -74682,7 +74682,7 @@
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	associação	associação	NOUN	_	Gender=Fem|Number=Sing	16	nsubj	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
 5	descrita	descrever	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	2	acl	O	_
 6-7	na	_	_	_	_	_	_	_	_
 6	em	em	ADP	_	_	8	case	O	_
@@ -74748,7 +74748,7 @@
 1	Esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	2	det	O	_
 2	associação	associação	NOUN	_	Gender=Fem|Number=Sing	6	nsubj:pass	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	2	nmod	B=LITOLOGIA	_
+4	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	2	nmod	B=LITOLOGIA	_
 5	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	6	aux:pass	O	_
 6	interpretada	interpretar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 7	como	como	ADP	_	_	8	case	O	_
@@ -74941,7 +74941,7 @@
 18	pin	pin	X	_	_	17	flat:foreign	O	_
 19	stripe	stripe	X	_	_	17	flat:foreign	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
-21	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	14	conj	B=LITOLOGIA	_
+21	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	14	conj	B=LITOLOGIA	_
 
 # text = Sr e grande porte das estratificações cruzadas acanaladas (St) são características de dunas eólicas (Mountney, 2006)..
 # sent_id = 283-20150917-MONOGRAFIA_0-383
@@ -75118,7 +75118,7 @@
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
 6	associação	associação	NOUN	_	Gender=Fem|Number=Sing	4	obj	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	6	nmod	B=LITOLOGIA	_
+8	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	6	nmod	B=LITOLOGIA	_
 9	St	St	PROPN	_	Gender=Masc|Number=Sing	6	nmod	I=LITOLOGIA	_
 10	,	,	PUNCT	_	_	11	punct	O	_
 11	Sm	Sm	PROPN	_	Gender=Masc|Number=Sing	9	conj	O	_
@@ -75128,7 +75128,7 @@
 # text = A fácies
 # sent_id = 283-20150917-MONOGRAFIA_0-392
 1	A	a	ADP	_	_	2	case	O	_
-2	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	0	root	B=LITOLOGIA	_
+2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	0	root	B=LITOLOGIA	_
 
 # text = St, com estruturas pin stripe, lentes com gradação inversa e estratificações de grande porte, é típica de dunas eólicas (Mountney, 2006)..
 # sent_id = 283-20150917-MONOGRAFIA_0-393
@@ -75165,7 +75165,7 @@
 # text = A fácies
 # sent_id = 283-20150917-MONOGRAFIA_0-394
 1	A	a	ADP	_	_	2	case	O	_
-2	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	0	root	B=LITOLOGIA	_
+2	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	0	root	B=LITOLOGIA	_
 
 # text = Sm corresponde ao interduna e são menos frequentes e menos espessas do que no sistema deposicional eólico descrito abaixo da descontinuidade regional..
 # sent_id = 283-20150917-MONOGRAFIA_0-395
@@ -75831,7 +75831,7 @@
 2	grande	grande	ADJ	_	Gender=Fem|Number=Sing	3	amod	O	_
 3	variação	variação	NOUN	_	Gender=Fem|Number=Sing	13	nsubj	O	_
 4	de	de	ADP	_	_	5	case	O	_
-5	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	3	nmod	B=LITOLOGIA	_
+5	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	3	nmod	B=LITOLOGIA	_
 6	observada	observar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	3	acl	O	_
 7	para	para	ADP	_	_	9	case	O	_
 8	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	9	det	O	_
@@ -76808,7 +76808,7 @@
 3	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	4	det	O	_
 4	associações	associação	NOUN	_	Gender=Fem|Number=Plur	1	nmod	O	_
 5	de	de	ADP	_	_	6	case	O	_
-6	fácies	fácies	NOUN	_	Gender=Masc|Number=Plur	4	nmod	B=LITOLOGIA	_
+6	fácies	fácies	NOUN	_	Gender=Fem|Number=Plur	4	nmod	B=LITOLOGIA	_
 7	e	e	CCONJ	_	_	8	cc	O	_
 8	sistemas	sistema	NOUN	_	Gender=Masc|Number=Plur	6	conj	O	_
 9	deposicionais	deposicional	ADJ	_	Gender=Masc|Number=Plur	8	amod	O	_
diff --git a/notebooks/facies.py b/notebooks/facies.py
new file mode 100644
index 0000000..975b1c4
--- /dev/null
+++ b/notebooks/facies.py
@@ -0,0 +1,30 @@
+import os
+if not os.path.isfile("estrutura_ud.py"):
+    os.system("wget https://raw.githubusercontent.com/alvelvis/ACDC-UD/master/estrutura_ud.py")
+import estrutura_ud
+import copy
+
+file = "../Petroles_3/Petroles_3_golden.conllu"
+corpus = estrutura_ud.Corpus(recursivo=True)
+corpus.load(file)
+
+fix_feats = lambda feats: "|".join(sorted([x for x in feats.split("|") if not any(x.startswith(y) for y in ["Gender", "_"])] + ["Gender=Fem"]))
+
+for sentence in corpus.sentences.values():
+    for t, token in enumerate(sentence.tokens):
+        if token.word.lower() == "fácies":
+            token.lemma = "fácies"
+            token.feats = fix_feats(token.feats)
+            for _token in sentence.tokens:
+                if _token.dephead == token.id and _token.deprel not in ["case", "nmod", "acl:relcl", "punct", "conj", "advmod", "nmod:appos", "cc", "mark"]:
+                    _token.feats = fix_feats(_token.feats)
+                if "PronType=Rel" in _token.feats and _token.head_token.deprel == "acl:relcl" and _token.head_token.dephead == token.id:
+                    _token.feats = fix_feats(_token.feats)
+
+file_name = file.rsplit("/", 1)[1]
+corpus.save("regras.conllu")
+os.system("python3 ../../conllu-merge-resolver/cosmo.py {} regras.conllu '.*'".format(file))
+os.system("cp {} antes-{}".format(file, file_name))
+os.system("rm regras-correcao.zip; zip regras-correcao.zip antes-{} regras.conllu".format(file_name))
+os.remove("antes-{}".format(file_name))
+os.remove("regras.conllu")
\ No newline at end of file
diff --git a/notebooks/facies_sedimentar.py b/notebooks/facies_sedimentar.py
deleted file mode 100644
index e619097..0000000
--- a/notebooks/facies_sedimentar.py
+++ /dev/null
@@ -1,51 +0,0 @@
-import os
-if not os.path.isfile("estrutura_ud.py"):
-    os.system("wget https://raw.githubusercontent.com/alvelvis/ACDC-UD/master/estrutura_ud.py")
-import estrutura_ud
-import copy
-
-file = "../Petroles_3/Petroles_3_golden.conllu"
-corpus = estrutura_ud.Corpus(recursivo=True)
-corpus.load(file)
-
-allowed_acl_head = ["PROPN", "NOUN", "SYM", "PRON"]
-
-for sentence in corpus.sentences.values():
-    for token in sentence.tokens:
-        if (token.deprel == "acl" and 
-        'gender' in token.__dict__ and 
-        token.__dict__.get("gender", None) != token.head_token.__dict__.get("gender", None)):
-            gender = token.__dict__.get("gender", "none")
-            number = token.__dict__.get("number", "none")
-            head = copy.copy(token.head_token)
-            previous = copy.copy(token.previous_token)
-            found = False
-            while head.dephead != "0":
-                head = copy.copy(head.head_token)
-                if (gender == head.__dict__.get("gender", None) and 
-                number == head.__dict__.get("number", None) and 
-                head.upos in allowed_acl_head):
-                    if found:
-                        token.misc = token.misc + " ou {} ({})".format(head.word, head.id)
-                    else:
-                        token.dephead = head.id
-                        found = True
-            if not found:
-                while previous.id != "0":
-                    previous = copy.copy(previous.previous_token)
-                    if (gender == previous.__dict__.get("gender", None) and 
-                    number == previous.__dict__.get("number", None) and
-                    previous.upos in allowed_acl_head):
-                        if found:
-                            token.misc = token.misc + " ou {} ({})".format(previous.word, previous.id)
-                        else:
-                            token.dephead = previous.id
-                            found = True
-
-file_name = file.rsplit("/", 1)[1]
-corpus.save("regras.conllu")
-os.system("python3 ../../conllu-merge-resolver/cosmo.py {} regras.conllu '.*'".format(file))
-os.system("cp {} antes-{}".format(file, file_name))
-os.system("rm regras-correcao.zip; zip regras-correcao.zip antes-{} regras.conllu".format(file_name))
-os.remove("antes-{}".format(file_name))
-os.remove("regras.conllu")
\ No newline at end of file
