commit 1ef93eb8496c7ffb4d3fc526dcb7754e79d876b9
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed Oct 20 17:28:02 2021 -0300

    Correções de flat:name com semas estranhos

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index a5368d0..d5907a3 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -279,11 +279,11 @@
 3	:	:	PUNCT	_	_	4	punct	O	_
 4	Localização	Localização	PROPN	_	Gender=Fem|Number=Sing	1	parataxis	O	_
 5-6	da	_	_	_	_	_	_	_	_
-5	de	de	ADP	_	_	4	flat:name	O	_
-6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	flat:name	O	_
-7	Bacia	Bacia	PROPN	_	Number=Sing	4	flat:name	B=BACIA	_
-8	de	de	ADP	_	_	4	flat:name	O	_
-9	Pelotas	Pelotas	PROPN	_	Number=Sing	4	flat:name	B=BACIA	_
+5	de	de	ADP	_	_	7	case	O	_
+6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
+7	Bacia	Bacia	PROPN	_	Number=Sing	4	nmod	B=BACIA	_
+8	de	de	ADP	_	_	7	flat:name	I=BACIA	_
+9	Pelotas	Pelotas	PROPN	_	Number=Sing	7	flat:name	I=BACIA	_
 
 # text = 2. - OBJETIVOS
 # sent_id = 247-20140910-MONOGRAFIA_0-8
@@ -1774,42 +1774,42 @@
 45	estratigráficos	estratigráfico	ADJ	_	Gender=Masc|Number=Plur	44	amod	O	_
 46	mapeados	mapear	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	44	acl	O	_
 47	(	(	PUNCT	_	_	48	punct	O	_
-48	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	44	appos	O	_
+48	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	44	appos	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 49-50	do	_	_	_	_	_	_	_	_
 49	de	de	ADP	_	_	48	flat:name	O	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	48	flat:name	O	_
-51	Embasamento	Embasamento	PROPN	_	Number=Sing	48	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+51	Embasamento	Embasamento	PROPN	_	Number=Sing	48	flat:name	O	_
 52	,	,	PUNCT	_	_	48	flat:name	O	_
-53	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	O	_
+53	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 54-55	do	_	_	_	_	_	_	_	_
 54	de	de	ADP	_	_	53	flat:name	O	_
 55	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	53	flat:name	O	_
 56	Rifte	Rifte	PROPN	_	Number=Sing	53	flat:name	O	_
 57	,	,	PUNCT	_	_	58	punct	O	_
-58	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	O	_
+58	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 59-60	do	_	_	_	_	_	_	_	_
 59	de	de	ADP	_	_	58	flat:name	O	_
 60	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	58	flat:name	O	_
-61	Albiano	Albiano	PROPN	_	Number=Sing	58	flat:name	B=GEOCRONOLOGIA	_
+61	Albiano	Albiano	PROPN	_	Number=Sing	58	flat:name	O	_
 62	,	,	PUNCT	_	_	63	punct	O	_
-63	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	O	_
+63	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 64-65	do	_	_	_	_	_	_	_	_
 64	de	de	ADP	_	_	63	flat:name	O	_
 65	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	63	flat:name	O	_
-66	Cretáceo	Cretáceo	PROPN	_	Number=Sing	63	flat:name	B=GEOCRONOLOGIA	_
-67	Superior	Superior	PROPN	_	Number=Sing	63	flat:name	I=GEOCRONOLOGIA	_
+66	Cretáceo	Cretáceo	PROPN	_	Number=Sing	63	flat:name	O	_
+67	Superior	Superior	PROPN	_	Number=Sing	63	flat:name	O	_
 68	,	,	PUNCT	_	_	69	punct	O	_
-69	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	O	_
+69	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 70-71	do	_	_	_	_	_	_	_	_
 70	de	de	ADP	_	_	69	flat:name	O	_
 71	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	69	flat:name	O	_
-72	Eoceno	Eoceno	PROPN	_	Number=Sing	69	flat:name	B=GEOCRONOLOGIA	_
+72	Eoceno	Eoceno	PROPN	_	Number=Sing	69	flat:name	O	_
 73	e	e	CCONJ	_	_	74	cc	O	_
-74	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	O	_
+74	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	48	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 75-76	do	_	_	_	_	_	_	_	_
 75	de	de	ADP	_	_	74	flat:name	O	_
 76	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	74	flat:name	O	_
-77	Mioceno	Mioceno	PROPN	_	Number=Sing	74	flat:name	B=GEOCRONOLOGIA	_
+77	Mioceno	Mioceno	PROPN	_	Number=Sing	74	flat:name	O	_
 78	)	)	PUNCT	_	_	48	punct	O	_
 79	.	.	PUNCT	_	_	15	punct	O	_
 
@@ -1931,30 +1931,30 @@
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	flat:name	O	_
 12	Rifte	Rifte	PROPN	_	Number=Sing	9	flat:name	O	_
 13	,	,	PUNCT	_	_	14	punct	O	_
-14	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+14	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=GEOCRONOLOGIA|B=UNIDADE_LITOESTRATIGRÁFICA	_
 15-16	do	_	_	_	_	_	_	_	_
 15	de	de	ADP	_	_	14	flat:name	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	flat:name	O	_
-17	Albiano	Albiano	PROPN	_	Number=Sing	14	flat:name	B=GEOCRONOLOGIA	_
+17	Albiano	Albiano	PROPN	_	Number=Sing	14	flat:name	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
-19	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+19	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=GEOCRONOLOGIA|B=UNIDADE_LITOESTRATIGRÁFICA	_
 20-21	do	_	_	_	_	_	_	_	_
 20	de	de	ADP	_	_	19	flat:name	O	_
 21	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	flat:name	O	_
-22	Cretáceo	Cretáceo	PROPN	_	Number=Sing	19	flat:name	B=GEOCRONOLOGIA	_
-23	Superior	Superior	PROPN	_	Number=Sing	19	flat:name	I=GEOCRONOLOGIA	_
+22	Cretáceo	Cretáceo	PROPN	_	Number=Sing	19	flat:name	O	_
+23	Superior	Superior	PROPN	_	Number=Sing	19	flat:name	O	_
 24	,	,	PUNCT	_	_	25	punct	O	_
-25	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+25	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=GEOCRONOLOGIA|B=UNIDADE_LITOESTRATIGRÁFICA	_
 26-27	do	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	25	flat:name	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	flat:name	O	_
-28	Eoceno	Eoceno	PROPN	_	Number=Sing	25	flat:name	B=GEOCRONOLOGIA	_
+28	Eoceno	Eoceno	PROPN	_	Number=Sing	25	flat:name	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
-30	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+30	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	conj	B=GEOCRONOLOGIA|B=UNIDADE_LITOESTRATIGRÁFICA	_
 31-32	do	_	_	_	_	_	_	_	_
 31	de	de	ADP	_	_	30	flat:name	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	flat:name	O	_
-33	Mioceno	Mioceno	PROPN	_	Number=Sing	30	flat:name	B=GEOCRONOLOGIA	_
+33	Mioceno	Mioceno	PROPN	_	Number=Sing	30	flat:name	O	_
 34	e	e	CCONJ	_	_	44	cc	O	_
 35	,	,	PUNCT	_	_	40	punct	O	_
 36	a	a	ADP	_	_	40	case	O	MWEPOS=ADP
@@ -2276,8 +2276,8 @@
 37	envolvendo	envolver	VERB	_	VerbForm=Ger	22	advcl	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	det	O	_
 39	Cráton	Cráton	PROPN	_	Gender=Masc|Number=Sing	37	obj	B=UNIDADE_LITOESTRATIGRÁFICA	_
-40	São	São	PROPN	_	Number=Sing	39	flat:name	B=BACIA	_
-41	Francisco	Francisco	PROPN	_	Number=Sing	39	flat:name	I=BACIA	_
+40	São	São	PROPN	_	Number=Sing	39	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+41	Francisco	Francisco	PROPN	_	Number=Sing	39	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 42	(	(	PUNCT	_	_	43	punct	O	_
 43	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	22	appos	O	_
 44	5	5	NUM	_	NumType=Card	43	nummod	O	_
@@ -2318,10 +2318,10 @@
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
 29	Cráton	Cráton	PROPN	_	Gender=Masc|Number=Sing	25	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 30-31	do	_	_	_	_	_	_	_	_
-30	de	de	ADP	_	_	29	flat:name	O	_
-31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	flat:name	O	_
-32	São	São	PROPN	_	Number=Sing	29	flat:name	B=BACIA	_
-33	Francisco	Francisco	PROPN	_	Number=Sing	29	flat:name	I=BACIA	_
+30	de	de	ADP	_	_	29	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+32	São	São	PROPN	_	Number=Sing	29	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+33	Francisco	Francisco	PROPN	_	Number=Sing	29	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 34	(	(	PUNCT	_	_	35	punct	O	_
 35	Barbosa	Barbosa	PROPN	_	Gender=Masc|Number=Sing	2	parataxis	O	_
 36	1966	1966	PROPN	_	Number=Sing	35	flat:name	O	_
@@ -2857,8 +2857,8 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-89
 1	Devoniana	Devoniana	PROPN	_	Gender=Fem|Number=Sing	0	root	O	_
 2	(	(	PUNCT	_	_	3	punct	O	_
-3	Supersequência	Supersequência	PROPN	_	Gender=Fem|Number=Sing	1	appos	O	_
-4	Paraná	Paraná	PROPN	_	Number=Sing	3	flat:name	B=BACIA	_
+3	Supersequência	Supersequência	PROPN	_	Gender=Fem|Number=Sing	1	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
+4	Paraná	Paraná	PROPN	_	Number=Sing	3	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 5	)	)	PUNCT	_	_	3	punct	O	_
 6	:	:	PUNCT	_	_	7	punct	O	_
 7	formada	formar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	1	acl	O	_
@@ -2866,7 +2866,7 @@
 8	por	por	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
 10	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	7	obl:agent	B=UNIDADE_LITOESTRATIGRÁFICA	_
-11	Paraná	Paraná	PROPN	_	Number=Sing	10	flat:name	B=BACIA	_
+11	Paraná	Paraná	PROPN	_	Number=Sing	10	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 12	,	,	PUNCT	_	_	13	punct	O	_
 13	apresentando	apresentar	VERB	_	VerbForm=Ger	7	advcl	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -2990,7 +2990,7 @@
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
 8	formações	formação	NOUN	_	Gender=Fem|Number=Plur	5	obl:agent	O	_
 9	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	8	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-10	Bonito	Bonito	PROPN	_	Number=Sing	9	flat:name	B=CAMPO	_
+10	Bonito	Bonito	PROPN	_	Number=Sing	9	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 11	e	e	CCONJ	_	_	12	cc	O	_
 12	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 13	(	(	PUNCT	_	_	16	punct	O	_
@@ -3009,7 +3009,7 @@
 24	a	a	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	23	obl:arg	B=UNIDADE_LITOESTRATIGRÁFICA	_
-27	Tatuí	Tatuí	PROPN	_	Number=Sing	26	flat:name	B=CAMPO	_
+27	Tatuí	Tatuí	PROPN	_	Number=Sing	26	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 28	)	)	PUNCT	_	_	23	punct	O	_
 29	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -3566,7 +3566,7 @@
 12	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	13	det	O	_
 13	formações	formação	NOUN	_	Gender=Fem|Number=Plur	10	obl:agent	O	_
 14	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-15	Bonito	Bonito	PROPN	_	Number=Sing	14	flat:name	B=CAMPO	_
+15	Bonito	Bonito	PROPN	_	Number=Sing	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 16	,	,	PUNCT	_	_	17	punct	O	_
 17	Palermo	Palermo	PROPN	_	Gender=Masc|Number=Sing	14	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 18	e	e	CCONJ	_	_	19	cc	O	_
@@ -3660,7 +3660,7 @@
 15	de	de	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	13	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-18	Serra	Serra	PROPN	_	Number=Sing	17	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+18	Serra	Serra	PROPN	_	Number=Sing	17	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 19	Geral	Geral	PROPN	_	Number=Sing	17	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 20	,	,	PUNCT	_	_	9	punct	O	_
 21	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	23	nsubj:pass	O	_
@@ -3835,11 +3835,11 @@
 3-4	pela	_	_	_	_	_	_	_	_
 3	por	por	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	Sequência	Sequência	PROPN	_	Gender=Fem|Number=Sing	2	obl:agent	O	_
+5	Sequência	Sequência	PROPN	_	Gender=Fem|Number=Sing	2	obl:agent	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 6-7	do	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	5	flat:name	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	flat:name	O	_
-8	Aptiano	Aptiano	PROPN	_	Number=Sing	5	flat:name	B=GEOCRONOLOGIA	_
+8	Aptiano	Aptiano	PROPN	_	Number=Sing	5	flat:name	O	_
 9	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	10	nsubj	O	_
 10	equivale	equivaler	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	acl:relcl	O	_
 11-12	à	_	_	_	_	_	_	_	_
@@ -3939,13 +3939,13 @@
 16	de	de	ADP	_	_	15	flat:name	O	_
 17	Torres	Torres	PROPN	_	Number=Sing	15	flat:name	O	_
 18	(	(	PUNCT	_	_	19	punct	O	_
-19	Norte	Norte	PROPN	_	Gender=Masc|Number=Sing	15	appos	O	_
+19	Norte	norte	NOUN	_	Gender=Masc|Number=Sing	15	appos	O	_
 20-21	da	_	_	_	_	_	_	_	_
-20	de	de	ADP	_	_	19	flat:name	O	_
-21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	flat:name	O	_
+20	de	de	ADP	_	_	22	case	O	_
+21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	22	det	O	_
 22	Bacia	Bacia	PROPN	_	Number=Sing	19	nmod	B=BACIA	_
-23	de	de	ADP	_	_	19	flat:name	O	_
-24	Pelotas	Pelotas	PROPN	_	Number=Sing	19	flat:name	B=BACIA	_
+23	de	de	ADP	_	_	22	flat:name	I=BACIA	_
+24	Pelotas	Pelotas	PROPN	_	Number=Sing	22	flat:name	I=BACIA	_
 25	)	)	PUNCT	_	_	19	punct	O	_
 26	ainda	ainda	ADV	_	_	27	advmod	O	_
 27	sofria	sofrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	12	ccomp	O	_
@@ -3956,9 +3956,9 @@
 32	,	,	PUNCT	_	_	27	punct	O	_
 33	enquanto	enquanto	SCONJ	_	_	42	mark	O	_
 34	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	35	det	O	_
-35	Sub-bacia	Sub-bacia	PROPN	_	Gender=Fem|Number=Sing	42	nsubj	O	_
-36	de	de	ADP	_	_	35	flat:name	O	_
-37	Pelotas	Pelotas	PROPN	_	Number=Sing	35	flat:name	B=BACIA	_
+35	Sub-bacia	Sub-bacia	PROPN	_	Gender=Fem|Number=Sing	42	nsubj	B=BACIA	_
+36	de	de	ADP	_	_	35	flat:name	I=BACIA	_
+37	Pelotas	Pelotas	PROPN	_	Number=Sing	35	flat:name	I=BACIA	_
 38	(	(	PUNCT	_	_	39	punct	O	_
 39	Sul	Sul	PROPN	_	Gender=Masc|Number=Sing	35	appos	O	_
 40	)	)	PUNCT	_	_	39	punct	O	_
@@ -4630,7 +4630,7 @@
 16	com	com	ADP	_	_	18	case	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
 18	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	15	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-19	Serra	Serra	PROPN	_	Number=Sing	18	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+19	Serra	Serra	PROPN	_	Number=Sing	18	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 20	Geral	Geral	PROPN	_	Number=Sing	18	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 21	devido	dever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	25	case	O	MWEPOS=ADP
 22-23	à	_	_	_	_	_	_	_	_
@@ -5021,11 +5021,11 @@
 4-5	À	_	_	_	_	_	_	_	_
 4	A	a	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
-6	BACIA	BACIA	PROPN	_	Gender=Fem|Number=Sing	3	obl	O	_
+6	BACIA	BACIA	PROPN	_	Gender=Fem|Number=Sing	3	obl	B=BACIA	_
 7-8	DO	_	_	_	_	_	_	_	_
-7	DE	de	ADP	_	_	6	flat:name	O	_
-8	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	flat:name	O	_
-9	PARANÁ	PARANÁ	PROPN	_	Gender=Masc|Number=Sing	6	flat:name	B=BACIA	_
+7	DE	de	ADP	_	_	6	flat:name	I=BACIA	_
+8	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	flat:name	I=BACIA	_
+9	PARANÁ	PARANÁ	PROPN	_	Gender=Masc|Number=Sing	6	flat:name	I=BACIA	_
 
 # text = A ligação entre os arcos de Ponta Grossa e do Rio Grande é representado pelo “Sinclinal” de Torres (Figura 10), com orientação NW-SE e caimento suave para NW.
 # sent_id = 247-20140910-MONOGRAFIA_0-149
@@ -8397,15 +8397,15 @@
 9-10	do	_	_	_	_	_	_	_	_
 9	de	de	ADP	_	_	11	case	O	_
 10	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	det	O	_
-11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	8	obl	O	_
+11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	8	obl	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	11	flat:name	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	O	_
-14	Embasamento	Embasamento	PROPN	_	Number=Sing	11	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+14	Embasamento	Embasamento	PROPN	_	Number=Sing	11	flat:name	O	_
 15-16	ao	_	_	_	_	_	_	_	_
 15	a	a	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
-17	Topo	Topo	PROPN	_	Number=Sing	8	obl	O	_
+17	Topo	Topo	PROPN	_	Number=Sing	8	obl	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 18-19	do	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	17	flat:name	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	flat:name	O	_
@@ -8935,13 +8935,13 @@
 8	horizontes	horizonte	NOUN	_	Gender=Masc|Number=Plur	4	obl	O	_
 9	interpretados	interpretar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	8	acl	O	_
 10	(	(	PUNCT	_	_	11	punct	O	_
-11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	8	appos	O	_
+11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	8	appos	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	11	flat:name	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	O	_
-14	Embasamento	Embasamento	PROPN	_	Number=Sing	11	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+14	Embasamento	Embasamento	PROPN	_	Number=Sing	11	flat:name	O	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Topo	Topo	PROPN	_	Number=Sing	11	conj	O	_
+16	Topo	Topo	PROPN	_	Number=Sing	11	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 17-18	do	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	16	flat:name	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
@@ -8952,35 +8952,35 @@
 23	e	e	CCONJ	_	_	24	cc	O	_
 24	31B	31B	PROPN	_	Number=Sing	11	conj	O	_
 25	;	;	PUNCT	_	_	26	punct	O	_
-26	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	11	conj	O	_
+26	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	11	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 27-28	do	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	26	flat:name	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	flat:name	O	_
-29	Albiano	Albiano	PROPN	_	Number=Sing	26	flat:name	B=GEOCRONOLOGIA	_
+29	Albiano	Albiano	PROPN	_	Number=Sing	26	flat:name	O	_
 30	e	e	CCONJ	_	_	31	cc	O	_
-31	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	26	conj	O	_
+31	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	26	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 32-33	do	_	_	_	_	_	_	_	_
 32	de	de	ADP	_	_	31	flat:name	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	31	flat:name	O	_
-34	Cretáceo	Cretáceo	PROPN	_	Number=Sing	31	flat:name	B=GEOCRONOLOGIA	_
-35	Superior	Superior	PROPN	_	Number=Sing	31	flat:name	I=GEOCRONOLOGIA	_
+34	Cretáceo	Cretáceo	PROPN	_	Number=Sing	31	flat:name	O	_
+35	Superior	Superior	PROPN	_	Number=Sing	31	flat:name	O	_
 36	,	,	PUNCT	_	_	37	punct	O	_
 37	Figuras	figuras	NOUN	_	Gender=Fem|Number=Sing	11	conj	O	_
 38	32A	32A	NUM	_	NumType=Card	37	nummod	O	_
 39	e	e	CCONJ	_	_	40	cc	O	_
 40	32B	32B	PROPN	_	Number=Sing	11	conj	O	_
 41	;	;	PUNCT	_	_	42	punct	O	_
-42	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	11	conj	O	_
+42	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	11	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 43-44	do	_	_	_	_	_	_	_	_
 43	de	de	ADP	_	_	42	flat:name	O	_
 44	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	42	flat:name	O	_
-45	Eoceno	Eoceno	PROPN	_	Number=Sing	42	flat:name	B=GEOCRONOLOGIA	_
+45	Eoceno	Eoceno	PROPN	_	Number=Sing	42	flat:name	O	_
 46	e	e	CCONJ	_	_	47	cc	O	_
-47	Topo	Topo	PROPN	_	Number=Sing	42	conj	O	_
+47	Topo	Topo	PROPN	_	Number=Sing	42	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 48-49	do	_	_	_	_	_	_	_	_
 48	de	de	ADP	_	_	47	flat:name	O	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	47	flat:name	O	_
-50	Mioceno	Mioceno	PROPN	_	Number=Sing	47	flat:name	B=GEOCRONOLOGIA	_
+50	Mioceno	Mioceno	PROPN	_	Number=Sing	47	flat:name	O	_
 51	,	,	PUNCT	_	_	52	punct	O	_
 52	Figuras	figuras	NOUN	_	Gender=Fem|Number=Sing	42	nmod:appos	O	_
 53	33A	33A	NUM	_	NumType=Card	52	nummod	O	_
@@ -9047,15 +9047,15 @@
 8-9	ao	_	_	_	_	_	_	_	_
 8	a	a	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	obl	O	_
+10	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	7	obl	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 11	de	de	ADP	_	_	10	flat:name	O	_
-12	Embasamento	Embasamento	PROPN	_	Number=Sing	10	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+12	Embasamento	Embasamento	PROPN	_	Number=Sing	10	flat:name	O	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	27	parataxis	O	_
 15	31A	31A	NUM	_	NumType=Card	14	nummod	O	_
 16	)	)	PUNCT	_	_	14	punct	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
-18	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	10	conj	O	_
+18	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	10	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	18	flat:name	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	flat:name	O	_
@@ -9243,7 +9243,7 @@
 34-35	ao	_	_	_	_	_	_	_	_
 34	a	a	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
-36	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	33	obl	O	_
+36	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	33	obl	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 37-38	do	_	_	_	_	_	_	_	_
 37	de	de	ADP	_	_	36	flat:name	O	_
 38	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	flat:name	O	_
@@ -9253,22 +9253,22 @@
 42	31	31	NUM	_	NumType=Card	41	nummod	O	_
 43	)	)	PUNCT	_	_	41	punct	O	_
 44	,	,	PUNCT	_	_	45	punct	O	_
-45	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	36	conj	O	_
+45	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	36	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 46-47	do	_	_	_	_	_	_	_	_
 46	de	de	ADP	_	_	45	flat:name	O	_
 47	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	45	flat:name	O	_
-48	Cretáceo	Cretáceo	PROPN	_	Number=Sing	45	flat:name	B=GEOCRONOLOGIA	_
-49	Superior	Superior	PROPN	_	Number=Sing	45	flat:name	I=GEOCRONOLOGIA	_
+48	Cretáceo	Cretáceo	PROPN	_	Number=Sing	45	flat:name	O	_
+49	Superior	Superior	PROPN	_	Number=Sing	45	flat:name	O	_
 50	(	(	PUNCT	_	_	51	punct	O	_
 51	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	6	parataxis	O	_
 52	32	32	NUM	_	NumType=Card	51	nummod	O	_
 53	)	)	PUNCT	_	_	51	punct	O	_
 54	e	e	CCONJ	_	_	55	cc	O	_
-55	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	36	conj	O	_
+55	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	36	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 56-57	do	_	_	_	_	_	_	_	_
 56	de	de	ADP	_	_	55	flat:name	O	_
 57	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	55	flat:name	O	_
-58	Eoceno	Eoceno	PROPN	_	Number=Sing	55	flat:name	B=GEOCRONOLOGIA	_
+58	Eoceno	Eoceno	PROPN	_	Number=Sing	55	flat:name	O	_
 59	(	(	PUNCT	_	_	60	punct	O	_
 60	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	6	parataxis	O	_
 61	33	33	NUM	_	NumType=Card	60	nummod	O	_
@@ -9479,11 +9479,11 @@
 28-29	do	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	det	O	_
-30	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	27	nmod	O	_
+30	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	27	nmod	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 31-32	do	_	_	_	_	_	_	_	_
 31	de	de	ADP	_	_	30	flat:name	O	_
 32	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	30	flat:name	O	_
-33	Embasamento	Embasamento	PROPN	_	Number=Sing	30	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+33	Embasamento	Embasamento	PROPN	_	Number=Sing	30	flat:name	O	_
 34	e	e	CCONJ	_	_	36	cc	O	_
 35	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	36	nsubj	O	_
 36	afetou	afetar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	24	conj	O	_
@@ -9492,7 +9492,7 @@
 39-40	do	_	_	_	_	_	_	_	_
 39	de	de	ADP	_	_	41	case	O	_
 40	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	det	O	_
-41	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	38	nmod	O	_
+41	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	38	nmod	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 42-43	do	_	_	_	_	_	_	_	_
 42	de	de	ADP	_	_	41	flat:name	O	_
 43	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	41	flat:name	O	_
@@ -9504,11 +9504,11 @@
 49	casos	caso	NOUN	_	Gender=Masc|Number=Plur	36	obl	O	_
 50	isolados	isolar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	49	acl	O	_
 51	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	52	det	O	_
-52	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	50	obj	O	_
+52	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	50	obj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 53-54	do	_	_	_	_	_	_	_	_
 53	de	de	ADP	_	_	52	flat:name	O	_
 54	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	52	flat:name	O	_
-55	Albiano	Albiano	PROPN	_	Number=Sing	52	flat:name	B=GEOCRONOLOGIA	_
+55	Albiano	Albiano	PROPN	_	Number=Sing	52	flat:name	O	_
 56	.	.	PUNCT	_	_	17	punct	O	_
 
 # text = Com o alinhamento das seções sísmicas (228-0322, 231-0498 e 231-0486 Anexo 15) pelo Arco de Torres foi visualizada uma sequência de grábens apresentando orientação NW-SE alinhados.
@@ -9583,13 +9583,13 @@
 8-9	do	_	_	_	_	_	_	_	_
 8	de	de	ADP	_	_	10	case	O	_
 9	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	det	O	_
-10	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+10	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	nmod	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 11-12	do	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	10	flat:name	O	_
 12	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	10	flat:name	O	_
-13	Embasamento	Embasamento	PROPN	_	Number=Sing	10	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+13	Embasamento	Embasamento	PROPN	_	Number=Sing	10	flat:name	O	_
 14	e	e	CCONJ	_	_	15	cc	O	_
-15	Topo	Topo	PROPN	_	Number=Sing	10	conj	O	_
+15	Topo	Topo	PROPN	_	Number=Sing	10	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 16-17	do	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	15	flat:name	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	flat:name	O	_
@@ -9997,23 +9997,23 @@
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	3	obl	O	_
+6	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	3	obl	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	6	flat:name	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	flat:name	O	_
-9	Mioceno	Mioceno	PROPN	_	Number=Sing	6	flat:name	B=GEOCRONOLOGIA	_
+9	Mioceno	Mioceno	PROPN	_	Number=Sing	6	flat:name	O	_
 10	,	,	PUNCT	_	_	11	punct	O	_
-11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	conj	O	_
+11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	11	flat:name	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	O	_
-14	Cretáceo	Cretáceo	PROPN	_	Number=Sing	11	flat:name	B=GEOCRONOLOGIA	_
+14	Cretáceo	Cretáceo	PROPN	_	Number=Sing	11	flat:name	O	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Topo	Topo	PROPN	_	Number=Sing	11	conj	O	_
+16	Topo	Topo	PROPN	_	Number=Sing	11	conj	B=UNIDADE_LITOESTRATIGRÁFICA|B=GEOCRONOLOGIA	_
 17-18	do	_	_	_	_	_	_	_	_
 17	de	de	ADP	_	_	16	flat:name	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
-19	Eoceno	Eoceno	PROPN	_	Number=Sing	16	flat:name	B=GEOCRONOLOGIA	_
+19	Eoceno	Eoceno	PROPN	_	Number=Sing	16	flat:name	O	_
 20-21	encontram-se	_	_	_	_	_	_	_	_
 20	encontram	encontrar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 21	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	20	expl	O	_
@@ -17718,8 +17718,8 @@
 4-5	ao	_	_	_	_	_	_	_	_
 4	a	a	ADP	_	_	6	case	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	det	O	_
-6	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	3	obl	O	_
-7	Alagoas	Alagoas	PROPN	_	Number=Sing	6	flat:name	B=BACIA	_
+6	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	3	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
+7	Alagoas	Alagoas	PROPN	_	Number=Sing	6	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 8-9	nesta	_	_	_	_	_	_	_	_
 8	em	em	ADP	_	_	10	case	O	_
 9	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	10	det	O	_
@@ -17729,7 +17729,7 @@
 12	a	a	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	11	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-15	Santana	Santana	PROPN	_	Number=Sing	14	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+15	Santana	Santana	PROPN	_	Number=Sing	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 16	,	,	PUNCT	_	_	11	punct	O	_
 17	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	18	nsubj	O	_
 18	contém	conter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	acl:relcl	O	_
@@ -18308,7 +18308,7 @@
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	18	flat:name	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	flat:name	O	_
-21	Araripe	Araripe	PROPN	_	Number=Sing	18	flat:name	B=BACIA	_
+21	Araripe	Araripe	PROPN	_	Number=Sing	18	flat:name	O	_
 22	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = Figura 5.
@@ -18476,7 +18476,7 @@
 2	.	.	PUNCT	_	_	3	punct	O	_
 3	Trabalhos	Trabalhos	PROPN	_	Gender=Masc|Number=Plur	0	root	O	_
 4	de	de	ADP	_	_	3	flat:name	O	_
-5	Campo	Campo	PROPN	_	Number=Sing	3	flat:name	B=BACIA	_
+5	Campo	Campo	PROPN	_	Number=Sing	3	flat:name	O	_
 
 # text = Esta etapa consistiu no tratamento e análise de dados de superfície da Bacia do Araripe e a delimitação das áreas a serem investigadas.
 # sent_id = 284-20150917-MONOGRAFIA_0-38
@@ -20841,9 +20841,9 @@
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	19	conj	B=BACIA	_
 40-41	do	_	_	_	_	_	_	_	_
-40	de	de	ADP	_	_	39	flat:name	O	_
-41	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	flat:name	O	_
-42	Jatobá	Jatobá	PROPN	_	Number=Sing	39	flat:name	B=BACIA|B=CAMPO	_
+40	de	de	ADP	_	_	39	flat:name	I=BACIA	_
+41	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	flat:name	I=BACIA	_
+42	Jatobá	Jatobá	PROPN	_	Number=Sing	39	flat:name	I=BACIA	_
 43	(	(	PUNCT	_	_	44	punct	O	_
 44	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	39	appos	B=CAMPO	_
 45	Negra	Negra	PROPN	_	Number=Sing	44	flat:name	O	_
@@ -20978,9 +20978,9 @@
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	7	nmod	B=BACIA	_
 12-13	do	_	_	_	_	_	_	_	_
-12	de	de	ADP	_	_	11	flat:name	O	_
-13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	O	_
-14	Jatobá	Jatobá	PROPN	_	Number=Sing	11	flat:name	B=BACIA|B=CAMPO	_
+12	de	de	ADP	_	_	11	flat:name	I=BACIA	_
+13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	I=BACIA	_
+14	Jatobá	Jatobá	PROPN	_	Number=Sing	11	flat:name	I=BACIA	_
 15	,	,	PUNCT	_	_	17	punct	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	idade	idade	NOUN	_	Gender=Fem|Number=Sing	7	nmod:appos	O	_
@@ -21067,9 +21067,9 @@
 26-27	do	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
-28	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	24	nmod	O	_
-29	Dom	Dom	PROPN	_	Number=Sing	28	flat:name	B=CAMPO	_
-30	João	João	PROPN	_	Number=Sing	28	flat:name	I=CAMPO	_
+28	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	24	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+29	Dom	Dom	PROPN	_	Number=Sing	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+30	João	João	PROPN	_	Number=Sing	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 31	.	.	PUNCT	_	_	12	punct	O	_
 
 # text = A presença de formas exclusivamente não-marinhas indica sedimentação lacustre em ambientes propícios à formação de red beds.
@@ -21399,8 +21399,8 @@
 18-19	do	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
-20	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	17	obl	O	_
-21	Buracica	Buracica	PROPN	_	Number=Sing	20	flat:name	B=CAMPO	_
+20	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	17	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
+21	Buracica	Buracica	PROPN	_	Number=Sing	20	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 22-23	ao	_	_	_	_	_	_	_	_
 22	a	a	ADP	_	_	24	case	O	_
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
@@ -21415,8 +21415,8 @@
 # text = O Andar Alagoas na Bacia do Araripe registra a sedimentação da sequência pós-rifte I, representada pelas Formações Barbalha e Santana (Grupo Araripe).
 # sent_id = 284-20150917-MONOGRAFIA_0-124
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
-2	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	10	nsubj	O	_
-3	Alagoas	Alagoas	PROPN	_	Number=Sing	2	flat:name	B=BACIA	_
+2	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	10	nsubj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+3	Alagoas	Alagoas	PROPN	_	Number=Sing	2	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 4-5	na	_	_	_	_	_	_	_	_
 4	em	em	ADP	_	_	6	case	O	_
 5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -21445,7 +21445,7 @@
 25	Santana	Santana	PROPN	_	Gender=Masc|Number=Sing	22	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 26	(	(	PUNCT	_	_	27	punct	O	_
 27	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	22	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
-28	Araripe	Araripe	PROPN	_	Number=Sing	27	flat:name	B=BACIA	_
+28	Araripe	Araripe	PROPN	_	Number=Sing	27	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 29	)	)	PUNCT	_	_	27	punct	O	_
 30	.	.	PUNCT	_	_	10	punct	O	_
 
@@ -21584,7 +21584,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-129
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	8	nsubj	B=UNIDADE_LITOESTRATIGRÁFICA	_
-3	Santana	Santana	PROPN	_	Number=Sing	2	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+3	Santana	Santana	PROPN	_	Number=Sing	2	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 4	(	(	PUNCT	_	_	5	punct	O	_
 5	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	8	parataxis	O	_
 6	8	8	NUM	_	NumType=Card	5	nummod	O	_
@@ -21604,7 +21604,7 @@
 19-20	do	_	_	_	_	_	_	_	_
 19	de	de	ADP	_	_	18	flat:name	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	flat:name	O	_
-21	Araripe	Araripe	PROPN	_	Number=Sing	18	flat:name	B=BACIA	_
+21	Araripe	Araripe	PROPN	_	Number=Sing	18	flat:name	O	_
 22	,	,	PUNCT	_	_	8	punct	O	_
 23	tanto	tanto	ADV	_	_	26	advmod	O	_
 24-25	na	_	_	_	_	_	_	_	_
@@ -21623,7 +21623,7 @@
 35-36	do	_	_	_	_	_	_	_	_
 35	de	de	ADP	_	_	34	flat:name	O	_
 36	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	flat:name	O	_
-37	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	34	flat:name	B=BACIA	_
+37	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	34	flat:name	O	_
 38	,	,	PUNCT	_	_	31	punct	O	_
 39	quanto	quanto	CCONJ	_	_	42	cc	O	_
 40-41	na	_	_	_	_	_	_	_	_
@@ -21994,9 +21994,9 @@
 2	.	.	PUNCT	_	_	3	punct	O	_
 3	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	0	root	B=UNIDADE_LITOESTRATIGRÁFICA	_
 4	Romualdo	Romualdo	PROPN	_	Number=Sing	3	flat:name	O	_
-5	-	-	PUNCT	_	_	3	flat:name	O	_
-6	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	3	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
-7	Santana	Santana	PROPN	_	Number=Sing	6	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+5	-	-	PUNCT	_	_	6	punct	O	_
+6	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	3	conj	B=UNIDADE_LITOESTRATIGRÁFICA	_
+7	Santana	Santana	PROPN	_	Number=Sing	6	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 
 # text = O Membro Romualdo foi distinguido primeiramente por Beurlen (1971) como Membro Romualdo superior, pertencente à Formação Santana, juntamente com os membros sotopostos Ipubi (intermediário) e Crato (inferior).
 # sent_id = 284-20150917-MONOGRAFIA_0-142
@@ -22021,7 +22021,7 @@
 18	a	a	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 20	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	17	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-21	Santana	Santana	PROPN	_	Number=Sing	20	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+21	Santana	Santana	PROPN	_	Number=Sing	20	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 22	,	,	PUNCT	_	_	17	punct	O	_
 23	juntamente	juntamente	ADV	_	_	17	advmod	O	_
 24	com	com	ADP	_	_	26	case	O	_
@@ -22090,7 +22090,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-144
 1	A	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	det	O	_
 2	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	11	nsubj:pass	B=UNIDADE_LITOESTRATIGRÁFICA	_
-3	Santana	Santana	PROPN	_	Number=Sing	2	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+3	Santana	Santana	PROPN	_	Number=Sing	2	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 4	definida	definir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	2	acl	O	_
 5	por	por	ADP	_	_	6	case	O	_
 6	Beurlen	Beurlen	PROPN	_	Gender=Masc|Number=Sing	4	obl:agent	O	_
@@ -22108,7 +22108,7 @@
 18	)	)	PUNCT	_	_	17	punct	O	_
 19	para	para	ADP	_	_	20	case	O	_
 20	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	11	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-21	Santana	Santana	PROPN	_	Number=Sing	20	flat:name	B=CAMPO	_
+21	Santana	Santana	PROPN	_	Number=Sing	20	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 22	,	,	PUNCT	_	_	23	punct	O	_
 23	englobando	englobar	VERB	_	VerbForm=Ger	11	advcl	O	_
 24	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	25	det	O	_
@@ -22159,7 +22159,7 @@
 32	categoria	categoria	NOUN	_	Gender=Fem|Number=Sing	30	obj	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	32	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-35	Santana	Santana	PROPN	_	Number=Sing	34	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+35	Santana	Santana	PROPN	_	Number=Sing	34	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 36	e	e	CCONJ	_	_	39	cc	O	_
 37	consequentemente	consequentemente	ADV	_	_	39	advmod	O	_
 38	,	,	PUNCT	_	_	39	punct	O	_
@@ -22199,7 +22199,7 @@
 25	a	a	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	24	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-28	Santana	Santana	PROPN	_	Number=Sing	27	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+28	Santana	Santana	PROPN	_	Number=Sing	27	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 29-30	da	_	_	_	_	_	_	_	_
 29	de	de	ADP	_	_	31	case	O	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
@@ -22949,8 +22949,8 @@
 34-35	no	_	_	_	_	_	_	_	_
 34	em	em	ADP	_	_	36	case	O	_
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
-36	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	32	nmod	O	_
-37	Alagoas	Alagoas	PROPN	_	Number=Sing	36	flat:name	B=BACIA	_
+36	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	32	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+37	Alagoas	Alagoas	PROPN	_	Number=Sing	36	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 38	.	.	PUNCT	_	_	21	punct	O	_
 
 # text = Apesar de possuir características e conteúdo fóssil predominantemente lacustres na base do membro, para Neumann (1999), o Membro Romualdo apresenta influência marinha no seu topo, segundo Beurlen (1966) e Coimbra et al. (2002).
@@ -26904,7 +26904,7 @@
 8	entre	entre	ADP	_	_	10	case	O	_
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	7	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-11	Santana	Santana	PROPN	_	Number=Sing	10	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+11	Santana	Santana	PROPN	_	Number=Sing	10	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	10	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
 14	Romualdo	Romualdo	PROPN	_	Number=Sing	13	flat:name	O	_
@@ -26950,7 +26950,7 @@
 27-28	do	_	_	_	_	_	_	_	_
 27	de	de	ADP	_	_	26	flat:name	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	26	flat:name	O	_
-29	Araripe	Araripe	PROPN	_	Number=Sing	26	flat:name	B=BACIA	_
+29	Araripe	Araripe	PROPN	_	Number=Sing	26	flat:name	O	_
 30	,	,	PUNCT	_	_	33	punct	O	_
 31	como	como	SCONJ	_	_	33	mark	O	_
 32	já	já	ADV	_	_	33	advmod	O	_
@@ -27127,7 +27127,7 @@
 44	Romualdo	Romualdo	PROPN	_	Number=Sing	43	flat:name	O	_
 45	(	(	PUNCT	_	_	46	punct	O	_
 46	Fm.	Fm.	PROPN	_	Gender=Masc|Number=Sing	43	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
-47	Santana	Santana	PROPN	_	Number=Sing	46	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+47	Santana	Santana	PROPN	_	Number=Sing	46	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 48	)	)	PUNCT	_	_	46	punct	O	_
 49	e	e	CCONJ	_	_	51	cc	O	_
 50	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	51	det	O	_
@@ -29587,7 +29587,7 @@
 21	de	de	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
 23	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	20	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-24	Santana	Santana	PROPN	_	Number=Sing	23	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+24	Santana	Santana	PROPN	_	Number=Sing	23	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 25	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = Acima desta desconformidade basal foi observada, no presente trabalho, fácies conglomeráticas (fácies Gt) registradas como conglomerados, matriz sustentados, com clastos e seixos arredondados e de distintas composições (quartzo, argila, feldspato), sendo interpretadas como registros de depósitos de canais fluviais.
@@ -30389,7 +30389,7 @@
 11	Romualdo	Romualdo	PROPN	_	Number=Sing	10	flat:name	O	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	Fm.	Fm.	PROPN	_	Gender=Masc|Number=Sing	10	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
-14	Santana	Santana	PROPN	_	Number=Sing	13	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+14	Santana	Santana	PROPN	_	Number=Sing	13	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 15	)	)	PUNCT	_	_	13	punct	O	_
 16	e	e	CCONJ	_	_	18	cc	O	_
 17	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	det	O	_
@@ -30637,14 +30637,14 @@
 42-43	do	_	_	_	_	_	_	_	_
 42	de	de	ADP	_	_	44	case	O	_
 43	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	44	det	O	_
-44	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	41	nmod	O	_
-45	Alagoas	Alagoas	PROPN	_	Number=Sing	44	flat:name	B=BACIA	_
+44	Andar	Andar	PROPN	_	Gender=Masc|Number=Sing	41	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
+45	Alagoas	Alagoas	PROPN	_	Number=Sing	44	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 46	sotoposta	sotopor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	41	acl	O	_
 47-48	à	_	_	_	_	_	_	_	_
 47	a	a	ADP	_	_	49	case	O	_
 48	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	49	det	O	_
 49	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	46	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-50	Santana	Santana	PROPN	_	Number=Sing	49	flat:name	B=CAMPO|I=UNIDADE_LITOESTRATIGRÁFICA	_
+50	Santana	Santana	PROPN	_	Number=Sing	49	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 51	(	(	PUNCT	_	_	52	punct	O	_
 52	Assine	Assine	PROPN	_	Gender=Fem|Number=Sing	4	parataxis	O	_
 53	1994	1994	PROPN	_	Number=Sing	52	flat:name	O	_
@@ -50033,7 +50033,7 @@
 7	Eduardo	Eduardo	PROPN	_	Number=Sing	6	flat:name	O	_
 8	Carvalhido	Carvalhido	PROPN	_	Number=Sing	6	flat:name	O	_
 9	de	de	ADP	_	_	6	flat:name	O	_
-10	Souza	Souza	PROPN	_	Number=Sing	6	flat:name	B=BACIA	_
+10	Souza	Souza	PROPN	_	Number=Sing	6	flat:name	O	_
 11	em	em	ADP	_	_	13	case	O	_
 12	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	13	det	O	_
 13	dissertação	dissertação	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
@@ -50077,7 +50077,7 @@
 7	Eduardo	Eduardo	PROPN	_	Number=Sing	6	flat:name	O	_
 8	Carvalhido	Carvalhido	PROPN	_	Number=Sing	6	flat:name	O	_
 9	de	de	ADP	_	_	6	flat:name	O	_
-10	Souza	Souza	PROPN	_	Number=Sing	6	flat:name	B=BACIA	_
+10	Souza	Souza	PROPN	_	Number=Sing	6	flat:name	O	_
 11	em	em	ADP	_	_	13	case	O	_
 12	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	13	det	O	_
 13	dissertação	dissertação	NOUN	_	Gender=Fem|Number=Sing	4	obl	O	_
@@ -50876,7 +50876,7 @@
 27	Eduardo	Eduardo	PROPN	_	Number=Sing	26	flat:name	O	_
 28	Carvalhido	Carvalhido	PROPN	_	Number=Sing	26	flat:name	O	_
 29	de	de	ADP	_	_	26	flat:name	O	_
-30	Souza	Souza	PROPN	_	Number=Sing	26	flat:name	B=BACIA	_
+30	Souza	Souza	PROPN	_	Number=Sing	26	flat:name	O	_
 31	,	,	PUNCT	_	_	24	punct	O	_
 32	em	em	ADP	_	_	34	case	O	_
 33	sua	seu	DET	_	Gender=Fem|Number=Sing|PronType=Prs	34	det	O	_
@@ -58893,10 +58893,10 @@
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	Cráton	Cráton	PROPN	_	Gender=Masc|Number=Sing	12	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 17-18	do	_	_	_	_	_	_	_	_
-17	de	de	ADP	_	_	16	flat:name	O	_
-18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
-19	São	São	PROPN	_	Number=Sing	16	flat:name	B=BACIA	_
-20	Francisco	Francisco	PROPN	_	Number=Sing	16	flat:name	I=BACIA	_
+17	de	de	ADP	_	_	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+19	São	São	PROPN	_	Number=Sing	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+20	Francisco	Francisco	PROPN	_	Number=Sing	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 21	.	.	PUNCT	_	_	6	punct	O	_
 
 # text = O Grupo Areado, objeto deste estudo, é o representante eocretácico desta bacia e divide-se nas Formações Abaeté, Quiricó e Três Barras, da base para o topo respectivamente.
@@ -59529,7 +59529,7 @@
 16	de	de	ADP	_	_	18	case	O	_
 17	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	18	det	O	_
 18	Grupo	Grupo	PROPN	_	Gender=Masc|Number=Sing	15	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-19	Areado	Areado	PROPN	_	Number=Sing	18	flat:name	O	_
+19	Areado	Areado	PROPN	_	Number=Sing	18	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 20-21	da	_	_	_	_	_	_	_	_
 20	de	de	ADP	_	_	18	flat:name	O	_
 21	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	18	flat:name	O	_
@@ -59735,10 +59735,10 @@
 23	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	det	O	_
 24	Alto	Alto	PROPN	_	Gender=Masc|Number=Sing	21	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 25-26	do	_	_	_	_	_	_	_	_
-25	de	de	ADP	_	_	24	flat:name	O	_
-26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	flat:name	O	_
-27	São	São	PROPN	_	Number=Sing	24	flat:name	B=BACIA	_
-28	Francisco	Francisco	PROPN	_	Number=Sing	24	flat:name	I=BACIA	_
+25	de	de	ADP	_	_	24	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	24	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+27	São	São	PROPN	_	Number=Sing	24	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+28	Francisco	Francisco	PROPN	_	Number=Sing	24	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 29	,	,	PUNCT	_	_	1	punct	O	_
 30	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	32	nsubj	O	_
 31	a	ela	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	32	obj	O	_
@@ -60426,7 +60426,7 @@
 # sent_id = 283-20150917-MONOGRAFIA_0-47
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	5	nsubj:pass	B=UNIDADE_LITOESTRATIGRÁFICA	_
-3	Canabrava	Canabrava	PROPN	_	Number=Sing	2	flat:name	B=CAMPO	_
+3	Canabrava	Canabrava	PROPN	_	Number=Sing	2	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 4	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	5	aux:pass	O	_
 5	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 6-7	na	_	_	_	_	_	_	_	_
@@ -61249,7 +61249,7 @@
 13	de	de	ADP	_	_	14	case	O	_
 14	Scorza	Scorza	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
 15	e	e	CCONJ	_	_	14	flat:name	O	_
-16	Santos	Santos	PROPN	_	Number=Sing	14	flat:name	B=BACIA	_
+16	Santos	Santos	PROPN	_	Number=Sing	14	flat:name	O	_
 17	(	(	PUNCT	_	_	18	punct	O	_
 18	1955	1955	NUM	_	NumType=Card	14	nmod:appos	O	_
 19	)	)	PUNCT	_	_	18	punct	O	_
@@ -63989,7 +63989,7 @@
 48	)	)	PUNCT	_	_	47	punct	O	_
 49	como	como	ADP	_	_	50	case	O	_
 50	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	43	xcomp	B=UNIDADE_LITOESTRATIGRÁFICA	_
-51	Canabrava	Canabrava	PROPN	_	Number=Sing	50	flat:name	B=CAMPO	_
+51	Canabrava	Canabrava	PROPN	_	Number=Sing	50	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 52	.	.	PUNCT	_	_	28	punct	O	_
 
 # text = A associação das fácies sedimentares observadas para a seção-colunar Córrego Tiririca (Fig. 6) é semelhante àquela encontrada na região de Canabrava, não ocorrendo apenas a fácies G, bem característica e recorrente na região norte da área.
@@ -64270,7 +64270,7 @@
 12	para	para	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	8	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-15	.	.	PROPN	_	_	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+15	.	.	PROPN	_	_	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 16	Abaeté	Abaeté	PROPN	_	Gender=Masc|Number=Sing	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 17	.	.	PUNCT	_	_	8	punct	O	_
 
@@ -65473,7 +65473,7 @@
 9	para	para	ADP	_	_	11	case	O	_
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	6	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-12	.	.	PROPN	_	_	13	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+12	.	.	PROPN	_	_	11	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 13	Quiricó	Quiricó	PROPN	_	Gender=Fem|Number=Sing	11	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 14	.	.	PUNCT	_	_	6	punct	O	_
 
@@ -65744,7 +65744,7 @@
 24	para	para	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	23	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
-27	.	.	PROPN	_	_	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+27	.	.	PROPN	_	_	26	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 28	Quiricó	Quiricó	PROPN	_	Gender=Fem|Number=Sing	26	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 29	.	.	PUNCT	_	_	13	punct	O	_
 
@@ -66192,7 +66192,7 @@
 41	para	para	ADP	_	_	43	case	O	_
 42	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	det	O	_
 43	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	40	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-44	.	.	PROPN	_	_	45	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+44	.	.	PROPN	_	_	43	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 45	Quiricó	Quiricó	PROPN	_	Gender=Masc|Number=Sing	43	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 46	.	.	PUNCT	_	_	36	punct	O	_
 
@@ -66248,7 +66248,7 @@
 17	(	(	PUNCT	_	_	18	punct	O	_
 18	Scorza	Scorza	PROPN	_	Gender=Masc|Number=Sing	9	parataxis	O	_
 19	e	e	CCONJ	_	_	18	flat:name	O	_
-20	Santos	Santos	PROPN	_	Number=Sing	18	flat:name	B=BACIA	_
+20	Santos	Santos	PROPN	_	Number=Sing	18	flat:name	O	_
 21	,	,	PUNCT	_	_	18	flat:name	O	_
 22	1955	1955	NUM	_	NumType=Card	18	flat:name	O	_
 23	)	)	PUNCT	_	_	18	punct	O	_
@@ -66600,7 +66600,7 @@
 5	para	para	ADP	_	_	7	case	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	4	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-8	.	.	PROPN	_	_	9	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+8	.	.	PROPN	_	_	7	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 9	Quiricó	Quiricó	PROPN	_	Gender=Fem|Number=Sing	7	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 10	correspondem	corresponder	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11-12	às	_	_	_	_	_	_	_	_
@@ -66693,7 +66693,7 @@
 21	para	para	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
 23	Fm	Fm	PROPN	_	Gender=Fem|Number=Sing	20	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
-24	.	.	PROPN	_	_	25	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+24	.	.	PROPN	_	_	23	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 25	Quiricó	Quiricó	PROPN	_	Gender=Fem|Number=Sing	23	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 26	,	,	PUNCT	_	_	27	punct	O	_
 27	formando	formar	VERB	_	VerbForm=Ger	8	advcl	O	_
@@ -69532,12 +69532,12 @@
 11	,	,	PUNCT	_	_	8	punct	O	_
 12	inclusive	inclusive	ADV	_	_	8	advmod	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	Arco	Arco	PROPN	_	Gender=Masc|Number=Sing	8	obl	O	_
+14	Arco	Arco	PROPN	_	Gender=Masc|Number=Sing	8	obl	B=UNIDADE_LITOESTRATIGRÁFICA	_
 15-16	do	_	_	_	_	_	_	_	_
-15	de	de	ADP	_	_	14	flat:name	O	_
-16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	flat:name	O	_
-17	Alto	Alto	PROPN	_	Number=Sing	14	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
-18	Paranaíba	Paranaíba	PROPN	_	Number=Sing	14	flat:name	O	_
+15	de	de	ADP	_	_	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+17	Alto	Alto	PROPN	_	Number=Sing	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+18	Paranaíba	Paranaíba	PROPN	_	Number=Sing	14	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 19	e	e	CCONJ	_	_	21	cc	O	_
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	Falha	Falha	PROPN	_	Gender=Fem|Number=Sing	14	conj	O	_
@@ -73771,7 +73771,7 @@
 26-27	da	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	25	flat:name	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	25	flat:name	O	_
-28	Costa	Costa	PROPN	_	Number=Sing	25	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+28	Costa	Costa	PROPN	_	Number=Sing	25	flat:name	O	_
 29	,	,	PUNCT	_	_	25	flat:name	O	_
 30	2009	2009	NUM	_	NumType=Card	25	flat:name	O	_
 31	)	)	PUNCT	_	_	25	punct	O	_
@@ -84872,8 +84872,8 @@
 13-14	do	_	_	_	_	_	_	_	_
 13	de	de	ADP	_	_	12	flat:name	O	_
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	flat:name	O	_
-15	Espírito	Espírito	PROPN	_	Number=Sing	12	flat:name	B=BACIA	_
-16	Santo	Santo	PROPN	_	Number=Sing	12	flat:name	I=BACIA	_
+15	Espírito	Espírito	PROPN	_	Number=Sing	12	flat:name	O	_
+16	Santo	Santo	PROPN	_	Number=Sing	12	flat:name	O	_
 17	,	,	PUNCT	_	_	3	punct	O	_
 18-19	nas	_	_	_	_	_	_	_	_
 18	em	em	ADP	_	_	20	case	O	_
@@ -88284,8 +88284,8 @@
 13	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	15	cop	O	_
 14	:	:	PUNCT	_	_	15	punct	O	_
 15	OOIP	OOIP	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
-16	de	de	ADP	_	_	15	flat:name	O	_
-17	Marlim	Marlim	PROPN	_	Number=Sing	15	flat:name	B=CAMPO	_
+16	de	de	ADP	_	_	17	case	O	_
+17	Marlim	Marlim	PROPN	_	Number=Sing	15	nmod	B=CAMPO	_
 18	=	=	PUNCT	_	_	19	punct	O	_
 19	8100	8100	NUM	_	NumType=Card	15	appos	O	_
 20	+	+	PUNCT	_	_	21	punct	O	_
@@ -88300,8 +88300,8 @@
 # text = OOIP de Barracuda = 2250 + 6,3 = 357,71 (10º m?)
 # sent_id = 161-20150810-MONOGRAFIA_0-610
 1	OOIP	OOIP	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
-2	de	de	ADP	_	_	1	flat:name	O	_
-3	Barracuda	Barracuda	PROPN	_	Number=Sing	1	flat:name	B=CAMPO	_
+2	de	de	ADP	_	_	3	case	O	_
+3	Barracuda	Barracuda	PROPN	_	Number=Sing	1	nmod	B=CAMPO	_
 4	=	=	PUNCT	_	_	1	flat:name	O	_
 5	2250	2250	NUM	_	NumType=Card	1	nmod:appos	O	_
 6	+	+	PUNCT	_	_	7	punct	O	_
@@ -88657,7 +88657,7 @@
 12	será	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	14	cop	O	_
 13	:	:	PUNCT	_	_	14	punct	O	_
 14	Vco2de	Vco2de	PROPN	_	Gender=Fem|Number=Sing	0	root	O	_
-15	Marlim	Marlim	PROPN	_	Number=Sing	14	flat:name	B=CAMPO	_
+15	Marlim	Marlim	PROPN	_	Number=Sing	14	nmod	B=CAMPO	_
 16	=	=	PUNCT	_	_	14	flat:name	O	_
 17	(	(	PUNCT	_	_	18	punct	O	_
 18	321,94/1000	321,94/1000	NUM	_	NumType=Card	14	appos	O	_
@@ -89352,8 +89352,8 @@
 # text = EOR de Barracuda = 0,085 x 2250 x C = 143,44 Milhões de barris de óleo
 # sent_id = 161-20150810-MONOGRAFIA_0-643
 1	EOR	EOR	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
-2	de	de	ADP	_	_	1	flat:name	O	_
-3	Barracuda	Barracuda	PROPN	_	Number=Sing	1	flat:name	B=CAMPO	_
+2	de	de	ADP	_	_	3	case	O	_
+3	Barracuda	Barracuda	PROPN	_	Number=Sing	1	nmod	B=CAMPO	_
 4	=	=	PUNCT	_	_	6	punct	O	_
 5	0,085	0,085	NUM	_	NumType=Card	6	nummod	O	_
 6	x	x	NOUN	_	Gender=Masc|Number=Plur	1	appos	O	_
@@ -98672,21 +98672,21 @@
 27	assim	assim	ADV	_	_	30	cc	O	MWEPOS=CCONJ
 28	como	como	ADP	_	_	27	fixed	O	_
 29	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	30	det	O	_
-30	Campos	Campos	PROPN	_	Gender=Masc|Number=Plur	23	conj	O	_
-31	de	de	ADP	_	_	30	flat:name	O	_
-32	Parati	Parati	PROPN	_	Number=Sing	30	flat:name	B=CAMPO	_
+30	Campos	Campos	PROPN	_	Gender=Masc|Number=Plur	23	conj	B=CAMPO	_
+31	de	de	ADP	_	_	30	flat:name	I=CAMPO	_
+32	Parati	Parati	PROPN	_	Number=Sing	30	flat:name	I=CAMPO	_
 33	,	,	PUNCT	_	_	34	punct	O	_
-34	Carioca	Carioca	PROPN	_	Gender=Masc|Number=Sing	30	conj	O	_
+34	Carioca	Carioca	PROPN	_	Gender=Masc|Number=Sing	30	conj	B=CAMPO	_
 35	,	,	PUNCT	_	_	36	punct	O	_
-36	Caramba	Caramba	PROPN	_	Gender=Fem|Number=Sing	30	conj	O	_
+36	Caramba	Caramba	PROPN	_	Gender=Fem|Number=Sing	30	conj	B=CAMPO	_
 37	,	,	PUNCT	_	_	38	punct	O	_
-38	Jupiter	Jupiter	PROPN	_	Gender=Fem|Number=Sing	30	conj	O	_
+38	Jupiter	Jupiter	PROPN	_	Gender=Fem|Number=Sing	30	conj	B=CAMPO	_
 39	,	,	PUNCT	_	_	40	punct	O	_
-40	Guará	Guará	PROPN	_	Gender=Fem|Number=Sing	30	conj	O	_
+40	Guará	Guará	PROPN	_	Gender=Fem|Number=Sing	30	conj	B=CAMPO	_
 41	,	,	PUNCT	_	_	42	punct	O	_
-42	Bem-te-Vi	Bem-te-Vi	PROPN	_	Gender=Masc|Number=Sing	30	conj	O	_
+42	Bem-te-Vi	Bem-te-Vi	PROPN	_	Gender=Masc|Number=Sing	30	conj	B=CAMPO	_
 43	e	e	CCONJ	_	_	44	cc	O	_
-44	Iara	Iara	PROPN	_	Gender=Fem|Number=Sing	30	conj	O	_
+44	Iara	Iara	PROPN	_	Gender=Fem|Number=Sing	30	conj	B=CAMPO	_
 45	(	(	PUNCT	_	_	46	punct	O	_
 46	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	6	parataxis	O	_
 47	4	4	NUM	_	NumType=Card	46	nummod	O	_
@@ -104179,7 +104179,7 @@
 45	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	46	det	O	_
 46	Bloco	Bloco	PROPN	_	Gender=Masc|Number=Sing	4	obl	O	_
 47	Green	Green	PROPN	_	Number=Sing	46	flat:name	O	_
-48	Canyon	Canyon	PROPN	_	Number=Sing	46	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+48	Canyon	Canyon	PROPN	_	Number=Sing	46	flat:name	O	_
 49	640	640	PROPN	_	Number=Sing	46	flat:name	O	_
 50	,	,	PUNCT	_	_	46	punct	O	_
 51	e	e	CCONJ	_	_	52	cc	O	_
@@ -186880,10 +186880,10 @@
 10	sobre	sobre	ADP	_	_	12	case	O	_
 11	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	12	det	O	_
 12	Bacias	Bacias	PROPN	_	Gender=Fem|Number=Plur	9	nmod	O	_
-13	de	de	ADP	_	_	12	flat:name	O	_
-14	Campos	Campos	PROPN	_	Number=Sing	12	flat:name	B=BACIA	_
+13	de	de	ADP	_	_	14	case	O	_
+14	Campos	Campos	PROPN	_	Number=Sing	12	nmod	B=BACIA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Santos	Santos	PROPN	_	Number=Sing	12	conj	B=BACIA	_
+16	Santos	Santos	PROPN	_	Number=Sing	14	conj	B=BACIA	_
 17	.	.	PUNCT	_	_	6	punct	O	_
 
 # text = A porção destes dados utilizada nesta pesquisa, que recobre as áreas proximais destas bacias incluindo o Alto de Cabo Frio (figura 1.1), é de valia para o entendimento de aspectos geológicos destas áreas que, por fazerem parte do contexto evolutivo de tais bacias, influenciaram os processos de acumulação de óleo e gás nas mesmas.
@@ -188741,10 +188741,10 @@
 1	2.2.1	2.2.1	NUM	_	NumType=Card	3	parataxis	O	_
 2	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	3	det	O	_
 3	Bacias	Bacias	PROPN	_	Number=Sing	0	root	O	_
-4	de	de	ADP	_	_	3	flat:name	O	_
-5	Santos	Santos	PROPN	_	Number=Sing	3	flat:name	B=BACIA	_
+4	de	de	ADP	_	_	5	case	O	_
+5	Santos	Santos	PROPN	_	Number=Sing	3	nmod	B=BACIA	_
 6	e	e	CCONJ	_	_	7	cc	O	_
-7	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	3	conj	O	_
+7	Campos	Campos	PROPN	_	Gender=Masc|Number=Sing	5	conj	B=BACIA	_
 
 # text = As bacias de Campos e Santos constituem importantes estruturas tectônicas subsidentes da margem continental sudeste que se posicionam na porção marinha adjacente à área desta pesquisa (figura 1.1).
 # sent_id = 20-20140904-TESEDSC_0-80
@@ -189340,7 +189340,7 @@
 14	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 15	(	(	PUNCT	_	_	16	punct	O	_
 16	Formação	Formação	PROPN	_	Gender=Fem|Number=Sing	14	appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
-17	Santos	Santos	PROPN	_	Number=Sing	16	flat:name	B=BACIA|I=UNIDADE_LITOESTRATIGRÁFICA	_
+17	Santos	Santos	PROPN	_	Number=Sing	16	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 18	)	)	PUNCT	_	_	16	punct	O	_
 19	e	e	CCONJ	_	_	20	cc	O	_
 20	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	14	conj	O	_
@@ -189940,7 +189940,7 @@
 16	turbidíticos	turbidítico	ADJ	_	Gender=Masc|Number=Plur	15	amod	O	_
 17	(	(	PUNCT	_	_	18	punct	O	_
 18	Membro	Membro	PROPN	_	Gender=Masc|Number=Sing	15	nmod:appos	B=UNIDADE_LITOESTRATIGRÁFICA	_
-19	Namorado	Namorado	PROPN	_	Number=Sing	18	flat:name	B=CAMPO	_
+19	Namorado	Namorado	PROPN	_	Number=Sing	18	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 20	)	)	PUNCT	_	_	18	punct	O	_
 21	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -190430,10 +190430,10 @@
 9	de	de	ADP	_	_	11	case	O	_
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
 11	Bacias	Bacia	PROPN	_	Gender=Fem|Number=Sing	7	nmod	O	_
-12	de	de	ADP	_	_	3	flat:name	O	_
-13	Campos	Campos	PROPN	_	Number=Sing	3	flat:name	O	_
-14	e	e	CCONJ	_	_	3	flat:name	O	_
-15	Santos	Santos	PROPN	_	Number=Sing	3	flat:name	B=BACIA	_
+12	de	de	ADP	_	_	13	case	O	_
+13	Campos	Campos	PROPN	_	Number=Sing	11	nmod	B=BACIA	_
+14	e	e	CCONJ	_	_	15	cc	O	_
+15	Santos	Santos	PROPN	_	Number=Sing	13	conj	B=BACIA	_
 
 # text = O embasamento raso nas áreas proximais das bacias de Campos e Santos, cujo trecho correspondente ao Estado do Rio está em foco nesta pesquisa (figura 1.1), representa uma porção da plataforma continental adjacente às respectivas bacias.
 # sent_id = 20-20140904-TESEDSC_0-130
@@ -191142,7 +191142,7 @@
 46-47	da	_	_	_	_	_	_	_	_
 46	de	de	ADP	_	_	43	flat:name	O	_
 47	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	43	flat:name	O	_
-48	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	43	flat:name	B=CAMPO	_
+48	Serra	Serra	PROPN	_	Gender=Fem|Number=Sing	43	flat:name	O	_
 49-50	do	_	_	_	_	_	_	_	_
 49	de	de	ADP	_	_	43	flat:name	O	_
 50	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	43	flat:name	O	_
@@ -191990,10 +191990,10 @@
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	Cráton	Cráton	PROPN	_	Gender=Masc|Number=Sing	26	obj	B=UNIDADE_LITOESTRATIGRÁFICA	_
 29-30	do	_	_	_	_	_	_	_	_
-29	de	de	ADP	_	_	28	flat:name	O	_
-30	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	flat:name	O	_
-31	São	São	PROPN	_	Gender=Masc|Number=Sing	28	flat:name	B=BACIA	_
-32	Francisco	Francisco	PROPN	_	Number=Sing	28	flat:name	I=BACIA	_
+29	de	de	ADP	_	_	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+30	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+31	São	São	PROPN	_	Gender=Masc|Number=Sing	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
+32	Francisco	Francisco	PROPN	_	Number=Sing	28	flat:name	I=UNIDADE_LITOESTRATIGRÁFICA	_
 33	a	a	ADP	_	_	34	case	O	_
 34	sudeste	sudeste	NOUN	_	Gender=Masc|Number=Sing	26	obl	O	_
 35	e	e	CCONJ	_	_	36	cc	O	_
@@ -192084,7 +192084,7 @@
 # sent_id = 20-20140904-TESEDSC_0-169
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	Terreno	Terreno	PROPN	_	Gender=Masc|Number=Sing	8	nsubj:pass	O	_
-3	Serra	Serra	PROPN	_	Number=Sing	2	flat:name	B=CAMPO	_
+3	Serra	Serra	PROPN	_	Number=Sing	2	flat:name	O	_
 4-5	do	_	_	_	_	_	_	_	_
 4	de	de	ADP	_	_	2	flat:name	O	_
 5	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	flat:name	O	_
@@ -192153,7 +192153,7 @@
 62	de	de	ADP	_	_	64	case	O	_
 63	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	64	det	O	_
 64	Unidades	Unidades	PROPN	_	Gender=Fem|Number=Plur	61	nmod	O	_
-65	Búzios	Búzios	PROPN	_	Number=Sing	64	flat:name	B=CAMPO	_
+65	Búzios	Búzios	PROPN	_	Number=Sing	64	flat:name	O	_
 66	(	(	PUNCT	_	_	67	punct	O	_
 67	Heilbron	Heilbron	PROPN	_	Gender=Fem|Number=Sing	8	parataxis	O	_
 68	et	et	PROPN	_	Number=Sing	67	flat:name	O	_
@@ -192790,7 +192790,7 @@
 37	Soarinho	Soarinho	PROPN	_	Gender=Masc|Number=Sing	31	conj	O	_
 38	,	,	PUNCT	_	_	39	punct	O	_
 39	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	31	conj	O	_
-40	Bonito	Bonito	PROPN	_	Number=Sing	39	flat:name	B=CAMPO	_
+40	Bonito	Bonito	PROPN	_	Number=Sing	39	flat:name	O	_
 41	e	e	CCONJ	_	_	42	cc	O	_
 42	Morro	Morro	PROPN	_	Gender=Masc|Number=Sing	31	conj	O	_
 43	de	de	ADP	_	_	42	flat:name	O	_
@@ -195020,9 +195020,9 @@
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
 2	gráben	gráben	NOUN	_	Gender=Masc|Number=Sing	26	nsubj:pass	O	_
 3-4	da	_	_	_	_	_	_	_	_
-3	de	de	ADP	_	_	2	flat:name	O	_
-4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	2	flat:name	O	_
-5	Barra	barra	PROPN	_	Gender=Fem|Number=Sing	2	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+3	de	de	ADP	_	_	5	case	O	_
+4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
+5	Barra	barra	PROPN	_	Gender=Fem|Number=Sing	2	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 6-7	do	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	5	flat:name	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	flat:name	O	_
@@ -196067,7 +196067,7 @@
 24	Monte	Monte	PROPN	_	Gender=Masc|Number=Sing	22	appos	O	_
 25	de	de	ADP	_	_	24	flat:name	O	_
 26	Trigo,	Trigo,	PROPN	_	Number=Sing	24	flat:name	O	_
-27	Búzios	Búzios	PROPN	_	Number=Sing	24	flat:name	B=CAMPO	_
+27	Búzios	Búzios	PROPN	_	Number=Sing	24	flat:name	O	_
 28	e	e	CCONJ	_	_	24	flat:name	O	_
 29	Vitória	Vitória	PROPN	_	Number=Sing	24	flat:name	O	_
 30	)	)	PUNCT	_	_	24	punct	O	_
@@ -197122,9 +197122,9 @@
 35	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	36	det	O	_
 36	gráben	gráben	NOUN	_	Gender=Masc|Number=Sing	33	nmod	O	_
 37-38	da	_	_	_	_	_	_	_	_
-37	de	de	ADP	_	_	36	flat:name	O	_
-38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	flat:name	O	_
-39	Barra	Barra	PROPN	_	Gender=Fem|Number=Sing	36	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+37	de	de	ADP	_	_	39	case	O	_
+38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
+39	Barra	Barra	PROPN	_	Gender=Fem|Number=Sing	36	nmod	B=UNIDADE_LITOESTRATIGRÁFICA	_
 40-41	do	_	_	_	_	_	_	_	_
 40	de	de	ADP	_	_	39	flat:name	O	_
 41	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	39	flat:name	O	_
@@ -197820,8 +197820,8 @@
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 15	Projeto	Projeto	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 16	Aerogeofisico	Aerogeofisico	PROPN	_	Number=Sing	15	flat:name	O	_
-17	São	São	PROPN	_	Number=Sing	15	flat:name	B=BACIA	_
-18	Paulo	Paulo	PROPN	_	Number=Sing	15	flat:name	I=BACIA	_
+17	São	São	PROPN	_	Number=Sing	15	flat:name	O	_
+18	Paulo	Paulo	PROPN	_	Number=Sing	15	flat:name	O	_
 19	–	–	PUNCT	_	_	15	flat:name	O	_
 20	Rio	Rio	PROPN	_	Gender=Masc|Number=Sing	15	flat:name	O	_
 21	de	de	ADP	_	_	15	flat:name	O	_
@@ -213832,17 +213832,17 @@
 2	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
 3	Gráben	Gráben	PROPN	_	Gender=Masc|Number=Sing	0	root	B=UNIDADE_LITOESTRATIGRÁFICA	_
 4-5	da	_	_	_	_	_	_	_	_
-4	de	de	ADP	_	_	3	flat:name	O	_
-5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	flat:name	O	_
-6	Porção	Porção	PROPN	_	Gender=Fem|Number=Sing	3	flat:name	O	_
-7	Proximal	Proximal	PROPN	_	Number=Sing	3	flat:name	O	_
-8	NE	NE	PROPN	_	Number=Sing	3	flat:name	O	_
+4	de	de	ADP	_	_	6	case	O	_
+5	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
+6	Porção	porção	NOUN	_	Gender=Fem|Number=Sing	3	nmod	O	_
+7	Proximal	proximal	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
+8	NE	NE	PROPN	_	Number=Sing	6	nmod	O	_
 9-10	da	_	_	_	_	_	_	_	_
-9	de	de	ADP	_	_	3	flat:name	O	_
-10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	flat:name	O	_
-11	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	3	flat:name	B=BACIA	_
-12	de	de	ADP	_	_	3	flat:name	O	_
-13	Santos	Santos	PROPN	_	Number=Sing	3	flat:name	B=BACIA	_
+9	de	de	ADP	_	_	11	case	O	_
+10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
+11	Bacia	Bacia	PROPN	_	Gender=Fem|Number=Sing	6	nmod	B=BACIA	_
+12	de	de	ADP	_	_	11	flat:name	I=BACIA	_
+13	Santos	Santos	PROPN	_	Number=Sing	11	flat:name	I=BACIA	_
 
 # text = A porção proximal NE da Bacia de Santos compreendida na área desta pesquisa (figura 1.2) apresenta um domínio magnético de baixa freqüência (Rbl no mapa de interpretação do anexo 1), bastante evidenciado no mapa de amplitude do sinal analítico (anexo 3).
 # sent_id = 20-20140904-TESEDSC_0-746
@@ -219650,7 +219650,7 @@
 14	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	15	det	O	_
 15	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 16	de	de	ADP	_	_	15	flat:name	O	_
-17	Búzios	Búzios	PROPN	_	Number=Sing	15	flat:name	B=CAMPO	_
+17	Búzios	Búzios	PROPN	_	Number=Sing	15	flat:name	O	_
 18	,	,	PUNCT	_	_	9	punct	O	_
 19	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	20	det	O	_
 20	baixo	baixo	NOUN	_	Gender=Masc|Number=Sing	23	nsubj	O	_
@@ -229210,7 +229210,7 @@
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
 29	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	26	nmod	O	_
 30	de	de	ADP	_	_	29	flat:name	O	_
-31	Búzios	Búzios	PROPN	_	Number=Sing	29	flat:name	B=CAMPO	_
+31	Búzios	Búzios	PROPN	_	Number=Sing	29	flat:name	O	_
 32	e	e	CCONJ	_	_	34	cc	O	_
 33	rápida	rápido	ADJ	_	Gender=Fem|Number=Sing	34	amod	O	_
 34	diminuição	diminuição	NOUN	_	Gender=Fem|Number=Sing	26	conj	O	_
@@ -281944,10 +281944,10 @@
 2	Estuário	Estuário	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
 3	Barra	Barra	PROPN	_	Number=Sing	2	flat:name	O	_
 4	Grande	Grande	PROPN	_	Number=Sing	2	flat:name	O	_
-5	-	-	PUNCT	_	_	2	flat:name	O	_
-6	Icapuí	Icapuí	PROPN	_	Gender=Masc|Number=Sing	2	flat:name	O	_
-7	-	-	PUNCT	_	_	2	flat:name	O	_
-8	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	2	flat:name	B=BACIA	_
+5	-	-	PUNCT	_	_	6	punct	O	_
+6	Icapuí	Icapuí	PROPN	_	Gender=Masc|Number=Sing	2	conj	O	_
+7	-	-	PUNCT	_	_	8	punct	O	_
+8	Ceará	Ceará	PROPN	_	Gender=Masc|Number=Sing	2	conj	O	_
 
 # text = Dentro do município de Icapuí encontra-se o estuário Barra Grande, com extensão de 28 km² georreferenciada sob as coordenadas geográficas (Norte/Oeste) 4º40'15.84'' S, 37º23'10.98'' W e (Sul/Leste) 4º43'45.39''S, 37º19'01.36'' W. local de grande biodiversidade marinha, lagoas costeiras, manguezais, terraços marinhos, delta de maré, dunas, banco de algas e a faixa de praia.
 # sent_id = 411-20150702-MONOGRAFIA_0-118
@@ -282373,7 +282373,7 @@
 1	2.2.4	2.2.4	NUM	_	NumType=Card	3	parataxis	O	_
 2	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
 3	Fluxo	Fluxo	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
-4	Costeiro	Costeiro	PROPN	_	Number=Sing	3	flat:name	B=UNIDADE_LITOESTRATIGRÁFICA	_
+4	Costeiro	Costeiro	PROPN	_	Number=Sing	3	flat:name	O	_
 5	e	e	CCONJ	_	_	7	cc	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
 7	Interferências	Interferências	PROPN	_	Gender=Fem|Number=Plur	3	conj	O	_
@@ -287216,7 +287216,7 @@
 
 # text = São elas :  Deflexão  Contenção e recolhimento  Dispersão química  Queima in situ  Limpeza da costa
 # sent_id = 411-20150702-MONOGRAFIA_0-261
-1	São	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	2	cop	O	_
+1	São	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	2	cop	O	sentenciação
 2	elas	ele	PRON	_	Case=Nom|Gender=Fem|Number=Plur|Person=3|PronType=Prs	0	root	O	_
 3	:	:	PUNCT	_	_	5	punct	O	_
 4			DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
