commit e43f36cf6f9f176c28cdec843f93393bf7bfc923
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Jan 4 16:49:30 2022 -0300

    correções 'NUM a NUM' como conj

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index f072407..21dcc59 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -6221,8 +6221,8 @@
 20	(	(	PUNCT	_	_	21	punct	O	_
 21	Figuras	figura	NOUN	_	Gender=Fem|Number=Plur	5	parataxis	O	_
 22	15	15	NUM	_	NumType=Card	21	nummod	O	_
-23	a	a	ADP	_	_	24	case	O	_
-24	18	18	NUM	_	NumType=Card	21	nummod	O	_
+23	a	a	CCONJ	_	_	24	cc	O	_
+24	18	18	NUM	_	NumType=Card	22	conj	O	_
 25	)	)	PUNCT	_	_	21	punct	O	_
 26	,	,	PUNCT	_	_	16	punct	O	_
 27	juntamente	juntamente	ADV	_	_	16	advmod	O	_
@@ -7039,7 +7039,7 @@
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
 8	afloramentos	afloramento	NOUN	_	Gender=Masc|Number=Plur	5	obl:arg	O	_
 9	194	194	NUM	_	NumType=Card	8	nummod	O	_
-10	a	a	ADP	_	_	11	case	O	_
+10	a	a	CCONJ	_	_	11	cc	O	_
 11	206	206	NUM	_	NumType=Card	9	conj	O	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	5	parataxis	O	_
@@ -8519,15 +8519,15 @@
 14	Eoceno	Eoceno	PROPN	_	Gender=Masc|Number=Sing	9	conj	B=UNIDADE_CRONOESTRATIGRÁFICA	_
 15	variam	variar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	de	de	ADP	_	_	17	case	O	_
-17	100	100	NUM	_	NumType=Card	15	obl	O	_
-18	a	a	ADP	_	_	20	case	O	_
-19	1000	1000	NUM	_	NumType=Card	20	nummod	O	_
-20	milissegundos	milissegundo	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
+17	100	100	NUM	_	NumType=Card	20	nummod	O	_
+18	a	a	CCONJ	_	_	19	cc	O	_
+19	1000	1000	NUM	_	NumType=Card	17	conj	O	_
+20	milissegundos	milissegundo	NOUN	_	Gender=Masc|Number=Plur	15	obl	O	_
 21	e	e	CCONJ	_	_	22	cc	O	_
-22	50	50	NUM	_	NumType=Card	20	conj	O	_
-23	a	a	ADP	_	_	25	case	O	_
-24	1700	1700	NUM	_	NumType=Card	25	nummod	O	_
-25	milissegundos	milissegundo	NOUN	_	Gender=Masc|Number=Plur	22	nmod	O	_
+22	50	50	NUM	_	NumType=Card	25	nummod	O	_
+23	a	a	CCONJ	_	_	24	cc	O	_
+24	1700	1700	NUM	_	NumType=Card	22	conj	O	_
+25	milissegundos	milissegundo	NOUN	_	Gender=Masc|Number=Plur	20	conj	O	_
 26	em	em	ADP	_	_	29	case	O	_
 27	todas	todo	DET	_	Gender=Fem|Number=Plur|PronType=Tot	29	det	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	29	det	O	_
@@ -11314,10 +11314,10 @@
 20	0,125	0,125	NUM	_	NumType=Card	21	nummod	O	_
 21	mm	mm	NOUN	_	Gender=Masc|Number=Plur	15	nmod	O	_
 22	(	(	PUNCT	_	_	23	punct	O	_
-23	25	25	NUM	_	NumType=Card	21	appos	O	_
-24	a	a	ADP	_	_	26	case	O	_
-25	125	125	NUM	_	NumType=Card	26	nummod	O	_
-26	microns	micron	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
+23	25	25	NUM	_	NumType=Card	26	nummod	O	_
+24	a	a	CCONJ	_	_	25	cc	O	_
+25	125	125	NUM	_	NumType=Card	23	conj	O	_
+26	microns	micron	NOUN	_	Gender=Masc|Number=Plur	21	appos	O	_
 27	)	)	PUNCT	_	_	26	punct	O	_
 28	.	.	PUNCT	_	_	9	punct	O	_
 
@@ -11750,10 +11750,10 @@
 12	dispersão	dispersão	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 13	composta	compor	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	12	acl	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	50	50	NUM	_	NumType=Card	13	obl	O	_
-16	a	a	ADP	_	_	18	case	O	_
-17	70	70	NUM	_	NumType=Card	18	nummod	O	_
-18	%	%	SYM	_	_	15	nmod	O	_
+15	50	50	NUM	_	NumType=Card	18	nummod	O	_
+16	a	a	CCONJ	_	_	17	cc	O	_
+17	70	70	NUM	_	NumType=Card	15	conj	O	_
+18	%	%	SYM	_	_	13	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	cimento	cimento	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
 21	asfáltico	asfáltico	ADJ	_	Gender=Masc|Number=Sing	20	amod	O	_
@@ -11764,8 +11764,8 @@
 26	emulsificante	emulsificante	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 27	(	(	PUNCT	_	_	31	punct	O	_
 28	1	1	NUM	_	NumType=Card	31	nummod	O	_
-29	a	a	ADP	_	_	30	case	O	_
-30	2	2	NUM	_	NumType=Card	28	nummod	O	_
+29	a	a	CCONJ	_	_	30	cc	O	_
+30	2	2	NUM	_	NumType=Card	28	conj	O	_
 31	%	%	SYM	_	_	24	appos	O	_
 32	)	)	PUNCT	_	_	31	punct	O	_
 33	e	e	CCONJ	_	_	35	cc	O	_
@@ -11859,10 +11859,10 @@
 25	tamanho	tamanho	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
 26	coloidal	coloidal	ADJ	_	Gender=Masc|Number=Sing	25	amod	O	_
 27	(	(	PUNCT	_	_	28	punct	O	_
-28	5	5	NUM	_	NumType=Card	25	appos	O	_
-29	a	a	ADP	_	_	31	case	O	_
-30	10	10	NUM	_	NumType=Card	31	nummod	O	_
-31	micra	micra	NOUN	_	Gender=Masc|Number=Sing	28	nmod	O	_
+28	5	5	NUM	_	NumType=Card	31	nummod	O	_
+29	a	a	CCONJ	_	_	30	cc	O	_
+30	10	10	NUM	_	NumType=Card	28	conj	O	_
+31	micra	micra	NOUN	_	Gender=Masc|Number=Sing	25	appos	O	_
 32	ou	ou	CCONJ	_	_	33	cc	O	_
 33	menor	pequeno	ADJ	_	Gender=Masc|Number=Sing	31	conj	O	_
 34	)	)	PUNCT	_	_	28	punct	O	_
@@ -15319,8 +15319,8 @@
 37	pH	ph	NOUN	_	Gender=Masc|Number=Sing	35	nmod	O	_
 38	de	de	ADP	_	_	39	case	O	_
 39	1	1	NUM	_	NumType=Card	37	nummod	O	_
-40	a	a	ADP	_	_	41	case	O	_
-41	14	14	NUM	_	NumType=Card	39	nmod	O	_
+40	a	a	CCONJ	_	_	41	cc	O	_
+41	14	14	NUM	_	NumType=Card	39	conj	O	_
 42	.	.	PUNCT	_	_	12	punct	O	_
 
 # text = O potenciômetro utilizado na pesquisa foi o Tec -3MP-Tecnal.
@@ -16140,15 +16140,15 @@
 18	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	14	obl	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	1	1	NUM	_	NumType=Card	18	nummod	O	_
-21	a	a	ADP	_	_	22	case	O	_
-22	7	7	NUM	_	NumType=Card	20	nmod	O	_
+21	a	a	CCONJ	_	_	22	cc	O	_
+22	7	7	NUM	_	NumType=Card	20	conj	O	_
 23	,	,	PUNCT	_	_	14	punct	O	_
 24	mais	mais	ADV	_	_	25	advmod	O	_
 25	especificamente	especificamente	ADV	_	_	14	advmod	O	_
 26	entre	entre	ADP	_	_	27	case	O	_
 27	2	2	NUM	_	NumType=Card	25	obl	O	_
-28	a	a	ADP	_	_	29	case	O	_
-29	4	4	NUM	_	NumType=Card	27	nmod	O	_
+28	a	a	CCONJ	_	_	29	cc	O	_
+29	4	4	NUM	_	NumType=Card	27	conj	O	_
 30	.	.	PUNCT	_	_	13	punct	O	_
 
 # text = De acordo com este mesmo autor, a dosagem de ácido adicionado a emulsão é o fator determinante no valor final do pH da emulsão.
@@ -18367,10 +18367,10 @@
 14	período	período	NOUN	_	Gender=Masc|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	04	04	NUM	_	NumType=Card	14	nummod	O	_
-17	a	a	ADP	_	_	18	case	O	_
-18	13	13	NUM	_	NumType=Card	16	nmod	O	_
+17	a	a	CCONJ	_	_	18	cc	O	_
+18	13	13	NUM	_	NumType=Card	16	conj	O	_
 19	de	de	ADP	_	_	20	case	O	_
-20	julho	julho	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
+20	julho	julho	NOUN	_	Gender=Masc|Number=Sing	16	nmod	O	_
 21	de	de	ADP	_	_	22	case	O	_
 22	2013	2013	NUM	_	NumType=Card	20	nummod	O	_
 23	,	,	PUNCT	_	_	24	punct	O	_
@@ -23695,10 +23695,10 @@
 7	normalmente	normalmente	ADV	_	_	8	advmod	O	_
 8	variando	variar	VERB	_	VerbForm=Ger	3	advcl	O	_
 9	de	de	ADP	_	_	10	case	O	_
-10	40	40	NUM	_	NumType=Card	8	obl:arg	O	_
-11	a	a	ADP	_	_	13	case	O	_
-12	60	60	NUM	_	NumType=Card	13	nummod	O	_
-13	cm	cm	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
+10	40	40	NUM	_	NumType=Card	13	nummod	O	_
+11	a	a	CCONJ	_	_	12	cc	O	_
+12	60	60	NUM	_	NumType=Card	10	conj	O	_
+13	cm	cm	NOUN	_	Gender=Masc|Number=Plur	8	obl:arg	O	_
 14	.	.	PUNCT	_	_	3	punct	O	_
 
 # text = A base é irregular, em contato abrupto com a fácies sotoposta, indicativa de corte e preenchimento.
@@ -24183,9 +24183,9 @@
 10	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	ordem	ordem	NOUN	_	Gender=Fem|Number=Sing	8	obl	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	1	1	NUM	_	NumType=Card	11	nummod	O	_
-14	a	a	ADP	_	_	16	case	O	_
-15	2	2	NUM	_	NumType=Card	16	nummod	O	_
+13	1	1	NUM	_	NumType=Card	16	nummod	O	_
+14	a	a	CCONJ	_	_	15	cc	O	_
+15	2	2	NUM	_	NumType=Card	13	conj	O	_
 16	centímetros	centímetro	NOUN	_	Gender=Masc|Number=Plur	11	nmod	O	_
 17	(	(	PUNCT	_	_	18	punct	O	_
 18	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	6	parataxis	O	_
@@ -24454,8 +24454,8 @@
 24	(	(	PUNCT	_	_	26	punct	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	2	2	NUM	_	NumType=Card	29	nummod	O	_
-27	a	a	ADP	_	_	29	case	O	_
-28	5	5	NUM	_	NumType=Card	29	nummod	O	_
+27	a	a	CCONJ	_	_	28	cc	O	_
+28	5	5	NUM	_	NumType=Card	26	conj	O	_
 29	cm	cm	NOUN	_	Gender=Masc|Number=Plur	22	nmod:appos	O	_
 30	)	)	PUNCT	_	_	29	punct	O	_
 31	de	de	ADP	_	_	32	case	O	_
@@ -25507,10 +25507,10 @@
 7	arredondadas	arredondar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	5	acl	O	_
 8	,	,	PUNCT	_	_	10	punct	O	_
 9	de	de	ADP	_	_	10	case	O	_
-10	9	9	NUM	_	NumType=Card	5	nummod	O	_
-11	a	a	ADP	_	_	13	case	O	_
-12	20	20	NUM	_	NumType=Card	13	nummod	O	_
-13	cm	cm	NOUN	_	Gender=Masc|Number=Plur	10	nmod	O	_
+10	9	9	NUM	_	NumType=Card	13	nummod	O	_
+11	a	a	CCONJ	_	_	12	cc	O	_
+12	20	20	NUM	_	NumType=Card	10	conj	O	_
+13	cm	cm	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 14	(	(	PUNCT	_	_	15	punct	O	_
 15	Figura	figura	NOUN	_	Gender=Fem|Number=Sing	21	parataxis	O	_
 16	22	22	NUM	_	NumType=Card	15	nummod	O	_
@@ -43374,10 +43374,10 @@
 22	de	de	ADP	_	_	23	case	O	_
 23	água	água	NOUN	_	Gender=Fem|Number=Sing	21	nmod	B=TIPO_FLUIDO	_
 24	de	de	ADP	_	_	25	case	O	_
-25	10	10	NUM	_	NumType=Card	21	nummod	O	_
-26	a	a	ADP	_	_	28	case	O	_
-27	45	45	NUM	_	NumType=Card	28	nummod	O	_
-28	%	%	SYM	_	_	25	nmod	O	_
+25	10	10	NUM	_	NumType=Card	28	nummod	O	_
+26	a	a	CCONJ	_	_	27	cc	O	_
+27	45	45	NUM	_	NumType=Card	25	conj	O	_
+28	%	%	SYM	_	_	21	nmod	O	_
 29	)	)	PUNCT	_	_	21	punct	O	_
 30	.	.	PUNCT	_	_	3	punct	O	_
 
@@ -47493,9 +47493,9 @@
 6	cátions	cátion	NOUN	_	Gender=Masc|Number=Plur	4	nmod	O	_
 7	varia	variar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	de	de	ADP	_	_	9	case	O	_
-9	80	80	NUM	_	NumType=Card	7	obl	O	_
-10	a	a	ADP	_	_	12	case	O	_
-11	150	150	NUM	_	NumType=Card	12	nummod	O	_
+9	80	80	NUM	_	NumType=Card	12	nummod	O	_
+10	a	a	CCONJ	_	_	11	cc	O	_
+11	150	150	NUM	_	NumType=Card	9	conj	O	_
 12	meq	meq	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
 13	por	por	ADP	_	_	14	case	O	_
 14	100g	100g	NOUN	_	Gender=Masc|Number=Plur	7	obl	O	_
@@ -47503,38 +47503,38 @@
 16	esmectita	esmectita	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 17	,	,	PUNCT	_	_	7	punct	O	_
 18	de	de	ADP	_	_	19	case	O	_
-19	3	3	NUM	_	NumType=Card	9	conj	O	_
-20	a	a	ADP	_	_	22	case	O	_
-21	15	15	NUM	_	NumType=Card	22	nummod	O	_
-22	meq	meq	NOUN	_	Gender=Masc|Number=Plur	19	nmod	O	_
+19	3	3	NUM	_	NumType=Card	22	nummod	O	_
+20	a	a	CCONJ	_	_	21	cc	O	_
+21	15	15	NUM	_	NumType=Card	19	conj	O	_
+22	meq	meq	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 23	por	por	ADP	_	_	24	case	O	_
 24	100g	100g	NOUN	_	Gender=Masc|Number=Plur	19	nmod	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	caulinita	caulinita	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 27	,	,	PUNCT	_	_	29	punct	O	_
 28	de	de	ADP	_	_	29	case	O	_
-29	5	5	NUM	_	NumType=Card	9	conj	O	_
-30	a	a	ADP	_	_	32	case	O	_
-31	10	10	NUM	_	NumType=Card	32	nummod	O	_
-32	meq	meq	NOUN	_	Gender=Masc|Number=Plur	29	nmod	O	_
+29	5	5	NUM	_	NumType=Card	32	nummod	O	_
+30	a	a	CCONJ	_	_	31	cc	O	_
+31	10	10	NUM	_	NumType=Card	29	conj	O	_
+32	meq	meq	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 33	de	de	ADP	_	_	34	case	O	_
 34	haloisita-2H2O	haloisita-2H2O	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
 35	,	,	PUNCT	_	_	37	punct	O	_
 36	de	de	ADP	_	_	37	case	O	_
-37	10	10	NUM	_	NumType=Card	9	conj	O	_
-38	a	a	ADP	_	_	40	case	O	_
-39	40	40	NUM	_	NumType=Card	40	nummod	O	_
-40	meq	meq	NOUN	_	Gender=Masc|Number=Plur	37	nmod	O	_
+37	10	10	NUM	_	NumType=Card	40	nummod	O	_
+38	a	a	CCONJ	_	_	39	cc	O	_
+39	40	40	NUM	_	NumType=Card	37	conj	O	_
+40	meq	meq	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 41	por	por	ADP	_	_	42	case	O	_
 42	100g	100g	NOUN	_	Gender=Masc|Number=Plur	40	nmod	O	_
 43	de	de	ADP	_	_	44	case	O	_
 44	haloisita-4H2O	haloisita-4H2O	NOUN	_	Gender=Masc|Number=Sing	42	nmod	O	_
 45	,	,	PUNCT	_	_	47	punct	O	_
 46	de	de	ADP	_	_	47	case	O	_
-47	10	10	NUM	_	NumType=Card	9	conj	O	_
-48	a	a	ADP	_	_	50	case	O	_
-49	40	40	NUM	_	NumType=Card	50	nummod	O	_
-50	meq	meq	NOUN	_	Gender=Masc|Number=Plur	47	nmod	O	_
+47	10	10	NUM	_	NumType=Card	50	nummod	O	_
+48	a	a	CCONJ	_	_	49	cc	O	_
+49	40	40	NUM	_	NumType=Card	47	conj	O	_
+50	meq	meq	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 51	por	por	ADP	_	_	52	case	O	_
 52	100g	100g	NOUN	_	Gender=Masc|Number=Plur	50	nmod	O	_
 53	de	de	ADP	_	_	54	case	O	_
@@ -47543,10 +47543,10 @@
 56	clorita	clorita	NOUN	_	Gender=Fem|Number=Sing	54	conj	O	_
 57	e	e	CCONJ	_	_	59	cc	O	_
 58	de	de	ADP	_	_	59	case	O	_
-59	100	100	NUM	_	NumType=Card	9	conj	O	_
-60	a	a	ADP	_	_	62	case	O	_
-61	150	150	NUM	_	NumType=Card	62	nummod	O	_
-62	meq	meq	NOUN	_	Gender=Masc|Number=Plur	59	nmod	O	_
+59	100	100	NUM	_	NumType=Card	62	nummod	O	_
+60	a	a	CCONJ	_	_	61	cc	O	_
+61	150	150	NUM	_	NumType=Card	59	conj	O	_
+62	meq	meq	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 63	por	por	ADP	_	_	64	case	O	_
 64	100g	100g	NOUN	_	Gender=Masc|Number=Plur	62	nmod	O	_
 65	de	de	ADP	_	_	66	case	O	_
@@ -58766,10 +58766,10 @@
 15	semi-úmido	semi-úmido	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	,	,	PUNCT	_	_	18	punct	O	_
 17	com	com	ADP	_	_	18	case	O	_
-18	4	4	NUM	_	NumType=Card	14	nummod	O	_
-19	a	a	ADP	_	_	21	case	O	_
-20	5	5	NUM	_	NumType=Card	21	nummod	O	_
-21	meses	mês	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
+18	4	4	NUM	_	NumType=Card	21	nummod	O	_
+19	a	a	CCONJ	_	_	20	cc	O	_
+20	5	5	NUM	_	NumType=Card	18	conj	O	_
+21	meses	mês	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 22	secos	seco	ADJ	_	Gender=Masc|Number=Plur	21	amod	O	_
 23	,	,	PUNCT	_	_	18	punct	O	_
 24	com	com	ADP	_	_	25	case	O	_
@@ -61384,10 +61384,10 @@
 13	período	período	NOUN	_	Gender=Masc|Number=Sing	10	obl	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	20	20	NUM	_	NumType=Card	13	nummod	O	_
-16	a	a	ADP	_	_	17	case	O	_
-17	28	28	NUM	_	NumType=Card	13	nummod	O	_
+16	a	a	CCONJ	_	_	17	cc	O	_
+17	28	28	NUM	_	NumType=Card	15	conj	O	_
 18	de	de	ADP	_	_	19	case	O	_
-19	julho	julho	NOUN	_	Gender=Masc|Number=Sing	17	nmod	O	_
+19	julho	julho	NOUN	_	Gender=Masc|Number=Sing	15	nmod	O	_
 20	de	de	ADP	_	_	21	case	O	_
 21	2013	2013	NUM	_	NumType=Card	19	nummod	O	_
 22	.	.	PUNCT	_	_	2	punct	O	_
@@ -66082,9 +66082,9 @@
 10	geralmente	geralmente	ADV	_	_	11	advmod	O	_
 11	variam	variar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	conj	O	_
 12	de	de	ADP	_	_	13	case	O	_
-13	30	30	NUM	_	NumType=Card	11	obl	O	_
-14	a	a	ADP	_	_	16	case	O	_
-15	40	40	NUM	_	NumType=Card	16	nummod	O	_
+13	30	30	NUM	_	NumType=Card	16	nummod	O	_
+14	a	a	CCONJ	_	_	15	cc	O	_
+15	40	40	NUM	_	NumType=Card	13	conj	O	_
 16	metros	metro	NOUN	_	Gender=Masc|Number=Plur	11	obl	O	_
 17	.	.	PUNCT	_	_	3	punct	O	_
 
@@ -68386,9 +68386,9 @@
 41	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	42	det	O	_
 42	ordem	ordem	NOUN	_	Gender=Fem|Number=Sing	31	obl	O	_
 43	de	de	ADP	_	_	44	case	O	_
-44	80	80	NUM	_	NumType=Card	42	nummod	O	_
-45	a	a	ADP	_	_	47	case	O	_
-46	100	100	NUM	_	NumType=Card	47	nummod	O	_
+44	80	80	NUM	_	NumType=Card	47	nummod	O	_
+45	a	a	CCONJ	_	_	46	cc	O	_
+46	100	100	NUM	_	NumType=Card	44	conj	O	_
 47	m	m	NOUN	_	Gender=Masc|Number=Plur	42	nmod	O	_
 48	,	,	PUNCT	_	_	53	punct	O	_
 49	sendo	ser	AUX	_	VerbForm=Ger	53	aux	O	_
@@ -72042,10 +72042,10 @@
 6	e	e	CCONJ	_	_	7	cc	O	_
 7	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	5	conj	O	_
 8	(	(	PUNCT	_	_	9	punct	O	_
-9	15	15	NUM	_	NumType=Card	2	appos	O	_
-10	a	a	ADP	_	_	12	case	O	_
-11	60	60	NUM	_	NumType=Card	12	nummod	O	_
-12	%	%	SYM	_	_	9	nmod	O	_
+9	15	15	NUM	_	NumType=Card	12	nummod	O	_
+10	a	a	CCONJ	_	_	11	cc	O	_
+11	60	60	NUM	_	NumType=Card	9	conj	O	_
+12	%	%	SYM	_	_	2	appos	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 15	)	)	PUNCT	_	_	9	punct	O	_
@@ -96755,11 +96755,11 @@
 11	Iara	Iara	PROPN	_	Gender=Fem|Number=Sing	10	nmod	O	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	3	3	NUM	_	NumType=Card	11	nmod:appos	O	_
-14	a	a	ADP	_	_	15	case	O	_
-15	4	4	NUM	_	NumType=Card	13	nmod	O	_
-16	bilhões	bilhão	NUM	_	NumType=Card	15	flat	O	_
+14	a	a	CCONJ	_	_	15	cc	O	_
+15	4	4	NUM	_	NumType=Card	13	conj	O	_
+16	bilhões	bilhão	NUM	_	NumType=Card	13	flat	O	_
 17	de	de	ADP	_	_	18	case	O	_
-18	barris	barril	NOUN	_	Gender=Masc|Number=Plur	15	nmod	O	_
+18	barris	barril	NOUN	_	Gender=Masc|Number=Plur	13	nmod	O	_
 19	)	)	PUNCT	_	_	13	punct	O	_
 20	e	e	CCONJ	_	_	21	cc	O	_
 21	Baleia	Baleia	PROPN	_	Gender=Fem|Number=Sing	11	conj	O	_
@@ -97647,8 +97647,8 @@
 234	,	,	PUNCT	_	_	236	punct	O	_
 235	de	de	ADP	_	_	236	case	O	_
 236	2006	2006	NUM	_	NumType=Card	231	nmod	O	_
-237	a	a	ADP	_	_	238	case	O	_
-238	2012	2012	NUM	_	NumType=Card	236	nmod	O	_
+237	a	a	CCONJ	_	_	238	cc	O	_
+238	2012	2012	NUM	_	NumType=Card	236	conj	O	_
 239	.	.	PUNCT	_	_	4	punct	O	_
 
 # text = I.2.1 Desafios do Pré-sal
@@ -102576,11 +102576,11 @@
 35	data	datar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	27	acl:relcl	O	_
 36	de	de	ADP	_	_	37	case	O	_
 37	24	24	NUM	_	NumType=Card	35	obl	O	_
-38	a	a	ADP	_	_	39	case	O	_
-39	65	65	NUM	_	NumType=Card	37	nummod	O	_
-40	milhões	milhão	NUM	_	NumType=Card	39	flat	O	_
+38	a	a	CCONJ	_	_	39	cc	O	_
+39	65	65	NUM	_	NumType=Card	37	conj	O	_
+40	milhões	milhão	NUM	_	NumType=Card	37	flat	O	_
 41	de	de	ADP	_	_	42	case	O	_
-42	anos	ano	NOUN	_	Gender=Masc|Number=Plur	39	nmod	O	_
+42	anos	ano	NOUN	_	Gender=Masc|Number=Plur	37	nmod	O	_
 43	)	)	PUNCT	_	_	27	punct	O	_
 44	,	,	PUNCT	_	_	45	punct	O	_
 45	dando	dar	VERB	_	VerbForm=Ger	8	advcl	O	_
@@ -102703,13 +102703,13 @@
 12	Tahiti	Tahiti	PROPN	_	Gender=Masc|Number=Sing	10	nmod	B=CAMPO	_
 13	(	(	PUNCT	_	_	14	punct	O	_
 14	400	400	NUM	_	NumType=Card	12	nmod:appos	O	_
-15	a	a	ADP	_	_	16	case	O	_
-16	500	500	NUM	_	NumType=Card	14	nmod	O	_
-17	milhões	milhão	NUM	_	NumType=Card	16	flat	O	_
+15	a	a	CCONJ	_	_	16	cc	O	_
+16	500	500	NUM	_	NumType=Card	14	conj	O	_
+17	milhões	milhão	NUM	_	NumType=Card	14	flat	O	_
 18	de	de	ADP	_	_	19	case	O	_
-19	barris	barril	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
+19	barris	barril	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 20	em	em	ADP	_	_	21	case	O	_
-21	reservas	reserva	NOUN	_	Gender=Fem|Number=Plur	16	nmod	O	_
+21	reservas	reserva	NOUN	_	Gender=Fem|Number=Plur	14	nmod	O	_
 22	)	)	PUNCT	_	_	14	punct	O	_
 23	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	24	nsubj	O	_
 24	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	10	acl:relcl	O	_
@@ -105402,10 +105402,10 @@
 104	com	com	ADP	_	_	105	case	O	_
 105	profundidades	profundidade	NOUN	_	Gender=Fem|Number=Plur	102	nmod	O	_
 106	entre	entre	ADP	_	_	107	case	O	_
-107	1000	1000	NUM	_	NumType=Card	105	nummod	O	_
-108	a	a	ADP	_	_	110	case	O	_
-109	2000	2000	NUM	_	NumType=Card	110	nummod	O	_
-110	metros	metro	NOUN	_	Gender=Masc|Number=Plur	107	nmod	O	_
+107	1000	1000	NUM	_	NumType=Card	110	nummod	O	_
+108	a	a	CCONJ	_	_	109	cc	O	_
+109	2000	2000	NUM	_	NumType=Card	107	conj	O	_
+110	metros	metro	NOUN	_	Gender=Masc|Number=Plur	105	nmod	O	_
 111	;	;	PUNCT	_	_	113	punct	O	_
 112	•	•	PUNCT	_	_	113	punct	O	_
 113	FASE	FASE	PROPN	_	Gender=Fem|Number=Sing	94	appos	O	_
@@ -107390,10 +107390,10 @@
 8	de	de	ADP	_	_	9	mark	O	_
 9	separar	separar	VERB	_	VerbForm=Inf	7	acl	O	_
 10	de	de	ADP	_	_	11	case	O	_
-11	50	50	NUM	_	NumType=Card	9	obl:arg	O	_
-12	a	a	ADP	_	_	14	case	O	_
-13	500	500	NUM	_	NumType=Card	14	nummod	O	_
-14	kg	kg	NOUN	_	Gender=Masc|Number=Plur	9	obl	O	_
+11	50	50	NUM	_	NumType=Card	14	nummod	O	_
+12	a	a	CCONJ	_	_	13	cc	O	_
+13	500	500	NUM	_	NumType=Card	11	conj	O	_
+14	kg	kg	NOUN	_	Gender=Masc|Number=Plur	9	obl:arg	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	areia	areia	NOUN	_	Gender=Fem|Number=Sing	14	nmod	B=LITOLOGIA	_
 17	por	por	ADP	_	_	18	case	O	_
@@ -109791,10 +109791,10 @@
 13	em	em	ADP	_	_	14	case	O	_
 14	profundidades	profundidade	NOUN	_	Gender=Fem|Number=Plur	5	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	250	250	NUM	_	NumType=Card	14	nummod	O	_
-17	a	a	ADP	_	_	19	case	O	_
-18	345	345	NUM	_	NumType=Card	19	nummod	O	_
-19	metros	metro	NOUN	_	Gender=Masc|Number=Plur	16	nmod	O	_
+16	250	250	NUM	_	NumType=Card	19	nummod	O	_
+17	a	a	CCONJ	_	_	18	cc	O	_
+18	345	345	NUM	_	NumType=Card	16	conj	O	_
+19	metros	metro	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 20	.	.	PUNCT	_	_	5	punct	O	_
 
 # text = O projeto conta também com um duto de retorno do CO2 separado no continente para ser reinjetado (Figura 32). (Fonte: KLEMP – Statoil)
@@ -125015,8 +125015,8 @@
 34	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	31	nmod	O	_
 35	de	de	ADP	_	_	36	case	O	_
 36	8	8	NUM	_	NumType=Card	34	nummod	O	_
-37	a	a	ADP	_	_	38	case	O	_
-38	14	14	NUM	_	NumType=Card	36	nmod	O	_
+37	a	a	CCONJ	_	_	38	cc	O	_
+38	14	14	NUM	_	NumType=Card	36	conj	O	_
 39	,	,	PUNCT	_	_	40	punct	O	_
 40	favorecem	favorecer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	24	ccomp	O	_
 41	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	42	det	O	_
@@ -125448,9 +125448,9 @@
 5	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	6	det	O	_
 6	pontos	ponto	NOUN	_	Gender=Masc|Number=Plur	20	obl	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	10	10	NUM	_	NumType=Card	6	nummod	O	_
-9	a	a	ADP	_	_	11	case	O	_
-10	65	65	NUM	_	NumType=Card	11	nummod	O	_
+8	10	10	NUM	_	NumType=Card	11	nummod	O	_
+9	a	a	CCONJ	_	_	10	cc	O	_
+10	65	65	NUM	_	NumType=Card	8	conj	O	_
 11	%	%	SYM	_	_	6	nmod	O	_
 12	de	de	ADP	_	_	13	case	O	_
 13	volume	volume	NOUN	_	Gender=Masc|Number=Sing	11	nmod	O	_
@@ -127265,10 +127265,10 @@
 5	representam	representar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	cerca	cerca	ADV	_	_	8	advmod	O	_
 7	de	de	ADP	_	_	6	fixed	O	_
-8	15	15	NUM	_	NumType=Card	5	obj	O	_
-9	a	a	ADP	_	_	11	case	O	_
-10	20	20	NUM	_	NumType=Card	11	nummod	O	_
-11	%	%	SYM	_	_	8	nmod	O	_
+8	15	15	NUM	_	NumType=Card	11	nummod	O	_
+9	a	a	CCONJ	_	_	10	cc	O	_
+10	20	20	NUM	_	NumType=Card	8	conj	O	_
+11	%	%	SYM	_	_	5	obj	O	_
 12-13	do	_	_	_	_	_	_	_	_
 12	de	de	ADP	_	_	14	case	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
@@ -127286,10 +127286,10 @@
 24	bastante	bastante	ADV	_	_	25	advmod	O	_
 25	amplos	amplo	ADJ	_	Gender=Masc|Number=Plur	23	amod	O	_
 26	(	(	PUNCT	_	_	27	punct	O	_
-27	3	3	NUM	_	NumType=Card	23	appos	O	_
-28	a	a	ADP	_	_	30	case	O	_
-29	35	35	NUM	_	NumType=Card	30	nummod	O	_
-30	%	%	SYM	_	_	27	nmod	O	_
+27	3	3	NUM	_	NumType=Card	30	nummod	O	_
+28	a	a	CCONJ	_	_	29	cc	O	_
+29	35	35	NUM	_	NumType=Card	27	conj	O	_
+30	%	%	SYM	_	_	23	appos	O	_
 31	)	)	PUNCT	_	_	27	punct	O	_
 32	.	.	PUNCT	_	_	5	punct	O	_
 
@@ -127757,7 +127757,7 @@
 5	constituída	constituir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	3	3	NUM	_	NumType=Card	5	obl:arg	O	_
-8	a	a	ADP	_	_	9	case	O	_
+8	a	a	CCONJ	_	_	9	cc	O	_
 9	10	10	NUM	_	NumType=Card	7	conj	O	_
 10	ou	ou	CCONJ	_	_	12	cc	O	_
 11	mais	mais	DET	_	Gender=Masc|Number=Plur|PronType=Ind	12	det	O	_
@@ -128146,9 +128146,9 @@
 7	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	6	obj	O	_
 9	de	de	ADP	_	_	10	case	O	_
-10	100	100	NUM	_	NumType=Card	8	nummod	O	_
-11	a	a	ADP	_	_	13	case	O	_
-12	10000	10000	NUM	_	NumType=Card	13	nummod	O	_
+10	100	100	NUM	_	NumType=Card	13	nummod	O	_
+11	a	a	CCONJ	_	_	12	cc	O	_
+12	10000	10000	NUM	_	NumType=Card	10	conj	O	_
 13	centipoises	centipoise	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
 14	em	em	ADP	_	_	15	case	O	_
 15	condições	condição	NOUN	_	Gender=Fem|Number=Plur	6	obl	O	_
@@ -128499,10 +128499,10 @@
 4	asfaltenos	asfalteno	NOUN	_	Gender=Masc|Number=Plur	2	conj	O	_
 5	somam	somar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	de	de	ADP	_	_	7	case	O	_
-7	25	25	NUM	_	NumType=Card	5	obl	O	_
-8	a	a	ADP	_	_	10	case	O	_
-9	70	70	NUM	_	NumType=Card	10	nummod	O	_
-10	%	%	SYM	_	_	7	nmod	O	_
+7	25	25	NUM	_	NumType=Card	10	nummod	O	_
+8	a	a	CCONJ	_	_	9	cc	O	_
+9	70	70	NUM	_	NumType=Card	7	conj	O	_
+10	%	%	SYM	_	_	5	obl	O	_
 11-12	da	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
@@ -129463,7 +129463,7 @@
 15	apenas	apenas	ADV	_	_	17	advmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	3	3	NUM	_	NumType=Card	12	obl	O	_
-18	a	a	ADP	_	_	19	case	O	_
+18	a	a	CCONJ	_	_	19	cc	O	_
 19	25	25	NUM	_	NumType=Card	17	conj	O	_
 20	%	%	SYM	_	_	17	nmod	O	_
 21-22	da	_	_	_	_	_	_	_	_
@@ -147170,10 +147170,10 @@
 18	,	,	PUNCT	_	_	13	punct	O	_
 19	responde	responder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	2	ccomp	O	_
 20	por	por	ADP	_	_	21	case	O	_
-21	75	75	NUM	_	NumType=Card	19	obl:arg	O	_
-22	a	a	ADP	_	_	24	case	O	_
-23	80	80	NUM	_	NumType=Card	24	nummod	O	_
-24	%	%	SYM	_	_	21	nmod	O	_
+21	75	75	NUM	_	NumType=Card	24	nummod	O	_
+22	a	a	CCONJ	_	_	23	cc	O	_
+23	80	80	NUM	_	NumType=Card	21	conj	O	_
+24	%	%	SYM	_	_	19	obl:arg	O	_
 25-26	do	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
@@ -158411,9 +158411,9 @@
 22	de	de	ADP	_	_	23	case	O	_
 23	reação	reação	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
 24	de	de	ADP	_	_	25	case	O	_
-25	4	4	NUM	_	NumType=Card	23	nummod	O	_
-26	a	a	ADP	_	_	28	case	O	_
-27	5	5	NUM	_	NumType=Card	28	nummod	O	_
+25	4	4	NUM	_	NumType=Card	28	nummod	O	_
+26	a	a	CCONJ	_	_	27	cc	O	_
+27	5	5	NUM	_	NumType=Card	25	conj	O	_
 28	horas	hora	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
 29	de	de	ADP	_	_	30	case	O	_
 30	reação	reação	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
@@ -164604,17 +164604,17 @@
 # sent_id = 391-20160914-MONOGRAFIA_0-43
 1	pH	ph	NOUN	_	Gender=Masc|Number=Sing	15	nsubj	O	_
 2	de	de	ADP	_	_	3	case	O	_
-3	4	4	NUM	_	NumType=Card	1	nummod	O	_
-4	a	a	ADP	_	_	9	case	O	_
-5	9	9	NUM	_	NumType=Card	9	nummod	O	_
+3	4	4	NUM	_	NumType=Card	9	nummod	O	_
+4	a	a	CCONJ	_	_	5	cc	O	_
+5	9	9	NUM	_	NumType=Card	3	conj	O	_
 6	→	→	PUNCT	_	_	9	punct	O	_
 7	não	não	ADV	_	Polarity=Neg	8	advmod	O	_
 8	iônicos	iônico	ADJ	_	Gender=Masc|Number=Plur	9	amod	O	_
 9	pH	ph	NOUN	_	Gender=Masc|Number=Sing	1	nmod	O	_
 10	de	de	ADP	_	_	11	case	O	_
 11	9	9	NUM	_	NumType=Card	9	nummod	O	_
-12	a	a	ADP	_	_	13	case	O	_
-13	10	10	NUM	_	NumType=Card	11	nmod	O	_
+12	a	a	CCONJ	_	_	13	cc	O	_
+13	10	10	NUM	_	NumType=Card	11	conj	O	_
 14	→	→	PUNCT	_	_	9	punct	O	_
 15	atua	atuar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	como	como	ADP	_	_	17	case	O	_
@@ -165448,8 +165448,8 @@
 15	variam	variar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	11	acl:relcl	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	1	1	NUM	_	NumType=Card	15	obl	O	_
-18	a	a	ADP	_	_	19	case	O	_
-19	18	18	NUM	_	NumType=Card	17	nmod	O	_
+18	a	a	CCONJ	_	_	19	cc	O	_
+19	18	18	NUM	_	NumType=Card	17	conj	O	_
 20	,	,	PUNCT	_	_	21	punct	O	_
 21	aumentando	aumentar	VERB	_	VerbForm=Ger	6	advcl	O	_
 22	estes	este	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	21	obj	O	_
@@ -165501,8 +165501,8 @@
 6	vai	ir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	advcl	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	3	3	NUM	_	NumType=Card	6	obl	O	_
-9	a	a	ADP	_	_	10	case	O	_
-10	9	9	NUM	_	NumType=Card	8	nmod	O	_
+9	a	a	CCONJ	_	_	10	cc	O	_
+10	9	9	NUM	_	NumType=Card	8	conj	O	_
 11	,	,	PUNCT	_	_	12	punct	O	_
 12	possuindo	possuir	VERB	_	VerbForm=Ger	6	advcl	O	_
 13	características	característica	NOUN	_	Gender=Fem|Number=Plur	12	obj	O	_
@@ -166593,9 +166593,9 @@
 20	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	21	det	O	_
 21	ordem	ordem	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 22	de	de	ADP	_	_	23	case	O	_
-23	50	50	NUM	_	NumType=Card	21	nummod	O	_
-24	a	a	ADP	_	_	26	case	O	_
-25	100	100	NUM	_	NumType=Card	26	nummod	O	_
+23	50	50	NUM	_	NumType=Card	26	nummod	O	_
+24	a	a	CCONJ	_	_	25	cc	O	_
+25	100	100	NUM	_	NumType=Card	23	conj	O	_
 26	%	%	SYM	_	_	21	nmod	O	_
 27	em	em	ADP	_	_	28	case	O	_
 28	volume	volume	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
@@ -173998,8 +173998,8 @@
 6	variam	variar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 7	de	de	ADP	_	_	8	case	O	_
 8	3	3	NUM	_	NumType=Card	6	obl	O	_
-9	a	a	ADP	_	_	10	case	O	_
-10	10	10	NUM	_	NumType=Card	8	nmod	O	_
+9	a	a	CCONJ	_	_	10	cc	O	_
+10	10	10	NUM	_	NumType=Card	8	conj	O	_
 
 # text = Ǻ conforme o tipo de estrutura, o teor de alumínio e o cátion de compensação, originando assim uma estrutura microporosa [17].
 # sent_id = 25-20150123-MONOGRAFIA_0-117
@@ -176835,10 +176835,10 @@
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
 23	região	região	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 24	de	de	ADP	_	_	25	case	O	_
-25	4000	4000	NUM	_	NumType=Card	23	nummod	O	_
-26	a	a	ADP	_	_	28	case	O	_
-27	400	400	NUM	_	NumType=Card	28	nummod	O	_
-28	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Plur	25	nmod	O	_
+25	4000	4000	NUM	_	NumType=Card	28	nummod	O	_
+26	a	a	CCONJ	_	_	27	cc	O	_
+27	400	400	NUM	_	NumType=Card	25	conj	O	_
+28	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 29	.	.	PUNCT	_	_	11	punct	O	_
 
 # text = Com estas medidas é possível estudar as bandas relativas às vibrações da estrutura da zeólita, e assim determinar a sua razão sílica/alumina (SAR).
@@ -179481,9 +179481,9 @@
 9	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	10	det	O	_
 10	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 11	de	de	ADP	_	_	12	case	O	_
-12	1400	1400	NUM	_	NumType=Card	10	nummod	O	_
-13	a	a	ADP	_	_	15	case	O	_
-14	600	600	NUM	_	NumType=Card	15	nummod	O	_
+12	1400	1400	NUM	_	NumType=Card	15	nummod	O	_
+13	a	a	CCONJ	_	_	14	cc	O	_
+14	600	600	NUM	_	NumType=Card	12	conj	O	_
 15	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Sing	10	nmod	O	_
 16-17	das	_	_	_	_	_	_	_	_
 16	de	de	ADP	_	_	18	case	O	_
@@ -179726,9 +179726,9 @@
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
 14	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 15	de	de	ADP	_	_	16	case	O	_
-16	1400	1400	NUM	_	NumType=Card	14	nummod	O	_
-17	a	a	ADP	_	_	19	case	O	_
-18	600	600	NUM	_	NumType=Card	19	nummod	O	_
+16	1400	1400	NUM	_	NumType=Card	19	nummod	O	_
+17	a	a	CCONJ	_	_	18	cc	O	_
+18	600	600	NUM	_	NumType=Card	16	conj	O	_
 19	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Plur	14	nmod	O	_
 20	.	.	PUNCT	_	_	7	punct	O	_
 
@@ -179973,10 +179973,10 @@
 16	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	17	det	O	_
 17	zeólitas	zeólita	NOUN	_	Gender=Fem|Number=Plur	14	nmod	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
-19	3850	3850	NUM	_	NumType=Card	17	appos	O	_
-20	a	a	ADP	_	_	22	case	O	_
-21	3350	3350	NUM	_	NumType=Card	22	nummod	O	_
-22	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Plur	19	nmod	O	_
+19	3850	3850	NUM	_	NumType=Card	22	nummod	O	_
+20	a	a	CCONJ	_	_	21	cc	O	_
+21	3350	3350	NUM	_	NumType=Card	19	conj	O	_
+22	cm-1	cm-1	NOUN	_	Gender=Masc|Number=Plur	17	appos	O	_
 23	,	,	PUNCT	_	_	19	punct	O	_
 24	estão	estar	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	25	aux:pass	O	_
 25	mostradas	mostrar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
@@ -187399,8 +187399,8 @@
 19	diabásio	diabásio	NOUN	_	Gender=Fem|Number=Sing	17	nmod	B=LITOLOGIA	_
 20	de	de	ADP	_	_	21	case	O	_
 21	90	90	NUM	_	NumType=Card	19	nummod	O	_
-22	a	a	ADP	_	_	23	case	O	_
-23	80	80	NUM	_	NumType=Card	21	nmod	O	_
+22	a	a	CCONJ	_	_	23	cc	O	_
+23	80	80	NUM	_	NumType=Card	21	conj	O	_
 
 # text = Ma de idade.
 # sent_id = 20-20140904-TESEDSC_0-107
@@ -187873,9 +187873,9 @@
 20	em	em	ADP	_	_	23	case	O	_
 21	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	23	det	O	_
 22	primeiros	primeiro	ADJ	_	Gender=Masc|Number=Plur|NumType=Ord	23	amod	O	_
-23	100	100	NUM	_	NumType=Card	29	nmod	O	_
-24	a	a	ADP	_	_	26	case	O	_
-25	200	200	NUM	_	NumType=Card	26	nummod	O	_
+23	100	100	NUM	_	NumType=Card	26	nmod	O	_
+24	a	a	CCONJ	_	_	25	cc	O	_
+25	200	200	NUM	_	NumType=Card	23	conj	O	_
 26	km	km	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 27	,	,	PUNCT	_	_	29	punct	O	_
 28	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -189172,10 +189172,10 @@
 22	em	em	ADP	_	_	23	case	O	_
 23	média	média	NOUN	_	Gender=Fem|Number=Sing	21	obl	O	_
 24	entre	entre	ADP	_	_	25	case	O	_
-25	1200	1200	NUM	_	NumType=Card	21	obl	O	_
-26	a	a	ADP	_	_	28	case	O	_
-27	1400	1400	NUM	_	NumType=Card	28	nummod	O	_
-28	metros	metro	NOUN	_	Gender=Masc|Number=Plur	25	nmod	O	_
+25	1200	1200	NUM	_	NumType=Card	28	nummod	O	_
+26	a	a	CCONJ	_	_	27	cc	O	_
+27	1400	1400	NUM	_	NumType=Card	25	conj	O	_
+28	metros	metro	NOUN	_	Gender=Masc|Number=Plur	21	obl	O	_
 29	de	de	ADP	_	_	30	case	O	_
 30	altitude	altitude	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	.	.	PUNCT	_	_	10	punct	O	_
@@ -192996,10 +192996,10 @@
 25	desníveis	desnível	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
 26	variando	variar	VERB	_	VerbForm=Ger	25	acl	O	_
 27	de	de	ADP	_	_	28	case	O	_
-28	1200	1200	NUM	_	NumType=Card	26	obl	O	_
-29	a	a	ADP	_	_	31	case	O	_
-30	20	20	NUM	_	NumType=Card	31	nummod	O	_
-31	m	m	NOUN	_	Gender=Masc|Number=Plur	28	nmod	O	_
+28	1200	1200	NUM	_	NumType=Card	31	nummod	O	_
+29	a	a	CCONJ	_	_	30	cc	O	_
+30	20	20	NUM	_	NumType=Card	28	conj	O	_
+31	m	m	NOUN	_	Gender=Masc|Number=Plur	26	obl	O	_
 32	.	.	PUNCT	_	_	8	punct	O	_
 
 # text = Rifte Marítimo : Devido à pequena quantidade de dados sísmicos existentes próximos à costa, Zalán e Oliveira (2005) lançaram mão de dados potenciais como principal ferramenta na investigação do embasamento adjacente às bacias de Campos e Santos.
@@ -196260,10 +196260,10 @@
 5	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	6	det	O	_
 6	isobatimétricas	isobatimétrica	NOUN	_	Gender=Fem|Number=Plur	3	obl	O	_
 7	de	de	ADP	_	_	8	case	O	_
-8	100	100	NUM	_	NumType=Card	6	nummod	O	_
-9	a	a	ADP	_	_	11	case	O	_
-10	200	200	NUM	_	NumType=Card	11	nummod	O	_
-11	metros	metro	NOUN	_	Gender=Masc|Number=Plur	8	nmod	O	_
+8	100	100	NUM	_	NumType=Card	11	nummod	O	_
+9	a	a	CCONJ	_	_	10	cc	O	_
+10	200	200	NUM	_	NumType=Card	8	conj	O	_
+11	metros	metro	NOUN	_	Gender=Masc|Number=Plur	6	nmod	O	_
 12	(	(	PUNCT	_	_	13	punct	O	_
 13	quebra	quebra	NOUN	_	Gender=Fem|Number=Sing	11	appos	O	_
 14-15	da	_	_	_	_	_	_	_	_
@@ -267324,7 +267324,7 @@
 33	,	,	PUNCT	_	_	35	punct	O	_
 34	de	de	ADP	_	_	35	case	O	_
 35	7	7	NUM	_	NumType=Card	24	obl	O	_
-36	a	a	ADP	_	_	37	case	O	_
+36	a	a	CCONJ	_	_	37	cc	O	_
 37	10	10	NUM	_	NumType=Card	35	conj	O	_
 38	.	.	PUNCT	_	_	12	punct	O	_
 
@@ -270852,9 +270852,9 @@
 19	de	de	ADP	_	_	20	case	O	_
 20	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
 21	de	de	ADP	_	_	22	case	O	_
-22	5	5	NUM	_	NumType=Card	20	nummod	O	_
-23	a	a	ADP	_	_	25	case	O	_
-24	1010	1010	NUM	_	NumType=Card	25	nummod	O	_
+22	5	5	NUM	_	NumType=Card	25	nummod	O	_
+23	a	a	CCONJ	_	_	24	cc	O	_
+24	1010	1010	NUM	_	NumType=Card	22	conj	O	_
 25	s-1	s-1	NOUN	_	Gender=Masc|Number=Plur	20	nmod	O	_
 26	por	por	ADP	_	_	28	case	O	_
 27	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
@@ -270892,10 +270892,10 @@
 25	de	de	ADP	_	_	26	case	O	_
 26	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 27	de	de	ADP	_	_	28	case	O	_
-28	5	5	NUM	_	NumType=Card	26	nummod	O	_
-29	a	a	ADP	_	_	31	case	O	_
-30	1010	1010	NUM	_	NumType=Card	31	nummod	O	_
-31	s-1	s-1	NOUN	_	Gender=Masc|Number=Plur	28	nmod	O	_
+28	5	5	NUM	_	NumType=Card	31	nummod	O	_
+29	a	a	CCONJ	_	_	30	cc	O	_
+30	1010	1010	NUM	_	NumType=Card	28	conj	O	_
+31	s-1	s-1	NOUN	_	Gender=Masc|Number=Plur	26	nmod	O	_
 32	,	,	PUNCT	_	_	37	punct	O	_
 33-34	no	_	_	_	_	_	_	_	_
 33	em	em	ADP	_	_	36	case	O	_
@@ -270927,10 +270927,10 @@
 56	de	de	ADP	_	_	57	case	O	_
 57	cisalhamento	cisalhamento	NOUN	_	Gender=Masc|Number=Sing	55	nmod	O	_
 58	de	de	ADP	_	_	59	case	O	_
-59	1010	1010	NUM	_	NumType=Card	57	nummod	O	_
-60	a	a	ADP	_	_	62	case	O	_
-61	5	5	NUM	_	NumType=Card	62	nummod	O	_
-62	s-1	s-1	NOUN	_	Gender=Masc|Number=Plur	59	nmod	O	_
+59	1010	1010	NUM	_	NumType=Card	62	nummod	O	_
+60	a	a	CCONJ	_	_	61	cc	O	_
+61	5	5	NUM	_	NumType=Card	59	conj	O	_
+62	s-1	s-1	NOUN	_	Gender=Masc|Number=Plur	57	nmod	O	_
 63	.	.	PUNCT	_	_	10	punct	O	_
 
 # text = Por fim, foi calculada a área do ciclo de histerese, caracterizando o grau de tixotropia do fluido.
@@ -277196,17 +277196,17 @@
 23	formado	formar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	18	conj	O	_
 24	por	por	ADP	_	_	26	case	O	_
 25	aproximadamente	aproximadamente	ADV	_	_	26	advmod	O	_
-26	82	82	NUM	_	NumType=Card	23	obl:agent	O	_
-27	a	a	ADP	_	_	29	case	O	_
-28	87	87	NUM	_	NumType=Card	29	nummod	O	_
-29	%	%	SYM	_	_	26	nmod	O	_
+26	82	82	NUM	_	NumType=Card	29	nummod	O	_
+27	a	a	CCONJ	_	_	28	cc	O	_
+28	87	87	NUM	_	NumType=Card	26	conj	O	_
+29	%	%	SYM	_	_	23	obl:agent	O	_
 30	em	em	ADP	_	_	31	case	O	_
 31	carbono	carbono	NOUN	_	Gender=Masc|Number=Sing	29	nmod	O	_
 32	e	e	CCONJ	_	_	33	cc	O	_
-33	11	11	NUM	_	NumType=Card	26	conj	O	_
-34	a	a	ADP	_	_	36	case	O	_
-35	15	15	NUM	_	NumType=Card	36	nummod	O	_
-36	%	%	SYM	_	_	33	nmod	O	_
+33	11	11	NUM	_	NumType=Card	36	nummod	O	_
+34	a	a	CCONJ	_	_	35	cc	O	_
+35	15	15	NUM	_	NumType=Card	33	conj	O	_
+36	%	%	SYM	_	_	26	conj	O	_
 37	em	em	ADP	_	_	38	case	O	_
 38	hidrogênio	hidrogênio	NOUN	_	Gender=Masc|Number=Sing	33	nmod	O	_
 39	,	,	PUNCT	_	_	43	punct	O	_
@@ -277229,10 +277229,10 @@
 55	,	,	PUNCT	_	_	56	punct	O	_
 56	enxofre	enxofre	NOUN	_	Gender=Masc|Number=Sing	48	conj	O	_
 57	(	(	PUNCT	_	_	58	punct	O	_
-58	0	0	NUM	_	NumType=Card	56	nmod:appos	O	_
-59	a	a	ADP	_	_	61	case	O	_
-60	8	8	NUM	_	NumType=Card	61	nummod	O	_
-61	%	%	SYM	_	_	58	nmod	O	_
+58	0	0	NUM	_	NumType=Card	61	nummod	O	_
+59	a	a	CCONJ	_	_	60	cc	O	_
+60	8	8	NUM	_	NumType=Card	58	conj	O	_
+61	%	%	SYM	_	_	56	nmod:appos	O	_
 62	)	)	PUNCT	_	_	58	punct	O	_
 63	e	e	CCONJ	_	_	65	cc	O	_
 64	de	de	ADP	_	_	65	case	O	_
@@ -283520,7 +283520,7 @@
 49	vai	ir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	46	acl:relcl	O	_
 50	de	de	ADP	_	_	51	case	O	_
 51	1	1	NUM	_	NumType=Card	49	obl	O	_
-52	a	a	ADP	_	_	53	case	O	_
+52	a	a	CCONJ	_	_	53	cc	O	_
 53	10	10	NUM	_	NumType=Card	51	conj	O	_
 54	.	.	PUNCT	_	_	4	punct	O	_
 
@@ -285572,8 +285572,8 @@
 26	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	31	cop	O	_
 27	entre	entre	ADP	_	_	31	case	O	_
 28	4	4	NUM	_	NumType=Card	31	nummod	O	_
-29	a	a	ADP	_	_	31	case	O	_
-30	8	8	NUM	_	NumType=Card	31	nummod	O	_
+29	a	a	CCONJ	_	_	30	cc	O	_
+30	8	8	NUM	_	NumType=Card	28	conj	O	_
 31	minutos	minuto	NOUN	_	Gender=Masc|Number=Plur	22	acl:relcl	O	_
 32	após	após	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
@@ -294426,10 +294426,10 @@
 26	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	27	det	O	_
 27	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 28	de	de	ADP	_	_	29	case	O	_
-29	1000	1000	NUM	_	NumType=Card	27	nummod	O	_
-30	a	a	ADP	_	_	32	case	O	_
-31	1400	1400	NUM	_	NumType=Card	32	nummod	O	_
-32	mg/	mg/	NOUN	_	Gender=Masc|Number=Plur	29	nmod	O	_
+29	1000	1000	NUM	_	NumType=Card	32	nummod	O	_
+30	a	a	CCONJ	_	_	31	cc	O	_
+31	1400	1400	NUM	_	NumType=Card	29	conj	O	_
+32	mg/	mg/	NOUN	_	Gender=Masc|Number=Plur	27	nmod	O	_
 33	L	L	PROPN	_	Gender=Masc|Number=Sing	32	nmod	O	_
 34	,	,	PUNCT	_	_	37	punct	O	_
 35	com	com	ADP	_	_	36	case	O	_
@@ -294485,9 +294485,9 @@
 79	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	80	det	O	_
 80	faixa	faixa	NOUN	_	Gender=Fem|Number=Sing	77	obl	O	_
 81	de	de	ADP	_	_	82	case	O	_
-82	1350	1350	NUM	_	NumType=Card	80	nummod	O	_
-83	a	a	ADP	_	_	85	case	O	_
-84	1400	1400	NUM	_	NumType=Card	85	nummod	O	_
+82	1350	1350	NUM	_	NumType=Card	85	nummod	O	_
+83	a	a	CCONJ	_	_	84	cc	O	_
+84	1400	1400	NUM	_	NumType=Card	82	conj	O	_
 85	mg	mg	NOUN	_	Gender=Masc|Number=Plur	80	nmod	O	_
 86	/L	/L	PROPN	_	Gender=Masc|Number=Sing	85	nmod	O	_
 87	,	,	PUNCT	_	_	88	punct	O	_
