commit 72ad91acb6b2dbcec9b8cc850ddda9fa39c36645
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Wed May 19 18:39:28 2021 -0300

    correções elvis petroles_3

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b4203c4..e3a6282 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -32,8 +32,8 @@
 22	a	a	ADP	_	_	24	case	O	_
 23	seu	seu	DET	_	Gender=Masc|Number=Sing|PronType=Prs	24	det	O	_
 24	respeito	respeito	NOUN	_	Gender=Masc|Number=Sing	21	obj	O	_
-25	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	26	expl	O	_
-26	comparada	comparar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	20	acl	O	_
+25	se	se	SCONJ	_	_	26	mark	O	_
+26	comparada	comparar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	18	advcl	O	_
 27-28	às	_	_	_	_	_	_	O	_
 27	a	a	ADP	_	_	30	case	O	_
 28	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
@@ -300,13 +300,13 @@
 # text = O objetivo deste trabalho é caracterizar as principais estruturas do arcabouço geológico do embasamento aflorante da Bacia de Pelotas e sua correlação com a porção offshore da mesma..
 # sent_id = 247-20140910-MONOGRAFIA_0-9
 1	O	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	2	det	O	_
-2	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	7	nsubj	O	_
+2	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	6	nsubj	O	_
 3-4	deste	_	_	_	_	_	_	O	_
 3	de	de	ADP	_	_	5	case	O	_
 4	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	5	det	O	_
 5	trabalho	trabalho	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	aux	O	_
-7	caracterizar	caracterizar	VERB	_	VerbForm=Inf	0	root	O	_
+6	é	ser	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+7	caracterizar	caracterizar	VERB	_	VerbForm=Inf	6	ccomp	O	_
 8	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
 9	principais	principal	ADJ	_	Gender=Fem|Number=Plur	10	amod	O	_
 10	estruturas	estrutura	NOUN	_	Gender=Fem|Number=Plur	7	obj	O	_
@@ -337,8 +337,8 @@
 31	de	de	ADP	_	_	33	case	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
 33	mesma	mesmo	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	29	nmod	O	_
-34	.	.	PUNCT	_	_	7	punct	O	_
-35	.	.	PUNCT	_	_	7	punct	O	_
+34	.	.	PUNCT	_	_	6	punct	O	_
+35	.	.	PUNCT	_	_	6	punct	O	_
 
 # text = A caracterização é realizada a partir da integração de dados de superfície do continente com dados de subsuperfície (offshore) da bacia adjacente, utilizando técnicas de sensoriamento remoto e interpretação sísmica..
 # sent_id = 247-20140910-MONOGRAFIA_0-10
@@ -2185,7 +2185,7 @@
 10	atividades	atividade	NOUN	_	Gender=Fem|Number=Plur	9	nsubj:pass	O	_
 11	de	de	ADP	_	_	12	case	O	_
 12	campo	campo	NOUN	_	Gender=Masc|Number=Sing	10	nmod	B=CAMPO:TIPO	_
-13	coordenadas	coordenada	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	12	acl	O	_
+13	coordenadas	coordenada	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	10	acl	O	_
 14-15	pelo	_	_	_	_	_	_	O	_
 14	por	por	ADP	_	_	16	case	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
@@ -6979,7 +6979,7 @@
 12	de	de	ADP	_	_	13	case	O	_
 13	diabásio	diabásio	NOUN	_	Gender=Masc|Number=Sing	11	nmod	B=LITOLOGIA	_
 14	com	com	ADP	_	_	16	case	O	_
-15	elevada	elevar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	16	acl	O	_
+15	elevada	elevado	ADJ	_	Gender=Fem|Number=Sing	16	amod	O	_
 16	frequência	frequência	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 17	próximo	próximo	ADJ	_	Gender=Masc|Number=Sing	16	amod	O	_
 18-19	ao	_	_	_	_	_	_	O	_
@@ -7930,7 +7930,7 @@
 2	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	7	nsubj:pass	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	fraturamento	fraturamento	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-5	encontrado	encontrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	4	acl	O	_
+5	encontrado	encontrar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
 6	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	aux:pass	O	_
 7	marcado	marcar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 8	por	por	ADP	_	_	10	case	O	_
@@ -10821,13 +10821,13 @@
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	6	flat:name	O	_
 9	Mioceno	Mioceno	PROPN	_	Number=Sing	6	flat:name	B=GEOCRONOLOGIA	_
 10	,	,	PUNCT	_	_	11	punct	O	_
-11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	appos	O	_
+11	Topo	Topo	PROPN	_	Gender=Masc|Number=Sing	6	conj	O	_
 12-13	do	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	11	flat:name	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	11	flat:name	O	_
 14	Cretáceo	Cretáceo	PROPN	_	Number=Sing	11	flat:name	B=GEOCRONOLOGIA	_
 15	e	e	CCONJ	_	_	16	cc	O	_
-16	Topo	Topo	PROPN	_	Number=Sing	11	flat:name	O	_
+16	Topo	Topo	PROPN	_	Number=Sing	11	conj	O	_
 17-18	do	_	_	_	_	_	_	O	_
 17	de	de	ADP	_	_	16	flat:name	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
@@ -25088,7 +25088,7 @@
 49	marinha	marinho	ADJ	_	Gender=Fem|Number=Sing	48	amod	O	_
 50	,	,	PUNCT	_	_	52	punct	O	_
 51	como	como	ADP	_	_	52	case	O	_
-52	dinoflagelados	dinoflagelar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	45	acl	O	_
+52	dinoflagelados	dinoflagelar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	45	nmod:appos	O	_
 53	e	e	CCONJ	_	_	54	cc	O	_
 54	foraminíferos	foraminífero	NOUN	_	Gender=Masc|Number=Plur	52	conj	O	_
 55	,	,	PUNCT	_	_	52	punct	O	_
@@ -48230,7 +48230,7 @@
 5	de	de	ADP	_	_	6	case	O	_
 6	dispersão	dispersão	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 7	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux:pass	O	_
-8	“fechada”	ofechar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
+8	“fechada”	fechar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
 10	como	como	SCONJ	_	_	11	mark	O	_
 11	observado	observar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	8	advcl	O	_
@@ -65740,7 +65740,7 @@
 3	Carmo	Carmo	PROPN	_	Number=Sing	2	flat:name	O	_
 4	,	,	PUNCT	_	_	6	punct	O	_
 5	segundo	segundo	ADP	_	_	6	case	O	_
-6	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	2	nmod	O	_
+6	Sgarbi	Sgarbi	PROPN	_	Gender=Masc|Number=Sing	11	obl	O	_
 7	(	(	PUNCT	_	_	6	flat:name	O	_
 8	2011	2011	NUM	_	NumType=Card	6	flat:name	O	_
 9	)	)	PUNCT	_	_	6	flat:name	O	_
@@ -65749,7 +65749,7 @@
 12-13	na	_	_	_	_	_	_	O	_
 12	em	em	ADP	_	_	14	case	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	região	região	NOUN	_	Gender=Fem|Number=Sing	11	obj	O	_
+14	região	região	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 15	de	de	ADP	_	_	16	case	O	_
 16	Carmo	Carmo	PROPN	_	Gender=Masc|Number=Sing	14	nmod	O	_
 17-18	do	_	_	_	_	_	_	O	_
@@ -65757,18 +65757,18 @@
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	flat:name	O	_
 19	Paranaíba	Paranaíba	PROPN	_	Number=Sing	16	flat:name	O	_
 20	(	(	PUNCT	_	_	21	punct	O	_
-21	MG	MG	PROPN	_	Gender=Masc|Number=Sing	16	appos	O	_
+21	MG	MG	PROPN	_	Gender=Masc|Number=Sing	16	nmod:appos	O	_
 22	)	)	PUNCT	_	_	21	punct	O	_
 23	e	e	CCONJ	_	_	25	cc	O	_
 24	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	25	aux:pass	O	_
 25	constituído	constituir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	11	conj	O	_
 26	por	por	ADP	_	_	27	case	O	_
 27	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	25	obl:agent	B=LITOLOGIA	_
-28	matriz-sustentado	matriz-sustentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	27	acl	O	_
+28	matriz-sustentado	matriz-sustentado	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
 29	e	e	CCONJ	_	_	30	cc	O	_
-30	arenitos	arenito	ADJ	_	Gender=Masc|Number=Plur	28	conj	B=LITOLOGIA	_
-31	líticos	lítico	ADJ	_	Gender=Masc|Number=Plur	27	amod	O	_
-32	depositados	depositar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	27	acl	O	_
+30	arenitos	arenito	ADJ	_	Gender=Masc|Number=Plur	27	conj	B=LITOLOGIA	_
+31	líticos	lítico	ADJ	_	Gender=Masc|Number=Plur	30	amod	O	_
+32	depositados	depositar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	30	acl	O	_
 33	por	por	ADP	_	_	34	case	O	_
 34	leques	leque	NOUN	_	Gender=Masc|Number=Plur	32	obl:agent	O	_
 35	aluviais	aluvial	ADJ	_	Gender=Masc|Number=Plur	34	amod	O	_
@@ -65776,7 +65776,7 @@
 37	fluxos	fluxo	NOUN	_	Gender=Masc|Number=Plur	34	conj	O	_
 38	aquosos	aquoso	ADJ	_	Gender=Masc|Number=Plur	37	amod	O	_
 39	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	40	nsubj	O	_
-40	geraram	gerar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	37	acl:relcl	O	_
+40	geraram	gerar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	34	acl:relcl	O	_
 41	depósitos	depósito	NOUN	_	Gender=Masc|Number=Plur	40	obj	B=LOCAL:GEO	_
 42-43	do	_	_	_	_	_	_	O	_
 42	de	de	ADP	_	_	44	case	O	_
@@ -93636,7 +93636,7 @@
 
 # text = Por exemplo, a maior distância entre uma fonte e o ponto onde o CO2 vai ser injetado é de 459 quilômetros, referente à extensão total de carbdouto entre a CSN e a plataforma FPSO P-37..
 # sent_id = 161-20150810-MONOGRAFIA_0-496
-1	Por	por	ADP	_	_	22	advmod	O	_
+1	Por	por	ADP	_	_	22	advmod	O	MWEPOS=ADV
 2	exemplo	exemplo	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	_
 3	,	,	PUNCT	_	_	1	punct	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	6	det	O	_
@@ -93658,18 +93658,18 @@
 20	de	de	ADP	_	_	22	case	O	_
 21	459	459	NUM	_	NumType=Card	22	nummod	O	_
 22	quilômetros	quilômetro	NOUN	_	Gender=Masc|Number=Plur	0	root	O	_
-23	,	,	PUNCT	_	_	22	punct	O	_
+23	,	,	PUNCT	_	_	24	punct	O	_
 24	referente	referente	ADV	_	_	22	advmod	O	_
 25-26	à	_	_	_	_	_	_	O	_
 25	a	a	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	extensão	extensão	NOUN	_	Gender=Fem|Number=Sing	24	obl	O	_
 28	total	total	ADJ	_	Gender=Fem|Number=Sing	27	amod	O	_
-29	de	de	ADP	_	_	30	mark	O	_
-30	carbdouto	carbdoutar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	27	acl	O	_
+29	de	de	ADP	_	_	30	case	O	_
+30	carbdouto	carboduto	NOUN	_	Gender=Masc|Number=Sing	27	nmod	O	_
 31	entre	entre	ADP	_	_	33	case	O	_
 32	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	33	det	O	_
-33	CSN	CSN	PROPN	_	Gender=Fem|Number=Sing	30	obl	O	_
+33	CSN	CSN	PROPN	_	Gender=Fem|Number=Sing	27	nmod	O	_
 34	e	e	CCONJ	_	_	36	cc	O	_
 35	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	36	det	O	_
 36	plataforma	plataforma	NOUN	_	Gender=Fem|Number=Sing	33	conj	O	_
@@ -93740,7 +93740,7 @@
 23	principalmente	principalmente	ADV	_	_	26	advmod	O	_
 24	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 25	“	“	PUNCT	_	_	26	punct	O	_
-26	campos	campo	NOUN	_	Gender=Masc|Number=Plur	18	appos	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
+26	campos	campo	NOUN	_	Gender=Masc|Number=Plur	21	appos	B=BACIA|B=CAMPO:TIPO|B=LOCAL:GEO	_
 27	gigantes	gigante	ADJ	_	Gender=Masc|Number=Plur	26	amod	O	_
 28	”	”	PUNCT	_	_	26	punct	O	_
 29	,	,	PUNCT	_	_	31	punct	O	_
@@ -95526,7 +95526,7 @@
 
 # text = Utilizando o valor da pressão original do campo de Marlim que é igual a 4082 psi ou 28,13 MPa (Oliveira, 2008), o valor de produção total acumulada de petróleo em Marlim, que é de 2131 milhões de barris e o valor de volume original de óleo in place, que é de 8100 milhões de barris, foi possível calcular o valor atual de pressão do reservatório: Com o auxilio do gráfico, o valor atual de pressão do reservatório foi de: Utilizando o valor da pressão original do campo de Barracuda que foi estimado, a partir do valor de pressão original de Marlim (devido as características e profundidades semelhantes), em 29 MPa, o valor de produção total acumulada de Barracuda, que é 386 milhões de barris e o valor de volume original de óleo in place, que é de 2250 milhões de barris, foi possível calcular o valor atual de pressão do reservatório: Com o auxilio do gráfico, o valor atual de pressão do reservatório foi de: Pressão atual do (MPa) = 82% x 29MPa =23,8 MPa
 # sent_id = 161-20150810-MONOGRAFIA_0-555
-1	Utilizando	utilizar	VERB	_	VerbForm=Ger	66	advcl	O	_
+1	Utilizando	utilizar	VERB	_	VerbForm=Ger	66	advcl	O	eliminar
 2	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	3	det	O	_
 3	valor	valor	NOUN	_	Gender=Masc|Number=Sing	1	obj	O	_
 4-5	da	_	_	_	_	_	_	O	_
@@ -112109,12 +112109,12 @@
 27	de	de	ADP	_	_	29	case	O	_
 28	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
 29	relatório	relatório	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
-30	com	com	ADP	_	_	31	case	O	_
-31	vistas	vestir	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	26	acl	O	_
+30	com	com	ADP	_	_	34	case	O	MWEPOS=ADP
+31	vistas	vista	NOUN	_	Gender=Fem|Number=Plur	30	fixed	O	_
 32-33	à	_	_	_	_	_	_	O	_
 32	a	a	ADP	_	_	34	case	O	_
 33	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	34	det	O	_
-34	valoração	valoração	NOUN	_	Gender=Fem|Number=Sing	31	obl	O	_
+34	valoração	valoração	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 35-36	dos	_	_	_	_	_	_	O	_
 35	de	de	ADP	_	_	37	case	O	_
 36	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	37	det	O	_
@@ -115170,15 +115170,15 @@
 31	ambas	ambos	DET	_	Gender=Fem|Number=Plur|PronType=Tot	33	det	O	_
 32	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	33	det	O	_
 33	regiões	região	NOUN	_	Gender=Fem|Number=Plur	34	nsubj	O	_
-34	possuem	possuir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	acl:relcl	O	_
+34	possuem	possuir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	14	parataxis	O	_
 35	áreas	área	NOUN	_	Gender=Fem|Number=Plur	34	obj	O	_
 36	de	de	ADP	_	_	37	case	O	_
 37	produção	produção	NOUN	_	Gender=Fem|Number=Sing	35	nmod	O	_
 38	offshore	offshore	ADJ	_	Gender=Fem|Number=Sing	37	amod	O	_
 39	bem	bem	ADV	_	_	40	advmod	O	_
-40	desenvolvidas	desenvolver	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	37	acl	O	_
+40	desenvolvidas	desenvolver	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	35	acl	O	_
 41	(	(	PUNCT	_	_	42	punct	O	_
-42	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	46	appos	O	_
+42	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	14	parataxis	O	_
 43	6	6	PROPN	_	Number=Sing	42	flat:name	O	_
 44	)	)	PUNCT	_	_	42	punct	O	_
 45	(	(	PUNCT	_	_	46	punct	O	_
@@ -117251,26 +117251,26 @@
 3	de	de	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 5	gás	gás	NOUN	_	Gender=Masc|Number=Sing	2	nmod	B=FLUIDO	_
-6	natural	natural	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
+6	natural	natural	ADJ	_	Gender=Masc|Number=Sing	5	compound	O	_
 7	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 8	por	por	ADP	_	_	9	case	O	_
-9	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
-10	,	,	PUNCT	_	_	7	punct	O	_
-11	além	além	ADV	_	_	14	cc	O	_
+9	objetivo	objetivo	NOUN	_	Gender=Masc|Number=Sing	7	xcomp	O	_
+10	,	,	PUNCT	_	_	14	punct	O	_
+11	além	além	ADV	_	_	14	cc	O	MWEPOS=CCONJ
 12-13	da	_	_	_	_	_	_	O	_
 12	de	de	ADP	_	_	11	fixed	O	_
 13	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	separação	separação	NOUN	_	Gender=Fem|Number=Sing	7	obl	O	_
+14	separação	separação	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 15-16	do	_	_	_	_	_	_	O	_
 15	de	de	ADP	_	_	17	case	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
 17	gás	gás	NOUN	_	Gender=Masc|Number=Sing	14	nmod	B=FLUIDO	_
-18	,	,	PUNCT	_	_	20	punct	O	_
+18	,	,	PUNCT	_	_	14	punct	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	remoção	remoção	NOUN	_	Gender=Fem|Number=Sing	14	appos	O	_
+20	remoção	remoção	NOUN	_	Gender=Fem|Number=Sing	7	obj	O	_
 21	,	,	PUNCT	_	_	23	punct	O	_
 22	ou	ou	CCONJ	_	_	23	cc	O	_
-23	redução	redução	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
+23	redução	redução	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
 24	a	a	ADP	_	_	25	case	O	_
 25	teores	teor	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 26	estabelecidos	estabelecer	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	25	acl	O	_
@@ -117278,11 +117278,11 @@
 28	normas	norma	NOUN	_	Gender=Fem|Number=Plur	26	obl:agent	O	_
 29	ou	ou	CCONJ	_	_	30	cc	O	_
 30	padrões	padrão	NOUN	_	Gender=Masc|Number=Plur	28	conj	O	_
-31	,	,	PUNCT	_	_	33	punct	O	_
+31	,	,	PUNCT	_	_	23	punct	O	_
 32	de	de	ADP	_	_	33	case	O	_
-33	compostos	compor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	28	acl	O	_
+33	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	20	nmod	O	_
 34	como	como	ADP	_	_	35	case	O	_
-35	água	água	NOUN	_	Gender=Fem|Number=Sing	33	xcomp	B=FLUIDO	_
+35	água	água	NOUN	_	Gender=Fem|Number=Sing	33	nmod	B=FLUIDO	_
 36	,	,	PUNCT	_	_	37	punct	O	_
 37	dióxido	dióxido	NOUN	_	Gender=Masc|Number=Sing	35	conj	O	_
 38	de	de	ADP	_	_	39	case	O	_
@@ -117291,18 +117291,18 @@
 41	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	37	appos	B=QUÍMICA	_
 42	)	)	PUNCT	_	_	41	punct	O	_
 43	,	,	PUNCT	_	_	44	punct	O	_
-44	compostos	compor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	37	acl	O	_
-45	sulfurados	sulfurar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	44	conj	O	_
+44	compostos	composto	NOUN	_	Gender=Masc|Number=Plur	35	conj	O	_
+45	sulfurados	sulfurar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	44	acl	O	_
 46	(	(	PUNCT	_	_	47	punct	O	_
-47	H2S	H2S	PROPN	_	Gender=Masc|Number=Sing	44	nmod:appos	O	_
+47	H2S	H2S	PROPN	_	Gender=Masc|Number=Sing	44	appos	O	_
 48	,	,	PUNCT	_	_	49	punct	O	_
 49	CS2	CS2	PROPN	_	Gender=Masc|Number=Sing	47	conj	O	_
 50	,	,	PUNCT	_	_	51	punct	O	_
 51	COS	COS	PROPN	_	Gender=Masc|Number=Sing	47	conj	O	_
-52	etc.	etc.	ADV	_	_	51	advmod	O	_
+52	etc.	etc.	ADV	_	_	47	conj	O	_
 53	)	)	PUNCT	_	_	47	punct	O	_
 54	,	,	PUNCT	_	_	55	punct	O	_
-55	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	28	conj	O	_
+55	sólidos	sólido	NOUN	_	Gender=Masc|Number=Plur	35	conj	O	_
 56	(	(	PUNCT	_	_	57	punct	O	_
 57	areia	areia	NOUN	_	Gender=Fem|Number=Sing	55	appos	B=LITOLOGIA	_
 58	,	,	PUNCT	_	_	59	punct	O	_
@@ -117315,14 +117315,14 @@
 65	corrosão	corrosão	NOUN	_	Gender=Fem|Number=Sing	63	nmod	O	_
 66	)	)	PUNCT	_	_	57	punct	O	_
 67	e	e	CCONJ	_	_	68	cc	O	_
-68	líquidos	líquido	NOUN	_	Gender=Masc|Number=Plur	28	conj	O	_
+68	líquidos	líquido	NOUN	_	Gender=Masc|Number=Plur	35	conj	O	_
 69	,	,	PUNCT	_	_	71	punct	O	_
 70	como	como	ADP	_	_	71	case	O	_
-71	condensados	condensar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	68	acl	O	_
+71	condensados	condensado	NOUN	_	Gender=Masc|Number=Plur	68	nmod:appos	O	_
 72	de	de	ADP	_	_	73	case	O	_
-73	gás	gás	NOUN	_	Gender=Masc|Number=Sing	71	obl	B=FLUIDO	_
+73	gás	gás	NOUN	_	Gender=Masc|Number=Sing	71	nmod	B=FLUIDO	_
 74	e	e	CCONJ	_	_	75	cc	O	_
-75	produtos	produto	NOUN	_	Gender=Masc|Number=Plur	73	conj	O	_
+75	produtos	produto	NOUN	_	Gender=Masc|Number=Plur	71	conj	O	_
 76	químicos	químico	ADJ	_	Gender=Masc|Number=Plur	75	amod	O	_
 77	.	.	PUNCT	_	_	7	punct	O	_
 78	.	.	PUNCT	_	_	7	punct	O	_
@@ -133271,21 +133271,21 @@
 17	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	15	obj	B=QUÍMICA	_
 18-19	nela	_	_	_	_	_	_	O	_
 18	em	em	ADP	_	_	19	case	O	_
-19	ela	ela	PRON	_	Gender=Fem|Number=Sing|Person=3|PronType=Prs	15	obl	O	_
-20	contido	conter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	19	acl	O	_
-21	removido	remover	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	19	acl	O	_
-22	antes	antes	ADV	_	_	25	mark	O	_
+19	ela	ela	PRON	_	Gender=Fem|Number=Sing|Person=3|PronType=Prs	20	obl	O	_
+20	contido	conter	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	acl	O	_
+21	removido	remover	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	15	xcomp	O	_
+22	antes	antes	ADV	_	_	25	mark	O	MWEPOS=SCONJ
 23	de	de	ADP	_	_	22	fixed	O	_
 24	ser	ser	AUX	_	VerbForm=Inf	25	aux:pass	O	_
 25	encaminhada	encaminhar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	21	advcl	O	_
 26-27	à	_	_	_	_	_	_	O	_
 26	a	a	ADP	_	_	28	case	O	_
 27	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	28	det	O	_
-28	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	25	obj	O	_
+28	unidade	unidade	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
 29	de	de	ADP	_	_	30	case	O	_
 30	recuperação	recuperação	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	de	de	ADP	_	_	32	case	O	_
-32	MEG..	MEG..	PROPN	_	Gender=Masc|Number=Sing	30	nmod	O	_
+32	MEG..	MEG..	PROPN	_	Gender=Masc|Number=Sing	30	nmod	O	tokenização
 
 # text = A fim de vaporizar o CO2 e facilitar sua remoção, esta corrente passa por despressurização seguida de aquecimento..
 # sent_id = 334-20150521-MONOGRAFIA_0-678
@@ -137961,24 +137961,24 @@
 # text = De caráter hidrofílico-lipofílico, essas substâncias são, em geral, moléculas com elevada massa molecular, de ocorrência natural nos petróleos, contendo heteroátomos como o nitrogênio, o enxofre, o oxigênio e metais, usualmente, fazem parte de grupos classificados como asfaltenos e resinas (BRASIL et al., 2012)..
 # sent_id = 202-20140404-MONOGRAFIA_0-25
 1	De	de	ADP	_	_	2	case	O	_
-2	caráter	caráter	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
-3	hidrofílico-lipofílico	hidrofílico-lipofílico	NOUN	_	Gender=Masc|Number=Sing	2	compound	O	_
+2	caráter	caráter	NOUN	_	Gender=Masc|Number=Sing	6	nmod	O	_
+3	hidrofílico-lipofílico	hidrofílico-lipofílico	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 4	,	,	PUNCT	_	_	2	punct	O	_
 5	essas	esse	DET	_	Gender=Fem|Number=Plur|PronType=Dem	6	det	O	_
 6	substâncias	substância	NOUN	_	Gender=Fem|Number=Plur	12	nsubj	O	_
 7	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	cop	O	_
 8	,	,	PUNCT	_	_	10	punct	O	_
 9	em	em	ADP	_	_	10	case	O	_
-10	geral	geral	ADJ	_	Gender=Masc|Number=Sing	12	nmod	O	_
+10	geral	geral	ADJ	_	Gender=Masc|Number=Sing	12	obl	O	_
 11	,	,	PUNCT	_	_	10	punct	O	_
 12	moléculas	molécula	NOUN	_	Gender=Fem|Number=Plur	0	root	O	_
 13	com	com	ADP	_	_	15	case	O	_
-14	elevada	elevar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	15	acl	O	_
+14	elevada	elevado	ADJ	_	Gender=Fem|Number=Sing	15	amod	O	_
 15	massa	massa	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
 16	molecular	molecular	ADJ	_	Gender=Fem|Number=Sing	15	amod	O	_
 17	,	,	PUNCT	_	_	19	punct	O	_
 18	de	de	ADP	_	_	19	case	O	_
-19	ocorrência	ocorrência	NOUN	_	Gender=Fem|Number=Sing	15	nmod	O	_
+19	ocorrência	ocorrência	NOUN	_	Gender=Fem|Number=Sing	12	nmod	O	_
 20	natural	natural	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21-22	nos	_	_	_	_	_	_	O	_
 21	em	em	ADP	_	_	23	case	O	_
@@ -137989,7 +137989,7 @@
 26	heteroátomos	heteroátomo	NOUN	_	Gender=Masc|Number=Plur	25	obj	O	_
 27	como	como	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
-29	nitrogênio	nitrogênio	NOUN	_	Gender=Masc|Number=Sing	25	xcomp	O	_
+29	nitrogênio	nitrogênio	NOUN	_	Gender=Masc|Number=Sing	26	nmod	O	_
 30	,	,	PUNCT	_	_	32	punct	O	_
 31	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	32	det	O	_
 32	enxofre	enxofre	NOUN	_	Gender=Masc|Number=Sing	29	conj	B=QUÍMICA	_
@@ -137997,26 +137997,26 @@
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 35	oxigênio	oxigênio	NOUN	_	Gender=Masc|Number=Sing	29	conj	O	_
 36	e	e	CCONJ	_	_	37	cc	O	_
-37	metais	metal	NOUN	_	Gender=Masc|Number=Plur	29	conj	O	_
+37	metais	metal	NOUN	_	Gender=Masc|Number=Plur	41	nsubj	O	_
 38	,	,	PUNCT	_	_	39	punct	O	_
 39	usualmente	usualmente	ADV	_	_	41	advmod	O	_
 40	,	,	PUNCT	_	_	39	punct	O	_
 41	fazem	fazer	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	12	conj	O	_
 42	parte	parte	NOUN	_	Gender=Fem|Number=Sing	41	obj	O	_
 43	de	de	ADP	_	_	44	case	O	_
-44	grupos	grupo	NOUN	_	Gender=Masc|Number=Plur	42	nmod	O	_
+44	grupos	grupo	NOUN	_	Gender=Masc|Number=Plur	41	obl	O	_
 45	classificados	classificar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	44	acl	O	_
 46	como	como	ADP	_	_	47	case	O	_
 47	asfaltenos	asfalteno	NOUN	_	Gender=Masc|Number=Plur	45	xcomp	B=QUÍMICA	_
 48	e	e	CCONJ	_	_	49	cc	O	_
 49	resinas	resina	NOUN	_	Gender=Fem|Number=Plur	47	conj	O	_
 50	(	(	PUNCT	_	_	51	punct	O	_
-51	BRASIL	Brasil	PROPN	_	Gender=Masc|Number=Sing	41	parataxis	O	_
+51	BRASIL	Brasil	PROPN	_	Gender=Masc|Number=Sing	12	parataxis	O	_
 52	et	et	PROPN	_	Number=Sing	51	flat:name	O	_
 53	al.	al.	PROPN	_	Number=Sing	51	flat:name	O	_
 54	,	,	PUNCT	_	_	51	flat:name	O	_
 55	2012	2012	NUM	_	NumType=Card	51	flat:name	O	_
-56	)	)	PUNCT	_	_	51	punct	O	_
+56	)	)	PUNCT	_	_	51	flat:name	O	_
 57	.	.	PUNCT	_	_	41	punct	O	_
 58	.	.	PUNCT	_	_	12	punct	O	_
 
@@ -354768,4 +354768,3 @@
 54	.	.	PUNCT	_	_	2	punct	O	_
 55	.	.	PUNCT	_	_	2	punct	O	_
 
-
