commit 3bedc0e8d1dcc3f29a5fcc9a598359310eafc495
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Thu Jul 8 01:29:46 2021 -0300

    assim_como

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index b6ea396..5ceb8b3 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -3585,11 +3585,11 @@
 3	de	de	ADP	_	_	2	flat:name	O	_
 4	Pelotas	Pelotas	PROPN	_	Number=Sing	2	flat:name	B=BACIA	_
 5	,	,	PUNCT	_	_	10	punct	O	_
-6	assim	assim	ADV	_	_	10	advmod	O	_
+6	assim	assim	ADV	_	_	10	cc	O	MWEPOS=CCONJ
 7	como	como	ADP	_	_	6	fixed	O	_
 8	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
 9	demais	demais	DET	_	Gender=Fem|Number=Plur|PronType=Ind	10	det	O	_
-10	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	2	nmod:appos	B=BACIA:TIPO	_
+10	bacias	bacia	NOUN	_	Gender=Fem|Number=Plur	2	conj	B=BACIA:TIPO	_
 11-12	da	_	_	_	_	_	_	_	_
 11	de	de	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
@@ -8339,7 +8339,7 @@
 25	de	de	ADP	_	_	24	flat:name	O	_
 26	Pelotas	Pelotas	PROPN	_	Number=Sing	24	flat:name	B=BACIA	_
 27	,	,	PUNCT	_	_	18	punct	O	_
-28	assim	assim	ADV	_	_	31	cc	O	_
+28	assim	assim	ADV	_	_	31	cc	O	MWEPOS=CCONJ
 29	como	como	ADP	_	_	28	fixed	O	_
 30	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
 31	quebra	quebra	NOUN	_	Gender=Fem|Number=Sing	20	conj	O	_
@@ -10796,8 +10796,8 @@
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
 23	Mioceno	Mioceno	PROPN	_	Gender=Masc|Number=Sing	12	obl	B=GEOCRONOLOGIA	_
 24	Inferior	Inferior	PROPN	_	Number=Sing	23	flat:name	I=GEOCRONOLOGIA	_
-25	assim	assim	ADV	_	_	27	advmod	O	_
-26	como	como	SCONJ	_	_	27	mark	O	_
+25	assim	assim	ADV	_	_	27	mark	O	MWEPOS=SCONJ
+26	como	como	ADP	_	_	25	fixed	O	_
 27	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	12	advcl	O	_
 28	por	por	ADP	_	_	29	case	O	_
 29	Bueno	Bueno	PROPN	_	Gender=Masc|Number=Sing	27	obl:agent	O	_
@@ -25105,10 +25105,10 @@
 
 # text = Assim como os autores citados acima, Sales (2005) considera, para o Membro Romualdo, paleoambiente transicional (lagunar), associado a incursões marinhas albianas, com ciclos deposicionais distintos e sucessivos com eventos de sedimentação episódica..
 # sent_id = 284-20150917-MONOGRAFIA_0-165
-1	Assim	assim	ADV	_	_	4	advmod	O	_
+1	Assim	assim	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	autores	autor	NOUN	_	Gender=Masc|Number=Plur	12	obl	O	_
+4	autores	autor	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
 5	citados	citar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	4	acl	O	_
 6	acima	acima	ADV	_	_	5	advmod	O	_
 7	,	,	PUNCT	_	_	5	punct	O	_
@@ -26920,7 +26920,7 @@
 10	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	11	det	O	_
 11	barras	barra	NOUN	_	Gender=Fem|Number=Plur	8	nmod	O	_
 12	,	,	PUNCT	_	_	15	punct	O	_
-13	assim	assim	ADV	_	_	15	cc	O	_
+13	assim	assim	ADV	_	_	15	cc	O	MWEPOS=CCONJ
 14	como	como	ADP	_	_	13	fixed	O	_
 15	clastos	clasto	NOUN	_	Gender=Masc|Number=Plur	8	conj	O	_
 16	arredondados	arredondar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	15	acl	O	_
@@ -32120,8 +32120,8 @@
 14	sistema	sistema	NOUN	_	Gender=Masc|Number=Sing	12	nmod	O	_
 15	fluvial	fluvial	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	,	,	PUNCT	_	_	10	punct	O	_
-17	assim	assim	ADV	_	_	19	mark	O	_
-18	como	como	SCONJ	_	_	17	fixed	O	_
+17	assim	assim	ADV	_	_	19	mark	O	MWEPOS=SCONJ
+18	como	como	ADP	_	_	17	fixed	O	_
 19	adotado	adotar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	10	advcl	O	_
 20-21	no	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	23	case	O	_
@@ -42692,12 +42692,12 @@
 
 # text = Assim como no Teste 1 do aquecimento na água, novamente não é, para estes valores de corrente e frequência, efetuado o aquecimento indutivo, conforme pode ser visto no Gráfico 7..
 # sent_id = 398-20160721-MONOGRAFIA_0-329
-1	Assim	assim	ADV	_	_	5	advmod	O	_
+1	Assim	assim	ADV	_	_	5	case	O	MWEPOS=ADP
 2	como	como	ADP	_	_	1	fixed	O	_
 3-4	no	_	_	_	_	_	_	_	_
 3	em	em	ADP	_	_	5	case	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
-5	Teste	Teste	PROPN	_	Gender=Masc|Number=Sing	0	root	B=MÉTODO	_
+5	Teste	Teste	PROPN	_	Gender=Masc|Number=Sing	26	obl	B=MÉTODO	_
 6	1	1	PROPN	_	Number=Sing	5	flat:name	O	_
 7-8	do	_	_	_	_	_	_	_	_
 7	de	de	ADP	_	_	9	case	O	_
@@ -42709,18 +42709,18 @@
 12	água	água	NOUN	_	Gender=Fem|Number=Sing	9	nmod	B=FLUIDO	_
 13	,	,	PUNCT	_	_	5	punct	O	_
 14	novamente	novamente	ADV	_	_	26	advmod	O	_
-15	não	não	ADV	_	Polarity=Neg	16	fixed	O	_
+15	não	não	ADV	_	Polarity=Neg	26	advmod	O	_
 16	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	26	cop	O	_
 17	,	,	PUNCT	_	_	20	punct	O	_
 18	para	para	ADP	_	_	20	case	O	_
 19	estes	este	DET	_	Gender=Masc|Number=Plur|PronType=Dem	20	det	O	_
-20	valores	valor	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
+20	valores	valor	NOUN	_	Gender=Masc|Number=Plur	26	obl	O	_
 21	de	de	ADP	_	_	22	case	O	_
 22	corrente	corrente	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 23	e	e	CCONJ	_	_	24	cc	O	_
 24	frequência	frequência	NOUN	_	Gender=Fem|Number=Sing	22	conj	O	_
 25	,	,	PUNCT	_	_	20	punct	O	_
-26	efetuado	efetuar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	20	acl	O	_
+26	efetuado	efetuar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	0	root	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
 28	aquecimento	aquecimento	NOUN	_	Gender=Masc|Number=Sing	26	obj	O	_
 29	indutivo	indutivo	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
@@ -42797,8 +42797,8 @@
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
 26	frequência	frequência	NOUN	_	Gender=Fem|Number=Sing	24	obj	O	_
 27	,	,	PUNCT	_	_	24	punct	O	_
-28	assim	assim	ADV	_	_	30	mark	O	_
-29	como	como	SCONJ	_	_	28	fixed	O	_
+28	assim	assim	ADV	_	_	30	mark	O	MWEPOS=SCONJ
+29	como	como	ADP	_	_	28	fixed	O	_
 30	visto	ver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	24	advcl	O	_
 31-32	na	_	_	_	_	_	_	_	_
 31	em	em	ADP	_	_	33	case	O	_
@@ -43487,12 +43487,12 @@
 
 # text = Assim como na Seção 4.1.3, é possível identificar no Gráfico 15, um crescimento da temperatura bastante interessante para recuperação de petróleo..
 # sent_id = 398-20160721-MONOGRAFIA_0-367
-1	Assim	assim	ADV	_	_	5	cc	O	_
+1	Assim	assim	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3-4	na	_	_	_	_	_	_	_	_
 3	em	em	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	Seção	seção	PROPN	_	Gender=Fem|Number=Sing	9	obl	O	_
+5	Seção	seção	PROPN	_	Gender=Fem|Number=Sing	13	conj	O	_
 6	4.1.3	4.1.3	PROPN	_	Number=Sing	5	flat:name	O	_
 7	,	,	PUNCT	_	_	5	punct	O	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
@@ -44068,12 +44068,12 @@
 
 # text = Assim como na Seção 4.1.3, é possível identificar no Gráfico 21, um crescimento da temperatura bastante interessante para recuperação de petróleo, no entanto a temperatura da parafina não a torna totalmente solúvel na mistura..
 # sent_id = 398-20160721-MONOGRAFIA_0-395
-1	Assim	assim	ADV	_	_	5	case	O	_
+1	Assim	assim	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3-4	na	_	_	_	_	_	_	_	_
 3	em	em	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	Seção	seção	PROPN	_	Gender=Fem|Number=Sing	9	obl	O	_
+5	Seção	seção	PROPN	_	Gender=Fem|Number=Sing	13	conj	O	_
 6	4.1.3	4.1.3	PROPN	_	Number=Sing	5	flat:name	O	_
 7	,	,	PUNCT	_	_	5	punct	O	_
 8	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
@@ -44461,9 +44461,9 @@
 7	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	8	expl	O	_
 8	esperar	esperar	VERB	_	VerbForm=Inf	5	xcomp	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
-10	assim	assim	ADV	_	_	12	mark	O	_
-11	como	como	SCONJ	_	_	10	fixed	O	_
-12	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	advcl	O	_
+10	assim	assim	ADV	_	_	12	mark	O	MWEPOS=SCONJ
+11	como	como	ADP	_	_	10	fixed	O	_
+12	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	30	advcl	O	_
 13	em	em	ADP	_	_	14	case	O	_
 14	transformadores	transformador	NOUN	_	Gender=Masc|Number=Plur	12	obl	O	_
 15	,	,	PUNCT	_	_	12	punct	O	_
@@ -59270,10 +59270,10 @@
 2	argila	argila	NOUN	_	Gender=Fem|Number=Sing	10	nsubj	B=LITOLOGIA	_
 3	A2	A2	PROPN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
-5	assim	assim	ADV	_	_	8	cc	O	_
+5	assim	assim	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6	como	como	ADP	_	_	5	fixed	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	demais	demais	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	2	nmod:appos	O	_
+8	demais	demais	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	2	conj	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
 10	possui	possuir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
@@ -59477,10 +59477,10 @@
 
 # text = Assim como a argila A2, a argila A3 possui alta concentração de ferro (Fe2O3, 11,1%p/p), o que sugere a substituição isomórfica do alumínio octaédrico por esse íon..
 # sent_id = 24-20150122-MONOGRAFIA_0-478
-1	Assim	assim	ADV	_	_	4	cc	O	_
+1	Assim	assim	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	argila	argila	NOUN	_	Gender=Fem|Number=Sing	10	obl	B=LITOLOGIA	_
+4	argila	argila	NOUN	_	Gender=Fem|Number=Sing	8	conj	B=LITOLOGIA	_
 5	A2	A2	PROPN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 6	,	,	PUNCT	_	_	4	punct	O	_
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
@@ -59824,7 +59824,7 @@
 39	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	40	det	O	_
 40	intercamadas	intercamada	NOUN	_	Gender=Fem|Number=Plur	37	obl	O	_
 41	,	,	PUNCT	_	_	37	punct	O	_
-42	assim	assim	ADV	_	_	45	cc	O	_
+42	assim	assim	ADV	_	_	45	cc	O	MWEPOS=CCONJ
 43	como	como	ADP	_	_	42	fixed	O	_
 44	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	45	det	O	_
 45	íons	íom	NOUN	_	Gender=Masc|Number=Plur	31	conj	O	_
@@ -60134,10 +60134,10 @@
 2	argila	argila	NOUN	_	Gender=Fem|Number=Sing	10	nsubj	B=LITOLOGIA	_
 3	A6	A6	PROPN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
-5	assim	assim	ADV	_	_	8	cc	O	_
+5	assim	assim	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6	como	como	ADP	_	_	5	fixed	O	_
 7	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	8	det	O	_
-8	demais	demais	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	2	nmod:appos	O	_
+8	demais	demais	PRON	_	Gender=Fem|Number=Plur|PronType=Ind	2	conj	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
 10	possui	possuir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 11	maior	grande	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
@@ -109562,10 +109562,10 @@
 24	de	de	ADP	_	_	25	case	O	_
 25	Tupi	Tupi	PROPN	_	Gender=Masc|Number=Sing	23	nmod	B=CAMPO	_
 26	,	,	PUNCT	_	_	20	punct	O	_
-27	assim	assim	ADV	_	_	30	cc	O	_
+27	assim	assim	ADV	_	_	30	cc	O	MWEPOS=CCONJ
 28	como	como	ADP	_	_	27	fixed	O	_
 29	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	30	det	O	_
-30	Campos	Campos	PROPN	_	Gender=Masc|Number=Plur	23	nmod:appos	O	_
+30	Campos	Campos	PROPN	_	Gender=Masc|Number=Plur	23	conj	O	_
 31	de	de	ADP	_	_	30	flat:name	O	_
 32	Parati	Parati	PROPN	_	Number=Sing	30	flat:name	B=CAMPO	_
 33	,	,	PUNCT	_	_	34	punct	O	_
@@ -110130,7 +110130,7 @@
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
 29	perfuração	perfuração	NOUN	_	Gender=Fem|Number=Sing	22	obl	O	_
 30	,	,	PUNCT	_	_	34	punct	O	_
-31	assim	assim	ADV	_	_	34	cc	O	_
+31	assim	assim	ADV	_	_	34	cc	O	MWEPOS=CCONJ
 32	como	como	ADP	_	_	31	fixed	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
 34	colapso	colapso	NOUN	_	Gender=Masc|Number=Sing	24	conj	O	_
@@ -110390,7 +110390,7 @@
 50	de	de	ADP	_	_	51	case	O	_
 51	produção	produção	NOUN	_	Gender=Fem|Number=Sing	49	nmod	O	_
 52	,	,	PUNCT	_	_	55	punct	O	_
-53	assim	assim	ADV	_	_	55	cc	O	_
+53	assim	assim	ADV	_	_	55	cc	O	MWEPOS=CCONJ
 54	como	como	ADP	_	_	53	fixed	O	_
 55	análise	análise	NOUN	_	Gender=Fem|Number=Sing	28	conj	O	_
 56	geomecânica	geomecânico	ADJ	_	Gender=Fem|Number=Sing	55	amod	O	_
@@ -110435,7 +110435,7 @@
 90	de	de	ADP	_	_	91	case	O	_
 91	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	89	nmod	B=QUÍMICA	_
 92	,	,	PUNCT	_	_	96	punct	O	_
-93	assim	assim	ADV	_	_	96	cc	O	_
+93	assim	assim	ADV	_	_	96	cc	O	MWEPOS=CCONJ
 94	como	como	ADP	_	_	93	fixed	O	_
 95	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	96	det	O	_
 96	resistência	resistência	NOUN	_	Gender=Fem|Number=Sing	79	conj	O	_
@@ -118897,10 +118897,10 @@
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
 12	Pré-sal	Pré-sal	PROPN	_	Gender=Masc|Number=Sing	9	nmod	O	_
 13	,	,	PUNCT	_	_	17	punct	O	_
-14	assim	assim	ADV	_	_	17	cc	O	_
+14	assim	assim	ADV	_	_	17	cc	O	MWEPOS=CCONJ
 15	como	como	ADP	_	_	14	fixed	O	_
 16	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	17	det	O	_
-17	alcance	alcance	NOUN	_	Gender=Masc|Number=Sing	1	nmod:appos	O	_
+17	alcance	alcance	NOUN	_	Gender=Masc|Number=Sing	1	conj	O	_
 18-19	da	_	_	_	_	_	_	_	_
 18	de	de	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
@@ -120930,9 +120930,9 @@
 
 # text = Assim como o de Troll, este separador trabalha em lâminas de água mais rasas (200 metros) que as encontradas no Pré-sal brasileiro..
 # sent_id = 334-20150521-MONOGRAFIA_0-311
-1	Assim	assim	ADV	_	_	3	cc	O	_
+1	Assim	assim	ADV	_	_	3	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
-3	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	9	obl	O	_
+3	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	8	conj	O	_
 4	de	de	ADP	_	_	5	case	O	_
 5	Troll	Troll	PROPN	_	Gender=Masc|Number=Sing	3	nmod	O	_
 6	,	,	PUNCT	_	_	3	punct	O	_
@@ -122482,7 +122482,7 @@
 15	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
 16	poço	poço	NOUN	_	Gender=Masc|Number=Sing	13	nmod	O	_
 17	,	,	PUNCT	_	_	21	punct	O	_
-18	assim	assim	ADV	_	_	21	cc	O	_
+18	assim	assim	ADV	_	_	21	cc	O	MWEPOS=CCONJ
 19	como	como	ADP	_	_	18	fixed	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
 21	bombeamento	bombeamento	NOUN	_	Gender=Masc|Number=Sing	10	conj	O	_
@@ -124459,8 +124459,8 @@
 10	de	de	ADP	_	_	11	case	O	_
 11	petróleo	petróleo	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=FLUIDO	_
 12	,	,	PUNCT	_	_	13	punct	O	_
-13	assim	assim	ADV	_	_	17	cc	O	_
-14	como	como	ADV	_	_	13	fixed	O	_
+13	assim	assim	ADV	_	_	17	cc	O	MWEPOS=CCONJ
+14	como	como	ADP	_	_	13	fixed	O	_
 15-16	nas	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
 16	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	17	det	O	_
@@ -126530,8 +126530,8 @@
 20	de	de	ADP	_	_	21	case	O	_
 21	hidrocarboneto	hidrocarboneto	NOUN	_	Gender=Masc|Number=Sing	19	nmod	O	_
 22	,	,	PUNCT	_	_	16	punct	O	_
-23	assim	assim	ADV	_	_	27	cc	O	_
-24	como	como	ADV	_	_	23	fixed	O	_
+23	assim	assim	ADV	_	_	27	cc	O	MWEPOS=CCONJ
+24	como	como	ADP	_	_	23	fixed	O	_
 25-26	da	_	_	_	_	_	_	_	_
 25	de	de	ADP	_	_	27	case	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
@@ -128004,7 +128004,7 @@
 15	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	16	det	O	_
 16	colunas	coluna	NOUN	_	Gender=Fem|Number=Plur	13	nmod	O	_
 17	,	,	PUNCT	_	_	9	punct	O	_
-18	assim	assim	ADV	_	_	22	cc	O	_
+18	assim	assim	ADV	_	_	22	cc	O	MWEPOS=CCONJ
 19	como	como	ADP	_	_	18	fixed	O	_
 20-21	na	_	_	_	_	_	_	_	_
 20	em	em	ADP	_	_	22	case	O	_
@@ -128629,8 +128629,8 @@
 37	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	38	det	O	_
 38	alimentação	alimentação	NOUN	_	Gender=Fem|Number=Sing	32	nmod	O	_
 39	,	,	PUNCT	_	_	42	punct	O	_
-40	assim	assim	ADV	_	_	42	cc	O	_
-41	como	como	ADV	_	_	40	fixed	O	_
+40	assim	assim	ADV	_	_	42	cc	O	MWEPOS=CCONJ
+41	como	como	ADP	_	_	40	fixed	O	_
 42	depende	depender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	22	conj	O	_
 43	também	também	ADV	_	_	42	advmod	O	_
 44-45	da	_	_	_	_	_	_	_	_
@@ -132956,9 +132956,9 @@
 11	estado	estado	NOUN	_	Gender=Masc|Number=Sing	4	obl	O	_
 12	vapor	vapor	ADJ	_	Gender=Masc|Number=Sing	11	amod	O	_
 13	,	,	PUNCT	_	_	16	punct	O	_
-14	assim	assim	ADV	_	_	16	cc	O	_
+14	assim	assim	ADV	_	_	16	cc	O	MWEPOS=CCONJ
 15	como	como	ADP	_	_	14	fixed	O	_
-16	condensado	condensado	NOUN	_	Gender=Masc|Number=Sing	11	nmod:appos	O	_
+16	condensado	condensado	NOUN	_	Gender=Masc|Number=Sing	11	conj	O	_
 17	e	e	CCONJ	_	_	18	cc	O	_
 18	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	16	conj	B=QUÍMICA	_
 19-20	no	_	_	_	_	_	_	_	_
@@ -134815,8 +134815,8 @@
 4	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	5	det	O	_
 5	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 6	,	,	PUNCT	_	_	10	punct	O	_
-7	assim	assim	ADV	_	_	10	cc	O	_
-8	como	como	ADV	_	_	7	fixed	O	_
+7	assim	assim	ADV	_	_	10	cc	O	MWEPOS=CCONJ
+8	como	como	ADP	_	_	7	fixed	O	_
 9	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	10	det	O	_
 10	frações	fração	NOUN	_	Gender=Fem|Number=Plur	2	conj	O	_
 11-12	dos	_	_	_	_	_	_	_	_
@@ -135586,7 +135586,7 @@
 19	de	de	ADP	_	_	20	case	O	_
 20	membranas	membrana	NOUN	_	Gender=Fem|Number=Plur	18	nmod	O	_
 21	,	,	PUNCT	_	_	26	punct	O	_
-22	assim	assim	ADV	_	_	26	cc	O	_
+22	assim	assim	ADV	_	_	26	cc	O	MWEPOS=CCONJ
 23	como	como	ADP	_	_	22	fixed	O	_
 24	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 25	respectivos	respectivo	ADJ	_	Gender=Masc|Number=Plur	26	amod	O	_
@@ -153516,7 +153516,7 @@
 57	de	de	ADP	_	_	56	flat:name	O	_
 58	Santos	Santos	PROPN	_	Number=Sing	56	flat:name	B=BACIA|B=LOCAL:POL	_
 59	,	,	PUNCT	_	_	64	punct	O	_
-60	assim	assim	ADV	_	_	64	cc	O	_
+60	assim	assim	ADV	_	_	64	cc	O	MWEPOS=CCONJ
 61	como	como	ADP	_	_	60	fixed	O	_
 62	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	64	det	O	_
 63	demais	demais	DET	_	Gender=Fem|Number=Plur|PronType=Ind	64	det	O	_
@@ -156403,10 +156403,10 @@
 
 # text = Assim como os dados de sísmica e poços, os dados de gravimetria foram disponibilizados pelo Banco de Dados de Exploração e Produção (BDEP) e requeridos dentro da política de gratuidade para universidades..
 # sent_id = 258-20160602-TESEMSC_0-332
-1	Assim	assim	ADV	_	_	4	advmod	O	_
+1	Assim	assim	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	dados	dado	NOUN	_	Gender=Masc|Number=Plur	15	obl	O	_
+4	dados	dado	NOUN	_	Gender=Masc|Number=Plur	11	conj	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	sísmica	sísmica	NOUN	_	Gender=Fem|Number=Sing	4	nmod	B=MÉTODO	_
 7	e	e	CCONJ	_	_	8	cc	O	_
@@ -159403,7 +159403,7 @@
 
 # text = Assim como o horizonte anterior, compreende sedimentos conglomeráticos avermelhados da Formação Santos; sedimentos arenosos folhelhos, siltitos e argilosos da Formação Juréia; folhelhos e argilitos da Formação Itajaí-Açu; e arenitos turbiditicos do Membro Ilha Bela (MOREIRA et al. 2007)..
 # sent_id = 258-20160602-TESEMSC_0-407
-1	Assim	assim	ADV	_	_	4	cc	O	_
+1	Assim	assim	ADV	_	_	4	case	O	MWEPOS=ADP
 2	como	como	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
@@ -159535,10 +159535,10 @@
 2	esse	esse	DET	_	Gender=Masc|Number=Sing|PronType=Dem	3	det	O	_
 3	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
 4	,	,	PUNCT	_	_	3	punct	O	_
-5	assim	assim	ADV	_	_	8	advmod	O	_
+5	assim	assim	ADV	_	_	8	cc	O	MWEPOS=CCONJ
 6	como	como	ADP	_	_	5	fixed	O	_
 7	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	8	det	O	_
-8	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	13	obl	O	_
+8	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	3	conj	O	_
 9	Eosantoniano	Eosantoniano	PROPN	_	Gender=Masc|Number=Sing	8	nmod	O	_
 10	e	e	CCONJ	_	_	11	cc	O	_
 11	Intracampaneano	Intracampaneano	PROPN	_	Gender=Masc|Number=Sing	9	conj	O	_
@@ -159761,10 +159761,10 @@
 13	seções	seção	NOUN	_	Gender=Fem|Number=Plur	9	obl	O	_
 14	,	,	PUNCT	_	_	4	punct	O	_
 15	e	e	CCONJ	_	_	21	cc	O	_
-16	assim	assim	ADV	_	_	21	advmod	O	_
+16	assim	assim	ADV	_	_	19	case	O	MWEPOS=ADP
 17	como	como	ADV	_	_	16	fixed	O	_
 18	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	19	det	O	_
-19	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	21	nsubj	O	_
+19	horizonte	horizonte	NOUN	_	Gender=Masc|Number=Sing	21	obl	O	_
 20	anterior	anterior	ADJ	_	Gender=Masc|Number=Sing	19	amod	O	_
 21	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	4	conj	O	_
 22	ser	ser	AUX	_	VerbForm=Inf	23	aux:pass	O	_
@@ -162473,10 +162473,10 @@
 
 # text = Assim como os dados de anomalia gravimétrica os mapas de contorno estrutural podem auxiliar em uma primeira fase de identificação de trends estruturais e possíveis falhas..
 # sent_id = 258-20160602-TESEMSC_0-475
-1	Assim	assim	ADV	_	_	13	advmod	O	_
-2	como	como	ADV	_	_	1	fixed	O	_
+1	Assim	assim	ADV	_	_	4	cc	O	MWEPOS=CCONJ
+2	como	como	ADP	_	_	1	fixed	O	_
 3	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	4	det	O	_
-4	dados	dado	NOUN	_	Gender=Masc|Number=Plur	13	nsubj	O	_
+4	dados	dado	NOUN	_	Gender=Masc|Number=Plur	9	conj	O	_
 5	de	de	ADP	_	_	6	case	O	_
 6	anomalia	anomalia	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 7	gravimétrica	gravimétrica	ADJ	_	Gender=Fem|Number=Sing	6	amod	O	_
@@ -163173,11 +163173,11 @@
 
 # text = Assim como o primeiro conjunto de dados analisados na direção 82º a análise do segundo e terceiro conjunto apresentam falhas normais que interceptam os horizontes embasamento e pré-sal e falhas associadas à halocinese, as quais ocorrem predominantemente sobre as estruturas salinas (Figura 6.5 e 6.6)..
 # sent_id = 258-20160602-TESEMSC_0-490
-1	Assim	assim	ADV	_	_	21	advmod	O	_
-2	como	como	ADV	_	_	1	fixed	O	_
+1	Assim	assim	ADV	_	_	5	cc	O	MWEPOS=CCONJ
+2	como	como	ADP	_	_	1	fixed	O	_
 3	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 4	primeiro	primeiro	ADJ	_	Gender=Masc|NumType=Ord|Number=Sing	5	amod	O	_
-5	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	21	nsubj	O	_
+5	conjunto	conjunto	NOUN	_	Gender=Masc|Number=Sing	14	conj	O	_
 6	de	de	ADP	_	_	7	case	O	_
 7	dados	dado	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 8	analisados	analisar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	7	acl	O	_
@@ -166458,10 +166458,10 @@
 55	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	56	aux:pass	O	_
 56	descrito	descrever	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	48	acl:relcl	O	_
 57	,	,	PUNCT	_	_	56	punct	O	_
-58	assim	assim	ADV	_	_	56	advmod	O	_
+58	assim	assim	ADV	_	_	61	cc	O	MWEPOS=CCONJ
 59	como	como	ADP	_	_	58	fixed	O	_
 60	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	61	det	O	_
-61	equipamentos	equipamento	NOUN	_	Gender=Masc|Number=Plur	56	xcomp	B=INSTRUMENTO	_
+61	equipamentos	equipamento	NOUN	_	Gender=Masc|Number=Plur	53	conj	B=INSTRUMENTO	_
 62	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	61	acl	O	_
 63	,	,	PUNCT	_	_	65	punct	O	_
 64	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	65	det	O	_
@@ -168290,10 +168290,10 @@
 
 # text = Assim como a densidade, a viscosidade é influenciada pela quantidade de resinas e asfaltenos..
 # sent_id = 77-20141209-TESEMSC_0-83
-1	Assim	assim	ADV	_	_	4	cc	O	_
+1	Assim	assim	ADV	_	_	4	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
-4	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
+4	densidade	densidade	NOUN	_	Gender=Fem|Number=Sing	7	conj	O	_
 5	,	,	PUNCT	_	_	4	punct	O	_
 6	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	7	det	O	_
 7	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	9	nsubj:pass	O	_
@@ -178708,10 +178708,10 @@
 8	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	4	nmod	O	_
 9	aumentar	aumentar	VERB	_	VerbForm=Inf	2	xcomp	O	_
 10	,	,	PUNCT	_	_	9	punct	O	_
-11	assim	assim	ADV	_	_	9	advmod	O	_
+11	assim	assim	ADV	_	_	14	cc	O	MWEPOS=CCONJ
 12	como	como	ADP	_	_	11	fixed	O	_
 13	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	14	det	O	_
-14	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	9	xcomp	O	_
+14	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	4	conj	O	_
 15	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	14	amod	O	_
 16	para	para	ADP	_	_	17	case	O	_
 17	limpeza	limpeza	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
@@ -181398,10 +181398,10 @@
 22	mistura	mistura	NOUN	_	Gender=Fem|Number=Sing	18	nmod	O	_
 23	aumente	aumentar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	14	ccomp	O	_
 24	,	,	PUNCT	_	_	23	punct	O	_
-25	assim	assim	ADV	_	_	23	advmod	O	_
+25	assim	assim	ADV	_	_	28	cc	O	MWEPOS=CCONJ
 26	como	como	ADP	_	_	25	fixed	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
-28	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	23	xcomp	O	_
+28	tempo	tempo	NOUN	_	Gender=Masc|Number=Sing	18	conj	O	_
 29	necessário	necessário	ADJ	_	Gender=Masc|Number=Sing	28	amod	O	_
 30	para	para	ADP	_	_	31	case	O	_
 31	limpeza	limpeza	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
@@ -186436,10 +186436,10 @@
 1	As	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	2	det	O	_
 2	medições	medição	NOUN	_	Gender=Fem|Number=Plur	17	nsubj:pass	O	_
 3	,	,	PUNCT	_	_	7	punct	O	_
-4	assim	assim	ADV	_	_	7	cc	O	_
+4	assim	assim	ADV	_	_	7	cc	O	MWEPOS=CCONJ
 5	como	como	ADP	_	_	4	fixed	O	_
 6	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	7	det	O	_
-7	incertezas	incerteza	NOUN	_	Gender=Fem|Number=Plur	2	nmod:appos	O	_
+7	incertezas	incerteza	NOUN	_	Gender=Fem|Number=Plur	2	conj	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	medição	medição	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 10-11	das	_	_	_	_	_	_	_	_
@@ -187616,9 +187616,9 @@
 7	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	8	det	O	_
 8	exaustão	exaustão	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
 9	,	,	PUNCT	_	_	12	punct	O	_
-10	assim	assim	ADV	_	_	12	cc	O	_
+10	assim	assim	ADV	_	_	12	cc	O	MWEPOS=CCONJ
 11	como	como	ADP	_	_	10	fixed	O	_
-12	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	5	xcomp	O	_
+12	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	2	conj	O	_
 13	de	de	ADP	_	_	14	case	O	_
 14	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	12	nmod	B=QUÍMICA	_
 15	,	,	PUNCT	_	_	12	punct	O	_
@@ -192192,7 +192192,7 @@
 42	não	não	ADV	_	Polarity=Neg	43	advmod	O	_
 43	voláteis	volátel	ADJ	_	Gender=Masc|Number=Plur	40	conj	O	_
 44	,	,	PUNCT	_	_	47	punct	O	_
-45	assim	assim	ADV	_	_	47	cc	O	_
+45	assim	assim	ADV	_	_	47	cc	O	MWEPOS=CCONJ
 46	como	como	ADP	_	_	45	fixed	O	_
 47	compostos	compor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	43	conj	O	_
 48	monoméricos	monomérico	ADJ	_	Gender=Masc|Number=Plur	47	amod	O	_
@@ -202186,7 +202186,7 @@
 28	de	de	ADP	_	_	29	case	O	_
 29	catalisador	catalisador	NOUN	_	Gender=Masc|Number=Sing	27	nmod	B=QUÍMICA	_
 30	,	,	PUNCT	_	_	34	punct	O	_
-31	assim	assim	ADV	_	_	34	cc	O	_
+31	assim	assim	ADV	_	_	34	cc	O	MWEPOS=CCONJ
 32	como	como	ADP	_	_	31	fixed	O	_
 33	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	34	det	O	_
 34	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	18	conj	B=FLUIDO	_
@@ -238033,7 +238033,7 @@
 22	relevo	relevo	NOUN	_	Gender=Masc|Number=Sing	18	conj	O	_
 23	adjacente	adjacente	ADJ	_	Gender=Masc|Number=Sing	22	amod	O	_
 24	,	,	PUNCT	_	_	3	punct	O	_
-25	assim	assim	ADV	_	_	29	cc	O	_
+25	assim	assim	ADV	_	_	29	cc	O	MWEPOS=CCONJ
 26	como	como	ADP	_	_	25	fixed	O	_
 27-28	na	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
@@ -281372,8 +281372,8 @@
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
 26	parâmetros	parâmetro	NOUN	_	Gender=Masc|Number=Plur	23	nmod	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
-28	assim	assim	ADV	_	_	30	cc	O	_
-29	como	como	ADV	_	_	28	fixed	O	_
+28	assim	assim	ADV	_	_	30	cc	O	MWEPOS=CCONJ
+29	como	como	ADP	_	_	28	fixed	O	_
 30	compatibilizar	compatibilizar	VERB	_	VerbForm=Inf	4	conj	O	_
 31	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	32	det	O	_
 32	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	30	obj	O	_
@@ -288078,9 +288078,9 @@
 7	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	8	aux:pass	O	_
 8	apresentado	apresentar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	0	root	O	_
 9	,	,	PUNCT	_	_	8	punct	O	_
-10	assim	assim	ADV	_	_	8	advmod	O	_
-11	como	como	ADP	_	_	12	case	O	_
-12	conceitos	conceito	NOUN	_	Gender=Masc|Number=Plur	8	xcomp	O	_
+10	assim	assim	ADV	_	_	12	cc	O	MWEPOS=CCONJ
+11	como	como	ADP	_	_	10	fixed	O	_
+12	conceitos	conceito	NOUN	_	Gender=Masc|Number=Plur	2	conj	O	_
 13	sobre	sobre	ADP	_	_	14	case	O	_
 14	polímeros	polímero	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
 15	,	,	PUNCT	_	_	16	punct	O	_
@@ -301627,10 +301627,10 @@
 13	poços	poço	NOUN	_	Gender=Masc|Number=Plur	11	nmod	B=POÇO:TIPO	_
 14	verticais	vertical	ADJ	_	Gender=Masc|Number=Plur	13	amod	I=POÇO:TIPO	_
 15	,	,	PUNCT	_	_	9	punct	O	_
-16	assim	assim	ADV	_	_	19	cc	O	_
+16	assim	assim	ADV	_	_	19	cc	O	MWEPOS=CCONJ
 17	como	como	ADP	_	_	16	fixed	O	_
 18	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	19	det	O	_
-19	permeabilidades	permeabilidade	NOUN	_	Gender=Fem|Number=Plur	13	nmod:appos	O	_
+19	permeabilidades	permeabilidade	NOUN	_	Gender=Fem|Number=Plur	13	conj	O	_
 20	horizontal	horizontal	ADJ	_	Gender=Fem|Number=Sing	19	amod	O	_
 21	e	e	CCONJ	_	_	22	cc	O	_
 22	vertical	vertical	ADJ	_	Gender=Fem|Number=Sing	20	conj	O	_
@@ -311123,12 +311123,12 @@
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	28	mark	O	_
 4	,	,	PUNCT	_	_	1	punct	O	_
-5	assim	assim	ADV	_	_	9	advmod	O	_
+5	assim	assim	ADV	_	_	9	cc	O	MWEPOS=CCONJ
 6	como	como	ADP	_	_	5	fixed	O	_
 7-8	nos	_	_	_	_	_	_	_	_
 7	em	em	ADP	_	_	9	case	O	_
 8	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	9	det	O	_
-9	experimentos	experimento	NOUN	_	Gender=Masc|Number=Plur	28	obl	O	_
+9	experimentos	experimento	NOUN	_	Gender=Masc|Number=Plur	26	conj	O	_
 10	realizados	realizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	acl	O	_
 11	com	com	ADP	_	_	12	case	O	_
 12	PHPA	PHPA	PROPN	_	Gender=Masc|Number=Sing	10	obl	B=QUÍMICA	_
@@ -311375,12 +311375,12 @@
 
 # text = Assim como na apresentação dos resultados para a PHPA, o retorno da vazão de óleo para os experimentos realizados com a GX será tratado também na forma relativa, ou seja, a fração de retorno obtida em relação à vazão de óleo inicial..
 # sent_id = 119-20141209-TESEMSC_0-717
-1	Assim	assim	ADV	_	_	5	case	O	_
+1	Assim	assim	ADV	_	_	5	cc	O	MWEPOS=CCONJ
 2	como	como	ADP	_	_	1	fixed	O	_
 3-4	na	_	_	_	_	_	_	_	_
 3	em	em	ADP	_	_	5	case	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
-5	apresentação	apresentação	NOUN	_	Gender=Fem|Number=Sing	28	obl	O	_
+5	apresentação	apresentação	NOUN	_	Gender=Fem|Number=Sing	14	conj	O	_
 6-7	dos	_	_	_	_	_	_	_	_
 6	de	de	ADP	_	_	8	case	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
@@ -312068,7 +312068,7 @@
 
 # text = Assim como nos experimentos com PHPA, aqui também foram utilizadas as Equações 3.5, 3.6 e 4.2 para o cálculo das saturações remanescentes de óleo ao final da invasão e do fluxo reverso, e do fluido invasor, ao final do fluxo reverso..
 # sent_id = 119-20141209-TESEMSC_0-734
-1	Assim	assim	ADV	_	_	5	advmod	O	_
+1	Assim	assim	ADV	_	_	5	case	O	MWEPOS=ADP
 2	como	como	ADP	_	_	1	fixed	O	_
 3-4	nos	_	_	_	_	_	_	_	_
 3	em	em	ADP	_	_	5	case	O	_
@@ -314075,10 +314075,10 @@
 5	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	4	nsubj	O	_
 6	que	que	SCONJ	_	_	17	mark	O	_
 7	,	,	PUNCT	_	_	4	punct	O	_
-8	assim	assim	ADV	_	_	17	advmod	O	_
+8	assim	assim	ADV	_	_	11	cc	O	MWEPOS=CCONJ
 9	como	como	ADP	_	_	8	fixed	O	_
 10	em	em	ADP	_	_	11	case	O	_
-11	Bourgoyne	Bourgoyne	PROPN	_	Gender=Masc|Number=Sing	17	obl	O	_
+11	Bourgoyne	Bourgoyne	PROPN	_	Gender=Masc|Number=Sing	3	conj	O	_
 12	,	,	PUNCT	_	_	11	flat:name	O	_
 13	1986	1986	NUM	_	NumType=Card	11	flat:name	O	_
 14	,	,	PUNCT	_	_	11	punct	O	_
@@ -332668,11 +332668,11 @@
 17	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	18	det	O	_
 18	tempos	tempo	NOUN	_	Gender=Masc|Number=Plur	15	nmod	O	_
 19	,	,	PUNCT	_	_	24	punct	O	_
-20	assim	assim	ADV	_	_	24	cc	O	_
+20	assim	assim	ADV	_	_	24	cc	O	MWEPOS=CCONJ
 21	como	como	ADP	_	_	20	fixed	O	_
 22	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	24	det	O	_
 23	altas	alto	ADJ	_	Gender=Fem|Number=Plur	24	amod	O	_
-24	temperaturas	temperatura	NOUN	_	Gender=Fem|Number=Plur	12	obl	O	_
+24	temperaturas	temperatura	NOUN	_	Gender=Fem|Number=Plur	2	conj	O	_
 25	,	,	PUNCT	_	_	27	punct	O	_
 26	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	27	det	O	_
 27	falta	falta	NOUN	_	Gender=Fem|Number=Sing	24	conj	O	_
