commit dfa248aeea78409c6418689d1168a49fe42efd8c
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Sat Jul 10 05:21:23 2021 -0300

    correções de obl e nmod

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index fb1c24d..50324d0 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -1708,10 +1708,10 @@
 # sent_id = 247-20140910-MONOGRAFIA_0-50
 1	Onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	3	parataxis	O	_
 2	:	:	PUNCT	_	_	3	punct	O	_
-3	V=	V=	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
+3	V=	V=	PROPN	_	Gender=Masc|Number=Sing	0	root	O	tokenização
 4	velocidade	velocidade	NOUN	_	Gender=Fem|Number=Sing	3	flat:name	O	_
 5	,	,	PUNCT	_	_	6	punct	O	_
-6	∆	∆	NOUN	_	Gender=Masc|Number=Sing	3	appos	O	_
+6	∆	∆	NOUN	_	Gender=Masc|Number=Sing	3	conj	O	_
 7	S=	S=	PROPN	_	Number=Sing	6	flat:name	O	_
 8	espessura	espessura	NOUN	_	Gender=Fem|Number=Sing	6	nmod	O	_
 9-10	da	_	_	_	_	_	_	_	_
@@ -1734,7 +1734,7 @@
 24-25	dos	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	23	fixed	O	_
 25	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	26	det	O	_
-26	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	3	obl	O	_
+26	sedimentos	sedimento	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
 27	.	.	PUNCT	_	_	3	punct	O	_
 28	.	.	PUNCT	_	_	3	punct	O	_
 
@@ -6211,7 +6211,7 @@
 4	e	e	CCONJ	_	_	5	cc	O	_
 5	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	1	conj	O	_
 6	com	com	ADP	_	_	7	case	O	_
-7	seixos	seixo	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
+7	seixos	seixo	NOUN	_	Gender=Masc|Number=Plur	5	nmod	O	_
 8	de	de	ADP	_	_	9	case	O	_
 9	vulcânicas	vulcânica	NOUN	_	Gender=Fem|Number=Plur	7	nmod	B=LITOLOGIA	_
 10	básicas	básico	ADJ	_	Gender=Fem|Number=Plur	9	amod	O	_
@@ -16614,16 +16614,16 @@
 5	em	em	ADP	_	_	6	case	O	_
 6	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	1	nmod	O	_
 7	contínuo	contínuo	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
-8	,	,	PUNCT	_	_	9	punct	O	_
+8	,	,	PUNCT	_	_	12	punct	O	_
 9	através	através	ADV	_	_	12	case	O	MWEPOS=ADP
 10-11	do	_	_	_	_	_	_	_	_
 10	de	de	ADP	_	_	9	fixed	O	_
 11	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	12	det	O	_
-12	orifício	orifício	NOUN	_	Gender=Masc|Number=Sing	1	obl	O	_
+12	orifício	orifício	NOUN	_	Gender=Masc|Number=Sing	1	nmod	O	_
 13	Furol	Furol	PROPN	_	Gender=Masc|Number=Sing	12	nmod	O	_
-14	,	,	PUNCT	_	_	9	punct	O	_
+14	,	,	PUNCT	_	_	16	punct	O	_
 15	sob	sob	ADP	_	_	16	case	O	_
-16	condições	condição	NOUN	_	Gender=Fem|Number=Plur	1	nmod	O	_
+16	condições	condição	NOUN	_	Gender=Fem|Number=Plur	12	nmod	O	_
 17	específicas	específico	ADJ	_	Gender=Fem|Number=Plur	16	amod	O	_
 18	.	.	PUNCT	_	_	1	punct	O	_
 19	.	.	PUNCT	_	_	1	punct	O	_
@@ -30092,7 +30092,7 @@
 11	(	(	PUNCT	_	_	12	punct	O	_
 12	LSDi	LSDi	PROPN	_	Gender=Fem|Number=Sing	7	appos	O	_
 13	)	)	PUNCT	_	_	12	punct	O	_
-14	,	,	PUNCT	_	_	4	punct	O	_
+14	,	,	PUNCT	_	_	16	punct	O	_
 15	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	16	nsubj	O	_
 16	marca	marcar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	7	acl:relcl	O	_
 17	passagem	passagem	NOUN	_	Gender=Fem|Number=Sing	16	obj	O	_
@@ -30106,7 +30106,7 @@
 25	Fl	Fl	PROPN	_	Gender=Fem|Number=Sing	24	flat:name	O	_
 26	)	)	PUNCT	_	_	24	punct	O	_
 27	em	em	ADP	_	_	28	case	O	_
-28	contato	contato	NOUN	_	Gender=Masc|Number=Sing	16	obl	O	_
+28	contato	contato	NOUN	_	Gender=Masc|Number=Sing	20	obl	O	_
 29	com	com	ADP	_	_	30	case	O	_
 30	conglomerados	conglomerado	NOUN	_	Gender=Masc|Number=Plur	28	nmod	B=LITOLOGIA	_
 31	(	(	PUNCT	_	_	32	punct	O	_
@@ -30117,7 +30117,7 @@
 36	matriz	matriz	NOUN	_	Gender=Fem|Number=Sing	30	conj	O	_
 37	sustentados	sustentado	ADJ	_	Gender=Masc|Number=Plur	36	compound	O	_
 38	a	a	ADP	_	_	39	case	O	_
-39	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	36	obl	B=LITOLOGIA	_
+39	arenitos	arenito	NOUN	_	Gender=Masc|Number=Plur	36	nmod	B=LITOLOGIA	_
 40	muito	muito	ADV	_	_	41	advmod	O	_
 41	grossos	grosso	ADJ	_	Gender=Masc|Number=Plur	39	amod	O	_
 42	.	.	PUNCT	_	_	4	punct	O	_
@@ -98646,7 +98646,7 @@
 
 # text = Onde:
 # sent_id = 161-20150810-MONOGRAFIA_0-633
-1	Onde	onde	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	4	obl	O	_
+1	Onde	onde	ADV	_	Gender=Masc|Number=Sing	4	advmod	O	_
 2	:	:	PUNCT	_	_	4	punct	O	_
 3			DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	EOR	EOR	PROPN	_	Gender=Fem|Number=Sing	0	root	O	_
@@ -98668,12 +98668,12 @@
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
 20	reservatório	reservatório	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
 21	(	(	PUNCT	_	_	22	punct	O	_
-22	milhões	milhão	NUM	_	NumType=Card	20	appos	O	_
+22	milhões	milhão	NUM	_	NumType=Card	11	appos	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	barris	barri	NOUN	_	Gender=Masc|Number=Plur	22	nmod	O	_
 25	)	)	PUNCT	_	_	22	punct	O	_
 26			NUM	_	NumType=Card	27	nummod	O	_
-27	%	%	SYM	_	_	17	obl	O	_
+27	%	%	SYM	_	_	4	conj	O	_
 28	EXTRA	EXTRA	PROPN	_	Gender=Masc|Number=Sing	27	compound	O	_
 29	=	=	PUNCT	_	_	30	punct	O	_
 30	porcentagem	porcentagem	NOUN	_	Gender=Fem|Number=Sing	27	appos	O	_
@@ -98682,7 +98682,7 @@
 33	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	30	nmod	B=FLUIDO	_
 34	recuperável	recuperável	ADJ	_	Gender=Masc|Number=Sing	33	amod	O	_
 35	em	em	ADP	_	_	36	case	O	_
-36	razão	razão	NOUN	_	Gender=Fem|Number=Sing	30	nmod	O	_
+36	razão	razão	NOUN	_	Gender=Fem|Number=Sing	34	obl	O	_
 37-38	da	_	_	_	_	_	_	_	_
 37	de	de	ADP	_	_	39	case	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
@@ -98690,7 +98690,7 @@
 40	de	de	ADP	_	_	41	case	O	_
 41	CO2	CO2	PROPN	_	Gender=Masc|Number=Sing	39	nmod	B=QUÍMICA	_
 42			PUNCT	_	_	43	punct	O	_
-43	OOIP	OOIP	PROPN	_	Gender=Fem|Number=Sing	41	conj	O	_
+43	OOIP	OOIP	PROPN	_	Gender=Fem|Number=Sing	4	conj	O	_
 44	=	=	PUNCT	_	_	45	punct	O	_
 45	Volume	volume	NOUN	_	Gender=Masc|Number=Sing	43	appos	O	_
 46	original	original	ADJ	_	Gender=Masc|Number=Sing	45	amod	O	_
@@ -98707,7 +98707,7 @@
 56	de	de	ADP	_	_	57	case	O	_
 57	barris	barri	NOUN	_	Gender=Masc|Number=Plur	55	nmod	O	_
 58	)	)	PUNCT	_	_	55	punct	O	_
-59			PUNCT	_	_	27	punct	O	_
+59			PUNCT	_	_	4	punct	O	_
 
 # text = C = fator de contato do CO2 com o óleo
 # sent_id = 161-20150810-MONOGRAFIA_0-634
@@ -166439,7 +166439,7 @@
 36	,	,	PUNCT	_	_	37	punct	O	_
 37	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	34	acl	O	_
 38	de	de	ADP	_	_	39	case	O	_
-39	linha	linha	NOUN	_	Gender=Fem|Number=Sing	37	obl	O	_
+39	linha	linha	NOUN	_	Gender=Fem|Number=Sing	37	nmod	O	_
 40	,	,	PUNCT	_	_	41	punct	O	_
 41	molhabilidade	molhabilidade	NOUN	_	Gender=Fem|Number=Sing	39	conj	O	_
 42	e	e	CCONJ	_	_	43	cc	O	_
@@ -169234,15 +169234,15 @@
 8	de	de	ADP	_	_	9	case	O	_
 9	carga	carga	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
 10	por	por	ADP	_	_	11	case	O	_
-11	atrito	atrito	NOUN	_	Gender=Masc|Number=Sing	9	nmod	B=LITOLOGIA	_
+11	atrito	atrito	NOUN	_	Gender=Masc|Number=Sing	7	nmod	B=LITOLOGIA	_
 12	chega	chegar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 13	a	a	SCONJ	_	_	17	mark	O	_
 14	ser	ser	AUX	_	VerbForm=Inf	17	cop	O	_
 15	apenas	apenas	ADV	_	_	17	advmod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	3	3	NUM	_	NumType=Card	12	obl	O	_
-18	a	a	ADP	_	_	20	case	O	_
-19	25	25	NUM	_	NumType=Card	20	nummod	O	_
+18	a	a	ADP	_	_	19	case	O	_
+19	25	25	NUM	_	NumType=Card	17	conj	O	_
 20	%	%	SYM	_	_	17	nmod	O	_
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
@@ -169254,7 +169254,7 @@
 27-28	à	_	_	_	_	_	_	_	_
 27	a	a	ADP	_	_	26	fixed	O	_
 28	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	29	det	O	_
-29	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	20	obl	O	_
+29	coluna	coluna	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 30	hidrostática	hidrostático	ADJ	_	Gender=Fem|Number=Sing	29	amod	O	_
 31	.	.	PUNCT	_	_	12	punct	O	_
 32	.	.	PUNCT	_	_	12	punct	O	_
@@ -169522,7 +169522,7 @@
 1	2.4	2.4	NUM	_	NumType=Card	2	nsubj	O	_
 2	Repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	0	root	O	_
 3	de	de	ADP	_	_	4	case	O	_
-4	linha	linha	NOUN	_	Gender=Fem|Number=Sing	2	obl	O	_
+4	linha	linha	NOUN	_	Gender=Fem|Number=Sing	2	nmod	O	_
 
 # text = Durante um escoamento de óleo pesado utilizando o método core flow podem ocorrer imprevistos que ocasionem uma parada abrupta das bombas de água e ou óleo..
 # sent_id = 77-20141209-TESEMSC_0-114
@@ -169692,7 +169692,7 @@
 9	de	de	ADP	_	_	11	case	O	_
 10	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	11	det	O	_
 11	linha	linha	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
-12	escoando	escoar	VERB	_	VerbForm=Ger	4	advcl	O	_
+12	escoando	escoar	VERB	_	VerbForm=Ger	5	advcl	O	_
 13	em	em	ADP	_	_	14	case	O	_
 14	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	12	obl	O	_
 15	core	core	X	_	_	14	nmod	O	_
@@ -169707,11 +169707,11 @@
 24-25	da	_	_	_	_	_	_	_	_
 24	de	de	ADP	_	_	26	case	O	_
 25	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	26	det	O	_
-26	linha	linha	NOUN	_	Gender=Fem|Number=Sing	23	obl	O	_
+26	linha	linha	NOUN	_	Gender=Fem|Number=Sing	23	nmod	O	_
 27	com	com	ADP	_	_	28	case	O	_
 28	água	água	NOUN	_	Gender=Fem|Number=Sing	26	nmod	B=FLUIDO	_
 29	(	(	PUNCT	_	_	30	punct	O	_
-30	“stop-and-go”	“stop-and-go”	NOUN	_	Gender=Masc|Number=Sing	28	appos	O	tokenização
+30	“stop-and-go”	“stop-and-go”	NOUN	_	Gender=Masc|Number=Sing	23	appos	O	tokenização
 31	)	)	PUNCT	_	_	30	punct	O	_
 32	.	.	PUNCT	_	_	4	punct	O	_
 33	.	.	PUNCT	_	_	4	punct	O	_
@@ -170024,7 +170024,7 @@
 11	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	9	nmod	O	_
 12	após	após	ADP	_	_	14	case	O	_
 13	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	14	det	O	_
-14	parada	parada	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
+14	parada	parada	NOUN	_	Gender=Fem|Number=Sing	11	nmod	O	_
 15	súbita	súbito	ADJ	_	Gender=Fem|Number=Sing	14	amod	O	_
 16	de	de	ADP	_	_	17	case	O	_
 17	bombas	bomba	NOUN	_	Gender=Fem|Number=Plur	14	nmod	B=INSTRUMENTO	_
@@ -170035,9 +170035,9 @@
 22	escoamento	escoamento	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
 23	de	de	ADP	_	_	24	case	O	_
 24	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	22	nmod	B=FLUIDO	_
-25	pesado	pesado	ADJ	_	Gender=Masc|Number=Sing	20	amod	I=FLUIDO	_
+25	pesado	pesado	ADJ	_	Gender=Masc|Number=Sing	24	amod	I=FLUIDO	_
 26	e	e	CCONJ	_	_	27	cc	O	_
-27	água	água	NOUN	_	Gender=Fem|Number=Sing	22	conj	B=FLUIDO	_
+27	água	água	NOUN	_	Gender=Fem|Number=Sing	24	conj	B=FLUIDO	_
 28	em	em	ADP	_	_	29	case	O	_
 29	padrão	padrão	NOUN	_	Gender=Masc|Number=Sing	22	nmod	O	_
 30	coreflow	coreflow	ADJ	_	Gender=Masc|Number=Sing	29	amod	O	_
@@ -172231,7 +172231,7 @@
 11	de	de	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
 13	métodos	método	NOUN	_	Gender=Masc|Number=Plur	9	conj	B=MÉTODO	_
-14	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	13	acl	O	_
+14	utilizados	utilizar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	acl	O	_
 15-16	na	_	_	_	_	_	_	_	_
 15	em	em	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
@@ -172241,16 +172241,16 @@
 19	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	20	det	O	_
 20	experimentos	experimento	NOUN	_	Gender=Masc|Number=Plur	17	nmod	O	_
 21	de	de	ADP	_	_	24	case	O	_
-22	:	:	PUNCT	_	_	4	punct	O	_
+22	:	:	PUNCT	_	_	24	punct	O	_
 23	•	•	PUNCT	_	_	24	punct	O	_
 24	Repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	20	nmod	O	_
 25	assistida	assistir	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	24	acl	O	_
 26	com	com	ADP	_	_	27	case	O	_
 27	água	água	NOUN	_	Gender=Fem|Number=Sing	25	obl	B=FLUIDO	_
-28	,	,	PUNCT	_	_	25	punct	O	_
+28	,	,	PUNCT	_	_	31	punct	O	_
 29	após	após	ADP	_	_	31	case	O	_
 30	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	31	det	O	_
-31	parada	parada	NOUN	_	Gender=Fem|Number=Sing	24	obl	O	_
+31	parada	parada	NOUN	_	Gender=Fem|Number=Sing	24	nmod	O	_
 32	súbita	súbito	ADJ	_	Gender=Fem|Number=Sing	31	amod	O	_
 33-34	das	_	_	_	_	_	_	_	_
 33	de	de	ADP	_	_	35	case	O	_
@@ -172276,11 +172276,11 @@
 53	água	água	NOUN	_	Gender=Fem|Number=Sing	51	obl	B=FLUIDO	_
 54	adicionada	adicionar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	53	acl	O	_
 55	de	de	ADP	_	_	56	case	O	_
-56	desemulsificante	desemulsificante	NOUN	_	Gender=Masc|Number=Sing	53	nmod	B=QUÍMICA	_
-57	,	,	PUNCT	_	_	54	punct	O	_
+56	desemulsificante	desemulsificante	NOUN	_	Gender=Masc|Number=Sing	54	nmod	B=QUÍMICA	_
+57	,	,	PUNCT	_	_	60	punct	O	_
 58	após	após	ADP	_	_	60	case	O	_
 59	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	60	det	O	_
-60	parada	parada	NOUN	_	Gender=Fem|Number=Sing	50	obl	O	_
+60	parada	parada	NOUN	_	Gender=Fem|Number=Sing	50	nmod	O	_
 61	súbita	súbito	ADJ	_	Gender=Fem|Number=Sing	60	amod	O	_
 62-63	das	_	_	_	_	_	_	_	_
 62	de	de	ADP	_	_	64	case	O	_
@@ -176195,7 +176195,7 @@
 23	de	de	ADP	_	_	24	case	O	_
 24	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	22	nmod	O	_
 25	sem	sem	ADP	_	_	26	case	O	_
-26	desemulsificante	desemulsificante	NOUN	_	Gender=Masc|Number=Sing	24	obl	B=QUÍMICA	_
+26	desemulsificante	desemulsificante	NOUN	_	Gender=Masc|Number=Sing	24	nmod	B=QUÍMICA	_
 27	.	.	PUNCT	_	_	8	punct	O	_
 28	.	.	PUNCT	_	_	8	punct	O	_
 
@@ -178105,7 +178105,7 @@
 18-19	na	_	_	_	_	_	_	_	_
 18	em	em	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
+20	tubulação	tubulação	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 21	bem	bem	ADV	_	_	24	cc	O	MWEPOS=CCONJ
 22	como	como	ADP	_	_	21	fixed	O	_
 23	suas	seu	DET	_	Gender=Fem|Number=Plur|PronType=Prs	24	det	O	_
@@ -178124,7 +178124,7 @@
 35	tais	tal	DET	_	Gender=Fem|Number=Plur|PronType=Dem	38	case	O	_
 36	como	como	ADV	_	_	35	fixed	O	_
 37	:	:	PUNCT	_	_	38	punct	O	_
-38	vazão	vazão	NOUN	_	Gender=Fem|Number=Sing	24	nmod:appos	O	_
+38	vazão	vazão	NOUN	_	Gender=Fem|Number=Sing	32	nmod:appos	O	_
 39	de	de	ADP	_	_	40	case	O	_
 40	água	água	NOUN	_	Gender=Fem|Number=Sing	38	nmod	B=FLUIDO	_
 41-42	na	_	_	_	_	_	_	_	_
@@ -178146,7 +178146,7 @@
 55	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	56	det	O	_
 56	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	53	obl	O	_
 57	com	com	ADP	_	_	58	case	O	_
-58	água	água	NOUN	_	Gender=Fem|Number=Sing	56	obl	B=FLUIDO	_
+58	água	água	NOUN	_	Gender=Fem|Number=Sing	56	nmod	B=FLUIDO	_
 59	.	.	PUNCT	_	_	3	punct	O	_
 60	.	.	PUNCT	_	_	3	punct	O	_
 
@@ -179549,7 +179549,7 @@
 11	de	de	ADP	_	_	12	case	O	_
 12	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	10	nmod	O	_
 13	com	com	ADP	_	_	14	case	O	_
-14	água	água	NOUN	_	Gender=Fem|Number=Sing	12	obl	B=FLUIDO	_
+14	água	água	NOUN	_	Gender=Fem|Number=Sing	12	nmod	B=FLUIDO	_
 15	de	de	ADP	_	_	17	case	O	_
 16	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	linha	linha	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
@@ -179567,7 +179567,7 @@
 29	de	de	ADP	_	_	30	case	O	_
 30	pressão	pressão	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 31	em	em	ADP	_	_	32	case	O	_
-32	função	função	NOUN	_	Gender=Fem|Number=Sing	25	obl	O	_
+32	função	função	NOUN	_	Gender=Fem|Number=Sing	26	nmod	O	_
 33-34	do	_	_	_	_	_	_	_	_
 33	de	de	ADP	_	_	35	case	O	_
 34	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
@@ -179577,7 +179577,7 @@
 38	duas	dois	NUM	_	NumType=Card	39	nummod	O	_
 39	regiões	região	NOUN	_	Gender=Fem|Number=Plur	36	obj	O	_
 40	distintas	distinto	ADJ	_	Gender=Fem|Number=Plur	39	amod	O	_
-41	,	,	PUNCT	_	_	36	punct	O	_
+41	,	,	PUNCT	_	_	45	punct	O	_
 42	as	o	DET	_	Gender=Fem|Number=Plur|PronType=Art	43	det	O	_
 43	quais	qual	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	45	nsubj:pass	O	_
 44	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	45	aux:pass	O	_
@@ -181696,17 +181696,17 @@
 17	essa	esse	DET	_	Gender=Fem|Number=Sing|PronType=Dem	18	det	O	_
 18	temática	temático	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
 19	:	:	PUNCT	_	_	20	punct	O	_
-20	l.	l.	NOUN	_	Gender=Masc|Number=Sing	18	appos	O	_
+20	l.	l.	NOUN	_	Gender=Masc|Number=Sing	12	appos	O	_
 21	1.	1.	NUM	_	NumType=Card	20	nummod	O	_
 22	-	-	PUNCT	_	_	23	punct	O	_
-23	Realizar	realizar	VERB	_	VerbForm=Inf	20	conj	O	_
+23	Realizar	realizar	VERB	_	VerbForm=Inf	20	appos	O	_
 24	novos	novo	ADJ	_	Gender=Masc|Number=Plur	25	amod	O	_
 25	experimentos	experimento	NOUN	_	Gender=Masc|Number=Plur	23	obj	O	_
 26	de	de	ADP	_	_	27	case	O	_
-27	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	25	acl	O	_
+27	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	25	nmod	O	_
 28	para	para	ADP	_	_	30	case	O	_
 29	diferentes	diferente	DET	_	Gender=Masc|Number=Plur|PronType=Ind	30	det	O	_
-30	comprimentos	comprimento	NOUN	_	Gender=Masc|Number=Plur	27	obl	O	_
+30	comprimentos	comprimento	NOUN	_	Gender=Masc|Number=Plur	25	nmod	O	_
 31	de	de	ADP	_	_	32	case	O	_
 32	entupimento	entupimento	NOUN	_	Gender=Masc|Number=Sing	30	nmod	O	_
 33	,	,	PUNCT	_	_	34	punct	O	_
@@ -181729,7 +181729,7 @@
 49	linha	linha	NOUN	_	Gender=Fem|Number=Sing	47	nmod	O	_
 50	;	;	PUNCT	_	_	52	punct	O	_
 51	com	com	ADP	_	_	52	case	O	_
-52	controle	controle	NOUN	_	Gender=Masc|Number=Sing	45	nmod	O	_
+52	controle	controle	NOUN	_	Gender=Masc|Number=Sing	25	conj	O	_
 53	de	de	ADP	_	_	54	case	O	_
 54	temperatura	temperatura	NOUN	_	Gender=Fem|Number=Sing	52	nmod	O	_
 55	e	e	CCONJ	_	_	56	cc	O	_
@@ -181743,13 +181743,13 @@
 63	em	em	ADP	_	_	64	case	O	_
 64	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	66	obl	O	_
 65	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	66	nsubj	O	_
-66	realize	realizar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	62	acl:relcl	O	_
+66	realize	realizar	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	52	acl:relcl	O	_
 67	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	68	det	O	_
 68	experimento	experimento	NOUN	_	Gender=Masc|Number=Sing	66	obj	O	_
 69	;	;	PUNCT	_	_	72	punct	O	_
 70	2.	2.	NUM	_	NumType=Card	72	parataxis	O	_
 71	-	-	PUNCT	_	_	72	punct	O	_
-72	Estudar	estudar	VERB	_	VerbForm=Inf	66	advcl	O	_
+72	Estudar	estudar	VERB	_	VerbForm=Inf	23	conj	O	_
 73	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	74	det	O	_
 74	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	72	obj	O	_
 75	com	com	ADP	_	_	76	case	O	_
@@ -181767,7 +181767,7 @@
 86	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	87	det	O	_
 87	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	84	nmod	O	_
 88	;	;	PUNCT	_	_	89	punct	O	_
-89	utilizar	utilizar	VERB	_	VerbForm=Inf	82	conj	O	_
+89	utilizar	utilizar	VERB	_	VerbForm=Inf	72	conj	O	_
 90	água	água	NOUN	_	Gender=Fem|Number=Sing	89	obj	B=FLUIDO	_
 91	destilada	destilar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	90	acl	I=FLUIDO	_
 92	e	e	CCONJ	_	_	93	cc	O	_
@@ -181775,7 +181775,7 @@
 94	com	com	ADP	_	_	95	case	O	_
 95	sal	sal	NOUN	_	Gender=Masc|Number=Sing	93	obl	B=LITOLOGIA	_
 96	para	para	SCONJ	_	_	97	mark	O	_
-97	verificar	verificar	VERB	_	VerbForm=Inf	93	advcl	O	_
+97	verificar	verificar	VERB	_	VerbForm=Inf	89	advcl	O	_
 98	possíveis	possível	ADJ	_	Gender=Fem|Number=Plur	99	amod	O	_
 99	alterações	alteração	NOUN	_	Gender=Fem|Number=Plur	97	obj	O	_
 100-101	no	_	_	_	_	_	_	_	_
@@ -181785,9 +181785,9 @@
 103	de	de	ADP	_	_	104	case	O	_
 104	limpeza	limpeza	NOUN	_	Gender=Fem|Number=Sing	102	nmod	O	_
 105	;	;	PUNCT	_	_	114	punct	O	_
-106	3.	3.	NUM	_	NumType=Card	97	parataxis	O	_
+106	3.	3.	NUM	_	NumType=Card	114	parataxis	O	_
 107	-	-	PUNCT	_	_	114	punct	O	_
-108	Paralelamente	paralelamente	ADV	_	_	106	advmod	O	_
+108	Paralelamente	paralelamente	ADV	_	_	114	advmod	O	_
 109-110	aos	_	_	_	_	_	_	_	_
 109	a	a	ADP	_	_	111	case	O	_
 110	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	111	det	O	_
@@ -181795,7 +181795,7 @@
 112	de	de	ADP	_	_	113	case	O	_
 113	repartida	repartida	NOUN	_	Gender=Fem|Number=Sing	111	acl	O	_
 114-115	sugere-se	_	_	_	_	_	_	_	_
-114	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	106	acl:relcl	O	_
+114	sugere	sugerir	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	23	acl:relcl	O	_
 115	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Sing|Person=3|PronType=Prs	114	expl	O	_
 116	sempre	sempre	ADV	_	_	117	advmod	O	_
 117	realizar	realizar	VERB	_	VerbForm=Inf	114	xcomp	O	_
@@ -181806,7 +181806,7 @@
 122	,	,	PUNCT	_	_	125	punct	O	_
 123	para	para	ADP	_	_	125	case	O	_
 124	melhor	bom	ADJ	_	Gender=Fem|Number=Sing	125	amod	O	_
-125	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	118	nmod	O	_
+125	interpretação	interpretação	NOUN	_	Gender=Fem|Number=Sing	117	obl	O	_
 126-127	dos	_	_	_	_	_	_	_	_
 126	de	de	ADP	_	_	128	case	O	_
 127	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	128	det	O	_
@@ -181814,7 +181814,7 @@
 129	;	;	PUNCT	_	_	132	punct	O	_
 130	4.	4.	NUM	_	NumType=Card	132	parataxis	O	_
 131	-	-	PUNCT	_	_	132	punct	O	_
-132	Realizar	realizar	VERB	_	VerbForm=Inf	117	conj	O	_
+132	Realizar	realizar	VERB	_	VerbForm=Inf	23	conj	O	_
 133	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	134	det	O	_
 134	estudo	estudo	NOUN	_	Gender=Masc|Number=Sing	132	obj	O	_
 135	mais	mais	ADV	_	_	136	advmod	O	_
