commit ad450c4185a5c03beff93a817b997a7963165059
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Mon May 17 03:45:07 2021 -0300

    abservar is not a word

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index 7236673..2f963d2 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -17627,7 +17627,7 @@
 # text = Observa-se que as emulsões asfálticas modificadas com vermiculita não apresentaram partículas retidas na peneira ASTM 20, mostrando a eficiência do processo de moagem realizado no moinho coloidal durante a obtenção da emulsão asfáltica..
 # sent_id = 241-20140227-MONOGRAFIA_0-208
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	11	mark	O	_
 4	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	5	det	O	_
@@ -59076,7 +59076,7 @@
 # text = Observando-se a Tabela IV.1, é possível notar que a argila apresenta 1,1 %p/p de óxido de cálcio (CaO)..
 # sent_id = 24-20150122-MONOGRAFIA_0-465
 1-2	Observando-se	_	_	_	_	_	_	O	_
-1	Abservando	abservar	VERB	_	VerbForm=Ger	8	advcl	O	_
+1	Observando	observar	VERB	_	VerbForm=Ger	8	advcl	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	expl	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	Tabela	Tabela	PROPN	_	Gender=Fem|Number=Sing	1	obj	O	_
@@ -61388,7 +61388,7 @@
 # text = Observou-se, em todas as sete argilas, rápido inchamento e em poucos minutos a total fragmentação no meio aquoso, evidenciado a significativa expansão das camadas interplanares basais..
 # sent_id = 24-20150122-MONOGRAFIA_0-541
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	,	,	PUNCT	_	_	1	punct	O	_
 4	em	em	ADP	_	_	8	case	O	_
@@ -61544,7 +61544,7 @@
 # text = Observou-se que a Argila A4 quando hidratada desfolha..
 # sent_id = 24-20150122-MONOGRAFIA_0-548
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	5	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -61669,7 +61669,7 @@
 # text = Observou-se que o poli (cloreto de dialildimetilamônio), inibiu a reatividade natural das argilas, mas em diferentes percentuais..
 # sent_id = 24-20150122-MONOGRAFIA_0-553
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	12	mark	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
@@ -61940,7 +61940,7 @@
 # text = Observou-se que N,N,N-trimetilquitosana (TMQ) inibiu a reatividade natural das argilas, mas em diferentes percentuais..
 # sent_id = 24-20150122-MONOGRAFIA_0-563
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	8	mark	O	_
 4	N,N,N-trimetilquitosana	N,N,N-trimetilquitosana	PROPN	_	Gender=Masc|Number=Sing	8	nsubj	O	_
@@ -62119,7 +62119,7 @@
 # text = Observou-se que o mesmo não foi eficiente em inibir a reatividade natural das argilas..
 # sent_id = 24-20150122-MONOGRAFIA_0-569
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	8	mark	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
@@ -62244,7 +62244,7 @@
 # text = Observou-se que a solução, não foi muito eficiente em inibir a reatividade natural das argilas..
 # sent_id = 24-20150122-MONOGRAFIA_0-575
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	10	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -71230,7 +71230,7 @@
 # text = Observou-se maior aporte psamítico para o topo da seção, até que passam a ocorrer apenas sedimentos arenosos da Fm..
 # sent_id = 283-20150917-MONOGRAFIA_0-245
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	maior	grande	ADJ	_	Gender=Masc|Number=Sing	4	amod	O	_
 4	aporte	aportar	NOUN	_	Gender=Masc|Number=Sing	1	nsubj	O	_
@@ -74120,7 +74120,7 @@
 # text = Observa-se fácies pelíticas subordinadas relacionadas com a associação de fácies descrita acima (St, Sr, Sh e Sd)..
 # sent_id = 283-20150917-MONOGRAFIA_0-351
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	1	expl	O	_
 3	fácies	fácie	ADJ	_	Gender=Fem|Number=Plur	4	amod	B=LITOLOGIA	_
 4	pelíticas	pelítica	NOUN	_	Gender=Fem|Number=Plur	1	nsubj	O	_
@@ -107720,7 +107720,7 @@
 # text = Observa-se que a obtenção dos RCEs pelo MDL quase não altera os lucros anuais do projeto, em razão da pouca quantidade e do baixo preço dos RCEs no principal mercado de emissões do mundo..
 # sent_id = 161-20150810-MONOGRAFIA_0-908
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	14	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -141358,7 +141358,7 @@
 # text = Observa-se que a estabilidade da emulsão a partir de 70% de óleo permanece inalterada durante o período de tempo analisado..
 # sent_id = 202-20140404-MONOGRAFIA_0-125
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	16	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -160311,7 +160311,7 @@
 # text = Observa-se que partir dela o depocentro da bacia sofre subsidência, separando a zona de ocorrência de sedimentos da fase rifte, pós-rifte e parte da fase drifte (horizonte
 # sent_id = 258-20160602-TESEMSC_0-428
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	12	mark	O	_
 4	partir	partir	NOUN	_	_	6	case	O	_
@@ -161871,7 +161871,7 @@
 # text = Observam-se de maneira bem definida feições como a quebra da plataforma e o depocentro da bacia, caracterizados respectivamente por uma variação brusca dos valores alinhada paralela à costa e por mínimos gravimétricos na porção central dos mapas..
 # sent_id = 258-20160602-TESEMSC_0-462
 1-2	Observam-se	_	_	_	_	_	_	O	_
-1	Abservam	abservar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observam	observar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	de	de	ADP	_	_	4	case	O	_
 4	maneira	maneira	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
@@ -177423,7 +177423,7 @@
 # text = Observa-se também efeitos físicos e químicos na água e no óleo..
 # sent_id = 77-20141209-TESEMSC_0-360
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Plur|Person=3|PronType=Prs	1	expl	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	efeitos	efeito	NOUN	_	Gender=Masc|Number=Plur	1	obj	O	_
@@ -179414,7 +179414,7 @@
 # text = Observa-se que a adição de desemulsificante reduziu consideravelmente o tempo de limpeza da linha e inclusive parece ter eliminado o transiente inicial de mistura das fases..
 # sent_id = 77-20141209-TESEMSC_0-412
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	8	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -185943,7 +185943,7 @@
 # text = Observa-se que todas as propriedades medidas atenderam aos valores limites estabelecidos pela ANP..
 # sent_id = 279-20140530-MONOGRAFIA_0-123
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	8	mark	O	_
 4	todas	todo	DET	_	Gender=Fem|Number=Plur|PronType=Tot	6	det	O	_
@@ -188511,7 +188511,7 @@
 # text = Observa-se no gráfico os aumentos de CO2 e CO, quando  diminui (ou  aumenta), observado majoritariamente nos experimentos, para um mesmo combustível..
 # sent_id = 279-20140530-MONOGRAFIA_0-211
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3-4	no	_	_	_	_	_	_	O	_
 3	em	em	ADP	_	_	5	case	O	_
@@ -199009,7 +199009,7 @@
 # text = Observa-se que a dopagem modificou a coloração do catalisador, tornando-o avermelhado e aglomerado..
 # sent_id = 271-20150810-MONOGRAFIA_0-290
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	6	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -199465,7 +199465,7 @@
 # text = Observou-se um resíduo de aproximadamente 65% a 900 °C, indicando perda de massa acumulada de 35%..
 # sent_id = 271-20150810-MONOGRAFIA_0-305
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	resíduo	resíduo	NOUN	_	Gender=Masc|Number=Sing	1	obj	O	_
@@ -199888,7 +199888,7 @@
 # text = Observou-se a presença de estruturas fibrilares, provavelmente devido a algum constituinte inorgânico. (c)
 # sent_id = 271-20150810-MONOGRAFIA_0-320
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	presença	presença	NOUN	_	Gender=Fem|Number=Sing	1	obj	O	_
@@ -200481,7 +200481,7 @@
 # text = Observou-se a presença de ácido palmítico, oleico e linoleico..
 # sent_id = 271-20150810-MONOGRAFIA_0-338
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	4	det	O	_
 4	presença	presença	NOUN	_	Gender=Fem|Number=Sing	1	nsubj	O	_
@@ -200633,7 +200633,7 @@
 # text = Observa-se também um aumento na acidez, no teor de água e no teor de metais como, Na, K, Ca e Mg, esta diferença nos resultados comparando com o óleo comercial podem ser explicados por causa dos alimentos que são utilizados durante as frituras, pois estes alimentos são temperados e com isso aumenta a quantidade de metais no óleo e também o alimento pode estar congelado influenciando no aumento do teor de água..
 # sent_id = 271-20150810-MONOGRAFIA_0-343
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
@@ -202156,7 +202156,7 @@
 # text = Observou-se também que o teor máximo foi alcançado (maior que 90%) após 6 h de reação para 5, 7 e 10% de catalisador, assim como o óleo residual..
 # sent_id = 271-20150810-MONOGRAFIA_0-392
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	que	que	SCONJ	_	_	9	mark	O	_
@@ -202365,7 +202365,7 @@
 # text = Observa-se também que apenas com 4 horas surgem sinais de formação do biodiesel em 3,6 ppm..
 # sent_id = 271-20150810-MONOGRAFIA_0-397
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	que	que	SCONJ	_	_	9	mark	O	_
@@ -202760,7 +202760,7 @@
 # text = Observa-se na Tabela 5 que todas as amostras apresentaram, massa específica, viscosidade, ponto de entupimento de filtro a frio, corrosividade ao cobre, índice de acidez e teor de éster, dentro dos limites previstos na especificação brasileira..
 # sent_id = 271-20150810-MONOGRAFIA_0-411
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3-4	na	_	_	_	_	_	_	O	_
 3	em	em	ADP	_	_	5	case	O	_
@@ -213470,7 +213470,7 @@
 # text = Observou-se também que a remoção de sais de ferro de amostras oleosas interfere na medida do TOC e a utilização do xileno como poluente modelo demonstrou ser apropriada para os estudos experimentais dos processos de tratamento e os dados experimentais obtidos da flotação em coluna apresentaram comportamento cinético de primeira ordem..
 # sent_id = 391-20160914-MONOGRAFIA_0-170
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	que	que	SCONJ	_	_	14	mark	O	_
@@ -228855,7 +228855,7 @@
 # text = Observa-se claramente que não ocorre adsorção irreversível de monóxido de carbono nas zeólitas suporte (Figura 21), pois a primeira isoterma praticamente coincide com a segunda..
 # sent_id = 25-20150123-MONOGRAFIA_0-421
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	claramente	claramente	ADV	_	_	1	advmod	O	_
 4	que	que	SCONJ	_	_	6	mark	O	_
@@ -259404,7 +259404,7 @@
 # text = Observa-se que a falha na borda sul controla um depocentro local em que o embasamento mostra forte basculamento, atingindo cerca de 800 milissegundos em tempo duplo (800 m de profundidade, utilizando-se a velocidade média sedimentar de 2.000 m/s), e com rápido acunhamento na direção do gráben interno, onde novamente é falhado..
 # sent_id = 20-20140904-TESEDSC_0-713
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	10	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -272373,7 +272373,7 @@
 # text = Observa-se um proeminente alinhamento no limite SW do trecho deslocado da faixa magmática que tem continuidade para NW e está representado no mapa de interpretação por um enxame de traços paralelos..
 # sent_id = 20-20140904-TESEDSC_0-1033
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
 4	proeminente	proeminente	ADJ	_	Gender=Masc|Number=Sing	5	amod	O	_
@@ -291933,7 +291933,7 @@
 # text = Observa-se que, quando o reboco é formado, o fluido invade menos a formação quando comparado com o poço sem a presença do reboco..
 # sent_id = 119-20141209-TESEMSC_0-165
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	13	mark	O	_
 4	,	,	PUNCT	_	_	9	punct	O	_
@@ -297745,7 +297745,7 @@
 # text = Observa-se também que as curvas de fluxo se convergem sendo a distância entre as curvas maior em A do que em B..
 # sent_id = 119-20141209-TESEMSC_0-358
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	também	também	ADV	_	_	1	advmod	O	_
 4	que	que	SCONJ	_	_	10	mark	O	_
@@ -304892,7 +304892,7 @@
 # text = Observa-se que os pontos iniciais e finais das curvas da viscosidade versus taxa de cisalhamento para a PHPA e para a GX ficaram próximos, exceto para a curva do fluido preparado com PHPA sem argila..
 # sent_id = 119-20141209-TESEMSC_0-553
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	15	mark	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -306595,7 +306595,7 @@
 # text = Observa-se pela Tabela 4.1 que, para todas as amostras, a permeabilidade absoluta ao óleo, obtida com a amostra 100% saturada, foi maior que a permeabilidade absoluta ao gás obtida com a amostra seca..
 # sent_id = 119-20141209-TESEMSC_0-607
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	1	expl	O	_
 3-4	pela	_	_	_	_	_	_	O	_
 3	por	por	ADP	_	_	5	case	O	_
@@ -307856,7 +307856,7 @@
 # text = Observa-se que a injeção reversa de óleo tende a aumentar a vazão, mostrando que o óleo está expulsando tanto o polímero quanto as partículas de argila presentes na amostra, desbloqueando os poros até então bloqueados..
 # sent_id = 119-20141209-TESEMSC_0-636
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	9	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -308046,7 +308046,7 @@
 # text = Observa-se que a vazão de óleo para o fluido que não continha argila ficou muito acima das demais e demandou, também, um menor tempo de teste..
 # sent_id = 119-20141209-TESEMSC_0-641
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	15	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -308894,7 +308894,7 @@
 # text = Observou-se que ocorre um processo de filtração do fluido na amostra de arenito, visto que, se comparado com o fluido injetado, um filtrado translúcido, apenas amarelado pelo fato de ter sido adicionado o iodeto de sódio (NaI), é produzido..
 # sent_id = 119-20141209-TESEMSC_0-660
 1-2	Observou-se	_	_	_	_	_	_	O	_
-1	Abservou	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
+1	Observou	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	4	mark	O	_
 4	ocorre	ocorrer	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	ccomp	O	_
@@ -309086,7 +309086,7 @@
 # text = Observa-se que os filtrados não apresentam as mesmas características dos fluidos injetados, principalmente para baixas taxas de cisalhamento, pois observa-se que a viscosidade, antes elevada, reduziu consideravelmente..
 # sent_id = 119-20141209-TESEMSC_0-665
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	7	mark	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -310962,7 +310962,7 @@
 # text = Observa-se que a taxa de invasão de filtrado para os fluidos preparados com GX foi mais reduzida em todos os experimentos..
 # sent_id = 119-20141209-TESEMSC_0-708
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	18	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -311119,7 +311119,7 @@
 # text = Observa-se que, assim como nos experimentos realizados com PHPA, para a GX, a concentração em massa de argila no fluido também teve influência no tamponamento dos poros, pois estes foram bloqueados mais rapidamente pelo fluido que continha a maior concentração de argila, e à medida que esta concentração foi reduzida, um maior VPI foi necessário para obstruir os poros das amostras..
 # sent_id = 119-20141209-TESEMSC_0-711
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	28	mark	O	_
 4	,	,	PUNCT	_	_	1	punct	O	_
@@ -311290,7 +311290,7 @@
 # text = Observa-se que a vazão de produção de óleo tende a aumentar com a injeção reversa de óleo, porém, este aumento possui um limite, visto que as curvas tendem a se horizontalizar..
 # sent_id = 119-20141209-TESEMSC_0-715
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	10	mark	O	_
 4	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	5	det	O	_
@@ -311463,7 +311463,7 @@
 # text = Observa-se que os retornos das vazões relativas de óleo para os fluidos preparados com GX foram aproximadamente os mesmos, exceto para o fluido com 2,0% de argila..
 # sent_id = 119-20141209-TESEMSC_0-719
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	21	mark	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -311988,7 +311988,7 @@
 # text = Observa-se que os resultados de retorno da vazão relativa de óleo foram muito próximos entre si, destacando-se o fluido com PHPA sem argila e o fluido com GX e 2,0% de argila..
 # sent_id = 119-20141209-TESEMSC_0-732
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	16	mark	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
@@ -312149,7 +312149,7 @@
 # text = Observa-se que o valor da saturação remanescente de óleo ao final da invasão #KH>LMNOPãQ$ para o fluido sem a presença de argila foi muito baixo (9,82%) e percebe-se, também, que à medida que a concentração de argila aumentou a saturação de óleo remanescente na amostra ao final da invasão ( também foi maior..
 # sent_id = 119-20141209-TESEMSC_0-736
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	30	mark	O	_
 4	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	5	det	O	_
@@ -313130,7 +313130,7 @@
 # text = Observa-se que os retornos da razão de produtividade para as amostras invadidas por fluidos de perfuração preparados com GX foram muito baixos..
 # sent_id = 119-20141209-TESEMSC_0-761
 1-2	Observa-se	_	_	_	_	_	_	O	_
-1	Abserva	abservar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+1	Observa	observar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 2	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	1	nsubj	O	_
 3	que	que	SCONJ	_	_	24	mark	O	_
 4	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	5	det	O	_
